{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "a8e006c9",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/justpqa/advpai/venv/lib/python3.10/site-packages/pypdf/_crypt_providers/_cryptography.py:32: CryptographyDeprecationWarning: ARC4 has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.ARC4 and will be removed from cryptography.hazmat.primitives.ciphers.algorithms in 48.0.0.\n",
      "  from cryptography.hazmat.primitives.ciphers.algorithms import AES, ARC4\n",
      "/Users/justpqa/advpai/venv/lib/python3.10/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "from copy import deepcopy\n",
    "# import xml.etree.ElementTree as ET\n",
    "# from rich.tree import Tree\n",
    "# from rich import print as rprint\n",
    "import io\n",
    "from typing import List, Union, Tuple\n",
    "# import pdfplumber\n",
    "# import fitz \n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "import requests\n",
    "# import xmltodict\n",
    "import re\n",
    "from pypdf import PdfReader, PdfWriter\n",
    "from difflib import SequenceMatcher\n",
    "from docling.datamodel.base_models import InputFormat\n",
    "from docling.datamodel.pipeline_options import PdfPipelineOptions, TableFormerMode\n",
    "from docling.document_converter import DocumentConverter, PdfFormatOption\n",
    "from docling.datamodel.document import DocumentStream\n",
    "from sentence_transformers import SentenceTransformer, util, CrossEncoder\n",
    "import torch\n",
    "import torch.nn.functional as F\n",
    "from transformers import AutoTokenizer, AutoModel, AutoModelForCausalLM"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "29cd6002",
   "metadata": {},
   "source": [
    "Try to look at the pdf"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2dfd7240",
   "metadata": {},
   "source": [
    "Try to look at the website of the paper instead"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5cc6bc82",
   "metadata": {},
   "source": [
    "Try to use Docling"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "870cbbf5",
   "metadata": {},
   "outputs": [],
   "source": [
    "# first we need to know the number of col of a table to make sure we try the right orientation with docling\n",
    "def extract_tables_num_col_lst_from_pmc(pmcid):\n",
    "    url = f\"https://www.ncbi.nlm.nih.gov/research/bionlp/RESTful/pmcoa.cgi/BioC_json/{pmcid}/unicode\"\n",
    "\n",
    "    response = requests.get(url)\n",
    "    tables_num_col_lst = []\n",
    "    if response.status_code == 200:\n",
    "        print(response.text)\n",
    "        data = response.json()\n",
    "        for d in data:\n",
    "            doc = d[\"documents\"]\n",
    "            for p in doc:\n",
    "                passage = p[\"passages\"]\n",
    "                for item in passage:\n",
    "                    if item.get(\"infons\", \"\").get(\"type\", \"\").lower() == \"table\" and \"text\" in item:\n",
    "                        table_str = item[\"text\"]\n",
    "                        num_col = 0\n",
    "                        for row in table_str.split(\"\\t \\t\"):\n",
    "                            row_lst = row.split(\"\\t\")\n",
    "                            num_col = max(num_col, len(row_lst))\n",
    "                        tables_num_col_lst.append(num_col)\n",
    "        print(f\"Successfully retrieve number of columns\")\n",
    "    else:\n",
    "        print(f\"Failed to retrieve number of columns: {response.status_code}\")\n",
    "        \n",
    "    return tables_num_col_lst"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "a478c37c",
   "metadata": {},
   "outputs": [],
   "source": [
    "def clean_cell(val):\n",
    "    tag_pattern = r'\\s[a-z]$'\n",
    "    if isinstance(val, str):\n",
    "        return re.sub(tag_pattern, '', val)\n",
    "    return val\n",
    "\n",
    "def clean_headers(df):\n",
    "    tag_pattern = r'\\s[a-z]$'\n",
    "    new_cols = []\n",
    "    seen = {}\n",
    "    for col in df.columns:\n",
    "        if pd.isna(col):\n",
    "            new_cols.append(\"\")\n",
    "        else:\n",
    "            # 1. Apply the regex to the name string\n",
    "            clean_name = re.sub(tag_pattern, '', str(col))\n",
    "            # 2. Handle duplicates (e.g., if 'Price a' and 'Price b' both become 'Price')\n",
    "            if clean_name in seen:\n",
    "                seen[clean_name] += 1\n",
    "                clean_name = f\"{clean_name}_{seen[clean_name]}\"\n",
    "            else:\n",
    "                seen[clean_name] = 0\n",
    "            new_cols.append(clean_name)\n",
    "    df.columns = new_cols\n",
    "    return df\n",
    "\n",
    "def extract_tables_lst_from_pdf_and_num_col(file_name, tables_num_col_lst):\n",
    "    reader = PdfReader(file_name)\n",
    "    options = PdfPipelineOptions()\n",
    "    options.table_structure_options.mode = TableFormerMode.ACCURATE\n",
    "    converter = DocumentConverter(\n",
    "        format_options={\n",
    "            InputFormat.PDF: PdfFormatOption(pipeline_options=options)\n",
    "        }\n",
    "    )\n",
    "    df_lst = []\n",
    "    row_pattern = r\"\\b(?:rs|s)\\d+\\S*\"\n",
    "\n",
    "    page_num = 1\n",
    "    while len(df_lst) < len(tables_num_col_lst) and page_num <= len(reader.pages):\n",
    "        for angle in [0, 90]: # Try normal, then try rotated\n",
    "            writer = PdfWriter()\n",
    "            page = reader.pages[page_num - 1]\n",
    "            \n",
    "            if angle != 0:\n",
    "                page.rotate(angle)\n",
    "            \n",
    "            writer.add_page(page)\n",
    "            \n",
    "            # Convert just this one page\n",
    "            pdf_buffer = io.BytesIO()\n",
    "            writer.write(pdf_buffer)\n",
    "            pdf_buffer.seek(0)\n",
    "            \n",
    "            doc_stream = DocumentStream(name=f\"page_{page_num}.pdf\", stream=pdf_buffer)\n",
    "            result = converter.convert(doc_stream)\n",
    "            \n",
    "            # Check if this rotation produced valid table rows\n",
    "            temp_dfs = []\n",
    "            for table in result.document.tables:\n",
    "                df = table.export_to_dataframe()\n",
    "                # check if table is empty\n",
    "                if (not df.empty):\n",
    "                    # Case 1: continue from previous table\n",
    "                    if len(df_lst) > 0 and df.shape[1] == df_lst[-1].shape[1]:\n",
    "                        # extra filters for tables that are snp related, we need to remove rows that do not have snp id\n",
    "                        # often are separation between sections\n",
    "                        for col in df.columns:\n",
    "                            # first modify \"\" -> nan\n",
    "                            df[col] = df[col].replace(r'^\\s*$', np.nan, regex=True).ffill()\n",
    "                        df[\"valid_row\"] = df.apply(lambda x: any(re.search(row_pattern, str(c)) for c in x), axis=1)\n",
    "                        df = df[df[\"valid_row\"]].drop(\"valid_row\", axis=1).reset_index().drop(\"index\", axis = 1)\n",
    "                        # some cell have the tag the end (often include a space and a small letter)\n",
    "                        df = df.map(clean_cell) \n",
    "                        df = clean_headers(df) \n",
    "                        if df_lst[-1].columns.equals(df.columns):\n",
    "                            df_lst[-1] = pd.concat([df_lst[-1], df], ignore_index = True)\n",
    "                        elif df.shape[1] == tables_num_col_lst[len(df_lst) + len(temp_dfs)]:\n",
    "                            # fail that test => add to temp since this is a new table\n",
    "                            temp_dfs.append(df)\n",
    "                    # Case 2: new table\n",
    "                    elif (len(df_lst) + len(temp_dfs)) < len(tables_num_col_lst) and df.shape[1] == tables_num_col_lst[len(df_lst) + len(temp_dfs)]: \n",
    "                        # extra filters for tables that are snp related, we need to remove rows that do not have snp id\n",
    "                        # often are separation between sections\n",
    "                        for col in df.columns:\n",
    "                            # first modify \"\" -> nan\n",
    "                            df[col] = df[col].replace(r'^\\s*$', np.nan, regex=True).ffill()\n",
    "                        df[\"valid_row\"] = df.apply(lambda x: any(re.search(row_pattern, str(c)) for c in x), axis=1)\n",
    "                        df = df[df[\"valid_row\"]].drop(\"valid_row\", axis=1).reset_index().drop(\"index\", axis = 1)\n",
    "                        # some cell have the tag the end (often include a space and a small letter)\n",
    "                        df = df.map(clean_cell) \n",
    "                        df = clean_headers(df)             \n",
    "                        temp_dfs.append(df)\n",
    "            if temp_dfs:\n",
    "                df_lst.extend(temp_dfs)\n",
    "                break\n",
    "            \n",
    "        page_num += 1\n",
    "\n",
    "    return df_lst"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "99bd8b1b",
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_tables_lst_from_paper(pmcid, file_name, table_inx_to_extract=[]):\n",
    "    tables_num_col_lst = extract_tables_num_col_lst_from_pmc(pmcid)\n",
    "    if len(table_inx_to_extract) > 0:\n",
    "        tables_num_col_lst = [tables_num_col_lst[i] for i in table_inx_to_extract]\n",
    "    df_lst = extract_tables_lst_from_pdf_and_num_col(file_name, tables_num_col_lst)\n",
    "    return df_lst"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "6513d3c2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{\"source\": \"PMC\", \"date\": \"20231103\", \"key\": \"pmc.key\", \"infons\": {}, \"documents\": [{\"id\": \"10497850\", \"infons\": {\"license\": \"CC BY\"}, \"passages\": [{\"offset\": 0, \"infons\": {\"alt-title\": \"Dato et al.\", \"article-id_doi\": \"10.1111/acel.13938\", \"article-id_other\": \"ACE-23-0275.R1\", \"article-id_pmc\": \"10497850\", \"article-id_pmid\": \"37621137\", \"article-id_publisher-id\": \"ACEL13938\", \"elocation-id\": \"e13938\", \"fn\": \"Serena Dato and Francesco De Rango\\u2014Co\\u2010first authors. Giuseppina Rose and Valerio Napolioni\\u2014Co\\u2010last authors.\", \"issue\": \"9\", \"kwd\": \"Alzheimer's Disease epistasis gene\\u2013gene interaction IGF1 longevity polymorphism\", \"license\": \"This is an open access article under the terms of the http://creativecommons.org/licenses/by/4.0/ License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.\", \"name_0\": \"surname:Dato;given-names:Serena\", \"name_1\": \"surname:De\\u00a0Rango;given-names:Francesco\", \"name_2\": \"surname:Crocco;given-names:Paolina\", \"name_3\": \"surname:Pallotti;given-names:Stefano\", \"name_4\": \"surname:Belloy;given-names:Michael E.\", \"name_5\": \"surname:Le\\u00a0Guen;given-names:Yann\", \"name_6\": \"surname:Greicius;given-names:Michael D.\", \"name_7\": \"surname:Passarino;given-names:Giuseppe\", \"name_8\": \"surname:Rose;given-names:Giuseppina\", \"name_9\": \"surname:Napolioni;given-names:Valerio\", \"notes\": \"\\n\\n, , , , , , , , , &  (). . , , . \\n\\n\", \"pub-id_doi\": \"10.1111/acel.13938\", \"section_type\": \"TITLE\", \"source\": \"Aging Cell\", \"type\": \"front\", \"volume\": \"22\", \"year\": \"2023\"}, \"text\": \"Sex\\u2010 and APOE\\u2010specific genetic risk factors for late\\u2010onset Alzheimer's disease: Evidence from gene\\u2013gene interaction of longevity\\u2010related loci\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 152, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract_title_1\"}, \"text\": \"Abstract\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 161, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Advanced age is the largest risk factor for late\\u2010onset Alzheimer's disease (LOAD), a disease in which susceptibility correlates to almost all hallmarks of aging. Shared genetic signatures between LOAD and longevity were frequently hypothesized, likely characterized by distinctive epistatic and pleiotropic interactions. Here, we applied a multidimensional reduction approach to detect gene\\u2013gene interactions affecting LOAD in a large dataset of genomic variants harbored by genes in the insulin/IGF1 signaling, DNA repair, and oxidative stress pathways, previously investigated in human longevity. The dataset was generated from a collection of publicly available Genome Wide Association Studies, comprising a total of 2,469 gene variants genotyped in 20,766 subjects of Northwestern European ancestry (11,038 LOAD cases and 9,728 controls). The stratified analysis according to APOE*4 status and sex corroborated evidence that pathways leading to longevity also contribute to LOAD. Among the significantly interacting genes, PTPN1, TXNRD1, and IGF1R were already found enriched in gene\\u2013gene interactions affecting survival to old age. Furthermore, interacting variants associated with LOAD in a sex\\u2010 and APOE\\u2010specific way. Indeed, while in APOE*4 female carriers we found several inter\\u2010pathway interactions, no significant epistasis was found in APOE*4 negative females; conversely, in males, significant intra\\u2010 and inter\\u2010pathways epistasis emerged according to APOE*4 status. These findings suggest that interactions of risk factors may drive different trajectories of cognitive aging. Beyond helping to disentangle the genetic architecture of LOAD, such knowledge may improve precision in predicting the risk of dementia and enable effective sex\\u2010 and APOE\\u2010stratified preventive and therapeutic interventions for LOAD.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2003, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Genetic studies over the last few years, have changed the perception of late\\u2010onset Alzheimer's disease (LOAD), highlighting its polygenic nature with multiple susceptibility genetic loci. We applied a multidimensional reduction approach to detect gene\\u2013gene interactions affecting LOAD by testing variants harbored by genes previously investigated in human longevity. This study suggests that longevity loci may also drive LOAD neuropathology through APOE\\u2010 and sex\\u2010related specific gene\\u2013gene interactions. \", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2519, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"title\"}, \"text\": \"Abbreviations\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2533, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"AD\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2536, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2556, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"APOE\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2561, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"apolipoprotein E\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2578, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"CV\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2581, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"cross\\u2010validation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2600, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"GWAS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2605, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"genome wide association study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2635, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"IBD\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2639, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"identity\\u2010by\\u2010descent\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2663, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"IG\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2666, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"information gain\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2683, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"IGF1\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2688, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"insulin growth factor 1\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2712, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"LD\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2715, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"linkage disequilibrium\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2738, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"LOAD\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2743, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"late\\u2010onset alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2776, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"MAF\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2780, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"minor allele frequency\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2803, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"MDR\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2807, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"multifactor dimensionality reduction\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2844, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"QTL\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2848, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"quantitative trait locus\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2873, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"SNP\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2877, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"single nucleotide polymorphism\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2908, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"title_1\"}, \"text\": \"INTRODUCTION\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2921, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Alzheimer's disease (AD) is the most prevalent disease among people over 85\\u2009years of age in western countries, posing a significant challenge to public health systems around the world. Most AD cases are sporadic or late onset (LOAD; >65\\u2009years of age), with biological measures of disease being detectable as early as 20\\u2009years before the first cognitive symptoms are observed (Jagust,\\u00a0). Decades of research have shed light on the neuropathological changes happening in the AD brain, and its complex etiology (Long & Holtzman,\\u00a0), characterized by sex differences in several aspects of the disease, including its onset and progression, and the effects of APOE*4 genotype, the strongest common genetic risk factor for LOAD (Nebel et al.,\\u00a0). A current challenge is to clarify the contribution of genetic, epigenetic, and environmental factors in the multifactorial nature of LOAD, which shows a heritability of 58%\\u201379%, with a large fraction attributable to the APOE locus. Genetic studies over the last few years, particularly coming from genome\\u2010wide association studies (GWAS) and large sequencing projects, have changed the perception of LOAD, highlighting its polygenic nature with multiple susceptibility genetic loci (Andrews et al.,\\u00a0). Also, functional genomic analyses pointed out that common LOAD risk variants operate in complex networks of genetic and metabolic interactions, regulated by \\u201chub\\u201d genes and \\u201cperipheral master regulators\\u201d (Gui et al.,\\u00a0). In the interactome network of the cell, each variant may show different effects (either in magnitude or in direction) on disease onset, in relation to alleles at other loci (Ridge et al.,\\u00a0). These epistatic effects may contribute substantially to the variation in disease susceptibility; that is, people carrying risk factors for LOAD but resilient to the disease, as well as people carrying the risk allele APOE*4 who live into their 90s without developing dementia.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 4872, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"While advances in technologies and increased availability of multi\\u2010omics data have expanded our knowledge about LOAD genetic architecture, many risk variants remain to be identified (Andrews et al.,\\u00a0). Such variants may be found in the underlying processes leading to LOAD, such as inflammation (Akiyama et al.,\\u00a0), apoptosis (Behl,\\u00a0), stress response (Iatrou et al.,\\u00a0), and mitochondrial decay (Kwong et al.,\\u00a0). Interestingly, many of these mechanisms are common to human longevity, suggesting that the search of susceptibility factors could be enhanced by testing genes belonging to pathways influencing longevity and survival. As a proof\\u2010of\\u2010concept alongside APOE, which is the locus consistently associated with longevity across different populations (Abondio et al.,\\u00a0), several other LOAD\\u2010associated loci were found to affect the human lifespan (Tesi et al.,\\u00a0). Moreover, several studies reported LOAD risk variants that are associated with both AD and longevity (Bacalini et al.,\\u00a0; Dato et al.,\\u00a0; Tesi et al.,\\u00a0), emphasizing the importance of studying the pleiotropic effects and epistatic interactions of genetic variants.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 6019, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Thus, searching for epistatic interactions between variants in genomic regions selected for their association with longevity may help to unravel some of the missing genetic variance of LOAD. To test this hypothesis, we leveraged a large collection of publicly available LOAD GWAS data and conducted gene\\u2013gene interaction analysis to find meaningful associations. Genomic regions were prioritized based on previous work by Dato and coworkers (Dato et al.,\\u00a0), who analyzed the joint effect on longevity of SNPs belonging to three candidate pathways, the insulin/insulin\\u2010like growth factor signaling (IIS), DNA repair, and stress response, respectively. These pathways were chosen as they regulate fundamental biological processes consistently recognized among the most relevant hallmarks of aging from model organisms to humans (Kuningas et al.,\\u00a0) and confirmed to have an important role in human longevity in large cohort studies (Deelen et al.,\\u00a0).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 6974, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"The analyzed dataset comprised genotypes for a total of 2469 gene variants from 20,766 subjects (11,038 LOAD cases and 9728 controls), belonging to several LOAD cohorts of Northwestern European ancestry. As we stated above growing evidence indicates sex\\u2010specific patterns of disease manifestation and sex\\u2010dependent effects of APOE on LOAD risk. Yet, beyond APOE*4, other genetic risk factors have been found that display sex\\u2010specific effects on LOAD, and the interplay between sex and the APOE allele has been also explored (Fan et al.,\\u00a0). In addition, sex\\u2010specific DNA methylation changes in LOAD pathology have been observed (Zhang et al.,\\u00a0). On the other hand, also the sex differences in genetic associations with longevity are remarkable (Zeng et al.,\\u00a0). Based on this evidence, we reasoned that stratifying genetic association analysis by sex and APOE status may facilitate the identification of sex\\u2010specific genetic risk loci and ultimately contribute to the understanding of disease heterogeneity between men and women.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 8015, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_1\"}, \"text\": \"MATERIALS AND METHODS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 8037, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Demographics & study datasets\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 8067, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Fifteen late\\u2010Onset Alzheimer's Disease (LOAD) GWAS datasets were obtained from publicly available data repositories (Table\\u00a01). Genotyping was performed using various high\\u2010density single\\u2010nucleotide variant microarrays across cohorts. Participants or their caregivers provided written informed consent in the original studies. The current study protocol was granted an exemption by the Stanford University institutional review board because the analyses were carried out on deidentified, off\\u2010the\\u2010shelf data.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 8584, \"infons\": {\"file\": \"acel13938-tbl-0001.xml\", \"id\": \"acel13938-tbl-0001\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Demographic information for individuals in the analysis dataset.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 8649, \"infons\": {\"file\": \"acel13938-tbl-0001.xml\", \"id\": \"acel13938-tbl-0001\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"hsides\\\" rules=\\\"groups\\\"><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><thead valign=\\\"bottom\\\"><tr style=\\\"border-bottom:solid 1px #000000\\\"><th align=\\\"left\\\" style=\\\"border-bottom:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/><th align=\\\"left\\\" style=\\\"border-bottom:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/><th align=\\\"left\\\" colspan=\\\"2\\\" style=\\\"border-bottom:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\">LOAD</th><th align=\\\"left\\\" colspan=\\\"2\\\" style=\\\"border-bottom:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\">Controls</th><th align=\\\"left\\\" colspan=\\\"2\\\" style=\\\"border-bottom:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\">Mean age LOAD (SD)</th><th align=\\\"left\\\" colspan=\\\"2\\\" style=\\\"border-bottom:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\">Mean age controls (SD)</th><th align=\\\"left\\\" colspan=\\\"2\\\" style=\\\"border-bottom:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">APOE</italic>*4 LOAD</th><th align=\\\"left\\\" colspan=\\\"2\\\" style=\\\"border-bottom:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">APOE</italic>*4 controls</th></tr><tr style=\\\"border-bottom:solid 1px #000000\\\"><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Cohort</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">N</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">F</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">M</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">F</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">M</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">F</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">M</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">F</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">M</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">F</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">M</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">F</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">M</th></tr></thead><tbody valign=\\\"top\\\"><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ACT</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2,115</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">343</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">190</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">872</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">710</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">81.7 (\\u00b16.4)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">80.4 (\\u00b16.4)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">82.5 (\\u00b16.1)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">81.7 (\\u00b15.9)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">149 (43.4%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">91 (47.9%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">201 (23.1%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">137 (19.3%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ADCC</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6,427</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2,171</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,800</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,561</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">895</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">74.1 (\\u00b18.0)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">72.5 (\\u00b17.2)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">79.5 (\\u00b19.0)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">80.1 (\\u00b18.6)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,411 (65.0%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,201 (66.7%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">389 (24.9%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">233 (26.3%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ADDNEUROMED</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">239</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">77</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">47</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">64</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">51</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.7 (\\u00b17.7)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">74.2 (\\u00b16)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">78.5 (\\u00b16.6)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">78.5 (\\u00b15.8)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">48 (62.3%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">28 (59.6%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">17 (26.6%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">15 (29.4%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ADNI</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">749</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">184</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">247</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">163</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">155</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">74.3 (\\u00b16.6)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">75.8 (\\u00b16.6)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">78.7 (\\u00b16.4)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">79.6 (\\u00b17.4)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">128 (69.6%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">168 (68.0%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">46 (28.2%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">42 (27.1%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">GenADA</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,375</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">400</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">286</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">436</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">253</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">74.0 (\\u00b17.3)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.9 (\\u00b16.2)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.9 (\\u00b17.2)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">75.0 (\\u00b16.9)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">257 (64.3%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">184 (64.3%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">100 (22.9%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">65 (25.7%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">HBTRC</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">302</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">123</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">100</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">22</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">57</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.9 (\\u00b15.5)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">70.3 (\\u00b16.1)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">70.3 (\\u00b16.9)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">68.9 (\\u00b16.8)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">67 (54.5%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">52 (52.0%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5 (22.7%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">23 (40.4%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NIA\\u2010LOAD</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,544</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">559</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">311</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">421</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">253</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">72.7 (\\u00b16.8)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">71.9 (\\u00b16.7)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">75.3 (\\u00b19.4)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">75.8 (\\u00b18.6)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">414 (74.1%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">236 (75.9%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">133 (31.6%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">83 (32.8%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">MAYO</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,730</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">432</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">292</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">533</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">473</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.8 (\\u00b15.0)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.9 (\\u00b15.0)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.7 (\\u00b14.5)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.4 (\\u00b14.3)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">286 (66.2%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">192 (65.8%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">153 (28.7%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">134 (28.3%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">MIRAGE</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">466</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">159</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">97</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">117</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">93</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">70.6 (\\u00b16.4)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">70.8 (\\u00b17.0)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">71.5 (\\u00b17.6)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">72.2 (\\u00b17.9)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">106 (66.7%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">59 (60.8%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">42 (35.9%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">37 (39.8%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">OHSU</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">379</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">114</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">64</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">108</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">93</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">86.9 (\\u00b16.3)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">84.6 (\\u00b15.1)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">87.7 (\\u00b17.4)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">86.5 (\\u00b17.3)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">39 (34.2%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">31 (48.4%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">23 (21.3%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">24 (25.8%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ROSMAP</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,131</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">468</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">180</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">356</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">127</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">84.3 (\\u00b16.6)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">83.8 (\\u00b16.5)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">85.8 (\\u00b17.1)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">86.2 (\\u00b16.8)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">160 (34.2%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">65 (36.1%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">74 (20.8%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">18 (14.2%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">TGEN</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,012</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">415</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">205</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">188</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">204</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">74.0 (\\u00b16.7)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">72.2 (\\u00b16.6)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">82.5 (\\u00b18.8)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">79.3 (\\u00b18.5)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">277 (66.8%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">132 (64.4%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">39 (20.7%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">43 (21.1%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">UPITT</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,667</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">627</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">356</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">437</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">247</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.2 (\\u00b16.8)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.0 (\\u00b16.5)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">75.9 (\\u00b16.3)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">75.2 (\\u00b15.9)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">365 (58.2%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">214 (60.1%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">84 (19.2%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">51 (20.7%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">UVM</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,257</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">354</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">193</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">444</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">266</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.6 (\\u00b17.7)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">71.5 (\\u00b17.3)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.4 (\\u00b17.6)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">74.2 (\\u00b17.4)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">225 (63.6%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">134 (69.4%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">111 (25.0%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">63 (23.7%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">WASHU</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">373</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">138</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">106</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">86</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">43</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">76.6 (\\u00b19.5)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">75.3 (\\u00b18.3)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">77.6 (\\u00b19.9)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">76.5 (\\u00b16.6)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">80 (58.0%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">55 (51.9%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">24 (27.9%)</td><td align=\\\"char\\\" char=\\\"(\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">12 (27.9%)</td></tr><tr><td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Total</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">20,766</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6,564</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">4,474</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5,808</td><td align=\\\"right\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">3,920</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/></tr></tbody></table>\\n\"}, \"text\": \"\\t\\tLOAD\\tControls\\tMean age LOAD (SD)\\tMean age controls (SD)\\tAPOE*4 LOAD\\tAPOE*4 controls\\t \\tCohort\\tN\\tF\\tM\\tF\\tM\\tF\\tM\\tF\\tM\\tF\\tM\\tF\\tM\\t \\tACT\\t2,115\\t343\\t190\\t872\\t710\\t81.7 (\\u00b16.4)\\t80.4 (\\u00b16.4)\\t82.5 (\\u00b16.1)\\t81.7 (\\u00b15.9)\\t149 (43.4%)\\t91 (47.9%)\\t201 (23.1%)\\t137 (19.3%)\\t \\tADCC\\t6,427\\t2,171\\t1,800\\t1,561\\t895\\t74.1 (\\u00b18.0)\\t72.5 (\\u00b17.2)\\t79.5 (\\u00b19.0)\\t80.1 (\\u00b18.6)\\t1,411 (65.0%)\\t1,201 (66.7%)\\t389 (24.9%)\\t233 (26.3%)\\t \\tADDNEUROMED\\t239\\t77\\t47\\t64\\t51\\t73.7 (\\u00b17.7)\\t74.2 (\\u00b16)\\t78.5 (\\u00b16.6)\\t78.5 (\\u00b15.8)\\t48 (62.3%)\\t28 (59.6%)\\t17 (26.6%)\\t15 (29.4%)\\t \\tADNI\\t749\\t184\\t247\\t163\\t155\\t74.3 (\\u00b16.6)\\t75.8 (\\u00b16.6)\\t78.7 (\\u00b16.4)\\t79.6 (\\u00b17.4)\\t128 (69.6%)\\t168 (68.0%)\\t46 (28.2%)\\t42 (27.1%)\\t \\tGenADA\\t1,375\\t400\\t286\\t436\\t253\\t74.0 (\\u00b17.3)\\t73.9 (\\u00b16.2)\\t73.9 (\\u00b17.2)\\t75.0 (\\u00b16.9)\\t257 (64.3%)\\t184 (64.3%)\\t100 (22.9%)\\t65 (25.7%)\\t \\tHBTRC\\t302\\t123\\t100\\t22\\t57\\t73.9 (\\u00b15.5)\\t70.3 (\\u00b16.1)\\t70.3 (\\u00b16.9)\\t68.9 (\\u00b16.8)\\t67 (54.5%)\\t52 (52.0%)\\t5 (22.7%)\\t23 (40.4%)\\t \\tNIA\\u2010LOAD\\t1,544\\t559\\t311\\t421\\t253\\t72.7 (\\u00b16.8)\\t71.9 (\\u00b16.7)\\t75.3 (\\u00b19.4)\\t75.8 (\\u00b18.6)\\t414 (74.1%)\\t236 (75.9%)\\t133 (31.6%)\\t83 (32.8%)\\t \\tMAYO\\t1,730\\t432\\t292\\t533\\t473\\t73.8 (\\u00b15.0)\\t73.9 (\\u00b15.0)\\t73.7 (\\u00b14.5)\\t73.4 (\\u00b14.3)\\t286 (66.2%)\\t192 (65.8%)\\t153 (28.7%)\\t134 (28.3%)\\t \\tMIRAGE\\t466\\t159\\t97\\t117\\t93\\t70.6 (\\u00b16.4)\\t70.8 (\\u00b17.0)\\t71.5 (\\u00b17.6)\\t72.2 (\\u00b17.9)\\t106 (66.7%)\\t59 (60.8%)\\t42 (35.9%)\\t37 (39.8%)\\t \\tOHSU\\t379\\t114\\t64\\t108\\t93\\t86.9 (\\u00b16.3)\\t84.6 (\\u00b15.1)\\t87.7 (\\u00b17.4)\\t86.5 (\\u00b17.3)\\t39 (34.2%)\\t31 (48.4%)\\t23 (21.3%)\\t24 (25.8%)\\t \\tROSMAP\\t1,131\\t468\\t180\\t356\\t127\\t84.3 (\\u00b16.6)\\t83.8 (\\u00b16.5)\\t85.8 (\\u00b17.1)\\t86.2 (\\u00b16.8)\\t160 (34.2%)\\t65 (36.1%)\\t74 (20.8%)\\t18 (14.2%)\\t \\tTGEN\\t1,012\\t415\\t205\\t188\\t204\\t74.0 (\\u00b16.7)\\t72.2 (\\u00b16.6)\\t82.5 (\\u00b18.8)\\t79.3 (\\u00b18.5)\\t277 (66.8%)\\t132 (64.4%)\\t39 (20.7%)\\t43 (21.1%)\\t \\tUPITT\\t1,667\\t627\\t356\\t437\\t247\\t73.2 (\\u00b16.8)\\t73.0 (\\u00b16.5)\\t75.9 (\\u00b16.3)\\t75.2 (\\u00b15.9)\\t365 (58.2%)\\t214 (60.1%)\\t84 (19.2%)\\t51 (20.7%)\\t \\tUVM\\t1,257\\t354\\t193\\t444\\t266\\t73.6 (\\u00b17.7)\\t71.5 (\\u00b17.3)\\t73.4 (\\u00b17.6)\\t74.2 (\\u00b17.4)\\t225 (63.6%)\\t134 (69.4%)\\t111 (25.0%)\\t63 (23.7%)\\t \\tWASHU\\t373\\t138\\t106\\t86\\t43\\t76.6 (\\u00b19.5)\\t75.3 (\\u00b18.3)\\t77.6 (\\u00b19.9)\\t76.5 (\\u00b16.6)\\t80 (58.0%)\\t55 (51.9%)\\t24 (27.9%)\\t12 (27.9%)\\t \\tTotal\\t20,766\\t6,564\\t4,474\\t5,808\\t3,920\\t\\t\\t\\t\\t\\t\\t\\t\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 10726, \"infons\": {\"file\": \"acel13938-tbl-0001.xml\", \"id\": \"acel13938-tbl-0001\", \"section_type\": \"TABLE\", \"type\": \"table_footnote\"}, \"text\": \"Abbreviations: F, Females; M, Males.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 10763, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Genotyping data: Harmonization & imputation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 10807, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The entire dataset includes 35,110 participants. Adopted SNP\\u2010array Quality Control (QC) procedures were like the ones previously reported (Napolioni et al.,\\u00a0). Subjects with autosomal missingness >5% and/or X\\u2010chromosome missingness >5% (compared to other subjects in the same dataset), age below 60\\u2009years, age information missing, or phenotype inconsistency [missing phenotype, diagnosis of mild cognitive impairment, or a neurodegenerative phenotype other than LOAD] were excluded from the analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 11315, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Individual ancestry was determined using SNPweights v.2.1 (Chen et al.,\\u00a0) by using reference populations from the 1000 Genomes Consortium (1000 Genomes Project Consortium,\\u00a0). By applying an ancestry percentage cut\\u2010off >\\u2009=\\u200980%, the samples were stratified into five super populations, South\\u2010Asians (SAS), East\\u2010Asians (EAS), Americans (AMR), Africans (AFR), and Europeans (EUR). Since most of the samples belonged to the European population, we also determined their ancestry percentage in the discovery sample according to three genetically distinct European sub\\u2010ancestries, Northwestern, Southeastern, and Ashkenazi Jewish, using reference populations available from SNPweights v.2.1. European subjects were stratified into the above\\u2010mentioned ancestries when their ancestry percentage was >\\u2009=\\u200950% for any of the three sub\\u2010ancestries. Subjects with genetic ancestry estimates discordant from self\\u2010reported ancestry, as well as for subjects showing sex\\u2010inconsistency, were excluded from the analyses.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 12341, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"After keeping only the EUR subjects, each GWAS dataset was QCed to remove the SNPs with a call rate \\u226495%; Minor Allele Frequency (MAF) \\u22641%; SNPs with MAF deviating more than 10% from the MAF reported in 1000 Genomes for the EUR population; SNPs with differential missingness between cases and controls (p\\u2009<\\u20090.05); SNPs deviating from Hardy\\u2013Weinberg Equilibrium (HWE) in controls (p\\u2009<\\u20095\\u2009\\u00d7\\u200910\\u22125); tri\\u2010allelic SNPs; and SNPs where the alleles are mismatched compared to the 1000 Genomes reference sequence. A/T and C/G SNPs were removed prior to imputation.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 12919, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"All the datasets were phased and imputed using the TopMed Imputation Server (Das et al.,\\u00a0). After imputation, variants with a r  2  info score \\u22640.75 were excluded. For the statistical analyses, inter\\u2010dataset duplicates (IBD >0.95) were removed from the dataset having the lowest SNP coverage, while, in case of relatedness (IBD >0.0625) the affected or older subjects were kept, independently of SNP coverage.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 13334, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"For association testing analyses, we selected only the Northwestern European subjects since they represented most of the EUR population (approx. 80%) available across the collected GWAS. Analyses were performed using PLINK 2.0 (Chang et al.,\\u00a0).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 13580, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Selection of candidate genomic regions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 13619, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Genomic regions harboring the genes selected for longevity in previous work (Dato et al.,\\u00a0) were queried. Genetic variants from the candidate regions were extracted from the full dataset (approx. 12\\u2009million variants), considering their hg19 genomic coordinates (+/\\u2212 10 kilobases from the gene boundaries), and further filtered by applying a MAF cut\\u2010off of 0.05 and genotyping rate of 0.95. The final list of variants was generated by Linkage disequilibrium pruning (r  2 \\u2009>\\u20090.75) to reduce the computational burden of the analysis through the removal of highly correlated variants. Finally, the final dataset comprised a total of 2,469 variants (2,360 SNPs and 109 Ins/dels), as reported in Table\\u00a0S1.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14332, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Statistical analyses\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14353, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"For all the analyses, the whole sample was split based on sex and the presence/absence of APOE*4, defining four groups: female APOE*4 carriers, male APOE*4 carriers, female APOE*4 non\\u2010carriers, and male APOE*4 non\\u2010carriers.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14581, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"After the QC phase, a logistic regression analysis, with an additive model of association, was performed using PLINK 2.0 on the filtered dataset composed of 20,766 subjects (11,038 LOAD cases and 9,728 controls) and 2,469 variants, to estimate single\\u2010marker effects on the predisposition to LOAD. Age, three principal components from the ancestry analysis, APOE*2 dosage, and study site were used as covariates in the regression models. Results from the univariate analysis of the four study groups were meta\\u2010analyzed using GWAMA (M\\u00e4gi & Morris,\\u00a0). To determine the statistical significance of all the univariate analysis results, a Bonferroni's multiple testing correction was applied [(0.05/2469 variants*4 study groups)], yielding a p\\u2010value threshold of 5\\u2009\\u00d7\\u200910\\u22126. A nominal p\\u2010value\\u2009<\\u20090.05 was used as a filter for the main effect estimation and for selecting the variants to include in the gene\\u2013gene interaction analysis carried out using Multifactor dimensionality reduction (MDR) (Ritchie et al.,\\u00a0).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 15610, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \" p\\u2010value distribution enrichment analysis was performed using Pearson's chi\\u2010square on two\\u2010way contingency tables testing the observed number of variants passing the nominal level of statistical significance (p\\u2009<\\u20090.05) versus the expected one, deeming as statistically significant a Bonferroni's multiple testing correction of p\\u2009<\\u20090.013 (0.05/4 study groups) for individual study group or p\\u2009<\\u20090.05 for the meta\\u2010analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 16050, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"To plot single and common variants between groups of samples, Venn diagrams were created with VennDiagram R Package version 1.7.1 (https://cran.r\\u2010project.org/web/packages/VennDiagram/index.html).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 16248, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Gene\\u2010based association test was performed by Versatile Gene\\u2010based Association Study\\u20102 version 2 (VEGAS2, https://vegas2.qimrberghofer.edu.au/) (Mishra & Macgregor,\\u00a0) particularly useful for analyzing GWAS summary statistics. Based on the association p\\u2010values of the individual variants, VEGAS2 sums the effects of all the variants within a gene and generates a gene\\u2010based test statistic by doing simulations of the multivariate normal distribution.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 16708, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The epistatic interaction of up to four bi\\u2010allelic variants was tested using MDR (Ritchie et al.,\\u00a0). This methodology estimates high\\u2010order interactions among variants, with respect to a given phenotype also when their individual effect is small to moderate, allowing the discovery of multi\\u2010loci genotype combinations associated with high or low disease risk. An entropy\\u2010based clustering algorithm sets a contingency table for k gene variants and calculates case\\u2013control ratios for each of the possible multi\\u2010loci genotypes; a genotype combination is considered high\\u2010level if it is more present in cases compared to controls. For each factor, the MDR interaction model describes percentage of entropy (information gain or IG) and plots a network of two\\u2010way interactions, with positive values of entropy indicating synergistic or non\\u2010additive interaction while negative entropy values indicate redundancy between the markers or lack of any synergistic interaction between them. In the network, red and orange connections indicate non\\u2010linear interactions, green and brown connections indicate independence or additivity, and blue connections indicate redundancy. For all 2\\u2010order, 3\\u2010order, or 4\\u2010order combinations, the best model is considered the one found more consistent in different replicates (expressed as CV, consistency values and accuracy, i.e., training balanced accuracy level). To calculate significance, permutation testing was applied, dividing the data set into 10 portions, and using nine portions as a training data set, and the remaining as a testing data set. Ten thousand permutations were performed, to determine a cutoff threshold for an alpha\\u2009=\\u20090.05 significance level. For each order of interaction tested, an odds ratio (OR) is outputted, referring to the best combination of variants (best model), while determining the multi\\u2010locus high\\u2010risk combinations.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 18621, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Multifactor dimensionality reduction analyses were implemented in the open\\u2010source MDR software package version 3.0.2 (https://sourceforge.net/projects/mdr/).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 18781, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Functional annotation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 18803, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Functional annotation of risk variants was performed by using multiple bioinformatic tools and databases, including the HaploReg database (https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php), the GTEX portal (https://gtexportal.org/home/), RegulomeDB database (https://beta.regulomedb.org/regulome\\u2010search/) version 2.1, SNP Nexus (https://www.snp\\u2010nexus.org/v4/), and MetaBrain (https://www.metabrain.nl/cis\\u2010eqtls.html). LDlink's LDproxy tool (https://analysistools.nci.nih.gov/LDlink/) (Machiela & Chanock,\\u00a0) and European population data was used for SNP LD analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 19389, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_1\"}, \"text\": \"RESULTS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 19397, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Single\\u2010variant and gene\\u2010based analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 19440, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"The flowchart in Figure\\u00a01 details the study design. After quality control and filtering, we performed logistic regression analysis on the sample divided in four sub\\u2010groups according to sex\\u2010 and APOE*4 status, and meta\\u2010analyzing them thereafter. In Table\\u00a0S2 we reported the variants associated with the disease risk at a nominal p\\u2010value, along with their chromosomal location, the assigned gene and the main functional pathway associated with each coded protein. A graphical representation of all the obtained p\\u2010values is presented as a Manhattan plot (Figure\\u00a0S1). These associations, however, did not stand upon correction for multiple comparisons (p\\u2009<\\u20095\\u2009x\\u200910\\u22126) either when considering the single study group or when meta\\u2010analyzed across the four study groups. Nonetheless, we observed a statistically significant 45% enrichment (p\\u2009=\\u20090.002) in the distribution of nominally significant variants (p\\u2009<\\u20090.05) when the four study groups were meta\\u2010analyzed (Table\\u00a02 ), even though no statistically significant enrichment was found when analyzing individually the four study groups. Interestingly, we also detected a statistically significant enrichment from the meta\\u2010analysis Cochran's heterogeneity statistic's p\\u2010values (p\\u2009=\\u20090.020, Table\\u00a02), supporting the existence of APOE*4 and sex\\u2010specific effects.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 20787, \"infons\": {\"file\": \"ACEL-22-e13938-g005.jpg\", \"id\": \"acel13938-fig-0001\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Flowchart describing the steps of the analysis after quality control and filtering.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 20871, \"infons\": {\"file\": \"acel13938-tbl-0002.xml\", \"id\": \"acel13938-tbl-0002\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"\\np\\u2010value distribution enrichment analysis performed across the four study groups and the meta\\u2010analysis results.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 20987, \"infons\": {\"file\": \"acel13938-tbl-0002.xml\", \"id\": \"acel13938-tbl-0002\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"hsides\\\" rules=\\\"groups\\\"><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><thead valign=\\\"bottom\\\"><tr style=\\\"border-bottom:solid 1px #000000\\\"><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Group</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">N</italic> of variants with <italic toggle=\\\"yes\\\">p</italic>\\u2009&lt;\\u20090.05</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">OR</th><th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Enrichment <italic toggle=\\\"yes\\\">p</italic>\\n</th></tr></thead><tbody valign=\\\"top\\\"><tr><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">APOE</italic>*4<sup>+</sup> females</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">148 (6.0%)</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.216</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.118</td></tr><tr><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">APOE</italic>*4<sup>+</sup> males</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">157 (6.4%)</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.295</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.036</td></tr><tr><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">APOE</italic>*4<sup>\\u2212</sup> females</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">115 (4.7%)</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.932</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.597</td></tr><tr><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">APOE</italic>*4<sup>\\u2212</sup> males</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">136 (5.5%)</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.112</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.406</td></tr><tr><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Meta\\u2010analysis</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/></tr><tr><td align=\\\"left\\\" style=\\\"padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Meta\\u2010analysis <italic toggle=\\\"yes\\\">p</italic>\\u2010value</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">174 (7.1%)</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.446</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.002</bold>\\n</td></tr><tr><td align=\\\"left\\\" style=\\\"padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Cochran's heterogeneity statistic's <italic toggle=\\\"yes\\\">p</italic>\\u2010value</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">161 (6.5%)</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.331</td><td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.020</bold>\\n</td></tr></tbody></table>\\n\"}, \"text\": \"Group\\tN of variants with p\\u2009<\\u20090.05\\tOR\\tEnrichment p\\t \\tAPOE*4+ females\\t148 (6.0%)\\t1.216\\t0.118\\t \\tAPOE*4+ males\\t157 (6.4%)\\t1.295\\t0.036\\t \\tAPOE*4\\u2212 females\\t115 (4.7%)\\t0.932\\t0.597\\t \\tAPOE*4\\u2212 males\\t136 (5.5%)\\t1.112\\t0.406\\t \\tMeta\\u2010analysis\\t\\t\\t\\t \\tMeta\\u2010analysis p\\u2010value\\t174 (7.1%)\\t1.446\\t0.002\\t \\tCochran's heterogeneity statistic's p\\u2010value\\t161 (6.5%)\\t1.331\\t0.020\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 21351, \"infons\": {\"file\": \"acel13938-tbl-0002.xml\", \"id\": \"acel13938-tbl-0002\", \"section_type\": \"TABLE\", \"type\": \"table_footnote\"}, \"text\": \"\\nNote: We deemed as statistically significant a Bonferroni's multiple testing correction of p\\u2009<\\u20090.013 (0.05/4 study groups) for individual study group or p\\u2009<\\u20090.05 for the meta\\u2010analysis. p\\u2010values reported in bold are statistically significant.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 21606, \"infons\": {\"file\": \"acel13938-tbl-0002.xml\", \"id\": \"acel13938-tbl-0002\", \"section_type\": \"TABLE\", \"type\": \"table_footnote\"}, \"text\": \"Abbreviations: OR, odds ratio; p, p\\u2010value.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 21651, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Within the four study groups, the nominally associated variants belong to 126 different genes, of which 20 are in the IIS, 31 in DNA repair, and 23 in stress response pathways, whereas the remaining 46 are related to other pathways, such as immunity and membrane trafficking (see Table\\u00a0S2). These last genes emerged probably because of a high gene density and/or overlapping genes in some of the genomic regions considered, which extend 10\\u2009kb upstream and downstream of the candidate gene boundaries.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 22155, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"More specifically, the variants significantly associated in the sub\\u2010groups were: 148 in APOE*4+ females, 157 in APOE*4+ males, 115 in APOE*4\\u2212 females, and 136 in APOE*4\\u2212 males. The top\\u2010variants (p\\u2009<\\u20090.001) were: rs17810889\\u2010C8orf49 and rs5742665\\u2010IGF1 in APOE*4+ females, rs56190996\\u2010IGF1R and rs8113762\\u2010IRGQ in APOE*4+ males, rs28362737\\u2010AQP1 and rs35519594\\u2010XDH in APOE*4\\u2212 females, rs3729587\\u2010XPC and rs142270994\\u2010CTD\\u20103094\\u2009K11.1 in APOE*4\\u2212 males. The Venn diagram for the gene variants listed in Table\\u00a0S2 shows the number of variants in each sub\\u2010group and those shared (Figure\\u00a02). As it is shown in Table\\u00a03, 12 variants were associated with LOAD risk in females, independently from APOE*4 status, five of which showed an opposite direction of effect in the two sub\\u2010groups (Table\\u00a03a). In males, eight variants were associated with LOAD independently from APOE*4 status, five of them showing a divergent effect in the two sub\\u2010groups (Table\\u00a03b). On the other hand, APOE*4 carriers, independent of sex, share eight markers, five of which show an opposite effect in the two sexes (Table\\u00a03c), while APOE*4\\u2212 share four markers, two of which show an opposite effect in the two sexes (Table\\u00a03d).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 23388, \"infons\": {\"file\": \"ACEL-22-e13938-g001.jpg\", \"id\": \"acel13938-fig-0002\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"VENN diagram built on Table\\u00a0S1 data, showing the number of variants associated with LOAD in each sub\\u2010group of samples and in common between different groups.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 23549, \"infons\": {\"file\": \"acel13938-tbl-0003.xml\", \"id\": \"acel13938-tbl-0003\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Gene variants nominally associated with LOAD, shared between: (A) APOE*4+ females and APOE*4\\u2212 females; (B) APOE*4+ males and APOE*4\\u2212 males; (C) APOE*4+ females and APOE*4+ males; (D) APOE*4\\u2212 females and APOE*4\\u2212 males.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 23775, \"infons\": {\"file\": \"acel13938-tbl-0003.xml\", \"id\": \"acel13938-tbl-0003\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"hsides\\\" rules=\\\"all\\\"><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><thead valign=\\\"bottom\\\"><tr style=\\\"border-bottom:solid 1px #000000\\\"><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Variant</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Chr.</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Position</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Gene SYMBOL</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">EA</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">OR</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">p</italic>\\n</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; border-bottom:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">OR</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">p</italic>\\n</th></tr></thead><tbody valign=\\\"top\\\"><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"5\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>A</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"2\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">APOE*4</italic>\\n</bold>\\n<sup>\\n<bold>\\n<italic toggle=\\\"yes\\\">+</italic>\\n</bold>\\n</sup>\\n<bold>females</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"2\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">APOE*4</italic>\\n</bold>\\n<sup>\\n<bold>\\n<italic toggle=\\\"yes\\\">\\u2212</italic>\\n</bold>\\n</sup>\\n<bold>females</bold>\\n</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs13183641</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">95,147,287</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">RHOBTB3</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">T</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.873</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.003</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.909</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.010</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs4561</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">95,152,313</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">RHOBTB3</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">G</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.886</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.008</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.910</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.011</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs2348974</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">95,143,394</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">RHOBTB3</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">C</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.119</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.015</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.093</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.014</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs6556881</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">95,134,419</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">RHOBTB3</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">G</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.908</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.038</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.925</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.034</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs34886287</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">80,077,309</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">MSH3</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">C</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.110</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.029</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.105</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.007</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs2645429</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">8</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">11,660,051</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">FDFT1</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.111</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.048</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.902</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.016</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs2645433</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">8</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">11,657,921</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">RP11\\u2010297\\u2009N6.4</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">C</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.902</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.027</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.081</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.035</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs112267867</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">9</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">118,905,353</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">PAPPA</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.260</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.037</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.179</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.044</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs6583817</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">10</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">94,247,247</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">IDE</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">T</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.836</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.006</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.859</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.005</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs4752254</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">10</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">120,910,136</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">SFXN4</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">C</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.895</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.042</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.094</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.036</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs2860173</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">19</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">7,129,086</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">INSR</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.230</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.016</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.870</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.039</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs12979722</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">19</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">7,118,878</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">INSR</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">C</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.169</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.042</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.846</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.006</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"5\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>B</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"2\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">APOE*4</italic>\\n</bold>\\n<sup>\\n<bold>\\n<italic toggle=\\\"yes\\\">+</italic>\\n</bold>\\n</sup>\\n<bold>males</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"2\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">APOE*4</italic>\\n</bold>\\n<sup>\\n<bold>\\n<italic toggle=\\\"yes\\\">\\u2212</italic>\\n</bold>\\n</sup>\\n<bold>males</bold>\\n</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs78686161</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">242,065,356</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">EXO1</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.450</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.005</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.303</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.006</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs670548</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">53,366,989</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">GCLC</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">C</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.859</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.009</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.123</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.015</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs7740677</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">46,649,231</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">SLC25A27</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">C</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.167</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.030</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.154</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.013</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs17069665</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">108,941,468</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">FOXO3</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">G</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.832</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.033</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.169</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.031</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs2684794</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">15</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">99,484,953</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">IGF1R</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">C</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.848</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.038</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.135</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.049</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs7245548</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">19</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">45,981,840</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">ERCC1</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">T</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.126</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.036</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.912</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.049</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs4803825</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">19</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">45,986,483</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">RTN2</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">G</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.889</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.042</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.128</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.012</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs4328554</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">19</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">7,249,830</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">INSR</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">T</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.129</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.046</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.130</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.014</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"5\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>C</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"2\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">APOE*4</italic>\\n</bold>\\n<sup>\\n<bold>\\n<italic toggle=\\\"yes\\\">+</italic>\\n</bold>\\n</sup>\\n<bold>females</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"2\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">APOE*4</italic>\\n</bold>\\n<sup>\\n<bold>\\n<italic toggle=\\\"yes\\\">+</italic>\\n</bold>\\n</sup>\\n<bold>males</bold>\\n</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs17882672</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">53,408,883</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">GCLC</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">T</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.858</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.018</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.236</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.018</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs7092649</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">10</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">60,005,202</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">IPMK</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.878</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.017</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.866</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.037</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs17636964</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">10</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">59,953,046</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">IPMK</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">C</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.139</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.025</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.167</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.029</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs1625716</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">10</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">59,960,083</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">IPMK</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">G</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.826</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.038</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.784</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.033</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs7138318</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">12</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">104,736,394</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">TXNRD1</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">C</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.861</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.002</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.146</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.024</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs7979495</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">12</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">104,625,779</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">TXNRD1</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">G</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.810</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.018</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.247</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.047</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs141516621</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">19</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">7,280,152</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">INSR</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.232</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.003</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.844</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.043</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs7258382</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">19</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">7,262,569</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">INSR</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">C</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.141</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.048</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.841</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.027</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"5\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>D</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"2\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">APOE*4</italic>\\n</bold>\\n<sup>\\n<bold>\\n<italic toggle=\\\"yes\\\">\\u2212</italic>\\n</bold>\\n</sup>\\n<bold>females</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"2\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">APOE*4</italic>\\n</bold>\\n<sup>\\n<bold>\\n<italic toggle=\\\"yes\\\">\\u2212</italic>\\n</bold>\\n</sup>\\n<bold>males</bold>\\n</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs2010704</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">31,622,465</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">XDH</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.098</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.011</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.106</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.030</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs147249797</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">31,601,541</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">XDH</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">T</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.092</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.034</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.123</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.029</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">s10759223</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">9</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">110,065,158</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">RAD23B</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">C</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.078</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.038</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.909</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.041</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs2494741</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">14</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">105,249,322</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">AKT1</italic>\\n</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">T</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.157</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.032</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.754</td><td align=\\\"center\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.002</td></tr></tbody></table>\\n\"}, \"text\": \"Variant\\tChr.\\tPosition\\tGene SYMBOL\\tEA\\tOR\\tp\\tOR\\tp\\t \\tA\\tAPOE*4+females\\tAPOE*4\\u2212females\\t \\trs13183641\\t5\\t95,147,287\\tRHOBTB3\\tT\\t0.873\\t0.003\\t0.909\\t0.010\\t \\trs4561\\t5\\t95,152,313\\tRHOBTB3\\tG\\t0.886\\t0.008\\t0.910\\t0.011\\t \\trs2348974\\t5\\t95,143,394\\tRHOBTB3\\tC\\t1.119\\t0.015\\t1.093\\t0.014\\t \\trs6556881\\t5\\t95,134,419\\tRHOBTB3\\tG\\t0.908\\t0.038\\t0.925\\t0.034\\t \\trs34886287\\t5\\t80,077,309\\tMSH3\\tC\\t1.110\\t0.029\\t1.105\\t0.007\\t \\trs2645429\\t8\\t11,660,051\\tFDFT1\\tA\\t1.111\\t0.048\\t0.902\\t0.016\\t \\trs2645433\\t8\\t11,657,921\\tRP11\\u2010297\\u2009N6.4\\tC\\t0.902\\t0.027\\t1.081\\t0.035\\t \\trs112267867\\t9\\t118,905,353\\tPAPPA\\tA\\t1.260\\t0.037\\t1.179\\t0.044\\t \\trs6583817\\t10\\t94,247,247\\tIDE\\tT\\t0.836\\t0.006\\t0.859\\t0.005\\t \\trs4752254\\t10\\t120,910,136\\tSFXN4\\tC\\t0.895\\t0.042\\t1.094\\t0.036\\t \\trs2860173\\t19\\t7,129,086\\tINSR\\tA\\t1.230\\t0.016\\t0.870\\t0.039\\t \\trs12979722\\t19\\t7,118,878\\tINSR\\tC\\t1.169\\t0.042\\t0.846\\t0.006\\t \\tB\\tAPOE*4+males\\tAPOE*4\\u2212males\\t \\trs78686161\\t1\\t242,065,356\\tEXO1\\tA\\t1.450\\t0.005\\t1.303\\t0.006\\t \\trs670548\\t6\\t53,366,989\\tGCLC\\tC\\t0.859\\t0.009\\t1.123\\t0.015\\t \\trs7740677\\t6\\t46,649,231\\tSLC25A27\\tC\\t1.167\\t0.030\\t1.154\\t0.013\\t \\trs17069665\\t6\\t108,941,468\\tFOXO3\\tG\\t0.832\\t0.033\\t1.169\\t0.031\\t \\trs2684794\\t15\\t99,484,953\\tIGF1R\\tC\\t0.848\\t0.038\\t1.135\\t0.049\\t \\trs7245548\\t19\\t45,981,840\\tERCC1\\tT\\t1.126\\t0.036\\t0.912\\t0.049\\t \\trs4803825\\t19\\t45,986,483\\tRTN2\\tG\\t0.889\\t0.042\\t1.128\\t0.012\\t \\trs4328554\\t19\\t7,249,830\\tINSR\\tT\\t1.129\\t0.046\\t1.130\\t0.014\\t \\tC\\tAPOE*4+females\\tAPOE*4+males\\t \\trs17882672\\t6\\t53,408,883\\tGCLC\\tT\\t0.858\\t0.018\\t1.236\\t0.018\\t \\trs7092649\\t10\\t60,005,202\\tIPMK\\tA\\t0.878\\t0.017\\t0.866\\t0.037\\t \\trs17636964\\t10\\t59,953,046\\tIPMK\\tC\\t1.139\\t0.025\\t1.167\\t0.029\\t \\trs1625716\\t10\\t59,960,083\\tIPMK\\tG\\t0.826\\t0.038\\t0.784\\t0.033\\t \\trs7138318\\t12\\t104,736,394\\tTXNRD1\\tC\\t0.861\\t0.002\\t1.146\\t0.024\\t \\trs7979495\\t12\\t104,625,779\\tTXNRD1\\tG\\t0.810\\t0.018\\t1.247\\t0.047\\t \\trs141516621\\t19\\t7,280,152\\tINSR\\tA\\t1.232\\t0.003\\t0.844\\t0.043\\t \\trs7258382\\t19\\t7,262,569\\tINSR\\tC\\t1.141\\t0.048\\t0.841\\t0.027\\t \\tD\\tAPOE*4\\u2212females\\tAPOE*4\\u2212males\\t \\trs2010704\\t2\\t31,622,465\\tXDH\\tA\\t1.098\\t0.011\\t1.106\\t0.030\\t \\trs147249797\\t2\\t31,601,541\\tXDH\\tT\\t1.092\\t0.034\\t1.123\\t0.029\\t \\ts10759223\\t9\\t110,065,158\\tRAD23B\\tC\\t1.078\\t0.038\\t0.909\\t0.041\\t \\trs2494741\\t14\\t105,249,322\\tAKT1\\tT\\t1.157\\t0.032\\t0.754\\t0.002\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 25819, \"infons\": {\"file\": \"acel13938-tbl-0003.xml\", \"id\": \"acel13938-tbl-0003\", \"section_type\": \"TABLE\", \"type\": \"table_footnote\"}, \"text\": \"Abbreviations: EA, effect allele; OR, Odds Ratio; p, p\\u2010value.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 25883, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"For testing the enrichment of associated variants in the same gene, we then performed a gene\\u2010based analysis using VEGAS2. Table\\u00a0S3 reports the top\\u2010genes (p\\u2009<\\u20090.01) and the top\\u2010variant associated with the disease risk in the four sub\\u2010groups of samples. The schematic representation of Table\\u00a0S3 data is reported in Figure\\u00a03. The INSR gene is the only one common to all the sub\\u2010groups, although through different variants. FDFT1 and MSH3 were the top\\u2010genes in females, while IGF1R, IGF2R, PAPPA, GCLC, and XPC genes were the top\\u2010genes in males, independent of APOE*4 status. These genes, thus, appear to be associated with LOAD in a sex\\u2010specific way. Conversely, APOE*4+ subjects, independent of by sex, share markers in IGF1R, GCLC, and IPMK genes; APOE*4\\u2212 subjects, by contrast show an enrichment of significant markers in IGF2R, AKT1, GSR, MSH3, and XDH genes, suggesting them as influenced by APOE status in LOAD susceptibility.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 26838, \"infons\": {\"file\": \"ACEL-22-e13938-g003.jpg\", \"id\": \"acel13938-fig-0003\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Schematic representation of gene\\u2010based analysis, reporting the top\\u2010genes (p\\u2009<\\u20090.01) associated with the disease in each different sub\\u2010group and those shared. Colors represent the three analyzed pathways.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 27052, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Analysis of epistatic interactions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 27087, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"With the aim of finding gene\\u2013gene epistatic interactions between 2\\u2010 or among 3\\u2010 and 4\\u2010markers associated with the disease risk, we used the MDR approach. Table\\u00a04 shows the gene\\u2013gene interactions for LOAD resulting from the analysis, while Figure\\u00a04 depicts the interaction networks between variants in each sub\\u2010group.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 27418, \"infons\": {\"file\": \"acel13938-tbl-0004.xml\", \"id\": \"acel13938-tbl-0004\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Significant results of gene\\u2013gene interaction associations, resulting from the MDR analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 27512, \"infons\": {\"file\": \"acel13938-tbl-0004.xml\", \"id\": \"acel13938-tbl-0004\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"hsides\\\" rules=\\\"all\\\"><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><col align=\\\"left\\\" span=\\\"1\\\"/><thead valign=\\\"bottom\\\"><tr style=\\\"border-bottom:solid 1px #000000\\\"><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; border-bottom:solid 1px #000000\\\" rowspan=\\\"2\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\"/><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; border-bottom:solid 1px #000000\\\" rowspan=\\\"2\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">Gene\\u2013gene combination</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; border-bottom:solid 1px #000000\\\" rowspan=\\\"2\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">Genes included in the combination</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; border-bottom:solid 1px #000000\\\" rowspan=\\\"2\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">Pathways involved</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; border-bottom:solid 1px #000000\\\" rowspan=\\\"2\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">OR(95% CI.)</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; border-bottom:solid 1px #000000\\\" colspan=\\\"4\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">p</italic>\\u2010value (10,000 permutations)</th></tr><tr style=\\\"border-bottom:solid 1px #000000\\\"><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">APOE</italic>*4<sup>+</sup> females</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">APOE</italic>*4<sup>+</sup> males</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">APOE</italic>*4<sup>\\u2212</sup> females</th><th align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">APOE</italic>*4<sup>\\u2212</sup> males</th></tr></thead><tbody valign=\\\"top\\\"><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"9\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>A: <italic toggle=\\\"yes\\\">APOE</italic>*4</bold>\\n<sup>\\n<bold>+</bold>\\n</sup>\\n<bold>females</bold>\\n</td></tr><tr style=\\\"border-bottom:solid 1px #000000\\\"><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" rowspan=\\\"2\\\" valign=\\\"top\\\" colspan=\\\"1\\\">2\\u2010order interactions</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs3757949/rs7092522</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">GATA4</italic>/<italic toggle=\\\"yes\\\">IDE</italic>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Stress response/Metabolism</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.42 (1.25, 1.62)</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0001</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.1945</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.7051</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.6106</td></tr><tr style=\\\"border-bottom:solid 1px #000000\\\"><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs62491484/rs35435718</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">NEIL2</italic>/<italic toggle=\\\"yes\\\">PGPEP1L</italic>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">DNA repair/Stress response</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.36 (1.19, 1.54)</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0002</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.6143</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.8744</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0294</bold>\\n</td></tr><tr style=\\\"border-bottom:solid 1px #000000\\\"><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" rowspan=\\\"2\\\" valign=\\\"top\\\" colspan=\\\"1\\\">3\\u2010 and 4\\u2010order interactions</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs62491484/rs28672744/rs35435718</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">NEIL2</italic>/<italic toggle=\\\"yes\\\">TXNRD1</italic>/<italic toggle=\\\"yes\\\">PGPEP1L</italic>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">DNA repair/Stress response</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.56 (1.38, 1.76)</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0001</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.8034</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.9658</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.4729</td></tr><tr style=\\\"border-bottom:solid 1px #000000\\\"><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs2686186/rs7092522/rs12316064/rs35511346</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">FDFT1</italic>/<italic toggle=\\\"yes\\\">IDE</italic>/<italic toggle=\\\"yes\\\">IGF1</italic>/<italic toggle=\\\"yes\\\">TXNRD1</italic>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Stress response/Metabolism</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.96 (1.72, 2.22)</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0001</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.9389</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.6944</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.8240</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"9\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>B: <italic toggle=\\\"yes\\\">APOE</italic>*4</bold>\\n<sup>\\n<bold>+</bold>\\n</sup>\\n<bold>males</bold>\\n</td></tr><tr style=\\\"border-bottom:solid 1px #000000\\\"><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" rowspan=\\\"2\\\" valign=\\\"top\\\" colspan=\\\"1\\\">2\\u2010order interactions</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs4674302/rs718630</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">AOX1</italic>/<italic toggle=\\\"yes\\\">PTPN1</italic>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Stress response/Metabolism</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.46 (1.27, 1.69)</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.8127</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0001</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.9709</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.9489</td></tr><tr style=\\\"border-bottom:solid 1px #000000\\\"><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs56190996/rs4325676</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">IGF1R</italic>/<italic toggle=\\\"yes\\\">INSR</italic>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Metabolism</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.49 (1.28, 1.73)</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.5017</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0002</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.2057</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.2499</td></tr><tr style=\\\"border-bottom:solid 1px #000000\\\"><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">3\\u2010 order interaction</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs7563682/rs670548/rs12972985</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">AOX1</italic>/<italic toggle=\\\"yes\\\">GCLC</italic>/<italic toggle=\\\"yes\\\">KLC3</italic>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Stress response/Metabolism</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.69 (1.46,1.96)</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.9976</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0001</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.9992</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.2060</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" colspan=\\\"9\\\" valign=\\\"top\\\" rowspan=\\\"1\\\">\\n<bold>C: <italic toggle=\\\"yes\\\">APOE</italic>*4</bold>\\n<sup>\\n<bold>\\u2212</bold>\\n</sup>\\n<bold>males</bold>\\n</td></tr><tr style=\\\"border-bottom:solid 1px #000000\\\"><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" rowspan=\\\"2\\\" valign=\\\"top\\\" colspan=\\\"1\\\">2\\u2010order interactions</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs11573680/rs4983559</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">RAD23B</italic>/<italic toggle=\\\"yes\\\">ZBTB42</italic>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">DNA repair</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.43 (1.24, 1.64)</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.7629</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0318</bold>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.7856</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0004</bold>\\n</td></tr><tr style=\\\"border-bottom:solid 1px #000000\\\"><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs6214/rs1879612</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">IGF1</italic>/<italic toggle=\\\"yes\\\">IGF1R</italic>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Metabolism</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.33 (1.18, 1.50)</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.6723</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.8911</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.5558</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0004</bold>\\n</td></tr><tr style=\\\"border-bottom:solid 1px #000000\\\"><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000; padding-left:10%\\\" rowspan=\\\"2\\\" valign=\\\"top\\\" colspan=\\\"1\\\">3\\u2010 and 4\\u2010order interactions</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs11573680/rs1520220/rs4983559</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">RAD23B</italic>/<italic toggle=\\\"yes\\\">IGF1</italic>/<italic toggle=\\\"yes\\\">ZBTB42</italic>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">DNA repair/Metabolism</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.55 (1.37, 1.76)</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.2953</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.8032</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.9899</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0001</bold>\\n</td></tr><tr><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs1776178/rs2010704/rs804281/rs1879612</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">EXO1</italic>/<italic toggle=\\\"yes\\\">XDH</italic>/<italic toggle=\\\"yes\\\">GATA4</italic>/<italic toggle=\\\"yes\\\">IGF1R</italic>\\n</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">DNA repair/Stress response/Metabolism</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.16 (1.09, 1.40)</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.5058</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.9903</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.9301</td><td align=\\\"left\\\" style=\\\"border-right:solid 1px #000000\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.0003</bold>\\n</td></tr></tbody></table>\\n\"}, \"text\": \"\\tGene\\u2013gene combination\\tGenes included in the combination\\tPathways involved\\tOR(95% CI.)\\tp\\u2010value (10,000 permutations)\\t \\tAPOE*4+ females\\tAPOE*4+ males\\tAPOE*4\\u2212 females\\tAPOE*4\\u2212 males\\t \\tA: APOE*4+females\\t \\t2\\u2010order interactions\\trs3757949/rs7092522\\tGATA4/IDE\\tStress response/Metabolism\\t1.42 (1.25, 1.62)\\t0.0001\\t0.1945\\t0.7051\\t0.6106\\t \\trs62491484/rs35435718\\tNEIL2/PGPEP1L\\tDNA repair/Stress response\\t1.36 (1.19, 1.54)\\t0.0002\\t0.6143\\t0.8744\\t0.0294\\t \\t3\\u2010 and 4\\u2010order interactions\\trs62491484/rs28672744/rs35435718\\tNEIL2/TXNRD1/PGPEP1L\\tDNA repair/Stress response\\t1.56 (1.38, 1.76)\\t0.0001\\t0.8034\\t0.9658\\t0.4729\\t \\trs2686186/rs7092522/rs12316064/rs35511346\\tFDFT1/IDE/IGF1/TXNRD1\\tStress response/Metabolism\\t1.96 (1.72, 2.22)\\t0.0001\\t0.9389\\t0.6944\\t0.8240\\t \\tB: APOE*4+males\\t \\t2\\u2010order interactions\\trs4674302/rs718630\\tAOX1/PTPN1\\tStress response/Metabolism\\t1.46 (1.27, 1.69)\\t0.8127\\t0.0001\\t0.9709\\t0.9489\\t \\trs56190996/rs4325676\\tIGF1R/INSR\\tMetabolism\\t1.49 (1.28, 1.73)\\t0.5017\\t0.0002\\t0.2057\\t0.2499\\t \\t3\\u2010 order interaction\\trs7563682/rs670548/rs12972985\\tAOX1/GCLC/KLC3\\tStress response/Metabolism\\t1.69 (1.46,1.96)\\t0.9976\\t0.0001\\t0.9992\\t0.2060\\t \\tC: APOE*4\\u2212males\\t \\t2\\u2010order interactions\\trs11573680/rs4983559\\tRAD23B/ZBTB42\\tDNA repair\\t1.43 (1.24, 1.64)\\t0.7629\\t0.0318\\t0.7856\\t0.0004\\t \\trs6214/rs1879612\\tIGF1/IGF1R\\tMetabolism\\t1.33 (1.18, 1.50)\\t0.6723\\t0.8911\\t0.5558\\t0.0004\\t \\t3\\u2010 and 4\\u2010order interactions\\trs11573680/rs1520220/rs4983559\\tRAD23B/IGF1/ZBTB42\\tDNA repair/Metabolism\\t1.55 (1.37, 1.76)\\t0.2953\\t0.8032\\t0.9899\\t0.0001\\t \\trs1776178/rs2010704/rs804281/rs1879612\\tEXO1/XDH/GATA4/IGF1R\\tDNA repair/Stress response/Metabolism\\t1.16 (1.09, 1.40)\\t0.5058\\t0.9903\\t0.9301\\t0.0003\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29162, \"infons\": {\"file\": \"acel13938-tbl-0004.xml\", \"id\": \"acel13938-tbl-0004\", \"section_type\": \"TABLE\", \"type\": \"table_footnote\"}, \"text\": \"\\nNote: 2\\u2010 3\\u2010 and 4\\u2010loci interactions are shown, with respective pathways, selected for CV (10/10 replicates) and training balanced accuracy (>0.5). The OR value of the best combination of variants (best model) is reported. p\\u2010values reported in bold are considered statistically significant.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29461, \"infons\": {\"file\": \"ACEL-22-e13938-g002.jpg\", \"id\": \"acel13938-fig-0004\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Interaction graphs, reporting the significant markers from MDR analysis, in the group of APOE*4+ females (a), in APOE*4\\u2212 females, in APOE*4+ males, and in APOE*4\\u2212 males (d). For each variant we reported the value of information gain (IG) in per cent, while numbers in the connections indicate the entropy\\u2010based IG for the variant pairs. Red bar and orange bar indicate the high\\u2010level synergies on the phenotype, while the brown indicate a medium\\u2010level interaction, green and blue connections with negative IG values indicate redundancy or lack of synergistic interactions between the markers.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 30064, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"In APOE*4+ females, two 2\\u2010order epistatic interactions (Figure\\u00a04a) were found significantly associated with LOAD, namely the combinations rs3757949\\u2010GATA4/rs7092522\\u2010IDE and rs62491484\\u2010NEIL2/rs35435718\\u2010PGPEP1L. These four variants were among the most significant associated with LOAD (p\\u2009<\\u20090.01) in this sub\\u2010group for the single\\u2010variant analysis. Both epistatic interactions were inter\\u2010pathway, combining markers of genes belonging, respectively, to stress response (GATA4) and metabolism (IDE), and to DNA repair (NEIL2) and stress response (PGPEP1L). The same pathways also emerged from the 3\\u2010 and 4\\u2010order loci interactions (Table\\u00a04a), where two markers of the gene TXNRD1 (rs28672744 in the 3\\u2010order and rs35511346 in the 4\\u2010order interactions) and one of FDFT1 (rs26861869) are included. As indicated by the interaction graph in Figure\\u00a04a, several variants show a redundant effect (blue lines) in the network. Among them, the rs5742665\\u2010IGF1 is the top\\u2010variant from the gene\\u2010based analysis in this sub\\u2010group (p\\u2009<\\u20090.001; Table\\u00a0S2) and seems to be a LOAD specific marker of APOE*4+ females, not being associated with disease in the other sub\\u2010groups.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 31256, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"No significant epistatic interactions were observed in the group of APOE*4\\u2212 females (Figure\\u00a04b). The variant rs28362737\\u2010AQP1 present in the network is the most associated with LOAD in this group, but it seems to have a univariate effect.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 31499, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"In APOE*4+ males (Figure\\u00a04c), the most important gene\\u2013gene interactions (red lines) were: rs4674302\\u2010AOX1/rs718630\\u2010PTPN1 and rs56190996\\u2010IGF1R/rs4325676\\u2010INSR. These interactions are driven by rs718630\\u2010PTPN1 and rs56190996\\u2010IGF1R variants, which are the most significantly associated with LOAD (p\\u2009<\\u20090.01) in the single\\u2010marker analysis, with the last representing the top\\u2010variant in IGF1R associated with LOAD in the subgroup. Both interactions are intra\\u2010pathway epistasis between genes involved in metabolism. The 3\\u2010order interaction is again an intra\\u2010pathway epistasis, involving AOX1, GCLC, and KCL3 genes whose products act in metabolism.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 32166, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"In APOE*4\\u2212 males (Figure\\u00a04d), two gene\\u2013gene interactions resulted from MDR analysis namely rs11573680\\u2010RAD32B/rs4983559\\u2010ZBTB42, relative to the DNA\\u2010repair pathway, and rs6214\\u2010IGF1/rs1879612\\u2010IGF1R to metabolism. 3\\u2010order and 4\\u2010order interactions resume the collaboration of DNA repair and metabolism, in addition to stress response pathway, in the susceptibility to LOAD in this sub\\u2010group.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 32574, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"To determine whether the gene\\u2013gene interactions found were exclusive between the four sub\\u2010groups, we ran MDR using those selected interactions across all the groups. Notably, we observed that the rs62491484\\u2010NEIL2/rs35435718\\u2010PGPEP1L epistatic interaction found in APOE*4+ females occurred also in APOE*4\\u2212 males (p\\u2009=\\u20090.028) (Table\\u00a04). However, the high\\u2010risk genotypic combinations were not comparable (Figure\\u00a0S2). Similarly, the rs11573680\\u2010RAD32B/rs4983559\\u2010ZBTB42 interaction found in APOE*4\\u2212 males occurred also in the APOE*4+ males (p\\u2009=\\u20090.038) (Table\\u00a04), although with a different pattern of high\\u2010risk genotypic combinations (Figure\\u00a0S2).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 33244, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Functional annotation analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 33275, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Next, we performed functional annotation to ascertain biological significance of the variants identified in the single variant and interaction analyses. To this end, several databases and tools were considered, as reported in the Materials & Methods section. Table\\u00a0S4 reports the most relevant results of eQTL analysis of the associated variants and their proxies in high LD (r  2 \\u2009>\\u2009=0.8) According to this analysis some of the risk variants act as cis\\u2010eQTL regulatory elements which modulate the expression of the corresponding gene or of nearby genes in specific brain regions. Moreover, evidence of a regulatory role (1f or 1b score in RegulomeDb and association with regulatory elements in SNPnexus) was found for other variants associated with the disease phenotype.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34055, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Some variants reported a relevant number of proxies in LD (r  2 \\u2009\\u2265\\u20090.8), although none showed evidence of stronger regulatory potential than the lead variant. As reported in Table\\u00a0S4, the significant variants we identified were not previously implicated in LOAD, yet some of them have been reported to affect some age\\u2010related disorders.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34401, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_1\"}, \"text\": \"DISCUSSION\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34412, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"The genetic architecture of LOAD has been widely studied in recent years, and so far, 100 of risk genes and related rare and common genetic variants have been identified, but many remain to be uncovered (Andrews et al.,\\u00a0). As advanced age is the greatest risk factor for LOAD, shared genetic pathways between LOAD and longevity are expected, although their connections are still not fully understood (Tesi et al.,\\u00a0). Phenotypic heterogeneity (i.e., distinct groups of subjects present with different clinical syndromes) and/or temporal heterogeneity (i.e., substantial inter\\u2010subject variance in age\\u2010at\\u2010onset and rate of decline) (Young et al.,\\u00a0) are common in LOAD. Another challenge is the small contribution of individual genetic variants in complex phenotypes. Moreover, several studies have emphasized that genetic interactions may be more important than single markers in neurodegenerative diseases and longevity (Gilbert\\u2010Diamond & Moore,\\u00a0), suggesting the existence of non\\u2010additive heritability to these traits.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 35444, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"In this study, we applied a gene\\u2013gene interaction approach to a set of genetic variants selected for being on or near genes involved in the IIS, the DNA repair, and the oxidative stress response pathways, previously linked to human longevity (Dato et al.,\\u00a0). To identify gene signatures for LOAD related to sex and/or to APOE genotype, we performed analyses stratified by APOE*4 status and sex. The validity of the choice of such study design was underscored by additional cross\\u2010validation analyses, which confirmed the presence of heterogeneity across the different sub\\u2010groups.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 36030, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Overall, this study highlights two aspects of the genetic complexity in LOAD. First, it supports the claim of shared genetic pathways between longevity and LOAD. Second, it suggests that sex and APOE genotype can drive different genetic risk factors. This evidence appeared clear in single variant analyses. In APOE*4+ females, the two top variants were rs17810889 upstream of C8orf49, and rs5742665 in an intron of IGF1 gene, one of the master genes in IIS metabolic pathway. In APOE*4\\u2212 females, the top\\u2010variant was rs28362737, an intronic variant of AQP1, the gene encoding Aquaporin 1, a protein associated with amyloid\\u2010beta deposition in AD brains (Misawa et al.,\\u00a0). In APOE*4+males, the top\\u2010variants were rs56190996, in an intron of IGF1R (insulin\\u2010like growth factor receptor), and rs8113762 located at the 3\\u2019\\u2010UTR of IRGQ (immunity\\u2010related GTPase Q), while in APOE*4\\u2212 subjects, the most significant variant was rs3729587 in the DNA repair gene XPC (Xeroderma pigmentosum, complementation group C). Many of these markers appear to be functionally relevant, being cis\\u2010eQTL or trans\\u2010eQTL, and specifically in brain regions, as reported in Table\\u00a0S4. However, we cannot exclude that one of their proxies in LD (r  2 \\u2009\\u2265\\u20090.8) could be the actual causal variant, although none showed a potential regulatory effect higher than the associated variants.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 37408, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"It is also worth noting that some of the associated variants showed an opposite effect in the two sexes. This is line with several studies demonstrating that some variants, the so\\u2010called \\u201csexually antagonistic variants\\u201d, have a beneficial effect in one sex but deleterious (or null) effects in the other, thus having a role in shaping differences between males and females in age\\u2010related disease outcomes as well as in survival (Iannuzzi et al.,\\u00a0; Lagou et al.,\\u00a0).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 37883, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Results from gene\\u2010based analysis further highlighted genes associated with LOAD in a sex\\u2010 and APOE\\u2010specific manner. INSR, is the only one gene shared among the four sub\\u2010groups of patients, which encodes the insulin receptor, an important component of the insulin pathway. Through its binding with insulin, INSR controls the glucose metabolism in the brain helping to maintain neuronal functioning (de la Monte & Wands,\\u00a0). Decline in glucose metabolism is indeed one of the earliest and most common anomalies observed in patients with LOAD (Akhtar & Sah,\\u00a0). A recent study by Leclerc et al.\\u00a0 reported that, in association with \\u03b2\\u2010amyloid pathology, defects in the activation of INSR at the blood\\u2013brain barrier strongly contribute to brain insulin resistance in LOAD. Genetic variants of this gene were found enriched in centenarians, thus indicating INSR a key mediator of human longevity (Barbieri et al.,\\u00a0).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 38808, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"The evaluation of the joint effect of different markers through gene\\u2013gene interactions further added insights into the genetic architecture of LOAD. Among the significantly interacting genes, PTPN1 (protein tyrosine phosphatase non\\u2010receptor type 1, IIS pathway), TXNRD1 (thioredoxin reductase 1; stress response), and IGF1R (insulin\\u2010like growth factor 1 receptor; IIS) were already found in combinations affecting survival to old age (Dato et al.,\\u00a0; Ukraintseva et al.,\\u00a0). PTPN1 and TXNRD1 were engaged in best risk combinations for longevity, respectively, with IGF1R and TP53 (DNA repair), while IGF1R, interacted with TP53 and TGFBR2 (cell proliferation). In LOAD PTPN1 interacts with a partner from the stress response pathway, like AOX1. Notably, TXNRD1 is engaged in 4\\u2010order risk combination for LOAD with genes from the IIS pathway, like IGF1, FDFT1 (farnesyl\\u2010diphosphate farnesyltransferase 1), and IDE (insulin degrading enzyme). Similarly, IGF1R is associated with the disease in a 4\\u2010order synergic combination with EXO1 (DNA repair), XDH (stress response), and GATA4 (IIS), behind its ligand IGF1. Overall, these findings suggest that in LOAD as in longevity TXNRD1 and even more IGF1R, may represent hubs interconnecting multiple signaling pathways. The different sub\\u2010processes may instead explain the sex\\u2010 and APOE\\u2010specific associations we found.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 40188, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Taken together, these results suggest that longevity loci may also drive LOAD neuropathology through APOE\\u2010 and sex\\u2010related specific gene\\u2013gene interactions. The complex interplay among sex, APOE, and age may influence the severity and the temporal trajectory of LOAD progression, creating a risk profile for LOAD that could serve to identify high\\u2010risk individuals (Riedel et al.,\\u00a0). Mechanisms underlying the sex differences are unknown; the literature largely supports the claim that it may be due to the known differences in longevity between men and women (Hossin,\\u00a0), while some propose that sex dimorphisms in stress responses can contribute to the increased prevalence of LOAD in women (Yan et al.,\\u00a0). Sex divergence in biochemical responses to stress were reported along the hypothalamic\\u2013pituitary\\u2013adrenal axis and in the activation of the cortical corticotrophin\\u2010releasing factor receptor 1 signaling pathway, leading to distinct female\\u2010biased increases in molecules associated with LOAD pathogenesis (Yan et al.,\\u00a0). In our analysis, effectors of stress response, such as TXNRD1 and GATA4, have been found in all the sub\\u2010groups of patients, suggesting that an impaired ability to induce a stress response represents an underlying risk factor for LOAD. Interestingly, in APOE*4+ females, potentially experiencing higher levels of hydroxyl radicals and reduced levels of mitochondrial antioxidants compared to non\\u2010APOE*4 carriers (Ihara et al.,\\u00a0), all significant epistatic interactions were among genes involved in oxidative stress and those belonging to the other pathways.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 41793, \"infons\": {\"section_type\": \"CONCL\", \"type\": \"title_1\"}, \"text\": \"CONCLUSIONS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 41805, \"infons\": {\"section_type\": \"CONCL\", \"type\": \"paragraph\"}, \"text\": \"Collectively, the present results indicate that several loci repeatedly implicated in aging and longevity also contribute to late\\u2010onset Alzheimer's disease (LOAD) risk. Most of the top genes associated are assigned to pathways related to metabolism, highlighting their relevance both in the aging process and the pathological events leading to LOAD. Interestingly, this study bolsters the evidence of specific interactions among established risk factors for LOAD, that is APOE genotype and sex, and these genes. This suggests that different trajectories of cognitive aging may be the result of specific epistatic effects between genetic and non\\u2010genetic risk factors. Although our conclusions are based on the evidence of \\u201cstatistical epistasis\\u201d, which magnitude and contribution to the variance of complex traits is a highly debated topic (Hivert et al.,\\u00a0), in accordance with other authors (Singhal et al.,\\u00a0) we think that the potential for \\u201cfunctional epistasis\\u201d to drive expressivity and explain clinical heterogeneity in complex diseases is mounting. In the complex scenario of LOAD, understanding the functional changes associated with different combinations of interacting entities (SNPs, genes, pathways, etc.) may help to disentangle the genetic architecture underlying disease development, and interindividual differences that underpin disease, finally leading to precision medicine approaches for early detection of individuals at higher risk for cognitive decline or dementia.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43307, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"title_1\"}, \"text\": \"AUTHOR CONTRIBUTIONS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43328, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"paragraph\"}, \"text\": \"Idea/concept: SD, FDR, GR, VN; Data curation/processing: VN, SP, MEB, YLG; Data analysis: SD, FDR, PC, GR, VN; Writing: SD, FDR, PC, GR, VN; Advice: MEB, YLG, MDG, GP.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43496, \"infons\": {\"section_type\": \"COMP_INT\", \"type\": \"title_1\"}, \"text\": \"CONFLICT OF INTEREST STATEMENT\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43527, \"infons\": {\"section_type\": \"COMP_INT\", \"type\": \"paragraph\"}, \"text\": \"The authors declare no competing interests.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43571, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_1\"}, \"text\": \"Supporting information\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43594, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_1\"}, \"text\": \"DATA AVAILABILITY STATEMENT\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43622, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"paragraph\"}, \"text\": \"Data sharing is not applicable to this article as no new data were created or analyzed in this study.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43724, \"infons\": {\"section_type\": \"REF\", \"type\": \"title\"}, \"text\": \"REFERENCES\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43735, \"infons\": {\"fpage\": \"68\", \"issue\": \"7571\", \"lpage\": \"74\", \"pub-id_doi\": \"10.1038/nature15393\", \"pub-id_pmid\": \"26432245\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"526\", \"year\": \"2015\"}, \"text\": \"A global reference for human genetic variation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43782, \"infons\": {\"elocation-id\": \"222\", \"issue\": \"3\", \"pub-id_doi\": \"10.3390/genes10030222\", \"pub-id_pmid\": \"30884759\", \"section_type\": \"REF\", \"source\": \"Genes\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2019\"}, \"text\": \"The Genetic variability of APOE in different human populations and its implications for longevity\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43880, \"infons\": {\"elocation-id\": \"104707\", \"pub-id_doi\": \"10.1016/j.neuint.2020.104707\", \"pub-id_pmid\": \"32092326\", \"section_type\": \"REF\", \"source\": \"Neurochemistry International\", \"type\": \"ref\", \"volume\": \"135\", \"year\": \"2020\"}, \"text\": \"Insulin signaling pathway and related molecules: Role in neurodegeneration and Alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43979, \"infons\": {\"fpage\": \"383\", \"issue\": \"3\", \"lpage\": \"421\", \"pub-id_doi\": \"10.1016/s0197-4580(00)00124-x\", \"pub-id_pmid\": \"10858586\", \"section_type\": \"REF\", \"source\": \"Neurobiology of Aging\", \"type\": \"ref\", \"volume\": \"21\", \"year\": \"2000\"}, \"text\": \"Inflammation and Alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44016, \"infons\": {\"elocation-id\": \"104511\", \"pub-id_doi\": \"10.1016/j.ebiom.2023.104511\", \"pub-id_pmid\": \"36907103\", \"section_type\": \"REF\", \"source\": \"eBioMedicine\", \"type\": \"ref\", \"volume\": \"90\", \"year\": \"2023\"}, \"text\": \"The complex genetic architecture of Alzheimer's disease: Novel insights and future directions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44110, \"infons\": {\"fpage\": \"881\", \"lpage\": \"896\", \"pub-id_doi\": \"10.1007/s11357-021-00477-0\", \"pub-id_pmid\": \"34921659\", \"section_type\": \"REF\", \"source\": \"GeroScience\", \"type\": \"ref\", \"volume\": \"44\", \"year\": \"2022\"}, \"text\": \"Association of rs3027178 polymorphism in the circadian clock gene PER1 with susceptibility to Alzheimer's disease and longevity in an Italian population\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44263, \"infons\": {\"fpage\": \"E1064\", \"issue\": \"5\", \"lpage\": \"E1071\", \"pub-id_doi\": \"10.1152/ajpendo.00296.2003\", \"pub-id_pmid\": \"14534077\", \"section_type\": \"REF\", \"source\": \"American Journal of Physiology. Endocrinology and Metabolism\", \"type\": \"ref\", \"volume\": \"285\", \"year\": \"2003\"}, \"text\": \"Insulin/IGF\\u2010I\\u2010signaling pathway: An evolutionarily conserved mechanism of longevity from yeast to humans\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44372, \"infons\": {\"fpage\": \"1325\", \"issue\": \"11\", \"lpage\": \"1344\", \"pub-id_doi\": \"10.1007/s007020070021\", \"section_type\": \"REF\", \"source\": \"Journal of Neural Transmission (Vienna)\", \"type\": \"ref\", \"volume\": \"107\", \"year\": \"2000\"}, \"text\": \"Apoptosis and Alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44406, \"infons\": {\"fpage\": \"7\", \"pub-id_doi\": \"10.1186/s13742-015-0047-8\", \"pub-id_pmid\": \"25722852\", \"section_type\": \"REF\", \"source\": \"Gigascience\", \"type\": \"ref\", \"volume\": \"4\", \"year\": \"2015\"}, \"text\": \"Second\\u2010generation PLINK: Rising to the challenge of larger and richer datasets\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44487, \"infons\": {\"fpage\": \"1399\", \"lpage\": \"1406\", \"pub-id_doi\": \"10.1093/bioinformatics/btt144\", \"pub-id_pmid\": \"23539302\", \"section_type\": \"REF\", \"source\": \"Bioinformatics\", \"type\": \"ref\", \"volume\": \"29\", \"year\": \"2013\"}, \"text\": \"Improved ancestry inference using weights from external reference panels\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44560, \"infons\": {\"fpage\": \"1284\", \"lpage\": \"1287\", \"pub-id_doi\": \"10.1038/ng.3656\", \"pub-id_pmid\": \"27571263\", \"section_type\": \"REF\", \"source\": \"Nature Genetics\", \"type\": \"ref\", \"volume\": \"48\", \"year\": \"2016\"}, \"text\": \"Next\\u2010generation genotype imputation service and methods\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44618, \"infons\": {\"elocation-id\": \"111439\", \"pub-id_doi\": \"10.1016/j.mad.2021.111439\", \"pub-id_pmid\": \"33497757\", \"section_type\": \"REF\", \"source\": \"Mechanisms of Ageing and Development\", \"type\": \"ref\", \"volume\": \"195\", \"year\": \"2021\"}, \"text\": \"IP6K3 and IPMK variations in LOAD and longevity: Evidence for a multifaceted signaling network at the crossroad between neurodegeneration and survival\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44769, \"infons\": {\"elocation-id\": \"e12755\", \"issue\": \"3\", \"pub-id_doi\": \"10.1111/acel.12755\", \"pub-id_pmid\": \"29577582\", \"section_type\": \"REF\", \"source\": \"Aging Cell\", \"type\": \"ref\", \"volume\": \"17\", \"year\": \"2018\"}, \"text\": \"The genetic component of human longevity: New insights from the analysis of pathway\\u2010based SNP\\u2010SNP interactions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44884, \"infons\": {\"fpage\": \"45\", \"issue\": \"1\", \"lpage\": \"61\", \"pub-id_doi\": \"10.3233/jad-2005-7106\", \"section_type\": \"REF\", \"source\": \"Journal of Alzheimer's Disease\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2005\"}, \"text\": \"Review of insulin and insulin\\u2010like growth factor expression, signaling, and malfunction in the central nervous system: Relevance to Alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45038, \"infons\": {\"fpage\": \"3669\", \"issue\": \"1\", \"pub-id_doi\": \"10.1038/s41467-019-11558-2\", \"section_type\": \"REF\", \"source\": \"Nature Communications\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2019\"}, \"text\": \"A meta\\u2010analysis of genome\\u2010wide association studies identifies multiple longevity genes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45129, \"infons\": {\"fpage\": \"2272\", \"issue\": \"7\", \"lpage\": \"2280\", \"pub-id_doi\": \"10.1093/brain/awaa164\", \"pub-id_pmid\": \"32591829\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"143\", \"year\": \"2020\"}, \"text\": \"Sex\\u2010dependent autosomal effects on clinical progression of Alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45210, \"infons\": {\"fpage\": \"14\", \"pub-id_doi\": \"10.1002/0471142905.hg0114s70\", \"section_type\": \"REF\", \"source\": \"Current Protocols in Human Genetics\", \"type\": \"ref\", \"volume\": \"1\", \"year\": \"2011\"}, \"text\": \"Analysis of gene\\u2010gene interactions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45247, \"infons\": {\"elocation-id\": \"6329041\", \"pub-id_doi\": \"10.1155/2021/6329041\", \"pub-id_pmid\": \"34326892\", \"section_type\": \"REF\", \"source\": \"Computational and Mathematical Methods in Medicine\", \"type\": \"ref\", \"volume\": \"2021\", \"year\": \"2021\"}, \"text\": \"Identification of the hub genes in Alzheimer's Disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45302, \"infons\": {\"fpage\": \"786\", \"issue\": \"5\", \"lpage\": \"798\", \"pub-id_doi\": \"10.1016/j.ajhg.2021.02.014\", \"pub-id_pmid\": \"33811805\", \"section_type\": \"REF\", \"source\": \"American Journal of Human Genetics\", \"type\": \"ref\", \"volume\": \"108\", \"year\": \"2021\"}, \"text\": \"Estimation of non\\u2010additive genetic variance in human complex traits from a large sample of unrelated individuals\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45417, \"infons\": {\"fpage\": \"482\", \"issue\": \"5\", \"lpage\": \"484\", \"pub-id_doi\": \"10.1093/inthealth/ihaa106\", \"pub-id_pmid\": \"33533409\", \"section_type\": \"REF\", \"source\": \"International Health\", \"type\": \"ref\", \"volume\": \"13\", \"year\": \"2021\"}, \"text\": \"The male disadvantage in life expectancy: Can we close the gender gap?\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45488, \"infons\": {\"elocation-id\": \"1\", \"pub-id_doi\": \"10.1186/s41118-023-00181-1\", \"section_type\": \"REF\", \"source\": \"Genus\", \"type\": \"ref\", \"volume\": \"79\", \"year\": \"2023\"}, \"text\": \"The role of genetics and epigenetics in sex differences in human survival\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45562, \"infons\": {\"fpage\": \"65\", \"issue\": \"1\", \"pub-id_doi\": \"10.1186/s13024-021-00489-6\", \"pub-id_pmid\": \"34535174\", \"section_type\": \"REF\", \"source\": \"Molecular Neurodegeneration\", \"type\": \"ref\", \"volume\": \"16\", \"year\": \"2021\"}, \"text\": \"Nuclear dynamics and stress responses in Alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45623, \"infons\": {\"fpage\": \"346\", \"issue\": \"6\", \"lpage\": \"349\", \"pub-id_doi\": \"10.1034/j.1600-0404.2000.102006346.x\", \"pub-id_pmid\": \"11125747\", \"section_type\": \"REF\", \"source\": \"Acta Neurologica Scandinavica\", \"type\": \"ref\", \"volume\": \"102\", \"year\": \"2000\"}, \"text\": \"Relationship between oxidative stress and apoE phenotype in Alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45703, \"infons\": {\"fpage\": \"687\", \"issue\": \"11\", \"lpage\": \"700\", \"pub-id_doi\": \"10.1038/s41583-018-0067-3\", \"pub-id_pmid\": \"30266970\", \"section_type\": \"REF\", \"source\": \"Nature Reviews Neuroscience\", \"type\": \"ref\", \"volume\": \"19\", \"year\": \"2018\"}, \"text\": \"Imaging the evolution and pathophysiology of Alzheimer disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45766, \"infons\": {\"fpage\": \"270\", \"lpage\": \"280\", \"pub-id_doi\": \"10.1111/j.1474-9726.2008.00366.x\", \"pub-id_pmid\": \"18208581\", \"section_type\": \"REF\", \"source\": \"Aging Cell\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2008\"}, \"text\": \"Genes encoding longevity: From model organisms to humans\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45823, \"infons\": {\"fpage\": \"1659\", \"issue\": \"6\", \"lpage\": \"1675\", \"pub-id_doi\": \"10.1111/j.1471-4159.2006.03990.x\", \"pub-id_pmid\": \"16805775\", \"section_type\": \"REF\", \"source\": \"Journal of Neurochemistry\", \"type\": \"ref\", \"volume\": \"97\", \"year\": \"2006\"}, \"text\": \"The role of mitochondria in inherited neurodegenerative diseases\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45888, \"infons\": {\"fpage\": \"24\", \"issue\": \"1\", \"pub-id_doi\": \"10.1038/s41467-020-19366-9\", \"section_type\": \"REF\", \"source\": \"Nature Communications\", \"type\": \"ref\", \"volume\": \"12\", \"year\": \"2021\"}, \"text\": \"Sex\\u2010dimorphic genetic effects and novel loci for fasting glucose and insulin variability\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45979, \"infons\": {\"fpage\": \"75\", \"issue\": \"1\", \"lpage\": \"90\", \"pub-id_doi\": \"10.1093/brain/awac309\", \"pub-id_pmid\": \"36280236\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"146\", \"year\": \"2023\"}, \"text\": \"Cerebrovascular insulin receptors are defective in Alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46050, \"infons\": {\"fpage\": \"312\", \"issue\": \"2\", \"lpage\": \"339\", \"pub-id_doi\": \"10.1016/j.cell.2019.09.001\", \"pub-id_pmid\": \"31564456\", \"section_type\": \"REF\", \"source\": \"Cell\", \"type\": \"ref\", \"volume\": \"179\", \"year\": \"2019\"}, \"text\": \"Alzheimer Disease: An update on pathobiology and treatment strategies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46120, \"infons\": {\"fpage\": \"3555\", \"lpage\": \"3557\", \"pub-id_doi\": \"10.1093/bioinformatics/btv402\", \"pub-id_pmid\": \"26139635\", \"section_type\": \"REF\", \"source\": \"Bioinformatics\", \"type\": \"ref\", \"volume\": \"31\", \"year\": \"2015\"}, \"text\": \"LDlink: A web\\u2010based application for exploring population\\u2010specific haplotype structure and linking correlated alleles of possible functional variants\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46273, \"infons\": {\"elocation-id\": \"288\", \"pub-id_doi\": \"10.1186/1471-2105-11-288\", \"pub-id_pmid\": \"20509871\", \"section_type\": \"REF\", \"source\": \"BMC Bioinformatics\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2010\"}, \"text\": \"GWAMA: Software for genome\\u2010wide association meta\\u2010analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46335, \"infons\": {\"fpage\": \"247\", \"issue\": \"3\", \"lpage\": \"260\", \"pub-id_doi\": \"10.1007/s00401-008-0387-x\", \"pub-id_pmid\": \"18509662\", \"section_type\": \"REF\", \"source\": \"Acta Neuropathologica\", \"type\": \"ref\", \"volume\": \"116\", \"year\": \"2008\"}, \"text\": \"Close association of water channel AQP1 with amyloid\\u2010beta deposition in Alzheimer disease brains\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46434, \"infons\": {\"fpage\": \"86\", \"issue\": \"1\", \"lpage\": \"91\", \"pub-id_doi\": \"10.1017/thg.2014.79\", \"pub-id_pmid\": \"25518859\", \"section_type\": \"REF\", \"source\": \"Twin Research and Human Genetics\", \"type\": \"ref\", \"volume\": \"18\", \"year\": \"2015\"}, \"text\": \"VEGAS2: Software for more flexible gene\\u2010based testing\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46490, \"infons\": {\"elocation-id\": \"629373\", \"pub-id_doi\": \"10.3389/fgene.2020.629373\", \"pub-id_pmid\": \"33584820\", \"section_type\": \"REF\", \"source\": \"Frontiers in Genetics\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2021\"}, \"text\": \"Recent consanguinity and outbred Autozygosity are associated with increased risk of late\\u2010onset Alzheimer's Disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46607, \"infons\": {\"fpage\": \"1171\", \"issue\": \"9\", \"lpage\": \"1183\", \"pub-id_doi\": \"10.1016/j.jalz.2018.04.008\", \"section_type\": \"REF\", \"source\": \"Alzheimer's & Dementia\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2018\"}, \"text\": \"Understanding the impact of sex and gender in Alzheimer's disease: A call to action\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46691, \"infons\": {\"fpage\": \"200.e13\", \"lpage\": \"200.e20\", \"pub-id_doi\": \"10.1016/j.neurobiolaging.2016.02.024\", \"section_type\": \"REF\", \"source\": \"Neurobiology of Aging\", \"type\": \"ref\", \"volume\": \"41\", \"year\": \"2016\"}, \"text\": \"Assessment of the genetic variance of late\\u2010onset Alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46762, \"infons\": {\"fpage\": \"134\", \"lpage\": \"147\", \"pub-id_doi\": \"10.1016/j.jsbmb.2016.03.012\", \"pub-id_pmid\": \"26969397\", \"section_type\": \"REF\", \"source\": \"The Journal of Steroid Biochemistry and Molecular Biology\", \"type\": \"ref\", \"volume\": \"160\", \"year\": \"2016\"}, \"text\": \"Age, APOE and sex: Triad of risk of Alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46818, \"infons\": {\"fpage\": \"138\", \"issue\": \"1\", \"lpage\": \"147\", \"pub-id_doi\": \"10.1086/321276\", \"pub-id_pmid\": \"11404819\", \"section_type\": \"REF\", \"source\": \"American Journal of Human Genetics\", \"type\": \"ref\", \"volume\": \"69\", \"year\": \"2001\"}, \"text\": \"Multifactor\\u2010dimensionality reduction reveals high\\u2010order interactions among estrogen\\u2010metabolism genes in sporadic breast cancer\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46951, \"infons\": {\"fpage\": \"377\", \"lpage\": \"395\", \"pub-id_doi\": \"10.1146/annurev-biodatasci-102022-120818\", \"section_type\": \"REF\", \"source\": \"Annual Review of Biomedical Data Science\", \"type\": \"ref\", \"volume\": \"6\", \"year\": \"2023\"}, \"text\": \"Gene interactions in human disease studies\\u2010evidence is mounting\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 47017, \"infons\": {\"elocation-id\": \"748781\", \"pub-id_doi\": \"10.3389/fgene.2021.748781\", \"pub-id_pmid\": \"34992629\", \"section_type\": \"REF\", \"source\": \"Frontiers in Genetics\", \"type\": \"ref\", \"volume\": \"12\", \"year\": \"2021\"}, \"text\": \"The effect of Alzheimer's Disease\\u2010associated Genetic variants on longevity\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 47094, \"infons\": {\"elocation-id\": \"692020\", \"pub-id_doi\": \"10.3389/fcell.2021.692020\", \"section_type\": \"REF\", \"source\": \"Frontiers in Cell and Development Biology\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2021\"}, \"text\": \"Interactions between genes from aging pathways may influence human lifespan and improve animal to human translation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 47210, \"infons\": {\"fpage\": \"120\", \"lpage\": \"126\", \"pub-id_doi\": \"10.1016/j.ynstr.2018.03.002\", \"pub-id_pmid\": \"29888307\", \"section_type\": \"REF\", \"source\": \"Neurobiology of Stress\", \"type\": \"ref\", \"volume\": \"8\", \"year\": \"2018\"}, \"text\": \"Sex differences in chronic stress responses and Alzheimer's disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 47278, \"infons\": {\"fpage\": \"4273\", \"pub-id_doi\": \"10.1038/s41467-018-05892-0\", \"section_type\": \"REF\", \"source\": \"Nature Communications\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2018\"}, \"text\": \"Uncovering the heterogeneity and temporal complexity of neurodegenerative diseases with subtype and stage inference\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 47394, \"infons\": {\"elocation-id\": \"e181670\", \"issue\": \"4\", \"pub-id_doi\": \"10.1001/jamanetworkopen.2018.1670\", \"pub-id_pmid\": \"30294719\", \"section_type\": \"REF\", \"source\": \"JAMA Network Open\", \"type\": \"ref\", \"volume\": \"1\", \"year\": \"2018\"}, \"text\": \"Sex differences in Genetic associations with longevity\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 47449, \"infons\": {\"fpage\": \"77\", \"issue\": \"1\", \"pub-id_doi\": \"10.1186/s40478-021-01177-8\", \"pub-id_pmid\": \"33902726\", \"section_type\": \"REF\", \"source\": \"Acta Neuropathologica Communications\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2021\"}, \"text\": \"Sex\\u2010specific DNA methylation differences in Alzheimer's disease pathology\", \"sentences\": [], \"annotations\": [], \"relations\": []}], \"annotations\": [], \"relations\": []}]}]\n",
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading weights: 100%|| 770/770 [00:00<00:00, 1486.31it/s, Materializing param=model.encoder_input_proj.2.1.weight]                                                 \n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    }
   ],
   "source": [
    "pmcid = \"PMC10497850\"\n",
    "file_name = \"papers/ACEL-22-e13938.pdf\"\n",
    "df_lst = extract_tables_lst_from_paper(pmcid, file_name)\n",
    "for i in [3, 4]:\n",
    "    df_lst[i-1].to_csv(f\"tables/{file_name.split('/')[-1].replace('.pdf', '')}_table_{i}.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "a7b4dc2b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{\"source\": \"PMC\", \"date\": \"20230422\", \"key\": \"pmc.key\", \"infons\": {}, \"documents\": [{\"id\": \"10115645\", \"infons\": {\"license\": \"CC BY\"}, \"passages\": [{\"offset\": 0, \"infons\": {\"article-id_doi\": \"10.1038/s41591-023-02268-w\", \"article-id_pmc\": \"10115645\", \"article-id_pmid\": \"37069360\", \"article-id_publisher-id\": \"2268\", \"fpage\": \"950\", \"issue\": \"4\", \"kwd\": \"Neurology Stroke Genome-wide association studies\", \"license\": \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article\\u2019s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article\\u2019s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit http://creativecommons.org/licenses/by/4.0/.\", \"lpage\": \"962\", \"name_0\": \"surname:Duperron;given-names:Marie-Gabrielle\", \"name_1\": \"surname:Knol;given-names:Maria J.\", \"name_10\": \"surname:Frenzel;given-names:Stefan\", \"name_11\": \"surname:Luciano;given-names:Michelle\", \"name_12\": \"surname:Hofer;given-names:Edith\", \"name_13\": \"surname:Bourgey;given-names:Mathieu\", \"name_14\": \"surname:Dueker;given-names:Nicole D.\", \"name_15\": \"surname:Delgado;given-names:Pilar\", \"name_16\": \"surname:Hilal;given-names:Saima\", \"name_17\": \"surname:Tankard;given-names:Rick M.\", \"name_18\": \"surname:Dubost;given-names:Florian\", \"name_19\": \"surname:Shin;given-names:Jean\", \"name_2\": \"surname:Le Grand;given-names:Quentin\", \"name_20\": \"surname:Saba;given-names:Yasaman\", \"name_21\": \"surname:Armstrong;given-names:Nicola J.\", \"name_22\": \"surname:Bordes;given-names:Constance\", \"name_23\": \"surname:Bastin;given-names:Mark E.\", \"name_24\": \"surname:Beiser;given-names:Alexa\", \"name_25\": \"surname:Brodaty;given-names:Henry\", \"name_26\": \"surname:B\\u00fclow;given-names:Robin\", \"name_27\": \"surname:Carrera;given-names:Caty\", \"name_28\": \"surname:Chen;given-names:Christopher\", \"name_29\": \"surname:Cheng;given-names:Ching-Yu\", \"name_3\": \"surname:Evans;given-names:Tavia E.\", \"name_30\": \"surname:Deary;given-names:Ian J.\", \"name_31\": \"surname:Gampawar;given-names:Piyush G.\", \"name_32\": \"surname:Himali;given-names:Jayandra J.\", \"name_33\": \"surname:Jiang;given-names:Jiyang\", \"name_34\": \"surname:Kawaguchi;given-names:Takahisa\", \"name_35\": \"surname:Li;given-names:Shuo\", \"name_36\": \"surname:Macalli;given-names:Melissa\", \"name_37\": \"surname:Marquis;given-names:Pascale\", \"name_38\": \"surname:Morris;given-names:Zoe\", \"name_39\": \"surname:Mu\\u00f1oz Maniega;given-names:Susana\", \"name_4\": \"surname:Mishra;given-names:Aniket\", \"name_40\": \"surname:Miyamoto;given-names:Susumu\", \"name_41\": \"surname:Okawa;given-names:Masakazu\", \"name_42\": \"surname:Paradise;given-names:Matthew\", \"name_43\": \"surname:Parva;given-names:Pedram\", \"name_44\": \"surname:Rundek;given-names:Tatjana\", \"name_45\": \"surname:Sargurupremraj;given-names:Muralidharan\", \"name_46\": \"surname:Schilling;given-names:Sabrina\", \"name_47\": \"surname:Setoh;given-names:Kazuya\", \"name_48\": \"surname:Soukarieh;given-names:Omar\", \"name_49\": \"surname:Tabara;given-names:Yasuharu\", \"name_5\": \"surname:Tsuchida;given-names:Ami\", \"name_50\": \"surname:Teumer;given-names:Alexander\", \"name_51\": \"surname:Thalamuthu;given-names:Anbupalam\", \"name_52\": \"surname:Trollor;given-names:Julian N.\", \"name_53\": \"surname:Vald\\u00e9s Hern\\u00e1ndez;given-names:Maria C.\", \"name_54\": \"surname:Vernooij;given-names:Meike W.\", \"name_55\": \"surname:V\\u00f6lker;given-names:Uwe\", \"name_56\": \"surname:Wittfeld;given-names:Katharina\", \"name_57\": \"surname:Wong;given-names:Tien Yin\", \"name_58\": \"surname:Wright;given-names:Margaret J.\", \"name_59\": \"surname:Zhang;given-names:Junyi\", \"name_6\": \"surname:Roshchupkin;given-names:Gennady\", \"name_60\": \"surname:Zhao;given-names:Wanting\", \"name_61\": \"surname:Zhu;given-names:Yi-Cheng\", \"name_62\": \"surname:Schmidt;given-names:Helena\", \"name_63\": \"surname:Sachdev;given-names:Perminder S.\", \"name_64\": \"surname:Wen;given-names:Wei\", \"name_65\": \"surname:Yoshida;given-names:Kazumichi\", \"name_66\": \"surname:Joutel;given-names:Anne\", \"name_67\": \"surname:Satizabal;given-names:Claudia L.\", \"name_68\": \"surname:Sacco;given-names:Ralph L.\", \"name_69\": \"surname:Bourque;given-names:Guillaume\", \"name_7\": \"surname:Konuma;given-names:Takahiro\", \"name_70\": \"surname:Le Grand;given-names:Quentin\", \"name_71\": \"surname:Lathrop;given-names:Mark\", \"name_72\": \"surname:Paus;given-names:Tomas\", \"name_73\": \"surname:Fernandez-Cadenas;given-names:Israel\", \"name_74\": \"surname:Yang;given-names:Qiong\", \"name_75\": \"surname:Mazoyer;given-names:Bernard\", \"name_76\": \"surname:Boutinaud;given-names:Philippe\", \"name_77\": \"surname:Okada;given-names:Yukinori\", \"name_78\": \"surname:Grabe;given-names:Hans J.\", \"name_79\": \"surname:Mather;given-names:Karen A.\", \"name_8\": \"surname:Tr\\u00e9gou\\u00ebt;given-names:David-Alexandre\", \"name_80\": \"surname:Schmidt;given-names:Reinhold\", \"name_81\": \"surname:Joliot;given-names:Marc\", \"name_82\": \"surname:Ikram;given-names:M. Arfan\", \"name_83\": \"surname:Matsuda;given-names:Fumihiko\", \"name_84\": \"surname:Tzourio;given-names:Christophe\", \"name_85\": \"surname:Wardlaw;given-names:Joanna M.\", \"name_86\": \"surname:Seshadri;given-names:Sudha\", \"name_87\": \"surname:Adams;given-names:Hieab H. H.\", \"name_88\": \"surname:Debette;given-names:St\\u00e9phanie\", \"name_9\": \"surname:Romero;given-names:Jose Rafael\", \"section_type\": \"TITLE\", \"title\": \"Subject terms\", \"type\": \"front\", \"volume\": \"29\", \"year\": \"2023\"}, \"text\": \"Genomics of perivascular space burden unravels early mechanisms of cerebral small vessel disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 97, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Perivascular space (PVS) burden is an emerging, poorly understood, magnetic resonance imaging marker of cerebral small vessel disease, a leading cause of stroke and dementia. Genome-wide association studies in up to 40,095 participants (18 population-based cohorts, 66.3\\u2009\\u00b1\\u20098.6\\u2009yr, 96.9% European ancestry) revealed 24 genome-wide significant PVS risk loci, mainly in the white matter. These were associated with white matter PVS already in young adults (N\\u2009=\\u20091,748; 22.1\\u2009\\u00b1\\u20092.3\\u2009yr) and were enriched in early-onset leukodystrophy genes and genes expressed in fetal brain endothelial cells, suggesting early-life mechanisms. In total, 53% of white matter PVS risk loci showed nominally significant associations (27% after multiple-testing correction) in a Japanese population-based cohort (N\\u2009=\\u20092,862; 68.3\\u2009\\u00b1\\u20095.3\\u2009yr). Mendelian randomization supported causal associations of high blood pressure with basal ganglia and hippocampal PVS, and of basal ganglia PVS and hippocampal PVS with stroke, accounting for blood pressure. Our findings provide insight into the biology of PVS and cerebral small vessel disease, pointing to pathways involving extracellular matrix, membrane transport and developmental processes, and the potential for genetically informed prioritization of drug targets.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 1410, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Genomic analyses of large population-based cohorts uncover the genetic determinants of perivascular space burden, an MRI marker of cerebral small vessel disease, across the lifespan, and reveal potential pathways implicated in the etiology of stroke and dementia.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 1674, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"title_1\"}, \"text\": \"Main\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 1679, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"PVS are physiological spaces surrounding small vessel walls as they run from the subarachnoid space through the brain parenchyma. Dilation of PVS observed on brain magnetic resonance imaging (MRI) is thought to be a marker of PVS dysfunction and, speculated from preclinical studies, to reflect impairment of brain fluid and waste clearance.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2021, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"PVS increase in number with age and vascular risk factors, especially hypertension. They are associated with white matter hyperintensities (WMH) of presumed vascular origin, lacunes and cerebral microbleeds, all MRI features of cerebral small vessel disease (cSVD), a leading cause of stroke and dementia with no specific mechanistic treatment to date. PVS are detected on brain MRI much earlier than WMH, lacunes or cerebral microbleeds, and are described as the earliest stage of cSVD lesions on neuropathology. Their pathophysiology is poorly understood.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2579, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"In experimental models, PVS appear to be important conduits for substrate delivery, flushing interstitial fluid, clearing metabolic waste (for example, beta-amyloid peptide) and brain fluid regulation, as part of the \\u2018glymphatic system\\u2019. These processes were described to increase during sleep. Mounting evidence suggests a major role of PVS in cerebral injury. Several studies suggested associations of PVS burden (number of visible PVS on brain MRI) with stroke, Alzheimer\\u2019s disease pathology and cerebral amyloid angiopathy (CAA). Post-stroke edema has been linked to post-stroke PVS enlargement, and in amyotrophic lateral sclerosis PVS dilation was observed and perivascular fibroblast proteins were associated with survival.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 3316, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"PVS burden is highly heritable. Identifying genetic risk variants for PVS could be a powerful tool to decipher underlying biological pathways. We conducted genome-wide association study (GWAS) meta-analyses and whole-exome/whole-genome sequencing (WES/WGS) studies of extensive PVS burden in up to 40,095 and 19,010 older community participants, respectively. Given differential associations with risk factors and neurological traits and anatomical differences, we ran analyses separately for white matter (WM)-PVS, basal ganglia (BG)-PVS and hippocampal (HIP)-PVS. We followed up identified risk loci in independent samples of young healthy adults and older Japanese community participants and examined shared genetic determinants with other vascular and neurological traits. Leveraging tissue and cell-specific gene expression databases and drug target libraries, we conducted extensive bioinformatics exploration of identified PVS risk loci.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 4261, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_1\"}, \"text\": \"Results\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 4269, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Genetic discovery\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 4287, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Twenty-one population-based cohorts were included, of which 18 were for GWAS and 8 for whole-exome association studies (Supplementary Table 1 and Methods). We tested associations of extensive PVS burden with ~8\\u2009million single-nucleotide polymorphisms (SNPs) (minor allele frequency (MAF)\\u2009\\u2265\\u20091%) in GWAS meta-analyses, gathering up to 40,095 participants (66.3\\u2009\\u00b1\\u20098.6\\u2009yr, 51.7% female, 66.7% with hypertension; Supplementary Tables 1\\u20133). We dichotomized PVS burden based on cut-offs closest to the top quartile of PVS distribution to account for differences in PVS quantification methods, image acquisition and participant characteristics (Supplementary Tables 1 and 2 and Methods). In total, 9,607 of 39,822, 9,189 of 40,000, and 9,339 of 40,095 participants had extensive PVS burden in WM, BG and hippocampus.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 5113, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Genetic variants associated with high PVS burden\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 5162, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table xmlns:xlink=\\\"http://www.w3.org/1999/xlink\\\" frame=\\\"hsides\\\" rules=\\\"groups\\\"><thead><tr><th>Region</th><th>SNP ALL</th><th>chr:position</th><th>EA/OA</th><th>EAF</th><th>Function</th><th>Nearest gene(s)</th><th>Effect (\\u03b2)<sup>a</sup></th><th>SE<sup>a</sup></th><th><italic>Z</italic>-score<sup>b</sup></th><th>Dir<sup>b</sup></th><th><italic>N</italic> ext-PVS/<italic>N</italic> total</th><th><italic>P</italic> value EUR</th><th><italic>P</italic> value All</th><th>Het <italic>P</italic> value</th></tr></thead><tbody><tr><td colspan=\\\"15\\\"><bold>PVS in white matter (WM-PVS)</bold></td></tr><tr><td>20q13.12</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs6011998\\\">rs6011998</ext-link></td><td>20:45269867</td><td>C/T</td><td>0.95</td><td>intronic</td><td><italic>SLC13A3</italic></td><td>0.087</td><td>0.009</td><td>10.65</td><td>++++</td><td>9,502/39,128</td><td><bold>1.90</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221224</bold></sup></td><td><bold>1.80</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221226</bold></sup></td><td>0.11</td></tr><tr><td>3p25.1</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs13079464\\\">rs13079464</ext-link></td><td>3:13822439</td><td>C/G</td><td>0.46</td><td>intergenic</td><td><italic>WNT7A</italic></td><td>0.026</td><td>0.004</td><td>8.70</td><td>++++</td><td>9,614/39,822</td><td><bold>8.64</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221217</bold></sup></td><td><bold>3.41</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221218</bold></sup></td><td>0.59</td></tr><tr><td>20q13.12</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs2425884\\\">rs2425884</ext-link></td><td>20:45258292</td><td>C/T</td><td>0.57</td><td>intronic</td><td><italic>SLC13A3</italic></td><td>0.029</td><td>0.004</td><td>8.63</td><td>+\\u2212+\\u2212</td><td>9,614/39,822</td><td><bold>2.60</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221218</bold></sup></td><td><bold>6.02</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221218</bold></sup></td><td>0.14</td></tr><tr><td>9q31.3</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs10817108\\\">rs10817108</ext-link><sup>c</sup></td><td>9:113658671</td><td>A/G</td><td>0.21</td><td>intronic</td><td><italic>LPAR1</italic></td><td>0.029</td><td>0.004</td><td>8.20</td><td>+++?</td><td>9,550/39,516</td><td><bold>1.07</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221215</bold></sup></td><td><bold>2.46</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221216</bold></sup></td><td>0.75</td></tr><tr><td>20q13.12</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs2425881\\\">rs2425881</ext-link></td><td>20:45255618</td><td>A/G</td><td>0.83</td><td>intronic</td><td><italic>SLC13A3</italic></td><td>0.033</td><td>0.005</td><td>7.68</td><td>+\\u2212+?</td><td>9,496/39,087</td><td><bold>2.02</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221215</bold></sup></td><td><bold>1.59</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221214</bold></sup></td><td>0.06</td></tr><tr><td>3q21.2</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs3772833\\\">rs3772833</ext-link></td><td>3:124518362</td><td>G/A</td><td>0.83</td><td>intronic</td><td><italic>ITGB5</italic>, <italic>UMPS</italic></td><td>0.032</td><td>0.005</td><td>7.67</td><td>+++?</td><td>9,496/39,087</td><td><bold>2.15</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221213</bold></sup></td><td><bold>1.76</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221214</bold></sup></td><td>0.39</td></tr><tr><td>20q13.12</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs112407396\\\">rs112407396</ext-link></td><td>20:45276381</td><td>T/A</td><td>0.03</td><td>intronic</td><td><italic>SLC13A3</italic></td><td>0.078</td><td>0.012</td><td>6.91</td><td>+???</td><td>8,426/34,530</td><td><bold>4.81</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221212</bold></sup></td><td><bold>4.81</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221212</bold></sup></td><td>1.00</td></tr><tr><td>1q41</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs10494988\\\">rs10494988</ext-link></td><td>1:215141570</td><td>C/T</td><td>0.63</td><td>intergenic</td><td><italic>CENPF</italic>, <italic>KCNK2</italic></td><td>0.021</td><td>0.004</td><td>6.54</td><td>++++</td><td>9,614/39,822</td><td><bold>2.23</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221210</bold></sup></td><td><bold>6.03</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221211</bold></sup></td><td>0.69</td></tr><tr><td>20q13.12</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs72485816\\\">rs72485816</ext-link><sup>d</sup></td><td>20:45314435</td><td>T/C</td><td>0.96</td><td>UTR3</td><td><italic>TP53RK</italic>, <italic>SLC13A3</italic></td><td>0.059</td><td>0.010</td><td>6.45</td><td>++?\\u2212</td><td>9,114/37,342</td><td><bold>1.47</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221210</bold></sup></td><td><bold>1.12</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221210</bold></sup></td><td>0.87</td></tr><tr><td>15q25.3</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs8041189\\\">rs8041189</ext-link></td><td>15:85686327</td><td>G/A</td><td>0.70</td><td>intergenic</td><td><italic>PDE8A</italic></td><td>0.022</td><td>0.004</td><td>6.44</td><td>+\\u2212??</td><td>9,486/39,315</td><td><bold>7.30</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221211</bold></sup></td><td><bold>1.24</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221210</bold></sup></td><td>0.31</td></tr><tr><td>3p25</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs4685022\\\">rs4685022</ext-link></td><td>3:13832611</td><td>G/A</td><td>0.65</td><td>intergenic</td><td><italic>WNT7A</italic></td><td>0.019</td><td>0.004</td><td>6.40</td><td>+++?</td><td>9,576/39,654</td><td><bold>2.36</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221209</bold></sup></td><td><bold>1.58</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221210</bold></sup></td><td>0.11</td></tr><tr><td>2p16.1</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs7596872\\\">rs7596872</ext-link></td><td>2:56128091</td><td>C/A</td><td>0.90</td><td>intronic</td><td><italic>EFEMP1</italic></td><td>0.033</td><td>0.006</td><td>6.31</td><td>+\\u2212??</td><td>9,333/38,442</td><td><bold>1.00</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221210</bold></sup></td><td><bold>2.80</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221210</bold></sup></td><td>0.11</td></tr><tr><td>17q21.31</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs1126642\\\">rs1126642</ext-link></td><td>17:42989063</td><td>C/T</td><td>0.96</td><td>exonic</td><td><italic>GFAP</italic></td><td>0.051</td><td>0.009</td><td>6.23</td><td>+?+?</td><td>9,119/37,466</td><td><bold>6.19</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221210</bold></sup></td><td><bold>4.67</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221210</bold></sup></td><td>0.72</td></tr><tr><td>3q29</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs687610\\\">rs687610</ext-link><sup>d</sup></td><td>3:193515781</td><td>G/C</td><td>0.43</td><td>intergenic</td><td><italic>OPA1</italic></td><td>0.021</td><td>0.004</td><td>6.20</td><td>+++\\u2212</td><td>9,614/39,822</td><td><bold>2.99</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221210</bold></sup></td><td><bold>5.81</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221210</bold></sup></td><td>0.76</td></tr><tr><td>6p25.2</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs4959689\\\">rs4959689</ext-link></td><td>6:2617122</td><td>C/A</td><td>0.58</td><td>intergenic</td><td><italic>C6orf195</italic></td><td>0.020</td><td>0.004</td><td>6.03</td><td>++++</td><td>9,582/39,695</td><td><bold>3.37</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221209</bold></sup></td><td><bold>1.63</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221209</bold></sup></td><td>1.00</td></tr><tr><td>20q13.12</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs56104388\\\">rs56104388</ext-link></td><td>20:45302135</td><td>T/C</td><td>0.99</td><td>intronic</td><td><italic>SLC13A3</italic></td><td>0.101</td><td>0.017</td><td>5.85</td><td>+???</td><td>7,626/30,916</td><td><bold>4.80</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221209</bold></sup></td><td><bold>4.80</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221209</bold></sup></td><td>1.00</td></tr><tr><td>11q13.3</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs12417836\\\">rs12417836</ext-link></td><td>11:70089700</td><td>T/C</td><td>0.07</td><td>intergenic</td><td><italic>FADD</italic>, <italic>PPFIA1</italic></td><td>0.034</td><td>0.007</td><td>5.58</td><td>+\\u2212+?</td><td>9,464/38,960</td><td><bold>1.56</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221208</bold></sup></td><td><bold>2.47</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221208</bold></sup></td><td>0.40</td></tr><tr><td>8p11.21</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs2923437\\\">rs2923437</ext-link><sup>d</sup></td><td>8:42425399</td><td>A/C</td><td>0.41</td><td>intergenic</td><td><italic>SMIM19</italic>, <italic>CHRNB3</italic>, <italic>SLC20A2</italic></td><td>0.018</td><td>0.004</td><td>5.49</td><td>++\\u2212\\u2212</td><td>9,614/39,822</td><td><bold>4.73</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221208</bold></sup></td><td><bold>4.08</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221208</bold></sup></td><td>0.14</td></tr><tr><td>6p25.3</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs1922930\\\">rs1922930</ext-link></td><td>6:1364691</td><td>C/A</td><td>0.12</td><td>intergenic</td><td><italic>FOXQ1</italic>, <italic>FOXF2</italic></td><td>0.029</td><td>0.006</td><td>5.47</td><td>++??</td><td>9,406/38,748</td><td><bold>3.60</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221208</bold></sup></td><td><bold>4.62</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221208</bold></sup></td><td>0.48</td></tr><tr><td>19p13.11</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs2385089\\\">rs2385089</ext-link></td><td>19:18550434</td><td>A/C</td><td>0.74</td><td>intergenic</td><td><italic>ISYNA1</italic>, <italic>ELL</italic>, <italic>LRRC25</italic><sup>c</sup></td><td>0.023</td><td>0.005</td><td>5.49</td><td>+++\\u2212</td><td>9,614/39,822</td><td><bold>4.14</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221208</bold></sup></td><td>5.73\\u2009\\u00d7\\u200910<sup>\\u221208</sup></td><td>0.57</td></tr><tr><td>7q33</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs10954468\\\">rs10954468</ext-link></td><td>7:134434661</td><td>C/A</td><td>0.40</td><td>intergenic</td><td><italic>BPGM</italic>, <italic>CALD1</italic><sup>c</sup></td><td>0.019</td><td>0.004</td><td>5.52</td><td>+\\u2212?+</td><td>9,524/39,483</td><td><bold>3.39</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221208</bold></sup></td><td>8.79\\u2009\\u00d7\\u200910<sup>\\u221208</sup></td><td>0.29</td></tr><tr><td colspan=\\\"15\\\"><bold>PVS in basal ganglia (BG-PVS)</bold></td></tr><tr><td>2q33.2</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs4675310\\\">rs4675310</ext-link><sup>d</sup></td><td>2:203880834</td><td>A/G</td><td>0.87</td><td>intronic</td><td><italic>NBEAL1</italic>, <italic>ICA1L</italic></td><td>0.027</td><td>0.005</td><td>5.92</td><td>++??</td><td>9,011/39,243</td><td><bold>2.71</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221209</bold></sup></td><td><bold>3.27</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221209</bold></sup></td><td>0.64</td></tr><tr><td>3q26.31</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs6769442\\\">rs6769442</ext-link></td><td>3:171565463</td><td>G/A</td><td>0.75</td><td>intronic</td><td><italic>TMEM212</italic></td><td>0.020</td><td>0.004</td><td>5.74</td><td>++?+</td><td>9,101/39,788</td><td><bold>1.68</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221208</bold></sup></td><td><bold>9.34</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221209</bold></sup></td><td>0.96</td></tr><tr><td colspan=\\\"15\\\"><bold>PVS in hippocampus (HIP-PVS)</bold></td></tr><tr><td>1q25.3</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs10797812\\\">rs10797812</ext-link><sup>d</sup></td><td>1:182984597</td><td>A/G</td><td>0.54</td><td>intergenic</td><td><italic>SHCBP1L</italic>, <italic>LAMC1</italic></td><td>0.027</td><td>0.004</td><td>7.84</td><td>++++</td><td>9,399/40,095</td><td><bold>1.67</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221214</bold></sup></td><td><bold>4.39</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221215</bold></sup></td><td>0.68</td></tr><tr><td>2p16.1</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs78857879\\\">rs78857879</ext-link><sup>d</sup></td><td>2:56135099</td><td>G/A</td><td>0.90</td><td>intronic</td><td><italic>EFEMP1</italic></td><td>0.038</td><td>0.006</td><td>6.43</td><td>+???</td><td>9,033/38,008</td><td><bold>8.20</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221211</bold></sup></td><td><bold>1.31</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221210</bold></sup></td><td>1.00</td></tr><tr><td>1q41</td><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs6540873\\\">rs6540873</ext-link></td><td>1:215137222</td><td>A/C</td><td>0.62</td><td>intergenic</td><td><italic>CENPF</italic>, <italic>KCNK2</italic></td><td>0.020</td><td>0.004</td><td>5.95</td><td>+\\u2212\\u2212+</td><td>9,399/40,095</td><td><bold>1.38</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221209</bold></sup></td><td><bold>2.72</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221209</bold></sup></td><td>0.11</td></tr></tbody></table>\\n\"}, \"text\": \"Region\\tSNP ALL\\tchr:position\\tEA/OA\\tEAF\\tFunction\\tNearest gene(s)\\tEffect (\\u03b2)a\\tSEa\\tZ-scoreb\\tDirb\\tN ext-PVS/N total\\tP value EUR\\tP value All\\tHet P value\\t \\tPVS in white matter (WM-PVS)\\t \\t20q13.12\\trs6011998\\t20:45269867\\tC/T\\t0.95\\tintronic\\tSLC13A3\\t0.087\\t0.009\\t10.65\\t++++\\t9,502/39,128\\t1.90\\u2009\\u00d7\\u200910\\u221224\\t1.80\\u2009\\u00d7\\u200910\\u221226\\t0.11\\t \\t3p25.1\\trs13079464\\t3:13822439\\tC/G\\t0.46\\tintergenic\\tWNT7A\\t0.026\\t0.004\\t8.70\\t++++\\t9,614/39,822\\t8.64\\u2009\\u00d7\\u200910\\u221217\\t3.41\\u2009\\u00d7\\u200910\\u221218\\t0.59\\t \\t20q13.12\\trs2425884\\t20:45258292\\tC/T\\t0.57\\tintronic\\tSLC13A3\\t0.029\\t0.004\\t8.63\\t+\\u2212+\\u2212\\t9,614/39,822\\t2.60\\u2009\\u00d7\\u200910\\u221218\\t6.02\\u2009\\u00d7\\u200910\\u221218\\t0.14\\t \\t9q31.3\\trs10817108c\\t9:113658671\\tA/G\\t0.21\\tintronic\\tLPAR1\\t0.029\\t0.004\\t8.20\\t+++?\\t9,550/39,516\\t1.07\\u2009\\u00d7\\u200910\\u221215\\t2.46\\u2009\\u00d7\\u200910\\u221216\\t0.75\\t \\t20q13.12\\trs2425881\\t20:45255618\\tA/G\\t0.83\\tintronic\\tSLC13A3\\t0.033\\t0.005\\t7.68\\t+\\u2212+?\\t9,496/39,087\\t2.02\\u2009\\u00d7\\u200910\\u221215\\t1.59\\u2009\\u00d7\\u200910\\u221214\\t0.06\\t \\t3q21.2\\trs3772833\\t3:124518362\\tG/A\\t0.83\\tintronic\\tITGB5, UMPS\\t0.032\\t0.005\\t7.67\\t+++?\\t9,496/39,087\\t2.15\\u2009\\u00d7\\u200910\\u221213\\t1.76\\u2009\\u00d7\\u200910\\u221214\\t0.39\\t \\t20q13.12\\trs112407396\\t20:45276381\\tT/A\\t0.03\\tintronic\\tSLC13A3\\t0.078\\t0.012\\t6.91\\t+???\\t8,426/34,530\\t4.81\\u2009\\u00d7\\u200910\\u221212\\t4.81\\u2009\\u00d7\\u200910\\u221212\\t1.00\\t \\t1q41\\trs10494988\\t1:215141570\\tC/T\\t0.63\\tintergenic\\tCENPF, KCNK2\\t0.021\\t0.004\\t6.54\\t++++\\t9,614/39,822\\t2.23\\u2009\\u00d7\\u200910\\u221210\\t6.03\\u2009\\u00d7\\u200910\\u221211\\t0.69\\t \\t20q13.12\\trs72485816d\\t20:45314435\\tT/C\\t0.96\\tUTR3\\tTP53RK, SLC13A3\\t0.059\\t0.010\\t6.45\\t++?\\u2212\\t9,114/37,342\\t1.47\\u2009\\u00d7\\u200910\\u221210\\t1.12\\u2009\\u00d7\\u200910\\u221210\\t0.87\\t \\t15q25.3\\trs8041189\\t15:85686327\\tG/A\\t0.70\\tintergenic\\tPDE8A\\t0.022\\t0.004\\t6.44\\t+\\u2212??\\t9,486/39,315\\t7.30\\u2009\\u00d7\\u200910\\u221211\\t1.24\\u2009\\u00d7\\u200910\\u221210\\t0.31\\t \\t3p25\\trs4685022\\t3:13832611\\tG/A\\t0.65\\tintergenic\\tWNT7A\\t0.019\\t0.004\\t6.40\\t+++?\\t9,576/39,654\\t2.36\\u2009\\u00d7\\u200910\\u221209\\t1.58\\u2009\\u00d7\\u200910\\u221210\\t0.11\\t \\t2p16.1\\trs7596872\\t2:56128091\\tC/A\\t0.90\\tintronic\\tEFEMP1\\t0.033\\t0.006\\t6.31\\t+\\u2212??\\t9,333/38,442\\t1.00\\u2009\\u00d7\\u200910\\u221210\\t2.80\\u2009\\u00d7\\u200910\\u221210\\t0.11\\t \\t17q21.31\\trs1126642\\t17:42989063\\tC/T\\t0.96\\texonic\\tGFAP\\t0.051\\t0.009\\t6.23\\t+?+?\\t9,119/37,466\\t6.19\\u2009\\u00d7\\u200910\\u221210\\t4.67\\u2009\\u00d7\\u200910\\u221210\\t0.72\\t \\t3q29\\trs687610d\\t3:193515781\\tG/C\\t0.43\\tintergenic\\tOPA1\\t0.021\\t0.004\\t6.20\\t+++\\u2212\\t9,614/39,822\\t2.99\\u2009\\u00d7\\u200910\\u221210\\t5.81\\u2009\\u00d7\\u200910\\u221210\\t0.76\\t \\t6p25.2\\trs4959689\\t6:2617122\\tC/A\\t0.58\\tintergenic\\tC6orf195\\t0.020\\t0.004\\t6.03\\t++++\\t9,582/39,695\\t3.37\\u2009\\u00d7\\u200910\\u221209\\t1.63\\u2009\\u00d7\\u200910\\u221209\\t1.00\\t \\t20q13.12\\trs56104388\\t20:45302135\\tT/C\\t0.99\\tintronic\\tSLC13A3\\t0.101\\t0.017\\t5.85\\t+???\\t7,626/30,916\\t4.80\\u2009\\u00d7\\u200910\\u221209\\t4.80\\u2009\\u00d7\\u200910\\u221209\\t1.00\\t \\t11q13.3\\trs12417836\\t11:70089700\\tT/C\\t0.07\\tintergenic\\tFADD, PPFIA1\\t0.034\\t0.007\\t5.58\\t+\\u2212+?\\t9,464/38,960\\t1.56\\u2009\\u00d7\\u200910\\u221208\\t2.47\\u2009\\u00d7\\u200910\\u221208\\t0.40\\t \\t8p11.21\\trs2923437d\\t8:42425399\\tA/C\\t0.41\\tintergenic\\tSMIM19, CHRNB3, SLC20A2\\t0.018\\t0.004\\t5.49\\t++\\u2212\\u2212\\t9,614/39,822\\t4.73\\u2009\\u00d7\\u200910\\u221208\\t4.08\\u2009\\u00d7\\u200910\\u221208\\t0.14\\t \\t6p25.3\\trs1922930\\t6:1364691\\tC/A\\t0.12\\tintergenic\\tFOXQ1, FOXF2\\t0.029\\t0.006\\t5.47\\t++??\\t9,406/38,748\\t3.60\\u2009\\u00d7\\u200910\\u221208\\t4.62\\u2009\\u00d7\\u200910\\u221208\\t0.48\\t \\t19p13.11\\trs2385089\\t19:18550434\\tA/C\\t0.74\\tintergenic\\tISYNA1, ELL, LRRC25c\\t0.023\\t0.005\\t5.49\\t+++\\u2212\\t9,614/39,822\\t4.14\\u2009\\u00d7\\u200910\\u221208\\t5.73\\u2009\\u00d7\\u200910\\u221208\\t0.57\\t \\t7q33\\trs10954468\\t7:134434661\\tC/A\\t0.40\\tintergenic\\tBPGM, CALD1c\\t0.019\\t0.004\\t5.52\\t+\\u2212?+\\t9,524/39,483\\t3.39\\u2009\\u00d7\\u200910\\u221208\\t8.79\\u2009\\u00d7\\u200910\\u221208\\t0.29\\t \\tPVS in basal ganglia (BG-PVS)\\t \\t2q33.2\\trs4675310d\\t2:203880834\\tA/G\\t0.87\\tintronic\\tNBEAL1, ICA1L\\t0.027\\t0.005\\t5.92\\t++??\\t9,011/39,243\\t2.71\\u2009\\u00d7\\u200910\\u221209\\t3.27\\u2009\\u00d7\\u200910\\u221209\\t0.64\\t \\t3q26.31\\trs6769442\\t3:171565463\\tG/A\\t0.75\\tintronic\\tTMEM212\\t0.020\\t0.004\\t5.74\\t++?+\\t9,101/39,788\\t1.68\\u2009\\u00d7\\u200910\\u221208\\t9.34\\u2009\\u00d7\\u200910\\u221209\\t0.96\\t \\tPVS in hippocampus (HIP-PVS)\\t \\t1q25.3\\trs10797812d\\t1:182984597\\tA/G\\t0.54\\tintergenic\\tSHCBP1L, LAMC1\\t0.027\\t0.004\\t7.84\\t++++\\t9,399/40,095\\t1.67\\u2009\\u00d7\\u200910\\u221214\\t4.39\\u2009\\u00d7\\u200910\\u221215\\t0.68\\t \\t2p16.1\\trs78857879d\\t2:56135099\\tG/A\\t0.90\\tintronic\\tEFEMP1\\t0.038\\t0.006\\t6.43\\t+???\\t9,033/38,008\\t8.20\\u2009\\u00d7\\u200910\\u221211\\t1.31\\u2009\\u00d7\\u200910\\u221210\\t1.00\\t \\t1q41\\trs6540873\\t1:215137222\\tA/C\\t0.62\\tintergenic\\tCENPF, KCNK2\\t0.020\\t0.004\\t5.95\\t+\\u2212\\u2212+\\t9,399/40,095\\t1.38\\u2009\\u00d7\\u200910\\u221209\\t2.72\\u2009\\u00d7\\u200910\\u221209\\t0.11\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 9100, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"EA, effect allele; OA, other allele; EAF, effect allele frequency; Z-scores of the sample size-weighted cross-ancestry meta-analysis are represented, except for the two SNPs reaching genome-wide significance in Europeans only (rs2385089, rs10954468) for which the European meta-analysis Z-score is reported; dir, the association direction of the EA with the phenotype (extensive PVS burden versus the rest) for European, Hispanic, Asian and African-American ancestry studies, in this order; N ext-PVS, the number of participants with extensive PVS burden in the cross-ancestry meta-analysis, in each location (WM-PVS, BG-PVS, HIP-PVS); N total, the total number of participants in the cross-ancestry meta-analysis; P value EUR, P value in the European meta-analysis; P value All, P value in the cross-ancestry meta-analysis; Het P value corresponds to the heterogeneity P value in the meta-analysis (except for rs2385089 and rs10954468 for which the European meta-analysis Het P value is reported); P values for genome-wide significant loci (P\\u2009<\\u20095\\u2009\\u00d7\\u200910\\u22128) are in bold; PVS GWAS analyses in individual cohorts were adjusted for age, sex and intracranial volume (or brain parenchymal fraction for ASPS), principal components of population stratification, and study site.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 10380, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"aFrom cross-ancestry inverse variance-weighted meta-analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 10442, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"bFrom cross-ancestry Z-score-based meta-analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 10492, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"cGenome-wide significant association in Europeans only.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 10548, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"dFor these loci, the lead SNP was different in the European meta-analysis (Cross-ancestry lead SNP \\u2192 European lead SNP: rs72485816\\u2192rs6094423; rs687610\\u2192rs6444747; rs2923437\\u2192rs62509329; rs4675310\\u2192rs140244541; rs10797812\\u2192rs2022392; rs78857879\\u2192rs7596872); the P value corresponding to the European lead SNP is reported under \\\"P value EUR\\\" (r\\u00b2\\u2009>\\u20090.50 between the European and cross-ancestry lead SNPs at these loci).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 10979, \"infons\": {\"file\": \"41591_2023_2268_Fig1_HTML.jpg\", \"id\": \"Fig1\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Illustration of extensive PVS burden and results of the cross-ancestry PVS GWAS meta-analysis, MTAG analysis and gene-based test.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 11109, \"infons\": {\"file\": \"41591_2023_2268_Fig1_HTML.jpg\", \"id\": \"Fig1\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"a, Extensive PVS burden (arrows) in WM (top, WM-PVS), BG (middle, BG-PVS) and hippocampus (bottom, HIP-PVS) on T1-weighted axial magnetic resonance images. b, Circular Manhattan plot. The inner circle corresponds to the cross-ancestry GWAS meta-analyses results, the middle circle to the results of the MTAG analysis and the outer circle to gene-based test results. Results for WM-PVS are in blue, for BG-PVS in purple and for HIP-PVS in green. The gray line corresponds to the genome-wide significance threshold (P\\u2009=\\u20095\\u2009\\u00d7\\u200910\\u22128, two-sided, correcting for multiple testing at the genome-wide level).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 11718, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"The GWAS meta-analysis comprised 17 cohorts from the Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) consortium (N\\u2009\\u2264\\u200911,511), with PVS quantification primarily on visual rating scales, and UK Biobank (UKB, N\\u2009\\u2264\\u200928,655), with computational PVS quantification (Methods). Participants were of European (N\\u2009=\\u200938,871), Hispanic (N\\u2009=\\u2009717), East-Asian (N\\u2009=\\u2009339) and African-American (N\\u2009=\\u2009168) ancestry. We identified 22 independent genome-wide significant risk loci for extensive PVS burden (WM-PVS: 19; BG-PVS: 2; HIP-PVS: 3 (2 shared with WM-PVS)) and two additional risk loci for WM-PVS in Europeans only, leading to 24 independent signals (Table 1, Fig. 1, Extended Data Fig. 1 and Supplementary Fig. 1). There was no systematic inflation of association statistics (Supplementary Table 4 and Extended Data Fig. 1).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 12578, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We performed conditional logistic regression using Genome-wide Complex Trait Analysis (GCTA)-COJO (Methods) to seek independent association signals within genome-wide significant loci. Consistent with linkage disequilibrium (LD)-based clumping, this identified two independent signals at chr3p25.1 (WNT7A) and six at chr20q13.12 (SLC13A3; Supplementary Fig. 1 and Supplementary Table 5a), four of which with low-frequency variants (Table 1). The six polymorphisms at chr20q13.12 generated eight haplotypes with haplotypic R2 (percentage of haplotypic variability explained by observed genotypes)\\u2009>\\u20090.7 in the Three-City Dijon Study (3C-Dijon) cohort, of European ancestry (N\\u2009=\\u20091,500; Supplementary Table 5b). The two common rs2425881-A and rs2425884-C alleles, in very low LD with each other (r2\\u2009(a measure of correlation of alleles for two genetic variants) ~\\u20090.05, D\\u2032 (a pairwise r2 standardized for allele frequencies)\\u2009~\\u20090.50), generated a common haplotype that was more frequent in individuals with extensive WM-PVS than in those without (0.50 versus 0.47, odds ratio (OR)\\u2009=\\u20091.19 (95% confidence interval (95% CI), 0.99\\u20131.43)). The effect of this haplotype was amplified by 1.7 in the presence of the rs112407396-T allele (MAF\\u2009=\\u20090.03), which has a high probability of being a regulatory variant (HaploReg, GTex, RegulomeDB). Next, to account for allelic heterogeneity between ancestries, we conducted cross-ancestry meta-analyses with MR-MEGA (Methods). There were no loci showing high heterogeneity in allelic effects across ancestries (PHet\\u2009<\\u20090.01) and reaching genome-wide significance (Supplementary Table 6).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14228, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Using MAGMA and VEGAS, we performed gene-based association analyses in participants of European ancestry, testing the combined association of variants within a gene with PVS (Methods). MAGMA identified 28 gene-wide significant associations (P\\u2009<\\u20092.63\\u2009\\u00d7\\u200910\\u22126), of which 12 in 8 loci did not reach genome-wide significance in the GWAS (WM-PVS: 3 (INS-IGF2/IGF2, PRKAG2, LRP4/CKAP5); BG-PVS: 4 (SH3PXD2A, WNT3, ZMYND15, KCNRG/TRIM13/SPRYD7); and HIP-PVS: 1 (PDZRN4); Fig. 1 and Supplementary Table 7). VEGAS identified one additional gene (NSF) for BG-PVS (same locus as WNT3; Supplementary Table 7). All were in suggestive GWAS loci (P\\u2009<\\u20095\\u2009\\u00d7\\u200910\\u22126; Supplementary Table 8).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14919, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Using LD-score regression, we estimated heritability at 11% for WM-PVS, 5% for BG-PVS and 8% for HIP-PVS (Methods and Supplementary Table 9). We found moderate genetic correlation between BG-PVS and HIP-PVS (rg (SE)\\u2009=\\u20090.63 (0.14), P\\u2009=\\u20097.23\\u2009\\u00d7\\u200910\\u22126), and modest genetic correlation of WM-PVS with BG-PVS (rg (SE)\\u2009=\\u20090.24 (0.12), P\\u2009=\\u20090.055) and HIP-PVS (rg (SE)\\u2009=\\u20090.27 (0.09), P\\u2009=\\u20090.003). The genetic correlation of PVS in CHARGE and UKB was moderate to high for WM-PVS and HIP-PVS and weaker for BG-PVS (Supplementary Table 10). Associations with genome-wide significant PVS loci were highly consistent between the UKB and CHARGE contributions and between the two dichotomous and the continuous PVS measures in UKB (Methods and Supplementary Tables 11 and 12). In sensitivity analyses in two representative cohorts (UKB and 3C-Dijon), continuous and dichotomous PVS measures were strongly correlated (Spearman\\u2019s \\u03c1, 0.61\\u20130.80; Supplementary Table 13).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 15903, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"To increase statistical power, we conducted secondary multivariate association analyses using Multi-Trait Analysis of GWAS (MTAG) (Methods), including summary statistics from GWAS of other cSVD markers (WMH volume, lacunes; Supplementary Table 14). We observed the highest gain in power for BG-PVS: ten additional loci reached genome-wide significance, of which two also for HIP-PVS (STN1, DEGS2/EVL). Two MTAG BG-PVS loci (CACNB2, NSF/WNT3) and one MTAG WM-PVS locus (VWA2) were not described before with any MRI marker of cSVD. Six loci showed greater significance in MTAG than with PVS, WMH volume or lacunes alone: at VWA2 (WM-PVS); SH3PXD2A/STN1, COL4A2, CACNB2 and NSF/WNT3 (BG-PVS); and DEGS2/EVL (BG-PVS, HIP-PVS).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 16626, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Using WES data and exome content of WGS data in 19,010 participants from UKB and the Brain Imaging, cognition, dementia, and next-generation genomics (BRIDGET) consortium (Methods and Supplementary Table 1), of whom 4,531, 4,424 and 4,497 had extensive PVS in WM, BG and hippocampus, we identified 19 variants in the chr1q25.3 locus associated with HIP-PVS, including two missense variants (rs20563 and rs20558) and one splice donor insertion (rs34133998) in LAMC1 at P\\u2009<\\u20095\\u2009\\u00d7\\u200910\\u22128, in strong LD with the GWAS sentinel variant (Supplementary Table 15a). Gene-based burden tests exploring protein-modifying rare variants (MAF\\u2009<\\u20090.01) did not identify any gene-wide significant association (Supplementary Table 15b).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 17355, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Follow-up of findings across the lifespan and ancestries\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 17412, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Association of genome-wide significant WM- and BG-PVS risk loci with PVS burden across the lifespan (i-Share study, N\\u2009=\\u20091,748) and across ancestries (Nagahama study, N\\u2009=\\u20092,862)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 17597, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table xmlns:xlink=\\\"http://www.w3.org/1999/xlink\\\" frame=\\\"hsides\\\" rules=\\\"groups\\\"><thead><tr><th colspan=\\\"4\\\">GWAS meta-analysis</th><th colspan=\\\"2\\\">i-Share (dichotomous)</th><th colspan=\\\"2\\\">i-Share (continuous)</th><th colspan=\\\"2\\\">Nagahama (dichotomous)</th><th colspan=\\\"2\\\">Nagahama (continuous)</th></tr><tr><th>SNP</th><th>chr:position</th><th>EA/OA</th><th>Nearest gene(s)</th><th>OR (95% CI)</th><th><italic>P</italic> value</th><th>\\u03b2 (SE)</th><th><italic>P</italic> value</th><th>OR (95% CI)</th><th><italic>P</italic> value</th><th>\\u03b2 (SE)</th><th><italic>P</italic> value</th></tr></thead><tbody><tr><td colspan=\\\"12\\\"><bold>PVS in white matter (WM-PVS)</bold></td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs6011998\\\">rs6011998</ext-link></td><td>20:45269867</td><td>C/T</td><td><italic>SLC13A3</italic></td><td>1.26 (0.83\\u20131.92)</td><td>0.28</td><td>0.164 (0.04)</td><td><bold>4.20</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221205</bold></sup><sup>a</sup></td><td>1.69 (1.33\\u20132.13)</td><td><bold>1.22</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221205</bold></sup><sup>a</sup></td><td>0.037 (0.008)</td><td><bold>6.21</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221207</bold></sup><sup>a</sup></td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs13079464\\\">rs13079464</ext-link></td><td>3:13822439</td><td>C/G</td><td><italic>WNT7A</italic></td><td>1.12 (0.91\\u20131.40)</td><td>0.29</td><td>0.014 (0.02)</td><td>0.50</td><td>1.16 (0.97\\u20131.40)</td><td>0.11</td><td>0.015 (0.006)</td><td><bold>1.50</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221202</bold></sup></td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs2425884\\\">rs2425884</ext-link></td><td>20:45258292</td><td>C/T</td><td><italic>SLC13A3</italic></td><td>1.18 (0.95\\u20131.45)</td><td>0.13</td><td>0.077 (0.02)</td><td><bold>2.98</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u2212</bold></sup><sup><bold>04</bold>a</sup></td><td>1.29 (1.09\\u20131.52)</td><td><bold>3.48</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221203</bold></sup></td><td>0.026 (0.005)</td><td><bold>1.77</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221206</bold></sup><sup>a</sup></td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs10817108\\\">rs10817108</ext-link></td><td>9:113658671</td><td>A/G</td><td><italic>LPAR1</italic></td><td>0.90 (0.69\\u20131.17)</td><td>0.44</td><td>0.058 (0.03)</td><td><bold>2.23</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u2212</bold></sup><sup><bold>02</bold></sup></td><td>1.18 (0.98\\u20131.43)</td><td>0.07</td><td>0.017 (0.006)</td><td><bold>4.10</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221203</bold></sup></td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs2425881\\\">rs2425881</ext-link></td><td>20:45255618</td><td>A/G</td><td><italic>SLC13A3</italic></td><td>1.47 (1.03\\u20132.01)</td><td><bold>1.40</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221202</bold></sup></td><td>0.063 (0.03)</td><td><bold>2.62</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221202</bold></sup></td><td>1.18 (1.01\\u20131.37)</td><td><bold>3.66</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221202</bold></sup></td><td>0.014 (0.005)</td><td><bold>4.68</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221203</bold></sup></td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs3772833\\\">rs3772833</ext-link></td><td>3:124518362</td><td>G/A</td><td><italic>ITGB5</italic>, <italic>UMPS</italic></td><td>1.22 (0.89\\u20131.66)</td><td>0.21</td><td>0.006 (0.03)</td><td>0.85</td><td>1.06 (0.88\\u20131.29)</td><td>0.51</td><td>0.008 (0.006)</td><td>0.16</td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs112407396\\\">rs112407396</ext-link></td><td>20:45276381</td><td>T/A</td><td><italic>SLC13A3</italic></td><td>1.47 (0.77\\u20132.78)</td><td>0.24</td><td>0.147 (0.07)</td><td><bold>3.13</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221202</bold></sup></td><td>NA</td><td>NA</td><td>NA</td><td>NA</td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs10494988\\\">rs10494988</ext-link></td><td>1:215141570</td><td>C/T</td><td><italic>CENPF</italic>, <italic>KCNK2</italic></td><td>1.18 (0.95\\u20131.47)</td><td>0.14</td><td>0.079 (0.02)</td><td><bold>1.94</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221204</bold></sup><sup>a</sup></td><td>1.01 (0.86\\u20131.18)</td><td>0.90</td><td>\\u22120.002 (0.005)</td><td>0.67</td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs72485816\\\">rs72485816</ext-link></td><td>20:45314435</td><td>T/C</td><td><italic>TP53RK</italic>, <italic>SLC13A3</italic></td><td>1.01 (0.56\\u20131.80)</td><td>0.98</td><td>0.093 (0.06)</td><td>0.095</td><td>1.32 (1.10\\u20131.59)</td><td><bold>2.83</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221203</bold></sup></td><td>0.033 (0.006)</td><td><bold>1.91</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221208</bold></sup><sup>b</sup></td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs8041189\\\">rs8041189</ext-link></td><td>15:85686327</td><td>G/A</td><td><italic>PDE8A</italic></td><td>1.14 (0.89\\u20131.44)</td><td>0.30</td><td>0.041 (0.02)</td><td>0.073</td><td>1.67 (0.84\\u20133.33)</td><td>0.14</td><td>0.046 (0.021)</td><td><bold>2.40</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221202</bold></sup></td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs4685022\\\">rs4685022</ext-link></td><td>3:13832611</td><td>G/A</td><td><italic>WNT7A</italic></td><td>1.12 (0.88\\u20131.42)</td><td>0.34</td><td>0.023 (0.02)</td><td>0.31</td><td>1.15 (0.97\\u20131.36)</td><td>0.10</td><td>0.010 (0.005)</td><td>0.075<sup>c</sup></td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs7596872\\\">rs7596872</ext-link></td><td>2:56128091</td><td>C/A</td><td><italic>EFEMP1</italic></td><td>1.65 (1.10\\u20132.46)</td><td><bold>1.14</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221202</bold></sup></td><td><bold>0.089 (0.03)</bold></td><td><bold>1.10</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221202</bold></sup></td><td>NA</td><td>NA</td><td>NA</td><td>NA</td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs1126642\\\">rs1126642</ext-link></td><td>17:42989063</td><td>C/T</td><td><italic>GFAP</italic></td><td>1.13 (0.65\\u20131.97)</td><td>0.67</td><td>0.127 (0.05)</td><td><bold>1.27</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221202</bold></sup></td><td>1.35 (1.09\\u20131.67)</td><td>0.11</td><td>0.033 (0.007)</td><td><bold>9.88</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221207</bold></sup><sup>a</sup></td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs687610\\\">rs687610</ext-link></td><td>3:193515781</td><td>G/C</td><td><italic>OPA1</italic></td><td>1.46 (1.18\\u20131.80)</td><td><bold>4.88</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221204</bold></sup><sup>a</sup></td><td><bold>0.109 (0.02)</bold></td><td><bold>1.29</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221207</bold></sup><sup>a</sup></td><td>0.95 (0.81\\u20131.13)</td><td>0.59</td><td>0.006 (0.005)</td><td>0.28</td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs4959689\\\">rs4959689</ext-link></td><td>6:2617122</td><td>C/A</td><td><italic>C6orf195</italic></td><td>1.10 (0.89\\u20131.37)</td><td>0.37</td><td>0.022 (0.02)</td><td>0.30</td><td>NA</td><td>NA</td><td>0.024 (0.026)</td><td>0.34<sup>c</sup></td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs56104388\\\">rs56104388</ext-link></td><td>20:45302135</td><td>T/C</td><td><italic>SLC13A3</italic></td><td>1.30 (0.40\\u20134.24)</td><td>0.67</td><td>0.274 (0.11)</td><td><bold>1.47</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221202</bold></sup></td><td>NA</td><td>NA</td><td>NA</td><td>NA</td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs12417836\\\">rs12417836</ext-link></td><td>11:70089700</td><td>T/C</td><td><italic>FADD</italic>, <italic>PPFIA1</italic></td><td>0.99 (0.64\\u20131.56)</td><td>0.99</td><td>0.045 (0.04)</td><td>0.29</td><td>0.87 (0.62\\u20131.21)</td><td>0.40</td><td>\\u22120.002 (0.011)</td><td>0.84</td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs2923437\\\">rs2923437</ext-link></td><td>8:42425399</td><td>A/C</td><td><italic>SMIM19</italic>, <italic>CHRNB3</italic>, <italic>SLC20A2</italic></td><td>0.98 (0.78\\u20131.23)</td><td>0.88</td><td>0.047 (0.02)</td><td><bold>2.60</bold>\\u2009<bold>\\u00d7</bold>\\u2009<bold>10</bold><sup><bold>\\u221202</bold></sup></td><td>1.11 (0.94\\u20131.31)</td><td>0.23</td><td>0.008 (0.005)</td><td>0.11</td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs1922930\\\">rs1922930</ext-link></td><td>6:1364691</td><td>C/A</td><td><italic>FOXQ1</italic>, <italic>FOXF2</italic></td><td>0.93 (0.65\\u20131.33)</td><td>0.70</td><td>0.035 (0.04)</td><td>0.33</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs2385089\\\">rs2385089</ext-link></td><td>19:18550434</td><td>A/C</td><td><italic>ISYNA1</italic>, <italic>ELL</italic>, <italic>LRRC25</italic></td><td>1.20 (0.94\\u20131.53)</td><td>0.14</td><td>0.049 (0.03)</td><td>0.057</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs10954468\\\">rs10954468</ext-link></td><td>7:134434661</td><td>C/A</td><td><italic>BPGM</italic>, <italic>CALD1</italic></td><td>1.08 (0.86\\u20131.36)</td><td>0.50</td><td>0.033 (0.02)</td><td>0.13</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td></tr><tr><td colspan=\\\"12\\\"><bold>PVS in basal ganglia (BG-PVS)</bold></td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs4675310\\\">rs4675310</ext-link><sup>d</sup></td><td>2:203880834</td><td>A/G</td><td><italic>NBEAL1</italic>, <italic>ICA1L</italic></td><td>1.07 (0.81\\u20131.41)</td><td>0.61</td><td>0.01 (0.04)</td><td>0.78</td><td>1.88 (0.63-5.60)</td><td>0.26</td><td>0.046 (0.03)</td><td>0.11</td></tr><tr><td><ext-link ext-link-type=\\\"uri\\\" xlink:href=\\\"https://www.ncbi.nlm.nih.gov/snp/?term=rs6769442\\\">rs6769442</ext-link></td><td>3:171565463</td><td>G/A</td><td><italic>TMEM212</italic></td><td>1.10 (0.87\\u20131.40)</td><td>0.37</td><td>0.03 (0.03)</td><td>0.33</td><td>1.04 (0.71-1.53)</td><td>0.82</td><td>0.008 (0.01)</td><td>0.35</td></tr></tbody></table>\\n\"}, \"text\": \"GWAS meta-analysis\\ti-Share (dichotomous)\\ti-Share (continuous)\\tNagahama (dichotomous)\\tNagahama (continuous)\\t \\tSNP\\tchr:position\\tEA/OA\\tNearest gene(s)\\tOR (95% CI)\\tP value\\t\\u03b2 (SE)\\tP value\\tOR (95% CI)\\tP value\\t\\u03b2 (SE)\\tP value\\t \\tPVS in white matter (WM-PVS)\\t \\trs6011998\\t20:45269867\\tC/T\\tSLC13A3\\t1.26 (0.83\\u20131.92)\\t0.28\\t0.164 (0.04)\\t4.20\\u2009\\u00d7\\u200910\\u221205a\\t1.69 (1.33\\u20132.13)\\t1.22\\u2009\\u00d7\\u200910\\u221205a\\t0.037 (0.008)\\t6.21\\u2009\\u00d7\\u200910\\u221207a\\t \\trs13079464\\t3:13822439\\tC/G\\tWNT7A\\t1.12 (0.91\\u20131.40)\\t0.29\\t0.014 (0.02)\\t0.50\\t1.16 (0.97\\u20131.40)\\t0.11\\t0.015 (0.006)\\t1.50\\u2009\\u00d7\\u200910\\u221202\\t \\trs2425884\\t20:45258292\\tC/T\\tSLC13A3\\t1.18 (0.95\\u20131.45)\\t0.13\\t0.077 (0.02)\\t2.98\\u2009\\u00d7\\u200910\\u221204a\\t1.29 (1.09\\u20131.52)\\t3.48\\u2009\\u00d7\\u200910\\u221203\\t0.026 (0.005)\\t1.77\\u2009\\u00d7\\u200910\\u221206a\\t \\trs10817108\\t9:113658671\\tA/G\\tLPAR1\\t0.90 (0.69\\u20131.17)\\t0.44\\t0.058 (0.03)\\t2.23\\u2009\\u00d7\\u200910\\u221202\\t1.18 (0.98\\u20131.43)\\t0.07\\t0.017 (0.006)\\t4.10\\u2009\\u00d7\\u200910\\u221203\\t \\trs2425881\\t20:45255618\\tA/G\\tSLC13A3\\t1.47 (1.03\\u20132.01)\\t1.40\\u2009\\u00d7\\u200910\\u221202\\t0.063 (0.03)\\t2.62\\u2009\\u00d7\\u200910\\u221202\\t1.18 (1.01\\u20131.37)\\t3.66\\u2009\\u00d7\\u200910\\u221202\\t0.014 (0.005)\\t4.68\\u2009\\u00d7\\u200910\\u221203\\t \\trs3772833\\t3:124518362\\tG/A\\tITGB5, UMPS\\t1.22 (0.89\\u20131.66)\\t0.21\\t0.006 (0.03)\\t0.85\\t1.06 (0.88\\u20131.29)\\t0.51\\t0.008 (0.006)\\t0.16\\t \\trs112407396\\t20:45276381\\tT/A\\tSLC13A3\\t1.47 (0.77\\u20132.78)\\t0.24\\t0.147 (0.07)\\t3.13\\u2009\\u00d7\\u200910\\u221202\\tNA\\tNA\\tNA\\tNA\\t \\trs10494988\\t1:215141570\\tC/T\\tCENPF, KCNK2\\t1.18 (0.95\\u20131.47)\\t0.14\\t0.079 (0.02)\\t1.94\\u2009\\u00d7\\u200910\\u221204a\\t1.01 (0.86\\u20131.18)\\t0.90\\t\\u22120.002 (0.005)\\t0.67\\t \\trs72485816\\t20:45314435\\tT/C\\tTP53RK, SLC13A3\\t1.01 (0.56\\u20131.80)\\t0.98\\t0.093 (0.06)\\t0.095\\t1.32 (1.10\\u20131.59)\\t2.83\\u2009\\u00d7\\u200910\\u221203\\t0.033 (0.006)\\t1.91\\u2009\\u00d7\\u200910\\u221208b\\t \\trs8041189\\t15:85686327\\tG/A\\tPDE8A\\t1.14 (0.89\\u20131.44)\\t0.30\\t0.041 (0.02)\\t0.073\\t1.67 (0.84\\u20133.33)\\t0.14\\t0.046 (0.021)\\t2.40\\u2009\\u00d7\\u200910\\u221202\\t \\trs4685022\\t3:13832611\\tG/A\\tWNT7A\\t1.12 (0.88\\u20131.42)\\t0.34\\t0.023 (0.02)\\t0.31\\t1.15 (0.97\\u20131.36)\\t0.10\\t0.010 (0.005)\\t0.075c\\t \\trs7596872\\t2:56128091\\tC/A\\tEFEMP1\\t1.65 (1.10\\u20132.46)\\t1.14\\u2009\\u00d7\\u200910\\u221202\\t0.089 (0.03)\\t1.10\\u2009\\u00d7\\u200910\\u221202\\tNA\\tNA\\tNA\\tNA\\t \\trs1126642\\t17:42989063\\tC/T\\tGFAP\\t1.13 (0.65\\u20131.97)\\t0.67\\t0.127 (0.05)\\t1.27\\u2009\\u00d7\\u200910\\u221202\\t1.35 (1.09\\u20131.67)\\t0.11\\t0.033 (0.007)\\t9.88\\u2009\\u00d7\\u200910\\u221207a\\t \\trs687610\\t3:193515781\\tG/C\\tOPA1\\t1.46 (1.18\\u20131.80)\\t4.88\\u2009\\u00d7\\u200910\\u221204a\\t0.109 (0.02)\\t1.29\\u2009\\u00d7\\u200910\\u221207a\\t0.95 (0.81\\u20131.13)\\t0.59\\t0.006 (0.005)\\t0.28\\t \\trs4959689\\t6:2617122\\tC/A\\tC6orf195\\t1.10 (0.89\\u20131.37)\\t0.37\\t0.022 (0.02)\\t0.30\\tNA\\tNA\\t0.024 (0.026)\\t0.34c\\t \\trs56104388\\t20:45302135\\tT/C\\tSLC13A3\\t1.30 (0.40\\u20134.24)\\t0.67\\t0.274 (0.11)\\t1.47\\u2009\\u00d7\\u200910\\u221202\\tNA\\tNA\\tNA\\tNA\\t \\trs12417836\\t11:70089700\\tT/C\\tFADD, PPFIA1\\t0.99 (0.64\\u20131.56)\\t0.99\\t0.045 (0.04)\\t0.29\\t0.87 (0.62\\u20131.21)\\t0.40\\t\\u22120.002 (0.011)\\t0.84\\t \\trs2923437\\t8:42425399\\tA/C\\tSMIM19, CHRNB3, SLC20A2\\t0.98 (0.78\\u20131.23)\\t0.88\\t0.047 (0.02)\\t2.60\\u2009\\u00d7\\u200910\\u221202\\t1.11 (0.94\\u20131.31)\\t0.23\\t0.008 (0.005)\\t0.11\\t \\trs1922930\\t6:1364691\\tC/A\\tFOXQ1, FOXF2\\t0.93 (0.65\\u20131.33)\\t0.70\\t0.035 (0.04)\\t0.33\\tNA\\tNA\\tNA\\tNA\\t \\trs2385089\\t19:18550434\\tA/C\\tISYNA1, ELL, LRRC25\\t1.20 (0.94\\u20131.53)\\t0.14\\t0.049 (0.03)\\t0.057\\tNA\\tNA\\tNA\\tNA\\t \\trs10954468\\t7:134434661\\tC/A\\tBPGM, CALD1\\t1.08 (0.86\\u20131.36)\\t0.50\\t0.033 (0.02)\\t0.13\\tNA\\tNA\\tNA\\tNA\\t \\tPVS in basal ganglia (BG-PVS)\\t \\trs4675310d\\t2:203880834\\tA/G\\tNBEAL1, ICA1L\\t1.07 (0.81\\u20131.41)\\t0.61\\t0.01 (0.04)\\t0.78\\t1.88 (0.63-5.60)\\t0.26\\t0.046 (0.03)\\t0.11\\t \\trs6769442\\t3:171565463\\tG/A\\tTMEM212\\t1.10 (0.87\\u20131.40)\\t0.37\\t0.03 (0.03)\\t0.33\\t1.04 (0.71-1.53)\\t0.82\\t0.008 (0.01)\\t0.35\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 20918, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"NA in the Nagahama Study correspond to variants that are rare (MAF\\u2009<\\u20091%: rs7596872; rs1922930; rs10954468) or monomorphic (rs112407396; rs56104388) in East Asians, or not available including in EAS 1000G data (rs2385089).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 21144, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"Analyses were adjusted for age, sex and intracranial volume, principal components of population stratification in the i-Share and Nagahama studies, and additionally adjusted for study center in the Nagahama study. In the Nagahama study, when the lead SNP from the PVS GWAS meta-analysis was not present, we used a tag SNP with r2\\u2009>\\u20090.80 using the 1000G Japanese reference panel.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 21527, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"SNPs or tag SNPs (r\\u00b2\\u2009>\\u20090.80, 1000G EAS) with a P\\u2009<\\u20090.05 are in bold.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 21605, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"aSNPs with a P\\u2009<\\u20091.09\\u2009\\u00d7\\u200910\\u22123 (Bonferroni correction for 23 independent loci and two PVS locations).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 21716, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"bSNPs reaching genome-wide significance.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 21757, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"cThe tag SNP (r\\u00b2\\u2009>\\u20090.80) is nominally significant: rs4685022 (r\\u00b2\\u2009=\\u20090.81 with rs934448, 1000G EAS), P\\u2009=\\u20090.048; rs4959689 (r\\u00b2\\u2009=\\u20090.83 with rs1772953, 1000G EAS), P\\u2009=\\u20090.02.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 21949, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"dThe lead SNP for this locus is not present in the Nagahama study; we used a tag SNP (rs150788469, r\\u00b2\\u2009=\\u20091.0 with rs4675310) where the A allele of rs4675310 is in phase with the G allele of rs150788469.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 22156, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We explored associations of WM-PVS and BG-PVS risk variants with these phenotypes in young adults (Internet-based Students\\u2019 HeAlth Research Enterprise (i-Share) study, N\\u2009=\\u20091,748, 22.1\\u2009\\u00b1\\u20092.3\\u2009yr) and in older Japanese community-dwelling people (Nagahama study, N\\u2009=\\u20092,862, 68.3\\u2009\\u00b1\\u20095.3\\u2009yr; Methods). We used an artificial intelligence-based method to derive quantitative WM-PVS and BG-PVS burden (HIP-PVS not available) and dichotomized it (top quartile versus the rest; Supplementary Table 2). In total, 67% of WM-PVS loci reached nominally significant associations in at least one of the two follow-up cohorts (P\\u2009<\\u20090.05 in i-Share and/or Nagahama), 43% of which at P\\u2009<\\u20091.09\\u2009\\u00d7\\u200910\\u22123 (correcting for the number of loci tested), with consistent directionality of effect (a binomial test showed significant concordance of risk alleles; Supplementary Table 12b). In i-Share, 52% of WM-PVS risk variants were associated with WM-PVS (P\\u2009<\\u20090.05, of which 4 at P\\u2009<\\u20091.09\\u2009\\u00d7\\u200910\\u22123; Table 2 and Supplementary Table 12a). A WM-PVS rescaled weighted genetic risk score (rwGRS) derived from European GWAS loci was associated with WM-PVS in i-Share (OR\\u2009=\\u20091.16 (95% CI, 1.08\\u20131.24), P\\u2009=\\u20095.89\\u2009\\u00d7\\u200910\\u22126 and \\u03b2 (SE)\\u2009=\\u20090.064 (0.007), P\\u2009=\\u20092.06\\u2009\\u00d7\\u200910\\u221219 for dichotomous and continuous measures; Supplementary Fig. 2). Although meta-regression suggested larger effect sizes at younger ages for lead variants at OPA1 and SLC13A3, differences were not significant after removing the much younger i-Share cohort (Supplementary Fig. 3). In Nagahama, out of 15 available WM-PVS risk loci (six were rare or monomorphic), eight loci (53%) were associated with continuous PVS burden at P\\u2009<\\u20090.05, of which four at P\\u2009<\\u20091.09\\u2009\\u00d7\\u200910\\u22123 and one at genome-wide significance (at SLC13A3; Table 2 and Supplementary Table 12a). A European WM-PVS weighted genetic risk score (wGRS) combining 14 independent loci (1000 Genomes project (1000G) Japanese reference panel) was associated with WM-PVS in Nagahama (OR\\u2009=\\u20091.18 (95% CI, 1.13\\u20131.24), P\\u2009=\\u20095.68\\u2009\\u00d7\\u200910\\u221213 and \\u03b2 (SE)\\u2009=\\u20090.01 (0.001), P\\u2009=\\u20097.18\\u2009\\u00d7\\u200910\\u221218 for dichotomous and continuous measures). Although HIP-PVS data were not available in the follow-up cohorts, two of the three HIP-PVS loci were shared with WM-PVS and replicated with that phenotype.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 24533, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Clinical correlates of identified PVS loci\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 24576, \"infons\": {\"file\": \"41591_2023_2268_Fig2_HTML.jpg\", \"id\": \"Fig2\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Association of PVS loci with vascular risk factors and other MRI markers of cSVD.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 24658, \"infons\": {\"file\": \"41591_2023_2268_Fig2_HTML.jpg\", \"id\": \"Fig2\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"a, Venn diagram displaying significant association of genome-wide significant risk loci for PVS burden with vascular risk factors and other MRI markers of cSVD: in italics for BG-PVS; underlined for HIP-PVS; underlined and in bold for HIP- and WM-PVS; all others for WM-PVS only (P\\u2009<\\u20093.3\\u2009\\u00d7\\u200910\\u22125, two-sided, correcting for multiple testing (21 independent phenotypes, 3 PVS locations and 24 independent loci)); *6 independent loci; **2 independent loci; \\u2020genome-wide significant in Europeans only; \\u2021in colocalization analyses the posterior probability PP4 was higher than 75% for these loci (only with WMH at NBEAL1-ICA1L). Exact P values are provided in Supplementary Table 16. b, Direction of association and level of significance of pleiotropic SNPs displayed in a: in red when the risk allele for extensive PVS burden is positively associated with the trait, in blue when the PVS risk allele is negatively associated with the trait (unexpected direction), in dark red and dark blue for genome-wide significant associations and in light red and light blue for significant association after multiple-testing correction (P\\u2009<\\u20093.3\\u2009\\u00d7\\u200910\\u22125, two-sided, correcting for multiple testing (21 independent phenotypes, 3 PVS locations and 24 independent loci)). PP, pulse pressure; BMI, body mass index; LDL, LDL cholesterol.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 25999, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We examined whether PVS risk loci (lead and proxy variants with r2\\u2009>\\u20090.9) were associated with MRI markers of brain aging, putative risk factors (vascular risk factors and sleep patterns) and common neurological diseases (stroke, Alzheimer\\u2019s disease, Parkinson\\u2019s disease), using the largest published GWAS (Methods). Of 24 independent PVS risk loci, five (21%) were significantly (P\\u2009<\\u20093.3\\u2009\\u00d7\\u200910\\u22125) associated with WMH volume and five (21%) with blood pressure traits (in the same and opposite directions; Fig. 2). Colocalization analyses suggested a shared causal variant for two-thirds of these associations (posterior probability for a shared causal variant, PP4\\u2009>\\u20090.75; Supplementary Table 16). Sixteen PVS loci (67%) did not show any association with vascular or neurological traits, thus pointing to pathways that do not seem mediated by established risk factors (Methods and Supplementary Tables 16 and 17).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 26935, \"infons\": {\"file\": \"41591_2023_2268_Fig3_HTML.jpg\", \"id\": \"Fig3\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Genetic correlations of extensive PVS burden with risk factors, neurological diseases and other MRI markers of brain aging.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 27059, \"infons\": {\"file\": \"41591_2023_2268_Fig3_HTML.jpg\", \"id\": \"Fig3\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"a\\u2013c, Genetic correlation using LD-score regression of extensive PVS burden with putative risk factors (a), neurological diseases (b) and other MRI markers of brain aging (c); two-sided exact P values are provided for nominally significant results (*P\\u2009<\\u20090.05) and significant results after multiple-testing correction (**P\\u2009<\\u20097.9\\u2009\\u00d7\\u200910\\u22124, correcting for 21 independent phenotypes and the three PVS locations); full results are provided in Supplementary Table 9. Larger colored squares correspond to more significant P values and the colors represent the direction of the genetic correlation (positive in red, negative in blue). HDL, high-density lipoprotein; amygdala, accumbens (nucleus), caudate (nucleus), pallidum, and putamen correspond to the volumes of these subcortical structures.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 27863, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Second, we explored genetic correlations of PVS burden with the same traits using LD-score regression (Methods, Fig. 3 and Supplementary Table 9). We observed significant (P\\u2009<\\u20097.9\\u2009\\u00d7\\u200910\\u22124) genetic correlation of BG-PVS with larger WMH and caudate nucleus volumes, and of HIP-PVS with larger hippocampal volume. BG-PVS and HIP-PVS showed significant genetic correlation with higher systolic blood pressure (SBP), diastolic blood pressure (DBP), any stroke and ischemic stroke, and nominally significant genetic correlation with (deep) intracerebral hemorrhage (ICH). Genetic correlations were consistent in secondary analyses conducted separately in CHARGE and UKB (Supplementary Table 9).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 28562, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Third, we used two-sample Mendelian randomization (MR) to seek evidence for a causal association of putative risk factors with PVS burden and of PVS burden with neurological diseases, using generalized summary-data-based MR (GSMR), and confirming significant associations (P\\u2009<\\u20091.19\\u2009\\u00d7\\u200910\\u22123) with RadialMR, TwoSampleMR and MR-CAUSE (Methods). Genetically determined higher SBP and DBP were consistently associated with BG-PVS, HIP-PVS and WM-PVS, although for WM-PVS the association with SBP was only nominally significant in RadialMR (Supplementary Table 18 and Extended Data Fig. 2). There was no evidence for reverse causation using MR-Steiger, but some evidence of residual pleiotropy after removal of outlier variants for SBP and DBP (RadialMR), with significant evidence for a causal model in MR-CAUSE for BG-PVS. Genetic liability to BG-PVS and HIP-PVS derived from a multi-trait analysis accounting for other MRI markers of cSVD (MTAG) was associated with an increased risk of any stroke, ischemic stroke and small vessel stroke (SVS) for BG-PVS, and SVS for HIP-PVS, suggesting that shared pathways between PVS, WMH and lacunes may be causally associated with stroke (Supplementary Table 18 and Extended Data Fig. 3). In multivariable MR analyses accounting for SBP and DBP, genetic liability to BG-PVS and HIP-PVS was significantly associated with an increased risk of any stroke, ischemic stroke and SVS (Supplementary Table 19).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 30012, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Functional exploration of identified PVS loci\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 30058, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Using MAGMA and VEGAS2Pathway (Methods), we identified significant enrichment of PVS loci in pathways involved in extracellular matrix (ECM) structure and function, lymphatic endothelial cell differentiation, cell motility and thyroid hormone transport (Supplementary Tables 20 and 21).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 30345, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Genes closest to PVS lead risk variants were significantly enriched in genes mutated in Online Mendelian Inheritance in Man (OMIM) syndromes associated with leukodystrophy, leukoencephalopathy or WMH, with a 20-fold enrichment in genes containing an intragenic lead variant. This enrichment was 30-fold when focusing on WM-PVS loci only, comprising several genes involved in early-onset leukodystrophies: GFAP (chr17q21.31), mutations of which cause Alexander disease, a rare neurodegenerative disorder of astrocytes leading to psychomotor regression and death; SLC13A3 (chr20q13.12), causing acute reversible leukoencephalopathy with increased urinary alpha-ketoglutarate; and PNPT1 (chr2p16.1), causing Aicardi\\u2013Gouti\\u00e8res syndrome and cystic leukoencephalopathy (Methods, Extended Data Fig. 4 and Supplementary Table 22). Although several genes near PVS lead risk variants were described to be involved in glioma, we found no significant enrichment for glioma genes (Methods).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 31326, \"infons\": {\"file\": \"41591_2023_2268_Fig4_HTML.jpg\", \"id\": \"Fig4\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Transcriptome-wide significant genes with extensive PVS burden.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 31390, \"infons\": {\"file\": \"41591_2023_2268_Fig4_HTML.jpg\", \"id\": \"Fig4\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"We used precomputed functional weights from 22 publicly available gene expression reference panels from brain (GTEx v7, CommonMind Consortium (CMC)), peripheral nerve tissues (GTEx v7), heart and arteries (GTEx v7), and blood (Netherlands Twin Registry (NTR) and Young Finns Study (YFS)). Transcriptome-wide significant genes (eGenes) and the corresponding eQTLs were determined using Bonferroni correction, based on the average number of features (4,235 genes) tested across all tissues and correcting for the three independent PVS locations (P\\u2009<\\u20093.93\\u2009\\u00d7\\u200910\\u22126). *Significant result in the TWAS and conditional analyses; **significant result in the TWAS and conditional analyses, and with a COLOC PP4\\u2009>\\u20090.75; eGenes for loci identified in the GWAS (\\u2020), gene-based test (\\u2021) or both GWAS and gene-based test (\\u00a7).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 32220, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"To seek evidence for a causal implication of specific genes and variants, we performed transcriptome-wide association studies (TWAS) using TWAS-Fusion (Methods), with European PVS GWAS summary statistics and the GTEx v7 multi-tissue (RNA sequencing) database, focusing on brain, vascular and blood tissues. We found 36 transcriptome-wide significant expression\\u2013trait associations for WM-PVS, 25 for BG-PVS and seven for HIP-PVS that were significant in colocalization analyses (TWAS-COLOC), providing evidence of a shared causal variant between the corresponding gene expression and PVS (Supplementary Table 23). Most genes with significant expression\\u2013trait associations (12) were in genome-wide significant PVS risk loci: eight genes in five WM-PVS GWAS loci (C6orf195, ITGB5, LPAR1, LRRC25, RP11-71H17.9, SLC20A2, SMIM19, UMPS), two genes in one BG-PVS GWAS locus (ICA1L, NBEAL1) and two genes in an HIP-PVS GWAS locus (LAMC1 and RP11-181K3.4), while nine were outside GWAS loci, requiring confirmation (Fig. 4). TWAS-COLOC signals were mostly observed in brain tissues (17 genes), but also in vascular tissues (ten genes) and blood (two genes).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 33372, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"To identify enrichment in specific brain cell types, we used a recently developed pipeline combining three cell type enrichment methods, stratified LD-score, MAGMA and H-MAGMA (Supplementary Table 24). We observed significant enrichment in brain vascular endothelial cells for all PVS locations, based on a human single-cell atlas of fetal gene expression, and in pericytes and astrocytes for WM-PVS (Supplementary Tables 24 and 25).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 33806, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We explored brain expression patterns from development to adulthood of genes nearest to PVS loci, prioritizing TWAS-COLOC genes (Methods). Several genes showed important variations in expression levels throughout the life course, some peaking in the prenatal period (for example, LAMC1, UMPS), suggestive of developmental mechanisms (Extended Data Fig. 5 and Supplementary Fig. 4).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34188, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Finally, we conducted an exploratory search for enrichment of PVS genes in targets of drugs validated in other indications (Methods). We found significant enrichment of BG-PVS genes in targets for anti-infectives, driven by CRHR1 (chr17q21.31, target for telavancin), and for diseases of the nervous system, driven by MAPT (chr17q21.31, target for davunetide); and of HIP-PVS genes in targets for ear disease drugs, driven by SERPIND1 (chr22q11.21, target for sulodexide, also used for venous thrombosis prevention; Extended Data Figs. 6 and 7). We also observed significant enrichment of TWAS-significant HIP-PVS genes in vascular disease drugs, including simvastatin, vincamine and macitentan (Extended Data Fig. 8).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34907, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_1\"}, \"text\": \"Discussion\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34918, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"In up to 40,095 participants from older population-based cohorts, we identified 24 genome-wide significant risk loci for extensive PVS burden, predominantly for WM-PVS, and six additional loci after accounting for other MRI markers of cSVD. Consistent with distinct risk factor profiles, the genetic architecture of PVS differed across PVS locations, with WM-PVS showing the highest heritability and low genetic correlation with BG-PVS and HIP-PVS. In line with the hypothesis that PVS is a marker of cSVD, moderate to high genetic correlation was observed with other MRI markers of cSVD, primarily for BG- and HIP-PVS. Pathway analyses highlight ECM structure and function, known to play an important role in cSVD, and several loci include genes involved in the matrisome (ECM and associated proteins), perturbations of which were proposed as a convergent pathologic pathway in cSVD (LAMC1, EFEMP1, COL4A2, SH3PXD2A, VWA2). Several PVS risk loci (at FOXF2, EFEMP1, KCNK2 and NBEAL1-ICA1L) are known risk loci for other cSVD features (WMH, SVS), and mutations in two MTAG genes cause monogenic SVD (at COL4A1-COL4A2 and STN1).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 36045, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"PVS have been described early in life, but their clinical significance at young ages is unknown. Our results suggest shared molecular mechanisms underlying PVS in young and older age. This corroborates recently described associations of WMH risk variants with changes in MRI-detected WM microstructure at age 20\\u2009yr (ref. ). The significant enrichment of PVS risk loci in genes involved in early-onset leukodystrophies and expressed in fetal brain vascular endothelial cells supports involvement of developmental processes. In spontaneously hypertensive stroke-prone rats, closely modeling cSVD, intrinsic endothelial cell dysfunction was observed at birth, including reduced tight junctions, as well as altered oligodendrocyte maturation and myelination. At the most significant WM-PVS locus in young adults, OPA1 harbors mutations causing autosomal-dominant optical atrophy, sometimes associated with multiple sclerosis-like illness, parkinsonism and dementia, and endothelial OPA1 plays an important role in developmental angiogenesis. These observations corroborate epidemiological associations of early-life factors with cSVD severity in older age.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 37200, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"The present effort has the largest East-Asian contribution compared with other large GWAS of MRI-defined phenotypes, with over half of available WM-PVS loci reaching nominally significant, directionally consistent associations in the Japanese follow-up study. The prevalence of cSVD is higher in East-Asian than European populations. Our results are an important initial step to establish the generalizability of cSVD genetic associations across ancestries. Efforts to further enhance the non-European contribution to MRI cSVD genomic studies, including in populations of African-ancestry in whom cSVD is also more frequent, are of paramount importance.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 37854, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"The combination of PVS GWAS findings with TWAS and WES/WGS strongly supports putative causal genes. WM-PVS associates with lower LPAR1 expression in vascular tissues. LPAR1 (chr9q31.3), expressed in oligodendrocytes, encodes a receptor for lysophosphatidic acid, an extracellular signaling small lipid, and is involved in postnatal myelination and functional connectivity across brain regions. An LPAR1 antagonist was found to attenuate brain damage after transient arterial occlusion, by decreasing inflammation, and LPAR1 modulation may also impact neural regeneration. Several drugs targeting LPAR1 are available (for example, the antidepressant mirtazapine) or in development. WNT7A (chr3p25.1) encodes a secreted signaling protein that targets the vascular endothelium, and was implicated in brain angiogenesis and blood brain barrier regulation. Loss of Wnt7a/b function in mice results in severe WM damage.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 38768, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"WM-PVS was associated with lower ITGB5 (chr3q21.2) expression in whole blood. ITGB5 encodes a beta subunit of integrin, and plays a central role in monogenic SVD. Higher ITGB5 plasma levels were associated with decreased odds of cognitive impairment or dementia, lower brain amyloid burden and slower brain atrophy rates. HIP-PVS was associated with lower expression of LAMC1 (chr1q25.3, encoding Laminin gamma-1) in brain and higher expression in vascular tissues, while WES/WGS identified a splice donor variant at LAMC1. Laminins are ECM glycoproteins, and the major noncollagenous constituent of basement membranes. Genes encoding other basement membrane proteins (NID2, COL4A1/2) are implicated in cSVD. Laminin regulates blood vessel diameter and blood brain barrier integrity and function, and astrocytic laminin loss decreases expression of tight junction proteins and aquaporin-4 (AQP4), a key modulator of glymphatic flow in experimental models.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 39724, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Some genes point to complex pleiotropic mechanisms. At chr2q33.2, also associated with WMH, SVS, Alzheimer\\u2019s disease and caudate volume, BG-PVS was associated with higher expression of ICA1L in brain tissues and of NBEAL1 in vascular tissues, similar to TWAS of WMH and SVS. ICA1L (encoding islet cell autoantigen-1-like and predominantly expressed in endothelial cells) harbors mutations causing juvenile amyotrophic lateral sclerosis, while NBEAL1 (encoding neurobeachin-like 1 protein) modulates low-density lipoprotein (LDL)-receptor expression.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 40276, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Our study points to an important involvement of solute carriers (SLCs), the largest family of transporters and candidates for drug target development, in PVS pathophysiology. The most significant PVS risk variants involve an intronic haplotype of SCL13A3, encoding a plasma membrane Na+/dicarboxylate cotransporter expressed in kidney, astrocytes and choroid plexus. Mutations in SLC13A3 cause acute reversible leukoencephalopathy with increased urinary alpha-ketoglutarate, where SLC13A3 loss-of-function may affect elimination of organic anions and xenobiotics from the cerebrospinal fluid (CSF). At the same locus (Supplementary Fig. 1), other genome-wide significant variants are located near SLC2A10, harboring mutations causing arterial tortuosity syndrome, described to be associated with PVS burden and cSVD. WM-PVS was associated with lower SLC20A2 expression in brain tissue. SLC20A2, involved in phosphate transport, harbors loss-of-function mutations causing idiopathic familial BG calcification, a neurodegenerative disorder with inorganic phosphate accumulation in the ECM. Given their role in CSF secretion and substance transport at the blood\\u2013CSF barrier, SLCs could be involved in interstitial fluid accumulation adjacent to the PVS.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 41529, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Consistent with other SVD phenotypes, we observed evidence for a causal association of blood pressure with PVS. Experimental work suggests that the perivascular pump becomes less efficient with increasing blood pressure, reducing net forward flow in the PVS. These effects were found to be larger at more distal locations, where arteries have thinner and less muscular walls. Such hemodynamic and anatomic differences could, perhaps, at least partly explain the more significant association of blood pressure with BG-PVS and HIP-PVS compared with WM-PVS. In contrast, WM-PVS were previously found to be associated with CAA and with higher brain amyloid deposition on positron emission tomography, across the clinical spectrum of CAA. The updated Boston Criteria (v.2.0) for CAA include severe WM-PVS as a new diagnostic criterion.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 42360, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"The clinical relevance of PVS is strongly supported by the significant genetic correlation of BG-PVS and HIP-PVS with any stroke and ischemic stroke and robust evidence for a possible causal association of BG-PVS and HIP-PVS with any stroke, ischemic stroke and SVS, accounting for blood pressure. The nominally significant genetic correlation of BG-PVS and HIP-PVS with (deep) ICH, based on smaller GWAS and thus less statistical power, is also consistent with epidemiological findings. Considering the association of HIP-PVS with lower LAMC1 expression in brain, it is striking to note that conditional knock-out of laminin in astrocytes leads to deep ICH in mice. This is reminiscent of known associations of variants in COL4A1/A2, encoding another basement membrane protein, with monogenic and multifactorial deep ICH.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43183, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Significant enrichment of PVS genes in targets of drugs validated or under investigation for vascular and cognitive disorders (for example, telavancin and davunetide) highlights the potential of PVS genetics for cSVD drug discovery.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43416, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"To our knowledge, this is the first study exploring the genetic determinants of PVS, using a comprehensive gene-mapping strategy and extensive bioinformatics follow-up. We acknowledge limitations. To account for heterogeneity in PVS quantification methods, we pragmatically dichotomized PVS variables based on the top quartile of the distribution, which may be less powerful than continuous measures. This may have been most prominent for BG-PVS, for which the genetic correlation pattern between CHARGE and UKB was low, in contrast with WM-PVS and HIP-PVS. Reassuringly, loci identified using dichotomous PVS phenotypes were also associated with continuous PVS burden in studies where computational methods were available (UKB, i-Share, Nagahama), mostly with more significant P values. A conservative approach will also have helped minimize the effect of accidentally including WMH in the PVS measures, a problem which some computational PVS methods have not yet overcome. Strikingly, 67% of WM-PVS loci were associated at least nominally with WM-PVS in one or both follow-up cohorts, despite considerably smaller samples and distinct age and ancestry, with consistent directionality. This suggests that our genomic discovery approach, although likely conservative, led to robust findings. With increasing development of artificial intelligence-based computational methods for PVS quantification, future genomic studies will likely have even greater power to detect genetic associations, to enable studying the genomics of total PVS volume, accounting for differences in individual PVS volume, width, length, shape, density, location and anatomical predominance, and to run sex-specific analyses.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45115, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"In conclusion, in this gene-mapping study of PVS, one of the earliest MRI markers of cSVD, we describe 24 genome-wide significant risk loci, with six additional loci in secondary multivariate analyses accounting for other cSVD markers. Our findings provide insight into the biology of PVS across the adult lifespan and its contribution to cSVD pathophysiology, with potential for genetically informed prioritization of drug targets for prevention trials of cSVD, a major cause of stroke and dementia worldwide.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45626, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_1\"}, \"text\": \"Methods\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45634, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Study design\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45647, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"This study complies with all relevant ethical regulations, and all participants gave written, informed consent. Analyses were performed on stroke-free participants from 22 population-based cohorts (18 for the GWAS meta-analysis), taking part in UKB, the CHARGE consortium and the BRIDGET initiative. Institutional review boards approved individual studies: UKB (National Research Ethics Service Committee North West\\u2013Haydock), 3C-Dijon (Ethical Committee of the University Hospital of Kremlin-Bic\\u00eatre), Austrian Stroke Prevention Study and Austrian Stroke Prevention Family Study (ASPS/ASPS-Fam) (Ethics Committee of the Medical University of Graz), Epidemiology of Dementia in Singapore Study (EDIS) (the Singapore Chinese Eye Study/Singapore Malay Eye Study-2, Singapore Eye Research Institute and the National Healthcare Group Domain-Specific Review Board), Framingham Heart Study (FHS) (Institutional Review Board of Boston University Medical Center), Investigating Silent Strokes in Hypertensives Study (ISSYS) (Comit\\u00e9 de \\u00e9tica de investigacion con medicamentos, Hospital Universitari Vall d\\u2019Hebron), Lothian Birth Cohort 1936 (LBC1936) (Lothian and Scottish Multicentre Research Ethics Committees), Northern Manhattan Study (NOMAS) (Columbia University Medical Center Institutional Review Board and the University of Miami Institutional Review Board), Rotterdam Study I, II and III (RS-I, RS-II and RS-III) (Ministry of Health, Welfare, and Sport of the Netherlands), Study of Health in Pomerania (SHIP) (SHIP-2, SHIP-Trend Batch 1 and 2, Ethics Commission of the University of Greifswald), i-Share study (Comit\\u00e9s de Protection des Personnes (CPP) Sud-Ouest Outre-Mer III, Sydney Memory and Ageing Study (MAS) (Ethics Committees of the University of New South Wales, South-Eastern Sydney, and the Illawarra Area Health Service), Older Australian Twins Study (OATS) (Ethical Committees of the Australian Twin Registry, the University of New South Wales, the University of Melbourne, the Queensland Institute of Medical Research, and the South-Eastern Sydney and Illawarra Area Health Service) and the Nagahama Study (Ethics Committee of Kyoto University Graduate School of Medicine and the Nagahama Municipal Review Board) (Supplementary Table 1). Characteristics of study participants are provided in Supplementary Tables 1\\u20133 and 26 and Supplementary Fig. 5.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 48021, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"PVS burden definition\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 48043, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"PVS were defined as fluid-filled spaces with a signal identical to that of CSF, of round, ovoid or linear shape depending on the slice direction, with usually a maximum diameter smaller than 3\\u2009mm, no hyperintense rim on T2-weighted or FLAIR sequences, and located in areas supplied by perforating arteries. In most CHARGE cohorts, visual semiquantitative rating scales were used to quantify PVS burden. As different scales were used across studies, we dichotomized PVS burden into \\u2018extensive PVS burden\\u2019 versus the rest, defined by a cut-off closest to the top quartile of the semiquantitative scale distribution within each cohort (Supplementary Tables 2, 27 and 28). This cohort-specific threshold definition was chosen because (1) small PVS counts are very sensitive to MRI field strength and less prominently associated with age and vascular risk factors; (2) extreme burden of other MRI markers of cSVD (for example, extensive WMH burden within the top quartile of the distribution) was previously shown to facilitate the identification of genetic variants underlying cSVD; and (3) PVS burden is highly dependent on participant characteristics, especially age, PVS quantification methods and image acquisition parameters. In RS-III and in UKB, a recently developed automated method was used to quantify the number of PVS (Supplementary Table 27), dichotomized according to the same cut-off (top quartile). For sensitivity analyses, we also compared results obtained in UKB with the dichotomized and continuous (log-transformed) PVS variables.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49597, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Covariates and descriptive variables\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49634, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Intracranial volume (sum of gray matter, WM and CSF volumes) was available in all studies except ASPS, where brain parenchymal fraction was used (ratio of brain parenchymal tissue volume to total volume within the surface contour of the whole brain). Other covariates are described in Supplementary Table 1.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49942, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Genotyping and imputation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49968, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Genome-wide genotypes were imputed to the 1000G project (1000G pIv3) or the Haplotype Reference Consortium reference panels (Supplementary Table 3).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 50117, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"PVS genome-wide association analyses in individual cohorts\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 50176, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Ancestry-specific logistic regression analyses with an additive genetic model were performed, adjusting for age, sex (genetically determined) and intracranial volume (or brain parenchymal fraction for ASPS), principal components of population stratification, and study site.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 50451, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"As sensitivity analyses, we ran linear mixed models in UKB, (1) using the log-transformed (log(variable\\u2009+\\u20091)) continuous PVS measurements, adjusting for the same covariates as above; (2) generating residuals adjusting for the same covariates and then dichotomizing the residuals (instead of adjusting for covariates after dichotomization).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 50795, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"PVS genome-wide association meta-analyses\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 50837, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We performed quality control in each study following the recommendations of Winkler et al.. Analyses were done on autosomal biallelic markers. Duplicate markers were removed, marker names and alleles were harmonized across studies, and PZ-plots, quantile\\u2013quantile plots and allele frequency plots were constructed. In each study, rare variants (MAF\\u2009<\\u20090.01) and variants with low imputation accuracy (R\\u00b2, oevar_imp or info score\\u2009<\\u20090.5) or extensive effect size values (\\u03b2\\u2009>\\u20095 or \\u03b2\\u2009<\\u2009\\u22125) were removed. We reported the number of SNPs passing quality control for each study (Supplementary Table 4). GWAS were run within each cohort using logistic regression (or linear regression for sensitivity analyses), using software described in Supplementary Table 3. We then conducted GWAS meta-analyses across participating cohorts in METAL, using sample size-weighted meta-analysis as PVS were measured on different scales. Meta-analyses were conducted within each ancestry (European (EUR), Asian (ASN), African-American (AA), Hispanic (HISP)) using METAL (https://github.com/statgen/METAL), followed by meta-analyses across ancestries. Ancestry was genetically inferred using principal components of population stratification (Supplementary Tables 1 and 3). Genomic control was applied to each study-specific GWAS with a genomic inflation factor greater than 1.00. Variants with an effective allele count (twice the product of MAF, imputation accuracy and number of participants with extensive PVS)\\u2009<\\u200910 and significant heterogeneity (PHet\\u2009<\\u20095.0\\u2009\\u00d7\\u200910\\u22128) were excluded from the meta-analysis. We performed LD-clumping, sorting the genome-wide significant SNPs by P value, keeping the most significant SNP and removing SNPs with an r\\u00b2\\u2009>\\u20090.1 within 1\\u2009megabase (Mb). Only variants present in at least half of participants of the final meta-analysis were used to construct quantile\\u2013quantile and Manhattan plots. In secondary analyses, we ran inverse variance-weighted meta-analyses to obtain effect estimates and standard errors for follow-up bioinformatics analyses.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52936, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Conditional and joint multiple-SNP analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52980, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We used GCTA-COJO to perform conditional and joint multiple-SNP analysis of PVS GWAS summary statistics, to identify secondary association signals at each of the genome-wide significant loci within 1\\u2009Mb of the lead SNP. We used European GWAS summary statistics as recommended to avoid population stratification. This method relied on a stepwise selection procedure to select SNPs based on the conditional P values, and the joint effects of all selected SNPs after optimization of the model were estimated. We used genotypes of 6,489 unrelated participants of European ancestry from the 1000G-imputed 3C-Dijon study data for LD correction. We performed haplotype association analyses on the six independent lead variants at chr20q13.12 (Supplementary Table 5b).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53743, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Cross-ancestry meta-regression of GWAS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53782, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We conducted cross-ancestry meta-analyses using MR-MEGA, which uses meta-regression to model allelic effects, including axes of genetic variation as covariates in the model.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53956, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Gene-based analyses\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53976, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We performed gene-based analyses on European PVS GWAS meta-analyses. We included variants within 10\\u2009kilobase (kb) of the 3\\u2032 and 5\\u2032 untranslated regions (UTRs) of a gene to capture regulatory variants. We used the MAGMA software implemented in FUMA to perform a gene-based association study, including 19,037 protein-coding genes. This method is based on a multiple linear principal components regression model. Gene-wide significance was defined at P\\u2009<\\u20092.63\\u2009\\u00d7\\u200910\\u22126. We also performed gene-based tests using VEGAS2 (ref. ), including 18,371 autosomal genes, leading to a gene-wide significance at P\\u2009<\\u20092.72\\u2009\\u00d7\\u200910\\u22126. Genes were considered in the same locus if they were within 200\\u2009kb of each other.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54701, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"PVS heritability estimates\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54728, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We used LD-score regression (ldsc package https://github.com/bulik/ldsc/) to estimate the heritability of extensive PVS burden in each location, overall and, in secondary analyses, separately, in CHARGE and UKB.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54940, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Multi-trait GWAS with PVS and other MRI markers of cSVD\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54996, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We conducted a joint analysis of summary statistics from GWAS of PVS, WMH and lacunes using MTAG, with the expectation to gain in power because of the genetic correlation between these MRI markers of cSVD. MTAG is a generalization of inverse variance-weighted GWAS meta-analysis of two or more traits, which accounts for sample overlap between GWAS results for different traits by employing LD-score regression. MTAG is based on the assumption that all SNPs share the same variance\\u2013covariance matrix of effect sizes across traits. We prioritized variants with a P\\u2009<\\u20095\\u2009\\u00d7\\u200910\\u22128 in the PVS MTAG analysis and P\\u2009<\\u20090.05 in the univariate PVS GWAS, which showed greater significance for association with PVS in MTAG than in univariate analyses for PVS, WMH and lacunes.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55775, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"PVS next-generation sequencing association analyses\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55827, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Using WES data and exome content of WGS data in 19,010 participants from UKB and BRIDGET, of whom 4,531, 4,424 and 4,497 had extensive PVS in WM, BG and HIP, respectively, we performed a whole-exome association study to identify (rare) exonic variants associated with extensive PVS (Supplementary Tables 1 and 29).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56142, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Follow-up of findings across lifespan and ancestries\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56195, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We explored associations of WM-PVS and BG-PVS risk variants identified in the GWAS meta-analysis with these phenotypes in young adults (i-Share study, N\\u2009=\\u20091,748, mean age 22.1\\u2009\\u00b1\\u20092.3\\u2009yr) and in older Japanese population-based cohort participants (Nagahama study, N\\u2009=\\u20092,862, 68.3\\u2009\\u00b1\\u20095.3\\u2009yr; Supplementary Tables 1 and 3). In each study, we used both quantitative PVS measurements derived from a computational artificial intelligence-based method (Supplementary Tables 27 and 28) and dichotomized PVS burden (top quartile of PVS distribution; Supplementary Table 2). HIP-PVS data were not available. Continuous PVS measurements were log-transformed (log(variable\\u2009+\\u20091)) to obtain a normal distribution.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56919, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"In i-Share participants of European ancestry, we also explored the association of WM-PVS with a wGRS of WM-PVS burden derived from the 21 independent genome-wide significant SNPs identified in the European GWAS meta-analysis (r\\u00b2\\u2009<\\u20090.10 based on the 1000G European reference panel). SNPs were weighted by the SNP effect sizes in the European GWAS meta-analysis (for the allele associated with larger PVS burden); the wGRS was rescaled (rwGRS) so that one unit of the wGRS corresponds to one additional WM-PVS risk allele. We tested for significant modifying effects of age on associations with WM-PVS for the three genome-wide significant WM-PVS loci in young adults (at chr2p16.1, chr3q29 and chr20q13.12). We collected effect estimates and standard errors for the lead SNPs at these three loci in each individual cohort, and fitted a meta-regression of the lead SNPs\\u2019 effect sizes onto an intercept and age. Meta-regression analysis was performed using Metafor, and any statistical evidence of linear association was corrected for multiple testing (P\\u2009<\\u20090.05/3\\u2009=\\u20091.7\\u2009\\u00d7\\u200910\\u22122).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58017, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"In Nagahama we explored the association of WM-PVS with a rwGRS of WM-PVS burden, including the 14 available independent SNPs identified in the European GWAS meta-analysis (r\\u00b2\\u2009<\\u20090.10 based on 1000G Japanese reference panel); SNPs were weighted by the SNP effect sizes in the European GWAS meta-analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58325, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Shared genetic variation with other phenotypes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58372, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"In the European ancestry meta-analysis, we explored shared genetic variation with vascular and neurological phenotypes: (1) putative risk factors (SBP, DBP, pulse pressure, body mass index, high-density lipoprotein cholesterol, LDL cholesterol, triglycerides, type 2 diabetes and sleep patterns); (2) other MRI markers of brain aging (WMH burden, covert MRI-defined brain infarcts and lacunes, and hippocampal, nucleus accumbens, amygdala, caudate nucleus, pallidum and putamen volumes); and (3) the most common neurological conditions previously reported to be associated with PVS, namely stroke (any stroke, any ischemic stroke, large artery stroke, cardio-embolic stroke, SVS, ICH), Alzheimer\\u2019s disease and Parkinson\\u2019s disease (Supplementary Table 30).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59132, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We explored whether genome-wide significant PVS risk loci (lead variants or in LD with r\\u00b2\\u2009>\\u20090.9, based on the 1000G European reference panel) were associated with these traits. A P value threshold <3.3\\u2009\\u00d7\\u200910\\u22125, correcting for 21 independent phenotypes, three PVS locations and 24 independent loci tested, was used (Supplementary Table 30). We performed a colocalization analysis using COLOC to search for evidence for a single causal variant between PVS and the other phenotypes, a posterior probability (PP4)\\u2009>\\u200975% supporting a single causal variant for both traits.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59715, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Second, we used LD-score regression (ldsc package: https://github.com/bulik/ldsc/) to estimate the genetic correlation of extensive PVS burden with these phenotypes (P\\u2009<\\u20097.9\\u2009\\u00d7\\u200910\\u22124 was used as a significance threshold, correcting for 21 phenotypes and three PVS locations). To decrease potential bias due to poor imputation quality, the summary statistics were filtered to the subset of HapMap3 SNPs for each trait. In secondary analyses, we estimated genetic correlation of PVS burden with the same traits separately in CHARGE and UKB.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60263, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We used FUMA to obtain extensive functional annotation for genome-wide significant SNPs and to identify SNPs associated with other traits at genome-wide significance from the GWAS catalog.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60452, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"MR\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60455, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We used an MR approach to explore the possible causal relation of putative risk factors (vascular risk factors and sleep patterns) with extensive PVS burden, and of extensive PVS burden with neurological traits (stroke, Alzheimer\\u2019s disease and Parkinson\\u2019s disease).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60725, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We used the GSMR method implemented in GCTA. Summary statistics were clumped using 1000G-imputed 3C-Dijon study data (r\\u00b2\\u2009<\\u20090.05 and P\\u2009<\\u20095\\u2009\\u00d7\\u200910\\u22128) using only SNPs with MAF\\u2009>\\u20090.01. The heterogeneity in independent instrument (HEIDI)-outlier method was used to remove genetic instruments that showed pleiotropic effects on both the exposure and the outcome.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61100, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"For (at least) nominally significant GSMR associations, we conducted secondary MR analyses using both TwoSampleMR and RadialMR. Only independent SNPs (r2\\u2009<\\u20090.01 based on 1000G European, window size\\u2009=\\u20091\\u2009Mb) reaching P\\u2009<\\u20095\\u2009\\u00d7\\u200910\\u22128 in the primary meta-analysis were included as recommended. Effect estimates (\\u03b2 values) and SE values were derived from the inverse variance-weighted GWAS meta-analyses. With TwoSampleMR, we estimated the effect of each exposure on each outcome using weighted median, random-effect inverse variance weighting (IVW) and MR-Egger. In addition, we confirmed the directionality of the observed associations with the Steiger test. With RadialMR (https://github.com/WSpiller/RadialMR), the putative causal effect of each exposure on each outcome was estimated using the fixed-effect IVW method using the modified second-order inverse variance weight. Cochran\\u2019s Q statistic was used to test for the heterogeneity (P\\u2009<\\u20090.05) due to horizontal pleiotropy. We excluded outlier SNPs, identified by regressing the predicted causal estimate against the inverse variance weights, and re-ran IVW tests, as well as MR-Egger regression, assessing heterogeneity with R\\u00fccker\\u2019s Q\\u2032 statistic. When the ratio of Q\\u2032 (Egger) on Q (IVW) (QR) was close to 1, indicating that both IVW and MR-Egger models fit the data equally, we selected the IVW model. We formally ruled out horizontal pleiotropy when the MR-Egger intercept after exclusion of outliers was nonsignificant (P\\u2009\\u2265\\u20090.05). To account for potential residual correlated pleiotropy, we used MR-CAUSE. Finally, we explored the association between genetic liability to PVS and stroke, conditioning on blood pressure (SBP and DBP separately), by running multivariable MR analyses using TwoSampleMR. A P\\u2009<\\u20091.19\\u2009\\u00d7\\u200910\\u22123, correcting for 14 independent phenotypes and the three PVS locations, was considered significant.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63024, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Pathway analyses\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63041, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We used MAGMA gene set analyses (in FUMA) to identify pathways overrepresented in the associations. We identified genes associated with extensive PVS burden and estimated the correlation between genes. The P values and gene correlation matrix were used in a generalized least squares model. A P\\u2009<\\u20093.2\\u2009\\u00d7\\u200910\\u22126 correction for 15,496 gene sets was considered significant. As a sensitivity analysis, we used VEGAS2Pathway, which aggregates association strengths of individual markers into prespecified biological pathways using VEGAS-derived gene association P values for extensive PVS burden, with an empirical significance threshold of P\\u2009<\\u20091\\u2009\\u00d7\\u200910\\u22125 (accounting for 6,213 correlated pathways).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63753, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Enrichment analyses in OMIM and COSMIC genes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63798, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Using hypergeometric tests, we performed enrichment analyses of genes within 1\\u2009Mb, 100\\u2009kb or 10\\u2009kb of the lead variants, but also of genes within 10\\u2009kb of the lead variants with intragenic variants, and genes within 10\\u2009kb of the genetic loci with intragenic lead variants. We used the rest of the protein-coding genome as a reference. We performed the analysis first combining loci of all PVS locations, and second including only WM-PVS loci. We searched for an enrichment in different gene groups from the OMIM database, including PVS (\\u2018perivascular space\\u2019 OR \\u2018virchow-robin space\\u2019), WMH (\\u2018leukoaraiosis\\u2019 OR \\u2018white matter lesion\\u2019 OR \\u2018white matter hyperintensities\\u2019) and leukodystrophy (\\u2018leukodystrophy\\u2019 OR \\u2018leukoencephalopathy\\u2019) genes. We also searched for an enrichment of genes involved in glioma and glioblastoma, identified in the Catalog Of Somatic Mutations In Cancer (COSMIC) (https://cancer.sanger.ac.uk).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64751, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"TWAS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64756, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We performed TWAS using TWAS-Fusion, to identify genes whose expression is significantly associated with PVS burden without directly measuring expression levels. We restricted the analysis to tissues considered relevant for cerebrovascular disease, and used precomputed functional weights from 22 publicly available gene expression reference panels from blood, arterial, brain and peripheral nerve tissues (Fig. 4). TWAS-Fusion was then used to estimate the TWAS association statistics between predicted gene expression and PVS burden by integrating information from expression reference panels (SNP expression weights), GWAS summary statistics (SNP PVS effect estimates) and LD reference panels (SNP correlation matrix). Transcriptome-wide significant genes (eGenes) and the corresponding expression quantitative trait loci (eQTLs) were determined using Bonferroni correction (P\\u2009<\\u20093.93\\u2009\\u00d7\\u200910\\u22126, correcting for 4,235 genes tested and three PVS locations). eGenes were then tested in conditional analyses as implemented in TWAS-Fusion. Next, we performed a genetic colocalization analysis of gene expression and PVS burden for each conditionally significant gene (P\\u2009<\\u20090.05) using COLOC, to estimate the posterior probability of a shared causal variant between the gene expression and the trait (PP4\\u2009\\u2265\\u20090.75). Gene regions with eQTLs not reaching genome-wide significance in association with PVS, and not in LD (r2\\u2009<\\u20090.01) with the lead SNP for genome-wide significant PVS risk loci, were considered as novel.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66287, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Cell type enrichment analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66317, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We conducted a cell type enrichment analysis using Single cell Type Enrichment Analysis for Phenotypes (https://github.com/erwinerdem/STEAP/). This is an extension to CELLECT and uses S-LDSC, MAGMA and H-MAGMA for enrichment analysis. PVS GWAS summary statistics were munged. Then, expression specificity profiles were calculated using human and mouse single-cell RNA sequencing databases (PsychENCODE DER-22, GSE67835, GSE101601, DroNc Human Hippocampus, Allen Brain Atlas MTG and LNG, Mousebrain, Tabula Muris, Descartes Human Cerebrum and Cerebellum; Supplementary Table 24). Cell type enrichment was calculated with MAGMA, H-MAGMA (incorporating chromatin interaction profiles from human brain tissues in MAGMA) and stratified LD-score regression. P values were corrected for the number of independent cell types in each database.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67152, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Lifetime brain gene expression profile\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67191, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We studied the lifetime expression of genes identified in the TWAS-COLOC analysis, and the three genes associated with WM-PVS burden in both the old and young populations, to search for developmental processes. We used a public database (https://hbatlas.org/) comprising genome-wide exon-level transcriptome data from 1,340 tissue samples from 16 brain regions (cerebellar cortex, mediodorsal nucleus of the thalamus, striatum, amygdala, hippocampus and 11 neocortex areas) of 57 postmortem human brains, from embryonic development to older adults of different ancestries.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67764, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Enrichment in drug target genes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67796, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We used the GREP (Genome for Repositioning) software tool, which quantifies an enrichment of gene sets from GWAS summary statistics in drugs of certain Anatomical Therapeutic Chemical Classification (ATC) classes, or indicated for some ICD10 (10th revision of the International Statistical Classification of Diseases and Related Health Problems) disease categories, and captures potentially repositionable drugs targeting the gene set. Genes with false discovery rate FDR q\\u2009<\\u20090.1 in MAGMA were used for enrichment analyses (in GREP) of target genes for approved or investigated drugs curated in DrugBank and the Therapeutic Target Database.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68441, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We used the Trans-Phar (integration of TWAS and Pharmacological database) software to identify drug target candidates in a specific tissue or cell type, using first FOCUS to identify up- and downregulated genes in participants with extensive PVS burden, followed by a negative Spearman\\u2019s rank correlation analysis between the gene expression (Z-score) of the top 10% genes with the highest expression variation and the LINCS CMap L1000 library database (Extended Data Fig. 8).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68920, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Reporting summary\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68938, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69056, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_1\"}, \"text\": \"Online content\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69071, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Any methods, additional references, Nature Portfolio reporting summaries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author contributions and competing interests; and statements of data and code availability are available at 10.1038/s41591-023-02268-w.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69390, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_1\"}, \"text\": \"Supplementary information\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69416, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_1\"}, \"text\": \"Extended data\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69430, \"infons\": {\"file\": \"41591_2023_2268_Fig5_ESM.jpg\", \"id\": \"Fig5\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Manhattan and QQ plots of extensive PVS burden in the cross-ancestry and European meta-analyses.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69527, \"infons\": {\"file\": \"41591_2023_2268_Fig5_ESM.jpg\", \"id\": \"Fig5\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Manhattan and Quantile-Quantile (QQ) plots of the p-values (observed versus expected) in the cross-ancestry (A) and European (B) GWAS meta-analyses are presented along with the genomic inflation factor (\\u03bb). For QQ plots the observed -log10(p) is represented in the y-axis and the expected -log10(p) in x-axis. The dotted line corresponds to the genome-wide significance threshold (p\\u2009=\\u20095\\u00d710\\u22128, two sided). PVS indicates perivascular spaces; WM, white matter; BG, basal ganglia; HIP, hippocampus.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70030, \"infons\": {\"file\": \"41591_2023_2268_Fig6_ESM.jpg\", \"id\": \"Fig6\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Two-sample Mendelian randomization analysis between putative risk factors and extensive PVS burden.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70130, \"infons\": {\"file\": \"41591_2023_2268_Fig6_ESM.jpg\", \"id\": \"Fig6\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Two-sample Mendelian randomization was conducted using European PVS GWAS summary statistics (N\\u2009=\\u200938,598 (WM-PVS), N\\u2009=\\u200938,903 (BG-PVS) and N\\u2009=\\u200938,871 (HIP-PVS)), combined with summary statistics for lacunes (N\\u2009=\\u20091,715 cases / N\\u2009=\\u200915,096 controls) and WMH volume (N\\u2009=\\u200948,454) in MTAG, for the outcomes and European GWAS summary statistics for blood pressure traits (N\\u2009=\\u2009757,601) to generate instruments for exposures (Supplementary Tables 1 and 29). Only significant associations after multiple testing correction (p\\u2009<\\u20091.19\\u00d710\\u22123) in GSMR are shown. Each dot (or triangle if p\\u2009<\\u20091.19\\u00d710\\u22123) represents the beta estimate from Mendelian randomization with lines representing the 95% confidence interval. Two-sided p-values of GSMR are reported. PVS indicates perivascular spaces; WM, white matter; BG, basal ganglia; HIP, hippocampus; MTAG, multi-trait analysis of genome-wide association summary statistics; DBP, diastolic blood pressure; PP, pulse pressure; SBP, systolic blood pressure; NS, not significant; GSMR, Generalised Summary-data-based Mendelian Randomisation; IVW, Inverse variance weighted; 2SMR, Two-SampleMR.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71291, \"infons\": {\"file\": \"41591_2023_2268_Fig7_ESM.jpg\", \"id\": \"Fig7\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Two-sample Mendelian randomization analysis between extensive PVS burden and neurological traits.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71389, \"infons\": {\"file\": \"41591_2023_2268_Fig7_ESM.jpg\", \"id\": \"Fig7\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Two-sample Mendelian randomization was conducted using European GWAS summary statistics for any stroke (N\\u2009=\\u200940,585 cases / N\\u2009=\\u2009406,111 controls), ischemic stroke (N\\u2009=\\u200934,217 cases / N\\u2009=\\u2009400,201 controls) and small vessel stroke (N\\u2009=\\u20095,386 cases / N\\u2009=\\u2009254,558 controls) for outcomes and European PVS GWAS summary statistics (N\\u2009=\\u200938,598 (WM-PVS), N\\u2009=\\u200938,903 (BG-PVS) and N\\u2009=\\u200938,871 (HIP-PVS)), combined with summary statistics for lacunes (N\\u2009=\\u20091,715 cases / N\\u2009=\\u200915,096 controls) and WMH volume (N\\u2009=\\u200948,454) in MTAG, to generate instruments for exposures (Supplementary Tables 1 and 29). Only significant associations after multiple testing correction (p\\u2009<\\u20091.19\\u00d710\\u22123) in GSMR are shown. Each dot (or triangle if p\\u2009<\\u20091.19\\u00d710\\u22123) represents the beta estimate from Mendelian randomization with lines representing the 95% confidence interval. Two-sided p-values of GSMR are reported. PVS indicates perivascular spaces; WM, white matter; BG, basal ganglia; HIP, hippocampus; MTAG, multi-trait analysis of genome-wide association summary statistics; NS, not significant; GSMR, Generalised Summary-data-based Mendelian Randomisation; IVW, Inverse variance weighted; 2SMR, Two-SampleMR.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72626, \"infons\": {\"file\": \"41591_2023_2268_Fig8_ESM.jpg\", \"id\": \"Fig8\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Enrichment of PVS risk loci in genes mutated in OMIM syndromes.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72690, \"infons\": {\"file\": \"41591_2023_2268_Fig8_ESM.jpg\", \"id\": \"Fig8\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Enrichment of all perivascular spaces (PVS) loci (left) and WM-PVS loci only (right) in genes mutated in OMIM syndromes associated with white matter hyperintensities, leukodystrophy, leukoencephalopathy, according to distance from the lead variant; * p\\u2009<\\u20090.05; ** p\\u2009<\\u2009(0.05/5); *** p\\u2009<\\u2009(0.05/5/2). Exact p-values for all PVS loci: p\\u2009=\\u20090.041 (genes with intragenic SNPs), p\\u2009=\\u20099,27\\u00d710\\u221206 (genes with intragenic lead SNPs); exact p-values for WM-PVS loci only: p\\u2009=\\u20090,018 (genes within 10\\u2009kb distance), p\\u2009=\\u20090.008 (genes with intragenic SNPs), p\\u2009=\\u20098,61\\u00d710\\u221208 (genes with intragenic lead SNPs), full results are provided in Supplementary Table 22.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 73372, \"infons\": {\"file\": \"41591_2023_2268_Fig9_ESM.jpg\", \"id\": \"Fig9\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Brain expression pattern across the lifespan of genes near genome-wide significant PVS loci that are also identified in the TWAS and peaking in the pre-natal period (UMPS and LAMC1).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 73555, \"infons\": {\"file\": \"41591_2023_2268_Fig9_ESM.jpg\", \"id\": \"Fig9\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"The figure displays brain expression patterns across the lifespan of the 2 significant PVS TWAS-COLOC genes from genome-wide significant PVS GWAS loci peaking in the pre-natal period (UMPS for WM-PVS and LAMC1 for HIP-PVS) (see Supplementary Table 4 for brain expression pattern across the lifespan of other genes). The spatio-temporal gene expression level is plotted as log2-transformed exon array signal intensity (y-axis) against the post conception days (x-axis) as provided by the Human Brain Transcriptome project database. Periods of human development and adulthood are indicated by vertical dashed lines: 4-8 post conception weeks [PCW] (period 1), 8-10 PCW (period 2), 10-13 PCW (period 3), 13-16 PCW (period 4), 16-19 PCW (period 5), 19-24 PCW (period 6), 24-38 PCW (period 7), birth-6 postnatal months (period 8), 6-12 postnatal months (period 9), 1-6 years (period 10), 6-12 years (period 11), 12-20 years (period 12), 20-40 years (period 13), 40-60 years (period 14), and 60 years\\u2009+\\u2009(period 15). The boundary between pre- and postnatal periods is indicated by the solid vertical line. Each colored point represents the expression level of each gene across 16 anatomical brain regions and ages. Brain structure includes 11 neocortical areas (NCX, blue), and 5 subcortical regions: hippocampus (HIP, cyan), amygdala (AMY, orange), striatum (STR, black), mediodorsal nucleus of thalamus (MD, dark green), and cerebellar cortex (CBC, red). Neocortical areas include orbital prefrontal cortex (OFC), dorsolateral prefrontal cortex (DFC), ventrolateral prefrontal cortex (VFC), medial prefrontal cortex (MFC), primary motor cortex (M1C), primary somatosensory cortex (S1C), posterior inferior parietal cortex (IPC), primary auditory cortex (A1C), posterior superior temporal cortex (STC), inferior temporal cortex (ITC), and primary visual cortex (V1C).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 75421, \"infons\": {\"file\": \"41591_2023_2268_Fig10_ESM.jpg\", \"id\": \"Fig10\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Enrichment of PVS genes in targets of drugs validated in other indications (Genome for REPositioning drugs, using ICD10 codes).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 75549, \"infons\": {\"file\": \"41591_2023_2268_Fig10_ESM.jpg\", \"id\": \"Fig10\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Using the Genome for REPositioning drugs (GREP) software (A), we found a significant enrichment of BG-PVS genes in targets of drugs for diseases of the nervous system (ICD10 codes G50-G59, G70-G73, and G80-G83, OR\\u2009=\\u200951.0, 49.7, and 60.4, p\\u2009=\\u20093.90\\u00d710\\u22122, 4.00\\u00d710\\u22122, and 3.32\\u00d710\\u22122) and for symptoms and signs involving cognition, perception, emotional state and behaviour (ICD10 codes R40-R46, OR\\u2009=\\u200948.4, p\\u2009=\\u20094.10\\u00d710\\u22122), driven by MAPT (chr17q21.31), a target for davunetide (B). We also found a significant enrichment of HIP-PVS genes in targets of drugs for diseases of the ear (ICD10 codes H90-H95, OR\\u2009=\\u200957.6, p\\u2009=\\u20092.34\\u00d710\\u22122), driven by SERPIND1 (chr22q11.21) a target for sulodexide, also used for the prophylaxis and treatment of vascular diseases with increased risk of thrombosis (B). PVS indicates perivascular spaces; ATC, Anatomical Therapeutic Chemical classification; ICD10, the 10th revision of the International Statistical Classification of Diseases and Related Health Problems; TTD, Therapeutic Target Database.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 76612, \"infons\": {\"file\": \"41591_2023_2268_Fig11_ESM.jpg\", \"id\": \"Fig11\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Enrichment of PVS genes in targets of drugs validated in other indications (Genome for REPositioning drugs, using ATC codes).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 76738, \"infons\": {\"file\": \"41591_2023_2268_Fig11_ESM.jpg\", \"id\": \"Fig11\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"We found a significant enrichment of BG-PVS genes in targets for antiinfectives for systemic use (ATC J01, OR\\u2009=\\u2009252.4, P\\u2009=\\u20098.85\\u00d710\\u22123), driven by CRHR1 (chr17q21.31), a target for telavancin (A, B); of note, CRHR1 antagonists were shown to attenuate blood brain barrier permeability, cortical vascular hyperpermeability and tight junction disruption. ATC indicates Anatomical Therapeutic Chemical classification.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 77161, \"infons\": {\"file\": \"41591_2023_2268_Fig12_ESM.jpg\", \"id\": \"Fig12\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Enrichment of PVS genes in targets of drugs validated in other indications (Trans-Phar).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 77250, \"infons\": {\"file\": \"41591_2023_2268_Fig12_ESM.jpg\", \"id\": \"Fig12\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"We used the Integration of Transcriptome-wide Association Study and Pharmacological Database (Trans-Phar) software, leveraging TWAS on all GTEx v7 tissues and a database of compound-regulated gene expression (C-map) (Methods). A TWAS using FOCUS, which demonstrates fine-mapping of causal gene sets from TWAS results, and 27 tissues in the GTEx v7 database (corresponding to defined 13 tissue-cell-type categories assigned by the 27 tissues in GTEx v7 database and 77 LINCS CMap L1000 library cell types) was performed to identify up- and down-regulated genes in participants with extensive PVS burden, and select the top 10% genes with the highest expression variation. We observed significant enrichment of HIP-PVS in drugs for vascular diseases, including simvastatin (lipid-lowering drug, p\\u2009=\\u20091.64\\u00d710\\u22124), vincamine (vasodilator increasing cerebral blood flow, p\\u2009=\\u20092.87\\u00d710\\u22124), and macitentan (used for pulmonary arterial hypertension, p\\u2009=\\u20097.46\\u00d710\\u22124). PVS indicates perivascular spaces.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78260, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Publisher\\u2019s note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78398, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"These authors contributed equally: Marie-Gabrielle Duperron, Maria J. Knol, Quentin Le Grand, Tavia E. Evans, Aniket Mishra, Ami Tsuchida.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78537, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"These authors jointly supervised this work: Fumihiko Matsuda, Christophe Tzourio, Joanna M. Wardlaw, Sudha Seshadri, Hieab H. H. Adams, St\\u00e9phanie Debette.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78693, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"A list of authors and their affiliations appears at the end of the paper\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78766, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_2\"}, \"text\": \"Extended data\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78780, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"paragraph\"}, \"text\": \"is available for this paper at 10.1038/s41591-023-02268-w.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78839, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_2\"}, \"text\": \"Supplementary information\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78865, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"paragraph\"}, \"text\": \"The online version contains supplementary material available at 10.1038/s41591-023-02268-w.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78957, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"title\"}, \"text\": \"Author contributions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78978, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"paragraph\"}, \"text\": \"F.M., C.T., J.M.W., S.Seshadri, H.H.H.A. and S.D. jointly supervised research. M.-G.D., M.J.K., Q.LG., T.E.E., A.M. and A.Tsuchida contributed equally. M.-G.D., M.J.K., H.H.H.A. and S.D. designed and conceived the study. J.R.R., S.F., M.L., E.H., M.B., N.D.D., P.D., S.H., R.M.T., F.D., J.S., Y.S., N.J.A., C.B., M.E.B., A.B., H.B., R.B., C. Carrera, C.Chen, C.-Y.C., I.J.D., P.G.G., J.J.H., J.J., T.K., S.L., M.M., P.M., Z.M., S.M.M., S.M., M.O., M.P., P.P., T.R., M.S., S. Schilling, K.S., O.S., Y.T., A.T., A.Thalamuthu, J.N.T., M.C.V.H., M.W.V., U.V., K.W., T.Y.W., M.J.W., J.Z., W.Z., Y.-C.Z., H.S., P.S.S., W.W., K.Y., C.L.S., R.L.S., G.B., M.Lathrop, I.F.-C., Q.Y., B.M., P.B., H.J.G., K.A.M., R.S., M.J. and M.A.I. generated the PVS phenotype and genomic data and conducted cohort-wise GWAS analyses. G.R., T.K., D.-A.T., A.J., T.P. and Y.O. contributed to bioinformatics analyses. M.-G.D., S.D., M.J.K., Q.L.G., T.E.E., A.M., A.Tsuchida, F.M., C.T., J.M.W., S.Seshadri and H.H.H.A. wrote and edited the manuscript.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 80002, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"title\"}, \"text\": \"Peer review\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 80014, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"title_1\"}, \"text\": \"Peer review information\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 80038, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"paragraph\"}, \"text\": \"Nature Medicine thanks Andreas Charidimou, Timothy Frayling and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Anna Maria Ranzoni, in collaboration with the Nature Medicine team.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 80285, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title\"}, \"text\": \"Data availability\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 80303, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"paragraph\"}, \"text\": \"Genome-wide summary statistics for the European and cross-ancestry meta-analysis generated and analyzed during the current study are deposited on the GWAS Catalog (study code GCST90244151-GCST90244156). As for other meta-analyses of GWAS or sequencing data, individual cohort data are subject to controlled access, for privacy and legal issues (national and European regulations, including GDPR). This applies to all participating cohorts (cohorts included in the meta-analyses and follow-up cohorts). UKB data (GWAS and sequencing) are accessible by submitting an application to the UKB portal (this research has been conducted under Application Number 23509). We used publicly available data for analyses described in this manuscript, including data from GTEx (https://gtexportal.org/home/), the Gusev laboratory (http://gusevlab.org/projects/fusion/), the CommonMind Consortium (https://www.nimhgenetics.org/resources/commonmind), the Netherlands Twin Registry (https://tweelingenregister.vu.nl/), the Young Finns Study (https://youngfinnsstudy.utu.fi/), OMIM (https://www.omim.org/), OMIM genes description are publicly available: GFAP (https://www.omim.org/entry/137780); SLC13A3 (https://www.omim.org/entry/618384); PNPT1 (https://www.omim.org/entry/610316), COSMIC (https://cancer.sanger.ac.uk), RNA sequencing datasets: PsychENCODE DER-22 (www.ncbi.nlm.nih.gov/geo/, accession code GSE97942), GSE67835 (www.ncbi.nlm.nih.gov/geo/, accession code GSE67835), GSE101601, DroNc_Human Hippocampus (https://www.gtexportal.org/home/datasets), Allen Brain Atlas (http://portal.brain-map.org/), Descartes_Human (https://descartes.brotmanbaty.org/), Mousebrain (http://mousebrain.org/), Tabula Muris (https://tabula-muris.ds.czbiohub.org/). All other data supporting the findings of this study are available within the article, the supplementary information or the supplementary data files.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 82191, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title\"}, \"text\": \"Code availability\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 82209, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"paragraph\"}, \"text\": \"The PVS quantification method used in the Nagahama Study is available using this link: https://github.com/pboutinaud/SHIVA_PVS. We used publicly available data from METAL (https://github.com/statgen/METAL), GCTA-cojo (https://yanglab.westlake.edu.cn/software/gcta/#COJO), FUMA (https://fuma.ctglab.nl/), MAGMA (https://ctg.cncr.nl/software/magma), LD-Score Regression (https://github.com/bulik/ldsc/), MTAG (https://github.com/JonJala/mtag), coloc (https://chr1swallace.github.io/coloc/) R package, RadialMR (https://github.com/WSpiller/RadialMR), TwoSampleMR (https://mrcieu.github.io/TwoSampleMR/), STEAP (https://github.com/erwinerdem/STEAP/), CELLECT (https://github.com/perslab/CELLECT), S-LSC (https://github.com/bulik/ldsc), H-MAGMA (https://github.com/thewonlab/H-MAGMA), HBT (https://hbatlas.org/). Drug discovery analysis was conducted using the following publicly available tools: GREP (https://github.com/saorisakaue/GREP), Trans-Phar (https://github.com/konumat/Trans-Phar).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 83197, \"infons\": {\"section_type\": \"COMP_INT\", \"type\": \"title\"}, \"text\": \"Competing interests\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 83217, \"infons\": {\"section_type\": \"COMP_INT\", \"type\": \"paragraph\"}, \"text\": \"The authors declare no competing interests.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 83261, \"infons\": {\"section_type\": \"REF\", \"type\": \"title\"}, \"text\": \"References\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 83272, \"infons\": {\"fpage\": \"337\", \"lpage\": \"346\", \"name_0\": \"surname:Pollock;given-names:H\", \"name_1\": \"surname:Hutchings;given-names:M\", \"name_2\": \"surname:Weller;given-names:RO\", \"name_3\": \"surname:Zhang;given-names:ET\", \"pub-id_doi\": \"10.1046/j.1469-7580.1997.19130337.x\", \"pub-id_pmid\": \"9418990\", \"section_type\": \"REF\", \"source\": \"J. Anat.\", \"type\": \"ref\", \"volume\": \"191\", \"year\": \"1997\"}, \"text\": \"Perivascular spaces in the basal ganglia of the human brain: their relationship to lacunes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 83363, \"infons\": {\"fpage\": \"137\", \"lpage\": \"153\", \"name_0\": \"surname:Wardlaw;given-names:JM\", \"pub-id_doi\": \"10.1038/s41582-020-0312-z\", \"pub-id_pmid\": \"32094487\", \"section_type\": \"REF\", \"source\": \"Nat. Rev. Neurol.\", \"type\": \"ref\", \"volume\": \"16\", \"year\": \"2020\"}, \"text\": \"Perivascular spaces in the brain: anatomy, physiology and pathology\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 83431, \"infons\": {\"fpage\": \"822\", \"lpage\": \"838\", \"name_0\": \"surname:Wardlaw;given-names:JM\", \"pub-id_doi\": \"10.1016/S1474-4422(13)70124-8\", \"pub-id_pmid\": \"23867200\", \"section_type\": \"REF\", \"source\": \"Lancet Neurol.\", \"type\": \"ref\", \"volume\": \"12\", \"year\": \"2013\"}, \"text\": \"Neuroimaging standards for research into small vessel disease and its contribution to ageing and neurodegeneration\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 83546, \"infons\": {\"fpage\": \"2583\", \"lpage\": \"2599\", \"name_0\": \"surname:Jessen;given-names:NA\", \"name_1\": \"surname:Munk;given-names:ASF\", \"name_2\": \"surname:Lundgaard;given-names:I\", \"name_3\": \"surname:Nedergaard;given-names:M\", \"pub-id_doi\": \"10.1007/s11064-015-1581-6\", \"pub-id_pmid\": \"25947369\", \"section_type\": \"REF\", \"source\": \"Neurochem. Res.\", \"type\": \"ref\", \"volume\": \"40\", \"year\": \"2015\"}, \"text\": \"The glymphatic system: a beginner\\u2019s guide\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 83590, \"infons\": {\"fpage\": \"6285\", \"name_0\": \"surname:Sargurupremraj;given-names:M\", \"pub-id_doi\": \"10.1038/s41467-020-19111-2\", \"pub-id_pmid\": \"33293549\", \"section_type\": \"REF\", \"source\": \"Nat. Commun.\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2020\"}, \"text\": \"Cerebral small vessel disease genomics and its implications across the lifespan\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 83670, \"infons\": {\"fpage\": \"81\", \"lpage\": \"94\", \"name_0\": \"surname:Debette;given-names:S\", \"name_1\": \"surname:Schilling;given-names:S\", \"name_2\": \"surname:Duperron;given-names:MG\", \"name_3\": \"surname:Larsson;given-names:SC\", \"name_4\": \"surname:Markus;given-names:HS\", \"pub-id_doi\": \"10.1001/jamaneurol.2018.3122\", \"pub-id_pmid\": \"30422209\", \"section_type\": \"REF\", \"source\": \"JAMA Neurol.\", \"type\": \"ref\", \"volume\": \"76\", \"year\": \"2019\"}, \"text\": \"Clinical significance of magnetic resonance imaging markers of vascular brain injury: a systematic review and meta-analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 83794, \"infons\": {\"fpage\": \"2257\", \"lpage\": \"2274\", \"name_0\": \"surname:Mestre;given-names:H\", \"name_1\": \"surname:Kostrikov;given-names:S\", \"name_2\": \"surname:Mehta;given-names:RI\", \"name_3\": \"surname:Nedergaard;given-names:M\", \"pub-id_doi\": \"10.1042/CS20160381\", \"pub-id_pmid\": \"28798076\", \"section_type\": \"REF\", \"source\": \"Clin. Sci. (Lond.)\", \"type\": \"ref\", \"volume\": \"131\", \"year\": \"2017\"}, \"text\": \"Perivascular spaces, glymphatic dysfunction, and small vessel disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 83864, \"infons\": {\"fpage\": \"1043\", \"lpage\": \"1050\", \"name_0\": \"surname:Deramecourt;given-names:V\", \"pub-id_doi\": \"10.1212/WNL.0b013e31824e8e7f\", \"pub-id_pmid\": \"22377814\", \"section_type\": \"REF\", \"source\": \"Neurology\", \"type\": \"ref\", \"volume\": \"78\", \"year\": \"2012\"}, \"text\": \"Staging and natural history of cerebrovascular pathology in dementia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 83933, \"infons\": {\"fpage\": \"101\", \"name_0\": \"surname:Bacyinski;given-names:A\", \"name_1\": \"surname:Xu;given-names:M\", \"name_2\": \"surname:Wang;given-names:W\", \"name_3\": \"surname:Hu;given-names:J\", \"pub-id_doi\": \"10.3389/fnana.2017.00101\", \"pub-id_pmid\": \"29163074\", \"section_type\": \"REF\", \"source\": \"Front. Neuroanat.\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2017\"}, \"text\": \"The paravascular pathway for brain waste clearance: current understanding, significance and controversy\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 84037, \"infons\": {\"fpage\": \"158\", \"lpage\": \"165\", \"name_0\": \"surname:Duperron;given-names:MG\", \"pub-id_doi\": \"10.1016/j.neurobiolaging.2019.08.031\", \"pub-id_pmid\": \"31629114\", \"section_type\": \"REF\", \"source\": \"Neurobiol. Aging\", \"type\": \"ref\", \"volume\": \"84\", \"year\": \"2019\"}, \"text\": \"High dilated perivascular space burden: a new MRI marker for risk of intracerebral hemorrhage\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 84131, \"infons\": {\"fpage\": \"1157\", \"lpage\": \"1164\", \"name_0\": \"surname:Charidimou;given-names:A\", \"pub-id_doi\": \"10.1212/WNL.0000000000003746\", \"pub-id_pmid\": \"28228568\", \"section_type\": \"REF\", \"source\": \"Neurology\", \"type\": \"ref\", \"volume\": \"88\", \"year\": \"2017\"}, \"text\": \"MRI-visible perivascular spaces in cerebral amyloid angiopathy and hypertensive arteriopathy\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 84224, \"infons\": {\"fpage\": \"2356\", \"lpage\": \"2362\", \"name_0\": \"surname:Tsai;given-names:HH\", \"pub-id_doi\": \"10.1161/STROKEAHA.120.032139\", \"pub-id_pmid\": \"33874751\", \"section_type\": \"REF\", \"source\": \"Stroke\", \"type\": \"ref\", \"volume\": \"52\", \"year\": \"2021\"}, \"text\": \"Centrum semiovale perivascular space and amyloid deposition in spontaneous intracerebral hemorrhage\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 84324, \"infons\": {\"fpage\": \"714\", \"lpage\": \"725\", \"name_0\": \"surname:Charidimou;given-names:A\", \"pub-id_doi\": \"10.1016/S1474-4422(22)00208-3\", \"pub-id_pmid\": \"35841910\", \"section_type\": \"REF\", \"source\": \"Lancet Neurol.\", \"type\": \"ref\", \"volume\": \"21\", \"year\": \"2022\"}, \"text\": \"The Boston criteria version 2.0 for cerebral amyloid angiopathy: a multicentre, retrospective, MRI-neuropathology diagnostic accuracy study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 84464, \"infons\": {\"fpage\": \"eaax7171\", \"name_0\": \"surname:Mestre;given-names:H\", \"pub-id_doi\": \"10.1126/science.aax7171\", \"pub-id_pmid\": \"32001524\", \"section_type\": \"REF\", \"source\": \"Science\", \"type\": \"ref\", \"volume\": \"367\", \"year\": \"2020\"}, \"text\": \"Cerebrospinal fluid influx drives acute ischemic tissue swelling\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 84529, \"infons\": {\"fpage\": \"640\", \"lpage\": \"646\", \"name_0\": \"surname:M\\u00e5nberg;given-names:A\", \"pub-id_doi\": \"10.1038/s41591-021-01295-9\", \"pub-id_pmid\": \"33859435\", \"section_type\": \"REF\", \"source\": \"Nat. Med\", \"type\": \"ref\", \"volume\": \"27\", \"year\": \"2021\"}, \"text\": \"Altered perivascular fibroblast activity precedes ALS disease onset\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 84597, \"infons\": {\"fpage\": \"282\", \"lpage\": \"287\", \"name_0\": \"surname:Duperron;given-names:MG\", \"pub-id_doi\": \"10.1161/STROKEAHA.117.019309\", \"pub-id_pmid\": \"29311265\", \"section_type\": \"REF\", \"source\": \"Stroke\", \"type\": \"ref\", \"volume\": \"49\", \"year\": \"2018\"}, \"text\": \"Burden of dilated perivascular spaces, an emerging marker of cerebral small vessel disease, is highly heritable\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 84709, \"infons\": {\"fpage\": \"2118\", \"lpage\": \"2125\", \"name_0\": \"surname:Yao;given-names:M\", \"pub-id_doi\": \"10.1016/j.neurobiolaging.2014.03.021\", \"pub-id_pmid\": \"24731517\", \"section_type\": \"REF\", \"source\": \"Neurobiol. Aging\", \"type\": \"ref\", \"volume\": \"35\", \"year\": \"2014\"}, \"text\": \"Hippocampal perivascular spaces are related to aging and blood pressure but not to cognition\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 84802, \"infons\": {\"fpage\": \"307\", \"lpage\": \"313\", \"name_0\": \"surname:Bouvy;given-names:WH\", \"pub-id_doi\": \"10.1097/RLI.0000000000000027\", \"pub-id_pmid\": \"24473365\", \"section_type\": \"REF\", \"source\": \"Invest. Radiol.\", \"type\": \"ref\", \"volume\": \"49\", \"year\": \"2014\"}, \"text\": \"Visualization of perivascular spaces and perforating arteries with 7 T magnetic resonance imaging\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 84900, \"infons\": {\"fpage\": \"73\", \"lpage\": \"80\", \"name_0\": \"surname:Psaty;given-names:BM\", \"pub-id_doi\": \"10.1161/CIRCGENETICS.108.829747\", \"pub-id_pmid\": \"20031568\", \"section_type\": \"REF\", \"source\": \"Circ. Cardiovasc. Genet.\", \"type\": \"ref\", \"volume\": \"2\", \"year\": \"2009\"}, \"text\": \"Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium: design of prospective meta-analyses of genome-wide association studies from 5 cohorts\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 85068, \"infons\": {\"fpage\": \"84\", \"lpage\": \"101\", \"name_0\": \"surname:Bordes;given-names:C\", \"name_1\": \"surname:Sargurupremraj;given-names:M\", \"name_2\": \"surname:Mishra;given-names:A\", \"name_3\": \"surname:Debette;given-names:S\", \"pub-id_doi\": \"10.1038/s41582-021-00592-8\", \"pub-id_pmid\": \"34987231\", \"section_type\": \"REF\", \"source\": \"Nat. Rev. Neurol.\", \"type\": \"ref\", \"volume\": \"18\", \"year\": \"2022\"}, \"text\": \"Genetics of common cerebral small vessel disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 85117, \"infons\": {\"fpage\": \"143\", \"lpage\": \"157\", \"name_0\": \"surname:Joutel;given-names:A\", \"name_1\": \"surname:Haddad;given-names:I\", \"name_2\": \"surname:Ratelade;given-names:J\", \"name_3\": \"surname:Nelson;given-names:MT\", \"pub-id_doi\": \"10.1038/jcbfm.2015.62\", \"pub-id_pmid\": \"25853907\", \"section_type\": \"REF\", \"source\": \"J. Cereb. Blood Flow. Metab.\", \"type\": \"ref\", \"volume\": \"36\", \"year\": \"2016\"}, \"text\": \"Perturbations of the cerebrovascular matrisome: a convergent mechanism in small vessel disease of the brain?\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 85226, \"infons\": {\"section_type\": \"REF\", \"type\": \"ref\"}, \"text\": \"Traylor, M. et al. Genetic basis of lacunar stroke: a pooled analysis of individual patient data and genome-wide association studies. 20(5):351-361 Lancet Neurol.20, 351\\u2013361 (2021).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 85410, \"infons\": {\"fpage\": \"2175\", \"name_0\": \"surname:Persyn;given-names:E\", \"pub-id_doi\": \"10.1038/s41467-020-15932-3\", \"pub-id_pmid\": \"32358547\", \"section_type\": \"REF\", \"source\": \"Nat. Commun.\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2020\"}, \"text\": \"Genome-wide association study of MRI markers of cerebral small vessel disease in 42,310 participants\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 85511, \"infons\": {\"fpage\": \"1429\", \"lpage\": \"1440\", \"name_0\": \"surname:Simon;given-names:AJ\", \"pub-id_doi\": \"10.1084/jem.20151618\", \"pub-id_pmid\": \"27432940\", \"section_type\": \"REF\", \"source\": \"J. Exp. Med.\", \"type\": \"ref\", \"volume\": \"213\", \"year\": \"2016\"}, \"text\": \"Mutations in STN1 cause Coats plus syndrome and are associated with genomic and telomere defects\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 85608, \"infons\": {\"fpage\": \"e025629\", \"name_0\": \"surname:Whittaker;given-names:E\", \"pub-id_doi\": \"10.1161/JAHA.121.025629\", \"pub-id_pmid\": \"35699195\", \"section_type\": \"REF\", \"source\": \"J. Am. Heart Assoc.\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2022\"}, \"text\": \"Systematic review of cerebral phenotypes associated with monogenic cerebral small-vessel disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 85705, \"infons\": {\"fpage\": \"2139\", \"lpage\": \"2145\", \"name_0\": \"surname:Piantino;given-names:J\", \"pub-id_doi\": \"10.3174/ajnr.A6789\", \"pub-id_pmid\": \"33033050\", \"section_type\": \"REF\", \"source\": \"AJNR Am. J. Neuroradiol.\", \"type\": \"ref\", \"volume\": \"41\", \"year\": \"2020\"}, \"text\": \"Characterization of MR imaging-visible perivascular spaces in the white matter of healthy adolescents at 3T\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 85813, \"infons\": {\"fpage\": \"eaam9507\", \"name_0\": \"surname:Rajani;given-names:RM\", \"pub-id_doi\": \"10.1126/scitranslmed.aam9507\", \"pub-id_pmid\": \"29973407\", \"section_type\": \"REF\", \"source\": \"Sci. Transl. Med.\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2018\"}, \"text\": \"Reversal of endothelial dysfunction reduces white matter vulnerability in cerebral small vessel disease in rats\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 85925, \"infons\": {\"fpage\": \"21\", \"lpage\": \"38\", \"name_0\": \"surname:Carelli;given-names:V\", \"pub-id_doi\": \"10.1002/ana.24410\", \"pub-id_pmid\": \"25820230\", \"section_type\": \"REF\", \"source\": \"Ann. Neurol.\", \"type\": \"ref\", \"volume\": \"78\", \"year\": \"2015\"}, \"text\": \"Syndromic parkinsonism and dementia associated with OPA1 missense mutations\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 86001, \"infons\": {\"fpage\": \"987\", \"lpage\": \"1003.e1008\", \"name_0\": \"surname:Herkenne;given-names:S\", \"pub-id_doi\": \"10.1016/j.cmet.2020.04.007\", \"pub-id_pmid\": \"32315597\", \"section_type\": \"REF\", \"source\": \"Cell Metab.\", \"type\": \"ref\", \"volume\": \"31\", \"year\": \"2020\"}, \"text\": \"Developmental and tumor angiogenesis requires the mitochondria-shaping protein Opa1\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 86085, \"infons\": {\"fpage\": \"3769\", \"lpage\": \"3778\", \"name_0\": \"surname:Backhouse;given-names:EV\", \"pub-id_doi\": \"10.1093/brain/awab331\", \"pub-id_pmid\": \"34581779\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"144\", \"year\": \"2021\"}, \"text\": \"Early life predictors of late life cerebral small vessel disease in four prospective cohort studies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 86185, \"infons\": {\"fpage\": \"eabf3736\", \"name_0\": \"surname:Zhao;given-names:B\", \"pub-id_doi\": \"10.1126/science.abf3736\", \"pub-id_pmid\": \"34140357\", \"section_type\": \"REF\", \"source\": \"Science\", \"type\": \"ref\", \"volume\": \"372\", \"year\": \"2021\"}, \"text\": \"Common genetic variation influencing human white matter microstructure\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 86256, \"infons\": {\"fpage\": \"eaay6690\", \"name_0\": \"surname:Grasby;given-names:KL\", \"pub-id_doi\": \"10.1126/science.aay6690\", \"pub-id_pmid\": \"32193296\", \"section_type\": \"REF\", \"source\": \"Science\", \"type\": \"ref\", \"volume\": \"367\", \"year\": \"2020\"}, \"text\": \"The genetic architecture of the human cerebral cortex\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 86310, \"infons\": {\"fpage\": \"36\", \"lpage\": \"42\", \"name_0\": \"surname:Mok;given-names:V\", \"pub-id_doi\": \"10.1111/ijs.12270\", \"pub-id_pmid\": \"24661839\", \"section_type\": \"REF\", \"source\": \"Int. J. Stroke\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2014\"}, \"text\": \"Race-ethnicity and cerebral small vessel disease \\u2013 comparison between Chinese and White populations\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 86412, \"infons\": {\"fpage\": \"634\", \"lpage\": \"656\", \"name_0\": \"surname:Akinyemi;given-names:RO\", \"pub-id_doi\": \"10.1038/s41582-021-00542-4\", \"pub-id_pmid\": \"34526674\", \"section_type\": \"REF\", \"source\": \"Nat. Rev. Neurol.\", \"type\": \"ref\", \"volume\": \"17\", \"year\": \"2021\"}, \"text\": \"Stroke in Africa: profile, progress, prospects and priorities\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 86474, \"infons\": {\"fpage\": \"809\", \"lpage\": \"819\", \"name_0\": \"surname:Mollink;given-names:J\", \"pub-id_doi\": \"10.1038/s41593-019-0379-2\", \"pub-id_pmid\": \"30988526\", \"section_type\": \"REF\", \"source\": \"Nat. Neurosci.\", \"type\": \"ref\", \"volume\": \"22\", \"year\": \"2019\"}, \"text\": \"The spatial correspondence and genetic influence of interhemispheric connectivity with white matter microstructure\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 86589, \"infons\": {\"fpage\": \"170\", \"name_0\": \"surname:Gaire;given-names:BP\", \"name_1\": \"surname:Sapkota;given-names:A\", \"name_2\": \"surname:Song;given-names:MR\", \"name_3\": \"surname:Choi;given-names:JW\", \"pub-id_doi\": \"10.1186/s12974-019-1555-8\", \"pub-id_pmid\": \"31429777\", \"section_type\": \"REF\", \"source\": \"J. Neuroinflammation\", \"type\": \"ref\", \"volume\": \"16\", \"year\": \"2019\"}, \"text\": \"Lysophosphatidic acid receptor 1 (LPA1) plays critical roles in microglial activation and brain damage after transient focal cerebral ischemia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 86732, \"infons\": {\"fpage\": \"53\", \"lpage\": \"54\", \"name_0\": \"surname:Gross;given-names:I\", \"name_1\": \"surname:Brauer;given-names:AU\", \"pub-id_doi\": \"10.4103/1673-5374.264452\", \"pub-id_pmid\": \"31535643\", \"section_type\": \"REF\", \"source\": \"Neural Regen. Res.\", \"type\": \"ref\", \"volume\": \"15\", \"year\": \"2020\"}, \"text\": \"Modulation of lysophosphatidic acid (LPA) receptor activity: the key to successful neural regeneration?\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 86836, \"infons\": {\"fpage\": \"172539\", \"name_0\": \"surname:Hisaoka-Nakashima;given-names:K\", \"pub-id_doi\": \"10.1016/j.ejphar.2019.172539\", \"section_type\": \"REF\", \"source\": \"Eur. J. Pharm.\", \"type\": \"ref\", \"volume\": \"860\", \"year\": \"2019\"}, \"text\": \"Mirtazapine increases glial cell line-derived neurotrophic factor production through lysophosphatidic acid 1 receptor-mediated extracellular signal-regulated kinase signaling in astrocytes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 87025, \"infons\": {\"fpage\": \"1634\", \"lpage\": \"1643\", \"name_0\": \"surname:Allanore;given-names:Y\", \"pub-id_doi\": \"10.1002/art.40547\", \"pub-id_pmid\": \"29732731\", \"section_type\": \"REF\", \"source\": \"Arthritis Rheumatol.\", \"type\": \"ref\", \"volume\": \"70\", \"year\": \"2018\"}, \"text\": \"Lysophosphatidic acid receptor 1 antagonist SAR100842 for patients with diffuse cutaneous systemic sclerosis: a double-blind, randomized, eight-week placebo-controlled study followed by a sixteen-week open-label extension study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 87253, \"infons\": {\"fpage\": \"1247\", \"lpage\": \"1250\", \"name_0\": \"surname:Stenman;given-names:JM\", \"pub-id_doi\": \"10.1126/science.1164594\", \"pub-id_pmid\": \"19023080\", \"section_type\": \"REF\", \"source\": \"Science\", \"type\": \"ref\", \"volume\": \"322\", \"year\": \"2008\"}, \"text\": \"Canonical Wnt signaling regulates organ-specific assembly and differentiation of CNS vasculature\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 87350, \"infons\": {\"fpage\": \"1130\", \"lpage\": \"1145.e1135\", \"name_0\": \"surname:Chavali;given-names:M\", \"pub-id_doi\": \"10.1016/j.neuron.2020.09.033\", \"pub-id_pmid\": \"33086038\", \"section_type\": \"REF\", \"source\": \"Neuron\", \"type\": \"ref\", \"volume\": \"108\", \"year\": \"2020\"}, \"text\": \"Wnt-dependent oligodendroglial-endothelial interactions regulate white matter vascularization and attenuate injury\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 87465, \"infons\": {\"fpage\": \"387\", \"lpage\": \"403\", \"name_0\": \"surname:Capone;given-names:C\", \"pub-id_doi\": \"10.1002/ana.24573\", \"pub-id_pmid\": \"26648042\", \"section_type\": \"REF\", \"source\": \"Ann. Neurol.\", \"type\": \"ref\", \"volume\": \"79\", \"year\": \"2016\"}, \"text\": \"Reducing Timp3 or vitronectin ameliorates disease manifestations in CADASIL mice\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 87546, \"infons\": {\"fpage\": \"e12018\", \"name_0\": \"surname:Tanaka;given-names:T\", \"pub-id_pmid\": \"32607407\", \"section_type\": \"REF\", \"source\": \"Alzheimers Dement. (N. Y.)\", \"type\": \"ref\", \"volume\": \"6\", \"year\": \"2020\"}, \"text\": \"Plasma proteomic signatures predict dementia and cognitive impairment\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 87616, \"infons\": {\"fpage\": \"1530\", \"lpage\": \"1539\", \"name_0\": \"surname:Jakobsson;given-names:L\", \"name_1\": \"surname:Domogatskaya;given-names:A\", \"name_2\": \"surname:Tryggvason;given-names:K\", \"name_3\": \"surname:Edgar;given-names:D\", \"name_4\": \"surname:Claesson-Welsh;given-names:L\", \"pub-id_doi\": \"10.1096/fj.07-9617com\", \"pub-id_pmid\": \"18073332\", \"section_type\": \"REF\", \"source\": \"FASEB J.\", \"type\": \"ref\", \"volume\": \"22\", \"year\": \"2008\"}, \"text\": \"Laminin deposition is dispensable for vasculogenesis but regulates blood vessel diameter independent of flow\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 87725, \"infons\": {\"fpage\": \"3413\", \"name_0\": \"surname:Yao;given-names:Y\", \"name_1\": \"surname:Chen;given-names:ZL\", \"name_2\": \"surname:Norris;given-names:EH\", \"name_3\": \"surname:Strickland;given-names:S\", \"pub-id_doi\": \"10.1038/ncomms4413\", \"pub-id_pmid\": \"24583950\", \"section_type\": \"REF\", \"source\": \"Nat. Commun.\", \"type\": \"ref\", \"volume\": \"5\", \"year\": \"2014\"}, \"text\": \"Astrocytic laminin regulates pericyte differentiation and maintains blood brain barrier integrity\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 87823, \"infons\": {\"fpage\": \"3176\", \"lpage\": \"3189\", \"name_0\": \"surname:Chung;given-names:J\", \"pub-id_doi\": \"10.1093/brain/awz233\", \"pub-id_pmid\": \"31430377\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"142\", \"year\": \"2019\"}, \"text\": \"Genome-wide association study of cerebral small vessel disease reveals established and novel loci\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 87921, \"infons\": {\"fpage\": \"2111\", \"lpage\": \"2121\", \"name_0\": \"surname:Armstrong;given-names:NJ\", \"pub-id_doi\": \"10.1161/STROKEAHA.119.027544\", \"pub-id_pmid\": \"32517579\", \"section_type\": \"REF\", \"source\": \"Stroke\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2020\"}, \"text\": \"Common genetic variation indicates separate causes for periventricular and deep white matter hyperintensities\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 88031, \"infons\": {\"fpage\": \"166\", \"lpage\": \"173\", \"name_0\": \"surname:Hadano;given-names:S\", \"pub-id_doi\": \"10.1038/ng1001-166\", \"pub-id_pmid\": \"11586298\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"29\", \"year\": \"2001\"}, \"text\": \"A gene encoding a putative GTPase regulator is mutated in familial amyotrophic lateral sclerosis 2\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 88130, \"infons\": {\"fpage\": \"4528\", \"name_0\": \"surname:Bindesb\\u00f8ll;given-names:C\", \"pub-id_doi\": \"10.1038/s41598-020-61352-0\", \"pub-id_pmid\": \"32161285\", \"section_type\": \"REF\", \"source\": \"Sci. Rep.\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2020\"}, \"text\": \"NBEAL1 controls SREBP2 processing and cholesterol metabolism and is a susceptibility locus for coronary artery disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 88249, \"infons\": {\"fpage\": \"3834\", \"lpage\": \"3867\", \"name_0\": \"surname:Wang;given-names:WW\", \"name_1\": \"surname:Gallo;given-names:L\", \"name_2\": \"surname:Jadhav;given-names:A\", \"name_3\": \"surname:Hawkins;given-names:R\", \"name_4\": \"surname:Parker;given-names:CG\", \"pub-id_doi\": \"10.1021/acs.jmedchem.9b01237\", \"pub-id_pmid\": \"31774679\", \"section_type\": \"REF\", \"source\": \"J. Med. Chem.\", \"type\": \"ref\", \"volume\": \"63\", \"year\": \"2020\"}, \"text\": \"The druggability of solute carriers\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 88285, \"infons\": {\"fpage\": \"385\", \"lpage\": \"395\", \"name_0\": \"surname:Dewulf;given-names:JP\", \"pub-id_doi\": \"10.1002/ana.25412\", \"pub-id_pmid\": \"30635937\", \"section_type\": \"REF\", \"source\": \"Ann. Neurol.\", \"type\": \"ref\", \"volume\": \"85\", \"year\": \"2019\"}, \"text\": \"SLC13A3 variants cause acute reversible leukoencephalopathy and \\u03b1-ketoglutarate accumulation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 88382, \"infons\": {\"fpage\": \"1236\", \"lpage\": \"1245\", \"name_0\": \"surname:Beyens;given-names:A\", \"pub-id_doi\": \"10.1038/gim.2017.253\", \"pub-id_pmid\": \"29323665\", \"section_type\": \"REF\", \"source\": \"Genet. Med.\", \"type\": \"ref\", \"volume\": \"20\", \"year\": \"2018\"}, \"text\": \"Arterial tortuosity syndrome: 40 new families and literature review\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 88450, \"infons\": {\"fpage\": \"567232\", \"name_0\": \"surname:Chen;given-names:YC\", \"pub-id_doi\": \"10.3389/fneur.2020.567232\", \"pub-id_pmid\": \"33193005\", \"section_type\": \"REF\", \"source\": \"Front. Neurol.\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2020\"}, \"text\": \"Correlation between internal carotid artery tortuosity and imaging of cerebral small vessel disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 88550, \"infons\": {\"fpage\": \"254\", \"lpage\": \"256\", \"name_0\": \"surname:Wang;given-names:C\", \"pub-id_doi\": \"10.1038/ng.1077\", \"pub-id_pmid\": \"22327515\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"44\", \"year\": \"2012\"}, \"text\": \"Mutations in SLC20A2 link familial idiopathic basal ganglia calcification with phosphate homeostasis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 88651, \"infons\": {\"fpage\": \"154\", \"name_0\": \"surname:Ho;given-names:HT\", \"name_1\": \"surname:Dahlin;given-names:A\", \"name_2\": \"surname:Wang;given-names:J\", \"pub-id_doi\": \"10.3389/fphar.2012.00154\", \"section_type\": \"REF\", \"source\": \"Front. Pharm.\", \"type\": \"ref\", \"volume\": \"3\", \"year\": \"2012\"}, \"text\": \"Expression profiling of solute carrier gene families at the blood-CSF barrier\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 88729, \"infons\": {\"fpage\": \"C1019\", \"lpage\": \"1030\", \"name_0\": \"surname:Wang;given-names:H\", \"pub-id_doi\": \"10.1152/ajpcell.2000.278.5.C1019\", \"pub-id_pmid\": \"10794676\", \"section_type\": \"REF\", \"source\": \"Am. J. Physiol. Cell Physiol.\", \"type\": \"ref\", \"volume\": \"278\", \"year\": \"2000\"}, \"text\": \"Structure, function, and genomic organization of human Na+-dependent high-affinity dicarboxylate transporter\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 88838, \"infons\": {\"fpage\": \"4878\", \"name_0\": \"surname:Mestre;given-names:H\", \"pub-id_doi\": \"10.1038/s41467-018-07318-3\", \"pub-id_pmid\": \"30451853\", \"section_type\": \"REF\", \"source\": \"Nat. Commun.\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2018\"}, \"text\": \"Flow of cerebrospinal fluid is driven by arterial pulsations and is reduced in hypertension\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 88930, \"infons\": {\"fpage\": \"381\", \"lpage\": \"395\", \"name_0\": \"surname:Chen;given-names:ZL\", \"pub-id_doi\": \"10.1083/jcb.201212032\", \"pub-id_pmid\": \"23857767\", \"section_type\": \"REF\", \"source\": \"J. Cell Biol.\", \"type\": \"ref\", \"volume\": \"202\", \"year\": \"2013\"}, \"text\": \"Ablation of astrocytic laminin impairs vascular smooth muscle cell function and leads to hemorrhagic stroke\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 89038, \"infons\": {\"fpage\": \"1829\", \"lpage\": \"1839\", \"name_0\": \"surname:Rannikm\\u00e4e;given-names:K\", \"pub-id_doi\": \"10.1212/WNL.0000000000004560\", \"pub-id_pmid\": \"28954878\", \"section_type\": \"REF\", \"source\": \"Neurology\", \"type\": \"ref\", \"volume\": \"89\", \"year\": \"2017\"}, \"text\": \"COL4A2 is associated with lacunar ischemic stroke and deep ICH: meta-analyses among 21,500 cases and 40,600 controls\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 89155, \"infons\": {\"fpage\": \"102120\", \"name_0\": \"surname:Ballerini;given-names:L\", \"pub-id_doi\": \"10.1016/j.nicl.2019.102120\", \"pub-id_pmid\": \"31887717\", \"section_type\": \"REF\", \"source\": \"Neuroimage Clin.\", \"type\": \"ref\", \"volume\": \"25\", \"year\": \"2020\"}, \"text\": \"Computational quantification of brain perivascular space morphologies: associations with vascular risk factors and white matter hyperintensities. A study in the Lothian Birth Cohort 1936\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 89342, \"infons\": {\"fpage\": \"1708\", \"lpage\": \"1717\", \"name_0\": \"surname:Bouvy;given-names:WH\", \"pub-id_doi\": \"10.1177/0271678X16648970\", \"pub-id_pmid\": \"27154503\", \"section_type\": \"REF\", \"source\": \"J. Cereb. Blood Flow. Metab.\", \"type\": \"ref\", \"volume\": \"36\", \"year\": \"2016\"}, \"text\": \"Perivascular spaces on 7 Tesla brain MRI are related to markers of small vessel disease but not to age or cardiovascular risk factors\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 89476, \"infons\": {\"fpage\": \"1992\", \"lpage\": \"2007\", \"name_0\": \"surname:Mishra;given-names:A\", \"pub-id_doi\": \"10.1093/brain/awab432\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"30\", \"year\": \"2022\"}, \"text\": \"Gene-mapping study of extremes of cerebral small vessel disease reveals TRIM47 as a strong candidate\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 89577, \"infons\": {\"fpage\": \"1192\", \"lpage\": \"1212\", \"name_0\": \"surname:Winkler;given-names:TW\", \"pub-id_doi\": \"10.1038/nprot.2014.071\", \"pub-id_pmid\": \"24762786\", \"section_type\": \"REF\", \"source\": \"Nat. Protoc.\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2014\"}, \"text\": \"Quality control and conduct of genome-wide association meta-analyses\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 89646, \"infons\": {\"fpage\": \"S361\", \"lpage\": \"363\", \"name_0\": \"surname:Yang;given-names:J\", \"pub-id_doi\": \"10.1038/ng.2213\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"44\", \"year\": \"2012\"}, \"text\": \"Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 89775, \"infons\": {\"fpage\": \"3639\", \"lpage\": \"3650\", \"name_0\": \"surname:Magi;given-names:R\", \"pub-id_doi\": \"10.1093/hmg/ddx280\", \"pub-id_pmid\": \"28911207\", \"section_type\": \"REF\", \"source\": \"Hum. Mol. Genet.\", \"type\": \"ref\", \"volume\": \"26\", \"year\": \"2017\"}, \"text\": \"Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 89930, \"infons\": {\"fpage\": \"1826\", \"name_0\": \"surname:Watanabe;given-names:K\", \"name_1\": \"surname:Taskesen;given-names:E\", \"name_2\": \"surname:van Bochoven;given-names:A\", \"name_3\": \"surname:Posthuma;given-names:D\", \"pub-id_doi\": \"10.1038/s41467-017-01261-5\", \"pub-id_pmid\": \"29184056\", \"section_type\": \"REF\", \"source\": \"Nat. Commun.\", \"type\": \"ref\", \"volume\": \"8\", \"year\": \"2017\"}, \"text\": \"Functional mapping and annotation of genetic associations with FUMA\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 89998, \"infons\": {\"fpage\": \"86\", \"lpage\": \"91\", \"name_0\": \"surname:Mishra;given-names:A\", \"name_1\": \"surname:Macgregor;given-names:S\", \"pub-id_doi\": \"10.1017/thg.2014.79\", \"pub-id_pmid\": \"25518859\", \"section_type\": \"REF\", \"source\": \"Twin Res. Hum. Genet.\", \"type\": \"ref\", \"volume\": \"18\", \"year\": \"2015\"}, \"text\": \"VEGAS2: software for more flexible gene-based testing\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 90052, \"infons\": {\"fpage\": \"229\", \"lpage\": \"237\", \"name_0\": \"surname:Turley;given-names:P\", \"pub-id_doi\": \"10.1038/s41588-017-0009-4\", \"pub-id_pmid\": \"29292387\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"50\", \"year\": \"2018\"}, \"text\": \"Multi-trait analysis of genome-wide association summary statistics using MTAG\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 90130, \"infons\": {\"fpage\": \"1\", \"issue\": \"3\", \"lpage\": \"48\", \"name_0\": \"surname:Viechtenbauer;given-names:W\", \"section_type\": \"REF\", \"source\": \"J. Stat. Softw.\", \"type\": \"ref\", \"volume\": \"36\", \"year\": \"2010\"}, \"text\": \"Conducting meta-analysis in R with the metafor package\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 90185, \"infons\": {\"fpage\": \"e1004383\", \"name_0\": \"surname:Giambartolomei;given-names:C\", \"pub-id_doi\": \"10.1371/journal.pgen.1004383\", \"pub-id_pmid\": \"24830394\", \"section_type\": \"REF\", \"source\": \"PLoS Genet.\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2014\"}, \"text\": \"Bayesian test for colocalisation between pairs of genetic association studies using summary statistics\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 90288, \"infons\": {\"fpage\": \"224\", \"name_0\": \"surname:Zhu;given-names:Z\", \"pub-id_doi\": \"10.1038/s41467-017-02317-2\", \"pub-id_pmid\": \"29335400\", \"section_type\": \"REF\", \"source\": \"Nat. Commun.\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2018\"}, \"text\": \"Causal associations between risk factors and common diseases inferred from GWAS summary data\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 90381, \"infons\": {\"fpage\": \"e34408\", \"name_0\": \"surname:Hemani;given-names:G\", \"pub-id_doi\": \"10.7554/eLife.34408\", \"pub-id_pmid\": \"29846171\", \"section_type\": \"REF\", \"source\": \"eLife\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2018\"}, \"text\": \"The MR-Base platform supports systematic causal inference across the human phenome\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 90464, \"infons\": {\"fpage\": \"1264\", \"lpage\": \"1278\", \"name_0\": \"surname:Bowden;given-names:J\", \"pub-id_doi\": \"10.1093/ije/dyy101\", \"pub-id_pmid\": \"29961852\", \"section_type\": \"REF\", \"source\": \"Int. J. Epidemiol.\", \"type\": \"ref\", \"volume\": \"47\", \"year\": \"2018\"}, \"text\": \"Improving the visualization, interpretation and analysis of two-sample summary data Mendelian randomization via the Radial plot and Radial regression\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 90614, \"infons\": {\"fpage\": \"e1007081\", \"name_0\": \"surname:Hemani;given-names:G\", \"name_1\": \"surname:Tilling;given-names:K\", \"name_2\": \"surname:Davey Smith;given-names:G\", \"pub-id_doi\": \"10.1371/journal.pgen.1007081\", \"pub-id_pmid\": \"29149188\", \"section_type\": \"REF\", \"source\": \"PLoS Genet.\", \"type\": \"ref\", \"volume\": \"13\", \"year\": \"2017\"}, \"text\": \"Orienting the causal relationship between imprecisely measured traits using GWAS summary data\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 90708, \"infons\": {\"fpage\": \"740\", \"lpage\": \"747\", \"name_0\": \"surname:Morrison;given-names:J\", \"name_1\": \"surname:Knoblauch;given-names:N\", \"name_2\": \"surname:Marcus;given-names:JH\", \"name_3\": \"surname:Stephens;given-names:M\", \"name_4\": \"surname:He;given-names:X\", \"pub-id_doi\": \"10.1038/s41588-020-0631-4\", \"pub-id_pmid\": \"32451458\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"52\", \"year\": \"2020\"}, \"text\": \"Mendelian randomization accounting for correlated and uncorrelated pleiotropic effects using genome-wide summary statistics\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 90832, \"infons\": {\"fpage\": \"1\", \"lpage\": \"9\", \"name_0\": \"surname:Mishra;given-names:A\", \"name_1\": \"surname:MacGregor;given-names:S\", \"pub-id_doi\": \"10.1017/thg.2016.100\", \"pub-id_pmid\": \"28105966\", \"section_type\": \"REF\", \"source\": \"Twin Res. Hum. Genet.\", \"type\": \"ref\", \"volume\": \"20\", \"year\": \"2017\"}, \"text\": \"A novel approach for pathway analysis of GWAS data highlights role of BMP signaling and muscle cell differentiation in colorectal cancer susceptibility\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 90984, \"infons\": {\"fpage\": \"D789\", \"lpage\": \"798\", \"name_0\": \"surname:Amberger;given-names:JS\", \"name_1\": \"surname:Bocchini;given-names:CA\", \"name_2\": \"surname:Schiettecatte;given-names:F\", \"name_3\": \"surname:Scott;given-names:AF\", \"name_4\": \"surname:Hamosh;given-names:A\", \"pub-id_doi\": \"10.1093/nar/gku1205\", \"pub-id_pmid\": \"25428349\", \"section_type\": \"REF\", \"source\": \"Nucleic Acids Res.\", \"type\": \"ref\", \"volume\": \"43\", \"year\": \"2015\"}, \"text\": \"OMIM.org: Online Mendelian Inheritance in Man (OMIM(R)), an online catalog of human genes and genetic disorders\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 91096, \"infons\": {\"fpage\": \"245\", \"lpage\": \"252\", \"name_0\": \"surname:Gusev;given-names:A\", \"pub-id_doi\": \"10.1038/ng.3506\", \"pub-id_pmid\": \"26854917\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"48\", \"year\": \"2016\"}, \"text\": \"Integrative approaches for large-scale transcriptome-wide association studies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 91174, \"infons\": {\"fpage\": \"3821\", \"lpage\": \"3823\", \"name_0\": \"surname:Sakaue;given-names:S\", \"name_1\": \"surname:Okada;given-names:Y\", \"pub-id_doi\": \"10.1093/bioinformatics/btz166\", \"pub-id_pmid\": \"30859178\", \"section_type\": \"REF\", \"source\": \"Bioinformatics\", \"type\": \"ref\", \"volume\": \"35\", \"year\": \"2019\"}, \"text\": \"GREP: Genome for REPositioning drugs\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 91211, \"infons\": {\"fpage\": \"294\", \"lpage\": \"304\", \"name_0\": \"surname:Konuma;given-names:T\", \"name_1\": \"surname:Ogawa;given-names:K\", \"name_2\": \"surname:Okada;given-names:Y\", \"pub-id_doi\": \"10.1093/hmg/ddab049\", \"pub-id_pmid\": \"33577681\", \"section_type\": \"REF\", \"source\": \"Hum. Mol. Genet.\", \"type\": \"ref\", \"volume\": \"30\", \"year\": \"2021\"}, \"text\": \"Integration of genetically regulated gene expression and pharmacological library provides therapeutic drug candidates\", \"sentences\": [], \"annotations\": [], \"relations\": []}], \"annotations\": [], \"relations\": []}]}]\n",
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading weights: 100%|| 770/770 [00:00<00:00, 1610.42it/s, Materializing param=model.encoder_input_proj.2.1.weight]                                                 \n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    }
   ],
   "source": [
    "pmcid = \"PMC10115645\"\n",
    "file_name = \"papers/41591_2023_Article_2268.pdf\"  # Can be a local path or a URL\n",
    "df_lst = extract_tables_lst_from_paper(pmcid, file_name)\n",
    "for i in [1, 2]:\n",
    "    df_lst[i-1].to_csv(f\"tables/{file_name.split('/')[-1].replace('.pdf', '')}_table_{i}.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "86c76427",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{\"bioctype\": \"BioCCollection\", \"source\": \"PMC\", \"date\": \"20250420\", \"key\": \"pmc.key\", \"version\": \"1.0\", \"infons\": {}, \"documents\": [{\"bioctype\": \"BioCDocument\", \"id\": \"9622429\", \"infons\": {\"license\": \"author_manuscript\"}, \"passages\": [{\"bioctype\": \"BioCPassage\", \"offset\": 0, \"infons\": {\"article-id_doi\": \"10.1002/alz.12686\", \"article-id_manuscript\": \"NIHMS1797266\", \"article-id_pmc\": \"9622429\", \"article-id_pmid\": \"35490390\", \"fpage\": \"611\", \"issue\": \"2\", \"kwd\": \"cognitive impairment age at onset genome wide association study Cox mixed-effects models SHISA6 gene Alzheimer\\u2019s disease protective variants\", \"license\": \"\\n          This file is available for text mining. It may also be used consistent with the principles of fair use under the copyright law.\\n        \", \"lpage\": \"620\", \"name_0\": \"surname:Ramos;given-names:Jairo\", \"name_1\": \"surname:Caywood;given-names:Laura J.\", \"name_10\": \"surname:Main;given-names:Leighanne R.\", \"name_11\": \"surname:Osterman;given-names:Michael D.\", \"name_12\": \"surname:Zaman;given-names:Andrew F.\", \"name_13\": \"surname:Whitehead;given-names:Patrice L.\", \"name_14\": \"surname:Adams;given-names:Larry D.\", \"name_15\": \"surname:Laux;given-names:Renee A.\", \"name_16\": \"surname:Song;given-names:Yeunjoo E.\", \"name_17\": \"surname:Foroud;given-names:Tatiana M.\", \"name_18\": \"surname:Mayeux;given-names:Richard P.\", \"name_19\": \"surname:George-Hyslop;given-names:Peter St.\", \"name_2\": \"surname:Prough;given-names:Michael B.\", \"name_20\": \"surname:Ogrocki;given-names:Paula K.\", \"name_21\": \"surname:Lerner;given-names:Alan J.\", \"name_22\": \"surname:Vance;given-names:Jeffery M.\", \"name_23\": \"surname:Cuccaro;given-names:Michael L.\", \"name_24\": \"surname:Haines;given-names:Jonathan L.\", \"name_25\": \"surname:Pericak-Vance;given-names:Margaret A.\", \"name_26\": \"surname:Scott;given-names:William K.\", \"name_3\": \"surname:Clouse;given-names:Jason E.\", \"name_4\": \"surname:Herington;given-names:Sharlene D.\", \"name_5\": \"surname:Slifer;given-names:Susan H.\", \"name_6\": \"surname:Fuzzell;given-names:M. Denise\", \"name_7\": \"surname:Fuzzell;given-names:Sarada L.\", \"name_8\": \"surname:Hochstetler;given-names:Sherri D.\", \"name_9\": \"surname:Miskimen;given-names:Kristy L.\", \"section_type\": \"TITLE\", \"type\": \"front\", \"volume\": \"19\", \"year\": \"2024\"}, \"text\": \"Genetic variants in the SHISA6 gene are associated with delayed cognitive impairment in two family datasets\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 108, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract_title_1\"}, \"text\": \"INTRODUCTION:\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 122, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Studies of cognitive impairment (CI) in Amish communities have identified sibships containing CI and cognitively unimpaired (CU) individuals. We hypothesize that CU individuals may carry protective alleles delaying age at onset (AAO) of CI.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 363, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract_title_1\"}, \"text\": \"METHODS:\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 372, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"1,522 individuals screened for CI were genotyped. The outcome studied was AAO for CI individuals or age at last normal exam for CU individuals. Cox mixed-effects models examined association between age and single nucleotide variants (SNV).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 612, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract_title_1\"}, \"text\": \"RESULTS:\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 621, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Three SNVs were significantly associated (p<5 \\u00d7 10\\u22128) with AAO on chromosomes 6 (rs14538074; Hazard Ratio (HR)=3.35), 9 (rs534551495; HR=2.82), and 17 (rs146729640; HR=6.38). The chromosome 17 association was replicated in the independent NIA-LOAD dataset.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 881, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract_title_1\"}, \"text\": \"DISCUSSION:\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 893, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"The replicated genome-wide significant association with AAO on chromosome 17 is located in the SHISA6 gene, which is involved in post-synaptic transmission in the hippocampus and is a biologically plausible candidate gene for Alzheimer\\u2019s disease (AD).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 1147, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"title_1\"}, \"text\": \"BACKGROUND\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 1158, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Dementia, which includes Alzheimer disease (AD), cerebrovascular, Lewy body, and mixed dementias, has a tremendous effect on the entire family, since the largest proportion, at least 60 % in the U.S., of caregivers are spouses, children and children-in-law. Dementia also carries a heavy emotional and financial toll, as people can live for many years cognitively impaired, requiring high levels of care with annual estimated costs in the U.S. in 2010 between $159 billion and $215 billion. Understanding the factors that influence the age at which cognitive impairment begins is foundational in developing approaches toward reducing the prevalence and progression of disease. Much effort has been made to identify genetic factors that increase the risk and result in earlier onset of dementia, and dozens of risk variants have been identified. The search for protective variants, including those that delay age at onset (AAO), has been less common. Studies of time to onset of dementia symptoms, employing survival analysis approaches, offer a potentially powerful means of identifying protective variants in age related neurodegenerative diseases that shorten the duration of cognitive impairment (CI) by delaying onset of symptoms. In complex diseases such as AD and other neurodegenerative disorders that cause dementia, the identification of both risk and protective variants is essential to elucidate the molecular mechanisms involved in disease development and progression. This is a crucial step in the discovery of new therapeutic targets. In this study, we aim to identify protective variants associated with delay onset of dementia using survival analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 2826, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"The complex genetic architecture of dementia and AD has complicated the search for both risk and protective variants. Reproducibility of genetic association is influenced by locus heterogeneity, smaller sample sizes, and population differences. Some of these limitations are addressed by large-scale consortium approaches to genome-wide association studies (GWAS). A second strategy to improve power and reduce heterogeneity is to study a genetically homogenous population such as the Amish. The Amish in the United States live in relatively genetically isolated communities; members of these communities have participated in longitudinal studies of aging, cognition, and cognitive impairment for over two decades which have identified sibships with both individuals with CI and siblings that are cognitively unimpaired (CU). Our hypothesis is that individuals that are CU carry genetic variants that delay AAO of CI despite having an increased risk due to advanced age (over age 75) and family history of CI (including 18.1% with siblings having CI).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 3878, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"In this study we performed a GWAS of AAO of CI in 1,522 people from the Amish communities living in Ohio and Indiana. Genome-wide significant results were examined for replication in an independent sample of 3,350 individuals from the NIA-LOAD family study of AD. A replicated association on chromosome 17 was identified, supporting the hypothesis that genetic variants delay the development of cognitive impairment in individuals who are CU despite elevated risk due to age and family history of dementia.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 4385, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_1\"}, \"text\": \"METHODS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 4393, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Study Populations and Clinical Assessment\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 4435, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"A description of the sample of 1,522 Amish used in this study is provided in Table 1. All participants were of European descent, with an average age of 73 at the last examination (range: 43 to 99). Women were 58.3% of the sample. Individuals were residents of Amish communities in Ohio (Holmes County) and Indiana (Adams, Elkhart, and LaGrange counties) and recruited as previously described by Pericak-Vance et al., Hahs et al., Edwards et al., Edwards et al., and Ramos et al.. This is part of a population-based study of aging and age-related phenotypes such as AD performed by researchers from the University of Miami and Case Western Reserve University after Institutional Review Board (IRB) review and approval.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 5153, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The cohort selected for this research come from three main studies. For the first study, we enrolled individuals from 1994 to 2017 in family-based studies of dementia and AD. Participants over 65 were initially identified by a population-based survey in 1991\\u20131993 and then by expansion of families from these index cases. Time-to-CI was established retrospectively through informant interview, or when that was not reliable, at the examination date where the individual was identified as impaired. Family members who were cognitively unimpaired (CU) were censored at the most recent examination date. Most of these individuals were examined only once. For the second study, we conducted a community-based study of cognitive and physical function in individuals over age 80 from 2003 to 2010. For people with CI, time-to-CI was established retrospectively through informant interview, or if that could not be reliably established, at the age at which the examination showed cognitive impairment. For CU, they were censored at the most recent examination date. Some CU individuals were examined a second time 2\\u20133 years later. For the third study, from 2017 to date, we contacted all siblings of people with CI identified in the two studies above and examined those 76 and older. We also contacted individuals in the community in the same age group who were not siblings. Some of these individuals were participants in the first two studies, and thus this visit was a re-examination 3\\u20135 years from the last visit. For people with CI, time-to-CI was established retrospectively through informant interview, or if that could not be reliably established, at the age at which the examination showed cognitive impairment. For CU, they were censored at the most recent examination date.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 6937, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"All Amish participants were evaluated for cognitive function at least once using the Modified Mini-Mental State exam (3MS). DNA was extracted from a blood sample obtained at this baseline examination, which also included an assessment of family history of dementia or AD, a medical history of common chronic diseases, and basic sociodemographic information. The average number of clinical examinations was 1.4 (range: 1 to 5).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 7364, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Individuals with education-adjusted 3MS scores below 87 at baseline or at a follow-up visit were classified as cognitively impaired (CI). Those scoring 87 and above at all visits were considered cognitively unimpaired (CU). AAO of CI was determined as the age at the examination when the participant was observed impaired for the first time. For individuals with CI at baseline, this may overestimate AAO of CI. For survival analysis, the outcome was defined as AAO of CI for those with cognitive impairment, and age at last examination for those classified as CU. The outcome of interest here was CI-free time, rather than the development of dementia, and therefore we chose a broad definition of CI (including AD and other types of dementia) as the event, rather than narrower definitions of AD. CI and CU classification was based on 3MS score as mentioned above. This allows the identification of loci protective against a wider range of dementia phenotypes, at the possible expense of greater heterogeneity of effects and lower power if the loci are specific to AD. We therefore attempted to replicate our findings in a second family dataset with individuals meeting this more specific AD case definition and their unaffected relatives.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 8605, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"For external replication we used the family-based National Institute on Aging Genetics Initiative for Late-Onset Alzheimer\\u2019s Disease (NIA-LOAD) dataset composed of 3,350 subjects (1,785 late onset AD (LOAD) cases and 1,565 controls neurologically evaluated as unaffected). Subjects were evaluated using standard research criteria for the diagnosis of LOAD as established in McKhann et al., including the Clinical Dementia Rating score. Cases were defined as those individuals with definite or probable LOAD diagnosis as per guidelines in McKhann et al., while AAO was established as the age at which the family first observed memory problems or the age at first exam where cognitive impairment was noted. For controls the age used was the age at last examination when the absence of dementia was determined. The demographics of the sample are provided in Table 2. More details about the sample are described in Lee et al. and Kunkle et al..\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 9548, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Written informed consent was obtained from study participants or, for those with substantial cognitive impairment, from a caregiver, legal guardian, or other proxy, and the study protocols for all populations were reviewed and approved by the appropriate Institutional review boards (IRB\\u2019s).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 9842, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Genotyping\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 9853, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"DNA extracted from peripheral blood samples was used for genotyping on the Illumina Infinium Global Screening (GSA) or Expanded Multi-Ethnic Genotyping (MEGAex) Array. After pre-imputation QC, which included removing markers missing more than 2% of the data, monomorphic markers, markers that had MAF < 0.05 and markers that failed Hardy Weinberg exact test at P value < 10\\u22126, the MEGAex final set before imputation was 552,801 markers and the GSA was 279,931. The two datasets were imputed separately using the Haplotype Reference Consortium (HRC) reference panel. Post-Imputation QC parameters were MAF >= 0.01, r2 >=0.4; MAF < 0.01, r2 >=0.8. Subsequently, the overlapping markers that passed QC from both sets were merged for a total of over 7 million common and rare imputed variants. Hardy-Weinberg Equilibrium (HWE) was tested for all SNPs and variants significantly out of HWE (P value < 10\\u22126) were removed before the analysis. Supplementary Table 1 summarizes the imputed SNPs retained for analysis by chromosome.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 10880, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The NIA-LOAD dataset was genotyped using the Illumina 610 array containing 592,532 SNPs. For genotyping details see Wijsman et al.. After genotype chip QC, the dataset was imputed to the 1,000 Genomes Project using IMPUTE2. The total number of imputed SNPs before QC was 39,127,740. For imputation details see Kunkle et al.. We concentrated the replication analysis on chromosomes 6, 9 and 17 where we had found the three genome-wide significant results in the Amish cohort. Supplementary Table 2 shows the number of imputed SNPs analyzed on each chromosome.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 11439, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Statistical Analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 11460, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The Cox proportional hazards model is widely used in the analysis of time-to-event phenotypes (such as AAO) and mixed-effects models are commonly used in family-based genetic studies to adjust for relatedness of individuals. We used Coxmeg an R package for conducting GWAS of age-at-onset traits in related individuals using Cox mixed-effects models (CMEMs). The CU survival time outcome was defined as AAO of CI (cases) or age at last normal cognitive examination (CU (censored) controls) in the Amish and NIA-LOAD (replication) cohorts. Familial relationships were controlled by including a random effect specified by the genetic relationship matrix (GRM) estimated from the genotype data in both Amish and NIA-LOAD datasets. In both datasets sex was included as a covariate. Individual SNPs, coded additively as imputed dosages of the less common allele, were added to this base model to test for association with CU survival time. The strength of this association is estimated by the hazard ratio (HR). The qqman package in R was used to summarize the significance of the association of each SNP with CU survival time in a Manhattan plot of p-values from the chi-square test of association. In the Amish dataset, genome-wide significant results were determined using the traditional GWAS threshold P < 5 \\u00d7 10\\u22128, and suggestive results were highlighted using the threshold P < 1 \\u00d7 10\\u22125. LocusZoom was used to generate regional association plots and Linkage Disequilibrium analysis (LD), based on 1000 Genomes, of all common variants located in a 1Mb interval centered on SNPs with genome-wide significant associations. Three genome-wide significant results were identified, and markers in a 1-Megabase (Mb) region centered on each peak SNP were selected for analysis in the NIA-LOAD replication dataset, using identical statistical models to the Amish discovery dataset. To assess potential interaction of genome-wide significant effects with APOE genotype, stratified analysis was conducted in Amish and NIA-LOAD samples subdivided into APOE-4 carriers and non-carriers. Meta-analysis of hazard ratios from the stratified analysis was also performed using the R package meta.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 13646, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_1\"}, \"text\": \"RESULTS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 13654, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Demographic statistics and genotype quality control for Amish dataset\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 13724, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"The demographic characteristics of the Amish dataset are shown in Table 1. A total of 362 subjects were cognitively impaired (CI). Of these, 58% were women. The remaining 1,160 were cognitively unimpaired (CU). Of these, 58.4% were women. Age at last examination was 81.6 for the CI group and 70.6 for the CU group. The mean age at first exam below 3MS cutoff point for the CI group was 81.2 years.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 14123, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"The total number of imputed variants after QC, including rare variants, was 7,819,581 (Supplementary Table 1). Tests of HWE identified 3,630 variants significantly deviating from HWE (p < 10\\u22126). After removing these variants, 7,815,951 were retained for GWAS analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 14394, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"GWAS of CU survival time\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 14419, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"The GWAS of CU survival time used a Cox mixed-effects model with each imputed variant and sex as a fixed-effects covariate, with the kinship matrix included as a random effect. The results for all autosomes are summarized in the Manhattan plot in Figure 1. Three common variants (minor allele frequency (MAF) > 0.01) were significantly associated (p < 5 \\u00d7 10\\u22128) with CI-free survival time on chromosomes 6 (rs145348074), 9 (rs534551495), and 17 (rs146729640). Table 3 lists for each associated region the p value, hazard ratio, minimum allele frequency (MAF), and overlapping or closest gene and genes located within +/\\u2212 500 thousand base pairs. The QQ plot of GWAS p-values of CU survival time, as well as the genomic inflation factor (lambda) statistic (\\u03bb=1.01) suggest that the significance of these GWAS results is not inflated (Figure 2). To determine if there were broad regions of association surrounding these peak markers, we examined regional association plots (Supplemental Figures 1-3), identified variants with suggestive evidence of association (p < 10\\u22125, Supplemental Table 3), and examined linkage disequilibrium patterns in the region to assess whether there was evidence for multiple association signals.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 15650, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Regional plot and local linkage disequilibrium (LD) analysis for each genome wide variant\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 15740, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"As summarized in Table 4, regional association analysis of 1-Megabase (Mb) regions on chromosomes 6 and 9 identified three additional variants in moderate to strong LD (r2>0.4) with the peak markers and suggestive evidence of association with AAO (p < 1 \\u00d7 10\\u22125). No additional markers were identified in the chromosome 17 region. Regional association plots are provided in Supplemental Figures 1, 2 and 3.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 16149, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Replication analysis of the three genome-wide significant results in the independent NIA-LOAD dataset.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 16252, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"The demographic characteristics of the NIA-LOAD replication dataset are shown in Table 2. There are 1,785 people with AD (65% women) and 1,565 CU controls (60% women). AAO was 73.1 for cases and the age at exam for the control group was 73.9. The replication analyses were conducted in the 1Mb interval centered on each peak variant from the Amish GWAS. For chromosome 6, variants numbered 14,532; for chromosome 9, variants were 13,602; for chromosome 17 variants were 17,000 (Supplementary Table 2).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 16754, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Analysis of CU survival time in the NIA-LOAD dataset detected nominal evidence of association in the chromosome 17 interval, at the same peak marker as the Amish GWAS (rs146729640; p=0.02, Table 5). Results for the other two regions were not significant. While the result on chromosome 17 did not exceed the Bonferroni-adjusted significance threshold (p=0.05/17,000 variants tested = 3 \\u00d7 10\\u22126), the replication of the result at the exact same variant with nominal significance suggests that the observed association is reproducible.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 17290, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"APOE-4 dose was strongly associated with CU survival time in the Amish (HR=1.61, 95% CI (1.28 \\u2013 2.02); p=5.08\\u00d710\\u20135). Because of this strong main effect, we tested for interaction between APOE-4 dose and the top SNP in the GWAS. Significant interactions (p<0.05) were evaluated by stratifying the dataset by APOE-4 carrier status to determine if these SNPs had different effects by APOE-4 genotype.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 17693, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Stratified Analysis of Chromosome 17 regions by APOE-4 carrier status\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 17763, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"To explore the possibility that APOE genotype might influences these results, the discovery and replication datasets were stratified by APOE-4 carrier status and the Cox mixed-effects models were run in each subset. In both the Amish (Table 6) and NIA-LOAD (Table 7) datasets, the effect (hazard ratio) at rs146729640 was stronger in the subset of individuals that did not carry the APOE-4 allele. This was also confirmed in the meta-analysis combining the results of the non APOE-4 carriers\\u2019 groups in the Amish and NIA-LOAD (Table 8; HR= 3.56, 95 %CI= 2.19\\u20135.78).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 18333, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_1\"}, \"text\": \"DISCUSSION\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 18344, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"This study identified a reproducible association of CU survival time with an intronic polymorphism of the SHISA6 gene on chromosome 17, where the more frequent allele \\u201cG\\u201d of rs146729640 is associated with later AAO of CI and the minor allele \\u201cA\\u201d was strongly associated with earlier AAO of CI (p=5.6 \\u00d7 10\\u22129) acting as a risk factor with a strong hazard ratio (HR= 6.38, 95 %CI= 3.42\\u201311.90). The \\u201cA\\u201d allele frequency in our cohort was 0.012, twice the frequency of 0.006 reported by The Allele Frequency Aggregator (ALFA) database (https://www.ncbi.nlm.nih.gov/snp/docs/gsr/alfa/) for European population.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 18966, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Large-scale genome-wide association studies (GWAS) of disease-free survival offer a powerful method of identifying factors that delay the onset of disease, manifesting in later AAO. Such discoveries might illuminate mechanisms underlying disease progression of age-related diseases such as CI and AD. In the present study we performed a GWAS of 7.8 M imputed variants, examining association with CU survival time in 1,520 Amish individuals longitudinally evaluated for CI. Three genome-wide significant results were examined for replication in the independent NIA-LOAD family sample of 1785 individuals with AD and 1565 CU controls, and the association at rs146729640 (chromosome 17) was replicated at a nominal significance level in the NIA-LOAD dataset. We note that the outcome of interest in both datasets, time free of cognitive impairment, was the same, despite differences in the definition used to identify CI. The association at rs146729640 in both datasets despite these differences is encouraging. This intronic polymorphism is located within the SHISA6 gene, an excellent candidate gene for AD.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 20073, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"SHISA6 has previously been implicated in AD pathogenesis in human proteomics and transgenic mouse model studies. The SHISA6 is among the 60 proteins, out of 4,582 analyzed, found significantly altered by increasing age in the human hippocampus. SHISA6 was found downregulated in this study, which was carried out using hippocampal samples from 16 Chinese brain tissue donors free of neurological diseases across four age groups using the youngest one (22\\u201349) as reference: 22\\u201349, 50\\u201369, 70\\u201389, and >90. Additionally, several experiments in vivo using APOE transgenic mice have revealed that the Shisa6 gene is differentially expressed (downregulated log2 fold change = - 0.53) in the entorhinal cortex (EC) of aged APOE mice (APOE3/4 vs. APOE3/3). Based on these findings, we performed a stratified analysis and found that the strength of association at rs145348074 (SHISA6 gene) was increased in the non-APOE4 carriers in both the Amish and the NIA-LOAD replication data sets. Notably, the EC is one of the first regions to be affected by AD pathology in humans. Shisa6 is also involved in maintenance of high-frequency synaptic transmission between neurons in the mouse hippocampus by regulating AMPA-type glutamate receptors (AMPAR).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 21318, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"We note that the family-based design and use of cognitive-impairment free survival time as the outcome introduces a few potential limitations to the study and interpretation of results. One potential limitation in our study is that for people in the Amish cohort who were CI at baseline (prevalent cases), the AAO of CI we used (age at exam) was probably later than the true AAO, as symptoms may have started earlier and been unrecognized or unreported prior to enrollment in our study. This could have biased our results towards the null; therefore, the true association may be stronger than the current estimate. A second potential limitation is that CI is a multifactorial condition driven by a mixture of genetic, vascular, metabolic, and lifestyle-related factors. Heterogeneity in types of CI (MCI, AD, other dementia) could lead to spurious associations (in direction or strength of association) due to Simpson\\u2019s paradox. However, based on the estimation that the most well-known cause of MCI (Mild Cognitive Impairment)- around 50 % in people age 65 and older- is AD, and since the results were replicated in the NIA-LOAD data set, where cases had been clinically diagnosed with LOAD, we are reassured that the detected association at SNP rs146729640 is not likely to be due to such bias.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 22617, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"The identification of variants that modify the AOO in AD will reveal potential pathways for intervention in the progression of the symptoms of the disease. Only a few studies have been conducted to identify genetic variants associated with AAO in AD. Lalli et al. identified the variant rs9909184 located near the CCL11 gene on chromosome 17 associated with delayed onset of mild-cognitive impairment and dementia using a cohort of 72 subjects affected with early-onset familial AD. Our study identified the variant rs146729640, that may be associated with AAO of LOAD, located on the same chromosome although in a different region. The discovery of AAO modifying factors is essential to develop targets for early interventions aimed to prevent the onset of the disease.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 23388, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"GWAS studies are susceptible to false positives results due to the large number of statistical tests. Therefore, an internally replicated design that focuses on signals that are independently detected in a second dataset provides stronger evidence of association. While our GWAS was conducted in a family-based study of 1,520 Amish, we tested our three genome-wide significant findings in a second family dataset (NIA-LOAD) with twice the sample size. The replicated result in the SHISA6 gene is bolstered by its biological plausibility; this gene has been implicated in transgenic mouse models of AD and human proteomic studies of the aging hippocampus. The effect in the NIA-LOAD is somewhat smaller, but still statistically significant. These results illustrate the utility of using isolated populations to facilitate replicable gene discoveries in complex traits. These specific results suggest that SHISA6 might modulate the rate of cognitive decline, and studies to elucidate the specific effect and mechanism underlying this reproducible association are warranted.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 24460, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_1\"}, \"text\": \"Supplementary Material\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 24483, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Conflict of Interest\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 24504, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Jairo Ramos, Laura J. Caywood, Michael B. Prough, Jason E. Clouse, Sharlene D. Herington, Susan H. Slifer, M. Denise Fuzzell, Sarada L. Fuzzell, Sherri D. Hochstetler, Kristy L. Miskimen, Leighanne R. Main, Michael D. Osterman, Andrew F. Zaman, Patrice L. Whitehead, Larry D. Adams, Renee A. Laux, Yeunjoo E. Song and Paula K. Ogrocki, have no conflict of interest to report.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 24880, \"infons\": {\"section_type\": \"REF\", \"type\": \"title\"}, \"text\": \"REFERENCES\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 24891, \"infons\": {\"fpage\": \"100577\", \"name_0\": \"surname:Hale;given-names:JM\", \"name_1\": \"surname:Schneider;given-names:DC\", \"name_2\": \"surname:Mehta;given-names:NK\", \"name_3\": \"surname:Myrskyla;given-names:M\", \"pub-id_pmid\": \"32300635\", \"section_type\": \"REF\", \"source\": \"SSM - Population Health\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2020\"}, \"text\": \"Cognitive impairment in the U.S.: Lifetime risk, age at onset, and years impaired.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 24974, \"infons\": {\"fpage\": \"217\", \"issue\": \"2\", \"lpage\": \"228\", \"name_0\": \"surname:Brodaty;given-names:H\", \"name_1\": \"surname:Donkin;given-names:M\", \"pub-id_pmid\": \"19585957\", \"section_type\": \"REF\", \"source\": \"Dialogues in Clinical Neuroscience\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2009\"}, \"text\": \"Family caregivers of people with dementia.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25017, \"infons\": {\"fpage\": \"1326\", \"issue\": \"14\", \"lpage\": \"1334\", \"name_0\": \"surname:Hurd;given-names:MD\", \"name_1\": \"surname:Martorell;given-names:P\", \"name_2\": \"surname:Delavande;given-names:A\", \"name_3\": \"surname:Mullen;given-names:KJ\", \"name_4\": \"surname:Langa;given-names:KM\", \"pub-id_pmid\": \"23550670\", \"section_type\": \"REF\", \"source\": \"The New England Journal of Medicine\", \"type\": \"ref\", \"volume\": \"368\", \"year\": \"2013\"}, \"text\": \"Monetary costs of dementia in the United States.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25066, \"infons\": {\"fpage\": \"558\", \"issue\": \"7\", \"lpage\": \"570\", \"name_0\": \"surname:Floris;given-names:M\", \"name_1\": \"surname:Olla;given-names:S\", \"name_2\": \"surname:Schlessinger;given-names:D\", \"name_3\": \"surname:Cucca;given-names:F\", \"pub-id_pmid\": \"29803319\", \"section_type\": \"REF\", \"source\": \"Trends in Genetics : TIG\", \"type\": \"ref\", \"volume\": \"34\", \"year\": \"2018\"}, \"text\": \"Genetic-driven druggable target identification and validation.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25129, \"infons\": {\"fpage\": \"1423\", \"issue\": \"9\", \"lpage\": \"1424\", \"name_0\": \"surname:Kunkle;given-names:BW\", \"name_1\": \"surname:Grenier-Boley;given-names:B\", \"name_2\": \"surname:Sims;given-names:R\", \"name_3\": \"surname:Bis;given-names:JC\", \"name_4\": \"surname:Damotte;given-names:V\", \"name_5\": \"surname:Naj;given-names:AC\", \"pub-id_pmid\": \"31417202\", \"section_type\": \"REF\", \"source\": \"Nature Genetics\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2019\"}, \"text\": \"Author correction: Genetic meta-analysis of diagnosed alzheimer\\u2019s disease identifies new risk loci and implicates A\\u03b2, tau, immunity and lipid processing.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25286, \"infons\": {\"fpage\": \"160\", \"issue\": \"2\", \"lpage\": \"166\", \"name_0\": \"surname:Hahs;given-names:DW\", \"name_1\": \"surname:McCauley;given-names:JL\", \"name_2\": \"surname:Crunk;given-names:AE\", \"name_3\": \"surname:McFarland;given-names:LL\", \"name_4\": \"surname:Gaskell;given-names:PC\", \"name_5\": \"surname:Jiang;given-names:L\", \"pub-id_pmid\": \"16389594\", \"section_type\": \"REF\", \"source\": \"American Journal of Medical Genetics.Part B, Neuropsychiatric Genetics: The Official Publication of the International Society of Psychiatric Genetics\", \"type\": \"ref\", \"volume\": \"141B\", \"year\": \"2006\"}, \"text\": \"A genome-wide linkage analysis of dementia in the amish.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25343, \"infons\": {\"fpage\": \"700\", \"issue\": \"6\", \"lpage\": \"704\", \"name_0\": \"surname:Pericak-Vance;given-names:MA\", \"name_1\": \"surname:Johnson;given-names:CC\", \"name_2\": \"surname:Rimmler;given-names:JB\", \"name_3\": \"surname:Saunders;given-names:AM\", \"name_4\": \"surname:Robinson;given-names:LC\", \"name_5\": \"surname:D\\u2019Hondt;given-names:EG\", \"pub-id_pmid\": \"8651641\", \"section_type\": \"REF\", \"source\": \"Annals of Neurology\", \"type\": \"ref\", \"volume\": \"39\", \"year\": \"1996\"}, \"text\": \"Alzheimer\\u2019s disease and apolipoprotein E-4 allele in an amish population.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25419, \"infons\": {\"fpage\": \"516\", \"issue\": \"4\", \"lpage\": \"528\", \"name_0\": \"surname:Edwards;given-names:DR\", \"name_1\": \"surname:Gilbert;given-names:JR\", \"name_2\": \"surname:Jiang;given-names:L\", \"name_3\": \"surname:Gallins;given-names:PJ\", \"name_4\": \"surname:Caywood;given-names:L\", \"name_5\": \"surname:Creason;given-names:M\", \"pub-id_pmid\": \"21668908\", \"section_type\": \"REF\", \"source\": \"Annals of Human Genetics\", \"type\": \"ref\", \"volume\": \"75\", \"year\": \"2011\"}, \"text\": \"Successful aging shows linkage to chromosomes 6, 7, and 14 in the amish.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25492, \"infons\": {\"fpage\": \"342\", \"issue\": \"5\", \"lpage\": \"351\", \"name_0\": \"surname:Cummings;given-names:AC\", \"name_1\": \"surname:Jiang;given-names:L\", \"name_2\": \"surname:Velez Edwards;given-names:DR\", \"name_3\": \"surname:McCauley;given-names:JL\", \"name_4\": \"surname:Laux;given-names:R\", \"name_5\": \"surname:McFarland;given-names:LL\", \"pub-id_pmid\": \"22881374\", \"section_type\": \"REF\", \"source\": \"Annals of Human Genetics\", \"type\": \"ref\", \"volume\": \"76\", \"year\": \"2012\"}, \"text\": \"Genome-wide association and linkage study in the amish detects a novel candidate late-onset alzheimer disease gene.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25608, \"infons\": {\"fpage\": \"1467\", \"issue\": \"4\", \"lpage\": \"1477\", \"name_0\": \"surname:Edwards;given-names:DR\", \"name_1\": \"surname:Gilbert;given-names:JR\", \"name_2\": \"surname:Hicks;given-names:JE\", \"name_3\": \"surname:Myers;given-names:JL\", \"name_4\": \"surname:Jiang;given-names:L\", \"name_5\": \"surname:Cummings;given-names:AC\", \"pub-id_pmid\": \"22773346\", \"section_type\": \"REF\", \"source\": \"Age (Dordrecht, Netherlands)\", \"type\": \"ref\", \"volume\": \"35\", \"year\": \"2013\"}, \"text\": \"Linkage and association of successful aging to the 6q25 region in large amish kindreds.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25696, \"infons\": {\"fpage\": \"e0118043\", \"issue\": \"2\", \"name_0\": \"surname:D\\u2019Aoust;given-names:LN\", \"name_1\": \"surname:Cummings;given-names:AC\", \"name_2\": \"surname:Laux;given-names:R\", \"name_3\": \"surname:Fuzzell;given-names:D\", \"name_4\": \"surname:Caywood;given-names:L\", \"name_5\": \"surname:Reinhart-Mercer;given-names:L\", \"pub-id_pmid\": \"25668194\", \"section_type\": \"REF\", \"source\": \"PloS One\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2015\"}, \"text\": \"Examination of candidate exonic variants for association to alzheimer disease in the amish.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25788, \"infons\": {\"fpage\": \"451\", \"issue\": \"1\", \"lpage\": \"458\", \"name_0\": \"surname:Ramos;given-names:J\", \"name_1\": \"surname:Chowdhury;given-names:AR\", \"name_2\": \"surname:Caywood;given-names:LJ\", \"name_3\": \"surname:Prough;given-names:M\", \"name_4\": \"surname:Denise Fuzzell;given-names:M\", \"name_5\": \"surname:Fuzzell;given-names:S\", \"section_type\": \"REF\", \"source\": \"Journal of Alzheimer\\u2019s Disease: JAD\", \"type\": \"ref\", \"volume\": \"79\", \"year\": \"2021\"}, \"text\": \"Lower levels of education are associated with cognitive impairment in the old order amish.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25879, \"infons\": {\"fpage\": \"387\", \"issue\": \"2\", \"lpage\": \"394\", \"name_0\": \"surname:Johnson;given-names:CC\", \"name_1\": \"surname:Rybicki;given-names:BA\", \"name_2\": \"surname:Brown;given-names:G\", \"name_3\": \"surname:D\\u2019Hondt;given-names:E\", \"name_4\": \"surname:Herpolsheimer;given-names:B\", \"name_5\": \"surname:Roth;given-names:D\", \"pub-id_pmid\": \"9169175\", \"section_type\": \"REF\", \"source\": \"International Journal of Epidemiology\", \"type\": \"ref\", \"volume\": \"26\", \"year\": \"1997\"}, \"text\": \"Cognitive impairment in the Amish: A four county survey.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25936, \"infons\": {\"fpage\": \"314\", \"issue\": \"8\", \"lpage\": \"318\", \"name_0\": \"surname:Teng;given-names:EL\", \"name_1\": \"surname:Chui;given-names:HC\", \"pub-id_pmid\": \"3611032\", \"section_type\": \"REF\", \"source\": \"The Journal of Clinical Psychiatry\", \"type\": \"ref\", \"volume\": \"48\", \"year\": \"1987\"}, \"text\": \"The modified mini-mental state (3MS) examination.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 25986, \"infons\": {\"fpage\": \"e1001308\", \"issue\": \"2\", \"name_0\": \"surname:Wijsman;given-names:EM\", \"name_1\": \"surname:Pankratz;given-names:ND\", \"name_2\": \"surname:Choi;given-names:Y\", \"name_3\": \"surname:Rothstein;given-names:JH\", \"name_4\": \"surname:Faber;given-names:KM\", \"name_5\": \"surname:Cheng;given-names:R\", \"pub-id_pmid\": \"21379329\", \"section_type\": \"REF\", \"source\": \"PLoS Genetics\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2011\"}, \"text\": \"Genome-wide association of familial late-onset alzheimer\\u2019s disease replicates BIN1 and CLU and nominates CUGBP2 in interaction with APOE.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 26126, \"infons\": {\"fpage\": \"1180\", \"issue\": \"12\", \"lpage\": \"1186\", \"name_0\": \"surname:Tosto;given-names:G\", \"name_1\": \"surname:Bird;given-names:TD\", \"name_2\": \"surname:Tsuang;given-names:D\", \"name_3\": \"surname:Bennett;given-names:DA\", \"name_4\": \"surname:Boeve;given-names:BF\", \"name_5\": \"surname:Cruchaga;given-names:C\", \"pub-id_pmid\": \"28213371\", \"section_type\": \"REF\", \"source\": \"Neurology\", \"type\": \"ref\", \"volume\": \"88\", \"year\": \"2017\"}, \"text\": \"Polygenic risk scores in familial alzheimer disease.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 26179, \"infons\": {\"fpage\": \"939\", \"issue\": \"7\", \"lpage\": \"944\", \"name_0\": \"surname:McKhann;given-names:G\", \"name_1\": \"surname:Drachman;given-names:D\", \"name_2\": \"surname:Folstein;given-names:M\", \"name_3\": \"surname:Katzman;given-names:R\", \"name_4\": \"surname:Price;given-names:D\", \"name_5\": \"surname:Stadlan;given-names:EM\", \"pub-id_pmid\": \"6610841\", \"section_type\": \"REF\", \"source\": \"Neurology\", \"type\": \"ref\", \"volume\": \"34\", \"year\": \"1984\"}, \"text\": \"Clinical diagnosis of alzheimer\\u2019s disease: Report of the NINCDS-ADRDA work group under the auspices of department of health and human services task force on alzheimer\\u2019s disease.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 26361, \"infons\": {\"fpage\": \"566\", \"lpage\": \"572\", \"name_0\": \"surname:Hughes;given-names:CP\", \"name_1\": \"surname:Berg;given-names:L\", \"name_2\": \"surname:Danziger;given-names:WL\", \"name_3\": \"surname:Coben;given-names:LA\", \"name_4\": \"surname:Martin;given-names:RL\", \"pub-id_pmid\": \"7104545\", \"section_type\": \"REF\", \"source\": \"The British Journal of Psychiatry: The Journal of Mental Science\", \"type\": \"ref\", \"volume\": \"140\", \"year\": \"1982\"}, \"text\": \"A new clinical scale for the staging of dementia.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 26411, \"infons\": {\"fpage\": \"1518\", \"issue\": \"11\", \"lpage\": \"1526\", \"name_0\": \"surname:Lee;given-names:JH\", \"name_1\": \"surname:Cheng;given-names:R\", \"name_2\": \"surname:Graff-Radford;given-names:N\", \"name_3\": \"surname:Foroud;given-names:T\", \"name_4\": \"surname:Mayeux;given-names:R\", \"pub-id_pmid\": \"19001172\", \"section_type\": \"REF\", \"source\": \"Archives of Neurology\", \"type\": \"ref\", \"volume\": \"65\", \"year\": \"2008\"}, \"text\": \"Analyses of the national institute on aging late-onset alzheimer\\u2019s disease family study: Implication of additional loci.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 26534, \"infons\": {\"fpage\": \"41\", \"issue\": \"1\", \"lpage\": \"58\", \"name_0\": \"surname:He;given-names:L\", \"name_1\": \"surname:Kulminski;given-names:AM\", \"pub-id_pmid\": \"32132097\", \"section_type\": \"REF\", \"source\": \"Genetics\", \"type\": \"ref\", \"volume\": \"215\", \"year\": \"2020\"}, \"text\": \"Fast algorithms for conducting large-scale GWAS of age-at-onset traits using cox mixed-effects models.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 26637, \"infons\": {\"fpage\": \"2336\", \"issue\": \"18\", \"lpage\": \"2337\", \"name_0\": \"surname:Pruim;given-names:RJ\", \"name_1\": \"surname:Welch;given-names:RP\", \"name_2\": \"surname:Sanna;given-names:S\", \"name_3\": \"surname:Teslovich;given-names:TM\", \"name_4\": \"surname:Chines;given-names:PS\", \"name_5\": \"surname:Gliedt;given-names:TP\", \"pub-id_pmid\": \"20634204\", \"section_type\": \"REF\", \"source\": \"Bioinformatics (Oxford, England)\", \"type\": \"ref\", \"volume\": \"26\", \"year\": \"2010\"}, \"text\": \"LocusZoom: Regional visualization of genome-wide association scan results.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 26712, \"infons\": {\"fpage\": \"153\", \"issue\": \"4\", \"lpage\": \"160\", \"name_0\": \"surname:Balduzzi;given-names:S\", \"name_1\": \"surname:Rucker;given-names:G\", \"name_2\": \"surname:Schwarzer;given-names:G\", \"pub-id_pmid\": \"31563865\", \"section_type\": \"REF\", \"source\": \"Evidence-Based Mental Health\", \"type\": \"ref\", \"volume\": \"22\", \"year\": \"2019\"}, \"text\": \"How to perform a meta-analysis with R: A practical tutorial.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 26773, \"infons\": {\"fpage\": \"46\", \"lpage\": \"56\", \"name_0\": \"surname:Xu;given-names:B\", \"name_1\": \"surname:Gao;given-names:Y\", \"name_2\": \"surname:Zhan;given-names:S\", \"name_3\": \"surname:Xiong;given-names:F\", \"name_4\": \"surname:Qiu;given-names:W\", \"name_5\": \"surname:Qian;given-names:X\", \"pub-id_pmid\": \"26923401\", \"section_type\": \"REF\", \"source\": \"Neurobiology of Aging\", \"type\": \"ref\", \"volume\": \"39\", \"year\": \"2016\"}, \"text\": \"Quantitative protein profiling of hippocampus during human aging.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 26839, \"infons\": {\"comment\": \"4277\\u2013020-61142\\u20138.\", \"issue\": \"1\", \"name_0\": \"surname:Area-Gomez;given-names:E\", \"name_1\": \"surname:Larrea;given-names:D\", \"name_2\": \"surname:Pera;given-names:M\", \"name_3\": \"surname:Agrawal;given-names:RR\", \"name_4\": \"surname:Guilfoyle;given-names:DN\", \"name_5\": \"surname:Pirhaji;given-names:L\", \"section_type\": \"REF\", \"source\": \"Scientific Reports\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2020\"}, \"text\": \"APOE4 is associated with differential regional vulnerability to bioenergetic deficits in aged APOE mice.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 26944, \"infons\": {\"fpage\": \"10682\", \"name_0\": \"surname:Klaassen;given-names:RV\", \"name_1\": \"surname:Stroeder;given-names:J\", \"name_2\": \"surname:Coussen;given-names:F\", \"name_3\": \"surname:Hafner;given-names:AS\", \"name_4\": \"surname:Petersen;given-names:JD\", \"name_5\": \"surname:Renancio;given-names:C\", \"section_type\": \"REF\", \"source\": \"Nature Communications\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2016\"}, \"text\": \"Shisa6 traps AMPA receptors at postsynaptic sites and prevents their desensitization during synaptic activity.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 27055, \"infons\": {\"fpage\": \"29\", \"issue\": \"1\", \"lpage\": \"36\", \"name_0\": \"surname:Rosenberg;given-names:A\", \"name_1\": \"surname:Mangialasche;given-names:F\", \"name_2\": \"surname:Ngandu;given-names:T\", \"name_3\": \"surname:Solomon;given-names:A\", \"name_4\": \"surname:Kivipelto;given-names:M\", \"section_type\": \"REF\", \"source\": \"The Journal of Prevention of Alzheimer\\u2019s Disease\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2020\"}, \"text\": \"Multidomain interventions to prevent cognitive impairment, alzheimer\\u2019s disease, and dementia: From FINGER to world-wide FINGERS.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 27186, \"infons\": {\"fpage\": \"327\", \"issue\": \"3\", \"lpage\": \"406\", \"section_type\": \"REF\", \"source\": \"Alzheimer\\u2019s & Dementia : The Journal of the Alzheimer\\u2019s Association\", \"type\": \"ref\", \"volume\": \"17\", \"year\": \"2021\"}, \"text\": \"\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 27187, \"infons\": {\"fpage\": \"1294\", \"issue\": \"11\", \"lpage\": \"1300\", \"name_0\": \"surname:Lalli;given-names:MA\", \"name_1\": \"surname:Bettcher;given-names:BM\", \"name_2\": \"surname:Arcila;given-names:ML\", \"name_3\": \"surname:Garcia;given-names:G\", \"name_4\": \"surname:Guzman;given-names:C\", \"name_5\": \"surname:Madrigal;given-names:L\", \"pub-id_pmid\": \"26324103\", \"section_type\": \"REF\", \"source\": \"Molecular Psychiatry\", \"type\": \"ref\", \"volume\": \"20\", \"year\": \"2015\"}, \"text\": \"Whole-genome sequencing suggests a chemokine gene cluster that modifies age at onset in familial alzheimer\\u2019s disease.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 27307, \"infons\": {\"file\": \"nihms-1797266-f0001.jpg\", \"id\": \"F1\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Manhattan plot showing results of GWAS of CU survival time using Cox mixed-effects model:\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 27397, \"infons\": {\"file\": \"nihms-1797266-f0001.jpg\", \"id\": \"F1\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Red line indicates the threshold for genome wide significance (P < 5 \\u00d7 10\\u22128) while the blue line represents the suggestive threshold (P < 1 \\u00d7 10\\u22125). Three variants reached genome wide significance: 6:149756107_C_G (rs145348074), 9:128627091_C_G (rs534551495), and 17:11183894_G_A (rs146729640). Mapping is based on GRCh37/hg19 assembly.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 27740, \"infons\": {\"file\": \"nihms-1797266-f0002.jpg\", \"id\": \"F2\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Quantile \\u2013 quantile (QQ) plot showing the results of GWAS p-values of CU survival time in the Amish cohort using Cox mixed-effects model:\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 27880, \"infons\": {\"file\": \"nihms-1797266-f0002.jpg\", \"id\": \"F2\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"This figure clearly shows that GWAS is not inflated. The genome-wide inflation factor (\\u03bb) was 1.01\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 27984, \"infons\": {\"file\": \"T1.xml\", \"id\": \"T1\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Demographic characteristics of the 1,522 Amish individuals included in the study. About 24% of the sample was cognitively impaired at the last examination. Cognitively impaired individuals were older at the last examination, and the two groups had similar proportions of men and women.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 28270, \"infons\": {\"file\": \"T1.xml\", \"id\": \"T1\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"box\\\" rules=\\\"all\\\">\\n        <colgroup span=\\\"1\\\">\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n        </colgroup>\\n        <thead>\\n          <tr>\\n            <th align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Column1</th>\\n            <th align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">COGNITIVELY <break/>IMPAIRED</th>\\n            <th align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">COGNITIVELY UNIMPAIRED</th>\\n            <th align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">TOTAL</th>\\n          </tr>\\n        </thead>\\n        <tbody>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">N</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">362</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1160</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1522</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">MALE</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">152 (42.0 %)</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">483 (41.6 %)</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">635 (41.7 %)</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FEMALE</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">210 (58.0 %)</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">677 (58.4 %)</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">887 (58.3 %)</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">AGE at the last exam</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">81.6</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">70.6</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.23</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">AAO</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">81.2</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">81.2</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">APOE4 carriers</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">105</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">290</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">395</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">non-APOE4 carriers</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">257</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">870</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1127</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">AGE (AAO for CI and Age at exam for CU) <break/>APOE4 carriers</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">80.1</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">70.1</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">72.7</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">AGE (AAO for CI and Age at exam for CU) <break/>non-APOE4 carriers</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">81.6</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">70.7</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.2</td>\\n          </tr>\\n        </tbody>\\n      </table>\\n\"}, \"text\": \"Column1\\tCOGNITIVELY IMPAIRED\\tCOGNITIVELY UNIMPAIRED\\tTOTAL\\t \\tN\\t362\\t1160\\t1522\\t \\tMALE\\t152 (42.0 %)\\t483 (41.6 %)\\t635 (41.7 %)\\t \\tFEMALE\\t210 (58.0 %)\\t677 (58.4 %)\\t887 (58.3 %)\\t \\tAGE at the last exam\\t81.6\\t70.6\\t73.23\\t \\tAAO\\t81.2\\tNA\\t81.2\\t \\tAPOE4 carriers\\t105\\t290\\t395\\t \\tnon-APOE4 carriers\\t257\\t870\\t1127\\t \\tAGE (AAO for CI and Age at exam for CU) APOE4 carriers\\t80.1\\t70.1\\t72.7\\t \\tAGE (AAO for CI and Age at exam for CU) non-APOE4 carriers\\t81.6\\t70.7\\t73.2\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 28712, \"infons\": {\"file\": \"T2.xml\", \"id\": \"T2\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Demographic characteristics of NIA-LOAD data set used for replication. About 53% of the sample was clinically diagnosed with LOAD. Age at onset for AD cases was similar to age at exam for controls. The group of AD has a higher proportion of women (65 vs 60%).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 28972, \"infons\": {\"file\": \"T2.xml\", \"id\": \"T2\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"box\\\" rules=\\\"all\\\">\\n        <colgroup span=\\\"1\\\">\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n        </colgroup>\\n        <thead>\\n          <tr>\\n            <th align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <th align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">AD</th>\\n            <th align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">CONTROLS</th>\\n            <th align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">TOTAL</th>\\n          </tr>\\n        </thead>\\n        <tbody>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>N</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,785</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,565</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">3,350</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>MALE</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">623 (34.9 %)</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">623 (39.8 %)</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,246 (37.2 %)</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>FEMALE</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,162 (65.1 %)</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">942 (60.2 %)</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2,104 (62.8 %)</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>AGE (AAO for AD and Age at exam for controls)</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.1</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.9</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">73.5</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>APOE4 carriers</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,326</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">572</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,898</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>non-APOE4 carriers</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">459</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">993</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,452</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>AGE (AAO for AD and Age at exam for controls) APOE4 carriers</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">72.5</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">71.5</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">72.2</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>AGE (AAO for AD and Age at exam for controls) non-APOE4 carriers</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">74.8</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">75.2</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">75.1</td>\\n          </tr>\\n        </tbody>\\n      </table>\\n\"}, \"text\": \"\\tAD\\tCONTROLS\\tTOTAL\\t \\tN\\t1,785\\t1,565\\t3,350\\t \\tMALE\\t623 (34.9 %)\\t623 (39.8 %)\\t1,246 (37.2 %)\\t \\tFEMALE\\t1,162 (65.1 %)\\t942 (60.2 %)\\t2,104 (62.8 %)\\t \\tAGE (AAO for AD and Age at exam for controls)\\t73.1\\t73.9\\t73.5\\t \\tAPOE4 carriers\\t1,326\\t572\\t1,898\\t \\tnon-APOE4 carriers\\t459\\t993\\t1,452\\t \\tAGE (AAO for AD and Age at exam for controls) APOE4 carriers\\t72.5\\t71.5\\t72.2\\t \\tAGE (AAO for AD and Age at exam for controls) non-APOE4 carriers\\t74.8\\t75.2\\t75.1\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 29407, \"infons\": {\"file\": \"T3.xml\", \"id\": \"T3\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Three common variants were significantly associated (p < 5 \\u00d7 10 \\u22128) with CU survival time in the Amish dataset. The strongest result was on chromosome 17, followed by chromosomes 6 and 9. Variants are annotated as overlapping a gene if located within the gene boundaries. Other genes within a 1Mb interval centered on the associated variant are also listed.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 29768, \"infons\": {\"file\": \"T3.xml\", \"id\": \"T3\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"box\\\" rules=\\\"all\\\">\\n        <colgroup span=\\\"1\\\">\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n        </colgroup>\\n        <thead>\\n          <tr>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Location and base pair change (build hg19)</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">SNP (rs)</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">MAF</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Variant type</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Hazard Ratio</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">P-value</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Overlapping Genes</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Genes within +/\\u2212 500Kb</th>\\n          </tr>\\n        </thead>\\n        <tbody>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6:149756107_C_G</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs145348074</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.0237382</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Intergenic</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">3.35</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5.26E-08</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Intergenic variant. Closest gene: <italic toggle=\\\"yes\\\">ZC3H12D</italic></td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">UST, UST-AS1, LOC105378047, TAB2, TAB2-AS1, SUMO4, ZC3H12D, PPIL4, GINM1, KATNA1, LATS1, LOC645967, NUP43, PCMT1, LRP11, RAET1E-AS1, RAET1E, RAET1G, LOC105378052</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">9:128627091_C_G</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs534551495</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.0379333</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Intron</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2.82</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">4.83E-08</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">PBX3</italic>\\n</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">GAPVD1, MAPKAP1, LOC51145, PBX3, LOC10192911, MVB12B</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">17:11183894_G_A</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs146729640</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.0119911</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Intron</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6.38</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5.59E-09</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">SHISA6</italic>\\n</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">TMEM220-AS1, DNAH9, TMEM238L, PIRT, SHISA6,</italic>\\n</td>\\n          </tr>\\n        </tbody>\\n      </table>\\n\"}, \"text\": \"Location and base pair change (build hg19)\\tSNP (rs)\\tMAF\\tVariant type\\tHazard Ratio\\tP-value\\tOverlapping Genes\\tGenes within +/\\u2212 500Kb\\t \\t6:149756107_C_G\\trs145348074\\t0.0237382\\tIntergenic\\t3.35\\t5.26E-08\\tIntergenic variant. Closest gene: ZC3H12D\\tUST, UST-AS1, LOC105378047, TAB2, TAB2-AS1, SUMO4, ZC3H12D, PPIL4, GINM1, KATNA1, LATS1, LOC645967, NUP43, PCMT1, LRP11, RAET1E-AS1, RAET1E, RAET1G, LOC105378052\\t \\t9:128627091_C_G\\trs534551495\\t0.0379333\\tIntron\\t2.82\\t4.83E-08\\tPBX3\\tGAPVD1, MAPKAP1, LOC51145, PBX3, LOC10192911, MVB12B\\t \\t17:11183894_G_A\\trs146729640\\t0.0119911\\tIntron\\t6.38\\t5.59E-09\\tSHISA6\\tTMEM220-AS1, DNAH9, TMEM238L, PIRT, SHISA6,\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 30404, \"infons\": {\"file\": \"T4.xml\", \"id\": \"T4\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Results of local linkage disequilibrium (LD) analysis of genome wide significant variants. Variants rs141138977 and rs140984313 were found in LD (1.0 > r2 \\u2265 0.8) with rs534551495, the genome wide signal found on chromosome 9. Variant rs143831028 was in LD (0.6 > r2 \\u2265 0.4) with rs145348074, the genome wide signal found on chromosome 6. None of the variants was found in LD (r2 \\u2265 0.4) with rs146729640, the genome wide signal found on chromosome 17.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 30860, \"infons\": {\"file\": \"T4.xml\", \"id\": \"T4\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"box\\\" rules=\\\"all\\\">\\n        <colgroup span=\\\"1\\\">\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n        </colgroup>\\n        <thead>\\n          <tr>\\n            <th rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">Genome wide Variant</th>\\n            <th rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">Variant (rs)</th>\\n            <th rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">MAF</th>\\n            <th rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">P value</th>\\n            <th rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">Genes</th>\\n            <th colspan=\\\"6\\\" align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\">Variants in LD (+/\\u2212500Kb)</th>\\n          </tr>\\n          <tr>\\n            <th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Variant</th>\\n            <th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Variant (rs)</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">r<sup>2</sup></th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">MAF</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">P value</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Genes</th>\\n          </tr>\\n        </thead>\\n        <tbody>\\n          <tr>\\n            <td rowspan=\\\"6\\\" align=\\\"left\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">\\n<bold>6:149756107_C_G</bold>\\n</td>\\n            <td rowspan=\\\"6\\\" align=\\\"left\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">\\n<bold>rs145348074</bold>\\n</td>\\n            <td rowspan=\\\"6\\\" align=\\\"left\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">0.024</td>\\n            <td rowspan=\\\"6\\\" align=\\\"left\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">5.26E-08</td>\\n            <td rowspan=\\\"6\\\" align=\\\"left\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">Intergenic variant. Closest gene: <bold><italic toggle=\\\"yes\\\">ZC3H12D</italic></bold></td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>6:149622242_A_T</bold>\\n</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>rs143831028</bold>\\n</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>0.6 &gt; r2 \\u2265 0.4</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.023</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.00E-7</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">TAB2</italic>\\n</bold>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6:149688198_G_A</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs192019712</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.4 &gt; r2 \\u2265 0.2</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.015</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.29E-6</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">TAB2</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6:149898888_T_A</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs188874345</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.4 &gt; r2 \\u2265 0.2</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.016</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.35E-6</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">GINM1</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6:149907017_C_T</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs530694742</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.4 &gt; r2 \\u2265 0.2</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.016</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.44E-6</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">GINM1</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6:149959451_T_C</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs142372487</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.4 &gt; r2 \\u2265 0.2</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.016</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.27E-6</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">KATNA1</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6:150057902_G_A</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs541233509</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.4 &gt; r2 \\u2265 0.2</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.016</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.58E-6</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">NUP43</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td rowspan=\\\"4\\\" align=\\\"left\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">\\n<bold>9:128627091_C_G</bold>\\n</td>\\n            <td rowspan=\\\"4\\\" align=\\\"left\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">\\n<bold>rs534551495</bold>\\n</td>\\n            <td rowspan=\\\"4\\\" align=\\\"left\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">0.038</td>\\n            <td rowspan=\\\"4\\\" align=\\\"left\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">4.83E-08</td>\\n            <td rowspan=\\\"4\\\" align=\\\"left\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">PBX3</italic>\\n</bold>\\n</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>9:128627225_G_T</bold>\\n</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>rs141138977</bold>\\n</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>1.0 &gt; r2 \\u2265 0.8</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.038</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5.45E-08</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">PBX3</italic>\\n</bold>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>9:128319404_G_A</bold>\\n</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>rs140984313</bold>\\n</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>1.0 &gt; r2 \\u2265 0.8</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.038</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6.46E-08</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">MAPKAP1</italic>\\n</bold>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">9:128481411_A_T</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs113929904</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.4 &gt; r2 \\u2265 0.2</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.046</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2.50E-7</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">Closest gene:</italic>\\n<break/>\\n<italic toggle=\\\"yes\\\">MAPKAP1</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">9:128175053_G_A</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs149607027</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.4 &gt; r2 \\u2265 0.2</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.042</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.33E-6</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">Closest gene:</italic>\\n<break/>\\n<italic toggle=\\\"yes\\\">MAPKAP1</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>17:11183894_G_A</bold>\\n</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>rs146729640</bold>\\n</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.012</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5.59E-09</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>\\n<italic toggle=\\\"yes\\\">SHISA6</italic>\\n</bold>\\n</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">17:11306227_G_A</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs111837116</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.4 &gt; r2 \\u2265 0.2</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.017</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">4.72E-04</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">SHISA6</td>\\n          </tr>\\n        </tbody>\\n      </table>\\n\"}, \"text\": \"Genome wide Variant\\tVariant (rs)\\tMAF\\tP value\\tGenes\\tVariants in LD (+/\\u2212500Kb)\\t \\tVariant\\tVariant (rs)\\tr2\\tMAF\\tP value\\tGenes\\t \\t6:149756107_C_G\\trs145348074\\t0.024\\t5.26E-08\\tIntergenic variant. Closest gene: ZC3H12D\\t6:149622242_A_T\\trs143831028\\t0.6 > r2 \\u2265 0.4\\t0.023\\t1.00E-7\\tTAB2\\t \\t6:149688198_G_A\\trs192019712\\t0.4 > r2 \\u2265 0.2\\t0.015\\t1.29E-6\\tTAB2\\t \\t6:149898888_T_A\\trs188874345\\t0.4 > r2 \\u2265 0.2\\t0.016\\t1.35E-6\\tGINM1\\t \\t6:149907017_C_T\\trs530694742\\t0.4 > r2 \\u2265 0.2\\t0.016\\t1.44E-6\\tGINM1\\t \\t6:149959451_T_C\\trs142372487\\t0.4 > r2 \\u2265 0.2\\t0.016\\t1.27E-6\\tKATNA1\\t \\t6:150057902_G_A\\trs541233509\\t0.4 > r2 \\u2265 0.2\\t0.016\\t1.58E-6\\tNUP43\\t \\t9:128627091_C_G\\trs534551495\\t0.038\\t4.83E-08\\tPBX3\\t9:128627225_G_T\\trs141138977\\t1.0 > r2 \\u2265 0.8\\t0.038\\t5.45E-08\\tPBX3\\t \\t9:128319404_G_A\\trs140984313\\t1.0 > r2 \\u2265 0.8\\t0.038\\t6.46E-08\\tMAPKAP1\\t \\t9:128481411_A_T\\trs113929904\\t0.4 > r2 \\u2265 0.2\\t0.046\\t2.50E-7\\tClosest gene:MAPKAP1\\t \\t9:128175053_G_A\\trs149607027\\t0.4 > r2 \\u2265 0.2\\t0.042\\t1.33E-6\\tClosest gene:MAPKAP1\\t \\t17:11183894_G_A\\trs146729640\\t0.012\\t5.59E-09\\tSHISA6\\t17:11306227_G_A\\trs111837116\\t0.4 > r2 \\u2265 0.2\\t0.017\\t4.72E-04\\tSHISA6\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 31940, \"infons\": {\"file\": \"T5.xml\", \"id\": \"T5\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Results of the Cox mixed effect model for replication using NIA-LOAD data set show nominal association (p = 0.02) in the chromosome 17 variant (rs146729640).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 32098, \"infons\": {\"file\": \"T5.xml\", \"id\": \"T5\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"box\\\" rules=\\\"all\\\">\\n        <colgroup span=\\\"1\\\">\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n        </colgroup>\\n        <thead>\\n          <tr>\\n            <th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Chr</th>\\n            <th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Variant</th>\\n            <th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Variant (rs)</th>\\n            <th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Data set</th>\\n            <th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"># of subjects</th>\\n            <th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">MAF</th>\\n            <th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Hazard ratio</th>\\n            <th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">P-value</th>\\n          </tr>\\n        </thead>\\n        <tbody>\\n          <tr>\\n            <td rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">17</td>\\n            <td rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">17:11183894_G_A</td>\\n            <td rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">rs146729640</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Amish</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1520</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.012</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6.38</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5.59E-09</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NIA-LOAD</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">3350</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.012</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.49</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.0217</td>\\n          </tr>\\n          <tr>\\n            <td rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">9</td>\\n            <td rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">9:128627091_C_G</td>\\n            <td rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">rs534551495</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Amish</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1520</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.038</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2.82</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">4.83E-08</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NIA-LOAD</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">3350</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.006</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.98</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.9356</td>\\n          </tr>\\n          <tr>\\n            <td rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">6</td>\\n            <td rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">6:149756107_C_G</td>\\n            <td rowspan=\\\"2\\\" align=\\\"center\\\" valign=\\\"bottom\\\" colspan=\\\"1\\\">rs145348074</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Amish</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1520</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.024</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">3.35</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5.26E-08</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NIA-LOAD</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">3350</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.013</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.16</td>\\n            <td align=\\\"right\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.4164</td>\\n          </tr>\\n        </tbody>\\n      </table>\\n\"}, \"text\": \"Chr\\tVariant\\tVariant (rs)\\tData set\\t# of subjects\\tMAF\\tHazard ratio\\tP-value\\t \\t17\\t17:11183894_G_A\\trs146729640\\tAmish\\t1520\\t0.012\\t6.38\\t5.59E-09\\t \\tNIA-LOAD\\t3350\\t0.012\\t1.49\\t0.0217\\t \\t9\\t9:128627091_C_G\\trs534551495\\tAmish\\t1520\\t0.038\\t2.82\\t4.83E-08\\t \\tNIA-LOAD\\t3350\\t0.006\\t0.98\\t0.9356\\t \\t6\\t6:149756107_C_G\\trs145348074\\tAmish\\t1520\\t0.024\\t3.35\\t5.26E-08\\t \\tNIA-LOAD\\t3350\\t0.013\\t1.16\\t0.4164\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 32466, \"infons\": {\"file\": \"T6.xml\", \"id\": \"T6\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Stratified survival analysis of rs146729640 variant in the Amish dataset. The strength of association was increased in the non- APOE4 carriers.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 32610, \"infons\": {\"file\": \"T6.xml\", \"id\": \"T6\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"box\\\" rules=\\\"all\\\">\\n        <colgroup span=\\\"1\\\">\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n        </colgroup>\\n        <thead>\\n          <tr>\\n            <th align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"><italic toggle=\\\"yes\\\">APOE4</italic> stratification <break/>Amish</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">SNP (chr:pos)</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">SNP (rs)</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">N (CI)</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">N (CU)</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Hazard Ratio</th>\\n            <th align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Hazard Ratio<break/>95% CI</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">P-value</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Gene</th>\\n          </tr>\\n        </thead>\\n        <tbody>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">No</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">17:11183894_G_A</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs146729640</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">362</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,160</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">6.38</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">3.42 ; 11.90</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">5.59E-09</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">SHISA6</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"><italic toggle=\\\"yes\\\">APOE4</italic> carriers</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">17:11183894_G_A</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs146729640</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">105</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">290</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2.93</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.16 ; 7.40</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2.32E-02</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">SHISA6</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">non-<italic toggle=\\\"yes\\\">APOE4</italic> carriers</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">17:11183894_G_A</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs146729640</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">257</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">870</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">8.96</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">4.25 ; 18.91</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">8.44E-09</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">SHISA6</td>\\n          </tr>\\n        </tbody>\\n      </table>\\n\"}, \"text\": \"APOE4 stratification Amish\\tSNP (chr:pos)\\tSNP (rs)\\tN (CI)\\tN (CU)\\tHazard Ratio\\tHazard Ratio95% CI\\tP-value\\tGene\\t \\tNo\\t17:11183894_G_A\\trs146729640\\t362\\t1,160\\t6.38\\t3.42 ; 11.90\\t5.59E-09\\tSHISA6\\t \\tAPOE4 carriers\\t17:11183894_G_A\\trs146729640\\t105\\t290\\t2.93\\t1.16 ; 7.40\\t2.32E-02\\tSHISA6\\t \\tnon-APOE4 carriers\\t17:11183894_G_A\\trs146729640\\t257\\t870\\t8.96\\t4.25 ; 18.91\\t8.44E-09\\tSHISA6\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 32976, \"infons\": {\"file\": \"T7.xml\", \"id\": \"T7\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Stratified survival analysis of rs146729640 variant in the NIA-LOAD dataset. Similar to the Amish discovery set, the strength of association was increased in the non-APOE4 carriers.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33158, \"infons\": {\"file\": \"T7.xml\", \"id\": \"T7\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"box\\\" rules=\\\"all\\\">\\n        <colgroup span=\\\"1\\\">\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n        </colgroup>\\n        <thead>\\n          <tr>\\n            <th align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"><italic toggle=\\\"yes\\\">APOE4</italic> stratification<break/>NIA-LOAD</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">SNP (chr:pos)</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">SNP (rs)</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">N (AD)</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">N (controls)</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Hazard Ratio</th>\\n            <th align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Hazard Ratio<break/>95% CI</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">P-value</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Gene</th>\\n          </tr>\\n        </thead>\\n        <tbody>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">No</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">17:11183894_G_A</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs146729640</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,785</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,565</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.49</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.06 ; 2.08</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.0217</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">SHISA6</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"><italic toggle=\\\"yes\\\">APOE4</italic> carriers</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">17:11183894_G_A</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs146729640</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1,326</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">572</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.35</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.95 ; 1.93</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.0950</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">SHISA6</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">non-<italic toggle=\\\"yes\\\">APOE4</italic> carriers</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">17:11183894_G_A</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs146729640</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">459</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">993</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.82</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.96 ; 3.43</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.0658</td>\\n            <td align=\\\"center\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">SHISA6</td>\\n          </tr>\\n        </tbody>\\n      </table>\\n\"}, \"text\": \"APOE4 stratificationNIA-LOAD\\tSNP (chr:pos)\\tSNP (rs)\\tN (AD)\\tN (controls)\\tHazard Ratio\\tHazard Ratio95% CI\\tP-value\\tGene\\t \\tNo\\t17:11183894_G_A\\trs146729640\\t1,785\\t1,565\\t1.49\\t1.06 ; 2.08\\t0.0217\\tSHISA6\\t \\tAPOE4 carriers\\t17:11183894_G_A\\trs146729640\\t1,326\\t572\\t1.35\\t0.95 ; 1.93\\t0.0950\\tSHISA6\\t \\tnon-APOE4 carriers\\t17:11183894_G_A\\trs146729640\\t459\\t993\\t1.82\\t0.96 ; 3.43\\t0.0658\\tSHISA6\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33528, \"infons\": {\"file\": \"T8.xml\", \"id\": \"T8\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Meta-analysis of rs146729640 variant in the Amish and NIA-LOAD datasets. Similar to the individual results of the discovery and replication studies, the strength of association was increased in the non-APOE4 carriers\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33745, \"infons\": {\"file\": \"T8.xml\", \"id\": \"T8\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"box\\\" rules=\\\"all\\\">\\n        <colgroup span=\\\"1\\\">\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n          <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n        </colgroup>\\n        <thead>\\n          <tr>\\n            <th align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"><italic toggle=\\\"yes\\\">APOE4</italic> stratification</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">SNP (rs)</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Studies combined</th>\\n            <th align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Number of observations</th>\\n            <th colspan=\\\"4\\\" align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\">Meta-Analysis results</th>\\n          </tr>\\n        </thead>\\n        <tbody>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>Model</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>Hazard Ratio</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>Hazard Ratio 95 % CI</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>P-value</bold>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>NO</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs146729640</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Amish and NIA-LOAD</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">4,872</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Fixed effect</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2.07</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.54; 2.78</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">&lt;0.0001</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Random effects</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">3.00</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.72; 12.50</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">0.1308</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>APOE4 carriers</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs146729640</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Amish and NIA-LOAD</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2,293</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Fixed effect</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.50</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.072; 2.09</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<italic toggle=\\\"yes\\\">0.0178</italic>\\n</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Random effects</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">1.76</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.86; 3.60</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.1219</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">\\n<bold>non-<italic toggle=\\\"yes\\\">APOE4</italic> carriers</bold>\\n</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">rs146729640</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Amish and NIA-LOAD</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2,579</td>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Fixed effect</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">3.56</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">2.19; 5.78</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">&lt;0.0001</td>\\n          </tr>\\n          <tr>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\"/>\\n            <td align=\\\"left\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Random effects</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">3.99</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.83; 19.04</td>\\n            <td align=\\\"center\\\" valign=\\\"bottom\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">0.0831</td>\\n          </tr>\\n        </tbody>\\n      </table>\\n\"}, \"text\": \"APOE4 stratification\\tSNP (rs)\\tStudies combined\\tNumber of observations\\tMeta-Analysis results\\t \\t\\t\\t\\t\\tModel\\tHazard Ratio\\tHazard Ratio 95 % CI\\tP-value\\t \\tNO\\trs146729640\\tAmish and NIA-LOAD\\t4,872\\tFixed effect\\t2.07\\t1.54; 2.78\\t<0.0001\\t \\t\\t\\t\\t\\tRandom effects\\t3.00\\t0.72; 12.50\\t0.1308\\t \\tAPOE4 carriers\\trs146729640\\tAmish and NIA-LOAD\\t2,293\\tFixed effect\\t1.50\\t1.072; 2.09\\t0.0178\\t \\t\\t\\t\\t\\tRandom effects\\t1.76\\t0.86; 3.60\\t0.1219\\t \\tnon-APOE4 carriers\\trs146729640\\tAmish and NIA-LOAD\\t2,579\\tFixed effect\\t3.56\\t2.19; 5.78\\t<0.0001\\t \\t\\t\\t\\t\\tRandom effects\\t3.99\\t0.83; 19.04\\t0.0831\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34293, \"infons\": {\"section_type\": \"REF\", \"type\": \"title_caption\"}, \"text\": \"HIGHLIGHTS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34304, \"infons\": {\"section_type\": \"REF\", \"type\": \"paragraph\"}, \"text\": \"Replicated genome-wide association of CI-free survival time in the SHISA6 gene.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34384, \"infons\": {\"section_type\": \"REF\", \"type\": \"paragraph\"}, \"text\": \"SHISA6 influences post-synaptic transmission.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34430, \"infons\": {\"section_type\": \"REF\", \"type\": \"paragraph\"}, \"text\": \"SHISA6 expression declines with age in hippocampus.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34482, \"infons\": {\"section_type\": \"REF\", \"type\": \"paragraph\"}, \"text\": \"SHISA6 is downregulated in entorhinal cortex of aged APOE4 transgenic mice.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34558, \"infons\": {\"section_type\": \"REF\", \"type\": \"paragraph\"}, \"text\": \"SHISA6 is thus a replicated, biologically plausible candidate gene for AAO of CI.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34640, \"infons\": {\"section_type\": \"REF\", \"type\": \"title_caption\"}, \"text\": \"RESEARCH IN CONTEXT\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34660, \"infons\": {\"section_type\": \"REF\", \"type\": \"paragraph\"}, \"text\": \"Systematic Review: We reviewed the literature, using traditional (e.g., PubMed) sources for longitudinal studies of aging and cognitive impairment. Several reports of sibships with CI and unaffected (CU) individuals paralleled our observations in the Amish. We reviewed publications on the SHISA6 gene and found influences on AD pathogenesis in humans and animal models.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 35031, \"infons\": {\"section_type\": \"REF\", \"type\": \"paragraph\"}, \"text\": \"Interpretation: In 1,522 individuals screened for CI and genotyped, we found three single nucleotide variants (SNV) associated with age at onset (AAO) of CI. One association on chromosome 17 (rs146729640) in the SHISA6 gene was replicated in the NIA-LOAD family study dataset (N=3,350). SHISA6, involved in post-synaptic transmission in the hippocampus, is a biologically plausible candidate gene for AD that might play a role in the rate of cognitive decline.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 35492, \"infons\": {\"section_type\": \"REF\", \"type\": \"paragraph\"}, \"text\": \"Future Directions: Future research should examine whether genetic variation influences SHISA6 gene expression in brain tissue from AD and control donors.\", \"sentences\": [], \"annotations\": [], \"relations\": []}], \"annotations\": [], \"relations\": []}]}]\n",
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading weights: 100%|| 770/770 [00:00<00:00, 1644.11it/s, Materializing param=model.encoder_input_proj.2.1.weight]                                                 \n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    }
   ],
   "source": [
    "pmcid = \"PMC9622429\"\n",
    "file_name = \"papers/nihms-1797266.pdf\"  # Can be a local path or a URL\n",
    "df_lst = extract_tables_lst_from_paper(pmcid, file_name)  \n",
    "for i in [3, 5, 6, 7, 8]:\n",
    "    df_lst[i-1].to_csv(f\"tables/{file_name.split('/')[-1].replace('.pdf', '')}_table_{i}.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "dbf1ea7d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{\"source\": \"PMC\", \"date\": \"20240127\", \"key\": \"pmc.key\", \"infons\": {}, \"documents\": [{\"id\": \"10615750\", \"infons\": {\"license\": \"CC BY\"}, \"passages\": [{\"offset\": 0, \"infons\": {\"article-id_doi\": \"10.1038/s41380-023-02089-w\", \"article-id_pmc\": \"10615750\", \"article-id_pmid\": \"37198259\", \"article-id_publisher-id\": \"2089\", \"fpage\": \"3121\", \"issue\": \"7\", \"kwd\": \"Neuroscience Genetics Diseases\", \"license\": \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article\\u2019s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article\\u2019s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit http://creativecommons.org/licenses/by/4.0/.\", \"lpage\": \"3132\", \"name_0\": \"surname:Lake;given-names:Julie\", \"name_1\": \"surname:Warly Solsberg;given-names:Caroline\", \"name_10\": \"surname:Kang;given-names:Sarang\", \"name_11\": \"surname:Gim;given-names:Jungsoo\", \"name_12\": \"surname:Lee;given-names:Kun Ho\", \"name_13\": \"surname:Pina-Escudero;given-names:Stefanie D.\", \"name_14\": \"surname:Ferrucci;given-names:Luigi\", \"name_15\": \"surname:Singleton;given-names:Andrew B.\", \"name_16\": \"surname:Blauwendraat;given-names:Cornelis\", \"name_17\": \"surname:Nalls;given-names:Mike A.\", \"name_18\": \"surname:Yokoyama;given-names:Jennifer S.\", \"name_19\": \"surname:Leonard;given-names:Hampton L.\", \"name_2\": \"surname:Kim;given-names:Jonggeol Jeffrey\", \"name_20\": \"surname:Lake;given-names:Julie\", \"name_21\": \"surname:Warly Solsberg;given-names:Caroline\", \"name_22\": \"surname:Yokoyama;given-names:Jennifer S.\", \"name_23\": \"surname:Leonard;given-names:Hampton L.\", \"name_24\": \"surname:Warly Solsberg;given-names:Caroline\", \"name_25\": \"surname:Yokoyama;given-names:Jennifer S.\", \"name_3\": \"surname:Acosta-Uribe;given-names:Juliana\", \"name_4\": \"surname:Makarious;given-names:Mary B.\", \"name_5\": \"surname:Li;given-names:Zizheng\", \"name_6\": \"surname:Levine;given-names:Kristin\", \"name_7\": \"surname:Heutink;given-names:Peter\", \"name_8\": \"surname:Alvarado;given-names:Chelsea X.\", \"name_9\": \"surname:Vitale;given-names:Dan\", \"section_type\": \"TITLE\", \"title\": \"Subject terms\", \"type\": \"front\", \"volume\": \"28\", \"year\": \"2023\"}, \"text\": \"Multi-ancestry meta-analysis and fine-mapping in Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Genome-wide association studies (GWAS) of Alzheimer\\u2019s disease are predominantly carried out in European ancestry individuals despite the known variation in genetic architecture and disease prevalence across global populations. We leveraged published GWAS summary statistics from European, East Asian, and African American populations, and an additional GWAS from a Caribbean Hispanic population using previously reported genotype data to perform the largest multi-ancestry GWAS meta-analysis of Alzheimer\\u2019s disease and related dementias to date. This method allowed us to identify two independent novel disease-associated loci on chromosome 3. We also leveraged diverse haplotype structures to fine-map nine loci with a posterior probability >0.8 and globally assessed the heterogeneity of known risk factors across populations. Additionally, we compared the generalizability of multi-ancestry- and single-ancestry-derived polygenic risk scores in a three-way admixed Colombian population. Our findings highlight the importance of multi-ancestry representation in uncovering and understanding putative factors that contribute to risk of Alzheimer\\u2019s disease and related dementias.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 1257, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"title_1\"}, \"text\": \"Introduction\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 1270, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Alzheimer\\u2019s disease (AD) is a complex genetic disorder with a range of deleterious variants across multiple genes attributed to both early and late-onset forms of sporadic AD. The strongest genetic risk factor for late-onset AD is APOE-e4, yet it has been estimated that there may be anywhere from 100 to 11,000 variants that also contribute to risk of late-onset AD. Large-scale genome-wide association studies (GWAS) in European ancestry populations have identified over 75 loci that are associated with AD and related dementias (ADD). However, genetic research in ADD that focuses solely on European populations limits additional discoveries afforded by studying diverse cohorts. Including non-European populations in genetic research provides new opportunities to uncover ancestry-specific risk variants and loci, increase statistical discovery power, improve fine-mapping resolution to identify putative causal variants, and identify loci with heterogeneous effects across ancestry groups.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2267, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Implementing existing ancestry-aware or heterogeneity penalizing meta-regression approaches have proven powerful at deconvoluting the genetic architecture of other phenotypes across populations. We leveraged such techniques, layering existing diverse data on top of more extensive European-derived data, to facilitate discovery of novel ADD risk loci. Here we report the results of our multi-ancestry genome-wide meta-analysis of the largest publicly available ADD GWAS from individuals of European, East Asian, and African American ancestry, and an additional GWAS of Caribbean Hispanic individuals using previously reported genotype data since those summary statistics were not available. Using a meta-regression approach implemented in MR-MEGA, we demonstrate improved fine-mapping at several known ADD loci and estimate the extent to which heterogeneity at these loci is attributable to genetic ancestry. This study highlights the utility of multi-ancestry analyses to further our understanding of disease biology and reduce health disparities in research by nominating novel loci and characterizing genetic differences across populations.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 3411, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_1\"}, \"text\": \"Results\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 3419, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Data included in this study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 3447, \"infons\": {\"file\": \"41380_2023_2089_Fig1_HTML.jpg\", \"id\": \"Fig1\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Study Design.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 3461, \"infons\": {\"file\": \"41380_2023_2089_Fig1_HTML.jpg\", \"id\": \"Fig1\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Outline of multi-ancestry meta-analysis procedure and downstream analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 3536, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Our multi-ancestry meta-analysis included a total of 54,233 AD cases, 46,828 proxy AD and related dementia (proxy-ADD) cases, and 543,127 controls (Fig.\\u00a01 and Table\\u00a0S1). Detailed information about the existing GWAS summary statistics used in this report are described elsewhere. In brief, the most recent publicly available ADD GWAS includes 39,106 clinically diagnosed AD cases, 46,828 proxy-ADD cases (defined as having a parent with AD/dementia) and 401,577 controls of European ancestry. FinnGen data from Release 6 includes 7329 AD cases and 131,102 controls free of any neurological disorder. We also included the largest publicly available AD GWAS of African American (2748 cases and 5222 controls) and East Asian (3962 cases and 4074 controls) populations and an additional GWAS including 1095 cases and 1179 controls of Caribbean Hispanic ancestry. Select SNPs from the Gwangju Alzheimer\\u2019s & Related Dementias (GARD) East Asian cohort (1119 cases and 1172 controls) were used to assess East Asian risk at our novel loci post-hoc since these SNPs were not present in the discovery East Asian dataset from Shigemizu et al. used in our meta-GWAS. In this study, we considered significant variants as passing the standard p value threshold of 5 \\u00d7 108, consistent with most GWAS meta-analyses and used previously in other multi-ancestry studies. Our analysis included only variants that passed quality control and with a minor allele frequency >1% in a minimum of three datasets to accurately quantify heterogeneity, effectively reducing the number of potential haplotypes and tests.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 5129, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Meta-GWAS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 5139, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Association summary statistics from all five datasets, representing four super populations, were aggregated via fixed and random effects models implemented in PLINK v1.9 and a multi-ancestry meta-regression implemented in MR-MEGA (Table\\u00a0S1). A fixed effect analysis was conducted in conjunction with random effects in PLINK as it is the standard choice for many GWAS meta-analyses. Since the fixed effect analysis did not identify any additional novel loci, we focused on the random effects and MR-MEGA results as these methods are generally more appropriate for multi-ancestry studies. In particular, MR-MEGA was specifically designed for multi-ancestry meta-analyses and random effects models penalize heterogeneity in their construction of effect estimates, allowing these estimates to be more generalizable across global populations. We did not observe any genomic inflation in these analyses after excluding rare variants (MAF\\u2009<\\u20091% per study) and correcting for case-control imbalance (Table\\u00a0S1 and Fig.\\u00a0S1). Chromosome 19 was also excluded from genomic inflation estimates to avoid bias from the APOE region. Association results from the random effects and MR-MEGA meta-analyses were moderately concordant for SNPs without heterogeneity (I2\\u2009=\\u20090, R2\\u2009=\\u20090.6). Our study also demonstrated that MR-MEGA is advantageous for SNPs with heterogeneous allelic effects (Fig.\\u00a0S2). Results from all meta-analyses (MR-MEGA, random effects, and fixed effect) can be found in Table\\u00a0S2a\\u2013c.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 6637, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"A total of 68 loci reached genome-wide significance (P\\u2009<\\u20095 \\u00d7 10\\u22128) in the fixed effect, random effects or MR-MEGA meta-analyses. While most of these loci overlapped all analyses, 1 was only significant using MR-MEGA (JAZF1), 1 using random effects (KANSL1), and 5 using the fixed effect model (ADAMTS1, MAF, PLEKHA1, TSPOAP1, and UMAD1) (Table\\u00a0S2a\\u2013c). 66 of these loci overlapped previously established genomic regions associated with ADD (see \\u201cDefining associated loci\\u201d in the \\u201cOnline Methods\\u201d). Our analysis may have been underpowered to detect the remaining loci without the replication summary statistics from Bellenguez et al. as only the discovery phase statistics from that study were available.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 7358, \"infons\": {\"file\": \"41380_2023_2089_Fig2_HTML.jpg\", \"id\": \"Fig2\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Summary of multi-ancestry meta-analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 7399, \"infons\": {\"file\": \"41380_2023_2089_Fig2_HTML.jpg\", \"id\": \"Fig2\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"a The Manhattan plot for the random effects meta-analysis P values, truncated at \\u2212log10(P)\\u2009<\\u200950. An orange dot indicates that the lead SNP at a locus reached a P value < 5 \\u00d7 10-8, while a red dot indicates a P value < 5 \\u00d7 10\\u22129. b, c The corresponding local association plots for the two loci of interest and\\u00a0forest plots summarizing the effect estimates per ancestry group for lead SNPs at the two loci of interest. Lead SNPs from both novel loci were absent in the East Asian dataset by Shigemizu et al. used for discovery. Abbreviations - MR MR-MEGA, RE Random effects, FE Fixed effect, EUR European, EUR (FIN) Finnish European, AFR African American, CAR HISP Caribbean Hispanic.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 8092, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Summary of novel loci.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 8115, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"hsides\\\" rules=\\\"groups\\\"><thead><tr><th>Locus</th><th>SNP</th><th>Chromosome</th><th>Position</th><th>Effect allele</th><th>Reference allele</th><th>P, MR-MEGA</th><th>P, ancestry heterogeneity</th><th>P, random effects</th><th>beta, random effects</th><th>SE, random effects</th><th>I2</th><th>Mean effect allele frequency</th><th>Minimum effect allele frequency</th><th>Maximum effect allele frequency</th></tr></thead><tbody><tr><td><italic>VWA5B2</italic></td><td>rs9837978</td><td>3</td><td>183936947</td><td>G</td><td>A</td><td>2.32E\\u221207</td><td>5.96E\\u221201</td><td>3.75E\\u221208</td><td>\\u22120.053</td><td>0.009</td><td>0.000</td><td>0.287</td><td>0.171</td><td>0.450</td></tr><tr><td><italic>TRANK1</italic></td><td>rs9867455</td><td>3</td><td>36953424</td><td>A</td><td>T</td><td>7.33E\\u221208</td><td>1.18E\\u221201</td><td>3.49E\\u221208</td><td>\\u22120.042</td><td>0.008</td><td>0.000</td><td>0.405</td><td>0.326</td><td>0.503</td></tr></tbody></table>\\n\"}, \"text\": \"Locus\\tSNP\\tChromosome\\tPosition\\tEffect allele\\tReference allele\\tP, MR-MEGA\\tP, ancestry heterogeneity\\tP, random effects\\tbeta, random effects\\tSE, random effects\\tI2\\tMean effect allele frequency\\tMinimum effect allele frequency\\tMaximum effect allele frequency\\t \\tVWA5B2\\trs9837978\\t3\\t183936947\\tG\\tA\\t2.32E\\u221207\\t5.96E\\u221201\\t3.75E\\u221208\\t\\u22120.053\\t0.009\\t0.000\\t0.287\\t0.171\\t0.450\\t \\tTRANK1\\trs9867455\\t3\\t36953424\\tA\\tT\\t7.33E\\u221208\\t1.18E\\u221201\\t3.49E\\u221208\\t\\u22120.042\\t0.008\\t0.000\\t0.405\\t0.326\\t0.503\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 8583, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We additionally identified two independent ADD risk loci on chromosome 3 near TRANK1 (rs9867455; PRE\\u2009=\\u20093.49 \\u00d7 10\\u22128, \\u03b2RE\\u2009=\\u2009\\u22120.0424, I2\\u2009=\\u20090) and VWA5B2 (rs9837978; PRE\\u2009=\\u20093.75 \\u00d7 10\\u22128, \\u03b2RE\\u2009=\\u2009\\u22120.0526, I2\\u2009=\\u20090) that are outside of the maximal linkage disequilibrium (LD) boundary for any known AD risk loci (Fig.\\u00a02). These two loci were identified using both the fixed and random effects models. These loci also showed similar P values using MR-MEGA, but did not reach genome-wide significance using this method (Table\\u00a01). The association signals are primarily driven by the European-focused study by Bellenguez et al., where they were also sub-significant (P-values of 6.95 \\u00d7 10\\u22127 for TRANK1 and 1.17 \\u00d7 10\\u22126 for VWA5B2, respectively). We did not identify any additional novel loci using MR-MEGA (Fig.\\u00a0S12a).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 9430, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Since the lead SNPs for these potential novel loci were absent in the East Asian dataset from Shigemizu et al. used for initial discovery, we attempted to test these SNPs in an independent East Asian ancestry dataset. Genome-wide data for this cohort was not available to include in the meta-analysis at the time of publication. We observed an association at P\\u2009<\\u20090.05 at VWA5B2-rs9837978 (P\\u2009=\\u20090.048, \\u03b2\\u2009=\\u20090.204) in the GARD cohort (n\\u2009=\\u20092,291). The direction of effect was not consistent with the direction seen in the other populations included in the discovery GWAS, suggesting the effect of this locus may be heterogeneous across populations, but more extensive testing in East Asian populations must be performed for confirmation. We were unable to test the association at TRANK1-rs9867455 in the GARD replication cohort since this SNP was not included in their GWAS and an LD proxy SNP was not available.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 10355, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Gene prioritization for novel loci\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 10390, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Using public expression quantitative trait locus (eQTL) evidence from Open Targets and multi-ancestry brain eQTL summary data, we assessed whether TRANK1-rs9867455 and VWA5B2-rs9837978 are associated with the expression of nearby genes. Open Targets reported rs9867455 as a significant eQTL (P\\u2009<\\u20091 \\u00d7 10\\u22126) for LRRFIP2, ITGA9, GOLGA4, MLH1, and TRANK1 across blood or other tissues. LRRFIP2, GOLGA4 and TRANK1 were also nominated in the multi-ancestry brain eQTL data. Open Targets reported rs9837978 as a significant eQTL for AP2M1, ABCF3, VWA5B2, ALG3, ABCC5, DVL3, and CLCN2. AP2M1, as well as two additional genes (EIF2B5 and ECE2) were nominated in the multi-ancestry brain eQTL data.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 11086, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"To prioritize susceptibility genes with expression effects on ADD risk, we performed summary-based Mendelian Randomization (SMR) to infer whether expression of the eQTL-nominated genes is causal for AD. More details regarding the purpose and methods used to perform SMR can be found in the \\u201cFunctional inferences\\u201d section of the \\u201cOnline Methods\\u201d. At the TRANK1-rs9867455 locus, TRANK1, LRRFIP2, GOLGA4, and ITGA9 were significant in our SMR results for affecting AD risk via expression across multiple tissue types. The strongest associations in cortex tissue were seen with TRANK1 and LRRFIP2. The GWAS signal at the TRANK1-rs9867455 locus colocalized most strongly with TRANK1 expression in cortex tissue (R2\\u2009=\\u20090.52; Fig.\\u00a0S3). At the VWA5B2-rs9837978 locus, VWA5B2, AP2M1, ABCF3, ALG3, EIF2B5, DVL3, CLCN2, ABCC5 were significant in our SMR results. The strongest associations in brain tissues were seen in ABCF3, ALG3 and EIF2B5, although colocalization between the GWAS signal and these eQTLS were not very strong (R2\\u2009<\\u20090.5; Fig.\\u00a0S4). For more details on directionality of these associations, see Table\\u00a0S3.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 12216, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Fine-mapping\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 12229, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Fine-mapping results for all SNPs with a posterior probability (PP)\\u2009>\\u20090.8.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 12308, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"hsides\\\" rules=\\\"groups\\\"><thead><tr><th>Locus</th><th>SNP</th><th>Chromosome</th><th>Position</th><th>Effect allele</th><th>Reference allele</th><th>Novel fine-mapping</th><th>Nsnps_in_credible_set</th><th>Posterior probability</th><th>P, MR-MEGA</th><th>P, ancestry heterogeneity</th><th>chisq, ancestry (% total)</th><th>I2</th></tr></thead><tbody><tr><td><italic>BIN1</italic></td><td>rs6733839</td><td>2</td><td>127892810</td><td>T</td><td>C</td><td>No</td><td>1</td><td>1</td><td>5.74E\\u221292</td><td>3.13E\\u221201</td><td>14.270</td><td>58.010</td></tr><tr><td><italic>INPP5D</italic></td><td>rs10933431</td><td>2</td><td>233981912</td><td>C</td><td>G</td><td>No</td><td>1</td><td>1</td><td>1.63E\\u221216</td><td>7.90E\\u221202</td><td>41.675</td><td>59.470</td></tr><tr><td><italic>RHOH</italic></td><td>rs2245466</td><td>4</td><td>40198846</td><td>C</td><td>G</td><td>Yes</td><td>1</td><td>1</td><td>3.82E\\u221209</td><td>9.31E\\u221201</td><td>0.128</td><td>49.160</td></tr><tr><td><italic>CTSB</italic></td><td>rs1065712</td><td>8</td><td>11702122</td><td>C</td><td>G</td><td>Yes</td><td>1</td><td>1</td><td>6.42E\\u221209</td><td>7.60E\\u221201</td><td>4.551</td><td>0.000</td></tr><tr><td><italic>ECHDC3</italic></td><td>rs7912495</td><td>10</td><td>11718713</td><td>C</td><td>G</td><td>No</td><td>1</td><td>1</td><td>1.35E\\u221213</td><td>6.02E\\u221201</td><td>48.894</td><td>0.000</td></tr><tr><td><italic>SLC24A4</italic></td><td>rs12590654</td><td>14</td><td>92938855</td><td>A</td><td>G</td><td>Yes</td><td>32</td><td>0.910</td><td>6.56E\\u221216</td><td>3.37E\\u221202</td><td>63.950</td><td>57.450</td></tr><tr><td><italic>APH1B</italic></td><td>rs117618017</td><td>15</td><td>63569902</td><td>T</td><td>C</td><td>No</td><td>2</td><td>0.965</td><td>3.63E\\u221220</td><td>4.98E\\u221201</td><td>13.608</td><td>10.950</td></tr><tr><td><italic>FAM157C/PRDM7</italic></td><td>rs56407236</td><td>16</td><td>90170095</td><td>A</td><td>G</td><td>Yes</td><td>1</td><td>1</td><td>7.55E\\u221213</td><td>1.33E\\u221201</td><td>63.528</td><td>15.600</td></tr><tr><td><italic>GRN</italic></td><td>rs5848</td><td>17</td><td>42430244</td><td>T</td><td>C</td><td>Yes</td><td>1</td><td>1</td><td>1.87E\\u221215</td><td>3.22E\\u221201</td><td>20.238</td><td>38.050</td></tr><tr><td><italic>ABCA7</italic></td><td>rs12151021</td><td>19</td><td>1050874</td><td>A</td><td>G</td><td>No</td><td>1</td><td>1</td><td>3.62E\\u221232</td><td>3.23E\\u221201</td><td>34.721</td><td>0.000</td></tr></tbody></table>\\n\"}, \"text\": \"Locus\\tSNP\\tChromosome\\tPosition\\tEffect allele\\tReference allele\\tNovel fine-mapping\\tNsnps_in_credible_set\\tPosterior probability\\tP, MR-MEGA\\tP, ancestry heterogeneity\\tchisq, ancestry (% total)\\tI2\\t \\tBIN1\\trs6733839\\t2\\t127892810\\tT\\tC\\tNo\\t1\\t1\\t5.74E\\u221292\\t3.13E\\u221201\\t14.270\\t58.010\\t \\tINPP5D\\trs10933431\\t2\\t233981912\\tC\\tG\\tNo\\t1\\t1\\t1.63E\\u221216\\t7.90E\\u221202\\t41.675\\t59.470\\t \\tRHOH\\trs2245466\\t4\\t40198846\\tC\\tG\\tYes\\t1\\t1\\t3.82E\\u221209\\t9.31E\\u221201\\t0.128\\t49.160\\t \\tCTSB\\trs1065712\\t8\\t11702122\\tC\\tG\\tYes\\t1\\t1\\t6.42E\\u221209\\t7.60E\\u221201\\t4.551\\t0.000\\t \\tECHDC3\\trs7912495\\t10\\t11718713\\tC\\tG\\tNo\\t1\\t1\\t1.35E\\u221213\\t6.02E\\u221201\\t48.894\\t0.000\\t \\tSLC24A4\\trs12590654\\t14\\t92938855\\tA\\tG\\tYes\\t32\\t0.910\\t6.56E\\u221216\\t3.37E\\u221202\\t63.950\\t57.450\\t \\tAPH1B\\trs117618017\\t15\\t63569902\\tT\\tC\\tNo\\t2\\t0.965\\t3.63E\\u221220\\t4.98E\\u221201\\t13.608\\t10.950\\t \\tFAM157C/PRDM7\\trs56407236\\t16\\t90170095\\tA\\tG\\tYes\\t1\\t1\\t7.55E\\u221213\\t1.33E\\u221201\\t63.528\\t15.600\\t \\tGRN\\trs5848\\t17\\t42430244\\tT\\tC\\tYes\\t1\\t1\\t1.87E\\u221215\\t3.22E\\u221201\\t20.238\\t38.050\\t \\tABCA7\\trs12151021\\t19\\t1050874\\tA\\tG\\tNo\\t1\\t1\\t3.62E\\u221232\\t3.23E\\u221201\\t34.721\\t0.000\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 13287, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"Fine-mapped SNPs were considered novel if they were not previously fine-mapped with PP\\u2009>\\u20090.8 in two recent European-focused studies by Wightman et al. and Schwartzentruber et al..\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 13471, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"A total of nine loci outside of the APOE, MAPT, and major histocompatibility complex (MHC) regions were fine-mapped to a credible set of \\u2264 2 SNPs with a combined posterior probability (PP) of 99% (Table\\u00a02 and Figs.\\u00a0S5, 6). The MHC and MAPT regions were excluded from fine-mapping due to a complex haplotype structure across populations and known haplotype inversions, respectively. Five of these loci were previously fine-mapped with PP\\u2009>\\u20090.8 in large GWAS of European populations (Fig.\\u00a0S5; BIN1-rs6733839; INPP5D-rs10933431; ECHDC3-rs7912495; APH1B-rs117618017; ABCA7-rs12151021). Four additional ADD loci with 1\\u20132 variants in their 99% credible sets have not been previously fine-mapped (Fig.\\u00a0S6a-b, d-e; RHOH-rs2245466; CTSB-rs1065712; FAM157C/PRDM7-rs56407236; GRN-rs5848). Interestingly, GRN is also a candidate risk locus for frontotemporal dementia (FTD) and Parkinson\\u2019s disease (PD). More than 70 pathogenic variants in GRN have been linked to familial FTD. While GRN mutations do exist in sporadic forms of FTD, we were unable to find publicly available FTD GWAS summary statistics with a significant association at this locus. Using publicly available PD GWAS summary statistics, we compared the regional genetic correlations between PD and ADD at both GRN and CTSB, another candidate risk locus for PD. LocusCompare plots show low genetic correlation between ADD and PD at both loci (GRN, R2\\u2009=\\u20090.36; CTSB, R2\\u2009=\\u20090.0048), which may indicate that distinct causal variants drive the associations (Fig.\\u00a0S7). One additional locus with a credible set size >2 was fine-mapped to a single SNP with PP\\u2009>\\u20090.8 (Fig.\\u00a0S6c; SLC24A4-rs12590654, PP\\u2009=\\u20090.91, n\\u2009=\\u200932 in 99% credible set). In addition, two SNPs with a PP\\u2009\\u2265\\u20090.3 were annotated as missense variants (Fig.\\u00a0S8; MS4A6A-rs7232, PP\\u2009=\\u20090.54, n\\u2009=\\u20094 in 99% credible set; SHARPIN-rs34674752, PP\\u2009=\\u20090.30, n\\u2009=\\u20095 in 99% credible set). Notably, our fine-mapping analysis did not replicate SORL1-rs11218343, which has been previously fine-mapped with a PP\\u2009>\\u20090.999 in two large European studies, likely due to a different regional architecture in the East Asian population as has been previously reported (Fig.\\u00a0S9). All 99% credible sets are provided in Table\\u00a0S4.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 15738, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Heterogeneity analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 15761, \"infons\": {\"file\": \"41380_2023_2089_Fig3_HTML.jpg\", \"id\": \"Fig3\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Graphical summary of heterogeneity at ADD genetic risk loci.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 15822, \"infons\": {\"file\": \"41380_2023_2089_Fig3_HTML.jpg\", \"id\": \"Fig3\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Lead SNPs were derived from MR-MEGA using maximal LD blocks, apart from APOE rs429358 and rs7412. Both APOE SNPs were absent in summary statistics from the most recent European-ancestry ADD GWAS. Aggregate effects were estimated using a random effects model since MR-MEGA assumes that effects differ across populations. Allelic effect heterogeneity that is attributable to genetic ancestry was estimated using Cochran\\u2019s Q statistics for ancestral and residual heterogeneity from the meta-regression (\\u201cOnline Methods\\u201d).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 16347, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We observed significant heterogeneity (I2\\u2009>\\u200930%) at 19 of the 48 loci that reached genome-wide significance (P\\u2009<\\u20095 \\u00d7 10\\u22128) in MR-MEGA (Table\\u00a0S2a), which does not include the additional loci (including 2 novel) identified by the fixed and random effects analyses (Table\\u00a0S2b, c). Several factors can account for the observed heterogeneity, such as differences in study design, geographical region, and diagnostic accuracy. We estimated the proportion of heterogeneity that is attributable to genetic ancestry using MR-MEGA (see \\u201cAssessment of allelic effect heterogeneity\\u201d in the \\u201cOnline Methods\\u201d) and observed that at least 50% of the heterogeneity was attributable to genetic ancestry at 10 of these loci (Figs.\\u00a03 and\\u00a0S10). We also assessed heterogeneity at lead SNPs from the most recent European GWAS and found that 37% of the lead SNPs tested presented significant heterogeneity (I2\\u2009>\\u200930%), of which 48% were primarily attributable to ancestry (Table\\u00a0S5). Five of the fine-mapped SNPs also showed significant heterogeneity (I2\\u2009>\\u200930%), of which only SLC24A4 showed heterogeneity that was primarily attributable to genetic ancestry (Table\\u00a02).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 17524, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"The genetic polymorphisms rs7412 and rs429358 that form the APOE e2/e3/e4 alleles presented very different allelic heterogeneity. Consistent with previous studies, we observed an attenuated signal at APOE-rs429358, which determines the APOE-e4 allele, among the cohorts of African Americans and Caribbean Hispanics (Fig.\\u00a0S11a). APOE-rs429358 had the highest heterogeneity (I2\\u2009=\\u200996.54) of the SNPs tested with ~42% attributable to genetic ancestry, although these polymorphisms were not available to test in the most recent European-ancestry AD GWAS. In contrast, APOE-rs7412, which determines the APOE-e2 allele, did not present any heterogeneity (Fig.\\u00a0S11b; I2\\u2009=\\u20090). Complementary to standard GWAS association tests, we also generated P values representing heterogeneity of effect estimates attributable to genetic ancestry in the multi-ancestry meta-regression and observed the strongest signal near APOE (Fig.\\u00a0S12). Additionally, we observed strong evidence of ancestry-related heterogeneity (PHET\\u2009<\\u20091e\\u22126) near SORL1, as well as PAPOLG, AC026202.5, and snoU13 which did not meet genome-wide significance in the association results. LocusZoom and beta-beta plots of these loci suggest that non-European populations primarily drive these association signals, and there are likely discordant effects across populations (Fig.\\u00a0S13).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 18873, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Polygenic risk scoring\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 18896, \"infons\": {\"file\": \"41380_2023_2089_Fig4_HTML.jpg\", \"id\": \"Fig4\", \"section_type\": \"FIG\", \"type\": \"fig_title_caption\"}, \"text\": \"Graphical summary of genetic risk scores.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 18938, \"infons\": {\"file\": \"41380_2023_2089_Fig4_HTML.jpg\", \"id\": \"Fig4\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"These genetic risk scores were derived from multi-ancestry and ancestry-specific risk estimates, then applied to an admixed Colombian cohort to evaluate significance and predictive power. The European-based (EUR) PRS was derived from a fixed effect meta-analysis of the summary statistics from Bellenguez et al. and FinnGen used in the meta-GWAS. a, b The maximal AUCs for each genetic risk score with color coding to delineate the source of the risk estimates for scores excluding and then including APOE-e4 variants. P value thresholds that correspond to the maximal AUCs are shown in Table\\u00a0S6.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 19536, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We tested the performance of the multi-ancestry fixed and random effects models and each of the GWAS from single ancestral populations in a Colombian cohort of AD cases (n\\u2009=\\u2009281) and neurologically normal controls (n\\u2009=\\u200987). This cohort is an admixture of three ancestral populations, with European substructure making up the highest proportion of global ancestry (mean of 64%, SD\\u2009=\\u200915%), followed by Indigenous American (mean of 27%, SD\\u2009=\\u200911%), and African (mean of 9%, SD\\u2009=\\u200911%). Colombian samples were used to test PRS applicability as they were a population not represented in the meta-analyses. While the Caribbean Hispanic cohort included in the meta-analyses is also three-way admixed, this cohort likely has a lower Indigenous American proportion and higher African ancestry proportion than the Colombian cohort. Single-ancestry PRS performed worse than multi-ancestry random-effects-derived PRS in terms of area under a receiver operating characteristic curve (AUC). We observed maximal AUCs of 79% and 68% including and then excluding APOE variants in this population, and 75% and 63% for the European-derived PRS (Fig.\\u00a04; Table\\u00a0S6). Non-European AUCs tended to improve with increasing sample size (Fig.\\u00a04), suggesting that the composite score, combining ancestry-specific PRS by population weights, may have performed as well or better than the random-effects-derived PRS if the component GWASs from underrepresented populations were better powered.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 21019, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_1\"}, \"text\": \"Discussion\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 21030, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"We performed a large, genome-wide meta-analysis of ADD across five datasets, representing four super-populations. By leveraging data from multiple ancestry groups, we replicated 66 known ADD loci and identified two novel risk loci on chromosome 3 (Tables\\u00a0S2a\\u2013c). These novel loci reached genome-wide significance using the fixed and random effects models, but were sub-significant using MR-MEGA as this model is not as well-powered to detect loci with homogenous allelic effects.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 21513, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"The first novel locus identified in this study is near TRANK1, which encodes tetratricopeptide repeat and ankyrin repeat containing 1. TRANK1 is associated with DNA- and ATP-binding and DNA repair and is highly expressed in brain tissue. Previously, TRANK1 has been cited as a robust risk locus for both schizophrenia (SZ). and bipolar disorder (BD), although subtype analyses suggest that this signal is primarily driven by the most heritable subtype, BD I, which is genetically correlated with SZ. Notably, BD has also been shown to increase risk for AD, with the two sharing significant genetic overlap. LocusCompare plots show a modest correlation between our random effects meta-analysis of ADD with BD I (R2\\u2009=\\u20090.46) and SZ (R2\\u2009=\\u20090.23) GWAS at this locus (Figs.\\u00a0S14 and 15). We also observed moderate LD (R2\\u2009=\\u20090.44, 1000 Genomes EUR) between the lead SNP identified in our study (rs9867455) and the lead SNP identified in the BD I GWAS (rs9834970), indicating potential cross-disease overlap for this locus.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 22539, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Previous SMR analysis of the TRANK1 region in BD suggested that both TRANK1 and GOLGA4 may be susceptibility genes. TRANK1 expression is decreased in both BD and AD, and decreased expression of TRANK1 was found to alter the expression of genes related to neuronal development and differentiation. Altered neurogenesis has been implicated in human AD brains and AD rodent models. Previous studies have suggested that TRANK1 may be involved in blood brain barrier permeability changes and neuroinflammation, both of which may be relevant to neurodegeneration. Interestingly, Kunkle et al. nominated TRANK1 through a gene-based analysis conducted in an African American population, but not through the single SNP association testing included in this study. A combination of eQTL and SMR nominated TRANK1, LRRFIP2, GOLGA4, and ITGA9 as potential genes underlying this SNP association, with the strongest associations in cortex tissue seen with TRANK1 and LRRFIP2 (Table\\u00a0S3). LRRFIP2 encodes LRR binding FLII interacting protein 2, which regulates Toll-like receptor 4 (TLR4) and can downregulate the NLRP3 inflammasome. TLR4 can induce microglial amyloid-\\u03b2 clearance in the brain in early stages of AD but can later induce an inflammatory response, suggesting that disruptions to LRRFIP2 may affect AD pathology in patients.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 23863, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"The second locus is nearest to VWA5B2, which encodes von Willebrand factor A domain-containing protein 5B1. Von Willebrand factor (VWF) is a glycoprotein that facilitates blood clotting at areas of injury. High VWF is associated with short-term risk of dementia, possibly due to the increased risk of blood clots restricting blood flow in the brain. Interestingly, VWA5B2 was found to be downregulated in AD patients, and other variants in VWA5B2 have been linked to decreased mean hemoglobin concentration. Low hemoglobin, or anemia, has been linked to an increase in risk for AD. This information seems to further implicate the involvement of the vasculature system with AD, complementing previous studies such as those investigating traumatic brain injury. Whether VWA5B2 has biological implications on risk for AD needs to be further investigated. The lead variant rs9837978 does not lie within any of the nearby genes at this locus, but eQTL and SMR evidence for this variant nominated eight nearby genes including VWA5B2 (Table\\u00a0S3). The strongest SMR association in brain tissue is seen with ABCF3. The ABCF3 protein is a member of the ATP-binding cassette (ABC) superfamily, all of which transport a variety of substrates across intra- and extracellular barriers. Members of the ABC A subfamily, such as ABCA7 and ABCA1, have previously been nominated as AD risk genes. ABCF3 is a unique family member in that it lacks a transmembrane domain but has been nominated as a candidate of TLR signaling, similar to LRRFIP2. In addition to inducing inflammatory responses, TLRs can affect microglial activity, synaptic plasticity, and tau phosphorylation, providing additional evidence to their potential importance in AD pathology. Additionally, downregulation of ABCF3 has been associated with an increase in viral load after infection by a flavivirus, specifically the West Nile virus which has been linked to long-term neurological problems and dementia. It\\u2019s plausible that in the presence of viral infection, changes in ABCF3 expression may affect immune response and inflammation, two processes that play a role in the amyloid cascade hypothesis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 26020, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Future studies will be required to further disentangle the potential roles of the nominated genes in the context of ADD risk. Although genome-wide summary statistics were unavailable for replication, we attempted to replicate the lead SNP VWA5B2-rs9837978 in a small East Asian cohort. However, this SNP demonstrated an opposite direction of effect and the replication dataset was underpowered to detect an association (Fig.\\u00a0S16). Assuming a disease prevalence of 2%, MAF of 0.1642 (gnomAD v3.2.1 East Asian), and a nominal significance threshold of 0.05, we had ~80% power to detect genotype relative risks \\u22651.245 but our odds ratio for this allele, which is generally an overestimate of risk, was 1.05. Replication in larger and more diverse cohorts is warranted in future studies. Further, the disparity seen at points between the results on Open Targets, which consists of largely European data, and the multi-ancestry eQTL results for nominated genes also highlights the need for more multi-modal reference data including diverse ancestries. However, it is also possible that there could be different mechanisms underlying disease risk conferred by the implicated loci across different populations.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 27227, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Our study highlights the utility of multi-ancestry datasets at uncovering putative mechanisms that contribute to ADD. Fine-mapping at several known ADD loci was better resolved using the multi-ancestry meta-regression compared to previous efforts in European populations. For example, fine-mapping near RHOH, CTSB and FAM157C/PRDM7 nominated variants that are located in untranslated regions that were not well-resolved in European-focused studies. Variants in the 3\\u2032UTR region can impact translation or protein stability, and transcription binding can be impacted by variants in the 5\\u2032UTR region. Additionally, GRN-rs5848 is associated with circulating progranulin levels and decreased GRN expression has been implicated in several neurodegenerative diseases, including AD and FTD. In contrast to previous studies in European populations, the SORL1 locus was not resolved to a single putative causal SNP. Lead SNPs in both the European (SORL1-rs11218343) and East Asian (SORL1-rs117807585) GWAS are more common among East Asians compared to all other populations in the Genome Aggregation Database v2.1.1 (rs11218343: AFEAS\\u2009=\\u20090.30, AFEUR\\u2009=\\u20090.039; rs117807585: AFEAS\\u2009=\\u20090.22, AFEUR\\u2009=\\u20090.020). It is possible that alternative fine-mapping approaches that allow for multiple causal variants per locus will provide greater insight into the SORL1 locus.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 28595, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"At the MS4A gene cluster, multi-ancestry fine-mapping resolved the signal to a credible set of five variants, with a common missense variant (rs7232, PP\\u2009=\\u20090.54) and an intergenic variant nearest MS4A4A (rs1582763, PP\\u2009=\\u20090.45) that are in moderate LD (R2\\u2009=\\u20090.55, 1000 Genomes all populations) having the highest probability of causality (Fig.\\u00a0S8). MS4A4A and/or MS4A6A modulate soluble TREM2 (sTREM2) in cerebrospinal fluid (CSF), which is correlated with AD progression. Previous studies have shown that rs7232 is associated with MS4A6A gene expression and CSF sTREM2, while rs1582763 is a cis-eQTL for MS4A4A and MS4A6A. Conditional analysis of CSF sTREM2 levels in this region have pointed to two independent signals represented by rs1582763 and rs6591561 (MS4A4A p.M159V). Therefore, a fine-mapping approach that allows for multiple causal variants may be more appropriate for this region.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29500, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"In addition to highlighting genetic risk factors that are shared across populations, our results also highlight ADD loci with significant heterogeneity that may reflect variation in effect sizes, allele frequencies or interaction(s) with environmental risk factors that vary by ancestral group. For example, we observed the strongest evidence of heterogeneity at APOE-rs429358. Around 42% of the heterogeneity at this allele was attributable to genetic ancestry, while the remaining heterogeneity may reflect other sources of variation such as imputation accuracy since this allele is rarely assayed successfully on genotyping arrays. At JAZF1-rs67250450 and CLU-rs1532276, we observed the strongest evidence of ancestry-related heterogeneity, both of which are most common among individuals of East Asian ancestry and showed the strongest effects in this population (Fig.\\u00a03). We also observed significant ancestry-related heterogeneity at SORL1 and TREM2, which have been previously shown to harbor population-specific risk variants. Given that our analyses focused on common variation, the effects of rare heterogeneous variants (e.g. ABCA7-rs115550680, which has comparable effects to APOE-rs429358 among African Americans) may not have been fully captured.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 30762, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"While this study marks progress towards assessing genetic risk of ADD across multiple populations, we acknowledge several limitations. First, we recognize that the magnitude of clinical and pathological diversity among ADD cases is extensive. The diagnostic inaccuracy rate of AD likely differs across studies and populations both early-onset forms of AD and other pathologies such as frontotemporal and lewy body dementias. This clinicopathological heterogeneity is further exacerbated by the inclusion of proxy-ADD cases in the Bellenguez et al. study, a study which comprised the majority of European samples included in our analysis. While proxy-ADD cases may introduce more variability than a clinical or pathological diagnosis of AD, prior studies have demonstrated strong genetic correlation between proxy-ADD and AD (rg = 0.81), further supporting the use of this data. Additionally, despite the phenotypic heterogeneity in the Bellenguez et al. study, the utility of ADD GWAS are supported by genes with well-defined involvement in relevant molecular pathways. For example, the APP locus was first identified at genome-wide significance in Bellenguez et al. (P\\u2009=\\u20091.02 \\u00d7 10\\u22129) and we found a stronger level of significance in our random effects meta-analysis (P\\u2009=\\u20098.1 \\u00d7 10\\u221212, I2\\u2009=\\u20090.79). This locus is likely driven by underlying AD pathology due to its role in the formation of amyloid-\\u03b2 and its previous implications in both LOAD and EOAD. Although disease subtype-specific conclusions that can be made from our analysis are limited by the diagnostic criteria of the included GWAS, similar analyses can be applied as larger datasets with high phenotypic specificity become available.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 32476, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Additionally, although MR-MEGA is a useful tool for fine-mapping and ancestral heterogeneity estimation, the software requirements of population overlap (K\\u2009>\\u20093) often result in reduced variant sets after study level quality control. This can bias fine-mapping results as we reduce the potential resolution on local haplotypes, and usually necessitates the inclusion of at least one of the larger European-focused studies. In our case, previous European and Finnish studies served as the backbone of our meta-GWAS. We did not replicate previous fine-mapping at NCK2, TREM2 and RNF223 from European-focused studies since study level quality control included filtering for common (MAF\\u2009>\\u20091%), biallelic SNVs due to potentially poor imputation and general low power for rare variants across ancestral groups. We acknowledge that variants with a minor allele frequency <1% in one or more populations, as well as indels and structural variants, may contribute to the observed associations. While less stringent QC may have allowed us to detect more variants, we used a more conservative approach to accommodate the smaller sample size of non-European GWAS and the need for MR-MEGA SNPs to be present in at least four studies. Future work should include imputation using diverse reference panels from long read sequence data specific to ADD to improve genomic coverage and provide insights into structural variation that may be population specific.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 33925, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"In addition, the number of axes of genetic variation (T) in MR-MEGA is restricted to T\\u2009\\u2264\\u2009K-2, where K is the number of input GWAS. The East Asian GWAS from Shigemizu et al. used in our meta-GWAS tested less than half as many SNPs as the others (Table\\u00a0S1), limiting the meta-regression to a single axis of genetic variation (PC0) at SNPs that overlap the remaining GWAS (K\\u2009=\\u20094). Including a larger number of input GWAS from underrepresented populations will likely improve the heterogeneity estimates outlined in this study and are worth pursuing when such data become available.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34515, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"While our study is inclusive, due to data availability and the European-dominated nature of genetic research, European-ancestry individuals make up approximately 85% of cases and the discovery efforts here maintain a baseline level of Eurocentric bias. Despite this bias, our random-effects-derived PRS including APOE variation achieved a higher maximal AUC of 79% in an independent admixed Colombian cohort compared to 75% achieved by the European-based PRS. Additionally, while our novel method of creating a composite PRS model that leverages admixture percentages is a potentially promising approach for assessing ADD risk across ancestrally heterogeneous and/or admixed cohorts, its performance relies on sufficient sample sizes and global genetic representation. As larger scale GWAS for multiple continental \\u201csuper populations\\u201d continue to become available, we believe this method of tuning PRS to an individual\\u2019s genetic admixture could have utility in a precision medicine context. Reducing the Eurocentric bias in AD genetics research will require the harmonization and refining of diagnosis in non-European research sites that serve communities with unique cultural and logistic concerns for participation in research. Overall, our study provides a critical framework for future ADD meta-analyses. It is our hope to improve representation in ADD genetic studies in the future, increasing the balance between European and well-powered non-European cohorts.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 35988, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_1\"}, \"text\": \"Online methods\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 36003, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Existing GWAS studies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 36025, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Summary statistics from Bellenguez et al. 2022 were accessed through the National Human Genome Research Institute-European Bioinformatics Institute GWAS catalog under accession number GCST90027158 in May 2022. Summary statistics from FinnGen Release 6 were accessed at https://www.finngen.fi/en/access_results in April of 2022. Summary statistics from Kunkle et al. were accessed through NIAGADS (https://www.niagads.org/) under accession number NG00100 in April of 2022. Summary statistics from Shigemizu et al. were accessed through the National Bioscience Database Center (NBDC) at the Japan Science and Technology Agency (JST) at https://humandbs.biosciencedbc.jp/en/ through accession number hum0237.v1.gwas.v1 in April of 2022. All summary statistics were aligned to GRCh37 and cleaned to remove indels, multi-allelics and rare variants (MAF\\u2009<\\u20091%) prior to multi-ancestry analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 36917, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Caribbean Hispanic GWAS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 36941, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Data from the Columbia University Study of Caribbean Hispanics and Late Onset Alzheimer\\u2019s disease were accessed via application to dbGaP accession number phs000496.v1.p1 in April of 2022. Samples were filtered to keep unrelated individuals without missing values for AD affection status, age, study category, education, and a missing call rate <0.02. Principal component analysis was performed on a combined dataset of study subjects and HapMap was used as a reference to identify potential outliers. Controls with a family history of dementia were removed to ensure that potential proxy-ADD cases were not present in the control group. Variant QC included exclusion filters for monomorphic SNPs, variants with MAF\\u2009<1%, missingness rates >2%, sex differences in allelic frequency \\u22650.2 and heterozygosity >0.3, duplicate SNPs, Hardy\\u2013Weinberg Equilibrium (HWE) P value <1 \\u00d7 10\\u22124, and >1 discordant calls or Mendelian errors. All variants with a significant frequency mismatch (\\u03c72\\u2009>\\u2009300) with the TOPMed reference panel were removed prior to imputation.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 38008, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"As increasing age is the most significant risk factor for ADD, age matching is commonly used to control for differences in age distributions between cases and controls. While we did not perform case-control age matching to maximize sample size, the distribution of ages between cases and controls largely overlaps and we include age as a covariate in our GWAS analysis. A demographic summary table detailing age, APOE-e4 status, sex for the Caribbean Hispanic cohort is provided in Table\\u00a0S7.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 38501, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Using PLINK v1.9, we evaluated the association between AD and imputed genotypes via logistic regression on allele dosages with imputation quality >0.3, adjusting for sex, age (age at disease onset for cases, age at last evaluation for controls), education, study category, and the first 10 principal components (PCs). Study category denotes subcategories within the Caribbean Hispanic dataset (individuals are from the United States, Puerto Rico and the Dominican Republic) and is included to account for potential batch effects.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 39031, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Meta-analysis and fine-mapping\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 39062, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_3\"}, \"text\": \"Meta-analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 39076, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Three models were used to conduct multi-ancestry meta-analyses. Fixed effect and random effects models were performed using PLINK v1.9, while a separate analysis was performed using MR-MEGA v0.2. PLINK v1.9 was preferred over METAL due to its capacity to perform fixed and random effects analyses in parallel. A random effects model provides a more conservative framework which allows each study to have unique effects, as can be expected in different populations. MR-MEGA was also employed since it is well-powered to detect associations at loci with allelic heterogeneity. MR-MEGA models allelic effects as a function of axes of genetic variation that are derived from the input GWAS summary statistics. This method can result in reduced variant sets since it requires that variants have sufficient overlap between the input datasets (K\\u2009>\\u20093), where K is the number of input GWAS, in contrast to both the fixed and random effects models implemented in PLINK v1.9 which were limited to K\\u2009>\\u20092 to accurately quantify heterogeneity.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 40114, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The European and Finnish European GWAS were included separately in all multi-ancestry meta-analyses to account for finer-scale differences in allele frequencies. To determine the optimal number of PCs needed to distinguish cis- and multi-ancestry ADD summary statistics using MR-MEGA, we visually inspected pairwise PC plots generated using all five GWAS referenced in Table\\u00a0S1. We observed adequate separation between the Caribbean Hispanic, European, African American, and East Asian GWAS using the first two meta-regression PCs (Fig.\\u00a0S17). To increase the variant set, we also ran MR-MEGA separately for each combination of four input GWAS. A single axis of genetic variation (T\\u2009=\\u20091) was included in this analysis since this is the maximum allowable given the constraints of the model (T\\u2009\\u2264\\u2009K\\u22122). Summary statistics were aggregated to maximize the effective sample size for each variant.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 41018, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_3\"}, \"text\": \"Defining associated loci\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 41043, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"FUMA was used to find maximal LD blocks around loci that reached P\\u2009<\\u20095 \\u00d7 10\\u22128 in the specified multi-ancestry meta-analysis. LD blocks of independent significant SNPs (R2\\u2009>\\u20090.3, 1000 Genomes all populations) were merged into a single genomic locus if the distance between LD blocks was less than 250\\u2009kb. These loci were compared to the previous GWAS by Bellenguez et al. and Open Targets to assess whether these regions were known to be associated with ADD. These genomic intervals were also used as inputs for fine-mapping as described below.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 41600, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_3\"}, \"text\": \"Fine-mapping\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 41613, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Fine-mapping was performed using approximate Bayes\\u2019 factors in favor of association from the meta-regression model implemented in MR-MEGA. Posterior probabilities (PP) were calculated using single-SNP Bayes factors and credible sets were generated for each locus (with genomic intervals defined as described above) until the cumulative PP exceeded 99%. All SNPs in the 99% credible sets were annotated with VEP (http://grch37.ensembl.org/Homo_sapiens/Tools/VEP) using default criteria to select one block of annotation per variant (Table\\u00a0S4).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 42159, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Assessment of allelic effect heterogeneity\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 42202, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Allelic effect heterogeneity between studies was assessed for all lead SNPs reaching genome-wide significance (P\\u2009<\\u20095 \\u00d7 10\\u22128) in the meta-regression, implemented in MR-MEGA. The meta-regression model derives axes of genetic variation from pairwise allele frequency differences between the input GWAS. Heterogeneity is then partitioned into (1) ancestry-related heterogeneity that is correlated with the axes of genetic variation and (2) residual heterogeneity that is likely due to other factors such as diagnostic accuracy, study design (e.g. covariate adjustments, phenotype definition, imputation quality, inclusion of proxy-ADD cases) and/or geographical region. Total heterogeneity at each index SNP was quantified using the I2 statistic in PLINK v1.9 to avoid bias due to sample size for SNPs not tested in the large European studies. The I2 statistic describes the proportion of variation in effect estimates that is due to heterogeneity. We considered SNPs with an I2\\u2009>\\u200930% as having significant heterogeneity since this suggests at least moderate variation in allelic effects. The percentage of this heterogeneity that is attributable to genetic ancestry was then calculated using Cochran\\u2019s Q statistics for ancestral and residual heterogeneity from the meta-regression (Eq.\\u00a01; ANC: ancestry, RESID: residual).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43535, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Functional inferences\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43557, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"To prioritize genes underlying the two novel loci, we first looked at public eQTL data to determine whether the GWAS-identified lead variants are eQTLs for nearby genes. This allowed us to cast a wide net of potential regional genes of interest. We employed Open Targets for this effort, which shares eQTL results for variants from blood, brain, and a wide array of tissues from multiple public eQTL datasets. We additionally investigated a multi-ancestry brain eQTL dataset which was not available on Open Targets at the time of publication. We considered the lead variants as significant eQTLs for a gene if they passed the significance threshold of P\\u2009<\\u20091 \\u00d7 106, which has been shown to correspond to a genome-wide false discovery rate (FDR) of 5%, although we do acknowledge this may be overly conservative in our regional analyses.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44398, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Once we had nominated potential genes for which our lead variants were significant eQTLs, we used summary-based Mendelian Randomization (SMR) to make functional inferences as to whether the disease risk SNPs in these regions mediate gene expression. We integrated summary-level data from the most recent ADD GWAS with data from multiple eQTL studies in different tissues using the SMR method. SMR uses summary statistics to determine if an exposure is associated with a trait through a shared casual variant. MR can be used to mimic a randomized controlled trial, as having a variant that increases or decreases expression of a gene may be comparable to life-long treatment with a drug targeting the encoded protein of that gene. For example, if SNP A affects gene B expression (the exposure), and SNP A is also associated with ADD risk (the outcome), you can infer the causal effect of the expression of gene B on ADD risk.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45323, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We limited our results to the genes that were prioritized by our eQTL search and considered a gene significant for expression effect on a disease if it passed an FDR-adjusted SMR significance threshold of P\\u2009<\\u20090.05 and a HEIDI threshold of P\\u2009>\\u20090.01. Filtering for a HEIDI P-value of this magnitude helps to remove associations that are likely due to polygenicity and have violated the central assumptions of SMR. Finally, we assessed the colocalization between the SMR-nominated genes in brain tissues and the multi-ancestry random effects GWAS using LocusCompare.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45895, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Polygenic risk scoring\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45918, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_3\"}, \"text\": \"PRS application cohort\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45941, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Whole genomes from the Colombian population were accessed from \\u201cThe Admixture and Neurodegeneration Genomic Landscape\\u201d (TANGL) study and quality controlled as previously described. The TANGL cohort was further quality controlled in PLINK v1.9 to remove carriers of pathogenic variants for mendelian forms of dementia, as well as related individuals for a final cohort of 281 cases and 87 controls.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46343, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_3\"}, \"text\": \"Pre-PRS variant alignment\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46369, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Base summary statistics were pruned with the MungeSumStats R package to remove multiallelic variants, align reference alleles to build GRCh37, and adjust weights for the appropriate reference alleles. The target TANGL cohort was also filtered to keep only biallelic variants and aligned to the same reference using PLINK v2.0.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46696, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_3\"}, \"text\": \"PRS method\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46707, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Polygenic risk score (PRS) analyses can be used to estimate an individual\\u2019s genetic liability to a phenotype by calculating the sum of risk allele effect size weights for an individual. Weights for the PRS were obtained from \\u03b2 estimates generated from multi-ancestry fixed and random effects meta-analyses as well as from individual ancestry summary statistics. PRS analyses were conducted using PRSice v2.3.5 including variants with minor allele frequency >5%, genotype missingness <10%, sample missingness <10%, and HWE P value <1 \\u00d7 10\\u22126. The APOE region (with ranges defined by FUMA as described previously) was excluded prior to variant clumping. For PRS analyses including APOE, the genetic polymorphisms rs7412 and rs429358 were added to the QC\\u2019d summary statistics prior to variant clumping. \\u03b2 estimates for the APOE polymorphisms were not available in the Bellenguez et al. summary statistics and therefore European-ancestry estimates were taken from another recent ADD GWAS by Schwartzentruber et al..\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 47727, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Variants were clumped in each 500\\u2009kb window with the index SNP at the center, an r2 threshold of 0.3, and a clump P value threshold of 1. Sex, age, and the first 5 PCs were used as covariates in the PRS analysis. PCs were generated from non-imputed genotype data using FlashPCA. Variants with a MAF <1%, genotype missingness <10%, sample missingness <10%, and HWE P values <5 \\u00d7 106 were excluded using PLINK v1.9. The remaining variants were pruned with a 1000-kb window, a 10-SNP shift per window and an r2 threshold of 0.02 prior to PC calculation. PRS analysis was performed at select P-value thresholds to determine the best fit model (P\\u2009=\\u20095 \\u00d7 1010, 5 \\u00d7 10\\u22129, 5 \\u00d7 10\\u22128, 5 \\u00d7 10\\u22127, 5 \\u00d7 10\\u22126, 5 \\u00d7 10\\u22125, 5 \\u00d7 10\\u22124, 5 \\u00d7 10\\u22123, 5 \\u00d7 10\\u22122). To assess the performance of each model, receiver operator characteristic curves were created using the pROC library in R for the best fit model from each analysis as shown in Table\\u00a0S6. Since MR-MEGA does not provide standard effect estimates per SNP, an additional \\u201ccomposite\\u201d ROC curve was generated through a linear combination of each super population to provide a comparison to the conservative random-effects-based PRS model. Each PRS was weighted by its associated admixture population percentage, previously determined in the TANGL cohort for each individual (Eq.\\u00a02; AFR: African American, EUR: European (including Finnish), EAS: East Asian, NAT: Native American). Given the population history and similarities in haplotype structure between the East Asian and Native American populations, Native American admixture proportions were used to weight the East Asian PRS.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49376, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Supplementary information\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49402, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"footnote\"}, \"text\": \"Publisher\\u2019s note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49540, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"footnote\"}, \"text\": \"These authors contributed equally: Julie Lake, Caroline Warly Solsberg, Jennifer S. Yokoyama, Hampton L. Leonard.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49654, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_1\"}, \"text\": \"Supplementary information\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49680, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"paragraph\"}, \"text\": \"The online version contains supplementary material available at 10.1038/s41380-023-02089-w.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49772, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"title\"}, \"text\": \"Author contributions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49793, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"paragraph\"}, \"text\": \"Designed the study: JL, CWS, PH, LF, ABS, MAN, JSY, and HLL. Acquired and processed data: JL, CWS, JA-U, MBM, ZL, CXA, DV, SK, JG, SDP, LF, ABS, CB, MAN, JSY, and HLL. Analyzed data: JL, CWS, JJK, JA-U, MBM, ZL, CXA, DV, SK, JG, SDP, ABS, CB, MAN, JSY, and HLL. Interpreted data, writing and revising the first draft of the manuscript: All authors.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 50142, \"infons\": {\"section_type\": \"ACK_FUND\", \"type\": \"title\"}, \"text\": \"Funding\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 50150, \"infons\": {\"section_type\": \"ACK_FUND\", \"type\": \"paragraph\"}, \"text\": \"This work was supported in part by the Intramural Research Programs of the National Institute on Aging (NIA) part of the National Institutes of Health, Department of Health and Human Services; project numbers 1ZIA-NS003154, Z01-AG000949-02, ZO1-AG000535 and Z01-ES101986. JSY is supported by NIH-NIA R01 AG062588, R01 AG057234, P30 AG062422; NIH-NINDS U54 NS123985; the Rainwater Charitable Foundation; the Alzheimer\\u2019s Association; the Global Brain Health Institute; and the Mary Oakley Foundation. This research was supported by the Korea Brain Research Institute basic research program funded by the Ministry of Science and ICT (22-BR-03-05). Funding support for the \\u201cGenetic Consortium for Late Onset Alzheimer\\u2019s Disease\\u201d was provided through the Division of Neuroscience, NIA. The Genetic Consortium for Late Onset Alzheimer\\u2019s Disease includes a genome-wide association study funded as part of the Division of Neuroscience, NIA.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51093, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title\"}, \"text\": \"Data availability\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51111, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"paragraph\"}, \"text\": \"Summary statistics from this study will be available to browse and download via our collaboration with the Broad\\u2019s Neurodegenerative Disease Knowledge Portal (https://ndkp.hugeamp.org/dinspector.html?dataset=Lake2023_AD_Mixed).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51341, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title\"}, \"text\": \"Code availability\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51359, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"paragraph\"}, \"text\": \"Code is available on the CARD GitHub (https://github.com/NIH-CARD/MA_MA_meta).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51438, \"infons\": {\"section_type\": \"COMP_INT\", \"type\": \"title\"}, \"text\": \"Competing interests\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51458, \"infons\": {\"section_type\": \"COMP_INT\", \"type\": \"paragraph\"}, \"text\": \"KL, DV, HL and MAN\\u2019s participation in this project was part of a competitive contract awarded to Data Tecnica International LLC by the National Institutes of Health to support open science research. MAN also currently serves on the scientific advisory board for Clover Therapeutics and is an advisor to Neuron23 Inc. PH is an employee at Alector and owns stock and options. JSY serves on the scientific advisory board for the Epstein Family Alzheimer\\u2019s Research Collaboration.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51939, \"infons\": {\"section_type\": \"REF\", \"type\": \"title\"}, \"text\": \"References\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51950, \"infons\": {\"fpage\": \"181\", \"lpage\": \"4\", \"name_0\": \"surname:Tanzi;given-names:RE\", \"name_1\": \"surname:Bertram;given-names:L\", \"pub-id_doi\": \"10.1016/S0896-6273(01)00476-7\", \"pub-id_pmid\": \"11683989\", \"section_type\": \"REF\", \"source\": \"Neuron.\", \"type\": \"ref\", \"volume\": \"32\", \"year\": \"2001\"}, \"text\": \"New frontiers in Alzheimer\\u2019s disease genetics\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51998, \"infons\": {\"fpage\": \"iyaa046\", \"name_0\": \"surname:Holland;given-names:D\", \"name_1\": \"surname:Frei;given-names:O\", \"name_2\": \"surname:Desikan;given-names:R\", \"name_3\": \"surname:Fan;given-names:C-C\", \"name_4\": \"surname:Shadrin;given-names:AA\", \"name_5\": \"surname:Smeland;given-names:OB\", \"pub-id_doi\": \"10.1093/genetics/iyaa046\", \"pub-id_pmid\": \"33789345\", \"section_type\": \"REF\", \"source\": \"Genetics.\", \"type\": \"ref\", \"volume\": \"217\", \"year\": \"2021\"}, \"text\": \"The genetic architecture of human complex phenotypes is modulated by linkage disequilibrium and heterozygosity\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52109, \"infons\": {\"fpage\": \"4799\", \"name_0\": \"surname:Zhang;given-names:Q\", \"name_1\": \"surname:Sidorenko;given-names:J\", \"name_2\": \"surname:Couvy-Duchesne;given-names:B\", \"name_3\": \"surname:Marioni;given-names:RE\", \"name_4\": \"surname:Wright;given-names:MJ\", \"name_5\": \"surname:Goate;given-names:AM\", \"pub-id_doi\": \"10.1038/s41467-020-18534-1\", \"pub-id_pmid\": \"32968074\", \"section_type\": \"REF\", \"source\": \"Nat Commun\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2020\"}, \"text\": \"Risk prediction of late-onset Alzheimer\\u2019s disease implies an oligogenic architecture\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52196, \"infons\": {\"fpage\": \"412\", \"lpage\": \"36\", \"name_0\": \"surname:Bellenguez;given-names:C\", \"name_1\": \"surname:K\\u00fc\\u00e7\\u00fckali;given-names:F\", \"name_2\": \"surname:Jansen;given-names:IE\", \"name_3\": \"surname:Kleineidam;given-names:L\", \"name_4\": \"surname:Moreno-Grau;given-names:S\", \"name_5\": \"surname:Amin;given-names:N\", \"pub-id_doi\": \"10.1038/s41588-022-01024-z\", \"pub-id_pmid\": \"35379992\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"54\", \"year\": \"2022\"}, \"text\": \"New insights into the genetic etiology of Alzheimer\\u2019s disease and related dementias\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52282, \"infons\": {\"fpage\": \"1631\", \"lpage\": \"3\", \"name_0\": \"surname:Hou;given-names:K\", \"name_1\": \"surname:Bhattacharya;given-names:A\", \"name_2\": \"surname:Mester;given-names:R\", \"name_3\": \"surname:Burch;given-names:KS\", \"name_4\": \"surname:Pasaniuc;given-names:B\", \"pub-id_doi\": \"10.1038/s41588-021-00953-5\", \"pub-id_pmid\": \"34824480\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"53\", \"year\": \"2021\"}, \"text\": \"On powerful GWAS in admixed populations\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52322, \"infons\": {\"fpage\": \"102\", \"lpage\": \"13\", \"name_0\": \"surname:Kunkle;given-names:BW\", \"name_1\": \"surname:Schmidt;given-names:M\", \"name_2\": \"surname:Klein;given-names:H-U\", \"name_3\": \"surname:Naj;given-names:AC\", \"name_4\": \"surname:Hamilton-Nelson;given-names:KL\", \"name_5\": \"surname:Larson;given-names:EB\", \"pub-id_doi\": \"10.1001/jamaneurol.2020.3536\", \"pub-id_pmid\": \"33074286\", \"section_type\": \"REF\", \"source\": \"JAMA Neurol\", \"type\": \"ref\", \"volume\": \"78\", \"year\": \"2021\"}, \"text\": \"Novel Alzheimer disease risk loci and pathways in African American individuals using the African genome resources panel: a meta-analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52459, \"infons\": {\"fpage\": \"1524\", \"lpage\": \"32\", \"name_0\": \"surname:Blue;given-names:EE\", \"name_1\": \"surname:Arvr;given-names:H\", \"name_2\": \"surname:Mukherjee;given-names:S\", \"name_3\": \"surname:Wijsman;given-names:EM\", \"name_4\": \"surname:Thornton;given-names:TA\", \"pub-id_doi\": \"10.1016/j.jalz.2019.07.016\", \"pub-id_pmid\": \"31606368\", \"section_type\": \"REF\", \"source\": \"Alzheimers Dement\", \"type\": \"ref\", \"volume\": \"15\", \"year\": \"2019\"}, \"text\": \"Local ancestry at APOE modifies Alzheimer\\u2019s disease risk in Caribbean Hispanics\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52541, \"infons\": {\"fpage\": \"675\", \"lpage\": \"9\", \"name_0\": \"surname:Graham;given-names:SE\", \"name_1\": \"surname:Clarke;given-names:SL\", \"name_2\": \"surname:Wu;given-names:K-HH\", \"name_3\": \"surname:Kanoni;given-names:S\", \"name_4\": \"surname:Zajac;given-names:GJM\", \"name_5\": \"surname:Ramdas;given-names:S\", \"pub-id_doi\": \"10.1038/s41586-021-04064-3\", \"pub-id_pmid\": \"34887591\", \"section_type\": \"REF\", \"source\": \"Nature.\", \"type\": \"ref\", \"volume\": \"600\", \"year\": \"2021\"}, \"text\": \"The power of genetic diversity in genome-wide association studies of lipids\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52617, \"infons\": {\"fpage\": \"560\", \"lpage\": \"72\", \"name_0\": \"surname:Mahajan;given-names:A\", \"name_1\": \"surname:Spracklen;given-names:CN\", \"name_2\": \"surname:Zhang;given-names:W\", \"name_3\": \"surname:Ng;given-names:MCY\", \"name_4\": \"surname:Petty;given-names:LE\", \"name_5\": \"surname:Kitajima;given-names:H\", \"pub-id_doi\": \"10.1038/s41588-022-01058-3\", \"pub-id_pmid\": \"35551307\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"54\", \"year\": \"2022\"}, \"text\": \"Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52739, \"infons\": {\"fpage\": \"194\", \"name_0\": \"surname:McCartney;given-names:DL\", \"name_1\": \"surname:Min;given-names:JL\", \"name_2\": \"surname:Richmond;given-names:RC\", \"name_3\": \"surname:Lu;given-names:AT\", \"name_4\": \"surname:Sobczyk;given-names:MK\", \"name_5\": \"surname:Davies;given-names:G\", \"pub-id_doi\": \"10.1186/s13059-021-02398-9\", \"pub-id_pmid\": \"34187551\", \"section_type\": \"REF\", \"source\": \"Genome Biol\", \"type\": \"ref\", \"volume\": \"22\", \"year\": \"2021\"}, \"text\": \"Genome-wide association studies identify 137 genetic loci for DNA methylation biomarkers of aging\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52837, \"infons\": {\"fpage\": \"5980\", \"name_0\": \"surname:Laisk;given-names:T\", \"name_1\": \"surname:Soares;given-names:ALG\", \"name_2\": \"surname:Ferreira;given-names:T\", \"name_3\": \"surname:Painter;given-names:JN\", \"name_4\": \"surname:Censin;given-names:JC\", \"name_5\": \"surname:Laber;given-names:S\", \"pub-id_doi\": \"10.1038/s41467-020-19742-5\", \"pub-id_pmid\": \"33239672\", \"section_type\": \"REF\", \"source\": \"Nat Commun\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2020\"}, \"text\": \"The genetic architecture of sporadic and multiple consecutive miscarriage\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52911, \"infons\": {\"fpage\": \"1198\", \"name_0\": \"surname:Chen;given-names:M-H\", \"name_1\": \"surname:Raffield;given-names:LM\", \"name_2\": \"surname:Mousas;given-names:A\", \"name_3\": \"surname:Sakaue;given-names:S\", \"name_4\": \"surname:Huffman;given-names:JE\", \"name_5\": \"surname:Moscati;given-names:A\", \"pub-id_doi\": \"10.1016/j.cell.2020.06.045\", \"pub-id_pmid\": \"32888493\", \"section_type\": \"REF\", \"source\": \"Cell.\", \"type\": \"ref\", \"volume\": \"182\", \"year\": \"2020\"}, \"text\": \"Trans-ethnic and ancestry-specific blood-cell genetics in 746,667 individuals from 5 global populations\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53015, \"infons\": {\"fpage\": \"1217\", \"name_0\": \"surname:Shu;given-names:X\", \"name_1\": \"surname:Long;given-names:J\", \"name_2\": \"surname:Cai;given-names:Q\", \"name_3\": \"surname:Kweon;given-names:S-S\", \"name_4\": \"surname:Choi;given-names:J-Y\", \"name_5\": \"surname:Kubo;given-names:M\", \"pub-id_doi\": \"10.1038/s41467-020-15046-w\", \"pub-id_pmid\": \"32139696\", \"section_type\": \"REF\", \"source\": \"Nat Commun\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2020\"}, \"text\": \"Identification of novel breast cancer susceptibility loci in meta-analyses conducted among Asian and European descendants\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53137, \"infons\": {\"fpage\": \"1459\", \"name_0\": \"surname:Tin;given-names:A\", \"name_1\": \"surname:Marten;given-names:J\", \"name_2\": \"surname:Halperin Kuhns;given-names:VL\", \"name_3\": \"surname:Li;given-names:Y\", \"name_4\": \"surname:Wuttke;given-names:M\", \"name_5\": \"surname:Kirsten;given-names:H\", \"pub-id_doi\": \"10.1038/s41588-019-0504-x\", \"pub-id_pmid\": \"31578528\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2019\"}, \"text\": \"Target genes, variants, tissues and transcriptional pathways influencing human serum urate levels\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53235, \"infons\": {\"fpage\": \"4130\", \"name_0\": \"surname:Teumer;given-names:A\", \"name_1\": \"surname:Li;given-names:Y\", \"name_2\": \"surname:Ghasemi;given-names:S\", \"name_3\": \"surname:Prins;given-names:BP\", \"name_4\": \"surname:Wuttke;given-names:M\", \"name_5\": \"surname:Hermle;given-names:T\", \"pub-id_doi\": \"10.1038/s41467-019-11576-0\", \"pub-id_pmid\": \"31511532\", \"section_type\": \"REF\", \"source\": \"Nat Commun\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2019\"}, \"text\": \"Genome-wide association meta-analyses and fine-mapping elucidate pathways influencing albuminuria\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53333, \"infons\": {\"fpage\": \"957\", \"name_0\": \"surname:Wuttke;given-names:M\", \"name_1\": \"surname:Li;given-names:Y\", \"name_2\": \"surname:Li;given-names:M\", \"name_3\": \"surname:Sieber;given-names:KB\", \"name_4\": \"surname:Feitosa;given-names:MF\", \"name_5\": \"surname:Gorski;given-names:M\", \"pub-id_doi\": \"10.1038/s41588-019-0407-x\", \"pub-id_pmid\": \"31152163\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2019\"}, \"text\": \"A catalog of genetic loci associated with kidney function from analyses of a million individuals\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53430, \"infons\": {\"fpage\": \"29\", \"name_0\": \"surname:Morris;given-names:AP\", \"name_1\": \"surname:Le;given-names:TH\", \"name_2\": \"surname:Wu;given-names:H\", \"name_3\": \"surname:Akbarov;given-names:A\", \"name_4\": \"surname:van der Most;given-names:PJ\", \"name_5\": \"surname:Hemani;given-names:G\", \"pub-id_doi\": \"10.1038/s41467-018-07867-7\", \"pub-id_pmid\": \"30604766\", \"section_type\": \"REF\", \"source\": \"Nat Commun\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2019\"}, \"text\": \"Trans-ethnic kidney function association study reveals putative causal genes and effects on kidney-specific disease aetiologies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53558, \"infons\": {\"fpage\": \"880\", \"name_0\": \"surname:Daya;given-names:M\", \"name_1\": \"surname:Rafaels;given-names:N\", \"name_2\": \"surname:Brunetti;given-names:TM\", \"name_3\": \"surname:Chavan;given-names:S\", \"name_4\": \"surname:Levin;given-names:AM\", \"name_5\": \"surname:Shetty;given-names:A\", \"pub-id_doi\": \"10.1038/s41467-019-08469-7\", \"pub-id_pmid\": \"30787307\", \"section_type\": \"REF\", \"source\": \"Nat Commun\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2019\"}, \"text\": \"Association study in African-admixed populations across the Americas recapitulates asthma risk loci in non-African populations\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53685, \"infons\": {\"fpage\": \"151\", \"name_0\": \"surname:Shigemizu;given-names:D\", \"name_1\": \"surname:Mitsumori;given-names:R\", \"name_2\": \"surname:Akiyama;given-names:S\", \"name_3\": \"surname:Miyashita;given-names:A\", \"name_4\": \"surname:Morizono;given-names:T\", \"name_5\": \"surname:Higaki;given-names:S\", \"pub-id_doi\": \"10.1038/s41398-021-01272-3\", \"pub-id_pmid\": \"33654092\", \"section_type\": \"REF\", \"source\": \"Transl Psychiatry.\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2021\"}, \"text\": \"Ethnic and trans-ethnic genome-wide association studies identify new loci influencing Japanese Alzheimer\\u2019s disease risk\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53807, \"infons\": {\"fpage\": \"1451\", \"lpage\": \"60\", \"name_0\": \"surname:Kang;given-names:S\", \"name_1\": \"surname:Gim;given-names:J\", \"name_2\": \"surname:Lee;given-names:J\", \"name_3\": \"surname:Gunasekaran;given-names:TI\", \"name_4\": \"surname:Choi;given-names:KY\", \"name_5\": \"surname:Lee;given-names:JJ\", \"pub-id_doi\": \"10.3233/JAD-210145\", \"pub-id_pmid\": \"34151794\", \"section_type\": \"REF\", \"source\": \"J Alzheimers Dis\", \"type\": \"ref\", \"volume\": \"82\", \"year\": \"2021\"}, \"text\": \"Potential novel genes for late-onset Alzheimer\\u2019s disease in East-Asian descent identified by APOE-stratified genome-wide association study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53948, \"infons\": {\"fpage\": \"65\", \"lpage\": \"75\", \"name_0\": \"surname:Conti;given-names:DV\", \"name_1\": \"surname:Darst;given-names:BF\", \"name_2\": \"surname:Moss;given-names:LC\", \"name_3\": \"surname:Saunders;given-names:EJ\", \"name_4\": \"surname:Sheng;given-names:X\", \"name_5\": \"surname:Chou;given-names:A\", \"pub-id_doi\": \"10.1038/s41588-020-00748-0\", \"pub-id_pmid\": \"33398198\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"53\", \"year\": \"2021\"}, \"text\": \"Trans-ancestry genome-wide association meta-analysis of prostate cancer identifies new susceptibility loci and informs genetic risk prediction\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54091, \"infons\": {\"fpage\": \"2542\", \"name_0\": \"surname:Ntalla;given-names:I\", \"name_1\": \"surname:Weng;given-names:L-C\", \"name_2\": \"surname:Cartwright;given-names:JH\", \"name_3\": \"surname:Hall;given-names:AW\", \"name_4\": \"surname:Sveinbjornsson;given-names:G\", \"name_5\": \"surname:Tucker;given-names:NR\", \"pub-id_doi\": \"10.1038/s41467-020-15706-x\", \"pub-id_pmid\": \"32439900\", \"section_type\": \"REF\", \"source\": \"Nat Commun\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2020\"}, \"text\": \"Multi-ancestry GWAS of the electrocardiographic PR interval identifies 202 loci underlying cardiac conduction\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54201, \"infons\": {\"fpage\": \"636\", \"lpage\": \"48\", \"name_0\": \"surname:Bentley;given-names:AR\", \"name_1\": \"surname:Sung;given-names:YJ\", \"name_2\": \"surname:Brown;given-names:MR\", \"name_3\": \"surname:Winkler;given-names:TW\", \"name_4\": \"surname:Kraja;given-names:AT\", \"name_5\": \"surname:Ntalla;given-names:I\", \"pub-id_doi\": \"10.1038/s41588-019-0378-y\", \"pub-id_pmid\": \"30926973\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2019\"}, \"text\": \"Multi-ancestry genome-wide gene-smoking interaction study of 387,272 individuals identifies new loci associated with serum lipids\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54331, \"infons\": {\"fpage\": \"7\", \"name_0\": \"surname:Chang;given-names:CC\", \"name_1\": \"surname:Chow;given-names:CC\", \"name_2\": \"surname:Tellier;given-names:LC\", \"name_3\": \"surname:Vattikuti;given-names:S\", \"name_4\": \"surname:Purcell;given-names:SM\", \"name_5\": \"surname:Lee;given-names:JJ\", \"pub-id_doi\": \"10.1186/s13742-015-0047-8\", \"pub-id_pmid\": \"25722852\", \"section_type\": \"REF\", \"source\": \"Gigascience.\", \"type\": \"ref\", \"volume\": \"4\", \"year\": \"2015\"}, \"text\": \"Second-generation PLINK: rising to the challenge of larger and richer datasets\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54410, \"infons\": {\"fpage\": \"3639\", \"lpage\": \"50\", \"name_0\": \"surname:M\\u00e4gi;given-names:R\", \"name_1\": \"surname:Horikoshi;given-names:M\", \"name_2\": \"surname:Sofer;given-names:T\", \"name_3\": \"surname:Mahajan;given-names:A\", \"name_4\": \"surname:Kitajima;given-names:H\", \"name_5\": \"surname:Franceschini;given-names:N\", \"pub-id_doi\": \"10.1093/hmg/ddx280\", \"pub-id_pmid\": \"28911207\", \"section_type\": \"REF\", \"source\": \"Hum Mol Genet\", \"type\": \"ref\", \"volume\": \"26\", \"year\": \"2017\"}, \"text\": \"Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54565, \"infons\": {\"fpage\": \"D1302\", \"lpage\": \"D1310.\", \"name_0\": \"surname:Ochoa;given-names:D\", \"name_1\": \"surname:Hercules;given-names:A\", \"name_2\": \"surname:Carmona;given-names:M\", \"name_3\": \"surname:Suveges;given-names:D\", \"name_4\": \"surname:Gonzalez-Uriarte;given-names:A\", \"name_5\": \"surname:Malangone;given-names:C\", \"pub-id_doi\": \"10.1093/nar/gkaa1027\", \"pub-id_pmid\": \"33196847\", \"section_type\": \"REF\", \"source\": \"Nucleic Acids Res\", \"type\": \"ref\", \"volume\": \"49\", \"year\": \"2021\"}, \"text\": \"Open Targets Platform: supporting systematic drug-target identification and prioritisation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54656, \"infons\": {\"fpage\": \"161\", \"lpage\": \"9\", \"name_0\": \"surname:Zeng;given-names:B\", \"name_1\": \"surname:Bendl;given-names:J\", \"name_2\": \"surname:Kosoy;given-names:R\", \"name_3\": \"surname:Fullard;given-names:JF\", \"name_4\": \"surname:Hoffman;given-names:GE\", \"name_5\": \"surname:Roussos;given-names:P\", \"pub-id_doi\": \"10.1038/s41588-021-00987-9\", \"pub-id_pmid\": \"35058635\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"54\", \"year\": \"2022\"}, \"text\": \"Multi-ancestry eQTL meta-analysis of human brain identifies candidate causal variants for brain-related traits\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54767, \"infons\": {\"fpage\": \"e1002272\", \"name_0\": \"surname:Steele;given-names:NZR\", \"name_1\": \"surname:Carr;given-names:JS\", \"name_2\": \"surname:Bonham;given-names:LW\", \"name_3\": \"surname:Geier;given-names:EG\", \"name_4\": \"surname:Damotte;given-names:V\", \"name_5\": \"surname:Miller;given-names:ZA\", \"pub-id_doi\": \"10.1371/journal.pmed.1002272\", \"pub-id_pmid\": \"28350795\", \"section_type\": \"REF\", \"source\": \"PLoS Med\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2017\"}, \"text\": \"Fine-mapping of the human leukocyte antigen locus as a risk factor for Alzheimer disease: a case\\u2013control study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54880, \"infons\": {\"fpage\": \"470\", \"lpage\": \"480\", \"name_0\": \"surname:Hirata;given-names:J\", \"name_1\": \"surname:Hosomichi;given-names:K\", \"name_2\": \"surname:Sakaue;given-names:S\", \"name_3\": \"surname:Kanai;given-names:M\", \"name_4\": \"surname:Nakaoka;given-names:H\", \"name_5\": \"surname:Ishigaki;given-names:K\", \"pub-id_doi\": \"10.1038/s41588-018-0336-0\", \"pub-id_pmid\": \"30692682\", \"section_type\": \"REF\", \"source\": \"Nat Genet.\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2019\"}, \"text\": \"Genetic and phenotypic landscape of the major histocompatibilty complex region in the Japanese population\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54986, \"infons\": {\"fpage\": \"327\", \"name_0\": \"surname:S\\u00e1nchez-Juan;given-names:P\", \"name_1\": \"surname:Moreno;given-names:S\", \"name_2\": \"surname:de Rojas;given-names:I\", \"name_3\": \"surname:Hern\\u00e1ndez;given-names:I\", \"name_4\": \"surname:Valero;given-names:S\", \"name_5\": \"surname:Alegret;given-names:M\", \"pub-id_doi\": \"10.3389/fnagi.2019.00327\", \"pub-id_pmid\": \"31866851\", \"section_type\": \"REF\", \"source\": \"Front Aging Neurosci\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2019\"}, \"text\": \"The MAPT H1 haplotype is a risk factor for Alzheimer\\u2019s disease in APOE \\u03b54 non-carriers\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55079, \"infons\": {\"fpage\": \"1276\", \"lpage\": \"82\", \"name_0\": \"surname:Wightman;given-names:DP\", \"name_1\": \"surname:Jansen;given-names:IE\", \"name_2\": \"surname:Savage;given-names:JE\", \"name_3\": \"surname:Shadrin;given-names:AA\", \"name_4\": \"surname:Bahrami;given-names:S\", \"name_5\": \"surname:Holland;given-names:D\", \"pub-id_doi\": \"10.1038/s41588-021-00921-z\", \"pub-id_pmid\": \"34493870\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"53\", \"year\": \"2021\"}, \"text\": \"A genome-wide association study with 1,126,563 individuals identifies new risk loci for Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55189, \"infons\": {\"fpage\": \"392\", \"lpage\": \"402\", \"name_0\": \"surname:Schwartzentruber;given-names:J\", \"name_1\": \"surname:Cooper;given-names:S\", \"name_2\": \"surname:Liu;given-names:JZ\", \"name_3\": \"surname:Barrio-Hernandez;given-names:I\", \"name_4\": \"surname:Bello;given-names:E\", \"name_5\": \"surname:Kumasaka;given-names:N\", \"pub-id_doi\": \"10.1038/s41588-020-00776-w\", \"pub-id_pmid\": \"33589840\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"53\", \"year\": \"2021\"}, \"text\": \"Genome-wide meta-analysis, fine-mapping and integrative prioritization implicate new Alzheimer\\u2019s disease risk genes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55307, \"infons\": {\"fpage\": \"41\", \"lpage\": \"52\", \"name_0\": \"surname:Sirkis;given-names:DW\", \"name_1\": \"surname:Geier;given-names:EG\", \"name_2\": \"surname:Bonham;given-names:LW\", \"name_3\": \"surname:Karch;given-names:CM\", \"name_4\": \"surname:Yokoyama;given-names:JS\", \"pub-id_doi\": \"10.1007/s40142-019-0160-6\", \"pub-id_pmid\": \"31687268\", \"section_type\": \"REF\", \"source\": \"Curr Genet Med Rep\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2019\"}, \"text\": \"Recent advances in the genetics of frontotemporal dementia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55366, \"infons\": {\"fpage\": \"1091\", \"name_0\": \"surname:Nalls;given-names:MA\", \"name_1\": \"surname:Blauwendraat;given-names:C\", \"name_2\": \"surname:Vallerga;given-names:CL\", \"name_3\": \"surname:Heilbron;given-names:K\", \"name_4\": \"surname:Bandres-Ciga;given-names:S\", \"name_5\": \"surname:Chang;given-names:D\", \"pub-id_doi\": \"10.1016/S1474-4422(19)30320-5\", \"pub-id_pmid\": \"31701892\", \"section_type\": \"REF\", \"source\": \"Lancet Neurol\", \"type\": \"ref\", \"volume\": \"18\", \"year\": \"2019\"}, \"text\": \"Identification of novel risk loci, causal insights, and heritable risk for Parkinson\\u2019s disease: a meta-genome wide association study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55501, \"infons\": {\"fpage\": \"fcab095\", \"name_0\": \"surname:Nalls;given-names:MA\", \"name_1\": \"surname:Blauwendraat;given-names:C\", \"name_2\": \"surname:Sargent;given-names:L\", \"name_3\": \"surname:Vitale;given-names:D\", \"name_4\": \"surname:Leonard;given-names:H\", \"name_5\": \"surname:Iwaki;given-names:H\", \"pub-id_doi\": \"10.1093/braincomms/fcab095\", \"pub-id_pmid\": \"34693284\", \"section_type\": \"REF\", \"source\": \"Brain Commun\", \"type\": \"ref\", \"volume\": \"3\", \"year\": \"2021\"}, \"text\": \"Evidence for GRN connecting multiple neurodegenerative diseases\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55565, \"infons\": {\"fpage\": \"e58618\", \"name_0\": \"surname:Miyashita;given-names:A\", \"name_1\": \"surname:Koike;given-names:A\", \"name_2\": \"surname:Jun;given-names:G\", \"name_3\": \"surname:Wang;given-names:L-S\", \"name_4\": \"surname:Takahashi;given-names:S\", \"name_5\": \"surname:Matsubara;given-names:E\", \"pub-id_doi\": \"10.1371/journal.pone.0058618\", \"pub-id_pmid\": \"23565137\", \"section_type\": \"REF\", \"source\": \"PLoS ONE\", \"type\": \"ref\", \"volume\": \"8\", \"year\": \"2013\"}, \"text\": \"SORL1 is genetically associated with late-onset Alzheimer\\u2019s disease in Japanese, Koreans and Caucasians\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55671, \"infons\": {\"fpage\": \"e1007791\", \"name_0\": \"surname:Rajabli;given-names:F\", \"name_1\": \"surname:Feliciano;given-names:BE\", \"name_2\": \"surname:Celis;given-names:K\", \"name_3\": \"surname:Hamilton-Nelson;given-names:KL\", \"name_4\": \"surname:Whitehead;given-names:PL\", \"name_5\": \"surname:Adams;given-names:LD\", \"pub-id_doi\": \"10.1371/journal.pgen.1007791\", \"pub-id_pmid\": \"30517106\", \"section_type\": \"REF\", \"source\": \"PLoS Genet\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2018\"}, \"text\": \"Ancestral origin of ApoE \\u03b54 Alzheimer disease risk in Puerto Rican and African American populations\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55777, \"infons\": {\"fpage\": \"810\", \"name_0\": \"surname:Tosto;given-names:G\", \"name_1\": \"surname:Fu;given-names:H\", \"name_2\": \"surname:Vardarajan;given-names:BN\", \"name_3\": \"surname:Lee;given-names:JH\", \"name_4\": \"surname:Cheng;given-names:R\", \"name_5\": \"surname:Reyes-Dumeyer;given-names:D\", \"pub-id_doi\": \"10.1002/acn3.223\", \"pub-id_pmid\": \"26339675\", \"section_type\": \"REF\", \"source\": \"Ann Clin Transl Neurol\", \"type\": \"ref\", \"volume\": \"2\", \"year\": \"2015\"}, \"text\": \"F-box/LRR-repeat protein 7 is genetically associated with Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55857, \"infons\": {\"fpage\": \"195\", \"lpage\": \"205\", \"name_0\": \"surname:Chen;given-names:DT\", \"name_1\": \"surname:Jiang;given-names:X\", \"name_2\": \"surname:Akula;given-names:N\", \"name_3\": \"surname:Shugart;given-names:YY\", \"name_4\": \"surname:Wendland;given-names:JR\", \"name_5\": \"surname:Steele;given-names:CJM\", \"pub-id_doi\": \"10.1038/mp.2011.157\", \"pub-id_pmid\": \"22182935\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"18\", \"year\": \"2011\"}, \"text\": \"Genome-wide association study meta-analysis of European and Asian-ancestry samples identifies three novel loci associated with bipolar disorder\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56001, \"infons\": {\"fpage\": \"996\", \"lpage\": \"1006\", \"name_0\": \"surname:Kent;given-names:WJ\", \"name_1\": \"surname:Sugnet;given-names:CW\", \"name_2\": \"surname:Furey;given-names:TS\", \"name_3\": \"surname:Roskin;given-names:KM\", \"name_4\": \"surname:Pringle;given-names:TH\", \"name_5\": \"surname:Zahler;given-names:AM\", \"pub-id_doi\": \"10.1101/gr.229102\", \"pub-id_pmid\": \"12045153\", \"section_type\": \"REF\", \"source\": \"Genome Res\", \"type\": \"ref\", \"volume\": \"12\", \"year\": \"2002\"}, \"text\": \"The human genome browser at UCSC\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56034, \"infons\": {\"section_type\": \"REF\", \"type\": \"ref\"}, \"text\": \"Schizophrenia Working Group of the Psychiatric Genomics Consortium, Ripke S, Neale BM, Corvin A, Walters JTR, Farh K-H, et al. Biological insights from 108 Schizophrenia-Associated Genetic Loci. Nature. 2014;511:421.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56251, \"infons\": {\"fpage\": \"1670\", \"lpage\": \"8\", \"name_0\": \"surname:Lam;given-names:M\", \"name_1\": \"surname:Chen;given-names:C-Y\", \"name_2\": \"surname:Li;given-names:Z\", \"name_3\": \"surname:Martin;given-names:AR\", \"name_4\": \"surname:Bryois;given-names:J\", \"name_5\": \"surname:Ma;given-names:X\", \"pub-id_doi\": \"10.1038/s41588-019-0512-x\", \"pub-id_pmid\": \"31740837\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2019\"}, \"text\": \"Comparative genetic architectures of schizophrenia in East Asian and European populations\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56341, \"infons\": {\"fpage\": \"824\", \"lpage\": \"34\", \"name_0\": \"surname:Ikeda;given-names:M\", \"name_1\": \"surname:Takahashi;given-names:A\", \"name_2\": \"surname:Kamatani;given-names:Y\", \"name_3\": \"surname:Momozawa;given-names:Y\", \"name_4\": \"surname:Saito;given-names:T\", \"name_5\": \"surname:Kondo;given-names:K\", \"pub-id_doi\": \"10.1093/schbul/sby140\", \"pub-id_pmid\": \"30285260\", \"section_type\": \"REF\", \"source\": \"Schizophr Bull\", \"type\": \"ref\", \"volume\": \"45\", \"year\": \"2019\"}, \"text\": \"Genome-wide association study detected novel susceptibility genes for schizophrenia and shared trans-populations/diseases genetic effect\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56478, \"infons\": {\"fpage\": \"e180\", \"name_0\": \"surname:Goes;given-names:FS\", \"name_1\": \"surname:Hamshere;given-names:ML\", \"name_2\": \"surname:Seifuddin;given-names:F\", \"name_3\": \"surname:Pirooznia;given-names:M\", \"name_4\": \"surname:Belmonte-Mahon;given-names:P\", \"name_5\": \"surname:Breuer;given-names:R\", \"pub-id_doi\": \"10.1038/tp.2012.106\", \"pub-id_pmid\": \"23092984\", \"section_type\": \"REF\", \"source\": \"Transl Psychiatry\", \"type\": \"ref\", \"volume\": \"2\", \"year\": \"2012\"}, \"text\": \"Genome-wide association of mood-incongruent psychotic bipolar disorder\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56549, \"infons\": {\"fpage\": \"1017\", \"lpage\": \"24\", \"name_0\": \"surname:Ruderfer;given-names:DM\", \"name_1\": \"surname:Fanous;given-names:AH\", \"name_2\": \"surname:Ripke;given-names:S\", \"name_3\": \"surname:McQuillin;given-names:A\", \"name_4\": \"surname:Amdur;given-names:RL\", \"pub-id_doi\": \"10.1038/mp.2013.138\", \"pub-id_pmid\": \"24280982\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"19\", \"year\": \"2014\"}, \"text\": \"Schizophrenia Working Group of the Psychiatric Genomics Consortium, et al. Polygenic dissection of diagnosis and clinical dimensions of bipolar disorder and schizophrenia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56720, \"infons\": {\"fpage\": \"817\", \"lpage\": \"29\", \"name_0\": \"surname:Mullins;given-names:N\", \"name_1\": \"surname:Forstner;given-names:AJ\", \"name_2\": \"surname:O\\u2019Connell;given-names:KS\", \"name_3\": \"surname:Coombes;given-names:B\", \"name_4\": \"surname:Coleman;given-names:JRI\", \"name_5\": \"surname:Qiao;given-names:Z\", \"pub-id_doi\": \"10.1038/s41588-021-00857-4\", \"pub-id_pmid\": \"34002096\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"53\", \"year\": \"2021\"}, \"text\": \"Genome-wide association study of more than 40,000 bipolar disorder cases provides new insights into the underlying biology\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56843, \"infons\": {\"fpage\": \"220\", \"name_0\": \"surname:Drange;given-names:OK\", \"name_1\": \"surname:Smeland;given-names:OB\", \"name_2\": \"surname:Shadrin;given-names:AA\", \"name_3\": \"surname:Finseth;given-names:PI\", \"name_4\": \"surname:Witoelar;given-names:A\", \"name_5\": \"surname:Frei;given-names:O\", \"pub-id_doi\": \"10.3389/fnins.2019.00220\", \"pub-id_pmid\": \"30930738\", \"section_type\": \"REF\", \"source\": \"Front Neurosci\", \"type\": \"ref\", \"volume\": \"13\", \"year\": \"2019\"}, \"text\": \"Genetic overlap between Alzheimer\\u2019s disease and bipolar disorder implicates the MARK2 and VAC14 Genes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56947, \"infons\": {\"fpage\": \"1103\", \"lpage\": \"12\", \"name_0\": \"surname:Li;given-names:W\", \"name_1\": \"surname:Cai;given-names:X\", \"name_2\": \"surname:Li;given-names:H-J\", \"name_3\": \"surname:Song;given-names:M\", \"name_4\": \"surname:Zhang;given-names:C-Y\", \"name_5\": \"surname:Yang;given-names:Y\", \"pub-id_doi\": \"10.1038/s41386-020-00788-4\", \"pub-id_pmid\": \"32791513\", \"section_type\": \"REF\", \"source\": \"Neuropsychopharmacology.\", \"type\": \"ref\", \"volume\": \"46\", \"year\": \"2021\"}, \"text\": \"Independent replications and integrative analyses confirm TRANK1 as a susceptibility gene for bipolar disorder\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57058, \"infons\": {\"fpage\": \"613\", \"lpage\": \"24\", \"name_0\": \"surname:Jiang;given-names:X\", \"name_1\": \"surname:Detera-Wadleigh;given-names:SD\", \"name_2\": \"surname:Akula;given-names:N\", \"name_3\": \"surname:Mallon;given-names:BS\", \"name_4\": \"surname:Hou;given-names:L\", \"name_5\": \"surname:Xiao;given-names:T\", \"pub-id_doi\": \"10.1038/s41380-018-0207-1\", \"pub-id_pmid\": \"30135510\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"24\", \"year\": \"2019\"}, \"text\": \"Sodium valproate rescues expression of TRANK1 in iPSC-derived neural cells that carry a genetic variant associated with serious mental illness\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57201, \"infons\": {\"fpage\": \"1542\", \"lpage\": \"58\", \"name_0\": \"surname:Gambuzza;given-names:ME\", \"name_1\": \"surname:Sofo;given-names:V\", \"name_2\": \"surname:Salmeri;given-names:FM\", \"name_3\": \"surname:Soraci;given-names:L\", \"name_4\": \"surname:Marino;given-names:S\", \"name_5\": \"surname:Bramanti;given-names:P\", \"pub-id_doi\": \"10.2174/1871527313666140806124850\", \"pub-id_pmid\": \"25106635\", \"section_type\": \"REF\", \"source\": \"CNS Neurol Disord Drug Targets\", \"type\": \"ref\", \"volume\": \"13\", \"year\": \"2014\"}, \"text\": \"Toll-like receptors in Alzheimer\\u2019s disease: a therapeutic perspective\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57273, \"infons\": {\"fpage\": \"5474\", \"name_0\": \"surname:Wolters;given-names:FJ\", \"name_1\": \"surname:Boender;given-names:J\", \"name_2\": \"surname:de Vries;given-names:PS\", \"name_3\": \"surname:Sonneveld;given-names:MA\", \"name_4\": \"surname:Koudstaal;given-names:PJ\", \"name_5\": \"surname:de Maat;given-names:MP\", \"pub-id_doi\": \"10.1038/s41598-018-23865-7\", \"pub-id_pmid\": \"29615758\", \"section_type\": \"REF\", \"source\": \"Sci Rep\", \"type\": \"ref\", \"volume\": \"8\", \"year\": \"2018\"}, \"text\": \"Von Willebrand factor and ADAMTS13 activity in relation to risk of dementia: a population-based study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57375, \"infons\": {\"fpage\": \"100227\", \"name_0\": \"surname:Li;given-names:QS\", \"name_1\": \"surname:De\", \"name_2\": \"surname:Muynck;given-names:L\", \"pub-id_doi\": \"10.1016/j.bbih.2021.100227\", \"pub-id_pmid\": \"34589742\", \"section_type\": \"REF\", \"source\": \"Brain Behav Immun Health\", \"type\": \"ref\", \"volume\": \"13\", \"year\": \"2021\"}, \"text\": \"Differentially expressed genes in Alzheimer\\u2019s disease highlighting the roles of microglia genes including OLR1 and astrocyte gene CDK2AP1\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57515, \"infons\": {\"fpage\": \"1227\", \"lpage\": \"34\", \"name_0\": \"surname:Faux;given-names:NG\", \"name_1\": \"surname:Rembach;given-names:A\", \"name_2\": \"surname:Wiley;given-names:J\", \"name_3\": \"surname:Ellis;given-names:KA\", \"name_4\": \"surname:Ames;given-names:D\", \"name_5\": \"surname:Fowler;given-names:CJ\", \"pub-id_doi\": \"10.1038/mp.2013.178\", \"pub-id_pmid\": \"24419041\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"19\", \"year\": \"2014\"}, \"text\": \"An anemia of Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57550, \"infons\": {\"fpage\": \"21\", \"lpage\": \"30\", \"name_0\": \"surname:Ramos-Cejudo;given-names:J\", \"name_1\": \"surname:Wisniewski;given-names:T\", \"name_2\": \"surname:Marmar;given-names:C\", \"name_3\": \"surname:Zetterberg;given-names:H\", \"name_4\": \"surname:Blennow;given-names:K\", \"name_5\": \"surname:de Leon;given-names:MJ\", \"pub-id_doi\": \"10.1016/j.ebiom.2018.01.021\", \"pub-id_pmid\": \"29396300\", \"section_type\": \"REF\", \"source\": \"EBioMedicine.\", \"type\": \"ref\", \"volume\": \"28\", \"year\": \"2018\"}, \"text\": \"Traumatic Brain Injury and Alzheimer\\u2019s Disease: the cerebrovascular link\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57625, \"infons\": {\"section_type\": \"REF\", \"type\": \"ref\"}, \"text\": \"Dean M. The human ATP-binding cassette (ABC) transporter superfamily. USA: National Center for Biotechnology Information; 2002.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57753, \"infons\": {\"fpage\": \"487\", \"name_0\": \"surname:Cao;given-names:QT\", \"name_1\": \"surname:Aguiar;given-names:JA\", \"name_2\": \"surname:Tremblay;given-names:BJ-M\", \"name_3\": \"surname:Abbas;given-names:N\", \"name_4\": \"surname:Tiessen;given-names:N\", \"name_5\": \"surname:Revill;given-names:S\", \"pub-id_doi\": \"10.3389/fcimb.2020.00487\", \"pub-id_pmid\": \"33042865\", \"section_type\": \"REF\", \"source\": \"Front Cell Infect Microbiol\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2020\"}, \"text\": \"ABCF1 regulates dsDNA-induced immune responses in human airway epithelial cells\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57833, \"infons\": {\"fpage\": \"577362\", \"name_0\": \"surname:Momtazmanesh;given-names:S\", \"name_1\": \"surname:Perry;given-names:G\", \"name_2\": \"surname:Rezaei;given-names:N\", \"pub-id_doi\": \"10.1016/j.jneuroim.2020.577362\", \"pub-id_pmid\": \"32858428\", \"section_type\": \"REF\", \"source\": \"J Neuroimmunol\", \"type\": \"ref\", \"volume\": \"348\", \"year\": \"2020\"}, \"text\": \"Toll-like receptors in Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57878, \"infons\": {\"fpage\": \"7953\", \"lpage\": \"63\", \"name_0\": \"surname:Courtney;given-names:SC\", \"name_1\": \"surname:Di;given-names:H\", \"name_2\": \"surname:Stockman;given-names:BM\", \"name_3\": \"surname:Liu;given-names:H\", \"name_4\": \"surname:Scherbik;given-names:SV\", \"name_5\": \"surname:Brinton;given-names:MA\", \"pub-id_doi\": \"10.1128/JVI.00333-12\", \"pub-id_pmid\": \"22623793\", \"section_type\": \"REF\", \"source\": \"J Virol\", \"type\": \"ref\", \"volume\": \"86\", \"year\": \"2012\"}, \"text\": \"Identification of novel host cell binding partners of Oas1b, the protein conferring resistance to flavivirus-induced disease in mice\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58011, \"infons\": {\"fpage\": \"25\", \"lpage\": \"36\", \"name_0\": \"surname:Vittor;given-names:AY\", \"name_1\": \"surname:Long;given-names:M\", \"name_2\": \"surname:Chakrabarty;given-names:P\", \"name_3\": \"surname:Aycock;given-names:L\", \"name_4\": \"surname:Kollu;given-names:V\", \"name_5\": \"surname:DeKosky;given-names:ST\", \"pub-id_doi\": \"10.1007/s40475-020-00200-7\", \"pub-id_pmid\": \"32775145\", \"section_type\": \"REF\", \"source\": \"Curr Trop Med Rep\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2020\"}, \"text\": \"West nile virus-induced neurologic sequelae-relationship to neurodegenerative cascades and dementias\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58112, \"infons\": {\"fpage\": \"2581\", \"name_0\": \"surname:de Oliveira;given-names:J\", \"name_1\": \"surname:Kucharska;given-names:E\", \"name_2\": \"surname:Garcez;given-names:ML\", \"name_3\": \"surname:Rodrigues;given-names:MS\", \"name_4\": \"surname:Quevedo;given-names:J\", \"name_5\": \"surname:Moreno-Gonzalez;given-names:I\", \"pub-id_doi\": \"10.3390/cells10102581\", \"pub-id_pmid\": \"34685563\", \"section_type\": \"REF\", \"source\": \"Cells.\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2021\"}, \"text\": \"Inflammatory cascade in Alzheimer\\u2019s disease pathogenesis: a review of experimental findings\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58206, \"infons\": {\"fpage\": \"209\", \"lpage\": \"13\", \"name_0\": \"surname:Skol;given-names:AD\", \"name_1\": \"surname:Scott;given-names:LJ\", \"name_2\": \"surname:Abecasis;given-names:GR\", \"name_3\": \"surname:Boehnke;given-names:M\", \"pub-id_doi\": \"10.1038/ng1706\", \"pub-id_pmid\": \"16415888\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"38\", \"year\": \"2006\"}, \"text\": \"Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58317, \"infons\": {\"section_type\": \"REF\", \"type\": \"ref\"}, \"text\": \"Vardarajan BN, Reyes-Dumeyer D, Piriz AL, Lantigua RA, Medrano M, Rivera D, et al. Progranulin mutations in clinical and neuropathological Alzheimer\\u2019s disease. Alzheimers Dement. 2022. 10.1002/alz.12567.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58523, \"infons\": {\"fpage\": \"546\", \"lpage\": \"58\", \"name_0\": \"surname:T\\u00f6njes;given-names:A\", \"name_1\": \"surname:Scholz;given-names:M\", \"name_2\": \"surname:Kr\\u00fcger;given-names:J\", \"name_3\": \"surname:Krause;given-names:K\", \"name_4\": \"surname:Schleinitz;given-names:D\", \"name_5\": \"surname:Kirsten;given-names:H\", \"pub-id_doi\": \"10.1093/hmg/ddx413\", \"pub-id_pmid\": \"29186428\", \"section_type\": \"REF\", \"source\": \"Hum Mol Genet\", \"type\": \"ref\", \"volume\": \"27\", \"year\": \"2018\"}, \"text\": \"Genome-wide meta-analysis identifies novel determinants of circulating serum progranulin\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58612, \"infons\": {\"fpage\": \"297\", \"name_0\": \"surname:Liu;given-names:C\", \"name_1\": \"surname:Yu;given-names:J\", \"pub-id_doi\": \"10.3389/fnagi.2019.00297\", \"pub-id_pmid\": \"31708768\", \"section_type\": \"REF\", \"source\": \"Front Aging Neurosci\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2019\"}, \"text\": \"Genome-wide association studies for cerebrospinal fluid soluble TREM2 in Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58707, \"infons\": {\"fpage\": \"241.e13\", \"lpage\": \"241.e20\", \"name_0\": \"surname:Hou;given-names:XH\", \"name_1\": \"surname:Bi;given-names:YL\", \"name_2\": \"surname:Tan;given-names:MS\", \"name_3\": \"surname:Xu;given-names:W\", \"name_4\": \"surname:Li;given-names:JQ\", \"name_5\": \"surname:Shen;given-names:XN\", \"pub-id_doi\": \"10.1016/j.neurobiolaging.2019.05.008\", \"pub-id_pmid\": \"31204042\", \"section_type\": \"REF\", \"source\": \"Neurobiol Aging\", \"type\": \"ref\", \"volume\": \"84\", \"year\": \"2019\"}, \"text\": \"Genome-wide association study identifies Alzheimer\\u2019s risk variant in MS4A6A influencing cerebrospinal fluid sTREM2 levels\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58831, \"infons\": {\"fpage\": \"eaau2291\", \"name_0\": \"surname:Deming;given-names:Y\", \"name_1\": \"surname:Filipello;given-names:F\", \"name_2\": \"surname:Cignarella;given-names:F\", \"name_3\": \"surname:Cantoni;given-names:C\", \"name_4\": \"surname:Hsu;given-names:S\", \"name_5\": \"surname:Mikesell;given-names:R\", \"pub-id_doi\": \"10.1126/scitranslmed.aau2291\", \"pub-id_pmid\": \"31413141\", \"section_type\": \"REF\", \"source\": \"Sci Transl Med\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2019\"}, \"text\": \"The MS4A gene cluster is a key modulator of soluble TREM2 and Alzheimer\\u2019s disease risk\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58920, \"infons\": {\"fpage\": \"27\", \"name_0\": \"surname:Acosta-Uribe;given-names:J\", \"name_1\": \"surname:Aguill\\u00f3n;given-names:D\", \"name_2\": \"surname:Cochran;given-names:JN\", \"name_3\": \"surname:Giraldo;given-names:M\", \"name_4\": \"surname:Madrigal;given-names:L\", \"name_5\": \"surname:Killingsworth;given-names:BW\", \"pub-id_doi\": \"10.1186/s13073-022-01035-9\", \"pub-id_pmid\": \"35260199\", \"section_type\": \"REF\", \"source\": \"Genome Med\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2022\"}, \"text\": \"A neurodegenerative disease landscape of rare mutations in Colombia due to founder effects\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59011, \"infons\": {\"fpage\": \"2422.e9\", \"lpage\": \"2422.e11\", \"name_0\": \"surname:Jiao;given-names:B\", \"name_1\": \"surname:Liu;given-names:X\", \"name_2\": \"surname:Tang;given-names:B\", \"name_3\": \"surname:Hou;given-names:L\", \"name_4\": \"surname:Zhou;given-names:L\", \"name_5\": \"surname:Zhang;given-names:F\", \"pub-id_doi\": \"10.1016/j.neurobiolaging.2014.04.025\", \"pub-id_pmid\": \"24866402\", \"section_type\": \"REF\", \"source\": \"Neurobiol Aging\", \"type\": \"ref\", \"volume\": \"35\", \"year\": \"2014\"}, \"text\": \"Investigation of TREM2, PLD3, and UNC5C variants in patients with Alzheimer\\u2019s disease from mainland China\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59119, \"infons\": {\"fpage\": \"1483\", \"lpage\": \"92\", \"name_0\": \"surname:Reitz;given-names:C\", \"name_1\": \"surname:Jun;given-names:G\", \"name_2\": \"surname:Naj;given-names:A\", \"name_3\": \"surname:Rajbhandary;given-names:R\", \"name_4\": \"surname:Vardarajan;given-names:BN\", \"name_5\": \"surname:Wang;given-names:L-S\", \"pub-id_doi\": \"10.1001/jama.2013.2973\", \"pub-id_pmid\": \"23571587\", \"section_type\": \"REF\", \"source\": \"JAMA.\", \"type\": \"ref\", \"volume\": \"309\", \"year\": \"2013\"}, \"text\": \"Variants in the ATP-binding cassette transporter (ABCA7), Apolipoprotein E \\u03f54, and the risk of late-onset Alzheimer disease in African Americans\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59270, \"infons\": {\"fpage\": \"891\", \"lpage\": \"8\", \"name_0\": \"surname:Gianattasio;given-names:KZ\", \"name_1\": \"surname:Prather;given-names:C\", \"name_2\": \"surname:Glymour;given-names:MM\", \"name_3\": \"surname:Ciarleglio;given-names:A\", \"name_4\": \"surname:Power;given-names:MC\", \"pub-id_doi\": \"10.1016/j.trci.2019.11.008\", \"section_type\": \"REF\", \"source\": \"Alzheimers Dement\", \"type\": \"ref\", \"volume\": \"5\", \"year\": \"2019\"}, \"text\": \"Racial disparities and temporal trends in dementia misdiagnosis risk in the United States\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59360, \"infons\": {\"fpage\": \"266\", \"lpage\": \"73\", \"name_0\": \"surname:Beach;given-names:TG\", \"name_1\": \"surname:Monsell;given-names:SE\", \"name_2\": \"surname:Phillips;given-names:LE\", \"name_3\": \"surname:Kukull;given-names:W\", \"pub-id_doi\": \"10.1097/NEN.0b013e31824b211b\", \"pub-id_pmid\": \"22437338\", \"section_type\": \"REF\", \"source\": \"J Neuropathol Exp Neurol\", \"type\": \"ref\", \"volume\": \"71\", \"year\": \"2012\"}, \"text\": \"Accuracy of the clinical diagnosis of Alzheimer disease at National Institute on Aging Alzheimer Disease Centers, 2005\\u201310\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59484, \"infons\": {\"fpage\": \"404\", \"lpage\": \"13\", \"name_0\": \"surname:Jansen;given-names:IE\", \"name_1\": \"surname:Savage;given-names:JE\", \"name_2\": \"surname:Watanabe;given-names:K\", \"name_3\": \"surname:Bryois;given-names:J\", \"name_4\": \"surname:Williams;given-names:DM\", \"name_5\": \"surname:Steinberg;given-names:S\", \"pub-id_doi\": \"10.1038/s41588-018-0311-9\", \"pub-id_pmid\": \"30617256\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2019\"}, \"text\": \"Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer\\u2019s disease risk\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59593, \"infons\": {\"fpage\": \"96\", \"lpage\": \"99\", \"name_0\": \"surname:Jonsson;given-names:T\", \"name_1\": \"surname:Atwal;given-names:JK\", \"name_2\": \"surname:Steinberg;given-names:S\", \"name_3\": \"surname:Snaedal;given-names:J\", \"name_4\": \"surname:Jonsson;given-names:PV\", \"name_5\": \"surname:Bjornsson;given-names:S\", \"pub-id_doi\": \"10.1038/nature11283\", \"pub-id_pmid\": \"22801501\", \"section_type\": \"REF\", \"source\": \"Nature.\", \"type\": \"ref\", \"volume\": \"488\", \"year\": \"2012\"}, \"text\": \"A mutation in APP protects against Alzheimer\\u2019s disease and age-related cognitive decline\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59684, \"infons\": {\"fpage\": \"2674\", \"lpage\": \"88\", \"name_0\": \"surname:Sirkis;given-names:DW\", \"name_1\": \"surname:Bonham;given-names:LW\", \"name_2\": \"surname:Johnson;given-names:TP\", \"name_3\": \"surname:La Joie;given-names:R\", \"name_4\": \"surname:Yokoyama;given-names:JS\", \"pub-id_doi\": \"10.1038/s41380-022-01531-9\", \"pub-id_pmid\": \"35393555\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"27\", \"year\": \"2022\"}, \"text\": \"Dissecting the clinical heterogeneity of early-onset Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59759, \"infons\": {\"section_type\": \"REF\", \"type\": \"ref\"}, \"text\": \"Deeks JJ, Higgins JPT, Altman DG, on behalf of the Cochrane Statistical Methods Group. Analysing data and undertaking meta\\u2010analyses. Cochrane Handbook for Systematic Reviews of Interventions. 2019:241\\u201384.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59968, \"infons\": {\"fpage\": \"233\", \"name_0\": \"surname:Gay;given-names:NR\", \"name_1\": \"surname:Gloudemans;given-names:M\", \"name_2\": \"surname:Antonio;given-names:ML\", \"name_3\": \"surname:Abell;given-names:NS\", \"name_4\": \"surname:Balliu;given-names:B\", \"name_5\": \"surname:Park;given-names:Y\", \"pub-id_doi\": \"10.1186/s13059-020-02113-0\", \"pub-id_pmid\": \"32912333\", \"section_type\": \"REF\", \"source\": \"Genome Biol\", \"type\": \"ref\", \"volume\": \"21\", \"year\": \"2020\"}, \"text\": \"Impact of admixture and ancestry on eQTL analysis and GWAS colocalization in GTEx\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60050, \"infons\": {\"fpage\": \"481\", \"lpage\": \"7\", \"name_0\": \"surname:Zhu;given-names:Z\", \"name_1\": \"surname:Zhang;given-names:F\", \"name_2\": \"surname:Hu;given-names:H\", \"name_3\": \"surname:Bakshi;given-names:A\", \"name_4\": \"surname:Robinson;given-names:MR\", \"name_5\": \"surname:Powell;given-names:JE\", \"pub-id_doi\": \"10.1038/ng.3538\", \"pub-id_pmid\": \"27019110\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"48\", \"year\": \"2016\"}, \"text\": \"Integration of summary data from GWAS and eQTL studies predicts complex trait gene targets\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60141, \"infons\": {\"fpage\": \"e1008489\", \"name_0\": \"surname:King;given-names:EA\", \"name_1\": \"surname:Wade Davis;given-names:J\", \"name_2\": \"surname:Degner;given-names:JF\", \"pub-id_doi\": \"10.1371/journal.pgen.1008489\", \"pub-id_pmid\": \"31830040\", \"section_type\": \"REF\", \"source\": \"PLoS Genet\", \"type\": \"ref\", \"volume\": \"15\", \"year\": \"2019\"}, \"text\": \"Are drug targets with genetic support twice as likely to be approved? Revised estimates of the impact of genetic support for drug mechanisms on the probability of drug approval\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60318, \"infons\": {\"fpage\": \"768\", \"name_0\": \"surname:Liu;given-names:B\", \"name_1\": \"surname:Gloudemans;given-names:M\", \"name_2\": \"surname:Rao;given-names:AS\", \"name_3\": \"surname:Ingelsson;given-names:E\", \"name_4\": \"surname:Montgomery;given-names:SB\", \"pub-id_doi\": \"10.1038/s41588-019-0404-0\", \"pub-id_pmid\": \"31043754\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2019\"}, \"text\": \"Abundant associations with gene expression complicate GWAS follow-up\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60387, \"infons\": {\"fpage\": \"4593\", \"lpage\": \"6\", \"name_0\": \"surname:Murphy;given-names:AE\", \"name_1\": \"surname:Schilder;given-names:BM\", \"name_2\": \"surname:Skene;given-names:NG\", \"pub-id_doi\": \"10.1093/bioinformatics/btab665\", \"pub-id_pmid\": \"34601555\", \"section_type\": \"REF\", \"source\": \"Bioinformatics.\", \"type\": \"ref\", \"volume\": \"37\", \"year\": \"2021\"}, \"text\": \"MungeSumstats: a bioconductor package for the standardization and quality control of many GWAS summary statistics\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60501, \"infons\": {\"fpage\": \"e93766\", \"name_0\": \"surname:Abraham;given-names:G\", \"name_1\": \"surname:Inouye;given-names:M\", \"pub-id_doi\": \"10.1371/journal.pone.0093766\", \"pub-id_pmid\": \"24718290\", \"section_type\": \"REF\", \"source\": \"PLoS ONE\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2014\"}, \"text\": \"Fast principal component analysis of large-scale genome-wide data\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60567, \"infons\": {\"fpage\": \"1\", \"lpage\": \"7\", \"name_0\": \"surname:Montinaro;given-names:F\", \"name_1\": \"surname:Busby;given-names:GBJ\", \"name_2\": \"surname:Pascali;given-names:VL\", \"name_3\": \"surname:Myers;given-names:S\", \"name_4\": \"surname:Hellenthal;given-names:G\", \"name_5\": \"surname:Capelli;given-names:C\", \"pub-id_doi\": \"10.1038/ncomms7596\", \"section_type\": \"REF\", \"source\": \"Nat Commun\", \"type\": \"ref\", \"volume\": \"6\", \"year\": \"2015\"}, \"text\": \"Unravelling the hidden ancestry of American admixed populations\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60631, \"infons\": {\"section_type\": \"REF\", \"type\": \"ref\"}, \"text\": \"Tokunaga K, Ohashi J, Bannai M, Juji T. Genetic link between Asians and native Americans: evidence from HLA genes and haplotypes. Hum Immunol. 2001;62:1001\\u20138.\", \"sentences\": [], \"annotations\": [], \"relations\": []}], \"annotations\": [], \"relations\": []}]}]\n",
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading weights: 100%|| 770/770 [00:00<00:00, 1570.57it/s, Materializing param=model.encoder_input_proj.2.1.weight]                                                 \n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    }
   ],
   "source": [
    "pmcid = \"PMC10615750\"\n",
    "file_name = \"papers/Recent paper on AD GWAS (1).pdf\"  # Can be a local path or a URL\n",
    "df_lst = extract_tables_lst_from_paper(pmcid, file_name)  \n",
    "for i in [1, 2]:\n",
    "    df_lst[i-1].to_csv(f\"tables/{file_name.split('/')[-1].replace('.pdf', '')}_table_{i}.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "a42ae61d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{\"bioctype\": \"BioCCollection\", \"source\": \"PMC\", \"date\": \"20260202\", \"key\": \"pmc.key\", \"version\": \"1.0\", \"infons\": {}, \"documents\": [{\"bioctype\": \"BioCDocument\", \"id\": \"PMC6677735\", \"infons\": {\"license\": \"CC BY\"}, \"passages\": [{\"bioctype\": \"BioCPassage\", \"offset\": 0, \"infons\": {\"article-id_doi\": \"10.1038/s42003-019-0537-9\", \"article-id_pmc\": \"PMC6677735\", \"article-id_pmid\": \"31396565\", \"article-id_publisher-id\": \"537\", \"elocation-id\": \"285\", \"kwd\": \"Biomarkers Genome-wide association studies Neurology\", \"license\": \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article\\u2019s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article\\u2019s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit http://creativecommons.org/licenses/by/4.0/.\", \"name_0\": \"surname:van der Lee;given-names:Sven J.\", \"name_1\": \"surname:Knol;given-names:Maria J.\", \"name_10\": \"surname:Arfanakis;given-names:Konstantinos\", \"name_11\": \"surname:Bernard;given-names:Manon\", \"name_12\": \"surname:Yanek;given-names:Lisa R.\", \"name_13\": \"surname:Amin;given-names:Najaf\", \"name_14\": \"surname:Crivello;given-names:Fabrice\", \"name_15\": \"surname:Cheung;given-names:Josh W.\", \"name_16\": \"surname:Harris;given-names:Tamara B.\", \"name_17\": \"surname:Saba;given-names:Yasaman\", \"name_18\": \"surname:Lopez;given-names:Oscar L.\", \"name_19\": \"surname:Li;given-names:Shuo\", \"name_2\": \"surname:Chauhan;given-names:Ganesh\", \"name_20\": \"surname:van der Grond;given-names:Jeroen\", \"name_21\": \"surname:Yu;given-names:Lei\", \"name_22\": \"surname:Paus;given-names:Tomas\", \"name_23\": \"surname:Roshchupkin;given-names:Gennady V.\", \"name_24\": \"surname:Amouyel;given-names:Philippe\", \"name_25\": \"surname:Jahanshad;given-names:Neda\", \"name_26\": \"surname:Taylor;given-names:Kent D.\", \"name_27\": \"surname:Yang;given-names:Qiong\", \"name_28\": \"surname:Mathias;given-names:Rasika A.\", \"name_29\": \"surname:Boehringer;given-names:Stefan\", \"name_3\": \"surname:Satizabal;given-names:Claudia L.\", \"name_30\": \"surname:Mazoyer;given-names:Bernard\", \"name_31\": \"surname:Rice;given-names:Ken\", \"name_32\": \"surname:Cheng;given-names:Ching Yu\", \"name_33\": \"surname:Maillard;given-names:Pauline\", \"name_34\": \"surname:van Heemst;given-names:Diana\", \"name_35\": \"surname:Wong;given-names:Tien Yin\", \"name_36\": \"surname:Niessen;given-names:Wiro J.\", \"name_37\": \"surname:Beiser;given-names:Alexa S.\", \"name_38\": \"surname:Beekman;given-names:Marian\", \"name_39\": \"surname:Zhao;given-names:Wanting\", \"name_4\": \"surname:Smith;given-names:Albert Vernon\", \"name_40\": \"surname:Nyquist;given-names:Paul A.\", \"name_41\": \"surname:Chen;given-names:Christopher\", \"name_42\": \"surname:Launer;given-names:Lenore J.\", \"name_43\": \"surname:Psaty;given-names:Bruce M.\", \"name_44\": \"surname:Ikram;given-names:M. Kamran\", \"name_45\": \"surname:Vernooij;given-names:Meike W.\", \"name_46\": \"surname:Schmidt;given-names:Helena\", \"name_47\": \"surname:Pausova;given-names:Zdenka\", \"name_48\": \"surname:Becker;given-names:Diane M.\", \"name_49\": \"surname:De Jager;given-names:Philip L.\", \"name_5\": \"surname:Hofer;given-names:Edith\", \"name_50\": \"surname:Thompson;given-names:Paul M.\", \"name_51\": \"surname:van Duijn;given-names:Cornelia M.\", \"name_52\": \"surname:Bennett;given-names:David A.\", \"name_53\": \"surname:Slagboom;given-names:P. Eline\", \"name_54\": \"surname:Schmidt;given-names:Reinhold\", \"name_55\": \"surname:Longstreth;given-names:W. T.\", \"name_56\": \"surname:Ikram;given-names:M. Arfan\", \"name_57\": \"surname:Seshadri;given-names:Sudha\", \"name_58\": \"surname:Debette;given-names:St\\u00e9phanie\", \"name_59\": \"surname:Gudnason;given-names:Vilmundur\", \"name_6\": \"surname:Bis;given-names:Joshua C.\", \"name_60\": \"surname:Adams;given-names:Hieab H. H.\", \"name_61\": \"surname:DeCarli;given-names:Charles\", \"name_7\": \"surname:Hibar;given-names:Derrek P.\", \"name_8\": \"surname:Hilal;given-names:Saima\", \"name_9\": \"surname:van den Akker;given-names:Erik B.\", \"section_type\": \"TITLE\", \"title\": \"Subject terms\", \"type\": \"front\", \"volume\": \"2\", \"year\": \"2019\"}, \"text\": \"A genome-wide association study identifies genetic loci associated with specific lobar brain volumes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 101, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Brain lobar volumes are heritable but genetic studies are limited. We performed genome-wide association studies of frontal, occipital, parietal and temporal lobe volumes in 16,016 individuals, and replicated our findings in 8,789 individuals. We identified six genetic loci associated with specific lobar volumes independent of intracranial volume. Two loci, associated with occipital (6q22.32) and temporal lobe volume (12q14.3), were previously reported to associate with intracranial and hippocampal volume, respectively. We identified four loci previously unknown to affect brain volumes: 3q24 for parietal lobe volume, and 1q22, 4p16.3 and 14q23.1 for occipital lobe volume. The associated variants were located in regions enriched for histone modifications (DAAM1 and THBS3), or close to genes causing Mendelian brain-related diseases (ZIC4 and FGFRL1). No genetic overlap between lobar volumes and neurological or psychiatric diseases was observed. Our findings reveal part of the complex genetics underlying brain development and suggest a role for regulatory regions in determining brain volumes.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 1207, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Sven van der Lee et al. report a genome-wide association study for lobar brain volumes in more than 16,000 individuals, with replication in nearly 9,000. They find 6 loci associated with occipital, parietal and temporal lobe volume, of which two were previously reported to associate with intracranial and hippocampal volume.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 1533, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"title_1\"}, \"text\": \"Introduction\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 1546, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Four lobes of the human brain are distinguished and several diseases can partially be attributed to lobe-specific structural changes. Functions of the frontal brain lobe include reasoning, movement, social behavior, planning, parts of speech, and problem solving; functions attributed to the parietal lobe include recognition and perception of stimuli; functions attributed to the temporal lobe include memory and speech; and lastly, visual input is mainly processed by the occipital lobe. Brain diseases with lobe-specific abnormalities include Alzheimer\\u2019s disease (in particular early onset), frontotemporal lobar degeneration, temporal lobe epilepsy, primary progressive aphasia, and cortical basilar ganglionic degeneration.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 2277, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Environmental factors, such as smoking and hypertension, affect lobar brain volumes, but previous studies have shown that genetic differences across individuals also contribute to variability in volumetric brain measures. The estimated heritability of brain lobar volumes is high, ranging from 26% to 84% for the frontal lobe, from 32% to 74% for the occipital lobe, from 30% to 86% for the parietal lobe, and from 55% to 88% for the temporal lobe. In addition, genetic analyses in families suggest that the lobes are determined by independent genetic factors. The observation that brain lobes are highly and differentially heritable makes them compelling targets to unravel the genetic architecture of the brain. Recent large genome-wide association studies (GWAS) have efficiently identified associations between genetic determinants and volumetric brain measures. However, to date no genetic variants influencing brain lobar volumes have been identified. GWAS of the four lobar volumes of the brain can contribute to our understanding of brain lobe development and may provide a biological link between brain lobar volumes and brain-related traits and diseases.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 3442, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"To identify genetic variants of influence on lobar brain volumes, we performed GWAS of four brain lobar volumes in 16,016 individuals and replicated our findings in a sample of 8,789 individuals. We identified six loci significantly associated with specific brain lobar volumes independent of intracranial volume. With this study, we shed light on common genetic variants determining human brain volume and allow for a deepened understanding of the genetic architecture of the brain lobes.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 3932, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_1\"}, \"text\": \"Results\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 3940, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"In total, 16,016 individuals from 19 population-based or family-based cohort study and one case\\u2013control study were included in the current study. Additional information regarding the population characteristics, genotyping, and imaging methods are provided in the \\u201cMethods\\u201d section and in Supplementary Data\\u00a01\\u20134.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 4261, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Heritability of lobar brain volumes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 4297, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Using a family-based approach, we found an age- and sex-adjusted heritability (h2) for occipital lobe of 50% (95% confidence interval (CI) 38\\u201362%), for frontal lobe of 52% (95% CI 40\\u201364%), for temporal lobe of 59% (95% CI 49\\u201369%), and for parietal lobe of 59% (95% CI 49\\u201369%) (all: p\\u2009\\u2264\\u20091.9\\u2009\\u00d7\\u200910\\u221219) (Supplementary Data\\u00a05). In comparison, the age- and sex-adjusted heritability estimate for total brain volume was 34% (95% CI 22\\u201346%, p\\u2009=\\u20098.8\\u2009\\u00d7\\u200910\\u221211).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 4783, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Novel genetic associations with brain lobar volumes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 4835, \"infons\": {\"file\": \"42003_2019_537_Fig1_HTML.jpg\", \"id\": \"Fig1\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Common genetic variants associated with frontal, parietal, temporal and occipital lobe volume. \\u2018Manhattan\\u2019 plot displaying the association p value for each tested single-nucleotide polymorphism (SNP) (displayed as \\u2013log10 of the p value). Genome-wide significance threshold is shown with a line at p\\u2009=\\u20095\\u2009\\u00d7\\u200910\\u22128 (solid black line) and also the suggestive threshold at p\\u2009=\\u20091\\u2009\\u00d7\\u200910\\u22125 (dashed line). Dots represent SNPs, results of the four lobes are shown in a single figure, and the nearby gene is labeled. Above the suggestive threshold SNPs are colored by the associated lobe: yellow\\u2009=\\u2009temporal lobe, green\\u2009=\\u2009occipital lobe, red\\u2009=\\u2009parietal lobe, blue\\u2009=\\u2009frontal lobe\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 5543, \"infons\": {\"file\": \"42003_2019_537_Fig2_HTML.jpg\", \"id\": \"Fig2\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Regional view of the genome-wide significant loci. For each panel, zoomed in Manhattan plots (\\u00b1kb from top single-nucleotide polymorphism (SNP)) are shown with gene models below (GENCODE version 19). Plots are zoomed in to highlight the genomic region that contains the index SNP and SNPs in linkage disequilibrium with the index SNP (R2\\u2009>\\u20090.8). Each plot was made using the LocusTrack software (http://gump.qimr.edu.au/general/gabrieC/LocusTrack/)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 5997, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Genetic variants at six loci significantly associated with lobar brain volumes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 6076, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"hsides\\\" rules=\\\"groups\\\"><thead><tr><th>Lobe</th><th>Annotation (nearby)</th><th>Gene</th><th>Rs-ID</th><th>Chr</th><th>Position</th><th>A1</th><th>A2</th><th colspan=\\\"4\\\">European ancestry meta-analysis</th><th colspan=\\\"4\\\">Multi-ethnic meta-analysis</th><th colspan=\\\"6\\\">Replication sample (<italic>n</italic>\\u2009=\\u20098,789)</th></tr><tr><th/><th/><th/><th/><th/><th/><th/><th/><th>Frq</th><th>Effect</th><th>SE</th><th><italic>p</italic></th><th>Frq</th><th>Effect</th><th>SE</th><th><italic>p</italic></th><th><italic>N</italic></th><th><italic>R</italic><sup>2</sup> (%)</th><th>Frq</th><th>Effect</th><th>SE</th><th><italic>p</italic></th></tr></thead><tbody><tr><td>Parietal</td><td>UTR3</td><td><italic>ZIC4</italic></td><td>rs2279829</td><td>3</td><td>147106319</td><td>T</td><td>C</td><td>0.21</td><td>\\u22120.92</td><td>0.16</td><td>5.3\\u2009\\u00d7\\u200910<sup>\\u22129</sup></td><td>0.21</td><td>\\u22120.95</td><td>0.15</td><td>4.4\\u2009\\u00d7\\u200910<sup>\\u221210</sup></td><td>16,015</td><td>0.24</td><td>0.22</td><td>\\u22120.63</td><td>0.18</td><td>6.0\\u2009\\u00d7\\u200910<sup>\\u22124</sup></td></tr><tr><td>Temporal</td><td>Intronic</td><td><italic>MSRB3</italic></td><td>rs146354218</td><td>12</td><td>65793942</td><td>A</td><td>G</td><td>0.37</td><td>0.69</td><td>0.11</td><td>1.2\\u2009\\u00d7\\u200910<sup>\\u22129</sup></td><td>0.37</td><td>0.69</td><td>0.11</td><td>6.4\\u2009\\u00d7\\u200910<sup>\\u221210</sup></td><td>15,789</td><td>0.24</td><td>0.37</td><td>0.54</td><td>0.14</td><td>6.4\\u2009\\u00d7\\u200910<sup>\\u22125</sup></td></tr><tr><td>Occipital</td><td>Intergenic</td><td><italic>DAAM1</italic> (dist\\u2009=\\u200924302)</td><td>rs147148763<sup>a</sup></td><td>14</td><td>59631075</td><td>G</td><td>GTTGT</td><td>0.13</td><td>\\u22120.83</td><td>0.14</td><td>6.6\\u2009\\u00d7\\u200910<sup>\\u22129</sup></td><td>0.13</td><td>\\u22120.85</td><td>0.14</td><td>2.9\\u2009\\u00d7\\u200910<sup>\\u22129</sup></td><td>15,220</td><td>0.23</td><td>0.12</td><td>\\u22120.80</td><td>0.12</td><td>3.1\\u2009\\u00d7\\u200910<sup>\\u221211</sup></td></tr><tr><td>Occipital</td><td>Intronic</td><td><italic>FGFRL1</italic></td><td>rs74921869</td><td>4</td><td>1013382</td><td>A</td><td>G</td><td>0.2</td><td>\\u22120.84</td><td>0.14</td><td>5.9\\u2009\\u00d7\\u200910<sup>\\u22129</sup></td><td>0.2</td><td>\\u22120.82</td><td>0.14</td><td>6.2\\u2009\\u00d7\\u200910<sup>\\u22129</sup></td><td>12,424</td><td>0.27</td><td>0.19</td><td>\\u22120.22</td><td>0.10</td><td>3.0\\u2009\\u00d7\\u200910<sup>\\u22122</sup></td></tr><tr><td>Occipital</td><td>Intergenic</td><td><italic>CENPW</italic> (dist\\u2009=\\u2009175626)</td><td>rs1337736</td><td>6</td><td>126845380</td><td>A</td><td>G</td><td>0.23</td><td>\\u22120.64</td><td>0.12</td><td>8.1\\u2009\\u00d7\\u200910<sup>\\u22128</sup></td><td>0.23</td><td>\\u22120.64</td><td>0.12</td><td>4.0\\u2009\\u00d7\\u200910<sup>\\u22128</sup></td><td>16,016</td><td>0.19</td><td>0.23</td><td>\\u22120.46</td><td>0.09</td><td>8.0\\u2009\\u00d7\\u200910<sup>\\u22127</sup></td></tr><tr><td>Occipital</td><td>Upstream</td><td><italic>MIR92B/THBS3</italic></td><td>rs12411216</td><td>1</td><td>155164480</td><td>A</td><td>C</td><td>0.46</td><td>\\u22120.52</td><td>0.10</td><td>3.9\\u2009\\u00d7\\u200910<sup>\\u22128</sup></td><td>0.46</td><td>\\u22120.49</td><td>0.09</td><td>1.4\\u2009\\u00d7\\u200910<sup>\\u22127</sup></td><td>16,016</td><td>0.17</td><td>0.45</td><td>\\u22120.19</td><td>0.08</td><td>1.6\\u2009\\u00d7\\u200910<sup>\\u22122</sup></td></tr></tbody></table>\\n\"}, \"text\": \"Lobe\\tAnnotation (nearby)\\tGene\\tRs-ID\\tChr\\tPosition\\tA1\\tA2\\tEuropean ancestry meta-analysis\\tMulti-ethnic meta-analysis\\tReplication sample (n\\u2009=\\u20098,789)\\t \\t\\t\\t\\t\\t\\t\\t\\t\\tFrq\\tEffect\\tSE\\tp\\tFrq\\tEffect\\tSE\\tp\\tN\\tR2 (%)\\tFrq\\tEffect\\tSE\\tp\\t \\tParietal\\tUTR3\\tZIC4\\trs2279829\\t3\\t147106319\\tT\\tC\\t0.21\\t\\u22120.92\\t0.16\\t5.3\\u2009\\u00d7\\u200910\\u22129\\t0.21\\t\\u22120.95\\t0.15\\t4.4\\u2009\\u00d7\\u200910\\u221210\\t16,015\\t0.24\\t0.22\\t\\u22120.63\\t0.18\\t6.0\\u2009\\u00d7\\u200910\\u22124\\t \\tTemporal\\tIntronic\\tMSRB3\\trs146354218\\t12\\t65793942\\tA\\tG\\t0.37\\t0.69\\t0.11\\t1.2\\u2009\\u00d7\\u200910\\u22129\\t0.37\\t0.69\\t0.11\\t6.4\\u2009\\u00d7\\u200910\\u221210\\t15,789\\t0.24\\t0.37\\t0.54\\t0.14\\t6.4\\u2009\\u00d7\\u200910\\u22125\\t \\tOccipital\\tIntergenic\\tDAAM1 (dist\\u2009=\\u200924302)\\trs147148763a\\t14\\t59631075\\tG\\tGTTGT\\t0.13\\t\\u22120.83\\t0.14\\t6.6\\u2009\\u00d7\\u200910\\u22129\\t0.13\\t\\u22120.85\\t0.14\\t2.9\\u2009\\u00d7\\u200910\\u22129\\t15,220\\t0.23\\t0.12\\t\\u22120.80\\t0.12\\t3.1\\u2009\\u00d7\\u200910\\u221211\\t \\tOccipital\\tIntronic\\tFGFRL1\\trs74921869\\t4\\t1013382\\tA\\tG\\t0.2\\t\\u22120.84\\t0.14\\t5.9\\u2009\\u00d7\\u200910\\u22129\\t0.2\\t\\u22120.82\\t0.14\\t6.2\\u2009\\u00d7\\u200910\\u22129\\t12,424\\t0.27\\t0.19\\t\\u22120.22\\t0.10\\t3.0\\u2009\\u00d7\\u200910\\u22122\\t \\tOccipital\\tIntergenic\\tCENPW (dist\\u2009=\\u2009175626)\\trs1337736\\t6\\t126845380\\tA\\tG\\t0.23\\t\\u22120.64\\t0.12\\t8.1\\u2009\\u00d7\\u200910\\u22128\\t0.23\\t\\u22120.64\\t0.12\\t4.0\\u2009\\u00d7\\u200910\\u22128\\t16,016\\t0.19\\t0.23\\t\\u22120.46\\t0.09\\t8.0\\u2009\\u00d7\\u200910\\u22127\\t \\tOccipital\\tUpstream\\tMIR92B/THBS3\\trs12411216\\t1\\t155164480\\tA\\tC\\t0.46\\t\\u22120.52\\t0.10\\t3.9\\u2009\\u00d7\\u200910\\u22128\\t0.46\\t\\u22120.49\\t0.09\\t1.4\\u2009\\u00d7\\u200910\\u22127\\t16,016\\t0.17\\t0.45\\t\\u22120.19\\t0.08\\t1.6\\u2009\\u00d7\\u200910\\u22122\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 7378, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"The allele frequency (frq) and effect size are given for A1. Effect sizes are given in units of cm3 per effect allele. Results are provided for the discovery samples and the meta-analysis of all European ancestry and the multi-ethnic meta-analysis. The variance explained gives the percentage variance explained of a SNP\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 7699, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"A1 effect allele, A2 reference allele, Chr chromosome, Frq effect allele frequency, N number of individuals with genetic variant, p\\np value, Rs-ID reference SNP cluster ID, SE standard error\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 7890, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"aIn the replication sample, another significant variant in high LD (R2\\u2009=\\u20090.99) with this variant was used (rs76341705, p value\\u2009=\\u20094.8\\u2009\\u00d7\\u200910\\u22129)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 8046, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"Abbreviations: effect allele (A1), reference allele (A2), chromosome (Chr), effect allele frequency (Frq), number of individuals with genetic variant (N), p-value (P), reference SNP cluster ID (Rs-ID), standard error (SE)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 8268, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Our multi-ethnic meta-analysis (n\\u2009=\\u200916,016 individuals of which 15,269 were of European ancestry) identified significant associations between genotypes and brain lobar volumes in five independent loci, even though we adjusted for intracranial volume (Figs.\\u00a01 and 2, Table\\u00a01). The quantile\\u2013quantile plots did not show high genomic inflation (\\u03bbGC\\u2009\\u2264\\u20091.05) (Supplementary Fig.\\u00a01). Of these five loci, variants in one locus associated with temporal lobe volume, in one with parietal lobe volume, and in three with occipital lobe volume. The variant rs146354218 (12q14.3, pmulti-ethnic\\u2009=\\u20096.4\\u2009\\u00d7\\u200910\\u221210) associated with temporal lobe volume and rs2279829 (3q24, pmulti-ethnic\\u2009=\\u20094.4\\u2009\\u00d7\\u200910\\u221210) associated with parietal lobe volume. Three loci associated with occipital lobe volume: index variants rs147148763 (small indel GTTGT\\u2192G, 14q23.1, pmulti-ethnic\\u2009=\\u20092.9\\u2009\\u00d7\\u200910\\u22129), rs74921869 (4p16.3, pmulti-ethnic\\u2009=\\u20096.2\\u2009\\u00d7\\u200910\\u22129), and rs1337736 (6q22.32, pmulti-ethnic\\u2009=\\u20094.0\\u2009\\u00d7\\u200910\\u22128). In the European ancestry-only meta-analysis, we found a significant association with occipital lobe volume in one additional independent locus (1q22, rs12411216, pEuropean ancestry-only\\u2009=\\u20093.9\\u2009\\u00d7\\u200910\\u22128). In the multi-ethnic meta-analysis, this association was below the genome-wide significance threshold (pmulti-ethnic\\u2009=\\u20091.3\\u2009\\u00d7\\u200910\\u22127). There was no significant heterogeneity observed for any of the six significant loci (Supplementary Figs.\\u00a02\\u20137). The sensitivity meta-analysis including only the studies using the k-Nearest-Neighbor (kNN) algorithm for measuring lobar volumes showed similar results compared to the studies using other methods (Supplementary Figs.\\u00a02\\u20137). The index variants of these total six loci were common (minor allele frequency ranging from 0.13 to 0.46) and associations with volume variations were between 0.48 and 0.95\\u2009cm3 per copy of the variant allele, explaining up to 0.27% of lobar volume variance per allele (Table\\u00a01). No variants were significantly associated with frontal lobe volume. All variants showing significant associations with brain lobar volumes are shown in Supplementary Data\\u00a06. Study-specific effects of all six significant loci are shown in Supplementary Figs.\\u00a02\\u20137.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 10544, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Notably, two genome-wide significant variants identified here, rs146354218 and rs2279827, were exclusively associated with the temporal lobe and parietal lobe, respectively (Supplementary Data\\u00a07). In contrast, rs147148763 and rs12411216 were not only significantly associated with occipital lobe volume but also appeared to be associated to some extent with parietal lobe volume (p\\u2009=\\u20092.5\\u2009\\u00d7\\u200910\\u22126; variance explained\\u2009=\\u20090.15% and p\\u2009=\\u20092.4\\u2009\\u00d7\\u200910\\u22125; variance explained\\u2009=\\u20090.11%, respectively). The other two variants also showed nominally significant associations with other lobar volumes.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 11157, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Replication\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 11169, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Five out of the six index variants were available in the imputation reference panel of our replication sample (n\\u2009=\\u20098,789). Unfortunately, the haplotype reference consortium (HRC) reference panel does not contain insertions and deletions. Therefore, for replication we selected rs76341705, a variant that showed a comparable signal in the meta-analysis (prs147148763\\u2009=\\u20092.9\\u2009\\u00d7\\u200910\\u22129, vs. prs76341705\\u2009=\\u20094.8\\u2009\\u00d7\\u200910\\u22129), and in high linkage disequilibrium (LD) with the index variant (R2\\u2009=\\u20090.99). We were able to replicate all these six variants at a nominal significance level (p values ranging from 3.0\\u2009\\u00d7\\u200910\\u22122 to 8.0\\u2009\\u00d7\\u200910\\u22127) with the same direction of effect as the discovery sample (Table\\u00a01, Supplementary Fig.\\u00a08).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 11923, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Variance explained in lobar volumes by common variants\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 11978, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Based on the LD score regression single-nucleotide polymorphism (SNP)-based heritability analyses, common variants across the whole genome explained as much as 20.3% (95% CI 13.2\\u201327.4%) of the variance in occipital lobe volume, 19.6% (95% CI 12.3\\u201326.9%) of frontal lobe volume, 17.5% (95% CI 10.7\\u201324.3%) of temporal lobe volume, and 17.9% (95% CI 11.7\\u201324.1%) of parietal lobe volume (Supplementary Data\\u00a08). Common genetic variants account for 30\\u201341% of the total heritability of brain lobar volumes (Supplementary Data\\u00a08).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 12513, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Genetic overlap with other brain volumes and related diseases\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 12575, \"infons\": {\"file\": \"42003_2019_537_Fig3_HTML.jpg\", \"id\": \"Fig3\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Genetic correlation between lobar brain volumes and other brain imaging measures and neuropsychiatric traits. Heatmap showing the genetic correlations estimates (rg) as calculated by linkage disequilibrium score regression. Larger blocks and darker colors present stronger correlations, with blue and red indicating positive and negative correlations, respectively. The strength of the significance levels are indicated by asterisks: *p\\u2009<\\u20090.05; **p\\u2009<\\u20091.3\\u2009\\u00d7\\u200910\\u22122 (0.05/4), adjusted for the lobe count; ***p\\u2009<\\u20095.8\\u2009\\u00d7\\u200910\\u22124 (0.05/86), adjusted for the number of correlations tested\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 13178, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Although no top variant was significantly associated with the volume of multiple lobes, nominally significant correlation (rg) was observed between genetic components of the parietal and temporal lobe (rg\\u2009=\\u20090.35, p\\u2009=\\u20091.5\\u2009\\u00d7\\u200910\\u22123), although this did not withstand correction for multiple testing (Fig.\\u00a03, Supplementary Data\\u00a09). Some suggestive correlation was observed between temporal and frontal lobe volume with genetic determinants of subcortical volumes; however, none survived multiple testing adjustments. When studying brain diseases, only occipital lobe volume showed a suggestive genetic correlation with Parkinson\\u2019s disease (rg\\u2009=\\u20090.18, p\\u2009=\\u20090.03). No significant genetic correlation was observed with any of the other tested neurological or psychiatric traits.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 13974, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_1\"}, \"text\": \"Discussion\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 13985, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"In our genome-wide association study of in up to 16,016 individuals, we identified 6 independent loci where variants had significant associations with brain lobar volumes, independent of intracranial volume. We were able to replicate these findings in a sample of 8,789 individuals. Four out of the six identified loci have not been linked to brain volume measures before; the other two loci are located in regions previously associated with brain volume measures (12q14.3 with hippocampal volume and 6q22.32 with intracranial volume). These new loci provide intriguing new insights into the genetics underlying brain lobar volumes.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 14618, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"We estimated that, after adjusting for intracranial volume, 17.5\\u201320.3% of the variance in lobar volumes could be explained by common genetic variation. This forms 30\\u201340% of the total heritability we estimated, suggesting a major contribution of common genetic variation in brain development. More genetic variants associated with brain volume may be discovered by increasing the sample sizes of genetic studies. An interesting observation is that we were able to replicate our findings using only gray matter volumes of each lobe, while the discovery studied the sum of gray and white matter. This difference might explain that not all loci replicated as strong as others and that differential effects might exist on gray and white matter volume effects of the genetic variants. Future studies will have to elucidate the biological mechanisms of the discovered associations.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 15497, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Interestingly, the majority of the identified loci contained variants associated with occipital lobe volume, whereas the other brain lobes have more often been linked to disease outcomes. Yet, the heritability estimates for the occipital lobe do not exceed the heritability estimates of the other brain lobes. One possible explanation for this finding is the smaller volume of the occipital lobe compared to the other lobes, making it a more specific region\\u2014also in terms of the genetic architecture. It may also be explained by a less polygenic nature of the occipital lobe compared to the other lobes, allowing one to identify stronger associations for a single genetic variant.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 16180, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Regarding the identified genome-wide significant loci, two identified loci have been previously associated with brain volume measurements. The locus 12q14.3 associated with temporal lobe volume in our study and was previously associated with hippocampal volume. Our index variant rs146354218 (p\\u2009=\\u20096.4\\u2009\\u00d7\\u200910\\u221210) is an intronic variant in the MSRB3 gene and lies 39 kilobases (kb) from the previously published rs61921502 variant associated with hippocampal volume; however, the LD (R2\\u2009=\\u20090.1, D\\u2019\\u2009=\\u20091, p\\u2009<\\u20090.0001) is low. This previously published variant also showed some evidence of association with total temporal lobe volume (effect\\u2009=\\u20090.57\\u2009cm3, p\\u2009=\\u20096.5\\u2009\\u00d7\\u200910\\u22124). Thus the 12q14.3 locus not only influences hippocampal volume but also seems to have a more generalized effect on the temporal lobe volume as would be expected by the genetic correlation between temporal lobe volume and hippocampal volume. The signal (rs1337736, p\\u2009=\\u20094.0\\u2009\\u00d7\\u200910\\u22128) at 6q22.32 near to the gene CENPW (Centromere Protein W) is associated with occipital volume. This signal overlapped previously associated signals with intracranial volume and is further implicated in bone mineral density, height, waist-hip ratio, and infant length. The index variant associated with intracranial volume (rs11759026) and our top variant are in linkage equilibrium (R2\\u2009=\\u20090.07, D\\u2019\\u2009=\\u20091, p\\u2009=\\u20090.0002). We also found suggestive associations between rs11759026 and both frontal (\\u22121.0\\u2009cm3, p\\u2009=\\u20096.3\\u2009\\u00d7\\u200910\\u22125) and occipital lobar volume (\\u22120.31\\u2009cm3, p\\u2009=\\u20096.6\\u2009\\u00d7\\u200910\\u22123). Each locus was located in regions that are under epigenetic regulation in brain tissue (Supplementary Data\\u00a06) or close to genes or genomic loci associated with Mendelian brain-related diseases. The variant rs2279829 (3q24) is located in the 3\\u2019-untranslated region (UTR) of the Zic Family Member 4 (ZIC4) gene and close to the related ZIC1 gene. This variant localizes within enhancer sites in predominantly neurological cell types, among which the brain germinal matrix (Supplementary Data\\u00a06) and both ZIC4 and ZIC1 are expressed throughout the brain (Supplementary Fig.\\u00a09). Heterozygous deletions of ZIC1 and ZIC4 cause Dandy\\u2013Walker malformation. Children with this malformation have no vermis, the part connecting the two cerebellar hemispheres. Gain-of-function mutations in ZIC1 lead to coronal craniosynostosis and learning disability. Variant rs147148763 was located 24\\u2009kb from the disheveled-associated activator of morphogenesis 1 (DAAM1). There is evidence for the most significant variants to localize within enhancer sites, as well as DNA-hypersensitivity sites in brain tissues. Also, genome-wide significant SNPs in the locus are expression quantitative trait loci (eQTLs) of DAAM1 in blood (Supplementary Data\\u00a06). Daam1 is a formin protein that has been linked to actin dynamics, is regulated by RhoA, and is expressed in the shafts of dendrites. Expression patterns in brain development of animals further suggest a role in neuronal cell differentiation and movement. Variant rs4647940 is located in the 3\\u2019-UTR of fibroblast growth factor receptor (FGFRL1) and is in LD with a missense variant in Alpha-L-iduronidase (IDUA) (rs3755955, R2\\u2009=\\u20090.87) that was previously associated with bone mineral density (p\\u2009=\\u20095.0\\u2009\\u00d7\\u200910\\u221215). Deletion of the 4p16.3 locus causes Wolf\\u2013Hirschhorn syndrome, a neurodevelopmental disorder characterized by mental retardation, craniofacial malformation, and defects in skeletal and heart development. Variant rs12411216 is located in an intron of MIR92B and THBS3, but the signal peak in this locus covers >20 genes. Promotor histone marks overlap the variant and it is an eQTL for multiple genes, both in a multitude of different tissues among which brain tissues (Supplementary Data\\u00a06). In summary, these findings link genes that cause Mendelian syndromes affecting cranial skeletal malformations, brain malformations, and intelligence with brain lobe volume in healthy individuals. One other interesting variant in tight LD with our index variant (rs4072037, R2\\u2009=\\u20090.94) is a missense SNP in the MUC1 gene that decreased levels of blood magnesium concentrations. It is not clear how decreased magnesium levels are involved in decreased occipital brain volume, but it is an interesting avenue to explore as magnesium is known to be important for neural transmissions and magnesium infusions have anti-convulsive effects and is still used to prevent convulsions in pre-eclampsia.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 20743, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Using genetic correlation analysis, we did not find a strong significant genetic correlation between most of the brain lobes, which suggests that the genetic basis of the brain lobes is largely independent. We also did not find significant genetic overlap between lobar brain volumes and neurological and psychiatric disease outcomes. The most significant genetic correlation with brain lobar volume and diseases we observed was between occipital lobe volume and Parkinson\\u2019s disease (rg\\u2009=\\u20090.18, p\\u2009=\\u20090.03). However, this finding was not significant after multiple testing correction, leading us to report this finding with caution. The absence of significant genetic correlations between other brain lobes and clinical diseases could be due to true absence of a genetic overlap. However, other explanations can be put forward. First, it could also mean that our lobar volume GWAS and those for other diseases were still too underpowered to show significant genetic correlations. Second, the anatomical boundaries for the different lobes can be quite arbitrary and do not necessarily have to coincide with underlying gene function or biological processes leading to neurological or psychiatric disorders.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 21956, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"There are several limitations to our study. First, we have accepted differences in analytical methods of the magnetic resonance imaging (MRI) scans to allow for the largest sample size to be studied. This might have resulted in different effects over the studies. However, we did not observe significant heterogeneity after correcting for multiple testing for the six loci. In addition, a sensitivity analysis showed similar effects of the genome-wide significant variants for the studies using the kNN algorithm in comparison to the other studies. False negative findings due to differences in analytical methods of the MRI scans cannot be excluded. Second, a limitation of our study is that a different reference panel for imputation was used for the discovery and replication sample. This is due to the historic limited availability of the HRC reference panel at the initiation of this study. As the variants were well imputed (R2\\u2009>\\u20090.5) in all studies, this is not expected to have influenced the results, although it is possible that additional variants may be discovered if the larger HRC reference panel would be used in future studies. Last, for the UK Biobank only gray matter parcellations were available to us. Despite this limitation, we were able to replicate our findings. This suggests that the identified variants have an effect on gray as well as white matter volumes.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 23346, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"In summary, brain lobar volumes are differentially heritable traits, which can in large part be explained by common genetic variation. We identified six loci where genotypes are associated with specific brain lobes, four of which have not been implicated in brain morphology before. These loci are compelling targets for functional research to identify the biology behind their genetic signals.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 23741, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_1\"}, \"text\": \"Methods\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 23749, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Study population\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 23766, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The study sample consisted of dementia- and stroke-free individuals with quantitative brain MRI and genome-wide genotypes from 19 population- and family-based cohort studies participating in the Cohorts of Heart and Aging Research in Genomic Epidemiology consortium and the case\\u2013control Alzheimer\\u2019s Disease Neuroimaging Initiative study. In total, 16,016 participants were included, 15,269 participants of European ancestry, 405 African Americans, 211 Chinese, and 131 Malay. We attempted to replicate our findings in 8,789 European ancestry individuals from the UK Biobank, an ongoing prospective population-based cohort study located in the United Kingdom. Descriptive statistics of all populations are provided in Supplementary Data\\u00a01. DNA from whole blood was extracted and genome-wide genotyping was performed using a range of commercially available genotyping arrays. Genotype imputations were performed in each discovery cohort using 1000 Genomes version 1 as reference and using the Haplotype Reference Consortium (HRC)\\u00a0version 1.1 in the replication cohort (Supplementary Data\\u00a02).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 24863, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"MRI methods\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 24875, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Three-dimensional T1-weighted brain MRI data were acquired by each cohort (Supplementary Data\\u00a01). Cohorts in the discovery sample segmented the T1-weighted images into supra-tentorial gray matter, white matter, and cerebrospinal fluid. The methods of image segmentation varied across study cohorts (Supplementary Methods). However, the majority used a previously described kNN algorithm, which was trained on six manually labeled atlases, or in-house image-processing pipelines. In each study, MRI scans were performed and processed with automated protocols, without reference to clinical or genetic information. We studied the total volume (sum of white and gray matter and the left and right hemisphere) of the frontal, parietal, temporal, and occipital brain lobes, adjusted for intracranial volume. Descriptive information of the lobar volumes across the different studies is provided in Supplementary Data\\u00a03. Differences in average brain lobar volumes were accepted as differences in MRI acquisition, processing, segmentation, and demographics, which exist over cohorts. As a replication, we used the released volume measurements of 8,789 UK Biobank participants, extracted using the FreeSurfer software version 6.0, which obtains lobar volumes by adding up regions of interest volumes. As only FreeSurfer gray matter volumes were available for this study sample, the replication sample volumes were smaller than the volumes in the discovery sample (Supplementary Data\\u00a03).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 26356, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Estimation of heritability\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 26383, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The heritability of lobar brain volumes was estimated using family structure in the Framingham Heart Study (n\\u2009=\\u20092080), which constitutes a community-based cohort of non-demented individuals without evidence of significant brain injury (e.g., stroke or multiple sclerosis). In total, 619 extended families with a family size of 3.6\\u2009\\u00b1\\u20096.6 individuals were included in the analyses. These families consisted of the following pairs of relatives: 316 parent\\u2013offspring, 1135 sibling, 340 avuncular, 1772 first cousin, and 826\\u2009second cousin pairs. We calculated additive genetic heritability without shared environmental effects (C) using a variance-components analysis under an AE model in SOLAR, adjusted for age, age2, and sex.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 27120, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"GWAS of lobar volumes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 27142, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Associations of imputed genotype dosages with lobar volumes were examined using linear regression analyses under an additive model. Associations were adjusted for age, age2, sex, the first four principal components to account for possible confounding due to population stratification, and study-specific covariates. Linear mixed models with estimated kinships were used for association analyses in cohorts with related samples. Details on the analysis methods used in each cohort are provided in Supplementary Data\\u00a01 and 2. Post-GWAS quality control (QC) was conducted using EasyQC and filtering. Genetic variants with a low imputation quality (R2\\u2009<\\u20090.5), a minor allele count <10, and allelic or locational mismatching of SNPs with the reference panel were removed prior to the meta-analyses. The number of variants after filtering and the genomic inflation per study are provided in Supplementary Data\\u00a04. After QC, summary statistics were adjusted by the genomic control method in each of the participating cohorts. We then performed two inverse-variance weighted fixed-effect meta-analyses in METAL. First, we meta-analyzed all participants of European ancestry, then performed a multi-ethnic meta-analysis including African Americans (n\\u2009=\\u2009405), Chinese (n\\u2009=\\u2009211), and Malay (n\\u2009=\\u2009131). After meta-analyses, genetic variants with a total sample size of <5000 were excluded. We performed conditional analysis on the index variants to determine whether there were multiple independent genome-wide significant variants in a locus using the Genome-Wide Complex Trait Analysis (GCTA) software (--cojo, --p-cojo). Genotypes in the Rotterdam-study (all 6291 individuals of the baseline cohort who were genotyped) were used as reference for this analysis. For loci with a genome-wide significant association (p\\u2009<\\u20095\\u2009\\u00d7\\u200910\\u22128), we tested for heterogeneity using the I2 statistic. In a sensitivity meta-analysis, we tested whether the studies using the kNN algorithm had similar effects of the genome-wide significant variants for the studies using the kNN-algorithm in comparison to the other studies. We also searched for candidate genes in the loci using publically available databases for differential expression of the SNPs (eQTL database in GTEx) and HaploReg, an online tool that summarizes the ENCODE database for epigenetic markings and proteins binding to DNA.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 29531, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Variance explained by common variants and genetic correlations\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 29594, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The variance explained by all SNPs, or SNP-based heritability, was calculated from summary statistics using LD score regression. The percentage of variance explained by all SNPs was determined based on meta-analysis results using the LD Hub. We used the same LD score regression to quantify the amount of genetic correlation between the four brain lobes and other brain-related traits and diseases, using summary statistics for meta-analyses of genetic studies of subcortical structures, intracranial volume, white matter hyperintensities, general cognitive ability, neuroticism, schizophrenia, attention-deficit/hyperactivity disorder, autism, major depressive disorder, bipolar disorder\\u00b8 Parkinson\\u2019s disease, and Alzheimer\\u2019s disease.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 30335, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Ethical compliance\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 30354, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"All participants, or their parents or guardians in the case of minors, provided written informed consent for study participation, the use of brain MRI data, and the use of their DNA for genetic research. Approval for the individual studies was obtained by the relevant local ethical committees and institutional review boards.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 30681, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Statistics and reproducibility\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 30712, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Software used for the data analysis of this study: EasyQC (www.genepi-regensburg.de/easyqc), FreeSurfer (https://surfer.nmr.mgh.harvard.edu/), GCTA (http://cnsgenomics.com/software/gcta/), GenABEL (http://www.genabel.org), GTeX (https://gtexportal.org/home/), HaploReg (https://www.encodeproject.org/software/haploreg/), HASE (https://github.com/roshchupkin/hase), LD-hub (http://ldsc.broadinstitute.org/ldhub/), LD score regression (https://github.com/bulik/ldsc), mach2qtl (https://www.nitrc.org/projects/mach2qtl/), METAL (http://csg.sph.umich.edu/abecasis/metal/), Perl (https://www.perl.org/), PLINK (https://www.cog-genomics.org/plink2), R (https://www.r-project.org/), SNPTEST (https://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html), and SOLAR (http://www.sfbr.org).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 31502, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Reporting summary\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 31520, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Further information on research design is available in the\\u00a0Nature Research Reporting Summary linked to this article.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 31638, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_1\"}, \"text\": \"Supplementary information\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 31664, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Publisher\\u2019s note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 31803, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_1\"}, \"text\": \"Supplementary information\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 31829, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"paragraph\"}, \"text\": \"Supplementary information accompanies this paper at 10.1038/s42003-019-0537-9.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 31908, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"title\"}, \"text\": \"Author contributions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 31929, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"paragraph\"}, \"text\": \"(Co)wrote draft: S.J.v.d.L., M.J.K., N.A., H.H.H.A., C.D. Meta-analysis: S.J.v.d.L. Analyzed data: G.C., C.S., A.V.S., E.H., J.C.B., D.P.H., S.H., E.B.v.d.A., M. Bernard, L.R.Y., F.C., J.W.C., Y.S., S.L., L.Y., G.V.R., N.J., Q.Y., R.A.M., S.B., C.Y.C., P.M., W.Z., P.A.N., B.M.P., Z.P., P.M.T. Critical revision of manuscript: G.C., C.S., A.V.S., E.H., J.C.B., D.P.H., E.B.v.A., M. Bernard, L.R.Y., N.A., J.W.C., Y.S., O.L.L., S.L., J.v.d.G., L.Y., T.P., G.V.R., P.A., N.J., Q.Y., R.A.M., S.B., B.M., C.Y.C., P.M., W.J.N., M. Beekman, W.Z., P.A.N., M.W.V., H.S., Z.P., D.M.B., P.L.D.J., P.M.T., C.M.v.D., R.S., W.T.L., M.A.I., S.S. Contributed to sample collection: J.C.B., S.H., K.A., T.B.H., O.L.L., J.v.d.G., T.P., K.D.T., B.M., C.Y.C., K.R., D.v.H., T.Y.W., W.J.N., A.S.B., M. Beekman, P.A.N., C.C., L.L., B.M.P., M.K.I., M.W.V., H.S., Z.P., D.M.B., P.L.D.J., C.M.v.D., D.A.B., P.E.S., R.S., W.T.L., M.A.I., S.S., S.D., V.G. Secured funding: K.A., T.B.H., O.L.L., P.A., K.D.T., B.M., K.R., D.v.H., T.Y.W., W.J.N., A.S.B., C.C., L.L., B.M.P., M.K.I., H.S., D.M.B., P.L.d.J., C.M.v.D., D.A.B., P.E.S., R.S., W.T.L., M.A.I., S.S., S.D., V.G. Study supervision: H.H.H.A., C.D.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33106, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title\"}, \"text\": \"Data availability\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33124, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"paragraph\"}, \"text\": \"The genome-wide summary statistics that support the findings of this study will be made available via the NHGRI-EBI GWAS Catalog website (https://www.ebi.ac.uk/gwas/downloads/summarystatistics) upon publication. Quantitative brain MRI and genotype data are available from the corresponding authors H.H.H.A. and C.D.C. upon reasonable request.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33467, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title\"}, \"text\": \"Code availability\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33485, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"paragraph\"}, \"text\": \"No previously unreported custom computer code or mathematical algorithm was used to generate results central to the conclusions. The code is available upon request from the corresponding authors H.H.H.A. and C.D.C.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33700, \"infons\": {\"section_type\": \"COMP_INT\", \"type\": \"title\"}, \"text\": \"Competing interests\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33720, \"infons\": {\"section_type\": \"COMP_INT\", \"type\": \"paragraph\"}, \"text\": \"The authors declare no competing interests.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33764, \"infons\": {\"section_type\": \"REF\", \"type\": \"title\"}, \"text\": \"References\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33775, \"infons\": {\"fpage\": \"167\", \"lpage\": \"202\", \"name_0\": \"surname:Miller;given-names:EK\", \"name_1\": \"surname:Cohen;given-names:JD\", \"pub-id_doi\": \"10.1146/annurev.neuro.24.1.167\", \"pub-id_pmid\": \"11283309\", \"section_type\": \"REF\", \"source\": \"Annu. Rev. Neurosci.\", \"type\": \"ref\", \"volume\": \"24\", \"year\": \"2001\"}, \"text\": \"An integrative theory of prefrontal cortex function\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33827, \"infons\": {\"fpage\": \"205\", \"lpage\": \"212\", \"name_0\": \"surname:Culham;given-names:JC\", \"name_1\": \"surname:Valyear;given-names:KF\", \"pub-id_doi\": \"10.1016/j.conb.2006.03.005\", \"pub-id_pmid\": \"16563735\", \"section_type\": \"REF\", \"source\": \"Curr. Opin. Neurobiol.\", \"type\": \"ref\", \"volume\": \"16\", \"year\": \"2006\"}, \"text\": \"Human parietal cortex in action\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33859, \"infons\": {\"fpage\": \"6710\", \"lpage\": \"6715\", \"name_0\": \"surname:Levy;given-names:DA\", \"name_1\": \"surname:Bayley;given-names:PJ\", \"name_2\": \"surname:Squire;given-names:LR\", \"pub-id_doi\": \"10.1073/pnas.0401679101\", \"pub-id_pmid\": \"15090653\", \"section_type\": \"REF\", \"source\": \"Proc. Natl Acad. Sci. USA\", \"type\": \"ref\", \"volume\": \"101\", \"year\": \"2004\"}, \"text\": \"The anatomy of semantic knowledge: Medial vs. lateral temporal lobe\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 33927, \"infons\": {\"fpage\": \"289\", \"lpage\": \"292\", \"name_0\": \"surname:Mendez;given-names:MF\", \"name_1\": \"surname:Selwood;given-names:A\", \"name_2\": \"surname:Mastri;given-names:AR\", \"name_3\": \"surname:Frey;given-names:WH;suffix:2nd\", \"pub-id_doi\": \"10.1212/WNL.43.2.289\", \"pub-id_pmid\": \"8437691\", \"section_type\": \"REF\", \"source\": \"Neurology\", \"type\": \"ref\", \"volume\": \"43\", \"year\": \"1993\"}, \"text\": \"Pick\\u2019s disease versus Alzheimer\\u2019s disease: a comparison of clinical characteristics\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34015, \"infons\": {\"fpage\": \"677\", \"lpage\": \"687\", \"name_0\": \"surname:de Lanerolle;given-names:NC\", \"pub-id_doi\": \"10.1046/j.1528-1157.2003.32701.x\", \"pub-id_pmid\": \"12752467\", \"section_type\": \"REF\", \"source\": \"Epilepsia\", \"type\": \"ref\", \"volume\": \"44\", \"year\": \"2003\"}, \"text\": \"A retrospective analysis of hippocampal pathology in human temporal lobe epilepsy: evidence for distinctive patient subcategories\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34145, \"infons\": {\"fpage\": \"198\", \"lpage\": \"204\", \"name_0\": \"surname:Srinivasa;given-names:RN\", \"pub-id_doi\": \"10.1148/radiol.2015142488\", \"pub-id_pmid\": \"26218598\", \"section_type\": \"REF\", \"source\": \"Radiology\", \"type\": \"ref\", \"volume\": \"278\", \"year\": \"2016\"}, \"text\": \"Cardiovascular risk factors associated with smaller brain volumes in regions identified as early predictors of cognitive decline\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34274, \"infons\": {\"fpage\": \"10235\", \"lpage\": \"10242\", \"name_0\": \"surname:Hulshoff Pol;given-names:HE\", \"pub-id_doi\": \"10.1523/JNEUROSCI.1312-06.2006\", \"pub-id_pmid\": \"17021179\", \"section_type\": \"REF\", \"source\": \"J. Neurosci.\", \"type\": \"ref\", \"volume\": \"26\", \"year\": \"2006\"}, \"text\": \"Genetic contributions to human brain morphology and intelligence\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34339, \"infons\": {\"fpage\": \"32760\", \"name_0\": \"surname:Wen;given-names:W\", \"pub-id_doi\": \"10.1038/srep32760\", \"pub-id_pmid\": \"27595976\", \"section_type\": \"REF\", \"source\": \"Sci. Rep.\", \"type\": \"ref\", \"volume\": \"6\", \"year\": \"2016\"}, \"text\": \"Distinct genetic influences on cortical and subcortical brain structures\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34412, \"infons\": {\"fpage\": \"937 e5\", \"lpage\": \"18\", \"name_0\": \"surname:Batouli;given-names:SA\", \"pub-id_doi\": \"10.1016/j.neurobiolaging.2013.10.079\", \"section_type\": \"REF\", \"source\": \"Neurobiol. Aging\", \"type\": \"ref\", \"volume\": \"35\", \"year\": \"2014\"}, \"text\": \"Heritability of brain volumes in older adults: the Older Australian Twins Study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34492, \"infons\": {\"fpage\": \"816\", \"lpage\": \"824\", \"name_0\": \"surname:Baare;given-names:WF\", \"pub-id_doi\": \"10.1093/cercor/11.9.816\", \"pub-id_pmid\": \"11532887\", \"section_type\": \"REF\", \"source\": \"Cereb. Cortex\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2001\"}, \"text\": \"Quantitative genetic modeling of variation in human brain morphology\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34561, \"infons\": {\"fpage\": \"3176\", \"lpage\": \"3181\", \"name_0\": \"surname:Geschwind;given-names:DH\", \"name_1\": \"surname:Miller;given-names:BL\", \"name_2\": \"surname:DeCarli;given-names:C\", \"name_3\": \"surname:Carmelli;given-names:D\", \"pub-id_doi\": \"10.1073/pnas.052494999\", \"pub-id_pmid\": \"11867730\", \"section_type\": \"REF\", \"source\": \"Proc. Natl Acad. Sci. USA\", \"type\": \"ref\", \"volume\": \"99\", \"year\": \"2002\"}, \"text\": \"Heritability of lobar brain volumes in twins supports genetic models of cerebral laterality and handedness\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34668, \"infons\": {\"fpage\": \"1174\", \"lpage\": \"1182\", \"name_0\": \"surname:Gilmore;given-names:JH\", \"pub-id_pmid\": \"20063301\", \"section_type\": \"REF\", \"source\": \"Hum. Brain Mapp.\", \"type\": \"ref\", \"volume\": \"31\", \"year\": \"2010\"}, \"text\": \"Genetic and environmental contributions to neonatal brain structure: A twin study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34750, \"infons\": {\"fpage\": \"1117\", \"lpage\": \"1125\", \"name_0\": \"surname:Yoon;given-names:U\", \"name_1\": \"surname:Fahim;given-names:C\", \"name_2\": \"surname:Perusse;given-names:D\", \"name_3\": \"surname:Evans;given-names:AC\", \"pub-id_doi\": \"10.1016/j.neuroimage.2010.01.007\", \"pub-id_pmid\": \"20074649\", \"section_type\": \"REF\", \"source\": \"Neuroimage\", \"type\": \"ref\", \"volume\": \"53\", \"year\": \"2010\"}, \"text\": \"Lateralized genetic and environmental influences on human brain morphology of 8-year-old twins\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34845, \"infons\": {\"fpage\": \"987\", \"lpage\": \"993\", \"name_0\": \"surname:Wallace;given-names:GL\", \"pub-id_doi\": \"10.1111/j.1469-7610.2006.01676.x\", \"pub-id_pmid\": \"17073977\", \"section_type\": \"REF\", \"source\": \"J. Child Psychol. Psychiatry\", \"type\": \"ref\", \"volume\": \"47\", \"year\": \"2006\"}, \"text\": \"A pediatric twin study of brain morphometry\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34889, \"infons\": {\"fpage\": \"218\", \"lpage\": \"223\", \"name_0\": \"surname:DeStefano;given-names:AL\", \"pub-id_doi\": \"10.1097/WAD.0b013e31819cadd8\", \"pub-id_pmid\": \"19812462\", \"section_type\": \"REF\", \"source\": \"Alzheimer Dis. Assoc. Disord.\", \"type\": \"ref\", \"volume\": \"23\", \"year\": \"2009\"}, \"text\": \"Bivariate heritability of total and regional brain volumes: the Framingham Study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 34970, \"infons\": {\"fpage\": \"224\", \"lpage\": \"229\", \"name_0\": \"surname:Hibar;given-names:DP\", \"pub-id_doi\": \"10.1038/nature14101\", \"pub-id_pmid\": \"25607358\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"520\", \"year\": \"2015\"}, \"text\": \"Common genetic variants influence human subcortical brain structures\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 35039, \"infons\": {\"section_type\": \"REF\", \"type\": \"ref\"}, \"text\": \"Adams, H. H. et al. Novel genetic loci underlying human intracranial volume identified through genome-wide association. Nat. Neurosci. 19,1569\\u20131582 (2016).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 35197, \"infons\": {\"fpage\": \"13624\", \"name_0\": \"surname:Hibar;given-names:DP\", \"pub-id_doi\": \"10.1038/ncomms13624\", \"pub-id_pmid\": \"28098162\", \"section_type\": \"REF\", \"source\": \"Nat. Commun.\", \"type\": \"ref\", \"volume\": \"8\", \"year\": \"2017\"}, \"text\": \"Novel genetic loci associated with hippocampal volume\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 35251, \"infons\": {\"fpage\": \"3555\", \"lpage\": \"3557\", \"name_0\": \"surname:Machiela;given-names:MJ\", \"name_1\": \"surname:Chanock;given-names:SJ\", \"pub-id_doi\": \"10.1093/bioinformatics/btv402\", \"pub-id_pmid\": \"26139635\", \"section_type\": \"REF\", \"source\": \"Bioinformatics\", \"type\": \"ref\", \"volume\": \"31\", \"year\": \"2015\"}, \"text\": \"LDlink: a web-based application for exploring population-specific haplotype structure and linking correlated alleles of possible functional variants\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 35400, \"infons\": {\"fpage\": \"491\", \"name_0\": \"surname:Estrada;given-names:K\", \"pub-id_doi\": \"10.1038/ng.2249\", \"pub-id_pmid\": \"22504420\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"44\", \"year\": \"2012\"}, \"text\": \"Genome-wide meta-analysis identifies 56 bone mineral density loci and reveals 14 loci associated with risk of fracture\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 35519, \"infons\": {\"fpage\": \"832\", \"lpage\": \"838\", \"name_0\": \"surname:Lango Allen;given-names:H\", \"pub-id_doi\": \"10.1038/nature09410\", \"pub-id_pmid\": \"20881960\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"467\", \"year\": \"2010\"}, \"text\": \"Hundreds of variants clustered in genomic loci and biological pathways affect human height\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 35610, \"infons\": {\"fpage\": \"1164\", \"name_0\": \"surname:Heid;given-names:IM\", \"pub-id_doi\": \"10.1038/ng1111-1164a\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"43\", \"year\": \"2011\"}, \"text\": \"Meta-analysis identifies 13 new loci associated with waist-hip ratio and reveals sexual dimorphism in the genetic basis of fat distribution\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 35750, \"infons\": {\"fpage\": \"1155\", \"lpage\": \"1168\", \"name_0\": \"surname:van der Valk;given-names:RJP\", \"pub-id_doi\": \"10.1093/hmg/ddu510\", \"pub-id_pmid\": \"25281659\", \"section_type\": \"REF\", \"source\": \"Hum. Mol. Genet.\", \"type\": \"ref\", \"volume\": \"24\", \"year\": \"2015\"}, \"text\": \"A novel common variant in DCST2 is associated with length in early life and height in adulthood\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 35846, \"infons\": {\"fpage\": \"1053\", \"lpage\": \"1055\", \"name_0\": \"surname:Grinberg;given-names:I\", \"pub-id_doi\": \"10.1038/ng1420\", \"pub-id_pmid\": \"15338008\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"36\", \"year\": \"2004\"}, \"text\": \"Heterozygous deletion of the linked genes ZIC1 and ZIC4 is involved in Dandy-Walker malformation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 35943, \"infons\": {\"fpage\": \"378\", \"lpage\": \"388\", \"name_0\": \"surname:Twigg;given-names:SR\", \"pub-id_doi\": \"10.1016/j.ajhg.2015.07.007\", \"pub-id_pmid\": \"26340333\", \"section_type\": \"REF\", \"source\": \"Am. J. Hum. Genet.\", \"type\": \"ref\", \"volume\": \"97\", \"year\": \"2015\"}, \"text\": \"Gain-of-function mutations in ZIC1 are associated with coronal craniosynostosis and learning disability\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 36047, \"infons\": {\"fpage\": \"843\", \"lpage\": \"854\", \"name_0\": \"surname:Habas;given-names:R\", \"name_1\": \"surname:Kato;given-names:Y\", \"name_2\": \"surname:He;given-names:X\", \"pub-id_doi\": \"10.1016/S0092-8674(01)00614-6\", \"pub-id_pmid\": \"11779461\", \"section_type\": \"REF\", \"source\": \"Cell\", \"type\": \"ref\", \"volume\": \"107\", \"year\": \"2001\"}, \"text\": \"Wnt/Frizzled activation of Rho regulates vertebrate gastrulation and requires a novel formin homology protein Daam1\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 36163, \"infons\": {\"fpage\": \"957\", \"lpage\": \"966\", \"name_0\": \"surname:Matusek;given-names:T\", \"pub-id_doi\": \"10.1242/dev.02266\", \"pub-id_pmid\": \"16469972\", \"section_type\": \"REF\", \"source\": \"Development\", \"type\": \"ref\", \"volume\": \"133\", \"year\": \"2006\"}, \"text\": \"The Drosophila formin DAAM regulates the tracheal cuticle pattern through organizing the actin cytoskeleton\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 36271, \"infons\": {\"fpage\": \"62\", \"lpage\": \"67\", \"name_0\": \"surname:Salomon;given-names:SN\", \"name_1\": \"surname:Haber;given-names:M\", \"name_2\": \"surname:Murai;given-names:KK\", \"name_3\": \"surname:Dunn;given-names:RJ\", \"pub-id_doi\": \"10.1016/j.neulet.2008.09.051\", \"pub-id_pmid\": \"18832009\", \"section_type\": \"REF\", \"source\": \"Neurosci. Lett.\", \"type\": \"ref\", \"volume\": \"447\", \"year\": \"2008\"}, \"text\": \"Localization of the Diaphanous-related formin Daam1 to neuronal dendrites\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 36345, \"infons\": {\"fpage\": \"143\", \"lpage\": \"150\", \"name_0\": \"surname:Kida;given-names:Y\", \"name_1\": \"surname:Shiraishi;given-names:T\", \"name_2\": \"surname:Ogura;given-names:T\", \"pub-id_doi\": \"10.1016/j.devbrainres.2004.07.014\", \"pub-id_pmid\": \"15464228\", \"section_type\": \"REF\", \"source\": \"Dev. Brain Res.\", \"type\": \"ref\", \"volume\": \"153\", \"year\": \"2004\"}, \"text\": \"Identification of chick and mouse Daam1 and Daam2 genes and their expression patterns in the central nervous system\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 36461, \"infons\": {\"fpage\": \"e1001045\", \"issue\": \"8\", \"name_0\": \"surname:Meyer;given-names:Tamra E.\", \"name_1\": \"surname:Verwoert;given-names:Germaine C.\", \"name_10\": \"surname:Harris;given-names:Tamara B.\", \"name_11\": \"surname:Yang;given-names:Qiong\", \"name_12\": \"surname:Dehghan;given-names:Abbas\", \"name_13\": \"surname:Aspelund;given-names:Thor\", \"name_14\": \"surname:Katz;given-names:Ronit\", \"name_15\": \"surname:Homuth;given-names:Georg\", \"name_16\": \"surname:Kocher;given-names:Thomas\", \"name_17\": \"surname:Rettig;given-names:Rainer\", \"name_18\": \"surname:Ried;given-names:Janina S.\", \"name_19\": \"surname:Gieger;given-names:Christian\", \"name_2\": \"surname:Hwang;given-names:Shih-Jen\", \"name_20\": \"surname:Prucha;given-names:Hanna\", \"name_21\": \"surname:Pfeufer;given-names:Arne\", \"name_22\": \"surname:Meitinger;given-names:Thomas\", \"name_23\": \"surname:Coresh;given-names:Josef\", \"name_24\": \"surname:Hofman;given-names:Albert\", \"name_25\": \"surname:Sarnak;given-names:Mark J.\", \"name_26\": \"surname:Chen;given-names:Yii-Der Ida\", \"name_27\": \"surname:Uitterlinden;given-names:Andr\\u00e9 G.\", \"name_28\": \"surname:Chakravarti;given-names:Aravinda\", \"name_29\": \"surname:Psaty;given-names:Bruce M.\", \"name_3\": \"surname:Glazer;given-names:Nicole L.\", \"name_30\": \"surname:van Duijn;given-names:Cornelia M.\", \"name_31\": \"surname:Kao;given-names:W. H. Linda\", \"name_32\": \"surname:Witteman;given-names:Jacqueline C. M.\", \"name_33\": \"surname:Gudnason;given-names:Vilmundur\", \"name_34\": \"surname:Siscovick;given-names:David S.\", \"name_35\": \"surname:Fox;given-names:Caroline S.\", \"name_36\": \"surname:K\\u00f6ttgen;given-names:Anna\", \"name_4\": \"surname:Smith;given-names:Albert V.\", \"name_5\": \"surname:van Rooij;given-names:Frank J. A.\", \"name_6\": \"surname:Ehret;given-names:Georg B.\", \"name_7\": \"surname:Boerwinkle;given-names:Eric\", \"name_8\": \"surname:Felix;given-names:Janine F.\", \"name_9\": \"surname:Leak;given-names:Tennille S.\", \"pub-id_doi\": \"10.1371/journal.pgen.1001045\", \"pub-id_pmid\": \"20700443\", \"section_type\": \"REF\", \"source\": \"PLoS Genetics\", \"type\": \"ref\", \"volume\": \"6\", \"year\": \"2010\"}, \"text\": \"Genome-Wide Association Studies of Serum Magnesium, Potassium, and Sodium Concentrations Identify Six Loci Influencing Serum Magnesium Levels\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 36603, \"infons\": {\"fpage\": \"81\", \"lpage\": \"91\", \"name_0\": \"surname:Borges;given-names:LF\", \"name_1\": \"surname:Gucer;given-names:G\", \"pub-id_doi\": \"10.1111/j.1528-1157.1978.tb05015.x\", \"pub-id_pmid\": \"414910\", \"section_type\": \"REF\", \"source\": \"Epilepsia\", \"type\": \"ref\", \"volume\": \"19\", \"year\": \"1978\"}, \"text\": \"Effect of magnesium on epileptic foci\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 36641, \"infons\": {\"fpage\": \"543\", \"lpage\": \"552\", \"name_0\": \"surname:Pritchard;given-names:JA\", \"name_1\": \"surname:Pritchard;given-names:SA\", \"pub-id_doi\": \"10.1016/0002-9378(75)90042-3\", \"pub-id_pmid\": \"1180300\", \"section_type\": \"REF\", \"source\": \"Am. J. Obstet. Gynecol.\", \"type\": \"ref\", \"volume\": \"123\", \"year\": \"1975\"}, \"text\": \"Standardized treatment of 154 consecutive cases of eclampsia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 36702, \"infons\": {\"fpage\": \"68\", \"lpage\": \"74\", \"name_0\": \"surname:Genomes Project;given-names:C\", \"pub-id_doi\": \"10.1038/nature15393\", \"pub-id_pmid\": \"26432245\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"526\", \"year\": \"2015\"}, \"text\": \"A global reference for human genetic variation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 36749, \"infons\": {\"fpage\": \"71\", \"lpage\": \"81\", \"name_0\": \"surname:Vrooman;given-names:HA\", \"pub-id_doi\": \"10.1016/j.neuroimage.2007.05.018\", \"pub-id_pmid\": \"17572111\", \"section_type\": \"REF\", \"source\": \"Neuroimage\", \"type\": \"ref\", \"volume\": \"37\", \"year\": \"2007\"}, \"text\": \"Multi-spectral brain tissue segmentation using automatically trained k-Nearest-Neighbor classification\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 36852, \"infons\": {\"fpage\": \"968\", \"lpage\": \"980\", \"name_0\": \"surname:Desikan;given-names:RS\", \"pub-id_doi\": \"10.1016/j.neuroimage.2006.01.021\", \"pub-id_pmid\": \"16530430\", \"section_type\": \"REF\", \"source\": \"Neuroimage\", \"type\": \"ref\", \"volume\": \"31\", \"year\": \"2006\"}, \"text\": \"An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 36973, \"infons\": {\"fpage\": \"11\", \"lpage\": \"22\", \"name_0\": \"surname:Fischl;given-names:B\", \"pub-id_doi\": \"10.1093/cercor/bhg087\", \"pub-id_pmid\": \"14654453\", \"section_type\": \"REF\", \"source\": \"Cereb. Cortex\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2004\"}, \"text\": \"Automatically parcellating the human cerebral cortex\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 37026, \"infons\": {\"fpage\": \"1198\", \"lpage\": \"1211\", \"name_0\": \"surname:Almasy;given-names:L\", \"name_1\": \"surname:Blangero;given-names:J\", \"pub-id_doi\": \"10.1086/301844\", \"pub-id_pmid\": \"9545414\", \"section_type\": \"REF\", \"source\": \"Am. J. Hum. Genet.\", \"type\": \"ref\", \"volume\": \"62\", \"year\": \"1998\"}, \"text\": \"Multipoint quantitative-trait linkage analysis in general pedigrees\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 37094, \"infons\": {\"fpage\": \"1192\", \"lpage\": \"1212\", \"name_0\": \"surname:Winkler;given-names:TW\", \"pub-id_doi\": \"10.1038/nprot.2014.071\", \"pub-id_pmid\": \"24762786\", \"section_type\": \"REF\", \"source\": \"Nat. Protoc.\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2014\"}, \"text\": \"Quality control and conduct of genome-wide association meta-analyses\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 37163, \"infons\": {\"fpage\": \"2190\", \"lpage\": \"2191\", \"name_0\": \"surname:Willer;given-names:CJ\", \"name_1\": \"surname:Li;given-names:Y\", \"name_2\": \"surname:Abecasis;given-names:GR\", \"pub-id_doi\": \"10.1093/bioinformatics/btq340\", \"pub-id_pmid\": \"20616382\", \"section_type\": \"REF\", \"source\": \"Bioinformatics\", \"type\": \"ref\", \"volume\": \"26\", \"year\": \"2010\"}, \"text\": \"METAL: fast and efficient meta-analysis of genomewide association scans\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 37235, \"infons\": {\"fpage\": \"76\", \"lpage\": \"82\", \"name_0\": \"surname:Yang;given-names:J\", \"name_1\": \"surname:Lee;given-names:SH\", \"name_2\": \"surname:Goddard;given-names:ME\", \"name_3\": \"surname:Visscher;given-names:PM\", \"pub-id_doi\": \"10.1016/j.ajhg.2010.11.011\", \"pub-id_pmid\": \"21167468\", \"section_type\": \"REF\", \"source\": \"Am. J. Hum. Genet.\", \"type\": \"ref\", \"volume\": \"88\", \"year\": \"2011\"}, \"text\": \"GCTA: a tool for genome-wide complex trait analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 37287, \"infons\": {\"fpage\": \"369\", \"lpage\": \"375\", \"name_0\": \"surname:Yang;given-names:J\", \"pub-id_doi\": \"10.1038/ng.2213\", \"pub-id_pmid\": \"22426310\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"44\", \"year\": \"2012\"}, \"text\": \"Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 37416, \"infons\": {\"fpage\": \"648\", \"lpage\": \"660\", \"name_0\": \"surname:Consortium;given-names:GT\", \"pub-id_doi\": \"10.1126/science.1262110\", \"pub-id_pmid\": \"25954001\", \"section_type\": \"REF\", \"source\": \"Science\", \"type\": \"ref\", \"volume\": \"348\", \"year\": \"2015\"}, \"text\": \"Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 37524, \"infons\": {\"fpage\": \"D930\", \"issue\": \"Database issue\", \"lpage\": \"D934\", \"name_0\": \"surname:Ward;given-names:LD\", \"name_1\": \"surname:Kellis;given-names:M\", \"pub-id_doi\": \"10.1093/nar/gkr917\", \"pub-id_pmid\": \"22064851\", \"section_type\": \"REF\", \"source\": \"Nucleic Acids Res.\", \"type\": \"ref\", \"volume\": \"40\", \"year\": \"2012\"}, \"text\": \"HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 37667, \"infons\": {\"fpage\": \"1236\", \"lpage\": \"1241\", \"name_0\": \"surname:Bulik-Sullivan;given-names:B\", \"pub-id_doi\": \"10.1038/ng.3406\", \"pub-id_pmid\": \"26414676\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"47\", \"year\": \"2015\"}, \"text\": \"An atlas of genetic correlations across human diseases and traits\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 37733, \"infons\": {\"fpage\": \"272\", \"issue\": \"2\", \"lpage\": \"279\", \"name_0\": \"surname:Zheng;given-names:Jie\", \"name_1\": \"surname:Erzurumluoglu;given-names:A. Mesut\", \"name_10\": \"surname:Warrington;given-names:Nicole M.\", \"name_11\": \"surname:Finucane;given-names:Hilary K.\", \"name_12\": \"surname:Price;given-names:Alkes L.\", \"name_13\": \"surname:Bulik-Sullivan;given-names:Brendan K.\", \"name_14\": \"surname:Anttila;given-names:Verneri\", \"name_15\": \"surname:Paternoster;given-names:Lavinia\", \"name_16\": \"surname:Gaunt;given-names:Tom R.\", \"name_17\": \"surname:Evans;given-names:David M.\", \"name_18\": \"surname:Neale;given-names:Benjamin M.\", \"name_2\": \"surname:Elsworth;given-names:Benjamin L.\", \"name_3\": \"surname:Kemp;given-names:John P.\", \"name_4\": \"surname:Howe;given-names:Laurence\", \"name_5\": \"surname:Haycock;given-names:Philip C.\", \"name_6\": \"surname:Hemani;given-names:Gibran\", \"name_7\": \"surname:Tansey;given-names:Katherine\", \"name_8\": \"surname:Laurin;given-names:Charles\", \"name_9\": \"surname:Pourcain;given-names:Beate St.\", \"pub-id_doi\": \"10.1093/bioinformatics/btw613\", \"pub-id_pmid\": \"27663502\", \"section_type\": \"REF\", \"source\": \"Bioinformatics\", \"type\": \"ref\", \"volume\": \"33\", \"year\": \"2016\"}, \"text\": \"LD Hub: a centralized database and web interface to perform LD score regression that maximizes the potential of summary level GWAS data for SNP heritability and genetic correlation analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 37923, \"infons\": {\"fpage\": \"398\", \"lpage\": \"409\", \"name_0\": \"surname:Verhaaren;given-names:BF\", \"pub-id_doi\": \"10.1161/CIRCGENETICS.114.000858\", \"pub-id_pmid\": \"25663218\", \"section_type\": \"REF\", \"source\": \"Circ. Cardiovasc. Genet.\", \"type\": \"ref\", \"volume\": \"8\", \"year\": \"2015\"}, \"text\": \"Multiethnic genome-wide association study of cerebral white matter hyperintensities on MRI\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 38014, \"infons\": {\"fpage\": \"183\", \"lpage\": \"192\", \"name_0\": \"surname:Davies;given-names:G\", \"pub-id_doi\": \"10.1038/mp.2014.188\", \"pub-id_pmid\": \"25644384\", \"section_type\": \"REF\", \"source\": \"Mol. Psychiatry\", \"type\": \"ref\", \"volume\": \"20\", \"year\": \"2015\"}, \"text\": \"Genetic contributions to variation in general cognitive function: a meta-analysis of genome-wide association studies in the CHARGE consortium (N=53 949)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 38167, \"infons\": {\"fpage\": \"229\", \"lpage\": \"237\", \"name_0\": \"surname:Turley;given-names:P\", \"pub-id_doi\": \"10.1038/s41588-017-0009-4\", \"pub-id_pmid\": \"29292387\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"50\", \"year\": \"2018\"}, \"text\": \"Multi-trait analysis of genome-wide association summary statistics using MTAG\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 38245, \"infons\": {\"section_type\": \"REF\", \"type\": \"ref\"}, \"text\": \"Bipolar Disorder and Schizophrenia Working Group of the Psychiatric Genomics Consortium. Genomic dissection of bipolar disorder and schizophrenia, including 28 subphenotypes. Cell173, 1705.e16\\u20131715.e16 (2018).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 38457, \"infons\": {\"fpage\": \"63\", \"lpage\": \"75\", \"name_0\": \"surname:Demontis;given-names:D\", \"pub-id_doi\": \"10.1038/s41588-018-0269-7\", \"pub-id_pmid\": \"30478444\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2019\"}, \"text\": \"Discovery of the first genome-wide significant risk loci for attention deficit/hyperactivity disorder\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 38559, \"infons\": {\"fpage\": \"431\", \"lpage\": \"444\", \"name_0\": \"surname:Grove;given-names:J\", \"pub-id_doi\": \"10.1038/s41588-019-0344-8\", \"pub-id_pmid\": \"30804558\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2019\"}, \"text\": \"Identification of common genetic risk variants for autism spectrum disorder\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 38635, \"infons\": {\"fpage\": \"668\", \"lpage\": \"681\", \"name_0\": \"surname:Wray;given-names:NR\", \"pub-id_doi\": \"10.1038/s41588-018-0090-3\", \"pub-id_pmid\": \"29700475\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"50\", \"year\": \"2018\"}, \"text\": \"Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 38750, \"infons\": {\"fpage\": \"1511\", \"lpage\": \"1516\", \"name_0\": \"surname:Chang;given-names:D\", \"pub-id_doi\": \"10.1038/ng.3955\", \"pub-id_pmid\": \"28892059\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"49\", \"year\": \"2017\"}, \"text\": \"A meta-analysis of genome-wide association studies identifies 17 new Parkinson\\u2019s disease risk loci\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"bioctype\": \"BioCPassage\", \"offset\": 38851, \"infons\": {\"fpage\": \"1452\", \"lpage\": \"1458\", \"name_0\": \"surname:Lambert;given-names:JC\", \"pub-id_doi\": \"10.1038/ng.2802\", \"pub-id_pmid\": \"24162737\", \"section_type\": \"REF\", \"source\": \"Nat. Genet.\", \"type\": \"ref\", \"volume\": \"45\", \"year\": \"2013\"}, \"text\": \"Meta-analysis of 74,046 individuals identifies 11 new susceptibility loci for Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}], \"annotations\": [], \"relations\": []}]}]\n",
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading weights: 100%|| 770/770 [00:00<00:00, 1574.38it/s, Materializing param=model.encoder_input_proj.2.1.weight]                                                 \n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    }
   ],
   "source": [
    "pmcid = \"PMC6677735\"\n",
    "file_name = \"papers/s42003-019-0537-9.pdf\"  # Can be a local path or a URL\n",
    "df_lst = extract_tables_lst_from_paper(pmcid, file_name)  \n",
    "for i in [1]:\n",
    "    df_lst[i-1].to_csv(f\"tables/{file_name.split('/')[-1].replace('.pdf', '')}_table_{i}.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "e9b843b9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{\"source\": \"PMC\", \"date\": \"20230704\", \"key\": \"pmc.key\", \"infons\": {}, \"documents\": [{\"id\": \"10286470\", \"infons\": {\"license\": \"CC BY\"}, \"passages\": [{\"offset\": 0, \"infons\": {\"article-id_doi\": \"10.1186/s13024-023-00633-4\", \"article-id_pmc\": \"10286470\", \"article-id_pmid\": \"37349795\", \"article-id_publisher-id\": \"633\", \"elocation-id\": \"40\", \"kwd\": \"Alzheimer\\u2019s disease Genome-wide association study Cognitive domains Longitudinal measures Pleiotropy Pathway analysis\", \"license\": \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. The Creative Commons Public Domain Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies to the data made available in this article, unless otherwise stated in a credit line to the data.\", \"name_0\": \"surname:Kang;given-names:Moonil\", \"name_1\": \"surname:Ang;given-names:Ting Fang Alvin\", \"name_10\": \"surname:Klinedinst;given-names:Brandon\", \"name_11\": \"surname:Nakano;given-names:Connie\", \"name_12\": \"surname:Dumitrescu;given-names:Logan C.\", \"name_13\": \"surname:Durant;given-names:Alaina\", \"name_14\": \"surname:Hohman;given-names:Timothy J.\", \"name_15\": \"surname:Cuccaro;given-names:Michael L.\", \"name_16\": \"surname:Saykin;given-names:Andrew J.\", \"name_17\": \"surname:Kukull;given-names:Walter A.\", \"name_18\": \"surname:Bennett;given-names:David A.\", \"name_19\": \"surname:Wang;given-names:Li-San\", \"name_2\": \"surname:Devine;given-names:Sherral A.\", \"name_20\": \"surname:Mayeux;given-names:Richard P.\", \"name_21\": \"surname:Haines;given-names:Jonathan L.\", \"name_22\": \"surname:Pericak-Vance;given-names:Margaret A.\", \"name_23\": \"surname:Schellenberg;given-names:Gerard D.\", \"name_24\": \"surname:Crane;given-names:Paul K.\", \"name_25\": \"surname:Au;given-names:Rhoda\", \"name_26\": \"surname:Lunetta;given-names:Kathryn L.\", \"name_27\": \"surname:Mez;given-names:Jesse B.\", \"name_28\": \"surname:Farrer;given-names:Lindsay A.\", \"name_29\": \"surname:Hohman;given-names:Timothy J.\", \"name_3\": \"surname:Sherva;given-names:Richard\", \"name_30\": \"surname:Saykin;given-names:Andrew J.\", \"name_31\": \"surname:Haines;given-names:Jonathan L.\", \"name_32\": \"surname:Schellenberg;given-names:Gerard D.\", \"name_33\": \"surname:Farrer;given-names:Lindsay A.\", \"name_4\": \"surname:Mukherjee;given-names:Shubhabrata\", \"name_5\": \"surname:Trittschuh;given-names:Emily H.\", \"name_6\": \"surname:Gibbons;given-names:Laura E.\", \"name_7\": \"surname:Scollard;given-names:Phoebe\", \"name_8\": \"surname:Lee;given-names:Michael\", \"name_9\": \"surname:Choi;given-names:Seo-Eun\", \"section_type\": \"TITLE\", \"title\": \"Keywords\", \"type\": \"front\", \"volume\": \"18\", \"year\": \"2023\"}, \"text\": \"A genome-wide search for pleiotropy in more than 100,000 harmonized longitudinal cognitive domain scores\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 105, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract_title_1\"}, \"text\": \"Background\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 116, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"More than 75 common variant loci account for only a portion of the heritability for Alzheimer\\u2019s disease (AD). A more complete understanding of the genetic basis of AD can be deduced by exploring associations with AD-related endophenotypes.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 358, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract_title_1\"}, \"text\": \"Methods\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 366, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"We conducted genome-wide scans for cognitive domain performance using harmonized and co-calibrated scores derived by confirmatory factor analyses for executive function, language, and memory. We analyzed 103,796 longitudinal observations from 23,066 members of community-based (FHS, ACT, and ROSMAP) and clinic-based (ADRCs and ADNI) cohorts using generalized linear mixed models including terms for SNP, age, SNP\\u2009\\u00d7\\u2009age interaction, sex, education, and five ancestry principal components. Significance was determined based on a joint test of the SNP\\u2019s main effect and interaction with age. Results across datasets were combined using inverse-variance meta-analysis. Genome-wide tests of pleiotropy for each domain pair as the outcome were performed using PLACO software.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 1144, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract_title_1\"}, \"text\": \"Results\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 1152, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Individual domain and pleiotropy analyses revealed genome-wide significant (GWS) associations with five established loci for AD and AD-related disorders (BIN1, CR1, GRN, MS4A6A, and APOE) and eight novel loci. ULK2 was associated with executive function in the community-based cohorts (rs157405, P\\u2009=\\u20092.19\\u2009\\u00d7\\u200910\\u20139). GWS associations for language were identified with CDK14 in the clinic-based cohorts (rs705353, P\\u2009=\\u20091.73\\u2009\\u00d7\\u200910\\u20138) and LINC02712 in the total sample (rs145012974, P\\u2009=\\u20093.66\\u2009\\u00d7\\u200910\\u20138). GRN (rs5848, P\\u2009=\\u20094.21\\u2009\\u00d7\\u200910\\u20138) and PURG (rs117523305, P\\u2009=\\u20091.73\\u2009\\u00d7\\u200910\\u20138) were associated with memory in the total and community-based cohorts, respectively. GWS pleiotropy was observed for language and memory with LOC107984373 (rs73005629, P\\u2009=\\u20093.12\\u2009\\u00d7\\u200910\\u20138) in the clinic-based cohorts, and with NCALD (rs56162098, P\\u2009=\\u20091.23\\u2009\\u00d7\\u200910\\u20139) and PTPRD (rs145989094, P\\u2009=\\u20098.34\\u2009\\u00d7\\u200910\\u20139) in the community-based cohorts. GWS pleiotropy was also found for executive function and memory with OSGIN1 (rs12447050, P\\u2009=\\u20094.09\\u2009\\u00d7\\u200910\\u20138) and PTPRD (rs145989094, P\\u2009=\\u20093.85\\u2009\\u00d7\\u200910\\u20138) in the community-based cohorts. Functional studies have previously linked AD to ULK2, NCALD, and PTPRD.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2407, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract_title_1\"}, \"text\": \"Conclusion\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2418, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Our results provide some insight into biological pathways underlying processes leading to domain-specific cognitive impairment and AD, as well as a conduit toward a syndrome-specific precision medicine approach to AD. Increasing the number of participants with harmonized cognitive domain scores will enhance the discovery of additional genetic factors of cognitive decline leading to AD and related dementias.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2829, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract_title_1\"}, \"text\": \"Supplementary Information\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2855, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"The online version contains supplementary material available at 10.1186/s13024-023-00633-4.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2947, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"title_1\"}, \"text\": \"Background\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 2958, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Late-onset Alzheimer\\u2019s disease (AD) occurring after age 65 is the most common type of dementia and is highly heritable, estimated at 60% to 80%. Common single-nucleotide polymorphisms (SNPs) explain 24% to 33% of the total phenotypic variance of AD, of which up to 6% is accounted for by\\u00a0APOE. More than 75 loci affecting AD risk have been identified in several large-scale genome-wide association studies (GWAS), but much of the underlying genetic architecture of AD remains unknown.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 3446, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Although GWAS conducted in larger samples will undoubtedly reveal additional AD loci, understanding of the genetic influence on AD risk can be improved by examining the association with endophenotypes that potentially highlight specific pathways underlying the complex disease phenotype. Previously, numerous genetic associations have been identified for AD-related endophenotypes, such as cognitive performance, brain imaging traits, neuropathological traits, and biomarkers measured in cerebrospinal fluid. GWAS have found several loci for general cognitive ability, but most findings for specific cognitive domains are not genome-wide significant (GWS), inconsistent, and rarely replicated in independent datasets, perhaps because of the variability in neuropsychological (NP) tests administered across cohorts. To address this concern, Mukherjee and colleagues applied confirmatory factor analysis models to co-calibrate and harmonize composite scores for several cognitive domains. The scores obtained are on the same scale, making them comparable to each other regardless of the NP protocol.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 4544, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Here, we conducted a GWAS for cognitive scores of three domains derived from longitudinal, prospectively collected NP tests administered to participants of several large cohort studies. The statistical power for detecting associations with endophenotypes can be increased by studying outcomes of multiple correlated traits under a model of pleiotropy\\u2014a phenomenon where a single gene or variant affects multiple phenotypes. This approach has successfully identified novel associations for neuropathological processes in AD. Because measures of cognitive performance are highly heritable and correlated with each other, they are well suited as outcomes for cross-phenotype genetic association studies. Therefore, we also tested pleiotropy models for each pair of the three cognitive domains to identify novel loci which may be involved in AD.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 5388, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_1\"}, \"text\": \"Methods\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 5396, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Participants\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 5409, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"This study included non-Hispanic white participants of the Framingham Heart Study (FHS), National Institute on Aging sponsored Alzheimer\\u2019s Disease Research Centers whose phenotypic information was assembled and curated by the National Alzheimer\\u2019s Coordinating Center (NACC), the Adult Changes in Thought (ACT) Study, the Alzheimer\\u2019s Disease Neuroimaging Initiative (ADNI), and the Religious Orders Study/Rush Memory and Aging Project (ROSMAP). Briefly, FHS is a long-running multi-generation community-based study of cardiovascular disease and other age-related disorders, including cognitive decline and dementia. ACT and ROSMAP are also community-based cohorts that recruited unrelated cognitively normal participants who are followed longitudinally for cognitive disorders. Participants of NACC and ADNI were clinically ascertained for AD research and were cognitively normal or met the criteria for mild cognitive impairment (MCI) or AD at the time of enrollment. Extensive cognitive testing of participants of all is conducted at all visits. Details regarding the ascertainment, evaluation, and diagnosis of members of these five cohorts were reported elsewhere.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 6583, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Cognitive domain scores\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 6607, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Scores for executive function, language, and memory domains were derived as previously described. Scores are co-calibrated to put them on the same scale regardless of the cognitive battery administered. Briefly, an expert panel of neuropsychologists (EHT, AJS) and a behavioral neurologist (JBM) assigned each NP test item to one of the three domains. Confirmatory factor analysis in Mplus was used for co-calibration. Cognitive data from the most recent visit were first used to derive scores for each domain, with each domain modeled separately. Test items administered in multiple cohorts functioned as anchors for co-calibration. Parameters for anchor items were forced to be the same across studies to put scores across studies on the same metric. Within study, multiple models (including single factor and bifactor models) were considered with the choice of model determined based on a combination of model fit and concordance with neuropsychological theory. Next, each study\\u2019s item parameters from calibration of data at the last visit were fixed and used to obtain scores for each person at each time point. Co-calibrated cognitive scores with a standard error (SE)\\u2009>\\u20090.6 or derived solely from the mini-mental state examination (MMSE), which has a ceiling effect, were excluded. Time points less than age 60 were only available for FHS and were excluded due to concern that cognitive performance under age 60 may have a different genetic architecture that was only being captured in a single study.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 8121, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Genotype data processing\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 8146, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We obtained genome-wide SNP data that were processed and imputed using the Trans-Omics for Precision Medicine (TOPMed) reference panel and aligned to Genome Research Consortium human build 38 (GRCh38). Variants with poor imputation quality (r2\\u2009<\\u20090.3), minor allele frequencies (MAF)\\u2009<\\u20090.01, call rates\\u2009<\\u200995%, and Hardy-Weinberg Equilibrium (HWE) test\\u00a0p-value\\u2009<\\u20091\\u2009\\u00d7\\u200910\\u20136 were excluded, and approximately nine million variants remained for each cohort after quality control (QC). Principal components (PCs) of population structure were generated for individuals within each cohort using the set of post-QC variants that were pruned on the basis of a linkage disequilibrium (LD) threshold of 0.1 using the R package GENESIS. Measures of relatedness, kinship coefficients for family-based samples and empirical identity by descent (IBD) in the other samples, were estimated using established procedures.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 9070, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Genetic and phenotypic correlation estimation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 9116, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Genetic correlations between each pair of cognitive domain scores (executive function, language, and memory) were estimated in each cohort using GREML. The kinship matrix derived from self-reported FHS pedigrees was incorporated in the estimates using the kinship2 package. We used the empirical genetic relationship matrix (GRM) to account for relatedness among individuals in the other cohorts. To concurrently investigate SNP associations with both performance at the median age and change in cognitive function over time in each domain, we applied a joint test of the marginal genetic effects and gene\\u2009\\u00d7\\u2009age interaction together in a generalized linear mixed model framework with a random slope and intercept as implemented in the mixed\\u2010model association test for gene-environment interactions (MAGEE) R package. Models included terms for SNP, the interaction between SNP and age, and covariates for age, sex, educational level (less than high school, high school, some college, or college graduate), and the first five PCs represented as follows:where \\u03b1A, \\u03b1B, and \\u03b1C indicate the effects of age, sex, and educational level, respectively; \\u03b2G is the main SNP effect; \\u03b3X represents the SNP\\u2009\\u00d7\\u2009age interaction effect; \\u03b1i is the effect of the ith PC (i between 1 and 5); and r is a random intercept. We subtracted the median age for all observations for all individuals in the dataset from the individual\\u2019s age at each exam in order to center age because the intercept will refer to the mean outcome value when an individual\\u2019s baseline age is equal to the mean age at baseline in each dataset. Models also incorporated the GRM as a random effect.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 10785, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Cross-trait analyses\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 10806, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We performed cross-traits LD score regression to estimate genetic correlations across general cognitive function, cognitive domain scores, and neuropsychiatric disorders. We used GWAS summary statistics for general cognitive function (n\\u2009=\\u2009300,486), cognitive factor scores (n\\u2009=\\u200923,066) from the current study, neuropsychiatric disorders\\u2014AD, bipolar disorder, schizophrenia (n\\u2009=\\u2009420,531), and depression (n\\u2009=\\u2009370,457)\\u2014from the Pan-UK Biobank, and LD scores derived from the 1000 Genomes Project (phase 3) European samples. We only included 4,815,014 variants with imputation quality r2\\u2009>\\u20090.6 and MAF\\u2009>\\u20090.01 in cross-trait analyses.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 11465, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Genetic association analyses\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 11494, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Analyses were performed in each dataset separately, and the GWAS results were combined across datasets by meta-analyses. To correct systematic inflation in a joint test of the SNP\\u2019s main and interaction effects, we applied the joint meta-analysis method which considers the covariance between the main and interaction effects, and the inverse variance weighted approach in METAL. Meta-analyses were performed for each cognitive domain in the total sample, clinic-based cohorts (NACC and ADNI), and community-based cohorts (FHS, ACT, and ROSMAP) separately (Fig. S1). Results for the clinic- and community-based cohorts were considered separately because some associations might be unique to one of the cohort groups due to disparity in age or proportion of participants with AD. The genomic inflation factor (\\u03bb) was calculated for each GWAS and applied to adjust\\u00a0p-values for each test. A GWS threshold was set at\\u00a0P\\u2009=\\u20095\\u2009\\u00d7\\u200910\\u20138.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 12438, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Genome-wide pleiotropy analyses\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 12470, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We conducted a pleiotropy GWAS for each pair of cognitive domains in the total sample, clinic-based cohorts, and community-based cohorts using the pooled GWAS results from the joint meta-analysis (Fig. S1) and the R package PLACO. Because rejecting the global null hypothesis that neither phenotype is associated does not specifically imply the existence of pleiotropy, PLACO tests the composite null hypothesis that no more than one phenotype is associated with a variant. Thus, rejecting the composite null hypothesis implies that both phenotypes are associated with the variant,\\u00a0i.e., pleiotropy. This approach uses the product of the Z-statistics as the test statistic for the association of a given variant with each individual trait. The null distribution of the test statistic takes the form of a mixture distribution that allows for the variant to be associated with none or only one of the traits. Variants with squared\\u00a0Z-scores\\u2009>\\u200980 for one trait were removed because they could cause spurious pleiotropic signals. Because correlations between the Z-statistics for the association between a variant and the two traits can result in inflated type I errors, we adjusted for the Pearson correlation for variants with no effect (P\\u2009>\\u20091\\u2009\\u00d7\\u200910\\u20134) as suggested by the developers of the method.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 13782, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Pathway enrichment analyses\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 13810, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We performed several pathway analyses, each of which was seeded with genes containing variants associated with a single cognitive domain or pleiotropy for paired domains (P\\u2009<\\u20091\\u2009\\u00d7\\u200910\\u20134) in the respective GWAS, using the Ingenuity Pathway Analysis software (QIAGEN Inc.). Enrichment p-values for each canonical pathway were adjusted for a false discovery rate (FDR) using the Benjamini-Hochberg method, and an FDR-adjusted P-value threshold was set at 0.001 to account for the 18 separate pathway analyses (six single and paired domains multiplied by three sample strata).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14392, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_1\"}, \"text\": \"Results\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14400, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Cognitive domains are phenotypically and genetically correlated\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14464, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Characteristics of study participants\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14502, \"infons\": {\"file\": \"Tab1.xml\", \"id\": \"Tab1\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"hsides\\\" rules=\\\"groups\\\"><thead><tr><th align=\\\"left\\\" rowspan=\\\"2\\\"/><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>Overall</bold></th><th align=\\\"left\\\" colspan=\\\"3\\\"><bold>Clinic-based cohorts</bold></th><th align=\\\"left\\\" colspan=\\\"4\\\"><bold>Community-based cohorts</bold></th></tr><tr><th align=\\\"left\\\"><bold>NACC</bold></th><th align=\\\"left\\\"><bold>ADNI</bold></th><th align=\\\"left\\\"><bold>Total</bold></th><th align=\\\"left\\\"><bold>FHS</bold></th><th align=\\\"left\\\"><bold>ACT</bold></th><th align=\\\"left\\\"><bold>ROSMAP</bold></th><th align=\\\"left\\\"><bold>Total</bold></th></tr></thead><tbody><tr><td align=\\\"left\\\">Observations, n</td><td align=\\\"left\\\">103,796</td><td align=\\\"left\\\">53,237</td><td align=\\\"left\\\">7,059</td><td align=\\\"left\\\">60,296</td><td align=\\\"left\\\">9,115</td><td align=\\\"left\\\">15,122</td><td align=\\\"left\\\">19,263</td><td align=\\\"left\\\">43,500</td></tr><tr><td align=\\\"left\\\">Unique participants, n</td><td align=\\\"left\\\">23,066</td><td align=\\\"left\\\">12,985</td><td align=\\\"left\\\">1,367</td><td align=\\\"left\\\">14,352</td><td align=\\\"left\\\">3,607</td><td align=\\\"left\\\">3,035</td><td align=\\\"left\\\">2,072</td><td align=\\\"left\\\">8,714</td></tr><tr><td align=\\\"left\\\" colspan=\\\"9\\\">Mean age, years (SD)</td></tr><tr><td align=\\\"left\\\">\\u2003Overall</td><td align=\\\"left\\\">77.5\\u2009\\u00b1\\u20098.9</td><td align=\\\"left\\\">75.7\\u2009\\u00b1\\u20098.9</td><td align=\\\"left\\\">76.6\\u2009\\u00b1\\u20097.4</td><td align=\\\"left\\\">75.8\\u2009\\u00b1\\u20098.8</td><td align=\\\"left\\\">75.2\\u2009\\u00b1\\u20099.6</td><td align=\\\"left\\\">79.3\\u2009\\u00b1\\u20097.2</td><td align=\\\"left\\\">82.8\\u2009\\u00b1\\u20097.6</td><td align=\\\"left\\\">80.0\\u2009\\u00b1\\u20098.4</td></tr><tr><td align=\\\"left\\\">\\u2003First visit</td><td align=\\\"left\\\">73.1\\u2009\\u00b1\\u20098.7</td><td align=\\\"left\\\">72.9\\u2009\\u00b1\\u20099.0</td><td align=\\\"left\\\">73.9\\u2009\\u00b1\\u20097.1</td><td align=\\\"left\\\">73.0\\u2009\\u00b1\\u20098.8</td><td align=\\\"left\\\">69.1\\u2009\\u00b1\\u20098.3</td><td align=\\\"left\\\">74.5\\u2009\\u00b1\\u20096.5</td><td align=\\\"left\\\">78.8\\u2009\\u00b1\\u20097.5</td><td align=\\\"left\\\">73.3\\u2009\\u00b1\\u20098.4</td></tr><tr><td align=\\\"left\\\">\\u2003Last visit</td><td align=\\\"left\\\">78.2\\u2009\\u00b1\\u20099.7</td><td align=\\\"left\\\">76.6\\u2009\\u00b1\\u20099.3</td><td align=\\\"left\\\">77.6\\u2009\\u00b1\\u20097.9</td><td align=\\\"left\\\">76.6\\u2009\\u00b1\\u20099.2</td><td align=\\\"left\\\">75.3\\u2009\\u00b1\\u20099.9</td><td align=\\\"left\\\">82.7\\u2009\\u00b1\\u20097.7</td><td align=\\\"left\\\">87.8\\u2009\\u00b1\\u20096.9</td><td align=\\\"left\\\">80.8\\u2009\\u00b1\\u20099.9</td></tr><tr><td align=\\\"left\\\" colspan=\\\"9\\\">Median age, years (IQR)</td></tr><tr><td align=\\\"left\\\">\\u2003Overall</td><td align=\\\"left\\\">78.0 (13.0)</td><td align=\\\"left\\\">76.0 (12.0)</td><td align=\\\"left\\\">76.7 (10.0)</td><td align=\\\"left\\\">76.0 (12.0)</td><td align=\\\"left\\\">75.0 (16.0)</td><td align=\\\"left\\\">79.0 (10.0)</td><td align=\\\"left\\\">83.2 (10.6)</td><td align=\\\"left\\\">80.4 (12.0)</td></tr><tr><td align=\\\"left\\\">\\u2003First visit</td><td align=\\\"left\\\">73.0 (12.3)</td><td align=\\\"left\\\">73.0 (12.0)</td><td align=\\\"left\\\">73.8 (9.8)</td><td align=\\\"left\\\">73.0 (12.0)</td><td align=\\\"left\\\">66.0 (11.5)</td><td align=\\\"left\\\">73.0 (10.0)</td><td align=\\\"left\\\">79.5 (10.6)</td><td align=\\\"left\\\">72.0 (14.0)</td></tr><tr><td align=\\\"left\\\">\\u2003Last visit</td><td align=\\\"left\\\">78.0 (14.3)</td><td align=\\\"left\\\">77.0 (13.0)</td><td align=\\\"left\\\">77.9 (11.4)</td><td align=\\\"left\\\">77.0 (13.0)</td><td align=\\\"left\\\">74.0 (16.0)</td><td align=\\\"left\\\">83.0 (11.0)</td><td align=\\\"left\\\">88.2 (9.0)</td><td align=\\\"left\\\">82.0 (15.6)</td></tr><tr><td align=\\\"left\\\" colspan=\\\"9\\\">Sex, n (%)</td></tr><tr><td align=\\\"left\\\">\\u2003Male</td><td align=\\\"left\\\">10,198 (44.2)</td><td align=\\\"left\\\">5,856 (45.1)</td><td align=\\\"left\\\">766 (56.0)</td><td align=\\\"left\\\">6,622 (46.1)</td><td align=\\\"left\\\">1,620 (44.9)</td><td align=\\\"left\\\">1,331 (43.9)</td><td align=\\\"left\\\">625 (30.2)</td><td align=\\\"left\\\">3,576 (41.0)</td></tr><tr><td align=\\\"left\\\">\\u2003Female</td><td align=\\\"left\\\">12,868 (55.8)</td><td align=\\\"left\\\">7,129 (54.9)</td><td align=\\\"left\\\">601 (44.0)</td><td align=\\\"left\\\">7,730 (53.9)</td><td align=\\\"left\\\">1,987 (55.1)</td><td align=\\\"left\\\">1,704 (56.1)</td><td align=\\\"left\\\">1,447 (69.8)</td><td align=\\\"left\\\">5,138 (59.0)</td></tr><tr><td align=\\\"left\\\" colspan=\\\"9\\\">Educational level, n (%)</td></tr><tr><td align=\\\"left\\\">\\u2003Under high school degree</td><td align=\\\"left\\\">974 (4.2)</td><td align=\\\"left\\\">302 (2.3)</td><td align=\\\"left\\\">41 (3.0)</td><td align=\\\"left\\\">343 (2.4)</td><td align=\\\"left\\\">305 (8.5)</td><td align=\\\"left\\\">240 (7.9)</td><td align=\\\"left\\\">86 (4.2)</td><td align=\\\"left\\\">631 (7.2)</td></tr><tr><td align=\\\"left\\\">\\u2003High school degree</td><td align=\\\"left\\\">3,988 (17.3)</td><td align=\\\"left\\\">1,958 (15.1)</td><td align=\\\"left\\\">160 (11.7)</td><td align=\\\"left\\\">2,118 (14.8)</td><td align=\\\"left\\\">927 (25.7)</td><td align=\\\"left\\\">620 (20.4)</td><td align=\\\"left\\\">323 (15.6)</td><td align=\\\"left\\\">1,870 (21.5)</td></tr><tr><td align=\\\"left\\\">\\u2003Some college</td><td align=\\\"left\\\">4,387 (19.0)</td><td align=\\\"left\\\">2,212 (17.0)</td><td align=\\\"left\\\">259 (18.9)</td><td align=\\\"left\\\">2,471 (17.2)</td><td align=\\\"left\\\">905 (25.1)</td><td align=\\\"left\\\">688 (22.7)</td><td align=\\\"left\\\">323 (15.6)</td><td align=\\\"left\\\">1,916 (22.0)</td></tr><tr><td align=\\\"left\\\">\\u2003Over college graduate</td><td align=\\\"left\\\">13,717 (59.5)</td><td align=\\\"left\\\">8,513 (65.6)</td><td align=\\\"left\\\">907 (66.3)</td><td align=\\\"left\\\">9,420 (65.6)</td><td align=\\\"left\\\">1,470 (40.8)</td><td align=\\\"left\\\">1,487 (49.0)</td><td align=\\\"left\\\">1,340 (64.7)</td><td align=\\\"left\\\">4,297 (49.3)</td></tr><tr><td align=\\\"left\\\" colspan=\\\"9\\\">Executive function, mean (SD)</td></tr><tr><td align=\\\"left\\\">\\u2003Overall</td><td align=\\\"left\\\">0.197\\u2009\\u00b1\\u20090.752</td><td align=\\\"left\\\">0.163\\u2009\\u00b1\\u20090.827</td><td align=\\\"left\\\">0.318\\u2009\\u00b1\\u20090.746</td><td align=\\\"left\\\">0.182\\u2009\\u00b1\\u20090.819</td><td align=\\\"left\\\">-0.007\\u2009\\u00b1\\u20090.688</td><td align=\\\"left\\\">0.189\\u2009\\u00b1\\u20090.499</td><td align=\\\"left\\\">0.343\\u2009\\u00b1\\u20090.698</td><td align=\\\"left\\\">0.217\\u2009\\u00b1\\u20090.649</td></tr><tr><td align=\\\"left\\\">\\u2003First visit</td><td align=\\\"left\\\">0.157\\u2009\\u00b1\\u20090.748</td><td align=\\\"left\\\">0.072\\u2009\\u00b1\\u20090.842</td><td align=\\\"left\\\">0.377\\u2009\\u00b1\\u20090.668</td><td align=\\\"left\\\">0.102\\u2009\\u00b1\\u20090.832</td><td align=\\\"left\\\">0.167\\u2009\\u00b1\\u20090.648</td><td align=\\\"left\\\">0.248\\u2009\\u00b1\\u20090.454</td><td align=\\\"left\\\">0.380\\u2009\\u00b1\\u20090.598</td><td align=\\\"left\\\">0.246\\u2009\\u00b1\\u20090.581</td></tr><tr><td align=\\\"left\\\">\\u2003Last visit</td><td align=\\\"left\\\">-0.091\\u2009\\u00b1\\u20090.894</td><td align=\\\"left\\\">-0.143\\u2009\\u00b1\\u20090.977</td><td align=\\\"left\\\">0.096\\u2009\\u00b1\\u20090.891</td><td align=\\\"left\\\">-0.119\\u2009\\u00b1\\u20090.971</td><td align=\\\"left\\\">-0.043\\u2009\\u00b1\\u20090.726</td><td align=\\\"left\\\">0.044\\u2009\\u00b1\\u20090.677</td><td align=\\\"left\\\">-0.181\\u2009\\u00b1\\u20090.865</td><td align=\\\"left\\\">-0.046\\u2009\\u00b1\\u20090.751</td></tr><tr><td align=\\\"left\\\" colspan=\\\"9\\\">Language, mean (SD)</td></tr><tr><td align=\\\"left\\\">\\u2003Overall</td><td align=\\\"left\\\">0.353\\u2009\\u00b1\\u20090.808</td><td align=\\\"left\\\">0.472\\u2009\\u00b1\\u20090.869</td><td align=\\\"left\\\">0.392\\u2009\\u00b1\\u20090.700</td><td align=\\\"left\\\">0.463\\u2009\\u00b1\\u20090.851</td><td align=\\\"left\\\">0.229\\u2009\\u00b1\\u20090.662</td><td align=\\\"left\\\">0.166\\u2009\\u00b1\\u20090.422</td><td align=\\\"left\\\">0.154\\u2009\\u00b1\\u20090.805</td><td align=\\\"left\\\">0.173\\u2009\\u00b1\\u20090.695</td></tr><tr><td align=\\\"left\\\">\\u2003First visit</td><td align=\\\"left\\\">0.360\\u2009\\u00b1\\u20090.739</td><td align=\\\"left\\\">0.392\\u2009\\u00b1\\u20090.843</td><td align=\\\"left\\\">0.467\\u2009\\u00b1\\u20090.610</td><td align=\\\"left\\\">0.399\\u2009\\u00b1\\u20090.824</td><td align=\\\"left\\\">0.360\\u2009\\u00b1\\u20090.626</td><td align=\\\"left\\\">0.239\\u2009\\u00b1\\u20090.366</td><td align=\\\"left\\\">0.247\\u2009\\u00b1\\u20090.625</td><td align=\\\"left\\\">0.291\\u2009\\u00b1\\u20090.555</td></tr><tr><td align=\\\"left\\\">\\u2003Last visit</td><td align=\\\"left\\\">0.088\\u2009\\u00b1\\u20090.935</td><td align=\\\"left\\\">0.135\\u2009\\u00b1\\u20091.016</td><td align=\\\"left\\\">0.186\\u2009\\u00b1\\u20090.863</td><td align=\\\"left\\\">0.140\\u2009\\u00b1\\u20091.003</td><td align=\\\"left\\\">0.222\\u2009\\u00b1\\u20090.722</td><td align=\\\"left\\\">0.039\\u2009\\u00b1\\u20090.537</td><td align=\\\"left\\\">-0.429\\u2009\\u00b1\\u20091.007</td><td align=\\\"left\\\">-0.004\\u2009\\u00b1\\u20090.797</td></tr><tr><td align=\\\"left\\\" colspan=\\\"9\\\">Memory, mean (SD)</td></tr><tr><td align=\\\"left\\\">\\u2003Overall</td><td align=\\\"left\\\">0.337\\u2009\\u00b1\\u20090.890</td><td align=\\\"left\\\">0.372\\u2009\\u00b1\\u20091.007</td><td align=\\\"left\\\">0.146\\u2009\\u00b1\\u20090.873</td><td align=\\\"left\\\">0.346\\u2009\\u00b1\\u20090.995</td><td align=\\\"left\\\">0.286\\u2009\\u00b1\\u20090.636</td><td align=\\\"left\\\">0.512\\u2009\\u00b1\\u20090.549</td><td align=\\\"left\\\">0.194\\u2009\\u00b1\\u20090.836</td><td align=\\\"left\\\">0.324\\u2009\\u00b1\\u20090.721</td></tr><tr><td align=\\\"left\\\">\\u2003First visit</td><td align=\\\"left\\\">0.293\\u2009\\u00b1\\u20090.798</td><td align=\\\"left\\\">0.216\\u2009\\u00b1\\u20090.927</td><td align=\\\"left\\\">0.235\\u2009\\u00b1\\u20090.717</td><td align=\\\"left\\\">0.218\\u2009\\u00b1\\u20090.909</td><td align=\\\"left\\\">0.406\\u2009\\u00b1\\u20090.551</td><td align=\\\"left\\\">0.590\\u2009\\u00b1\\u20090.456</td><td align=\\\"left\\\">0.181\\u2009\\u00b1\\u20090.573</td><td align=\\\"left\\\">0.417\\u2009\\u00b1\\u20090.548</td></tr><tr><td align=\\\"left\\\">\\u2003Last visit</td><td align=\\\"left\\\">0.010\\u2009\\u00b1\\u20091.042</td><td align=\\\"left\\\">-0.019\\u2009\\u00b1\\u20091.155</td><td align=\\\"left\\\">-0.083\\u2009\\u00b1\\u20091.035</td><td align=\\\"left\\\">-0.025\\u2009\\u00b1\\u20091.144</td><td align=\\\"left\\\">0.241\\u2009\\u00b1\\u20090.688</td><td align=\\\"left\\\">0.176\\u2009\\u00b1\\u20090.731</td><td align=\\\"left\\\">-0.393\\u2009\\u00b1\\u20091.055</td><td align=\\\"left\\\">0.068\\u2009\\u00b1\\u20090.845</td></tr><tr><td align=\\\"left\\\" colspan=\\\"9\\\">Cognitive status, n (%)</td></tr><tr><td align=\\\"left\\\">\\u2003Cognitively normal</td><td align=\\\"left\\\">10,919 (47.3)</td><td align=\\\"left\\\">4,891 (37.7)</td><td align=\\\"left\\\">437 (32.0)</td><td align=\\\"left\\\">5,328 (37.1)</td><td align=\\\"left\\\">2,740 (76.0)</td><td align=\\\"left\\\">2,267 (74.7)</td><td align=\\\"left\\\">584 (28.2)</td><td align=\\\"left\\\">5,591 (64.2)</td></tr><tr><td align=\\\"left\\\">\\u2003MCI</td><td align=\\\"left\\\">2,248 (9.7)</td><td align=\\\"left\\\">820 (6.3)</td><td align=\\\"left\\\">444 (32.5)</td><td align=\\\"left\\\">1,264 (8.8)</td><td align=\\\"left\\\">287 (8.0)</td><td align=\\\"left\\\">-</td><td align=\\\"left\\\">697 (33.6)</td><td align=\\\"left\\\">984 (11.3)</td></tr><tr><td align=\\\"left\\\">\\u2003AD</td><td align=\\\"left\\\">9,272 (40.2)</td><td align=\\\"left\\\">7,090 (54.6)</td><td align=\\\"left\\\">486 (35.6)</td><td align=\\\"left\\\">7,576 (52.8)</td><td align=\\\"left\\\">445 (12.3)</td><td align=\\\"left\\\">485 (16.0)</td><td align=\\\"left\\\">766 (37.0)</td><td align=\\\"left\\\">1,696 (19.5)</td></tr><tr><td align=\\\"left\\\">\\u2003Dementia (other than AD)</td><td align=\\\"left\\\">627 (2.7)</td><td align=\\\"left\\\">184 (1.4)</td><td align=\\\"left\\\">-</td><td align=\\\"left\\\">184 (1.3)</td><td align=\\\"left\\\">135 (3.7)</td><td align=\\\"left\\\">283 (9.3)</td><td align=\\\"left\\\">25 (1.2)</td><td align=\\\"left\\\">443 (5.1)</td></tr></tbody></table>\\n\"}, \"text\": \"\\tOverall\\tClinic-based cohorts\\tCommunity-based cohorts\\t \\tNACC\\tADNI\\tTotal\\tFHS\\tACT\\tROSMAP\\tTotal\\t \\tObservations, n\\t103,796\\t53,237\\t7,059\\t60,296\\t9,115\\t15,122\\t19,263\\t43,500\\t \\tUnique participants, n\\t23,066\\t12,985\\t1,367\\t14,352\\t3,607\\t3,035\\t2,072\\t8,714\\t \\tMean age, years (SD)\\t \\t\\u2003Overall\\t77.5\\u2009\\u00b1\\u20098.9\\t75.7\\u2009\\u00b1\\u20098.9\\t76.6\\u2009\\u00b1\\u20097.4\\t75.8\\u2009\\u00b1\\u20098.8\\t75.2\\u2009\\u00b1\\u20099.6\\t79.3\\u2009\\u00b1\\u20097.2\\t82.8\\u2009\\u00b1\\u20097.6\\t80.0\\u2009\\u00b1\\u20098.4\\t \\t\\u2003First visit\\t73.1\\u2009\\u00b1\\u20098.7\\t72.9\\u2009\\u00b1\\u20099.0\\t73.9\\u2009\\u00b1\\u20097.1\\t73.0\\u2009\\u00b1\\u20098.8\\t69.1\\u2009\\u00b1\\u20098.3\\t74.5\\u2009\\u00b1\\u20096.5\\t78.8\\u2009\\u00b1\\u20097.5\\t73.3\\u2009\\u00b1\\u20098.4\\t \\t\\u2003Last visit\\t78.2\\u2009\\u00b1\\u20099.7\\t76.6\\u2009\\u00b1\\u20099.3\\t77.6\\u2009\\u00b1\\u20097.9\\t76.6\\u2009\\u00b1\\u20099.2\\t75.3\\u2009\\u00b1\\u20099.9\\t82.7\\u2009\\u00b1\\u20097.7\\t87.8\\u2009\\u00b1\\u20096.9\\t80.8\\u2009\\u00b1\\u20099.9\\t \\tMedian age, years (IQR)\\t \\t\\u2003Overall\\t78.0 (13.0)\\t76.0 (12.0)\\t76.7 (10.0)\\t76.0 (12.0)\\t75.0 (16.0)\\t79.0 (10.0)\\t83.2 (10.6)\\t80.4 (12.0)\\t \\t\\u2003First visit\\t73.0 (12.3)\\t73.0 (12.0)\\t73.8 (9.8)\\t73.0 (12.0)\\t66.0 (11.5)\\t73.0 (10.0)\\t79.5 (10.6)\\t72.0 (14.0)\\t \\t\\u2003Last visit\\t78.0 (14.3)\\t77.0 (13.0)\\t77.9 (11.4)\\t77.0 (13.0)\\t74.0 (16.0)\\t83.0 (11.0)\\t88.2 (9.0)\\t82.0 (15.6)\\t \\tSex, n (%)\\t \\t\\u2003Male\\t10,198 (44.2)\\t5,856 (45.1)\\t766 (56.0)\\t6,622 (46.1)\\t1,620 (44.9)\\t1,331 (43.9)\\t625 (30.2)\\t3,576 (41.0)\\t \\t\\u2003Female\\t12,868 (55.8)\\t7,129 (54.9)\\t601 (44.0)\\t7,730 (53.9)\\t1,987 (55.1)\\t1,704 (56.1)\\t1,447 (69.8)\\t5,138 (59.0)\\t \\tEducational level, n (%)\\t \\t\\u2003Under high school degree\\t974 (4.2)\\t302 (2.3)\\t41 (3.0)\\t343 (2.4)\\t305 (8.5)\\t240 (7.9)\\t86 (4.2)\\t631 (7.2)\\t \\t\\u2003High school degree\\t3,988 (17.3)\\t1,958 (15.1)\\t160 (11.7)\\t2,118 (14.8)\\t927 (25.7)\\t620 (20.4)\\t323 (15.6)\\t1,870 (21.5)\\t \\t\\u2003Some college\\t4,387 (19.0)\\t2,212 (17.0)\\t259 (18.9)\\t2,471 (17.2)\\t905 (25.1)\\t688 (22.7)\\t323 (15.6)\\t1,916 (22.0)\\t \\t\\u2003Over college graduate\\t13,717 (59.5)\\t8,513 (65.6)\\t907 (66.3)\\t9,420 (65.6)\\t1,470 (40.8)\\t1,487 (49.0)\\t1,340 (64.7)\\t4,297 (49.3)\\t \\tExecutive function, mean (SD)\\t \\t\\u2003Overall\\t0.197\\u2009\\u00b1\\u20090.752\\t0.163\\u2009\\u00b1\\u20090.827\\t0.318\\u2009\\u00b1\\u20090.746\\t0.182\\u2009\\u00b1\\u20090.819\\t-0.007\\u2009\\u00b1\\u20090.688\\t0.189\\u2009\\u00b1\\u20090.499\\t0.343\\u2009\\u00b1\\u20090.698\\t0.217\\u2009\\u00b1\\u20090.649\\t \\t\\u2003First visit\\t0.157\\u2009\\u00b1\\u20090.748\\t0.072\\u2009\\u00b1\\u20090.842\\t0.377\\u2009\\u00b1\\u20090.668\\t0.102\\u2009\\u00b1\\u20090.832\\t0.167\\u2009\\u00b1\\u20090.648\\t0.248\\u2009\\u00b1\\u20090.454\\t0.380\\u2009\\u00b1\\u20090.598\\t0.246\\u2009\\u00b1\\u20090.581\\t \\t\\u2003Last visit\\t-0.091\\u2009\\u00b1\\u20090.894\\t-0.143\\u2009\\u00b1\\u20090.977\\t0.096\\u2009\\u00b1\\u20090.891\\t-0.119\\u2009\\u00b1\\u20090.971\\t-0.043\\u2009\\u00b1\\u20090.726\\t0.044\\u2009\\u00b1\\u20090.677\\t-0.181\\u2009\\u00b1\\u20090.865\\t-0.046\\u2009\\u00b1\\u20090.751\\t \\tLanguage, mean (SD)\\t \\t\\u2003Overall\\t0.353\\u2009\\u00b1\\u20090.808\\t0.472\\u2009\\u00b1\\u20090.869\\t0.392\\u2009\\u00b1\\u20090.700\\t0.463\\u2009\\u00b1\\u20090.851\\t0.229\\u2009\\u00b1\\u20090.662\\t0.166\\u2009\\u00b1\\u20090.422\\t0.154\\u2009\\u00b1\\u20090.805\\t0.173\\u2009\\u00b1\\u20090.695\\t \\t\\u2003First visit\\t0.360\\u2009\\u00b1\\u20090.739\\t0.392\\u2009\\u00b1\\u20090.843\\t0.467\\u2009\\u00b1\\u20090.610\\t0.399\\u2009\\u00b1\\u20090.824\\t0.360\\u2009\\u00b1\\u20090.626\\t0.239\\u2009\\u00b1\\u20090.366\\t0.247\\u2009\\u00b1\\u20090.625\\t0.291\\u2009\\u00b1\\u20090.555\\t \\t\\u2003Last visit\\t0.088\\u2009\\u00b1\\u20090.935\\t0.135\\u2009\\u00b1\\u20091.016\\t0.186\\u2009\\u00b1\\u20090.863\\t0.140\\u2009\\u00b1\\u20091.003\\t0.222\\u2009\\u00b1\\u20090.722\\t0.039\\u2009\\u00b1\\u20090.537\\t-0.429\\u2009\\u00b1\\u20091.007\\t-0.004\\u2009\\u00b1\\u20090.797\\t \\tMemory, mean (SD)\\t \\t\\u2003Overall\\t0.337\\u2009\\u00b1\\u20090.890\\t0.372\\u2009\\u00b1\\u20091.007\\t0.146\\u2009\\u00b1\\u20090.873\\t0.346\\u2009\\u00b1\\u20090.995\\t0.286\\u2009\\u00b1\\u20090.636\\t0.512\\u2009\\u00b1\\u20090.549\\t0.194\\u2009\\u00b1\\u20090.836\\t0.324\\u2009\\u00b1\\u20090.721\\t \\t\\u2003First visit\\t0.293\\u2009\\u00b1\\u20090.798\\t0.216\\u2009\\u00b1\\u20090.927\\t0.235\\u2009\\u00b1\\u20090.717\\t0.218\\u2009\\u00b1\\u20090.909\\t0.406\\u2009\\u00b1\\u20090.551\\t0.590\\u2009\\u00b1\\u20090.456\\t0.181\\u2009\\u00b1\\u20090.573\\t0.417\\u2009\\u00b1\\u20090.548\\t \\t\\u2003Last visit\\t0.010\\u2009\\u00b1\\u20091.042\\t-0.019\\u2009\\u00b1\\u20091.155\\t-0.083\\u2009\\u00b1\\u20091.035\\t-0.025\\u2009\\u00b1\\u20091.144\\t0.241\\u2009\\u00b1\\u20090.688\\t0.176\\u2009\\u00b1\\u20090.731\\t-0.393\\u2009\\u00b1\\u20091.055\\t0.068\\u2009\\u00b1\\u20090.845\\t \\tCognitive status, n (%)\\t \\t\\u2003Cognitively normal\\t10,919 (47.3)\\t4,891 (37.7)\\t437 (32.0)\\t5,328 (37.1)\\t2,740 (76.0)\\t2,267 (74.7)\\t584 (28.2)\\t5,591 (64.2)\\t \\t\\u2003MCI\\t2,248 (9.7)\\t820 (6.3)\\t444 (32.5)\\t1,264 (8.8)\\t287 (8.0)\\t-\\t697 (33.6)\\t984 (11.3)\\t \\t\\u2003AD\\t9,272 (40.2)\\t7,090 (54.6)\\t486 (35.6)\\t7,576 (52.8)\\t445 (12.3)\\t485 (16.0)\\t766 (37.0)\\t1,696 (19.5)\\t \\t\\u2003Dementia (other than AD)\\t627 (2.7)\\t184 (1.4)\\t-\\t184 (1.3)\\t135 (3.7)\\t283 (9.3)\\t25 (1.2)\\t443 (5.1)\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 18333, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Compared to the community-based cohorts, the clinic-based cohorts had more males, participants who were younger and better educated, and higher proportion of participants who were diagnosed with MCI or AD (Table 1). Even though the mean and median ages at the last visits of individuals in the clinic-based cohorts were slightly lower than those in the community-based cohorts, scores for executive function and memory were significantly lower (P\\u2009<\\u20090.001), and the language score was significantly higher (P\\u2009<\\u20090.001) in the clinic-based cohorts. Phenotypic and genetic correlations for each pair of factor scores in each dataset were moderate to high (phenotypic r\\u2009=\\u20090.56\\u20130.86, genetic r\\u2009=\\u20090.57\\u20130.72) (Table S1). Most phenotypic and genetic correlations were higher for ROSMAP compared to the other cohorts. Cross-trait analyses revealed that factor scores for all three cognitive domains are significantly genetically correlated with general cognitive function (0.51\\u2009\\u2264\\u2009r\\u2009\\u2264\\u20090.77) (Table S2). Although none of the traits were significantly correlated with AD or other psychiatric disorders, the language domain score was moderately associated with depression (r\\u2009=\\u20090.60, P\\u2009=\\u20090.11) and the memory domain score was strongly associated with AD (r\\u2009=\\u20090.90, P\\u2009=\\u20090.40). Lack of significance for these results may be due to insufficient power for genetic correlations with dichotomous outcomes.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 19766, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"GWAS identifies multiple established AD and novel loci associated with individual cognitive domains\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 19866, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Genome-wide significant associations for cognitive domain scores\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 19931, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"hsides\\\" rules=\\\"groups\\\"><thead><tr><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>Chr</bold></th><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>Position</bold></th><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>Variant</bold></th><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>A1/A2</bold><sup><bold>a</bold></sup><bold> (MAF)</bold></th><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>Nearest Gene</bold></th><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>Cognitive Domain</bold></th><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>Cohorts</bold><sup><bold>b</bold></sup></th><th align=\\\"left\\\" colspan=\\\"5\\\"><bold>Genetic effects</bold></th></tr><tr><th align=\\\"left\\\"><bold><italic>\\u03b2</italic></bold><sub><bold>G</bold></sub><bold> (SE)</bold></th><th align=\\\"left\\\"><bold><italic>P</italic></bold><sub><bold>G</bold></sub></th><th align=\\\"left\\\"><bold><italic>\\u03b2</italic></bold><sub><bold>G\\u00d7Age</bold></sub><bold> (SE)</bold></th><th align=\\\"left\\\"><bold><italic>P</italic></bold><sub><bold>G\\u00d7Age</bold></sub></th><th align=\\\"left\\\"><bold><italic>P</italic></bold><sub><bold>Joint</bold></sub></th></tr></thead><tbody><tr><td align=\\\"left\\\" rowspan=\\\"9\\\">1</td><td align=\\\"left\\\" rowspan=\\\"9\\\">207573951</td><td align=\\\"left\\\" rowspan=\\\"9\\\">rs1752684</td><td align=\\\"left\\\" rowspan=\\\"9\\\">A/G (0.198)</td><td align=\\\"left\\\" rowspan=\\\"9\\\"><italic>CR1</italic></td><td align=\\\"left\\\" rowspan=\\\"3\\\">Executive Function</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.028 (0.007)</td><td align=\\\"left\\\">1.53\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">6.05\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">5.05\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.042 (0.012)</td><td align=\\\"left\\\">5.91\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">7.52\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">6.88\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.020 (0.009)</td><td align=\\\"left\\\">2.98\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">4.90\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">2.40\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Language</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.026 (0.007)</td><td align=\\\"left\\\">3.90\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">-0.001 (0.001)</td><td align=\\\"left\\\">6.84\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">6.25\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.054 (0.012)</td><td align=\\\"left\\\">1.26\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">5.07\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">1.63\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.011 (0.009)</td><td align=\\\"left\\\">2.14\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.001 (0.001)</td><td align=\\\"left\\\">5.59\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">4.20\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Memory</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.043 (0.008)</td><td align=\\\"left\\\">1.75\\u2009\\u00d7\\u200910<sup>\\u20137</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">1.83\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">4.42\\u2009\\u00d7\\u200910<sup>\\u20137</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.080 (0.014)</td><td align=\\\"left\\\">2.44\\u2009\\u00d7\\u200910<sup>\\u20138</sup></td><td align=\\\"left\\\">-0.004 (0.001)</td><td align=\\\"left\\\">1.05\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\"><bold>8.85\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20139</bold></sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.022 (0.010)</td><td align=\\\"left\\\">2.46\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">0.000 (0.001)</td><td align=\\\"left\\\">6.45\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">6.88\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"9\\\">2</td><td align=\\\"left\\\" rowspan=\\\"9\\\">127135234</td><td align=\\\"left\\\" rowspan=\\\"9\\\">rs6733839</td><td align=\\\"left\\\" rowspan=\\\"9\\\">T/C (0.403)</td><td align=\\\"left\\\" rowspan=\\\"9\\\"><p><italic>BIN1</italic></p><p><italic>CYP27C1</italic></p></td><td align=\\\"left\\\" rowspan=\\\"3\\\">Executive Function</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.009 (0.006)</td><td align=\\\"left\\\">1.25\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.000 (0.001)</td><td align=\\\"left\\\">6.54\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">2.92\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.023 (0.010)</td><td align=\\\"left\\\">1.77\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">0.000 (0.001)</td><td align=\\\"left\\\">6.84\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">5.58\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.001 (0.007)</td><td align=\\\"left\\\">8.70\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.000 (0.001)</td><td align=\\\"left\\\">9.27\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">9.84\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Language</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.031 (0.006)</td><td align=\\\"left\\\">6.90\\u2009\\u00d7\\u200910<sup>\\u20138</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">3.62\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\"><bold>2.70\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20138</bold></sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.063 (0.010)</td><td align=\\\"left\\\">4.23\\u2009\\u00d7\\u200910<sup>\\u201310</sup></td><td align=\\\"left\\\">-0.001 (0.001)</td><td align=\\\"left\\\">2.09\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\"><bold>1.98\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20139</bold></sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.017 (0.007)</td><td align=\\\"left\\\">1.68\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">1.70\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">6.57\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Memory</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.039 (0.006)</td><td align=\\\"left\\\">2.20\\u2009\\u00d7\\u200910<sup>\\u20139</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">1.62\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\"><bold>2.37\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20139</bold></sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.068 (0.012)</td><td align=\\\"left\\\">5.23\\u2009\\u00d7\\u200910<sup>\\u20139</sup></td><td align=\\\"left\\\">-0.001 (0.001)</td><td align=\\\"left\\\">2.08\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\"><bold>1.60\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20138</bold></sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.027 (0.008)</td><td align=\\\"left\\\">5.71\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">1.69\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">1.11\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"9\\\">7</td><td align=\\\"left\\\" rowspan=\\\"9\\\">91069758</td><td align=\\\"left\\\" rowspan=\\\"9\\\">rs705353</td><td align=\\\"left\\\" rowspan=\\\"9\\\">G/A (0.252)</td><td align=\\\"left\\\" rowspan=\\\"9\\\"><italic>CDK14</italic></td><td align=\\\"left\\\" rowspan=\\\"3\\\">Executive Function</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">0.003 (0.006)</td><td align=\\\"left\\\">6.40\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">1.53\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">4.58\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">0.000 (0.011)</td><td align=\\\"left\\\">9.75\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.005 (0.001)</td><td align=\\\"left\\\">2.03\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">1.12\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.006 (0.008)</td><td align=\\\"left\\\">4.20\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.001 (0.001)</td><td align=\\\"left\\\">4.11\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">4.43\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Language</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">0.007 (0.006)</td><td align=\\\"left\\\">2.68\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">5.14\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">5.93\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.004 (0.011)</td><td align=\\\"left\\\">7.53\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.007 (0.001)</td><td align=\\\"left\\\">2.54\\u2009\\u00d7\\u200910<sup>\\u20139</sup></td><td align=\\\"left\\\"><bold>1.73\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20138</bold></sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.015 (0.008)</td><td align=\\\"left\\\">5.28\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">0.000 (0.001)</td><td align=\\\"left\\\">8.28\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">1.41\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Memory</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.003 (0.007)</td><td align=\\\"left\\\">6.95\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">3.03\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">8.72\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.008 (0.013)</td><td align=\\\"left\\\">5.46\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.005 (0.001)</td><td align=\\\"left\\\">1.36\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">6.11\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.005 (0.009)</td><td align=\\\"left\\\">5.93\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.000 (0.001)</td><td align=\\\"left\\\">9.19\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">8.59\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"9\\\">8</td><td align=\\\"left\\\" rowspan=\\\"9\\\">30963282</td><td align=\\\"left\\\" rowspan=\\\"9\\\">rs117523305</td><td align=\\\"left\\\" rowspan=\\\"9\\\">C/G (0.016)</td><td align=\\\"left\\\" rowspan=\\\"9\\\"><p><italic>PURG</italic></p><p><italic>TEX15</italic></p></td><td align=\\\"left\\\" rowspan=\\\"3\\\">Executive Function</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.001 (0.022)</td><td align=\\\"left\\\">9.67\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.006 (0.002)</td><td align=\\\"left\\\">5.35\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">2.02\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.012 (0.038)</td><td align=\\\"left\\\">7.42\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.002 (0.004)</td><td align=\\\"left\\\">5.89\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">8.23\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.003 (0.026)</td><td align=\\\"left\\\">9.13\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.008 (0.003)</td><td align=\\\"left\\\">2.65\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">9.18\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Language</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.018 (0.021)</td><td align=\\\"left\\\">4.02\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.008 (0.002)</td><td align=\\\"left\\\">8.23\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">3.01\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">0.028 (0.039)</td><td align=\\\"left\\\">4.80\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.001 (0.004)</td><td align=\\\"left\\\">7.03\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">7.11\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.041 (0.025)</td><td align=\\\"left\\\">1.19\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.011 (0.003)</td><td align=\\\"left\\\">9.16\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">1.87\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Memory</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.069 (0.024)</td><td align=\\\"left\\\">5.27\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">-0.010 (0.002)</td><td align=\\\"left\\\">5.18\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">3.62\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.041 (0.044)</td><td align=\\\"left\\\">3.60\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.004 (0.004)</td><td align=\\\"left\\\">3.34\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">3.88\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.098 (0.029)</td><td align=\\\"left\\\">7.87\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">-0.018 (0.003)</td><td align=\\\"left\\\">1.41\\u2009\\u00d7\\u200910<sup>\\u20138</sup></td><td align=\\\"left\\\"><bold>1.73\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20138</bold></sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"9\\\">11</td><td align=\\\"left\\\" rowspan=\\\"9\\\">60173126</td><td align=\\\"left\\\" rowspan=\\\"9\\\">rs7232</td><td align=\\\"left\\\" rowspan=\\\"9\\\">A/T (0.361)</td><td align=\\\"left\\\" rowspan=\\\"9\\\"><italic>MS4A6A</italic></td><td align=\\\"left\\\" rowspan=\\\"3\\\">Executive Function</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">0.012 (0.006)</td><td align=\\\"left\\\">3.46\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">0.002 (0.001)</td><td align=\\\"left\\\">2.90\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">2.38\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">0.021 (0.010)</td><td align=\\\"left\\\">3.46\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">0.000 (0.001)</td><td align=\\\"left\\\">8.60\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">1.06\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.009 (0.007)</td><td align=\\\"left\\\">2.07\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.002 (0.001)</td><td align=\\\"left\\\">6.28\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">2.09\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Language</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">0.016 (0.006)</td><td align=\\\"left\\\">6.40\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">0.002 (0.001)</td><td align=\\\"left\\\">1.04\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">3.61\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">0.045 (0.010)</td><td align=\\\"left\\\">1.40\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">0.002 (0.001)</td><td align=\\\"left\\\">4.91\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">1.55\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.003 (0.007)</td><td align=\\\"left\\\">6.38\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.002 (0.001)</td><td align=\\\"left\\\">1.93\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">6.31\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Memory</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">0.027 (0.006)</td><td align=\\\"left\\\">5.19\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">0.003 (0.001)</td><td align=\\\"left\\\">2.60\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">9.43\\u2009\\u00d7\\u200910<sup>\\u20137</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">0.068 (0.012)</td><td align=\\\"left\\\">1.25\\u2009\\u00d7\\u200910<sup>\\u20138</sup></td><td align=\\\"left\\\">0.001 (0.001)</td><td align=\\\"left\\\">2.27\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\"><bold>3.97\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20138</bold></sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.011 (0.008)</td><td align=\\\"left\\\">1.50\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.003 (0.001)</td><td align=\\\"left\\\">4.62\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">1.80\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"9\\\">11</td><td align=\\\"left\\\" rowspan=\\\"9\\\">127595711</td><td align=\\\"left\\\" rowspan=\\\"9\\\">rs145012974</td><td align=\\\"left\\\" rowspan=\\\"9\\\">A/G (0.014)</td><td align=\\\"left\\\" rowspan=\\\"9\\\"><p><italic>LINC02712</italic></p><p><italic>LINC02098</italic></p></td><td align=\\\"left\\\" rowspan=\\\"3\\\">Executive Function</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.040 (0.024)</td><td align=\\\"left\\\">1.03\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.007 (0.002)</td><td align=\\\"left\\\">4.72\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">7.25\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.013 (0.041)</td><td align=\\\"left\\\">7.53\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.004 (0.004)</td><td align=\\\"left\\\">3.14\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">5.72\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.056 (0.030)</td><td align=\\\"left\\\">6.46\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">-0.008 (0.003)</td><td align=\\\"left\\\">5.11\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">6.48\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Language</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.029 (0.024)</td><td align=\\\"left\\\">2.29\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.014 (0.002)</td><td align=\\\"left\\\">7.69\\u2009\\u00d7\\u200910<sup>\\u20139</sup></td><td align=\\\"left\\\"><bold>3.66\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20138</bold></sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.026 (0.041)</td><td align=\\\"left\\\">5.31\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.012 (0.004)</td><td align=\\\"left\\\">2.16\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">7.23\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.033 (0.029)</td><td align=\\\"left\\\">2.70\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.016 (0.003)</td><td align=\\\"left\\\">1.00\\u2009\\u00d7\\u200910<sup>\\u20136</sup></td><td align=\\\"left\\\">3.29\\u2009\\u00d7\\u200910<sup>\\u20136</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Memory</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.052 (0.026)</td><td align=\\\"left\\\">5.08\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">-0.008 (0.003)</td><td align=\\\"left\\\">2.46\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">3.99\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.051 (0.047)</td><td align=\\\"left\\\">2.87\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.010 (0.005)</td><td align=\\\"left\\\">3.06\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">5.42\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.050 (0.032)</td><td align=\\\"left\\\">1.24\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.007 (0.003)</td><td align=\\\"left\\\">3.00\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">6.31\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"9\\\">17</td><td align=\\\"left\\\" rowspan=\\\"9\\\">19806731</td><td align=\\\"left\\\" rowspan=\\\"9\\\">rs157405</td><td align=\\\"left\\\" rowspan=\\\"9\\\">G/A (0.023)</td><td align=\\\"left\\\" rowspan=\\\"9\\\"><italic>ULK2</italic></td><td align=\\\"left\\\" rowspan=\\\"3\\\">Executive Function</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">0.063 (0.018)</td><td align=\\\"left\\\">8.61\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">0.008 (0.002)</td><td align=\\\"left\\\">1.04\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">1.49\\u2009\\u00d7\\u200910<sup>\\u20136</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.024 (0.032)</td><td align=\\\"left\\\">4.62\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.002 (0.003)</td><td align=\\\"left\\\">4.25\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">5.20\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.107 (0.023)</td><td align=\\\"left\\\">3.21\\u2009\\u00d7\\u200910<sup>\\u20136</sup></td><td align=\\\"left\\\">0.011 (0.002)</td><td align=\\\"left\\\">7.65\\u2009\\u00d7\\u200910<sup>\\u20137</sup></td><td align=\\\"left\\\"><bold>2.19\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20139</bold></sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Language</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">0.029 (0.018)</td><td align=\\\"left\\\">1.19\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.002 (0.002)</td><td align=\\\"left\\\">2.04\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">1.70\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.040 (0.033)</td><td align=\\\"left\\\">2.29\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.002 (0.003)</td><td align=\\\"left\\\">4.81\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">3.35\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.059 (0.022)</td><td align=\\\"left\\\">8.39\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">0.003 (0.002)</td><td align=\\\"left\\\">2.54\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">2.21\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Memory</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">0.046 (0.020)</td><td align=\\\"left\\\">2.78\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">0.001 (0.002)</td><td align=\\\"left\\\">5.87\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">8.07\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.005 (0.038)</td><td align=\\\"left\\\">9.07\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.002 (0.004)</td><td align=\\\"left\\\">5.39\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">8.14\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.064 (0.024)</td><td align=\\\"left\\\">9.43\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">0.001 (0.003)</td><td align=\\\"left\\\">6.48\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">2.71\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"9\\\">17</td><td align=\\\"left\\\" rowspan=\\\"9\\\">44352876</td><td align=\\\"left\\\" rowspan=\\\"9\\\">rs5848</td><td align=\\\"left\\\" rowspan=\\\"9\\\">T/C (0.305)</td><td align=\\\"left\\\" rowspan=\\\"9\\\"><italic>GRN</italic></td><td align=\\\"left\\\" rowspan=\\\"3\\\">Executive Function</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.007 (0.006)</td><td align=\\\"left\\\">2.53\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">1.32\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">5.43\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">0.003 (0.010)</td><td align=\\\"left\\\">7.50\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.003 (0.001)</td><td align=\\\"left\\\">4.56\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">1.67\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.012 (0.008)</td><td align=\\\"left\\\">1.14\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">5.76\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">1.15\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Language</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.012 (0.006)</td><td align=\\\"left\\\">5.99\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">2.86\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">3.53\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.015 (0.011)</td><td align=\\\"left\\\">1.71\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.001 (0.001)</td><td align=\\\"left\\\">4.67\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">3.11\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.011 (0.007)</td><td align=\\\"left\\\">1.34\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.003 (0.001)</td><td align=\\\"left\\\">1.56\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">3.43\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Memory</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.029 (0.007)</td><td align=\\\"left\\\">2.19\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">-0.003 (0.001)</td><td align=\\\"left\\\">1.07\\u2009\\u00d7\\u200910<sup>\\u20136</sup></td><td align=\\\"left\\\"><bold>4.21\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20138</bold></sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.032 (0.012)</td><td align=\\\"left\\\">9.58\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">-0.003 (0.001)</td><td align=\\\"left\\\">3.30\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">5.28\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.028 (0.008)</td><td align=\\\"left\\\">8.11\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">-0.003 (0.001)</td><td align=\\\"left\\\">1.29\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">6.50\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"9\\\">19<sup>c</sup></td><td align=\\\"left\\\" rowspan=\\\"9\\\">44908684</td><td align=\\\"left\\\" rowspan=\\\"9\\\">rs429358</td><td align=\\\"left\\\" rowspan=\\\"9\\\">C/T (0.231)</td><td align=\\\"left\\\" rowspan=\\\"9\\\"><italic>APOE</italic></td><td align=\\\"left\\\" rowspan=\\\"3\\\">Executive Function</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.192 (0.008)</td><td align=\\\"left\\\">6.75\\u2009\\u00d7\\u200910<sup>\\u2013135</sup></td><td align=\\\"left\\\">-0.009 (0.001)</td><td align=\\\"left\\\">1.26\\u2009\\u00d7\\u200910<sup>\\u201328</sup></td><td align=\\\"left\\\"><bold>8.58\\u2009\\u00d7\\u200910</bold><sup><bold>\\u2013145</bold></sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.308 (0.011)</td><td align=\\\"left\\\">5.51\\u2009\\u00d7\\u200910<sup>\\u2013166</sup></td><td align=\\\"left\\\">-0.010 (0.001)</td><td align=\\\"left\\\">1.71\\u2009\\u00d7\\u200910<sup>\\u201319</sup></td><td align=\\\"left\\\"><bold>1.86\\u2009\\u00d7\\u200910</bold><sup><bold>\\u2013170</bold></sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.089 (0.010)</td><td align=\\\"left\\\">5.21\\u2009\\u00d7\\u200910<sup>\\u201317</sup></td><td align=\\\"left\\\">-0.006 (0.001)</td><td align=\\\"left\\\">4.58\\u2009\\u00d7\\u200910<sup>\\u20139</sup></td><td align=\\\"left\\\"><bold>7.05\\u2009\\u00d7\\u200910</bold><sup><bold>\\u201320</bold></sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Language</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.193 (0.008)</td><td align=\\\"left\\\">1.81\\u2009\\u00d7\\u200910<sup>\\u2013137</sup></td><td align=\\\"left\\\">-0.011 (0.001)</td><td align=\\\"left\\\">5.99\\u2009\\u00d7\\u200910<sup>\\u201345</sup></td><td align=\\\"left\\\"><bold>2.99\\u2009\\u00d7\\u200910</bold><sup><bold>\\u2013158</bold></sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.334 (0.011)</td><td align=\\\"left\\\">7.15\\u2009\\u00d7\\u200910<sup>\\u2013187</sup></td><td align=\\\"left\\\">-0.016 (0.001)</td><td align=\\\"left\\\">2.15\\u2009\\u00d7\\u200910<sup>\\u201346</sup></td><td align=\\\"left\\\"><bold>3.89\\u2009\\u00d7\\u200910</bold><sup><bold>\\u2013206</bold></sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.079 (0.010)</td><td align=\\\"left\\\">2.00\\u2009\\u00d7\\u200910<sup>\\u201314</sup></td><td align=\\\"left\\\">-0.006 (0.001)</td><td align=\\\"left\\\">5.59\\u2009\\u00d7\\u200910<sup>\\u20137</sup></td><td align=\\\"left\\\"><bold>2.52\\u2009\\u00d7\\u200910</bold><sup><bold>\\u201316</bold></sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Memory</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">-0.323 (0.008)</td><td align=\\\"left\\\">2.16\\u2009\\u00d7\\u200910<sup>\\u2013298</sup></td><td align=\\\"left\\\">-0.017 (0.001)</td><td align=\\\"left\\\">2.19\\u2009\\u00d7\\u200910<sup>\\u201376</sup></td><td align=\\\"left\\\"><bold>7.51\\u2009\\u00d7\\u200910</bold><sup><bold>\\u2013321</bold></sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">-0.536 (0.013)</td><td align=\\\"left\\\">2.38\\u2009\\u00d7\\u200910<sup>\\u2013359</sup></td><td align=\\\"left\\\">-0.017 (0.001)</td><td align=\\\"left\\\">8.14\\u2009\\u00d7\\u200910<sup>\\u201338</sup></td><td align=\\\"left\\\"><bold>3.31\\u2009\\u00d7\\u200910</bold><sup><bold>\\u2013374</bold></sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">-0.168 (0.011)</td><td align=\\\"left\\\">3.45\\u2009\\u00d7\\u200910<sup>\\u201347</sup></td><td align=\\\"left\\\">-0.012 (0.001)</td><td align=\\\"left\\\">1.00\\u2009\\u00d7\\u200910<sup>\\u201322</sup></td><td align=\\\"left\\\"><bold>4.04\\u2009\\u00d7\\u200910</bold><sup><bold>\\u201351</bold></sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"9\\\">19<sup>c</sup></td><td align=\\\"left\\\" rowspan=\\\"9\\\">44908822</td><td align=\\\"left\\\" rowspan=\\\"9\\\">rs7412</td><td align=\\\"left\\\" rowspan=\\\"9\\\">T/C (0.059)</td><td align=\\\"left\\\" rowspan=\\\"9\\\"><italic>APOE</italic></td><td align=\\\"left\\\" rowspan=\\\"3\\\">Executive Function</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">0.063 (0.011)</td><td align=\\\"left\\\">2.78\\u2009\\u00d7\\u200910<sup>\\u20138</sup></td><td align=\\\"left\\\">0.005 (0.001)</td><td align=\\\"left\\\">5.40\\u2009\\u00d7\\u200910<sup>\\u20136</sup></td><td align=\\\"left\\\"><bold>8.28\\u2009\\u00d7\\u200910</bold><sup><bold>\\u201311</bold></sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">0.180 (0.020)</td><td align=\\\"left\\\">2.98\\u2009\\u00d7\\u200910<sup>\\u201318</sup></td><td align=\\\"left\\\">0.002 (0.002)</td><td align=\\\"left\\\">2.73\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\"><bold>1.42\\u2009\\u00d7\\u200910</bold><sup><bold>\\u201317</bold></sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.016 (0.013)</td><td align=\\\"left\\\">2.39\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.005 (0.001)</td><td align=\\\"left\\\">6.71\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">2.88\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Language</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">0.055 (0.011)</td><td align=\\\"left\\\">9.77\\u2009\\u00d7\\u200910<sup>\\u20137</sup></td><td align=\\\"left\\\">0.004 (0.001)</td><td align=\\\"left\\\">1.40\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\"><bold>2.93\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20138</bold></sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">0.194 (0.021)</td><td align=\\\"left\\\">1.23\\u2009\\u00d7\\u200910<sup>\\u201319</sup></td><td align=\\\"left\\\">0.007 (0.002)</td><td align=\\\"left\\\">1.95\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\"><bold>1.19\\u2009\\u00d7\\u200910</bold><sup><bold>\\u201321</bold></sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.001 (0.013)</td><td align=\\\"left\\\">9.51\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.002 (0.001)</td><td align=\\\"left\\\">2.35\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">4.67\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Memory</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">0.105 (0.012)</td><td align=\\\"left\\\">9.64\\u2009\\u00d7\\u200910<sup>\\u201317</sup></td><td align=\\\"left\\\">0.008 (0.001)</td><td align=\\\"left\\\">1.11\\u2009\\u00d7\\u200910<sup>\\u20139</sup></td><td align=\\\"left\\\"><bold>1.11\\u2009\\u00d7\\u200910</bold><sup><bold>\\u201319</bold></sup></td></tr><tr><td align=\\\"left\\\">Clinic</td><td align=\\\"left\\\">0.302 (0.025)</td><td align=\\\"left\\\">5.81\\u2009\\u00d7\\u200910<sup>\\u201334</sup></td><td align=\\\"left\\\">0.005 (0.002)</td><td align=\\\"left\\\">2.09\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\"><bold>1.18\\u2009\\u00d7\\u200910</bold><sup><bold>\\u201334</bold></sup></td></tr><tr><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">0.040 (0.015)</td><td align=\\\"left\\\">6.93\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">0.006 (0.002)</td><td align=\\\"left\\\">9.90\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">1.73\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr></tbody></table>\\n\"}, \"text\": \"Chr\\tPosition\\tVariant\\tA1/A2a (MAF)\\tNearest Gene\\tCognitive Domain\\tCohortsb\\tGenetic effects\\t \\t\\u03b2G (SE)\\tPG\\t\\u03b2G\\u00d7Age (SE)\\tPG\\u00d7Age\\tPJoint\\t \\t1\\t207573951\\trs1752684\\tA/G (0.198)\\tCR1\\tExecutive Function\\tAll\\t-0.028 (0.007)\\t1.53\\u2009\\u00d7\\u200910\\u20134\\t-0.002 (0.001)\\t6.05\\u2009\\u00d7\\u200910\\u20133\\t5.05\\u2009\\u00d7\\u200910\\u20135\\t \\tClinic\\t-0.042 (0.012)\\t5.91\\u2009\\u00d7\\u200910\\u20134\\t-0.002 (0.001)\\t7.52\\u2009\\u00d7\\u200910\\u20132\\t6.88\\u2009\\u00d7\\u200910\\u20134\\t \\tCommunity\\t-0.020 (0.009)\\t2.98\\u2009\\u00d7\\u200910\\u20132\\t-0.002 (0.001)\\t4.90\\u2009\\u00d7\\u200910\\u20132\\t2.40\\u2009\\u00d7\\u200910\\u20132\\t \\tLanguage\\tAll\\t-0.026 (0.007)\\t3.90\\u2009\\u00d7\\u200910\\u20134\\t-0.001 (0.001)\\t6.84\\u2009\\u00d7\\u200910\\u20132\\t6.25\\u2009\\u00d7\\u200910\\u20134\\t \\tClinic\\t-0.054 (0.012)\\t1.26\\u2009\\u00d7\\u200910\\u20135\\t-0.002 (0.001)\\t5.07\\u2009\\u00d7\\u200910\\u20132\\t1.63\\u2009\\u00d7\\u200910\\u20135\\t \\tCommunity\\t-0.011 (0.009)\\t2.14\\u2009\\u00d7\\u200910\\u20131\\t-0.001 (0.001)\\t5.59\\u2009\\u00d7\\u200910\\u20131\\t4.20\\u2009\\u00d7\\u200910\\u20131\\t \\tMemory\\tAll\\t-0.043 (0.008)\\t1.75\\u2009\\u00d7\\u200910\\u20137\\t-0.002 (0.001)\\t1.83\\u2009\\u00d7\\u200910\\u20132\\t4.42\\u2009\\u00d7\\u200910\\u20137\\t \\tClinic\\t-0.080 (0.014)\\t2.44\\u2009\\u00d7\\u200910\\u20138\\t-0.004 (0.001)\\t1.05\\u2009\\u00d7\\u200910\\u20132\\t8.85\\u2009\\u00d7\\u200910\\u20139\\t \\tCommunity\\t-0.022 (0.010)\\t2.46\\u2009\\u00d7\\u200910\\u20132\\t0.000 (0.001)\\t6.45\\u2009\\u00d7\\u200910\\u20131\\t6.88\\u2009\\u00d7\\u200910\\u20132\\t \\t2\\t127135234\\trs6733839\\tT/C (0.403)\\tBIN1CYP27C1\\tExecutive Function\\tAll\\t-0.009 (0.006)\\t1.25\\u2009\\u00d7\\u200910\\u20131\\t0.000 (0.001)\\t6.54\\u2009\\u00d7\\u200910\\u20131\\t2.92\\u2009\\u00d7\\u200910\\u20131\\t \\tClinic\\t-0.023 (0.010)\\t1.77\\u2009\\u00d7\\u200910\\u20132\\t0.000 (0.001)\\t6.84\\u2009\\u00d7\\u200910\\u20131\\t5.58\\u2009\\u00d7\\u200910\\u20132\\t \\tCommunity\\t-0.001 (0.007)\\t8.70\\u2009\\u00d7\\u200910\\u20131\\t0.000 (0.001)\\t9.27\\u2009\\u00d7\\u200910\\u20131\\t9.84\\u2009\\u00d7\\u200910\\u20131\\t \\tLanguage\\tAll\\t-0.031 (0.006)\\t6.90\\u2009\\u00d7\\u200910\\u20138\\t-0.002 (0.001)\\t3.62\\u2009\\u00d7\\u200910\\u20133\\t2.70\\u2009\\u00d7\\u200910\\u20138\\t \\tClinic\\t-0.063 (0.010)\\t4.23\\u2009\\u00d7\\u200910\\u201310\\t-0.001 (0.001)\\t2.09\\u2009\\u00d7\\u200910\\u20131\\t1.98\\u2009\\u00d7\\u200910\\u20139\\t \\tCommunity\\t-0.017 (0.007)\\t1.68\\u2009\\u00d7\\u200910\\u20132\\t-0.002 (0.001)\\t1.70\\u2009\\u00d7\\u200910\\u20132\\t6.57\\u2009\\u00d7\\u200910\\u20133\\t \\tMemory\\tAll\\t-0.039 (0.006)\\t2.20\\u2009\\u00d7\\u200910\\u20139\\t-0.002 (0.001)\\t1.62\\u2009\\u00d7\\u200910\\u20133\\t2.37\\u2009\\u00d7\\u200910\\u20139\\t \\tClinic\\t-0.068 (0.012)\\t5.23\\u2009\\u00d7\\u200910\\u20139\\t-0.001 (0.001)\\t2.08\\u2009\\u00d7\\u200910\\u20131\\t1.60\\u2009\\u00d7\\u200910\\u20138\\t \\tCommunity\\t-0.027 (0.008)\\t5.71\\u2009\\u00d7\\u200910\\u20134\\t-0.002 (0.001)\\t1.69\\u2009\\u00d7\\u200910\\u20132\\t1.11\\u2009\\u00d7\\u200910\\u20133\\t \\t7\\t91069758\\trs705353\\tG/A (0.252)\\tCDK14\\tExecutive Function\\tAll\\t0.003 (0.006)\\t6.40\\u2009\\u00d7\\u200910\\u20131\\t-0.002 (0.001)\\t1.53\\u2009\\u00d7\\u200910\\u20133\\t4.58\\u2009\\u00d7\\u200910\\u20133\\t \\tClinic\\t0.000 (0.011)\\t9.75\\u2009\\u00d7\\u200910\\u20131\\t-0.005 (0.001)\\t2.03\\u2009\\u00d7\\u200910\\u20135\\t1.12\\u2009\\u00d7\\u200910\\u20134\\t \\tCommunity\\t0.006 (0.008)\\t4.20\\u2009\\u00d7\\u200910\\u20131\\t-0.001 (0.001)\\t4.11\\u2009\\u00d7\\u200910\\u20131\\t4.43\\u2009\\u00d7\\u200910\\u20131\\t \\tLanguage\\tAll\\t0.007 (0.006)\\t2.68\\u2009\\u00d7\\u200910\\u20131\\t-0.002 (0.001)\\t5.14\\u2009\\u00d7\\u200910\\u20134\\t5.93\\u2009\\u00d7\\u200910\\u20134\\t \\tClinic\\t-0.004 (0.011)\\t7.53\\u2009\\u00d7\\u200910\\u20131\\t-0.007 (0.001)\\t2.54\\u2009\\u00d7\\u200910\\u20139\\t1.73\\u2009\\u00d7\\u200910\\u20138\\t \\tCommunity\\t0.015 (0.008)\\t5.28\\u2009\\u00d7\\u200910\\u20132\\t0.000 (0.001)\\t8.28\\u2009\\u00d7\\u200910\\u20131\\t1.41\\u2009\\u00d7\\u200910\\u20131\\t \\tMemory\\tAll\\t-0.003 (0.007)\\t6.95\\u2009\\u00d7\\u200910\\u20131\\t-0.002 (0.001)\\t3.03\\u2009\\u00d7\\u200910\\u20132\\t8.72\\u2009\\u00d7\\u200910\\u20132\\t \\tClinic\\t-0.008 (0.013)\\t5.46\\u2009\\u00d7\\u200910\\u20131\\t-0.005 (0.001)\\t1.36\\u2009\\u00d7\\u200910\\u20134\\t6.11\\u2009\\u00d7\\u200910\\u20134\\t \\tCommunity\\t0.005 (0.009)\\t5.93\\u2009\\u00d7\\u200910\\u20131\\t0.000 (0.001)\\t9.19\\u2009\\u00d7\\u200910\\u20131\\t8.59\\u2009\\u00d7\\u200910\\u20131\\t \\t8\\t30963282\\trs117523305\\tC/G (0.016)\\tPURGTEX15\\tExecutive Function\\tAll\\t-0.001 (0.022)\\t9.67\\u2009\\u00d7\\u200910\\u20131\\t-0.006 (0.002)\\t5.35\\u2009\\u00d7\\u200910\\u20133\\t2.02\\u2009\\u00d7\\u200910\\u20132\\t \\tClinic\\t-0.012 (0.038)\\t7.42\\u2009\\u00d7\\u200910\\u20131\\t-0.002 (0.004)\\t5.89\\u2009\\u00d7\\u200910\\u20131\\t8.23\\u2009\\u00d7\\u200910\\u20131\\t \\tCommunity\\t0.003 (0.026)\\t9.13\\u2009\\u00d7\\u200910\\u20131\\t-0.008 (0.003)\\t2.65\\u2009\\u00d7\\u200910\\u20133\\t9.18\\u2009\\u00d7\\u200910\\u20133\\t \\tLanguage\\tAll\\t-0.018 (0.021)\\t4.02\\u2009\\u00d7\\u200910\\u20131\\t-0.008 (0.002)\\t8.23\\u2009\\u00d7\\u200910\\u20134\\t3.01\\u2009\\u00d7\\u200910\\u20133\\t \\tClinic\\t0.028 (0.039)\\t4.80\\u2009\\u00d7\\u200910\\u20131\\t-0.001 (0.004)\\t7.03\\u2009\\u00d7\\u200910\\u20131\\t7.11\\u2009\\u00d7\\u200910\\u20131\\t \\tCommunity\\t-0.041 (0.025)\\t1.19\\u2009\\u00d7\\u200910\\u20131\\t-0.011 (0.003)\\t9.16\\u2009\\u00d7\\u200910\\u20135\\t1.87\\u2009\\u00d7\\u200910\\u20134\\t \\tMemory\\tAll\\t-0.069 (0.024)\\t5.27\\u2009\\u00d7\\u200910\\u20133\\t-0.010 (0.002)\\t5.18\\u2009\\u00d7\\u200910\\u20135\\t3.62\\u2009\\u00d7\\u200910\\u20135\\t \\tClinic\\t-0.041 (0.044)\\t3.60\\u2009\\u00d7\\u200910\\u20131\\t0.004 (0.004)\\t3.34\\u2009\\u00d7\\u200910\\u20131\\t3.88\\u2009\\u00d7\\u200910\\u20131\\t \\tCommunity\\t-0.098 (0.029)\\t7.87\\u2009\\u00d7\\u200910\\u20134\\t-0.018 (0.003)\\t1.41\\u2009\\u00d7\\u200910\\u20138\\t1.73\\u2009\\u00d7\\u200910\\u20138\\t \\t11\\t60173126\\trs7232\\tA/T (0.361)\\tMS4A6A\\tExecutive Function\\tAll\\t0.012 (0.006)\\t3.46\\u2009\\u00d7\\u200910\\u20132\\t0.002 (0.001)\\t2.90\\u2009\\u00d7\\u200910\\u20133\\t2.38\\u2009\\u00d7\\u200910\\u20133\\t \\tClinic\\t0.021 (0.010)\\t3.46\\u2009\\u00d7\\u200910\\u20132\\t0.000 (0.001)\\t8.60\\u2009\\u00d7\\u200910\\u20131\\t1.06\\u2009\\u00d7\\u200910\\u20131\\t \\tCommunity\\t0.009 (0.007)\\t2.07\\u2009\\u00d7\\u200910\\u20131\\t0.002 (0.001)\\t6.28\\u2009\\u00d7\\u200910\\u20134\\t2.09\\u2009\\u00d7\\u200910\\u20133\\t \\tLanguage\\tAll\\t0.016 (0.006)\\t6.40\\u2009\\u00d7\\u200910\\u20133\\t0.002 (0.001)\\t1.04\\u2009\\u00d7\\u200910\\u20134\\t3.61\\u2009\\u00d7\\u200910\\u20135\\t \\tClinic\\t0.045 (0.010)\\t1.40\\u2009\\u00d7\\u200910\\u20135\\t0.002 (0.001)\\t4.91\\u2009\\u00d7\\u200910\\u20132\\t1.55\\u2009\\u00d7\\u200910\\u20135\\t \\tCommunity\\t0.003 (0.007)\\t6.38\\u2009\\u00d7\\u200910\\u20131\\t0.002 (0.001)\\t1.93\\u2009\\u00d7\\u200910\\u20133\\t6.31\\u2009\\u00d7\\u200910\\u20133\\t \\tMemory\\tAll\\t0.027 (0.006)\\t5.19\\u2009\\u00d7\\u200910\\u20135\\t0.003 (0.001)\\t2.60\\u2009\\u00d7\\u200910\\u20135\\t9.43\\u2009\\u00d7\\u200910\\u20137\\t \\tClinic\\t0.068 (0.012)\\t1.25\\u2009\\u00d7\\u200910\\u20138\\t0.001 (0.001)\\t2.27\\u2009\\u00d7\\u200910\\u20131\\t3.97\\u2009\\u00d7\\u200910\\u20138\\t \\tCommunity\\t0.011 (0.008)\\t1.50\\u2009\\u00d7\\u200910\\u20131\\t0.003 (0.001)\\t4.62\\u2009\\u00d7\\u200910\\u20134\\t1.80\\u2009\\u00d7\\u200910\\u20133\\t \\t11\\t127595711\\trs145012974\\tA/G (0.014)\\tLINC02712LINC02098\\tExecutive Function\\tAll\\t-0.040 (0.024)\\t1.03\\u2009\\u00d7\\u200910\\u20131\\t-0.007 (0.002)\\t4.72\\u2009\\u00d7\\u200910\\u20133\\t7.25\\u2009\\u00d7\\u200910\\u20133\\t \\tClinic\\t-0.013 (0.041)\\t7.53\\u2009\\u00d7\\u200910\\u20131\\t-0.004 (0.004)\\t3.14\\u2009\\u00d7\\u200910\\u20131\\t5.72\\u2009\\u00d7\\u200910\\u20131\\t \\tCommunity\\t-0.056 (0.030)\\t6.46\\u2009\\u00d7\\u200910\\u20132\\t-0.008 (0.003)\\t5.11\\u2009\\u00d7\\u200910\\u20133\\t6.48\\u2009\\u00d7\\u200910\\u20133\\t \\tLanguage\\tAll\\t-0.029 (0.024)\\t2.29\\u2009\\u00d7\\u200910\\u20131\\t-0.014 (0.002)\\t7.69\\u2009\\u00d7\\u200910\\u20139\\t3.66\\u2009\\u00d7\\u200910\\u20138\\t \\tClinic\\t-0.026 (0.041)\\t5.31\\u2009\\u00d7\\u200910\\u20131\\t-0.012 (0.004)\\t2.16\\u2009\\u00d7\\u200910\\u20133\\t7.23\\u2009\\u00d7\\u200910\\u20133\\t \\tCommunity\\t-0.033 (0.029)\\t2.70\\u2009\\u00d7\\u200910\\u20131\\t-0.016 (0.003)\\t1.00\\u2009\\u00d7\\u200910\\u20136\\t3.29\\u2009\\u00d7\\u200910\\u20136\\t \\tMemory\\tAll\\t-0.052 (0.026)\\t5.08\\u2009\\u00d7\\u200910\\u20132\\t-0.008 (0.003)\\t2.46\\u2009\\u00d7\\u200910\\u20133\\t3.99\\u2009\\u00d7\\u200910\\u20133\\t \\tClinic\\t-0.051 (0.047)\\t2.87\\u2009\\u00d7\\u200910\\u20131\\t-0.010 (0.005)\\t3.06\\u2009\\u00d7\\u200910\\u20132\\t5.42\\u2009\\u00d7\\u200910\\u20132\\t \\tCommunity\\t-0.050 (0.032)\\t1.24\\u2009\\u00d7\\u200910\\u20131\\t-0.007 (0.003)\\t3.00\\u2009\\u00d7\\u200910\\u20132\\t6.31\\u2009\\u00d7\\u200910\\u20132\\t \\t17\\t19806731\\trs157405\\tG/A (0.023)\\tULK2\\tExecutive Function\\tAll\\t0.063 (0.018)\\t8.61\\u2009\\u00d7\\u200910\\u20134\\t0.008 (0.002)\\t1.04\\u2009\\u00d7\\u200910\\u20135\\t1.49\\u2009\\u00d7\\u200910\\u20136\\t \\tClinic\\t-0.024 (0.032)\\t4.62\\u2009\\u00d7\\u200910\\u20131\\t0.002 (0.003)\\t4.25\\u2009\\u00d7\\u200910\\u20131\\t5.20\\u2009\\u00d7\\u200910\\u20131\\t \\tCommunity\\t0.107 (0.023)\\t3.21\\u2009\\u00d7\\u200910\\u20136\\t0.011 (0.002)\\t7.65\\u2009\\u00d7\\u200910\\u20137\\t2.19\\u2009\\u00d7\\u200910\\u20139\\t \\tLanguage\\tAll\\t0.029 (0.018)\\t1.19\\u2009\\u00d7\\u200910\\u20131\\t0.002 (0.002)\\t2.04\\u2009\\u00d7\\u200910\\u20131\\t1.70\\u2009\\u00d7\\u200910\\u20131\\t \\tClinic\\t-0.040 (0.033)\\t2.29\\u2009\\u00d7\\u200910\\u20131\\t0.002 (0.003)\\t4.81\\u2009\\u00d7\\u200910\\u20131\\t3.35\\u2009\\u00d7\\u200910\\u20131\\t \\tCommunity\\t0.059 (0.022)\\t8.39\\u2009\\u00d7\\u200910\\u20133\\t0.003 (0.002)\\t2.54\\u2009\\u00d7\\u200910\\u20131\\t2.21\\u2009\\u00d7\\u200910\\u20132\\t \\tMemory\\tAll\\t0.046 (0.020)\\t2.78\\u2009\\u00d7\\u200910\\u20132\\t0.001 (0.002)\\t5.87\\u2009\\u00d7\\u200910\\u20131\\t8.07\\u2009\\u00d7\\u200910\\u20132\\t \\tClinic\\t-0.005 (0.038)\\t9.07\\u2009\\u00d7\\u200910\\u20131\\t0.002 (0.004)\\t5.39\\u2009\\u00d7\\u200910\\u20131\\t8.14\\u2009\\u00d7\\u200910\\u20131\\t \\tCommunity\\t0.064 (0.024)\\t9.43\\u2009\\u00d7\\u200910\\u20133\\t0.001 (0.003)\\t6.48\\u2009\\u00d7\\u200910\\u20131\\t2.71\\u2009\\u00d7\\u200910\\u20132\\t \\t17\\t44352876\\trs5848\\tT/C (0.305)\\tGRN\\tExecutive Function\\tAll\\t-0.007 (0.006)\\t2.53\\u2009\\u00d7\\u200910\\u20131\\t-0.002 (0.001)\\t1.32\\u2009\\u00d7\\u200910\\u20134\\t5.43\\u2009\\u00d7\\u200910\\u20134\\t \\tClinic\\t0.003 (0.010)\\t7.50\\u2009\\u00d7\\u200910\\u20131\\t-0.003 (0.001)\\t4.56\\u2009\\u00d7\\u200910\\u20133\\t1.67\\u2009\\u00d7\\u200910\\u20132\\t \\tCommunity\\t-0.012 (0.008)\\t1.14\\u2009\\u00d7\\u200910\\u20131\\t-0.002 (0.001)\\t5.76\\u2009\\u00d7\\u200910\\u20133\\t1.15\\u2009\\u00d7\\u200910\\u20132\\t \\tLanguage\\tAll\\t-0.012 (0.006)\\t5.99\\u2009\\u00d7\\u200910\\u20132\\t-0.002 (0.001)\\t2.86\\u2009\\u00d7\\u200910\\u20133\\t3.53\\u2009\\u00d7\\u200910\\u20133\\t \\tClinic\\t-0.015 (0.011)\\t1.71\\u2009\\u00d7\\u200910\\u20131\\t-0.001 (0.001)\\t4.67\\u2009\\u00d7\\u200910\\u20131\\t3.11\\u2009\\u00d7\\u200910\\u20131\\t \\tCommunity\\t-0.011 (0.007)\\t1.34\\u2009\\u00d7\\u200910\\u20131\\t-0.003 (0.001)\\t1.56\\u2009\\u00d7\\u200910\\u20133\\t3.43\\u2009\\u00d7\\u200910\\u20133\\t \\tMemory\\tAll\\t-0.029 (0.007)\\t2.19\\u2009\\u00d7\\u200910\\u20135\\t-0.003 (0.001)\\t1.07\\u2009\\u00d7\\u200910\\u20136\\t4.21\\u2009\\u00d7\\u200910\\u20138\\t \\tClinic\\t-0.032 (0.012)\\t9.58\\u2009\\u00d7\\u200910\\u20133\\t-0.003 (0.001)\\t3.30\\u2009\\u00d7\\u200910\\u20133\\t5.28\\u2009\\u00d7\\u200910\\u20134\\t \\tCommunity\\t-0.028 (0.008)\\t8.11\\u2009\\u00d7\\u200910\\u20134\\t-0.003 (0.001)\\t1.29\\u2009\\u00d7\\u200910\\u20134\\t6.50\\u2009\\u00d7\\u200910\\u20135\\t \\t19c\\t44908684\\trs429358\\tC/T (0.231)\\tAPOE\\tExecutive Function\\tAll\\t-0.192 (0.008)\\t6.75\\u2009\\u00d7\\u200910\\u2013135\\t-0.009 (0.001)\\t1.26\\u2009\\u00d7\\u200910\\u201328\\t8.58\\u2009\\u00d7\\u200910\\u2013145\\t \\tClinic\\t-0.308 (0.011)\\t5.51\\u2009\\u00d7\\u200910\\u2013166\\t-0.010 (0.001)\\t1.71\\u2009\\u00d7\\u200910\\u201319\\t1.86\\u2009\\u00d7\\u200910\\u2013170\\t \\tCommunity\\t-0.089 (0.010)\\t5.21\\u2009\\u00d7\\u200910\\u201317\\t-0.006 (0.001)\\t4.58\\u2009\\u00d7\\u200910\\u20139\\t7.05\\u2009\\u00d7\\u200910\\u201320\\t \\tLanguage\\tAll\\t-0.193 (0.008)\\t1.81\\u2009\\u00d7\\u200910\\u2013137\\t-0.011 (0.001)\\t5.99\\u2009\\u00d7\\u200910\\u201345\\t2.99\\u2009\\u00d7\\u200910\\u2013158\\t \\tClinic\\t-0.334 (0.011)\\t7.15\\u2009\\u00d7\\u200910\\u2013187\\t-0.016 (0.001)\\t2.15\\u2009\\u00d7\\u200910\\u201346\\t3.89\\u2009\\u00d7\\u200910\\u2013206\\t \\tCommunity\\t-0.079 (0.010)\\t2.00\\u2009\\u00d7\\u200910\\u201314\\t-0.006 (0.001)\\t5.59\\u2009\\u00d7\\u200910\\u20137\\t2.52\\u2009\\u00d7\\u200910\\u201316\\t \\tMemory\\tAll\\t-0.323 (0.008)\\t2.16\\u2009\\u00d7\\u200910\\u2013298\\t-0.017 (0.001)\\t2.19\\u2009\\u00d7\\u200910\\u201376\\t7.51\\u2009\\u00d7\\u200910\\u2013321\\t \\tClinic\\t-0.536 (0.013)\\t2.38\\u2009\\u00d7\\u200910\\u2013359\\t-0.017 (0.001)\\t8.14\\u2009\\u00d7\\u200910\\u201338\\t3.31\\u2009\\u00d7\\u200910\\u2013374\\t \\tCommunity\\t-0.168 (0.011)\\t3.45\\u2009\\u00d7\\u200910\\u201347\\t-0.012 (0.001)\\t1.00\\u2009\\u00d7\\u200910\\u201322\\t4.04\\u2009\\u00d7\\u200910\\u201351\\t \\t19c\\t44908822\\trs7412\\tT/C (0.059)\\tAPOE\\tExecutive Function\\tAll\\t0.063 (0.011)\\t2.78\\u2009\\u00d7\\u200910\\u20138\\t0.005 (0.001)\\t5.40\\u2009\\u00d7\\u200910\\u20136\\t8.28\\u2009\\u00d7\\u200910\\u201311\\t \\tClinic\\t0.180 (0.020)\\t2.98\\u2009\\u00d7\\u200910\\u201318\\t0.002 (0.002)\\t2.73\\u2009\\u00d7\\u200910\\u20131\\t1.42\\u2009\\u00d7\\u200910\\u201317\\t \\tCommunity\\t0.016 (0.013)\\t2.39\\u2009\\u00d7\\u200910\\u20131\\t0.005 (0.001)\\t6.71\\u2009\\u00d7\\u200910\\u20135\\t2.88\\u2009\\u00d7\\u200910\\u20134\\t \\tLanguage\\tAll\\t0.055 (0.011)\\t9.77\\u2009\\u00d7\\u200910\\u20137\\t0.004 (0.001)\\t1.40\\u2009\\u00d7\\u200910\\u20134\\t2.93\\u2009\\u00d7\\u200910\\u20138\\t \\tClinic\\t0.194 (0.021)\\t1.23\\u2009\\u00d7\\u200910\\u201319\\t0.007 (0.002)\\t1.95\\u2009\\u00d7\\u200910\\u20134\\t1.19\\u2009\\u00d7\\u200910\\u201321\\t \\tCommunity\\t0.001 (0.013)\\t9.51\\u2009\\u00d7\\u200910\\u20131\\t0.002 (0.001)\\t2.35\\u2009\\u00d7\\u200910\\u20131\\t4.67\\u2009\\u00d7\\u200910\\u20131\\t \\tMemory\\tAll\\t0.105 (0.012)\\t9.64\\u2009\\u00d7\\u200910\\u201317\\t0.008 (0.001)\\t1.11\\u2009\\u00d7\\u200910\\u20139\\t1.11\\u2009\\u00d7\\u200910\\u201319\\t \\tClinic\\t0.302 (0.025)\\t5.81\\u2009\\u00d7\\u200910\\u201334\\t0.005 (0.002)\\t2.09\\u2009\\u00d7\\u200910\\u20132\\t1.18\\u2009\\u00d7\\u200910\\u201334\\t \\tCommunity\\t0.040 (0.015)\\t6.93\\u2009\\u00d7\\u200910\\u20133\\t0.006 (0.002)\\t9.90\\u2009\\u00d7\\u200910\\u20135\\t1.73\\u2009\\u00d7\\u200910\\u20134\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29446, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"aA1 Minor allele, A2 Reference allele, MAF Minor allele frequency\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29512, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"bAll Total sample, Clinic Clinic-based cohorts, Community Community-based cohorts\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29594, \"infons\": {\"file\": \"Tab2.xml\", \"id\": \"Tab2\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"cWe presented GWAS results for the APOE SNPs\\u2014\\u03b54 (rs429358) and \\u03b52 (rs7412) alleles\\u2014but excluded all the other results in the APOE region\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29743, \"infons\": {\"file\": \"13024_2023_633_Fig1_HTML.jpg\", \"id\": \"Fig1\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Locus Zoom plots showing the association of SNPs in the regions of novel loci with cognitive domains. The SNP with the lowest p-value at each locus is indicated with a purple diamond. Computed estimates of linkage disequilibrium (r2) of SNPs in the region with top-ranked SNP are color-coded according to the key. Vertical blue lines indicate locations of high recombination rates. Locations of genes in the region are shown below the diagram. a Association of rs157405 with executive function in the community-based cohorts. b Association of rs705353 with language in the clinic-based cohorts. c Association of rs117523305 with memory in the community-based cohorts. d Association of rs145012974 with language in the total sample. e Association of rs5848 with memory in the total sample\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 30531, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"There was little evidence of genomic inflation (\\u03bb\\u2009=\\u20091.006\\u20131.023) in the GWAS for each cognitive domain and strata of the sample (Figs. S2, S3 and S4). GWS associations were observed for many SNPs in the\\u00a0APOE\\u00a0region for all traits (Table S3). We also identified associations with several other established AD loci (Table 2). BIN1 SNP rs6733839 was associated with language (PJoint\\u2009=\\u20092.70\\u2009\\u00d7\\u200910\\u20138) and memory (PJoint\\u2009=\\u20092.37\\u2009\\u00d7\\u200910\\u20139) in the total sample and with both language (PJoint\\u2009=\\u20091.98\\u2009\\u00d7\\u200910\\u20139) and memory (PJoint\\u2009=\\u20091.60\\u2009\\u00d7\\u200910\\u20138) in the clinic-based cohorts. The significant joint effect of rs6733839 is due primarily to the SNP\\u2019s main effect rather than its interaction with age and is supported by multiple adjacent variants (Figs. S5 and S6). GWS associations for memory were also observed with CR1 SNP rs1752684 (PJoint\\u2009=\\u20098.85\\u2009\\u00d7\\u200910\\u20139) and\\u00a0MS4A6A\\u00a0SNP rs7232 (PJoint\\u2009=\\u20093.97\\u2009\\u00d7\\u200910\\u20138) in the clinic-based cohorts, findings which were supported by adjacent SNPs (Fig. S7). Similar to\\u00a0BIN1, results of the joint test of the main and interaction effects for rs1752684 and rs7232 reflect the SNPs\\u2019 main effects. GWS associations were also detected with SNPs in four additional loci,\\u00a0ULK2 (rs157405,\\u00a0PJoint\\u2009=\\u20092.19\\u2009\\u00d7\\u200910\\u20139) with executive function in the community-based cohorts,\\u00a0CDK14 (rs705353,\\u00a0PJoint\\u2009=\\u20091.73\\u2009\\u00d7\\u200910\\u20138) with language in the clinic-based cohorts, PURG\\u00a0(rs117523305, PJoint\\u2009=\\u20091.73\\u2009\\u00d7\\u200910\\u20138) with memory in the community-based cohorts, and LINC02712 (rs145012974, PJoint\\u2009=\\u20093.66\\u2009\\u00d7\\u200910\\u20138) with language in the total sample (Fig.\\u00a01). We also identified a GWS association of memory with GRN (rs5848, PJoint\\u2009=\\u20094.21\\u2009\\u00d7\\u200910\\u20138) in the total sample (Fig.\\u00a01). Unlike the associations with the other known AD loci, the interactions of the ULK2 (PG\\u00d7Age\\u2009=\\u20097.65\\u2009\\u00d7\\u200910\\u20137),\\u00a0CDK14 (PG\\u00d7Age\\u2009=\\u20092.54\\u2009\\u00d7\\u200910\\u20139), PURG (PG\\u00d7Age\\u2009=\\u20091.41\\u2009\\u00d7\\u200910\\u20138), LINC02712 (PG\\u00d7Age\\u2009=\\u20097.69\\u2009\\u00d7\\u200910\\u20139), and GRN (PG\\u00d7Age\\u2009=\\u20091.07\\u2009\\u00d7\\u200910\\u20136) SNPs with age accounted for the significant joint test findings (Table 2).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 32691, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"The\\u00a0APOE\\u00a0region comprised many variants significantly associated with all the cognitive domains in all cohort groupings (Table S3). The high LD between these variants suggests that there are not multiple independent association signals, a conclusion supported by evidence that no genes in this region other than the\\u00a0APOE\\u00a0account for the observed association with AD risk or onset age. Focusing on the\\u00a0APOE\\u00a0SNPs encoding the\\u00a0\\u03b54 (rs429358) and\\u00a0\\u03b52 (rs7412) alleles, the\\u00a0\\u03b54 SNP was significantly associated with lower (worse) scores for all cognitive domains in an age-dependent manner, based on the negative sign of \\u03b2G\\u00d7Age. Notably, the magnitude of the effect of the\\u00a0\\u03b54 SNP on memory was approximately 1.7 times greater than on executive function or language in the total sample at the median age (Table 2). In the same sample, the effect of interaction between\\u00a0\\u03b54 and age was 1.5\\u20131.9 times larger for memory compared to executive function or language. Conversely, the\\u00a0\\u03b52 SNP was significantly associated with higher (better) cognitive domain scores in the clinic-based cohorts at the median age with some limited age-dependent effect (Table 2).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 33874, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Numerous highly suggestive associations (P\\u2009<\\u20091\\u2009\\u00d7\\u200910\\u20136), including several that were nearly GWS (P\\u2009<\\u20091\\u2009\\u00d7\\u200910\\u20137), were found for individual cognitive domains with other loci (Table S3). Notably, language was associated with 18\\u00a0ADCY2\\u00a0SNPs in the community-based cohorts (top SNP: rs7734697, PJoint\\u2009=\\u20096.34\\u2009\\u00d7\\u200910\\u20138) and with 19\\u00a0DAPK2\\u00a0SNPs in the clinic-based cohorts (top SNP: rs112972763, PJoint\\u2009=\\u20097.76\\u2009\\u00d7\\u200910\\u20138). Six\\u00a0PLXDC2\\u00a0SNPs were associated with executive function in the total sample (top SNP: rs7083449, PJoint\\u2009=\\u20097.02\\u2009\\u00d7\\u200910\\u20138), and most of the evidence was derived from the community-based cohorts.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34532, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Genome-wide pleiotropy analysis identifies the association of cognitive domains with the progranulin gene and four novel loci\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34658, \"infons\": {\"file\": \"Tab3.xml\", \"id\": \"Tab3\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Genome-wide significant pleiotropic loci for each pair of cognitive domains (excluding the APOE region)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34762, \"infons\": {\"file\": \"Tab3.xml\", \"id\": \"Tab3\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"hsides\\\" rules=\\\"groups\\\"><thead><tr><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>Chr</bold></th><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>Position</bold></th><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>Variant</bold></th><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>A1/A2</bold><sup><bold>a</bold></sup><bold> (MAF)</bold></th><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>Nearest Gene</bold></th><th align=\\\"left\\\" rowspan=\\\"2\\\"><bold>Cognitive Domain</bold></th><th align=\\\"left\\\" colspan=\\\"5\\\"><bold>Individual locus</bold></th><th align=\\\"left\\\" colspan=\\\"3\\\"><bold>Pleiotropy</bold></th></tr><tr><th align=\\\"left\\\"><bold><italic>\\u03b2</italic></bold><sub><bold>G</bold></sub><bold> (SE)</bold></th><th align=\\\"left\\\"><bold><italic>P</italic></bold><sub><bold>G</bold></sub></th><th align=\\\"left\\\"><bold><italic>\\u03b2</italic></bold><sub><bold>G\\u00d7Age</bold></sub><bold> (SE)</bold></th><th align=\\\"left\\\"><bold><italic>P</italic></bold><sub><bold>G\\u00d7Age</bold></sub></th><th align=\\\"left\\\"><bold><italic>P</italic></bold><sub><bold>Joint</bold></sub></th><th align=\\\"left\\\"><bold><italic>P</italic></bold><sub><bold>Placo, G</bold></sub></th><th align=\\\"left\\\"><bold><italic>P</italic></bold><sub><bold>Placo, G\\u00d7Age</bold></sub></th><th align=\\\"left\\\"><bold><italic>P</italic></bold><sub><bold>Placo, Joint</bold></sub></th></tr></thead><tbody><tr><td align=\\\"left\\\" colspan=\\\"14\\\"><bold>Total sample</bold></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20032</td><td align=\\\"left\\\" rowspan=\\\"2\\\">127135234</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs6733839</td><td align=\\\"left\\\" rowspan=\\\"2\\\">T/C (0.403)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><p><italic>BIN1</italic></p><p><italic>CYP27C1</italic></p></td><td align=\\\"left\\\">Language</td><td align=\\\"left\\\">-0.031 (0.006)</td><td align=\\\"left\\\">6.90\\u2009\\u00d7\\u200910<sup>\\u20138</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">3.62\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">2.70\\u2009\\u00d7\\u200910<sup>\\u20138</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">2.72\\u2009\\u00d7\\u200910<sup>\\u201310</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">9.11\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\"><bold>9.01\\u2009\\u00d7\\u200910</bold><sup><bold>\\u201312</bold></sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">-0.039 (0.006)</td><td align=\\\"left\\\">2.20\\u2009\\u00d7\\u200910<sup>\\u20139</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">1.62\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">2.37\\u2009\\u00d7\\u200910<sup>\\u20139</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20034</td><td align=\\\"left\\\" rowspan=\\\"2\\\">179150192</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs73005629</td><td align=\\\"left\\\" rowspan=\\\"2\\\">C/T (0.022)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><italic>LOC107984373</italic></td><td align=\\\"left\\\">Language</td><td align=\\\"left\\\">-0.048 (0.019)</td><td align=\\\"left\\\">1.21\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">-0.007 (0.002)</td><td align=\\\"left\\\">2.06\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">1.19\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">3.69\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">9.32\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">6.16\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">-0.062 (0.021)</td><td align=\\\"left\\\">3.97\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">-0.006 (0.002)</td><td align=\\\"left\\\">8.12\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">2.11\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20038</td><td align=\\\"left\\\" rowspan=\\\"2\\\">102060792</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs56162098</td><td align=\\\"left\\\" rowspan=\\\"2\\\">C/T (0.138)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><italic>NCALD</italic></td><td align=\\\"left\\\">Language</td><td align=\\\"left\\\">0.019 (0.008)</td><td align=\\\"left\\\">1.64\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">0.001 (0.001)</td><td align=\\\"left\\\">1.84\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">3.08\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">5.85\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">5.74\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">6.86\\u2009\\u00d7\\u200910<sup>\\u20136</sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">-0.013 (0.009)</td><td align=\\\"left\\\">1.68\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.001 (0.001)</td><td align=\\\"left\\\">1.71\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">2.17\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20039</td><td align=\\\"left\\\" rowspan=\\\"2\\\">10155013</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs145989094</td><td align=\\\"left\\\" rowspan=\\\"2\\\">A/T (0.013)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><italic>PTPRD</italic></td><td align=\\\"left\\\">Exec Function</td><td align=\\\"left\\\">-0.056 (0.026)</td><td align=\\\"left\\\">3.22\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">-0.006 (0.002)</td><td align=\\\"left\\\">1.26\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">1.04\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">2.70\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">2.79\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">9.93\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">-0.058 (0.028)</td><td align=\\\"left\\\">4.56\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">-0.008 (0.003)</td><td align=\\\"left\\\">3.14\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">5.92\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20039</td><td align=\\\"left\\\" rowspan=\\\"2\\\">10155013</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs145989094</td><td align=\\\"left\\\" rowspan=\\\"2\\\">A/T (0.013)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><italic>PTPRD</italic></td><td align=\\\"left\\\">Language</td><td align=\\\"left\\\">-0.093 (0.025)</td><td align=\\\"left\\\">3.40\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">-0.004 (0.003)</td><td align=\\\"left\\\">9.29\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">8.84\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">1.22\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">3.47\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">4.65\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">-0.058 (0.028)</td><td align=\\\"left\\\">4.56\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">-0.008 (0.003)</td><td align=\\\"left\\\">3.14\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">5.92\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u200316</td><td align=\\\"left\\\" rowspan=\\\"2\\\">83947805</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs12447050</td><td align=\\\"left\\\" rowspan=\\\"2\\\">C/T (0.400)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><p><italic>OSGIN1</italic></p><p><italic>MLYCD</italic></p></td><td align=\\\"left\\\">Exec Function</td><td align=\\\"left\\\">0.016 (0.006)</td><td align=\\\"left\\\">4.70\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">0.002 (0.001)</td><td align=\\\"left\\\">4.01\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">1.03\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">2.03\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">9.76\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">3.72\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">0.018 (0.006)</td><td align=\\\"left\\\">4.92\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">0.002 (0.001)</td><td align=\\\"left\\\">9.28\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">2.61\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" colspan=\\\"14\\\"><bold>Clinic-based cohorts</bold></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20032</td><td align=\\\"left\\\" rowspan=\\\"2\\\">127135234</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs6733839</td><td align=\\\"left\\\" rowspan=\\\"2\\\">T/C (0.409)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><p><italic>BIN1</italic></p><p><italic>CYP27C1</italic></p></td><td align=\\\"left\\\">Language</td><td align=\\\"left\\\">-0.063 (0.010)</td><td align=\\\"left\\\">4.23\\u2009\\u00d7\\u200910<sup>\\u201310</sup></td><td align=\\\"left\\\">-0.001 (0.001)</td><td align=\\\"left\\\">2.09\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">1.98\\u2009\\u00d7\\u200910<sup>\\u20139</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">1.78\\u2009\\u00d7\\u200910<sup>\\u201310</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">2.20\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\"><bold>6.85\\u2009\\u00d7\\u200910</bold><sup><bold>\\u201310</bold></sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">-0.068 (0.012)</td><td align=\\\"left\\\">5.23\\u2009\\u00d7\\u200910<sup>\\u20139</sup></td><td align=\\\"left\\\">-0.001 (0.001)</td><td align=\\\"left\\\">2.08\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">1.60\\u2009\\u00d7\\u200910<sup>\\u20138</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20034</td><td align=\\\"left\\\" rowspan=\\\"2\\\">179150192</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs73005629</td><td align=\\\"left\\\" rowspan=\\\"2\\\">C/T (0.022)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><italic>LOC107984373</italic></td><td align=\\\"left\\\">Language</td><td align=\\\"left\\\">-0.112 (0.033)</td><td align=\\\"left\\\">7.70\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">-0.014 (0.003)</td><td align=\\\"left\\\">1.68\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">4.66\\u2009\\u00d7\\u200910<sup>\\u20137</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">2.06\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">5.11\\u2009\\u00d7\\u200910<sup>\\u20136</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\"><bold>3.12\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20138</bold></sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">-0.149 (0.038)</td><td align=\\\"left\\\">1.13\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">-0.015 (0.004)</td><td align=\\\"left\\\">4.04\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">1.47\\u2009\\u00d7\\u200910<sup>\\u20137</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20038</td><td align=\\\"left\\\" rowspan=\\\"2\\\">102060792</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs56162098</td><td align=\\\"left\\\" rowspan=\\\"2\\\">C/T (0.138)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><italic>NCALD</italic></td><td align=\\\"left\\\">Language</td><td align=\\\"left\\\">0.006 (0.014)</td><td align=\\\"left\\\">6.89\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.000 (0.001)</td><td align=\\\"left\\\">7.49\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">8.81\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">7.74\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">9.29\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">9.56\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">0.009 (0.017)</td><td align=\\\"left\\\">5.73\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.001 (0.002)</td><td align=\\\"left\\\">3.54\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">5.62\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20039</td><td align=\\\"left\\\" rowspan=\\\"2\\\">10155013</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs145989094</td><td align=\\\"left\\\" rowspan=\\\"2\\\">A/T (0.013)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><italic>PTPRD</italic></td><td align=\\\"left\\\">Exec Function</td><td align=\\\"left\\\">0.016 (0.044)</td><td align=\\\"left\\\">7.25\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.002 (0.004)</td><td align=\\\"left\\\">7.27\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">8.85\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">4.38\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">8.92\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">7.33\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">0.081 (0.053)</td><td align=\\\"left\\\">1.28\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.000 (0.005)</td><td align=\\\"left\\\">9.83\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">3.08\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20039</td><td align=\\\"left\\\" rowspan=\\\"2\\\">10155013</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs145989094</td><td align=\\\"left\\\" rowspan=\\\"2\\\">A/T (0.013)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><italic>PTPRD</italic></td><td align=\\\"left\\\">Language</td><td align=\\\"left\\\">0.026 (0.046)</td><td align=\\\"left\\\">5.78\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.002 (0.004)</td><td align=\\\"left\\\">6.27\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">7.68\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">5.27\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">7.63\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">7.82\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">0.081 (0.053)</td><td align=\\\"left\\\">1.28\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.000 (0.005)</td><td align=\\\"left\\\">9.83\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">3.08\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u200316</td><td align=\\\"left\\\" rowspan=\\\"2\\\">83947805</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs12447050</td><td align=\\\"left\\\" rowspan=\\\"2\\\">C/T (0.404)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><p><italic>OSGIN1</italic></p><p><italic>MLYCD</italic></p></td><td align=\\\"left\\\">Exec Function</td><td align=\\\"left\\\">0.005 (0.010)</td><td align=\\\"left\\\">6.31\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.001 (0.001)</td><td align=\\\"left\\\">4.59\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">6.83\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">8.02\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">3.63\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">6.41\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">0.001 (0.012)</td><td align=\\\"left\\\">9.56\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.000 (0.001)</td><td align=\\\"left\\\">7.65\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">9.54\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\" colspan=\\\"14\\\"><bold>Community-based cohorts</bold></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20032</td><td align=\\\"left\\\" rowspan=\\\"2\\\">127135234</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs6733839</td><td align=\\\"left\\\" rowspan=\\\"2\\\">T/C (0.388)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><p><italic>BIN1</italic></p><p><italic>CYP27C1</italic></p></td><td align=\\\"left\\\">Language</td><td align=\\\"left\\\">-0.017 (0.007)</td><td align=\\\"left\\\">1.68\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">1.70\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">6.57\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">1.99\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">8.10\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">2.52\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">-0.027 (0.008)</td><td align=\\\"left\\\">5.71\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">-0.002 (0.001)</td><td align=\\\"left\\\">1.69\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">1.11\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20034</td><td align=\\\"left\\\" rowspan=\\\"2\\\">179150192</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs73005629</td><td align=\\\"left\\\" rowspan=\\\"2\\\">C/T (0.021)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><italic>LOC107984373</italic></td><td align=\\\"left\\\">Language</td><td align=\\\"left\\\">-0.011 (0.023)</td><td align=\\\"left\\\">6.32\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">-0.003 (0.002)</td><td align=\\\"left\\\">2.25\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">4.48\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">8.73\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">2.86\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">5.58\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">-0.006 (0.026)</td><td align=\\\"left\\\">8.24\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">0.000 (0.003)</td><td align=\\\"left\\\">8.79\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">9.42\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20038</td><td align=\\\"left\\\" rowspan=\\\"2\\\">102060792</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs56162098</td><td align=\\\"left\\\" rowspan=\\\"2\\\">C/T (0.138)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><italic>NCALD</italic></td><td align=\\\"left\\\">Language</td><td align=\\\"left\\\">0.026 (0.010)</td><td align=\\\"left\\\">7.77\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">0.002 (0.001)</td><td align=\\\"left\\\">1.39\\u2009\\u00d7\\u200910<sup>\\u20131</sup></td><td align=\\\"left\\\">1.46\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">4.69\\u2009\\u00d7\\u200910<sup>\\u20137</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">7.64\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\"><bold>1.23\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20139</bold></sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">-0.026 (0.011)</td><td align=\\\"left\\\">1.65\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">-0.003 (0.001)</td><td align=\\\"left\\\">1.01\\u2009\\u00d7\\u200910<sup>\\u20132</sup></td><td align=\\\"left\\\">9.76\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20039</td><td align=\\\"left\\\" rowspan=\\\"2\\\">10155013</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs145989094</td><td align=\\\"left\\\" rowspan=\\\"2\\\">A/T (0.013)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><italic>PTPRD</italic></td><td align=\\\"left\\\">Exec Function</td><td align=\\\"left\\\">-0.105 (0.032)</td><td align=\\\"left\\\">1.25\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">-0.011 (0.003)</td><td align=\\\"left\\\">3.18\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">9.49\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">1.09\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">1.19\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\"><bold>3.85\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20138</bold></sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">-0.131 (0.034)</td><td align=\\\"left\\\">1.62\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">-0.015 (0.003)</td><td align=\\\"left\\\">2.68\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">1.02\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u20039</td><td align=\\\"left\\\" rowspan=\\\"2\\\">10155013</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs145989094</td><td align=\\\"left\\\" rowspan=\\\"2\\\">A/T (0.013)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><italic>PTPRD</italic></td><td align=\\\"left\\\">Language</td><td align=\\\"left\\\">-0.154 (0.031)</td><td align=\\\"left\\\">8.58\\u2009\\u00d7\\u200910<sup>\\u20137</sup></td><td align=\\\"left\\\">-0.010 (0.003)</td><td align=\\\"left\\\">4.26\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">1.15\\u2009\\u00d7\\u200910<sup>\\u20136</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">1.56\\u2009\\u00d7\\u200910<sup>\\u20136</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">1.71\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\"><bold>8.34\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20139</bold></sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">-0.131 (0.034)</td><td align=\\\"left\\\">1.62\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">-0.015 (0.003)</td><td align=\\\"left\\\">2.68\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">1.02\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\" rowspan=\\\"2\\\">\\u200316</td><td align=\\\"left\\\" rowspan=\\\"2\\\">83947805</td><td align=\\\"left\\\" rowspan=\\\"2\\\">rs12447050</td><td align=\\\"left\\\" rowspan=\\\"2\\\">C/T (0.391)</td><td align=\\\"left\\\" rowspan=\\\"2\\\"><p><italic>OSGIN1</italic></p><p><italic>MLYCD</italic></p></td><td align=\\\"left\\\">Exec Function</td><td align=\\\"left\\\">0.024 (0.007)</td><td align=\\\"left\\\">1.02\\u2009\\u00d7\\u200910<sup>\\u20133</sup></td><td align=\\\"left\\\">0.003 (0.001)</td><td align=\\\"left\\\">8.06\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\">1.16\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">9.49\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\">1.46\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td><td align=\\\"left\\\" rowspan=\\\"2\\\"><bold>4.09\\u2009\\u00d7\\u200910</bold><sup><bold>\\u20138</bold></sup></td></tr><tr><td align=\\\"left\\\">Memory</td><td align=\\\"left\\\">0.029 (0.008)</td><td align=\\\"left\\\">1.71\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">0.003 (0.001)</td><td align=\\\"left\\\">2.00\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td><td align=\\\"left\\\">3.00\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr></tbody></table>\\n\"}, \"text\": \"Chr\\tPosition\\tVariant\\tA1/A2a (MAF)\\tNearest Gene\\tCognitive Domain\\tIndividual locus\\tPleiotropy\\t \\t\\u03b2G (SE)\\tPG\\t\\u03b2G\\u00d7Age (SE)\\tPG\\u00d7Age\\tPJoint\\tPPlaco, G\\tPPlaco, G\\u00d7Age\\tPPlaco, Joint\\t \\tTotal sample\\t \\t\\u20032\\t127135234\\trs6733839\\tT/C (0.403)\\tBIN1CYP27C1\\tLanguage\\t-0.031 (0.006)\\t6.90\\u2009\\u00d7\\u200910\\u20138\\t-0.002 (0.001)\\t3.62\\u2009\\u00d7\\u200910\\u20133\\t2.70\\u2009\\u00d7\\u200910\\u20138\\t2.72\\u2009\\u00d7\\u200910\\u201310\\t9.11\\u2009\\u00d7\\u200910\\u20134\\t9.01\\u2009\\u00d7\\u200910\\u201312\\t \\tMemory\\t-0.039 (0.006)\\t2.20\\u2009\\u00d7\\u200910\\u20139\\t-0.002 (0.001)\\t1.62\\u2009\\u00d7\\u200910\\u20133\\t2.37\\u2009\\u00d7\\u200910\\u20139\\t \\t\\u20034\\t179150192\\trs73005629\\tC/T (0.022)\\tLOC107984373\\tLanguage\\t-0.048 (0.019)\\t1.21\\u2009\\u00d7\\u200910\\u20132\\t-0.007 (0.002)\\t2.06\\u2009\\u00d7\\u200910\\u20134\\t1.19\\u2009\\u00d7\\u200910\\u20134\\t3.69\\u2009\\u00d7\\u200910\\u20133\\t9.32\\u2009\\u00d7\\u200910\\u20134\\t6.16\\u2009\\u00d7\\u200910\\u20135\\t \\tMemory\\t-0.062 (0.021)\\t3.97\\u2009\\u00d7\\u200910\\u20133\\t-0.006 (0.002)\\t8.12\\u2009\\u00d7\\u200910\\u20133\\t2.11\\u2009\\u00d7\\u200910\\u20133\\t \\t\\u20038\\t102060792\\trs56162098\\tC/T (0.138)\\tNCALD\\tLanguage\\t0.019 (0.008)\\t1.64\\u2009\\u00d7\\u200910\\u20132\\t0.001 (0.001)\\t1.84\\u2009\\u00d7\\u200910\\u20131\\t3.08\\u2009\\u00d7\\u200910\\u20132\\t5.85\\u2009\\u00d7\\u200910\\u20135\\t5.74\\u2009\\u00d7\\u200910\\u20133\\t6.86\\u2009\\u00d7\\u200910\\u20136\\t \\tMemory\\t-0.013 (0.009)\\t1.68\\u2009\\u00d7\\u200910\\u20131\\t-0.001 (0.001)\\t1.71\\u2009\\u00d7\\u200910\\u20131\\t2.17\\u2009\\u00d7\\u200910\\u20131\\t \\t\\u20039\\t10155013\\trs145989094\\tA/T (0.013)\\tPTPRD\\tExec Function\\t-0.056 (0.026)\\t3.22\\u2009\\u00d7\\u200910\\u20132\\t-0.006 (0.002)\\t1.26\\u2009\\u00d7\\u200910\\u20132\\t1.04\\u2009\\u00d7\\u200910\\u20132\\t2.70\\u2009\\u00d7\\u200910\\u20132\\t2.79\\u2009\\u00d7\\u200910\\u20133\\t9.93\\u2009\\u00d7\\u200910\\u20134\\t \\tMemory\\t-0.058 (0.028)\\t4.56\\u2009\\u00d7\\u200910\\u20132\\t-0.008 (0.003)\\t3.14\\u2009\\u00d7\\u200910\\u20133\\t5.92\\u2009\\u00d7\\u200910\\u20133\\t \\t\\u20039\\t10155013\\trs145989094\\tA/T (0.013)\\tPTPRD\\tLanguage\\t-0.093 (0.025)\\t3.40\\u2009\\u00d7\\u200910\\u20134\\t-0.004 (0.003)\\t9.29\\u2009\\u00d7\\u200910\\u20132\\t8.84\\u2009\\u00d7\\u200910\\u20134\\t1.22\\u2009\\u00d7\\u200910\\u20132\\t3.47\\u2009\\u00d7\\u200910\\u20132\\t4.65\\u2009\\u00d7\\u200910\\u20133\\t \\tMemory\\t-0.058 (0.028)\\t4.56\\u2009\\u00d7\\u200910\\u20132\\t-0.008 (0.003)\\t3.14\\u2009\\u00d7\\u200910\\u20133\\t5.92\\u2009\\u00d7\\u200910\\u20133\\t \\t\\u200316\\t83947805\\trs12447050\\tC/T (0.400)\\tOSGIN1MLYCD\\tExec Function\\t0.016 (0.006)\\t4.70\\u2009\\u00d7\\u200910\\u20133\\t0.002 (0.001)\\t4.01\\u2009\\u00d7\\u200910\\u20134\\t1.03\\u2009\\u00d7\\u200910\\u20134\\t2.03\\u2009\\u00d7\\u200910\\u20133\\t9.76\\u2009\\u00d7\\u200910\\u20134\\t3.72\\u2009\\u00d7\\u200910\\u20135\\t \\tMemory\\t0.018 (0.006)\\t4.92\\u2009\\u00d7\\u200910\\u20133\\t0.002 (0.001)\\t9.28\\u2009\\u00d7\\u200910\\u20133\\t2.61\\u2009\\u00d7\\u200910\\u20133\\t \\tClinic-based cohorts\\t \\t\\u20032\\t127135234\\trs6733839\\tT/C (0.409)\\tBIN1CYP27C1\\tLanguage\\t-0.063 (0.010)\\t4.23\\u2009\\u00d7\\u200910\\u201310\\t-0.001 (0.001)\\t2.09\\u2009\\u00d7\\u200910\\u20131\\t1.98\\u2009\\u00d7\\u200910\\u20139\\t1.78\\u2009\\u00d7\\u200910\\u201310\\t2.20\\u2009\\u00d7\\u200910\\u20131\\t6.85\\u2009\\u00d7\\u200910\\u201310\\t \\tMemory\\t-0.068 (0.012)\\t5.23\\u2009\\u00d7\\u200910\\u20139\\t-0.001 (0.001)\\t2.08\\u2009\\u00d7\\u200910\\u20131\\t1.60\\u2009\\u00d7\\u200910\\u20138\\t \\t\\u20034\\t179150192\\trs73005629\\tC/T (0.022)\\tLOC107984373\\tLanguage\\t-0.112 (0.033)\\t7.70\\u2009\\u00d7\\u200910\\u20134\\t-0.014 (0.003)\\t1.68\\u2009\\u00d7\\u200910\\u20135\\t4.66\\u2009\\u00d7\\u200910\\u20137\\t2.06\\u2009\\u00d7\\u200910\\u20134\\t5.11\\u2009\\u00d7\\u200910\\u20136\\t3.12\\u2009\\u00d7\\u200910\\u20138\\t \\tMemory\\t-0.149 (0.038)\\t1.13\\u2009\\u00d7\\u200910\\u20134\\t-0.015 (0.004)\\t4.04\\u2009\\u00d7\\u200910\\u20135\\t1.47\\u2009\\u00d7\\u200910\\u20137\\t \\t\\u20038\\t102060792\\trs56162098\\tC/T (0.138)\\tNCALD\\tLanguage\\t0.006 (0.014)\\t6.89\\u2009\\u00d7\\u200910\\u20131\\t0.000 (0.001)\\t7.49\\u2009\\u00d7\\u200910\\u20131\\t8.81\\u2009\\u00d7\\u200910\\u20131\\t7.74\\u2009\\u00d7\\u200910\\u20131\\t9.29\\u2009\\u00d7\\u200910\\u20131\\t9.56\\u2009\\u00d7\\u200910\\u20131\\t \\tMemory\\t0.009 (0.017)\\t5.73\\u2009\\u00d7\\u200910\\u20131\\t0.001 (0.002)\\t3.54\\u2009\\u00d7\\u200910\\u20131\\t5.62\\u2009\\u00d7\\u200910\\u20131\\t \\t\\u20039\\t10155013\\trs145989094\\tA/T (0.013)\\tPTPRD\\tExec Function\\t0.016 (0.044)\\t7.25\\u2009\\u00d7\\u200910\\u20131\\t0.002 (0.004)\\t7.27\\u2009\\u00d7\\u200910\\u20131\\t8.85\\u2009\\u00d7\\u200910\\u20131\\t4.38\\u2009\\u00d7\\u200910\\u20131\\t8.92\\u2009\\u00d7\\u200910\\u20131\\t7.33\\u2009\\u00d7\\u200910\\u20131\\t \\tMemory\\t0.081 (0.053)\\t1.28\\u2009\\u00d7\\u200910\\u20131\\t0.000 (0.005)\\t9.83\\u2009\\u00d7\\u200910\\u20131\\t3.08\\u2009\\u00d7\\u200910\\u20131\\t \\t\\u20039\\t10155013\\trs145989094\\tA/T (0.013)\\tPTPRD\\tLanguage\\t0.026 (0.046)\\t5.78\\u2009\\u00d7\\u200910\\u20131\\t0.002 (0.004)\\t6.27\\u2009\\u00d7\\u200910\\u20131\\t7.68\\u2009\\u00d7\\u200910\\u20131\\t5.27\\u2009\\u00d7\\u200910\\u20131\\t7.63\\u2009\\u00d7\\u200910\\u20131\\t7.82\\u2009\\u00d7\\u200910\\u20131\\t \\tMemory\\t0.081 (0.053)\\t1.28\\u2009\\u00d7\\u200910\\u20131\\t0.000 (0.005)\\t9.83\\u2009\\u00d7\\u200910\\u20131\\t3.08\\u2009\\u00d7\\u200910\\u20131\\t \\t\\u200316\\t83947805\\trs12447050\\tC/T (0.404)\\tOSGIN1MLYCD\\tExec Function\\t0.005 (0.010)\\t6.31\\u2009\\u00d7\\u200910\\u20131\\t0.001 (0.001)\\t4.59\\u2009\\u00d7\\u200910\\u20131\\t6.83\\u2009\\u00d7\\u200910\\u20131\\t8.02\\u2009\\u00d7\\u200910\\u20131\\t3.63\\u2009\\u00d7\\u200910\\u20131\\t6.41\\u2009\\u00d7\\u200910\\u20131\\t \\tMemory\\t0.001 (0.012)\\t9.56\\u2009\\u00d7\\u200910\\u20131\\t0.000 (0.001)\\t7.65\\u2009\\u00d7\\u200910\\u20131\\t9.54\\u2009\\u00d7\\u200910\\u20131\\t \\tCommunity-based cohorts\\t \\t\\u20032\\t127135234\\trs6733839\\tT/C (0.388)\\tBIN1CYP27C1\\tLanguage\\t-0.017 (0.007)\\t1.68\\u2009\\u00d7\\u200910\\u20132\\t-0.002 (0.001)\\t1.70\\u2009\\u00d7\\u200910\\u20132\\t6.57\\u2009\\u00d7\\u200910\\u20133\\t1.99\\u2009\\u00d7\\u200910\\u20133\\t8.10\\u2009\\u00d7\\u200910\\u20133\\t2.52\\u2009\\u00d7\\u200910\\u20134\\t \\tMemory\\t-0.027 (0.008)\\t5.71\\u2009\\u00d7\\u200910\\u20134\\t-0.002 (0.001)\\t1.69\\u2009\\u00d7\\u200910\\u20132\\t1.11\\u2009\\u00d7\\u200910\\u20133\\t \\t\\u20034\\t179150192\\trs73005629\\tC/T (0.021)\\tLOC107984373\\tLanguage\\t-0.011 (0.023)\\t6.32\\u2009\\u00d7\\u200910\\u20131\\t-0.003 (0.002)\\t2.25\\u2009\\u00d7\\u200910\\u20131\\t4.48\\u2009\\u00d7\\u200910\\u20131\\t8.73\\u2009\\u00d7\\u200910\\u20131\\t2.86\\u2009\\u00d7\\u200910\\u20131\\t5.58\\u2009\\u00d7\\u200910\\u20131\\t \\tMemory\\t-0.006 (0.026)\\t8.24\\u2009\\u00d7\\u200910\\u20131\\t0.000 (0.003)\\t8.79\\u2009\\u00d7\\u200910\\u20131\\t9.42\\u2009\\u00d7\\u200910\\u20131\\t \\t\\u20038\\t102060792\\trs56162098\\tC/T (0.138)\\tNCALD\\tLanguage\\t0.026 (0.010)\\t7.77\\u2009\\u00d7\\u200910\\u20133\\t0.002 (0.001)\\t1.39\\u2009\\u00d7\\u200910\\u20131\\t1.46\\u2009\\u00d7\\u200910\\u20132\\t4.69\\u2009\\u00d7\\u200910\\u20137\\t7.64\\u2009\\u00d7\\u200910\\u20135\\t1.23\\u2009\\u00d7\\u200910\\u20139\\t \\tMemory\\t-0.026 (0.011)\\t1.65\\u2009\\u00d7\\u200910\\u20132\\t-0.003 (0.001)\\t1.01\\u2009\\u00d7\\u200910\\u20132\\t9.76\\u2009\\u00d7\\u200910\\u20133\\t \\t\\u20039\\t10155013\\trs145989094\\tA/T (0.013)\\tPTPRD\\tExec Function\\t-0.105 (0.032)\\t1.25\\u2009\\u00d7\\u200910\\u20133\\t-0.011 (0.003)\\t3.18\\u2009\\u00d7\\u200910\\u20134\\t9.49\\u2009\\u00d7\\u200910\\u20135\\t1.09\\u2009\\u00d7\\u200910\\u20134\\t1.19\\u2009\\u00d7\\u200910\\u20135\\t3.85\\u2009\\u00d7\\u200910\\u20138\\t \\tMemory\\t-0.131 (0.034)\\t1.62\\u2009\\u00d7\\u200910\\u20134\\t-0.015 (0.003)\\t2.68\\u2009\\u00d7\\u200910\\u20135\\t1.02\\u2009\\u00d7\\u200910\\u20135\\t \\t\\u20039\\t10155013\\trs145989094\\tA/T (0.013)\\tPTPRD\\tLanguage\\t-0.154 (0.031)\\t8.58\\u2009\\u00d7\\u200910\\u20137\\t-0.010 (0.003)\\t4.26\\u2009\\u00d7\\u200910\\u20133\\t1.15\\u2009\\u00d7\\u200910\\u20136\\t1.56\\u2009\\u00d7\\u200910\\u20136\\t1.71\\u2009\\u00d7\\u200910\\u20134\\t8.34\\u2009\\u00d7\\u200910\\u20139\\t \\tMemory\\t-0.131 (0.034)\\t1.62\\u2009\\u00d7\\u200910\\u20134\\t-0.015 (0.003)\\t2.68\\u2009\\u00d7\\u200910\\u20135\\t1.02\\u2009\\u00d7\\u200910\\u20135\\t \\t\\u200316\\t83947805\\trs12447050\\tC/T (0.391)\\tOSGIN1MLYCD\\tExec Function\\t0.024 (0.007)\\t1.02\\u2009\\u00d7\\u200910\\u20133\\t0.003 (0.001)\\t8.06\\u2009\\u00d7\\u200910\\u20135\\t1.16\\u2009\\u00d7\\u200910\\u20135\\t9.49\\u2009\\u00d7\\u200910\\u20135\\t1.46\\u2009\\u00d7\\u200910\\u20135\\t4.09\\u2009\\u00d7\\u200910\\u20138\\t \\tMemory\\t0.029 (0.008)\\t1.71\\u2009\\u00d7\\u200910\\u20134\\t0.003 (0.001)\\t2.00\\u2009\\u00d7\\u200910\\u20134\\t3.00\\u2009\\u00d7\\u200910\\u20135\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 40373, \"infons\": {\"file\": \"Tab3.xml\", \"id\": \"Tab3\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"aA1 Minor allele, A2 Reference allele, MAF Minor allele frequency\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 40439, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"GWAS for the three pairs of cognitive domains collectively identified GWS evidence of association with SNPs in five independent loci (Table 3, Table S4) with little evidence of genomic inflation in the total sample or separately within the clinic-based and community-based cohorts (\\u03bb\\u2009=\\u20090.964\\u20130.994, Figs. S8, S9 and S10). Consistent with the findings from analyses of individual cognitive domains, GWS evidence of pleiotropy was found for the association of the APOE region SNPs with all cognitive domain pairs in the total sample and the clinic-based and community-based cohorts (Table S4). GWS pleiotropy was also observed with rs6733839, located between\\u00a0BIN1\\u00a0and\\u00a0CYP27C1, in the total sample (PJoint\\u2009=\\u20099.01\\u2009\\u00d7\\u200910\\u201312) and the clinic-based cohorts (PJoint\\u2009=\\u20096.85\\u2009\\u00d7\\u200910\\u201310) for language and memory (Table 3). The association with rs6733839 was evident in the community-based cohorts (PJoint\\u2009=\\u20092.52\\u2009\\u00d7\\u200910\\u20134), strengthened in the total sample (PJoint\\u2009=\\u20099.01\\u2009\\u00d7\\u200910\\u201312), well supported by association with neighboring variants (Fig. S11), and attributable primarily to its main effect for each domain (Table 3).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 41597, \"infons\": {\"file\": \"13024_2023_633_Fig2_HTML.jpg\", \"id\": \"Fig2\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Locus Zoom plots showing genome-wide significant pleiotropy for SNPs in the regions of novel loci. The SNP with the lowest p-value at each locus is indicated with a purple diamond. Computed estimates of linkage disequilibrium (r2) of SNPs in the region with top-ranked SNP are color-coded according to the key. Vertical blue lines indicate locations of high recombination rates. Locations of genes in the region are shown below the diagram. a Association of rs12447050 with executive function and memory in the community-based cohorts. b Association of rs56162098 with language and memory in the community-based cohorts. c Association of rs145989094 with executive function and memory in the community-based cohorts. d Association of rs145989094 with language and memory in the community-based cohorts. e Association of rs73005629 with language and memory in the clinic-based cohorts\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 42481, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"In the clinic-based cohorts, there was GWS pleiotropy for language and memory with rs73005629 (PJoint\\u2009=\\u20093.12\\u2009\\u00d7\\u200910\\u20138) located in an intergenic region on chromosome 4 (Table 3, Fig.\\u00a02). The joint effect of rs73005629 on language (PJoint\\u2009=\\u20094.66\\u2009\\u00d7\\u200910\\u20137) and memory (PJoint\\u2009=\\u20091.47\\u2009\\u00d7\\u200910\\u20137) was equally attributable to its main and interaction effects. There was no evidence of pleiotropy for rs73005629 in the community-based cohorts. Conversely, significant pleiotropy for the same domain pair was observed with\\u00a0NCALD SNP rs56162098 (PJoint\\u2009=\\u20091.23\\u2009\\u00d7\\u200910\\u20139) and\\u00a0PTPRD\\u00a0SNP rs145989094 (PJoint\\u2009=\\u20098.34\\u2009\\u00d7\\u200910\\u20139) in the community-based cohorts (Table 3). The association with rs56162098 was not evident in the clinic-based cohorts but was supported by the association with neighboring variants (Fig.\\u00a02). The same PTPRD SNP was also pleiotropic for executive function and memory (PJoint\\u2009=\\u20093.85\\u2009\\u00d7\\u200910\\u20138), but this association is not supported by findings in the clinic-based cohorts (Table 3) or neighboring SNPs (Fig.\\u00a02).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43560, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We also identified significant pleiotropy in the community-based cohorts for executive function and memory with rs12447050, located 5.5\\u00a0kb upstream from\\u00a0OSGIN1\\u00a0(PJoint\\u2009=\\u20094.09\\u2009\\u00d7\\u200910\\u20138). This association was comparably supported by each domain and the SNP\\u2019s main effect and interaction with age (Table 3), as well as by neighboring SNPs (Fig.\\u00a02). There was no evidence of association with the individual domains or in the pleiotropy model in the clinic-based cohorts. However, the magnitude of effect for rs12447050 and its interaction with age in each domain, as well as the significance levels for the main, interaction, and joint pleiotropy tests in the community-based cohorts and the total sample, were nearly identical (Table 3).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44310, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Highly suggestive pleiotropy was observed in the community-based cohorts with two SNPs (rs7081658 and rs7070729) located in the\\u00a0USP6NL/ECHDC3\\u00a0region, an established AD risk locus, for executive function and language (PJoint\\u2009=\\u20093.54\\u2009\\u00d7\\u200910\\u20137 and PJoint\\u2009=\\u20098.76\\u2009\\u00d7\\u200910\\u20138, respectively) and for executive function and memory (PJoint\\u2009=\\u20092.11\\u2009\\u00d7\\u200910\\u20137 and PJoint\\u2009=\\u20095.09\\u2009\\u00d7\\u200910\\u20138, respectively); rs7070729 was also pleiotropic for language and memory (PJoint\\u2009=\\u20097.76\\u2009\\u00d7\\u200910\\u20137) (Table S4). There was also suggestive pleiotropy for executive function and language with two SNPs in the AD risk locus\\u00a0WWOX\\u00a0(rs13329990, PJoint\\u2009=\\u20098.45\\u2009\\u00d7\\u200910\\u20137; rs11862902, PJoint\\u2009=\\u20099.60\\u2009\\u00d7\\u200910\\u20137).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45043, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Pathways involved in neuronal development or signaling, vascular and endocrine systems are related to cognitive domain performance\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45174, \"infons\": {\"file\": \"Tab4.xml\", \"id\": \"Tab4\", \"section_type\": \"TABLE\", \"type\": \"table_caption\"}, \"text\": \"Canonical pathways significantly enriched for top-ranked GWAS genes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45242, \"infons\": {\"file\": \"Tab4.xml\", \"id\": \"Tab4\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"hsides\\\" rules=\\\"groups\\\"><thead><tr><th align=\\\"left\\\"><bold>Pathway</bold></th><th align=\\\"left\\\"><bold>Sample</bold></th><th align=\\\"left\\\"><bold>GWAS Outcome</bold></th><th align=\\\"left\\\"><bold>Number of genes</bold></th><th align=\\\"left\\\"><bold><italic>P</italic></bold><sub><bold>FDR</bold></sub><sup><bold>1</bold></sup></th></tr></thead><tbody><tr><td align=\\\"left\\\" rowspan=\\\"3\\\">Synaptogenesis signaling</td><td align=\\\"left\\\" rowspan=\\\"3\\\">Community</td><td align=\\\"left\\\">L / M</td><td align=\\\"left\\\">334</td><td align=\\\"left\\\">4.07\\u2009\\u00d7\\u200910<sup>\\u20136</sup></td></tr><tr><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">1.62\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">EF / M</td><td align=\\\"left\\\">286</td><td align=\\\"left\\\">6.61\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Opioid signaling</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">8.13\\u2009\\u00d7\\u200910<sup>\\u20136</sup></td></tr><tr><td align=\\\"left\\\">Amyotrophic lateral sclerosis signaling</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">211</td><td align=\\\"left\\\">2.14\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Maturity onset diabetes of young signaling</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">211</td><td align=\\\"left\\\">2.14\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Corticotropin releasing hormone signaling</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">3.39\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Dilated cardiomyopathy signaling</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">3.39\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Netrin signaling</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">3.39\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Neuronal nitric oxide synthase signaling in skeletal muscle cells</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">3.39\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Circadian rhythm signaling</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">5.62\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">Insulin secretion signaling</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">5.62\\u2009\\u00d7\\u200910<sup>\\u20135</sup></td></tr><tr><td align=\\\"left\\\">White adipose tissue browning</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">1.05\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Synaptic long-term depression</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">1.62\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Nitric oxide signaling in the cardiovascular system</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">211</td><td align=\\\"left\\\">1.74\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">GPCR-mediated nutrient sensing in enteroendocrine cells</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">2.63\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Protein kinase A signaling</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">2.63\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Hepatic fibrosis signaling</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">3.24\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">GABA receptor signaling</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">3.98\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Endocannabinoid neuronal synapse</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">211</td><td align=\\\"left\\\">4.68\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Type II diabetes mellitus signaling</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">211</td><td align=\\\"left\\\">4.68\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">GNRH signaling</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">5.25\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Cardiac hypertrophy signaling (enhanced)</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">6.92\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Gustation pathway</td><td align=\\\"left\\\">Community</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">191</td><td align=\\\"left\\\">6.92\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Androgen signaling</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">211</td><td align=\\\"left\\\">7.94\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Calcium signaling</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">211</td><td align=\\\"left\\\">7.94\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Oxytocin signaling</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">211</td><td align=\\\"left\\\">7.94\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">G beta gamma signaling</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">EF / L</td><td align=\\\"left\\\">211</td><td align=\\\"left\\\">8.32\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">AMPK signaling</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">EF / M</td><td align=\\\"left\\\">210</td><td align=\\\"left\\\">9.55\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr><tr><td align=\\\"left\\\">Glutamate receptor signaling</td><td align=\\\"left\\\">All</td><td align=\\\"left\\\">EF / M</td><td align=\\\"left\\\">210</td><td align=\\\"left\\\">9.55\\u2009\\u00d7\\u200910<sup>\\u20134</sup></td></tr></tbody></table>\\n\"}, \"text\": \"Pathway\\tSample\\tGWAS Outcome\\tNumber of genes\\tPFDR1\\t \\tSynaptogenesis signaling\\tCommunity\\tL / M\\t334\\t4.07\\u2009\\u00d7\\u200910\\u20136\\t \\tEF / L\\t191\\t1.62\\u2009\\u00d7\\u200910\\u20134\\t \\tEF / M\\t286\\t6.61\\u2009\\u00d7\\u200910\\u20134\\t \\tOpioid signaling\\tCommunity\\tEF / L\\t191\\t8.13\\u2009\\u00d7\\u200910\\u20136\\t \\tAmyotrophic lateral sclerosis signaling\\tAll\\tEF / L\\t211\\t2.14\\u2009\\u00d7\\u200910\\u20135\\t \\tMaturity onset diabetes of young signaling\\tAll\\tEF / L\\t211\\t2.14\\u2009\\u00d7\\u200910\\u20135\\t \\tCorticotropin releasing hormone signaling\\tCommunity\\tEF / L\\t191\\t3.39\\u2009\\u00d7\\u200910\\u20135\\t \\tDilated cardiomyopathy signaling\\tCommunity\\tEF / L\\t191\\t3.39\\u2009\\u00d7\\u200910\\u20135\\t \\tNetrin signaling\\tCommunity\\tEF / L\\t191\\t3.39\\u2009\\u00d7\\u200910\\u20135\\t \\tNeuronal nitric oxide synthase signaling in skeletal muscle cells\\tCommunity\\tEF / L\\t191\\t3.39\\u2009\\u00d7\\u200910\\u20135\\t \\tCircadian rhythm signaling\\tCommunity\\tEF / L\\t191\\t5.62\\u2009\\u00d7\\u200910\\u20135\\t \\tInsulin secretion signaling\\tCommunity\\tEF / L\\t191\\t5.62\\u2009\\u00d7\\u200910\\u20135\\t \\tWhite adipose tissue browning\\tCommunity\\tEF / L\\t191\\t1.05\\u2009\\u00d7\\u200910\\u20134\\t \\tSynaptic long-term depression\\tCommunity\\tEF / L\\t191\\t1.62\\u2009\\u00d7\\u200910\\u20134\\t \\tNitric oxide signaling in the cardiovascular system\\tAll\\tEF / L\\t211\\t1.74\\u2009\\u00d7\\u200910\\u20134\\t \\tGPCR-mediated nutrient sensing in enteroendocrine cells\\tCommunity\\tEF / L\\t191\\t2.63\\u2009\\u00d7\\u200910\\u20134\\t \\tProtein kinase A signaling\\tCommunity\\tEF / L\\t191\\t2.63\\u2009\\u00d7\\u200910\\u20134\\t \\tHepatic fibrosis signaling\\tCommunity\\tEF / L\\t191\\t3.24\\u2009\\u00d7\\u200910\\u20134\\t \\tGABA receptor signaling\\tCommunity\\tEF / L\\t191\\t3.98\\u2009\\u00d7\\u200910\\u20134\\t \\tEndocannabinoid neuronal synapse\\tAll\\tEF / L\\t211\\t4.68\\u2009\\u00d7\\u200910\\u20134\\t \\tType II diabetes mellitus signaling\\tAll\\tEF / L\\t211\\t4.68\\u2009\\u00d7\\u200910\\u20134\\t \\tGNRH signaling\\tCommunity\\tEF / L\\t191\\t5.25\\u2009\\u00d7\\u200910\\u20134\\t \\tCardiac hypertrophy signaling (enhanced)\\tCommunity\\tEF / L\\t191\\t6.92\\u2009\\u00d7\\u200910\\u20134\\t \\tGustation pathway\\tCommunity\\tEF / L\\t191\\t6.92\\u2009\\u00d7\\u200910\\u20134\\t \\tAndrogen signaling\\tAll\\tEF / L\\t211\\t7.94\\u2009\\u00d7\\u200910\\u20134\\t \\tCalcium signaling\\tAll\\tEF / L\\t211\\t7.94\\u2009\\u00d7\\u200910\\u20134\\t \\tOxytocin signaling\\tAll\\tEF / L\\t211\\t7.94\\u2009\\u00d7\\u200910\\u20134\\t \\tG beta gamma signaling\\tAll\\tEF / L\\t211\\t8.32\\u2009\\u00d7\\u200910\\u20134\\t \\tAMPK signaling\\tAll\\tEF / M\\t210\\t9.55\\u2009\\u00d7\\u200910\\u20134\\t \\tGlutamate receptor signaling\\tAll\\tEF / M\\t210\\t9.55\\u2009\\u00d7\\u200910\\u20134\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 47318, \"infons\": {\"file\": \"Tab4.xml\", \"id\": \"Tab4\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"All Total sample, Community Community-based cohorts, EF Executive function, L Language, M Memory\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 47415, \"infons\": {\"file\": \"Tab4.xml\", \"id\": \"Tab4\", \"section_type\": \"TABLE\", \"type\": \"table_foot\"}, \"text\": \"1P-values were adjusted for a false discovery rate and 18 separate pathway analyses (threshold P\\u2009<\\u20090.001)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 47525, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"A total of 28 canonical pathways were significantly enriched for loci associated with pleiotropy for paired domains (Table 4), noting that none of these pathways were specific to the clinic-based cohorts, and no significant pathways were identified in analyses seeded with top-ranked genes in the GWAS for individual cognitive domains. The evidence for approximately 60% (17/28) of these pathways was derived from analyses of the community-based cohorts only. The top-ranked pathway, synaptogenesis signaling, was significantly enriched for genes that emerged from pleiotropy analysis for all three pairs of cognitive domains. Several pathways are related to neuronal development or signaling (e.g., synaptogenesis signaling, synaptic long-term depression, endocannabinoid neuronal synapse, netrin signaling, GABA and glutamate receptor signaling, and calcium signaling), AD-associated vascular risk factors (e.g., type II diabetes and maturity onset diabetes of young signaling, insulin secretion signaling, dilated cardiomyopathy and cardiac hypertrophy signaling, and nitric oxide signaling in the cardiovascular system), and the endocrine system (e.g., G protein-coupled receptor-mediated nutrient sensing in enteroendocrine cells, insulin, corticotropin-releasing hormone, gonadotropin-releasing hormone, androgen, and oxytocin signaling). Details for suggestive pathways (FDR-adjusted\\u00a0P\\u2009<\\u20090.05) and the number of seed genes selected from each GWAS and pleiotropy analysis are summarized in\\u00a0Tables S5 and S6, respectively.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49059, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_1\"}, \"text\": \"Discussion\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49070, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Genome-wide scans for performance measures in three cognitive domains in two large clinically ascertained and three community-based cohorts revealed GWS associations with four well-established AD loci (BIN1,\\u00a0CR1, MS4A6A, and\\u00a0APOE) and eight loci not previously genetically linked to AD or cognitive decline (ULK2,\\u00a0CDK14, PURG, LINC02712, LOC107984373, NCALD, PTPRD, and\\u00a0OSGIN1), as well as with GRN which has been associated with AD and several other dementing illnesses. These findings were based on analyses that leveraged data obtained from one or more cognitive examinations, considered cognitive performance changes over time, and examined genetic effects on individual or pairs of domains. In comparison to previous GWAS of cognitive performance, which were limited to the availability of data for particular NP tests and focused primarily on clinic-based or community-based samples, our study utilized harmonized measures that enabled pooling data obtained using multiple NP protocols and considered associations that may be common or unique to differentially ascertained samples.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 50162, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"To our knowledge, this is the first genome-wide pleiotropy study using harmonized cognitive domain scores. Compared to previous conventional GWAS or pleiotropy studies of individual cognitive traits, our genetic analysis of harmonized cognitive scores allows combining results from studies using different NP protocols and permits greater opportunities for replication and meta-analyses. This approach has been successfully used in a variety of studies of cognitive aging. A recent study of five preclinical AD cohorts conducted a factor analysis on three domains\\u2014general cognitive performance, episodic memory, and executive function\\u2014and established a common algorithm for classifying MCI progression across the heterogeneously evaluated samples. Cognitive factor scores derived in an identical fashion as those used in this study have been utilized for a variety of investigations of AD subgroups, which linked cognition to imaging, neuropathology, and genetics. Similarly, they have been used in genetic studies of cognitive resilience to AD.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51212, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"We identified three novel loci that have functional relevance to processes implicated in AD. Dysfunction of the protein encoded by ULK2, unc51 like autophagy activating kinase 2, has been suggested to cause multiple diseases.\\u00a0ULK2\\u00a0SNPs have been associated with schizophrenia, and a ULK2 circular RNA is expressed more than tenfold in a vascular dementia rat model. Lee and colleagues recently demonstrated that amyloid-\\u03b2 42 oligomer-mediated loss of excitatory synapses in cortical neurons and hippocampal CA1 neurons requires AMPK-mediated activation of ULK2-dependent mitophagy. PTPRD, protein tyrosine phosphatase receptor type D, was previously reported to be associated with AD susceptibility. A recent study identified a significant association of PTPRD with the accumulation of neurofibrillary tangles that was independent of amyloid-\\u03b2 pathology. NCALD encodes a member of the neuronal calcium sensor family of calcium-binding proteins, which mediates signal transduction in response to calcium in neurons. NCALD is downregulated in the AD brain and may play a protective role in hippocampal CA1 and CA3 regions. This observation is consistent with a finding from a study of differentially expressed proteins in rats fed a high-fat diet suggesting that the memory-impairing effects of diet-induced obesity might potentially be mediated by down-regulated NCALD within the hippocampus.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 52610, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"We also identified a GWS signal in\\u00a0GRN, the gene that encodes the anti-inflammatory and neurotrophic factor progranulin (PGRN).\\u00a0GRN\\u00a0mutations are a well-established cause of frontotemporal lobar degeneration (FTLD). More than 60 disease-causing\\u00a0GRN\\u00a0mutations have been identified, accounting for 20% to 25% of familial FTLD cases and about 10% of all FTLD cases. The most significantly associated\\u00a0GRN\\u00a0SNP in our study, rs5848, was found in the 3\\u2019-untranslated region, which is predicted to be a microRNA binding site. Rs5848 is the GRN\\u00a0variant most frequently\\u00a0associated with FTLD and is associated with a reduction in PGRN in plasma and cerebrospinal fluid. In addition to FTLD, several studies have shown an association between clinical AD and the rs5848\\u00a0T\\u00a0allele, which we found to be linked to lower memory performance in both clinic- and community-based cohorts. A recent large GWAS meta-analysis found a GWS association of AD risk with rs5848-T. A recent study examining neuropathological AD correlates showed that rs5848\\u00a0T\\u00a0allele carriers had a higher frequency of hippocampal sclerosis and TDP-43 deposits, significantly increased tau pathology burden, but showed no specific association with \\u03b2-amyloid load or AD neuropathological diagnosis. Interestingly, our finding was exclusive to the memory domain, which is affected in early stages of AD, but also commonly affected in hippocampal sclerosis and limbic-predominant age-related TDP-43 encephalopathy (LATE). Effects were driven by both the SNP\\u2019s main and SNP\\u2009\\u00d7\\u2009age interaction effects. This finding provides additional evidence that variation in\\u00a0GRN\\u00a0may be related to neurodegeneration more broadly and that restoring PGRN levels may be an effective way to prevent and treat dementia.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54384, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Highly suggestive pleiotropy (P\\u2009<\\u20091\\u2009\\u00d7\\u200910\\u20137) was also found with other established AD risk loci, including USP6NL/ECHDC3 for all three paired cognitive domains and WWOX for executive function and language. ECHDC3, enoyl-CoA hydratase domain containing 3, was previously reported to be associated with AD. ADCY2, adenylate cyclase 2, was reported to be associated with AD-related changes in hippocampal gene expression, as well as AD-associated structural changes detected by brain imaging. A recent GWAS reported the association of DAPK2, death associated protein kinase 2, with amyloid deposition in the brain, a finding consistent with studies showing that DAPK1 promotes APP phosphorylation and amyloidogenic processing. PLXDC2, plexin domain containing 2, is upregulated with increasing \\u03b2-amyloid plaque load or Braak stages.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55225, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"There are no established links of OSGIN1, CDK14, and PURG to AD. The product encoded by OSGIN1 is an oxidative stress response protein that regulates cell death and appears to be a key regulator of both inflammatory and anti-inflammatory molecules. CDK14 encodes a protein kinase whose expression is more than two-fold higher in the brain than in any other tissue. However, it has been linked to cancer in various tissues, primarily outside of the brain. Although the function of PURG is unknown, a SNP in this gene showed significant associations in GWAS of cognitive performance and intelligence. The biological significance of the pleiotropic association of memory and language with a chromosome 4 variant located about 170\\u00a0kb from LOC107984373, which encodes a long non-protein coding RNA, is also puzzling at this time.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56051, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Bioinformatic analyses of the top-ranked genes emerging from GWAS implicated several biological pathways related to neuronal development and signaling, AD-associated vascular risk factors, and endocrine pathways. Notably, all of the significant pathways were identified from analyses of findings from the pleiotropy GWAS analyses, especially those supported by the community-based cohorts. Because pathways were constructed using information from well-established metabolic and cell signaling pathways, they tend to reflect more common or shared mechanisms rather than particular or trait-specific mechanisms. Therefore, pleiotropic loci affecting multiple cognitive domains may be more suitable as seed genes for canonical pathways than loci associated with a single domain. Indeed, in the analyses of community-based cohorts, the numbers of seed genes from pleiotropy GWAS were 1.5\\u20131.7 times larger than those from GWAS of individual cognitive domains (Table S6). Considering that AD pathology results in progressive dysfunction in several cognitive domains over time, the majority of our findings, which emerged from analyses of the community-based rather than the clinic-based cohorts, may represent pathways underlying cognitive processes related to AD progression rather than AD risk.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57344, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Interestingly, associations with several well-established AD loci, including BIN1, CR1, and MS4A6A, were observed only in the clinic-based cohorts. Lack of replication in the community-based cohorts might be due to the relative paucity of AD cases and the higher likelihood of mixed pathologies. Conversely, the associations with the known AD locus USP6NL/ECHDC3 and novel loci, including ULK2, NCALD, PTPRD, ADCY2, and OSGIN1, were observed only in the community-based cohorts. Lack of replication in the clinic-based cohorts may indicate that these loci are associated with normal age-related cognitive changes rather than an AD process. However, this explanation seems less likely given their previous association with AD risk (USP6NL/ECHDC3) or functional relevance to processes implicated in AD and/or their association with other AD-related endophenotypes (NCALD, ULK2 and PTPRD). Alternatively, community-based cohort-specific findings may indicate that the effects of these genes are age-dependent or detectable when tracked over time. This idea is supported by the observation of the highly significant SNP\\u2009\\u00d7\\u2009age interaction term for these loci, which for\\u00a0ULK2, PTPRD, ADCY2, and OSGIN1 were responsible for the significant joint effect to a much greater extent than the SNP\\u2019s main effect.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58651, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"We employed a joint test that combines the main genetic effects and SNP\\u2009\\u00d7\\u2009age interaction together to increase our power to detect genetic associations. Nonetheless, interpreting a joint test can be challenging, requiring examination of effect estimates for both the main and interaction terms. The contribution of the SNP\\u2019s main effect to the joint association for some findings, including the well-established AD loci and several novel ones (e.g., ULK2, CDK14, PURG, LINC02712, and GRN), was much stronger than its interaction with age. This may reflect that these loci are associated with the development rather than the progression of AD. This aligns with the fact that these loci were initially identified using a case-control design. In contrast, particularly for the more novel loci, the contribution of the SNP\\u2009\\u00d7\\u2009age interaction to the joint association was stronger than the main effect. This may reflect that these loci are associated with the progression rather than the development of AD and could explain why they have not been identified previously, as few genetic studies of AD have utilized a longitudinal design.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59793, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Of note, all of the GWS pleiotropic associations and half of the GWS single-domain associations involved the memory domain. This is not surprising as prominent memory impairment is the most common cognitive feature in AD. Nonetheless, prominent impairment in other cognitive domains occurs in a reasonable number of AD cases. Those specific loci were implicated for particular cognitive domains may provide syndrome-specific therapeutic targets with an eye toward a precision medicine approach to AD.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60294, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Our results also highlight that the biology underlying cognitive performance in older individuals is complex and likely a function of multiple processes including lifelong ability, neurodegeneration and resilience to neurodegeneration. Genetic architecture may be influencing cognition through each of these processes. Without a measure of underlying pathology, disentangling the mechanism by which genes are affecting cognition is difficult. This point applies to both the current study and the large AD GWAS in which most participants received a diagnosis based only upon assessment of cognition in life. Our finding that AD risk is strongly genetically correlated with the factor score for memory but not executive function or language might provide some insight into these processes. GWAS findings for the individual cognitive domains showing that memory was associated only with established AD risk genes. However, all of the novel associations identified in the pleiotropy analysis included memory as part of the paired outcome. These genetic association patterns might argue that our phenotypes for executive function and language could reflect decline from AD (rather than development of AD), underlying cognitive ability and/or cognitive resilience. Future studies that utilize both measures of cognition and underlying pathology will be needed to better disentangle the genetic architecture underlying these different processes that influence cognition.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61758, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"This study has several limitations. Although we included several large cohorts whose cognitive test data were co-calibrated and harmonized with each other, the sample size was small compared to the previous GWAS of AD risk. In addition, there was a reduction in power for tests of marginal genetic effects or SNP\\u2009\\u00d7\\u2009age interactions because a large portion of the subjects had only one visit (FHS-21.5%, NACC-21.1%, ACT-9.9%, ADNI-13.3%, and ROSMAP-6.0%). The interpretation of our findings based on the joint effect of the SNP and SNP\\u2009\\u00d7\\u2009age interaction is complicated because the identified loci could imply several meanings to cognitive domain functions or AD. Those findings may reflect genetic associations with the development or progression of AD or both, but additional work is needed to address this issue confidently. Further, our model assumes linearity in the cognitive trajectories, but cognitive trajectories at different disease stages may be non-linear. The observed associations with known AD loci provide validation for our modeling approach. Our results are not adjusted for the number of genome-wide scans performed, but the analyses for each cognitive domain and paired cognitive domain are testing separate hypotheses. Correction for analyses conducted separately in the clinic- and community-based cohorts would raise the significance threshold to 2.5\\u2009\\u00d7\\u200910\\u20138, which would render associations for\\u00a0MS4A6A, LINC02712, GRN, LOC107984373, PTPRD,\\u00a0and OSGIN1\\u00a0as borderline GWS. Another concern is the lack of replication which will require the availability of co-calibrated longitudinally obtained cognitive data from independent samples which are informative for AD. Ongoing phenotype harmonization efforts of the Alzheimer\\u2019s Disease Genetics Consortium, Alzheimer\\u2019s Disease Sequencing Project, and other studies will likely yield the data necessary for replication testing. Because some of the identified loci have no obvious connection to AD or cognition, further research is required to determine their mechanistic pathways. Finally, datasets from other population groups containing cognitive domain factor scores for adequately powered samples will be needed to extend our findings which were derived from non-Hispanic whites only.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64033, \"infons\": {\"section_type\": \"CONCL\", \"type\": \"title_1\"}, \"text\": \"Conclusion\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64044, \"infons\": {\"section_type\": \"CONCL\", \"type\": \"paragraph\"}, \"text\": \"Our results provide some insight into biological pathways underlying processes leading to domain-specific cognitive impairment and AD. The findings may provide a conduit toward a syndrome-specific precision medicine approach to AD. Increasing the number of datasets by harmonizing measures of cognitive performance in other cohorts, as applied in this study, would likely enhance the discovery of additional genetic factors of cognitive decline leading to AD and related dementias.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64526, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_1\"}, \"text\": \"Supplementary Information\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64552, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"title\"}, \"text\": \"Abbreviations\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64566, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"ACT\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64570, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Adult Changes in Thought\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64595, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"AD\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64598, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64620, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"ADNI\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64625, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Alzheimer\\u2019s Disease Neuroimaging Initiative\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64671, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"FDR\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64675, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"False discovery rate\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64696, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"FHS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64700, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Framingham Heart Study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64723, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"FTLD\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64728, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Frontotemporal lobar degeneration\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64762, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"GRM\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64766, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Genetic relationship matrix\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64794, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"GWAS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64799, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Genome-wide association study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64829, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"GWS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64833, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Genome-wide significant\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64857, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"HWE\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64861, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Hardy-Weinberg equilibrium\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64888, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"IBD\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64892, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Identical by descent\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64913, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"LATE\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64918, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Limbic-predominant age-related TDP-43 encephalopathy\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64971, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"LD\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64974, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Linkage disequilibrium\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64997, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"MAF\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65001, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Minor allele frequency\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65024, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"MAGEE\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65030, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Mixed\\u2010model association test for gene-environment interactions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65095, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"MCI\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65099, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Mild cognitive impairment\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65125, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"MMSE\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65130, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Mini-mental State Examination\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65160, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"NACC\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65165, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"National Alzheimer\\u2019s Coordinating Center\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65208, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"NP\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65211, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Neuropsychological\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65230, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"PC\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65233, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Principal component\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65253, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"PRGN\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65258, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Progranulin\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65270, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"QC\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65273, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Quality control\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65289, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"ROSMAP\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65296, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Religious Order Study and Rush Memory Aging Project\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65348, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"SE\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65351, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Standard error\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65366, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"SNP\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65370, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Single nucleotide polymorphism\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65401, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"TOPMed\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65408, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"paragraph\"}, \"text\": \"Trans-Omics for Precision Medicine\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65443, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"footnote\"}, \"text\": \"Publisher\\u2019s Note\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65462, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"footnote\"}, \"text\": \"Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65581, \"infons\": {\"section_type\": \"ABBR\", \"type\": \"footnote\"}, \"text\": \"Jesse B. Mez and Lindsay A. Farrer equally supervised this work.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65646, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"title\"}, \"text\": \"Authors\\u2019 contributions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65671, \"infons\": {\"section_type\": \"AUTH_CONT\", \"type\": \"paragraph\"}, \"text\": \"MK, RS, KLL, JBM, and LAF planned and designed the study. SM, LEG, PS, ML, S-EC, BK, CN, PKC, and JBM derived the cognitive domain factor scores. EHT, LCD, AD, TJH, MLC, and AJS harmonized phenotypic data for the ACT, ADNI, ROSMAP, and NACC datasets. TFAA, SAD, RA, and JBM collected and harmonized FHS data. WAK assembled and harmonized ADRC phenotypic data into NACC. DAB collected and harmonized ROSMAP phenotype data. MK performed the statistical analyses. KLL, JBM, and LAF supervised the analyses. MK and LAF wrote the manuscript. All authors edited the manuscript. L-SW, RPM, JLH, MAP-V, GDS, RA, and LAF obtained funding for this study. All authors read and approved the final manuscript.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66368, \"infons\": {\"section_type\": \"ACK_FUND\", \"type\": \"title\"}, \"text\": \"Funding\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66376, \"infons\": {\"section_type\": \"ACK_FUND\", \"type\": \"paragraph\"}, \"text\": \"This work was supported in part by NIH grants U19-AG068753, R01-AG048927, U54-AG052427, U01-AG058654, U01-AG032984, RF1-AG057519, U01-AG062602, P30-AG072878, U24-AG074855, U01-AG068057, and R01-AG059716.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66580, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title\"}, \"text\": \"Availability of data and materials\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66615, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Genetic and phenotype data for the ACT, ADNI, NACC, and ROSMAP GWAS datasets are available through the National Institute on Aging Genetics of Alzheimer Disease Data Storage site (https://www.niagads.org/). Framingham Heart Study data are available in dbGaP and/or through application to the FHS Research Committee (https://www.framinghamheartstudy.org/fhs-for-researchers/). GWAS summary statistics are available in NIAGADS.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67041, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title\"}, \"text\": \"Declarations\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67054, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title\"}, \"text\": \"Ethics approval and consent to participate\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67097, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"This study was performed in accordance with the Declaration of Helsinki and approved by the Boston University Institutional Review Board (Protocol # H-28168). Informed consent was obtained from all subjects and/or their legal guardians.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67334, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title\"}, \"text\": \"Consent for publication\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67358, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Not applicable.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67374, \"infons\": {\"section_type\": \"COMP_INT\", \"type\": \"title\"}, \"text\": \"Competing interests\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67394, \"infons\": {\"section_type\": \"COMP_INT\", \"type\": \"paragraph\"}, \"text\": \"The authors declare they have no competing interests.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67448, \"infons\": {\"section_type\": \"REF\", \"type\": \"title\"}, \"text\": \"References\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67459, \"infons\": {\"fpage\": \"168\", \"lpage\": \"174\", \"name_0\": \"surname:Gatz;given-names:M\", \"name_1\": \"surname:Reynolds;given-names:CA\", \"name_2\": \"surname:Fratiglioni;given-names:L\", \"name_3\": \"surname:Johansson;given-names:B\", \"name_4\": \"surname:Mortimer;given-names:JA\", \"name_5\": \"surname:Berg;given-names:S\", \"name_6\": \"surname:Fiske;given-names:A\", \"name_7\": \"surname:Pedersen;given-names:NL\", \"pub-id_doi\": \"10.1001/archpsyc.63.2.168\", \"pub-id_pmid\": \"16461860\", \"section_type\": \"REF\", \"source\": \"Arch Gen Psychiatry\", \"type\": \"ref\", \"volume\": \"63\", \"year\": \"2006\"}, \"text\": \"Role of genes and environments for explaining Alzheimer disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67523, \"infons\": {\"fpage\": \"832\", \"lpage\": \"841\", \"name_0\": \"surname:Lee;given-names:SH\", \"name_1\": \"surname:Harold;given-names:D\", \"name_2\": \"surname:Nyholt;given-names:DR\", \"name_3\": \"surname:Goddard;given-names:ME\", \"name_4\": \"surname:Zondervan;given-names:KT\", \"name_5\": \"surname:Williams;given-names:J\", \"pub-id_doi\": \"10.1093/hmg/dds491\", \"pub-id_pmid\": \"23193196\", \"section_type\": \"REF\", \"source\": \"Hum Mol Genet\", \"type\": \"ref\", \"volume\": \"22\", \"year\": \"2013\"}, \"text\": \"Estimation and partitioning of polygenic variation captured by common SNPs for Alzheimer\\u2019s disease, multiple sclerosis and endometriosis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67662, \"infons\": {\"fpage\": \"e79771\", \"name_0\": \"surname:Ridge;given-names:PG\", \"name_1\": \"surname:Mukherjee;given-names:S\", \"name_2\": \"surname:Crane;given-names:PK\", \"name_3\": \"surname:Kauwe;given-names:JS\", \"pub-id_doi\": \"10.1371/journal.pone.0079771\", \"pub-id_pmid\": \"24244562\", \"section_type\": \"REF\", \"source\": \"PLoS ONE\", \"type\": \"ref\", \"volume\": \"8\", \"year\": \"2013\"}, \"text\": \"Alzheimer\\u2019s Disease Genetics C: Alzheimer\\u2019s disease: analyzing the missing heritability\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67754, \"infons\": {\"fpage\": \"eaap8757\", \"issue\": \"6395\", \"name_0\": \"surname:Anttila;given-names:V\", \"name_1\": \"surname:Bulik-Sullivan;given-names:B\", \"name_2\": \"surname:Finucane;given-names:HK\", \"name_3\": \"surname:Walters;given-names:RK\", \"name_4\": \"surname:Bras;given-names:J\", \"name_5\": \"surname:Duncan;given-names:L\", \"name_6\": \"surname:Escott-Price;given-names:V\", \"name_7\": \"surname:Falcone;given-names:GJ\", \"name_8\": \"surname:Gormley;given-names:P\", \"pub-id_doi\": \"10.1126/science.aap8757\", \"pub-id_pmid\": \"29930110\", \"section_type\": \"REF\", \"source\": \"Science\", \"type\": \"ref\", \"volume\": \"360\", \"year\": \"2018\"}, \"text\": \"Analysis of shared heritability in common disorders of the brain\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67819, \"infons\": {\"fpage\": \"1094\", \"lpage\": \"1099\", \"name_0\": \"surname:Lambert;given-names:JC\", \"name_1\": \"surname:Heath;given-names:S\", \"name_2\": \"surname:Even;given-names:G\", \"name_3\": \"surname:Campion;given-names:D\", \"name_4\": \"surname:Sleegers;given-names:K\", \"name_5\": \"surname:Hiltunen;given-names:M\", \"name_6\": \"surname:Combarros;given-names:O\", \"name_7\": \"surname:Zelenika;given-names:D\", \"name_8\": \"surname:Bullido;given-names:MJ\", \"name_9\": \"surname:Tavernier;given-names:B\", \"pub-id_doi\": \"10.1038/ng.439\", \"pub-id_pmid\": \"19734903\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"41\", \"year\": \"2009\"}, \"text\": \"Genome-wide association study identifies variants at CLU and CR1 associated with Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67922, \"infons\": {\"fpage\": \"1452\", \"lpage\": \"1458\", \"name_0\": \"surname:Lambert;given-names:JC\", \"name_1\": \"surname:Ibrahim-Verbaas;given-names:CA\", \"name_2\": \"surname:Harold;given-names:D\", \"name_3\": \"surname:Naj;given-names:AC\", \"name_4\": \"surname:Sims;given-names:R\", \"name_5\": \"surname:Bellenguez;given-names:C\", \"name_6\": \"surname:DeStafano;given-names:AL\", \"name_7\": \"surname:Bis;given-names:JC\", \"name_8\": \"surname:Beecham;given-names:GW\", \"name_9\": \"surname:Grenier-Boley;given-names:B\", \"pub-id_doi\": \"10.1038/ng.2802\", \"pub-id_pmid\": \"24162737\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"45\", \"year\": \"2013\"}, \"text\": \"Meta-analysis of 74,046 individuals identifies 11 new susceptibility loci for Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68022, \"infons\": {\"fpage\": \"404\", \"lpage\": \"413\", \"name_0\": \"surname:Jansen;given-names:IE\", \"name_1\": \"surname:Savage;given-names:JE\", \"name_2\": \"surname:Watanabe;given-names:K\", \"name_3\": \"surname:Bryois;given-names:J\", \"name_4\": \"surname:Williams;given-names:DM\", \"name_5\": \"surname:Steinberg;given-names:S\", \"name_6\": \"surname:Sealock;given-names:J\", \"name_7\": \"surname:Karlsson;given-names:IK\", \"name_8\": \"surname:Hagg;given-names:S\", \"name_9\": \"surname:Athanasiu;given-names:L\", \"pub-id_doi\": \"10.1038/s41588-018-0311-9\", \"pub-id_pmid\": \"30617256\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2019\"}, \"text\": \"Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer\\u2019s disease risk\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68131, \"infons\": {\"fpage\": \"414\", \"lpage\": \"430\", \"name_0\": \"surname:Kunkle;given-names:BW\", \"name_1\": \"surname:Grenier-Boley;given-names:B\", \"name_2\": \"surname:Sims;given-names:R\", \"name_3\": \"surname:Bis;given-names:JC\", \"name_4\": \"surname:Damotte;given-names:V\", \"name_5\": \"surname:Naj;given-names:AC\", \"name_6\": \"surname:Boland;given-names:A\", \"name_7\": \"surname:Vronskaya;given-names:M\", \"name_8\": \"surname:van der Lee;given-names:SJ\", \"name_9\": \"surname:Amlie-Wolf;given-names:A\", \"pub-id_doi\": \"10.1038/s41588-019-0358-2\", \"pub-id_pmid\": \"30820047\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"51\", \"year\": \"2019\"}, \"text\": \"Genetic meta-analysis of diagnosed Alzheimer\\u2019s disease identifies new risk loci and implicates Abeta, tau, immunity and lipid processing\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68270, \"infons\": {\"fpage\": \"1276\", \"lpage\": \"1282\", \"name_0\": \"surname:Wightman;given-names:DP\", \"name_1\": \"surname:Jansen;given-names:IE\", \"name_2\": \"surname:Savage;given-names:JE\", \"name_3\": \"surname:Shadrin;given-names:AA\", \"name_4\": \"surname:Bahrami;given-names:S\", \"name_5\": \"surname:Holland;given-names:D\", \"name_6\": \"surname:Rongve;given-names:A\", \"name_7\": \"surname:Borte;given-names:S\", \"name_8\": \"surname:Winsvold;given-names:BS\", \"name_9\": \"surname:Drange;given-names:OK\", \"pub-id_doi\": \"10.1038/s41588-021-00921-z\", \"pub-id_pmid\": \"34493870\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"53\", \"year\": \"2021\"}, \"text\": \"A genome-wide association study with 1,126,563 individuals identifies new risk loci for Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68380, \"infons\": {\"fpage\": \"412\", \"lpage\": \"436\", \"name_0\": \"surname:Bellenguez;given-names:C\", \"name_1\": \"surname:Kucukali;given-names:F\", \"name_2\": \"surname:Jansen;given-names:IE\", \"name_3\": \"surname:Kleineidam;given-names:L\", \"name_4\": \"surname:Moreno-Grau;given-names:S\", \"name_5\": \"surname:Amin;given-names:N\", \"name_6\": \"surname:Naj;given-names:AC\", \"name_7\": \"surname:Campos-Martin;given-names:R\", \"name_8\": \"surname:Grenier-Boley;given-names:B\", \"name_9\": \"surname:Andrade;given-names:V\", \"pub-id_doi\": \"10.1038/s41588-022-01024-z\", \"pub-id_pmid\": \"35379992\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"54\", \"year\": \"2022\"}, \"text\": \"New insights into the genetic etiology of Alzheimer\\u2019s disease and related dementias\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68466, \"infons\": {\"fpage\": \"iyaa046\", \"issue\": \"3\", \"name_0\": \"surname:Holland;given-names:D\", \"name_1\": \"surname:Frei;given-names:O\", \"name_2\": \"surname:Desikan;given-names:R\", \"name_3\": \"surname:Fan;given-names:CC\", \"name_4\": \"surname:Shadrin;given-names:AA\", \"name_5\": \"surname:Smeland;given-names:OB\", \"name_6\": \"surname:Andreassen;given-names:OA\", \"name_7\": \"surname:Dale;given-names:AM\", \"pub-id_doi\": \"10.1093/genetics/iyaa046\", \"pub-id_pmid\": \"33789345\", \"section_type\": \"REF\", \"source\": \"Genetics\", \"type\": \"ref\", \"volume\": \"217\", \"year\": \"2021\"}, \"text\": \"The genetic architecture of human complex phenotypes is modulated by linkage disequilibrium and heterozygosity\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68577, \"infons\": {\"fpage\": \"13790\", \"lpage\": \"13794\", \"name_0\": \"surname:Rietveld;given-names:CA\", \"name_1\": \"surname:Esko;given-names:T\", \"name_2\": \"surname:Davies;given-names:G\", \"name_3\": \"surname:Pers;given-names:TH\", \"name_4\": \"surname:Turley;given-names:P\", \"name_5\": \"surname:Benyamin;given-names:B\", \"name_6\": \"surname:Chabris;given-names:CF\", \"name_7\": \"surname:Emilsson;given-names:V\", \"name_8\": \"surname:Johnson;given-names:AD\", \"name_9\": \"surname:Lee;given-names:JJ\", \"pub-id_doi\": \"10.1073/pnas.1404623111\", \"pub-id_pmid\": \"25201988\", \"section_type\": \"REF\", \"source\": \"Proc Natl Acad Sci U S A\", \"type\": \"ref\", \"volume\": \"111\", \"year\": \"2014\"}, \"text\": \"Common genetic variants associated with cognitive performance identified using the proxy-phenotype method\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68683, \"infons\": {\"fpage\": \"2098\", \"name_0\": \"surname:Davies;given-names:G\", \"name_1\": \"surname:Lam;given-names:M\", \"name_2\": \"surname:Harris;given-names:SE\", \"name_3\": \"surname:Trampush;given-names:JW\", \"name_4\": \"surname:Luciano;given-names:M\", \"name_5\": \"surname:Hill;given-names:WD\", \"name_6\": \"surname:Hagenaars;given-names:SP\", \"name_7\": \"surname:Ritchie;given-names:SJ\", \"name_8\": \"surname:Marioni;given-names:RE\", \"name_9\": \"surname:Fawns-Ritchie;given-names:C\", \"pub-id_doi\": \"10.1038/s41467-018-04362-x\", \"pub-id_pmid\": \"29844566\", \"section_type\": \"REF\", \"source\": \"Nat Commun\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2018\"}, \"text\": \"Study of 300,486 individuals identifies 148 independent genetic loci influencing general cognitive function\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68791, \"infons\": {\"fpage\": \"1134\", \"lpage\": \"1145\", \"name_0\": \"surname:Sherva;given-names:R\", \"name_1\": \"surname:Gross;given-names:A\", \"name_2\": \"surname:Mukherjee;given-names:S\", \"name_3\": \"surname:Koesterer;given-names:R\", \"name_4\": \"surname:Amouyel;given-names:P\", \"name_5\": \"surname:Bellenguez;given-names:C\", \"name_6\": \"surname:Dufouil;given-names:C\", \"name_7\": \"surname:Bennett;given-names:DA\", \"name_8\": \"surname:Chibnik;given-names:L\", \"name_9\": \"surname:Cruchaga;given-names:C\", \"pub-id_doi\": \"10.1002/alz.12106\", \"pub-id_pmid\": \"32573913\", \"section_type\": \"REF\", \"source\": \"Alzheimers Dement\", \"type\": \"ref\", \"volume\": \"16\", \"year\": \"2020\"}, \"text\": \"Genome-wide association study of rate of cognitive decline in Alzheimer\\u2019s disease patients identifies novel genes and pathways\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68920, \"infons\": {\"fpage\": \"2942\", \"lpage\": \"2951\", \"name_0\": \"surname:Mukherjee;given-names:S\", \"name_1\": \"surname:Mez;given-names:J\", \"name_2\": \"surname:Trittschuh;given-names:EH\", \"name_3\": \"surname:Saykin;given-names:AJ\", \"name_4\": \"surname:Gibbons;given-names:LE\", \"name_5\": \"surname:Fardo;given-names:DW\", \"name_6\": \"surname:Wessels;given-names:M\", \"name_7\": \"surname:Bauman;given-names:J\", \"name_8\": \"surname:Moore;given-names:M\", \"name_9\": \"surname:Choi;given-names:SE\", \"pub-id_doi\": \"10.1038/s41380-018-0298-8\", \"pub-id_pmid\": \"30514930\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"25\", \"year\": \"2020\"}, \"text\": \"Genetic data and cognitively defined late-onset Alzheimer\\u2019s disease subgroups\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69000, \"infons\": {\"fpage\": \"159\", \"lpage\": \"169\", \"name_0\": \"surname:Xu;given-names:Z\", \"name_1\": \"surname:Wu;given-names:C\", \"name_2\": \"surname:Pan;given-names:W\", \"pub-id_doi\": \"10.1016/j.neuroimage.2017.07.036\", \"pub-id_pmid\": \"28736311\", \"section_type\": \"REF\", \"source\": \"Neuroimage\", \"type\": \"ref\", \"volume\": \"159\", \"year\": \"2017\"}, \"text\": \"Imaging-wide association study: integrating imaging endophenotypes in GWAS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69075, \"infons\": {\"fpage\": \"117347\", \"name_0\": \"surname:Knutson;given-names:KA\", \"name_1\": \"surname:Deng;given-names:Y\", \"name_2\": \"surname:Pan;given-names:W\", \"pub-id_doi\": \"10.1016/j.neuroimage.2020.117347\", \"pub-id_pmid\": \"32898681\", \"section_type\": \"REF\", \"source\": \"Neuroimage\", \"type\": \"ref\", \"volume\": \"223\", \"year\": \"2020\"}, \"text\": \"Implicating causal brain imaging endophenotypes in Alzheimer\\u2019s disease using multivariable IWAS and GWAS summary data\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69195, \"infons\": {\"fpage\": \"185\", \"lpage\": \"200\", \"name_0\": \"surname:Knutson;given-names:KA\", \"name_1\": \"surname:Pan;given-names:W\", \"pub-id_pmid\": \"35399757\", \"section_type\": \"REF\", \"source\": \"Quant Biol\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2021\"}, \"text\": \"Integrating brain imaging endophenotypes with GWAS for Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69272, \"infons\": {\"fpage\": \"623\", \"lpage\": \"633\", \"name_0\": \"surname:Chung;given-names:J\", \"name_1\": \"surname:Wang;given-names:X\", \"name_2\": \"surname:Maruyama;given-names:T\", \"name_3\": \"surname:Ma;given-names:Y\", \"name_4\": \"surname:Zhang;given-names:X\", \"name_5\": \"surname:Mez;given-names:J\", \"name_6\": \"surname:Sherva;given-names:R\", \"name_7\": \"surname:Takeyama;given-names:H\", \"name_8\": \"surname:Lunetta;given-names:KL\", \"pub-id_doi\": \"10.1016/j.jalz.2017.11.006\", \"pub-id_pmid\": \"29274321\", \"section_type\": \"REF\", \"source\": \"Alzheimers Dement\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2018\"}, \"text\": \"Genome-wide association study of Alzheimer\\u2019s disease endophenotypes at prediagnosis stages\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69365, \"infons\": {\"fpage\": \"1521\", \"lpage\": \"1529\", \"name_0\": \"surname:Chibnik;given-names:LB\", \"name_1\": \"surname:White;given-names:CC\", \"name_2\": \"surname:Mukherjee;given-names:S\", \"name_3\": \"surname:Raj;given-names:T\", \"name_4\": \"surname:Yu;given-names:L\", \"name_5\": \"surname:Larson;given-names:EB\", \"name_6\": \"surname:Montine;given-names:TJ\", \"name_7\": \"surname:Keene;given-names:CD\", \"name_8\": \"surname:Sonnen;given-names:J\", \"name_9\": \"surname:Schneider;given-names:JA\", \"pub-id_doi\": \"10.1038/mp.2017.20\", \"pub-id_pmid\": \"28322283\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"23\", \"year\": \"2018\"}, \"text\": \"Susceptibility to neurofibrillary tangles: role of the PTPRD locus and limited pleiotropy with other neuropathologies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69483, \"infons\": {\"fpage\": \"699\", \"name_0\": \"surname:Tasaki;given-names:S\", \"name_1\": \"surname:Gaiteri;given-names:C\", \"name_2\": \"surname:Mostafavi;given-names:S\", \"name_3\": \"surname:De Jager;given-names:PL\", \"name_4\": \"surname:Bennett;given-names:DA\", \"pub-id_doi\": \"10.3389/fnins.2018.00699\", \"pub-id_pmid\": \"30349450\", \"section_type\": \"REF\", \"source\": \"Front Neurosci\", \"type\": \"ref\", \"volume\": \"12\", \"year\": \"2018\"}, \"text\": \"The molecular and neuropathological consequences of genetic risk for Alzheimer\\u2019s dementia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69575, \"infons\": {\"fpage\": \"4558\", \"lpage\": \"4571\", \"name_0\": \"surname:Cruchaga;given-names:C\", \"name_1\": \"surname:Kauwe;given-names:JS\", \"name_2\": \"surname:Nowotny;given-names:P\", \"name_3\": \"surname:Bales;given-names:K\", \"name_4\": \"surname:Pickering;given-names:EH\", \"name_5\": \"surname:Mayo;given-names:K\", \"name_6\": \"surname:Bertelsen;given-names:S\", \"name_7\": \"surname:Hinrichs;given-names:A\", \"name_8\": \"surname:Fagan;given-names:AM\", \"pub-id_doi\": \"10.1093/hmg/dds296\", \"pub-id_pmid\": \"22821396\", \"section_type\": \"REF\", \"source\": \"Hum Mol Genet\", \"type\": \"ref\", \"volume\": \"21\", \"year\": \"2012\"}, \"text\": \"Cerebrospinal fluid APOE levels: an endophenotype for genetic studies for Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69671, \"infons\": {\"fpage\": \"208.e201\", \"lpage\": \"208.e209\", \"name_0\": \"surname:Deming;given-names:Y\", \"name_1\": \"surname:Xia;given-names:J\", \"name_2\": \"surname:Cai;given-names:Y\", \"name_3\": \"surname:Lord;given-names:J\", \"name_4\": \"surname:Holmans;given-names:P\", \"name_5\": \"surname:Bertelsen;given-names:S\", \"name_6\": \"surname:Holtzman;given-names:D\", \"name_7\": \"surname:Morris;given-names:JC\", \"name_8\": \"surname:Bales;given-names:K\", \"name_9\": \"surname:Pickering;given-names:EH\", \"pub-id_doi\": \"10.1016/j.neurobiolaging.2015.09.009\", \"section_type\": \"REF\", \"source\": \"Neurobiol Aging\", \"type\": \"ref\", \"volume\": \"37\", \"year\": \"2016\"}, \"text\": \"A potential endophenotype for Alzheimer\\u2019s disease: cerebrospinal fluid clusterin\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69754, \"infons\": {\"fpage\": \"839\", \"lpage\": \"856\", \"name_0\": \"surname:Deming;given-names:Y\", \"name_1\": \"surname:Li;given-names:Z\", \"name_2\": \"surname:Kapoor;given-names:M\", \"name_3\": \"surname:Harari;given-names:O\", \"name_4\": \"surname:Del-Aguila;given-names:JL\", \"name_5\": \"surname:Black;given-names:K\", \"name_6\": \"surname:Carrell;given-names:D\", \"name_7\": \"surname:Cai;given-names:Y\", \"name_8\": \"surname:Fernandez;given-names:MV\", \"name_9\": \"surname:Budde;given-names:J\", \"pub-id_doi\": \"10.1007/s00401-017-1685-y\", \"pub-id_pmid\": \"28247064\", \"section_type\": \"REF\", \"source\": \"Acta Neuropathol\", \"type\": \"ref\", \"volume\": \"133\", \"year\": \"2017\"}, \"text\": \"Genome-wide association study identifies four novel loci associated with Alzheimer\\u2019s endophenotypes and disease modifiers\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69878, \"infons\": {\"fpage\": \"336\", \"lpage\": \"345\", \"name_0\": \"surname:Trampush;given-names:JW\", \"name_1\": \"surname:Yang;given-names:ML\", \"name_2\": \"surname:Yu;given-names:J\", \"name_3\": \"surname:Knowles;given-names:E\", \"name_4\": \"surname:Davies;given-names:G\", \"name_5\": \"surname:Liewald;given-names:DC\", \"name_6\": \"surname:Starr;given-names:JM\", \"name_7\": \"surname:Djurovic;given-names:S\", \"name_8\": \"surname:Melle;given-names:I\", \"name_9\": \"surname:Sundet;given-names:K\", \"pub-id_doi\": \"10.1038/mp.2016.244\", \"pub-id_pmid\": \"28093568\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"22\", \"year\": \"2017\"}, \"text\": \"GWAS meta-analysis reveals novel loci and genetic correlates for general cognitive function: a report from the COGENT consortium\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70007, \"infons\": {\"fpage\": \"749\", \"lpage\": \"763\", \"name_0\": \"surname:Debette;given-names:S\", \"name_1\": \"surname:Ibrahim Verbaas;given-names:CA\", \"name_2\": \"surname:Bressler;given-names:J\", \"name_3\": \"surname:Schuur;given-names:M\", \"name_4\": \"surname:Smith;given-names:A\", \"name_5\": \"surname:Bis;given-names:JC\", \"name_6\": \"surname:Davies;given-names:G\", \"name_7\": \"surname:Wolf;given-names:C\", \"name_8\": \"surname:Gudnason;given-names:V\", \"name_9\": \"surname:Chibnik;given-names:LB\", \"pub-id_doi\": \"10.1016/j.biopsych.2014.08.027\", \"pub-id_pmid\": \"25648963\", \"section_type\": \"REF\", \"source\": \"Biol Psychiatry\", \"type\": \"ref\", \"volume\": \"77\", \"year\": \"2015\"}, \"text\": \"Genome-wide studies of verbal declarative memory in nondemented older people: the Cohorts for Heart and Aging Research in Genomic Epidemiology consortium\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70161, \"infons\": {\"fpage\": \"189\", \"lpage\": \"197\", \"name_0\": \"surname:Ibrahim-Verbaas;given-names:CA\", \"name_1\": \"surname:Bressler;given-names:J\", \"name_2\": \"surname:Debette;given-names:S\", \"name_3\": \"surname:Schuur;given-names:M\", \"name_4\": \"surname:Smith;given-names:AV\", \"name_5\": \"surname:Bis;given-names:JC\", \"name_6\": \"surname:Davies;given-names:G\", \"name_7\": \"surname:Trompet;given-names:S\", \"name_8\": \"surname:Smith;given-names:JA\", \"name_9\": \"surname:Wolf;given-names:C\", \"pub-id_doi\": \"10.1038/mp.2015.37\", \"pub-id_pmid\": \"25869804\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"21\", \"year\": \"2016\"}, \"text\": \"GWAS for executive function and processing speed suggests involvement of the CADM2 gene\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70249, \"infons\": {\"fpage\": \"758\", \"lpage\": \"767\", \"name_0\": \"surname:Davies;given-names:G\", \"name_1\": \"surname:Marioni;given-names:RE\", \"name_2\": \"surname:Liewald;given-names:DC\", \"name_3\": \"surname:Hill;given-names:WD\", \"name_4\": \"surname:Hagenaars;given-names:SP\", \"name_5\": \"surname:Harris;given-names:SE\", \"name_6\": \"surname:Ritchie;given-names:SJ\", \"name_7\": \"surname:Luciano;given-names:M\", \"name_8\": \"surname:Fawns-Ritchie;given-names:C\", \"name_9\": \"surname:Lyall;given-names:D\", \"pub-id_doi\": \"10.1038/mp.2016.45\", \"pub-id_pmid\": \"27046643\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"21\", \"year\": \"2016\"}, \"text\": \"Genome-wide association study of cognitive functions and educational attainment in UK Biobank (N=112 151)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70355, \"infons\": {\"fpage\": \"840651\", \"name_0\": \"surname:Homann;given-names:J\", \"name_1\": \"surname:Osburg;given-names:T\", \"name_2\": \"surname:Ohlei;given-names:O\", \"name_3\": \"surname:Dobricic;given-names:V\", \"name_4\": \"surname:Deecke;given-names:L\", \"name_5\": \"surname:Bos;given-names:I\", \"name_6\": \"surname:Vandenberghe;given-names:R\", \"name_7\": \"surname:Gabel;given-names:S\", \"name_8\": \"surname:Scheltens;given-names:P\", \"name_9\": \"surname:Teunissen;given-names:CE\", \"pub-id_doi\": \"10.3389/fnagi.2022.840651\", \"pub-id_pmid\": \"35386118\", \"section_type\": \"REF\", \"source\": \"Front Aging Neurosci\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2022\"}, \"text\": \"Genome-wide association study of Alzheimer\\u2019s disease brain imaging biomarkers and neuropsychological phenotypes in the European Medical Information Framework for Alzheimer\\u2019s Disease Multimodal Biomarker Discovery dataset\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70580, \"infons\": {\"fpage\": \"409\", \"lpage\": \"23\", \"name_0\": \"surname:Mukherjee;given-names:S\", \"name_1\": \"surname:Choi;given-names:SE\", \"name_2\": \"surname:Lee;given-names:ML\", \"name_3\": \"surname:Scollard;given-names:P\", \"name_4\": \"surname:Trittschuh;given-names:EH\", \"name_5\": \"surname:Mez;given-names:J\", \"pub-id_doi\": \"10.1037/neu0000835\", \"pub-id_pmid\": \"35925737\", \"section_type\": \"REF\", \"source\": \"Neuropsychology\", \"type\": \"ref\", \"volume\": \"37\", \"year\": \"2023\"}, \"text\": \"Cognitive domain harmonization and cocalibration in studies of older adults\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70656, \"infons\": {\"fpage\": \"767\", \"lpage\": \"773\", \"name_0\": \"surname:Stearns;given-names:FW\", \"pub-id_doi\": \"10.1534/genetics.110.122549\", \"pub-id_pmid\": \"21062962\", \"section_type\": \"REF\", \"source\": \"Genetics\", \"type\": \"ref\", \"volume\": \"186\", \"year\": \"2010\"}, \"text\": \"One hundred years of pleiotropy: a retrospective\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70705, \"infons\": {\"fpage\": \"66\", \"lpage\": \"73\", \"name_0\": \"surname:Paaby;given-names:AB\", \"name_1\": \"surname:Rockman;given-names:MV\", \"pub-id_doi\": \"10.1016/j.tig.2012.10.010\", \"pub-id_pmid\": \"23140989\", \"section_type\": \"REF\", \"source\": \"Trends Genet\", \"type\": \"ref\", \"volume\": \"29\", \"year\": \"2013\"}, \"text\": \"The many faces of pleiotropy\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70734, \"infons\": {\"fpage\": \"483\", \"lpage\": \"495\", \"name_0\": \"surname:Solovieff;given-names:N\", \"name_1\": \"surname:Cotsapas;given-names:C\", \"name_2\": \"surname:Lee;given-names:PH\", \"name_3\": \"surname:Purcell;given-names:SM\", \"name_4\": \"surname:Smoller;given-names:JW\", \"pub-id_doi\": \"10.1038/nrg3461\", \"pub-id_pmid\": \"23752797\", \"section_type\": \"REF\", \"source\": \"Nat Rev Genet\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2013\"}, \"text\": \"Pleiotropy in complex traits: challenges and strategies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70790, \"infons\": {\"fpage\": \"567\", \"lpage\": \"581\", \"name_0\": \"surname:van Rheenen;given-names:W\", \"name_1\": \"surname:Peyrot;given-names:WJ\", \"name_2\": \"surname:Schork;given-names:AJ\", \"name_3\": \"surname:Lee;given-names:SH\", \"name_4\": \"surname:Wray;given-names:NR\", \"pub-id_doi\": \"10.1038/s41576-019-0137-z\", \"pub-id_pmid\": \"31171865\", \"section_type\": \"REF\", \"source\": \"Nat Rev Genet\", \"type\": \"ref\", \"volume\": \"20\", \"year\": \"2019\"}, \"text\": \"Genetic correlations of polygenic disease traits: from theory to practice\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70864, \"infons\": {\"fpage\": \"e43728\", \"name_0\": \"surname:Jun;given-names:G\", \"name_1\": \"surname:Moncaster;given-names:JA\", \"name_10\": \"surname:Farrer;given-names:LA\", \"name_2\": \"surname:Koutras;given-names:C\", \"name_3\": \"surname:Seshadri;given-names:S\", \"name_4\": \"surname:Buros;given-names:J\", \"name_5\": \"surname:McKee;given-names:AC\", \"name_6\": \"surname:Levesque;given-names:G\", \"name_7\": \"surname:Wolf;given-names:PA\", \"name_8\": \"surname:St George-Hyslop;given-names:P\", \"name_9\": \"surname:Goldstein;given-names:LE\", \"pub-id_doi\": \"10.1371/journal.pone.0043728\", \"pub-id_pmid\": \"22984439\", \"section_type\": \"REF\", \"source\": \"PLoS ONE\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2012\"}, \"text\": \"delta-Catenin is genetically and biologically associated with cortical cataract and future Alzheimer-related structural and functional brain changes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71013, \"infons\": {\"fpage\": \"22\", \"name_0\": \"surname:Chung;given-names:J\", \"name_1\": \"surname:Zhang;given-names:X\", \"name_2\": \"surname:Allen;given-names:M\", \"name_3\": \"surname:Wang;given-names:X\", \"name_4\": \"surname:Ma;given-names:Y\", \"name_5\": \"surname:Beecham;given-names:G\", \"name_6\": \"surname:Montine;given-names:TJ\", \"name_7\": \"surname:Younkin;given-names:SG\", \"name_8\": \"surname:Dickson;given-names:DW\", \"name_9\": \"surname:Golde;given-names:TE\", \"pub-id_doi\": \"10.1186/s13195-018-0349-z\", \"pub-id_pmid\": \"29458411\", \"section_type\": \"REF\", \"source\": \"Alzheimers Res Ther\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2018\"}, \"text\": \"Genome-wide pleiotropy analysis of neuropathological traits related to Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71106, \"infons\": {\"fpage\": \"34\", \"name_0\": \"surname:Bone;given-names:WP\", \"name_1\": \"surname:Siewert;given-names:KM\", \"name_10\": \"surname:Voight;given-names:BF\", \"name_2\": \"surname:Jha;given-names:A\", \"name_3\": \"surname:Klarin;given-names:D\", \"name_4\": \"surname:Damrauer;given-names:SM\", \"name_5\": \"surname:Program;given-names:VAMV\", \"name_6\": \"surname:Chang;given-names:KM\", \"name_7\": \"surname:Tsao;given-names:PS\", \"name_8\": \"surname:Assimes;given-names:TL\", \"name_9\": \"surname:Ritchie;given-names:MD\", \"pub-id_doi\": \"10.1186/s13195-021-00773-z\", \"pub-id_pmid\": \"33541420\", \"section_type\": \"REF\", \"source\": \"Alzheimers Res Ther\", \"type\": \"ref\", \"volume\": \"13\", \"year\": \"2021\"}, \"text\": \"Multi-trait association studies discover pleiotropic loci between Alzheimer\\u2019s disease and cardiometabolic traits\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71221, \"infons\": {\"fpage\": \"388\", \"lpage\": \"394\", \"name_0\": \"surname:Harris;given-names:SE\", \"name_1\": \"surname:Deary;given-names:IJ\", \"pub-id_pmid\": \"21840749\", \"section_type\": \"REF\", \"source\": \"Trends Cogn Sci\", \"type\": \"ref\", \"volume\": \"15\", \"year\": \"2011\"}, \"text\": \"The genetics of cognitive ability and cognitive ageing in healthy older people\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71300, \"infons\": {\"fpage\": \"702\", \"lpage\": \"709\", \"name_0\": \"surname:Polderman;given-names:TJ\", \"name_1\": \"surname:Benyamin;given-names:B\", \"name_2\": \"surname:de Leeuw;given-names:CA\", \"name_3\": \"surname:Sullivan;given-names:PF\", \"name_4\": \"surname:van Bochoven;given-names:A\", \"name_5\": \"surname:Visscher;given-names:PM\", \"name_6\": \"surname:Posthuma;given-names:D\", \"pub-id_doi\": \"10.1038/ng.3285\", \"pub-id_pmid\": \"25985137\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"47\", \"year\": \"2015\"}, \"text\": \"Meta-analysis of the heritability of human traits based on fifty years of twin studies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71387, \"infons\": {\"fpage\": \"98\", \"lpage\": \"108\", \"name_0\": \"surname:Plomin;given-names:R\", \"name_1\": \"surname:Deary;given-names:IJ\", \"pub-id_doi\": \"10.1038/mp.2014.105\", \"pub-id_pmid\": \"25224258\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"20\", \"year\": \"2015\"}, \"text\": \"Genetics and intelligence differences: five special findings\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71448, \"infons\": {\"fpage\": \"553\", \"lpage\": \"555\", \"name_0\": \"surname:Dawber;given-names:TR\", \"name_1\": \"surname:Kannel;given-names:WB\", \"pub-id_doi\": \"10.1161/01.CIR.34.4.553\", \"pub-id_pmid\": \"5921755\", \"section_type\": \"REF\", \"source\": \"Circulation\", \"type\": \"ref\", \"volume\": \"34\", \"year\": \"1966\"}, \"text\": \"The Framingham study. An epidemiological approach to coronary heart disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71524, \"infons\": {\"fpage\": \"518\", \"lpage\": \"525\", \"name_0\": \"surname:Feinleib;given-names:M\", \"name_1\": \"surname:Kannel;given-names:WB\", \"name_2\": \"surname:Garrison;given-names:RJ\", \"name_3\": \"surname:McNamara;given-names:PM\", \"name_4\": \"surname:Castelli;given-names:WP\", \"pub-id_doi\": \"10.1016/0091-7435(75)90037-7\", \"pub-id_pmid\": \"1208363\", \"section_type\": \"REF\", \"source\": \"Prev Med\", \"type\": \"ref\", \"volume\": \"4\", \"year\": \"1975\"}, \"text\": \"The Framingham offspring study. Design and preliminary data\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71584, \"infons\": {\"fpage\": \"1328\", \"lpage\": \"1335\", \"name_0\": \"surname:Splansky;given-names:GL\", \"name_1\": \"surname:Corey;given-names:D\", \"name_2\": \"surname:Yang;given-names:Q\", \"name_3\": \"surname:Atwood;given-names:LD\", \"name_4\": \"surname:Cupples;given-names:LA\", \"name_5\": \"surname:Benjamin;given-names:EJ\", \"name_6\": \"surname:D\\u2019Agostino;given-names:RB;suffix:Sr\", \"name_7\": \"surname:Fox;given-names:CS\", \"name_8\": \"surname:Larson;given-names:MG\", \"name_9\": \"surname:Murabito;given-names:JM\", \"pub-id_doi\": \"10.1093/aje/kwm021\", \"pub-id_pmid\": \"17372189\", \"section_type\": \"REF\", \"source\": \"Am J Epidemiol\", \"type\": \"ref\", \"volume\": \"165\", \"year\": \"2007\"}, \"text\": \"The third generation cohort of the National Heart, Lung, and Blood Institute\\u2019s Framingham Heart Study: design, recruitment, and initial examination\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71734, \"infons\": {\"fpage\": \"1498\", \"lpage\": \"1504\", \"name_0\": \"surname:Seshadri;given-names:S\", \"name_1\": \"surname:Wolf;given-names:PA\", \"name_2\": \"surname:Beiser;given-names:A\", \"name_3\": \"surname:Au;given-names:R\", \"name_4\": \"surname:McNulty;given-names:K\", \"name_5\": \"surname:White;given-names:R\", \"name_6\": \"surname:D\\u2019Agostino;given-names:RB\", \"pub-id_doi\": \"10.1212/WNL.49.6.1498\", \"pub-id_pmid\": \"9409336\", \"section_type\": \"REF\", \"source\": \"Neurology\", \"type\": \"ref\", \"volume\": \"49\", \"year\": \"1997\"}, \"text\": \"Lifetime risk of dementia and Alzheimer\\u2019s disease. The impact of mortality on risk estimates in the Framingham study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71853, \"infons\": {\"fpage\": \"35\", \"lpage\": \"52\", \"name_0\": \"surname:Seshadri;given-names:S\", \"name_1\": \"surname:Beiser;given-names:A\", \"name_2\": \"surname:Au;given-names:R\", \"name_3\": \"surname:Wolf;given-names:PA\", \"name_4\": \"surname:Evans;given-names:DA\", \"name_5\": \"surname:Wilson;given-names:RS\", \"name_6\": \"surname:Petersen;given-names:RC\", \"name_7\": \"surname:Knopman;given-names:DS\", \"name_8\": \"surname:Rocca;given-names:WA\", \"name_9\": \"surname:Kawas;given-names:CH\", \"pub-id_doi\": \"10.1016/j.jalz.2010.12.002\", \"pub-id_pmid\": \"21255742\", \"section_type\": \"REF\", \"source\": \"Alzheimers Dement\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2011\"}, \"text\": \"Operationalizing diagnostic criteria for Alzheimer\\u2019s disease and other age-related cognitive impairment-part 2\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71966, \"infons\": {\"fpage\": \"1737\", \"lpage\": \"1746\", \"name_0\": \"surname:Kukull;given-names:WA\", \"name_1\": \"surname:Higdon;given-names:R\", \"name_2\": \"surname:Bowen;given-names:JD\", \"name_3\": \"surname:McCormick;given-names:WC\", \"name_4\": \"surname:Teri;given-names:L\", \"name_5\": \"surname:Schellenberg;given-names:GD\", \"name_6\": \"surname:van Belle;given-names:G\", \"name_7\": \"surname:Jolley;given-names:L\", \"name_8\": \"surname:Larson;given-names:EB\", \"pub-id_doi\": \"10.1001/archneur.59.11.1737\", \"pub-id_pmid\": \"12433261\", \"section_type\": \"REF\", \"source\": \"Arch Neurol\", \"type\": \"ref\", \"volume\": \"59\", \"year\": \"2002\"}, \"text\": \"Dementia and Alzheimer disease incidence: a prospective cohort study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72035, \"infons\": {\"fpage\": \"S161\", \"lpage\": \"S189\", \"name_0\": \"surname:Bennett;given-names:DA\", \"name_1\": \"surname:Buchman;given-names:AS\", \"name_2\": \"surname:Boyle;given-names:PA\", \"name_3\": \"surname:Barnes;given-names:LL\", \"name_4\": \"surname:Wilson;given-names:RS\", \"name_5\": \"surname:Schneider;given-names:JA\", \"pub-id_doi\": \"10.3233/JAD-179939\", \"pub-id_pmid\": \"29865057\", \"section_type\": \"REF\", \"source\": \"J Alzheimers Dis\", \"type\": \"ref\", \"volume\": \"64\", \"year\": \"2018\"}, \"text\": \"Religious orders study and rush memory and aging project\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72092, \"infons\": {\"fpage\": \"270\", \"lpage\": \"277\", \"name_0\": \"surname:Beekly;given-names:DL\", \"name_1\": \"surname:Ramos;given-names:EM\", \"name_2\": \"surname:van Belle;given-names:G\", \"name_3\": \"surname:Deitrich;given-names:W\", \"name_4\": \"surname:Clark;given-names:AD\", \"name_5\": \"surname:Jacka;given-names:ME\", \"name_6\": \"surname:Kukull;given-names:WA\", \"pub-id_pmid\": \"15592144\", \"section_type\": \"REF\", \"source\": \"Alzheimer Dis Assoc Disord\", \"type\": \"ref\", \"volume\": \"18\", \"year\": \"2004\"}, \"text\": \"The National Alzheimer\\u2019s Coordinating Center (NACC) Database: an Alzheimer disease database\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72186, \"infons\": {\"fpage\": \"249\", \"lpage\": \"258\", \"name_0\": \"surname:Beekly;given-names:DL\", \"name_1\": \"surname:Ramos;given-names:EM\", \"name_2\": \"surname:Lee;given-names:WW\", \"name_3\": \"surname:Deitrich;given-names:WD\", \"name_4\": \"surname:Jacka;given-names:ME\", \"name_5\": \"surname:Wu;given-names:J\", \"name_6\": \"surname:Hubbard;given-names:JL\", \"name_7\": \"surname:Koepsell;given-names:TD\", \"name_8\": \"surname:Morris;given-names:JC\", \"name_9\": \"surname:Kukull;given-names:WA\", \"pub-id_doi\": \"10.1097/WAD.0b013e318142774e\", \"pub-id_pmid\": \"17804958\", \"section_type\": \"REF\", \"source\": \"Alzheimer Dis Assoc Disord\", \"type\": \"ref\", \"volume\": \"21\", \"year\": \"2007\"}, \"text\": \"The National Alzheimer\\u2019s Coordinating Center (NACC) database: the uniform data set\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72271, \"infons\": {\"fpage\": \"869\", \"lpage\": \"877\", \"name_0\": \"surname:Mueller;given-names:SG\", \"name_1\": \"surname:Weiner;given-names:MW\", \"name_2\": \"surname:Thal;given-names:LJ\", \"name_3\": \"surname:Petersen;given-names:RC\", \"name_4\": \"surname:Jack;given-names:C\", \"name_5\": \"surname:Jagust;given-names:W\", \"name_6\": \"surname:Trojanowski;given-names:JQ\", \"name_7\": \"surname:Toga;given-names:AW\", \"name_8\": \"surname:Beckett;given-names:L\", \"pub-id_doi\": \"10.1016/j.nic.2005.09.008\", \"pub-id_pmid\": \"16443497\", \"section_type\": \"REF\", \"source\": \"Neuroimaging Clin N Am\", \"type\": \"ref\", \"volume\": \"15\", \"year\": \"2005\"}, \"text\": \"The Alzheimer\\u2019s disease neuroimaging initiative\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72321, \"infons\": {\"fpage\": \"202\", \"lpage\": \"211.e207\", \"name_0\": \"surname:Weiner;given-names:MW\", \"name_1\": \"surname:Aisen;given-names:PS\", \"name_2\": \"surname:Jack;given-names:CR;suffix:Jr\", \"name_3\": \"surname:Jagust;given-names:WJ\", \"name_4\": \"surname:Trojanowski;given-names:JQ\", \"name_5\": \"surname:Shaw;given-names:L\", \"name_6\": \"surname:Saykin;given-names:AJ\", \"name_7\": \"surname:Morris;given-names:JC\", \"name_8\": \"surname:Cairns;given-names:N\", \"name_9\": \"surname:Beckett;given-names:LA\", \"pub-id_doi\": \"10.1016/j.jalz.2010.03.007\", \"pub-id_pmid\": \"20451868\", \"section_type\": \"REF\", \"source\": \"Alzheimers Dement\", \"type\": \"ref\", \"volume\": \"6\", \"year\": \"2010\"}, \"text\": \"The Alzheimer\\u2019s disease neuroimaging initiative: progress report and future plans\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72405, \"infons\": {\"fpage\": \"201\", \"lpage\": \"209\", \"name_0\": \"surname:Petersen;given-names:RC\", \"name_1\": \"surname:Aisen;given-names:PS\", \"name_2\": \"surname:Beckett;given-names:LA\", \"name_3\": \"surname:Donohue;given-names:MC\", \"name_4\": \"surname:Gamst;given-names:AC\", \"name_5\": \"surname:Harvey;given-names:DJ\", \"name_6\": \"surname:Jack;given-names:CR;suffix:Jr\", \"name_7\": \"surname:Jagust;given-names:WJ\", \"name_8\": \"surname:Shaw;given-names:LM\", \"name_9\": \"surname:Toga;given-names:AW\", \"pub-id_doi\": \"10.1212/WNL.0b013e3181cb3e25\", \"pub-id_pmid\": \"20042704\", \"section_type\": \"REF\", \"source\": \"Neurology\", \"type\": \"ref\", \"volume\": \"74\", \"year\": \"2010\"}, \"text\": \"Alzheimer\\u2019s Disease Neuroimaging Initiative (ADNI): clinical characterization\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72485, \"infons\": {\"fpage\": \"383\", \"lpage\": \"97\", \"name_0\": \"surname:Scollard;given-names:P\", \"name_1\": \"surname:Choi;given-names:S-E\", \"name_2\": \"surname:Lee;given-names:ML\", \"name_3\": \"surname:Mukherjee;given-names:S\", \"name_4\": \"surname:Trittschuh;given-names:E\", \"name_5\": \"surname:Sanders;given-names:RE\", \"pub-id_doi\": \"10.1037/neu0000828\", \"pub-id_pmid\": \"37276135\", \"section_type\": \"REF\", \"source\": \"Neuropsychology\", \"type\": \"ref\", \"volume\": \"37\", \"year\": \"2023\"}, \"text\": \"Ceiling effects and differential measurement precision across calibrated cognitive scores in the Framingham study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72599, \"infons\": {\"name_0\": \"surname:Muth\\u00e9n;given-names:LK\", \"name_1\": \"surname:Muth\\u00e9n;given-names:BO\", \"section_type\": \"REF\", \"source\": \"Mplus: statistical analysis with latent variables: user\\u2019s guide (version 8)\", \"type\": \"ref\", \"year\": \"2017\"}, \"text\": \"\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72600, \"infons\": {\"fpage\": \"290\", \"lpage\": \"299\", \"name_0\": \"surname:Taliun;given-names:D\", \"name_1\": \"surname:Harris;given-names:DN\", \"name_2\": \"surname:Kessler;given-names:MD\", \"name_3\": \"surname:Carlson;given-names:J\", \"name_4\": \"surname:Szpiech;given-names:ZA\", \"name_5\": \"surname:Torres;given-names:R\", \"name_6\": \"surname:Taliun;given-names:SAG\", \"name_7\": \"surname:Corvelo;given-names:A\", \"name_8\": \"surname:Gogarten;given-names:SM\", \"name_9\": \"surname:Kang;given-names:HM\", \"pub-id_doi\": \"10.1038/s41586-021-03205-y\", \"pub-id_pmid\": \"33568819\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"590\", \"year\": \"2021\"}, \"text\": \"Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72667, \"infons\": {\"fpage\": \"5346\", \"lpage\": \"5348\", \"name_0\": \"surname:Gogarten;given-names:SM\", \"name_1\": \"surname:Sofer;given-names:T\", \"name_2\": \"surname:Chen;given-names:H\", \"name_3\": \"surname:Yu;given-names:C\", \"name_4\": \"surname:Brody;given-names:JA\", \"name_5\": \"surname:Thornton;given-names:TA\", \"name_6\": \"surname:Rice;given-names:KM\", \"name_7\": \"surname:Conomos;given-names:MP\", \"pub-id_doi\": \"10.1093/bioinformatics/btz567\", \"pub-id_pmid\": \"31329242\", \"section_type\": \"REF\", \"source\": \"Bioinformatics\", \"type\": \"ref\", \"volume\": \"35\", \"year\": \"2019\"}, \"text\": \"Genetic association testing using the GENESIS R/Bioconductor package\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72736, \"infons\": {\"fpage\": \"2867\", \"lpage\": \"2873\", \"name_0\": \"surname:Manichaikul;given-names:A\", \"name_1\": \"surname:Mychaleckyj;given-names:JC\", \"name_2\": \"surname:Rich;given-names:SS\", \"name_3\": \"surname:Daly;given-names:K\", \"name_4\": \"surname:Sale;given-names:M\", \"name_5\": \"surname:Chen;given-names:WM\", \"pub-id_doi\": \"10.1093/bioinformatics/btq559\", \"pub-id_pmid\": \"20926424\", \"section_type\": \"REF\", \"source\": \"Bioinformatics\", \"type\": \"ref\", \"volume\": \"26\", \"year\": \"2010\"}, \"text\": \"Robust relationship inference in genome-wide association studies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72801, \"infons\": {\"fpage\": \"276\", \"lpage\": \"293\", \"name_0\": \"surname:Conomos;given-names:MP\", \"name_1\": \"surname:Miller;given-names:MB\", \"name_2\": \"surname:Thornton;given-names:TA\", \"pub-id_doi\": \"10.1002/gepi.21896\", \"pub-id_pmid\": \"25810074\", \"section_type\": \"REF\", \"source\": \"Genet Epidemiol\", \"type\": \"ref\", \"volume\": \"39\", \"year\": \"2015\"}, \"text\": \"Robust inference of population structure for ancestry prediction and correction of stratification in the presence of relatedness\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72930, \"infons\": {\"fpage\": \"127\", \"lpage\": \"148\", \"name_0\": \"surname:Conomos;given-names:MP\", \"name_1\": \"surname:Reiner;given-names:AP\", \"name_2\": \"surname:Weir;given-names:BS\", \"name_3\": \"surname:Thornton;given-names:TA\", \"pub-id_doi\": \"10.1016/j.ajhg.2015.11.022\", \"pub-id_pmid\": \"26748516\", \"section_type\": \"REF\", \"source\": \"Am J Hum Genet\", \"type\": \"ref\", \"volume\": \"98\", \"year\": \"2016\"}, \"text\": \"Model-free estimation of recent genetic relatedness\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 72982, \"infons\": {\"fpage\": \"76\", \"lpage\": \"82\", \"name_0\": \"surname:Yang;given-names:J\", \"name_1\": \"surname:Lee;given-names:SH\", \"name_2\": \"surname:Goddard;given-names:ME\", \"name_3\": \"surname:Visscher;given-names:PM\", \"pub-id_doi\": \"10.1016/j.ajhg.2010.11.011\", \"pub-id_pmid\": \"21167468\", \"section_type\": \"REF\", \"source\": \"Am J Hum Genet\", \"type\": \"ref\", \"volume\": \"88\", \"year\": \"2011\"}, \"text\": \"GCTA: a tool for genome-wide complex trait analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 73034, \"infons\": {\"fpage\": \"2540\", \"lpage\": \"2542\", \"name_0\": \"surname:Lee;given-names:SH\", \"name_1\": \"surname:Yang;given-names:J\", \"name_2\": \"surname:Goddard;given-names:ME\", \"name_3\": \"surname:Visscher;given-names:PM\", \"name_4\": \"surname:Wray;given-names:NR\", \"pub-id_doi\": \"10.1093/bioinformatics/bts474\", \"pub-id_pmid\": \"22843982\", \"section_type\": \"REF\", \"source\": \"Bioinformatics\", \"type\": \"ref\", \"volume\": \"28\", \"year\": \"2012\"}, \"text\": \"Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 73185, \"infons\": {\"fpage\": \"91\", \"lpage\": \"93\", \"name_0\": \"surname:Sinnwell;given-names:JP\", \"name_1\": \"surname:Therneau;given-names:TM\", \"name_2\": \"surname:Schaid;given-names:DJ\", \"pub-id_doi\": \"10.1159/000363105\", \"pub-id_pmid\": \"25074474\", \"section_type\": \"REF\", \"source\": \"Hum Hered\", \"type\": \"ref\", \"volume\": \"78\", \"year\": \"2014\"}, \"text\": \"The kinship2 R package for pedigree data\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 73226, \"infons\": {\"fpage\": \"653\", \"lpage\": \"666\", \"name_0\": \"surname:Chen;given-names:H\", \"name_1\": \"surname:Wang;given-names:C\", \"name_2\": \"surname:Conomos;given-names:MP\", \"name_3\": \"surname:Stilp;given-names:AM\", \"name_4\": \"surname:Li;given-names:Z\", \"name_5\": \"surname:Sofer;given-names:T\", \"name_6\": \"surname:Szpiro;given-names:AA\", \"name_7\": \"surname:Chen;given-names:W\", \"name_8\": \"surname:Brehm;given-names:JM\", \"name_9\": \"surname:Celedon;given-names:JC\", \"pub-id_doi\": \"10.1016/j.ajhg.2016.02.012\", \"pub-id_pmid\": \"27018471\", \"section_type\": \"REF\", \"source\": \"Am J Hum Genet\", \"type\": \"ref\", \"volume\": \"98\", \"year\": \"2016\"}, \"text\": \"Control for population structure and relatedness for binary traits in genetic association studies via logistic mixed models\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 73350, \"infons\": {\"fpage\": \"908\", \"lpage\": \"923\", \"name_0\": \"surname:Wang;given-names:X\", \"name_1\": \"surname:Lim;given-names:E\", \"name_2\": \"surname:Liu;given-names:CT\", \"name_3\": \"surname:Sung;given-names:YJ\", \"name_4\": \"surname:Rao;given-names:DC\", \"name_5\": \"surname:Morrison;given-names:AC\", \"name_6\": \"surname:Boerwinkle;given-names:E\", \"name_7\": \"surname:Manning;given-names:AK\", \"name_8\": \"surname:Chen;given-names:H\", \"pub-id_doi\": \"10.1002/gepi.22351\", \"pub-id_pmid\": \"32864785\", \"section_type\": \"REF\", \"source\": \"Genet Epidemiol\", \"type\": \"ref\", \"volume\": \"44\", \"year\": \"2020\"}, \"text\": \"Efficient gene-environment interaction tests for large biobank-scale sequencing studies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 73438, \"infons\": {\"fpage\": \"1236\", \"lpage\": \"1241\", \"name_0\": \"surname:Bulik-Sullivan;given-names:B\", \"name_1\": \"surname:Finucane;given-names:HK\", \"name_2\": \"surname:Anttila;given-names:V\", \"name_3\": \"surname:Gusev;given-names:A\", \"name_4\": \"surname:Day;given-names:FR\", \"name_5\": \"surname:Loh;given-names:PR\", \"name_6\": \"surname:Duncan;given-names:L\", \"pub-id_doi\": \"10.1038/ng.3406\", \"pub-id_pmid\": \"26414676\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"47\", \"year\": \"2015\"}, \"text\": \"An atlas of genetic correlations across human diseases and traits\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 73504, \"infons\": {\"fpage\": \"e19416\", \"name_0\": \"surname:Voorman;given-names:A\", \"name_1\": \"surname:Lumley;given-names:T\", \"name_2\": \"surname:McKnight;given-names:B\", \"name_3\": \"surname:Rice;given-names:K\", \"pub-id_doi\": \"10.1371/journal.pone.0019416\", \"pub-id_pmid\": \"21589913\", \"section_type\": \"REF\", \"source\": \"PLoS ONE\", \"type\": \"ref\", \"volume\": \"6\", \"year\": \"2011\"}, \"text\": \"Behavior of QQ-plots and genomic control in studies of gene-environment interaction\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 73588, \"infons\": {\"fpage\": \"1392\", \"lpage\": \"1399\", \"name_0\": \"surname:Almli;given-names:LM\", \"name_1\": \"surname:Duncan;given-names:R\", \"name_2\": \"surname:Feng;given-names:H\", \"name_3\": \"surname:Ghosh;given-names:D\", \"name_4\": \"surname:Binder;given-names:EB\", \"name_5\": \"surname:Bradley;given-names:B\", \"name_6\": \"surname:Ressler;given-names:KJ\", \"name_7\": \"surname:Conneely;given-names:KN\", \"name_8\": \"surname:Epstein;given-names:MP\", \"pub-id_doi\": \"10.1001/jamapsychiatry.2014.1339\", \"section_type\": \"REF\", \"source\": \"JAMA Psychiat\", \"type\": \"ref\", \"volume\": \"71\", \"year\": \"2014\"}, \"text\": \"Correcting systematic inflation in genetic association tests that consider interaction effects: application to a genome-wide association study of posttraumatic stress disorder\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 73764, \"infons\": {\"fpage\": \"11\", \"lpage\": \"18\", \"name_0\": \"surname:Manning;given-names:AK\", \"name_1\": \"surname:LaValley;given-names:M\", \"name_2\": \"surname:Liu;given-names:CT\", \"name_3\": \"surname:Rice;given-names:K\", \"name_4\": \"surname:An;given-names:P\", \"name_5\": \"surname:Liu;given-names:Y\", \"name_6\": \"surname:Miljkovic;given-names:I\", \"name_7\": \"surname:Rasmussen-Torvik;given-names:L\", \"name_8\": \"surname:Harris;given-names:TB\", \"name_9\": \"surname:Province;given-names:MA\", \"pub-id_doi\": \"10.1002/gepi.20546\", \"pub-id_pmid\": \"21181894\", \"section_type\": \"REF\", \"source\": \"Genet Epidemiol\", \"type\": \"ref\", \"volume\": \"35\", \"year\": \"2011\"}, \"text\": \"Meta-analysis of gene-environment interaction: joint estimation of SNP and SNP x environment regression coefficients\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 73881, \"infons\": {\"fpage\": \"2190\", \"lpage\": \"2191\", \"name_0\": \"surname:Willer;given-names:CJ\", \"name_1\": \"surname:Li;given-names:Y\", \"name_2\": \"surname:Abecasis;given-names:GR\", \"pub-id_doi\": \"10.1093/bioinformatics/btq340\", \"pub-id_pmid\": \"20616382\", \"section_type\": \"REF\", \"source\": \"Bioinformatics\", \"type\": \"ref\", \"volume\": \"26\", \"year\": \"2010\"}, \"text\": \"METAL: fast and efficient meta-analysis of genomewide association scans\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 73953, \"infons\": {\"fpage\": \"e1009218\", \"name_0\": \"surname:Ray;given-names:D\", \"name_1\": \"surname:Chatterjee;given-names:N\", \"pub-id_doi\": \"10.1371/journal.pgen.1009218\", \"pub-id_pmid\": \"33290408\", \"section_type\": \"REF\", \"source\": \"PLoS Genet\", \"type\": \"ref\", \"volume\": \"16\", \"year\": \"2020\"}, \"text\": \"A powerful method for pleiotropic analysis under composite null hypothesis identifies novel shared loci between Type 2 Diabetes and Prostate Cancer\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 74101, \"infons\": {\"fpage\": \"e1009584\", \"name_0\": \"surname:Ray;given-names:D\", \"name_1\": \"surname:Venkataraghavan;given-names:S\", \"name_10\": \"surname:Beaty;given-names:TH\", \"name_2\": \"surname:Zhang;given-names:W\", \"name_3\": \"surname:Leslie;given-names:EJ\", \"name_4\": \"surname:Hetmanski;given-names:JB\", \"name_5\": \"surname:Weinberg;given-names:SM\", \"name_6\": \"surname:Murray;given-names:JC\", \"name_7\": \"surname:Marazita;given-names:ML\", \"name_8\": \"surname:Ruczinski;given-names:I\", \"name_9\": \"surname:Taub;given-names:MA\", \"pub-id_doi\": \"10.1371/journal.pgen.1009584\", \"pub-id_pmid\": \"34242216\", \"section_type\": \"REF\", \"source\": \"PLoS Genet\", \"type\": \"ref\", \"volume\": \"17\", \"year\": \"2021\"}, \"text\": \"Pleiotropy method reveals genetic overlap between orofacial clefts at multiple novel loci from GWAS of multi-ethnic trios\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 74223, \"infons\": {\"fpage\": \"291\", \"lpage\": \"295\", \"name_0\": \"surname:Bulik-Sullivan;given-names:BK\", \"name_1\": \"surname:Loh;given-names:PR\", \"name_2\": \"surname:Finucane;given-names:HK\", \"name_3\": \"surname:Ripke;given-names:S\", \"name_4\": \"surname:Yang;given-names:J\", \"name_5\": \"surname:Patterson;given-names:N\", \"name_6\": \"surname:Daly;given-names:MJ\", \"name_7\": \"surname:Price;given-names:AL\", \"name_8\": \"surname:Neale;given-names:BM\", \"pub-id_doi\": \"10.1038/ng.3211\", \"pub-id_pmid\": \"25642630\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"47\", \"year\": \"2015\"}, \"text\": \"LD Score regression distinguishes confounding from polygenicity in genome-wide association studies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 74322, \"infons\": {\"fpage\": \"272\", \"lpage\": \"279\", \"name_0\": \"surname:Zheng;given-names:J\", \"name_1\": \"surname:Erzurumluoglu;given-names:AM\", \"name_2\": \"surname:Elsworth;given-names:BL\", \"name_3\": \"surname:Kemp;given-names:JP\", \"name_4\": \"surname:Howe;given-names:L\", \"name_5\": \"surname:Haycock;given-names:PC\", \"name_6\": \"surname:Hemani;given-names:G\", \"name_7\": \"surname:Tansey;given-names:K\", \"name_8\": \"surname:Laurin;given-names:C\", \"name_9\": \"surname:Early;given-names:G\", \"pub-id_doi\": \"10.1093/bioinformatics/btw613\", \"pub-id_pmid\": \"27663502\", \"section_type\": \"REF\", \"source\": \"Bioinformatics\", \"type\": \"ref\", \"volume\": \"33\", \"year\": \"2017\"}, \"text\": \"LD Hub: a centralized database and web interface to perform LD score regression that maximizes the potential of summary level GWAS data for SNP heritability and genetic correlation analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 74512, \"infons\": {\"fpage\": \"862\", \"lpage\": \"872\", \"name_0\": \"surname:Lin;given-names:DY\", \"name_1\": \"surname:Sullivan;given-names:PF\", \"pub-id_doi\": \"10.1016/j.ajhg.2009.11.001\", \"pub-id_pmid\": \"20004761\", \"section_type\": \"REF\", \"source\": \"Am J Hum Genet\", \"type\": \"ref\", \"volume\": \"85\", \"year\": \"2009\"}, \"text\": \"Meta-analysis of genome-wide association studies with overlapping subjects\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 74587, \"infons\": {\"fpage\": \"523\", \"lpage\": \"530\", \"name_0\": \"surname:Kramer;given-names:A\", \"name_1\": \"surname:Green;given-names:J\", \"name_2\": \"surname:Pollard;given-names:J;suffix:Jr\", \"name_3\": \"surname:Tugendreich;given-names:S\", \"pub-id_doi\": \"10.1093/bioinformatics/btt703\", \"pub-id_pmid\": \"24336805\", \"section_type\": \"REF\", \"source\": \"Bioinformatics\", \"type\": \"ref\", \"volume\": \"30\", \"year\": \"2014\"}, \"text\": \"Causal analysis approaches in Ingenuity Pathway Analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 74644, \"infons\": {\"fpage\": \"289\", \"lpage\": \"300\", \"name_0\": \"surname:Benjamini;given-names:Y\", \"name_1\": \"surname:Hochberg;given-names:Y\", \"section_type\": \"REF\", \"source\": \"J R Stat Soc B\", \"type\": \"ref\", \"volume\": \"57\", \"year\": \"1995\"}, \"text\": \"Controlling the false discovery rate - a practical and powerful approach to multiple testing\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 74737, \"infons\": {\"fpage\": \"1270\", \"lpage\": \"1279\", \"name_0\": \"surname:Jun;given-names:G\", \"name_1\": \"surname:Vardarajan;given-names:BN\", \"name_2\": \"surname:Buros;given-names:J\", \"name_3\": \"surname:Yu;given-names:CE\", \"name_4\": \"surname:Hawk;given-names:MV\", \"name_5\": \"surname:Dombroski;given-names:BA\", \"name_6\": \"surname:Crane;given-names:PK\", \"name_7\": \"surname:Larson;given-names:EB\", \"name_8\": \"surname:Mayeux;given-names:R\", \"pub-id_doi\": \"10.1001/archneurol.2012.2052\", \"pub-id_pmid\": \"22869155\", \"section_type\": \"REF\", \"source\": \"Arch Neurol\", \"type\": \"ref\", \"volume\": \"69\", \"year\": \"2012\"}, \"text\": \"Comprehensive search for Alzheimer disease susceptibility loci in the APOE region\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 74819, \"infons\": {\"fpage\": \"2808\", \"lpage\": \"2810\", \"name_0\": \"surname:Goedert;given-names:M\", \"name_1\": \"surname:Spillantini;given-names:MG\", \"pub-id_doi\": \"10.1093/brain/awl291\", \"pub-id_pmid\": \"17071919\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"129\", \"year\": \"2006\"}, \"text\": \"Frontotemporal lobar degeneration through loss of progranulin function\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 74890, \"infons\": {\"fpage\": \"287\", \"lpage\": \"297\", \"name_0\": \"surname:Eriksen;given-names:JL\", \"name_1\": \"surname:Mackenzie;given-names:IR\", \"pub-id_pmid\": \"17953663\", \"section_type\": \"REF\", \"source\": \"J Neurochem\", \"type\": \"ref\", \"volume\": \"104\", \"year\": \"2008\"}, \"text\": \"Progranulin: normal function and role in neurodegeneration\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 74949, \"infons\": {\"fpage\": \"1091\", \"lpage\": \"1102\", \"name_0\": \"surname:Nalls;given-names:MA\", \"name_1\": \"surname:Blauwendraat;given-names:C\", \"name_2\": \"surname:Vallerga;given-names:CL\", \"name_3\": \"surname:Heilbron;given-names:K\", \"name_4\": \"surname:Bandres-Ciga;given-names:S\", \"name_5\": \"surname:Chang;given-names:D\", \"name_6\": \"surname:Tan;given-names:M\", \"name_7\": \"surname:Kia;given-names:DA\", \"name_8\": \"surname:Noyce;given-names:AJ\", \"name_9\": \"surname:Xue;given-names:A\", \"pub-id_doi\": \"10.1016/S1474-4422(19)30320-5\", \"pub-id_pmid\": \"31701892\", \"section_type\": \"REF\", \"source\": \"Lancet Neurol\", \"type\": \"ref\", \"volume\": \"18\", \"year\": \"2019\"}, \"text\": \"Identification of novel risk loci, causal insights, and heritable risk for Parkinson\\u2019s disease: a meta-analysis of genome-wide association studies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 75098, \"infons\": {\"fpage\": \"1307\", \"lpage\": \"1316\", \"name_0\": \"surname:Crane;given-names:PK\", \"name_1\": \"surname:Trittschuh;given-names:E\", \"name_2\": \"surname:Mukherjee;given-names:S\", \"name_3\": \"surname:Saykin;given-names:AJ\", \"name_4\": \"surname:Sanders;given-names:RE\", \"name_5\": \"surname:Larson;given-names:EB\", \"name_6\": \"surname:McCurry;given-names:SM\", \"name_7\": \"surname:McCormick;given-names:W\", \"name_8\": \"surname:Bowen;given-names:JD\", \"name_9\": \"surname:Grabowski;given-names:T\", \"pub-id_doi\": \"10.1016/j.jalz.2017.04.011\", \"pub-id_pmid\": \"28623677\", \"section_type\": \"REF\", \"source\": \"Alzheimers Dement\", \"type\": \"ref\", \"volume\": \"13\", \"year\": \"2017\"}, \"text\": \"Incidence of cognitively defined late-onset Alzheimer\\u2019s dementia subgroups from a prospective cohort study\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 75207, \"infons\": {\"fpage\": \"2561\", \"lpage\": \"2575\", \"name_0\": \"surname:Dumitrescu;given-names:L\", \"name_1\": \"surname:Mahoney;given-names:ER\", \"name_2\": \"surname:Mukherjee;given-names:S\", \"name_3\": \"surname:Lee;given-names:ML\", \"name_4\": \"surname:Bush;given-names:WS\", \"name_5\": \"surname:Engelman;given-names:CD\", \"name_6\": \"surname:Lu;given-names:Q\", \"name_7\": \"surname:Fardo;given-names:DW\", \"name_8\": \"surname:Trittschuh;given-names:EH\", \"name_9\": \"surname:Mez;given-names:J\", \"pub-id_doi\": \"10.1093/brain/awaa209\", \"pub-id_pmid\": \"32844198\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"143\", \"year\": \"2020\"}, \"text\": \"Genetic variants and functional pathways associated with resilience to Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 75300, \"infons\": {\"fpage\": \"102660\", \"name_0\": \"surname:Groot;given-names:C\", \"name_1\": \"surname:Grothe;given-names:MJ\", \"name_2\": \"surname:Mukherjee;given-names:S\", \"name_3\": \"surname:Jelistratova;given-names:I\", \"name_4\": \"surname:Jansen;given-names:I\", \"name_5\": \"surname:van Loenhoud;given-names:AC\", \"name_6\": \"surname:Risacher;given-names:SL\", \"name_7\": \"surname:Saykin;given-names:AJ\", \"name_8\": \"surname:Mac Donald;given-names:CL\", \"name_9\": \"surname:Mez;given-names:J\", \"pub-id_doi\": \"10.1016/j.nicl.2021.102660\", \"pub-id_pmid\": \"33895633\", \"section_type\": \"REF\", \"source\": \"Neuroimage Clin\", \"type\": \"ref\", \"volume\": \"30\", \"year\": \"2021\"}, \"text\": \"Differential patterns of gray matter volumes and associated gene expression profiles in cognitively-defined Alzheimer\\u2019s disease subgroups\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 75440, \"infons\": {\"fpage\": \"e12201\", \"name_0\": \"surname:Uretsky;given-names:M\", \"name_1\": \"surname:Gibbons;given-names:LE\", \"name_10\": \"surname:Mez;given-names:J\", \"name_2\": \"surname:Mukherjee;given-names:S\", \"name_3\": \"surname:Trittschuh;given-names:EH\", \"name_4\": \"surname:Fardo;given-names:DW\", \"name_5\": \"surname:Boyle;given-names:PA\", \"name_6\": \"surname:Keene;given-names:CD\", \"name_7\": \"surname:Saykin;given-names:AJ\", \"name_8\": \"surname:Crane;given-names:PK\", \"name_9\": \"surname:Schneider;given-names:JA\", \"pub-id_pmid\": \"34604500\", \"section_type\": \"REF\", \"source\": \"Alzheimers Dement (N Y)\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2021\"}, \"text\": \"Longitudinal cognitive performance of Alzheimer\\u2019s disease neuropathological subtypes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 75527, \"infons\": {\"fpage\": \"147\", \"lpage\": \"155\", \"name_0\": \"surname:Gross;given-names:AL\", \"name_1\": \"surname:Hassenstab;given-names:JJ\", \"name_2\": \"surname:Johnson;given-names:SC\", \"name_3\": \"surname:Clark;given-names:LR\", \"name_4\": \"surname:Resnick;given-names:SM\", \"name_5\": \"surname:Kitner-Triolo;given-names:M\", \"name_6\": \"surname:Masters;given-names:CL\", \"name_7\": \"surname:Maruff;given-names:P\", \"name_8\": \"surname:Morris;given-names:JC\", \"name_9\": \"surname:Soldan;given-names:A\", \"pub-id_doi\": \"10.1016/j.dadm.2017.05.003\", \"pub-id_pmid\": \"28653035\", \"section_type\": \"REF\", \"source\": \"Alzheimers Dement (Amst)\", \"type\": \"ref\", \"volume\": \"8\", \"year\": \"2017\"}, \"text\": \"A classification algorithm for predicting progression from normal cognition to mild cognitive impairment across five cohorts: the preclinical AD consortium\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 75683, \"infons\": {\"fpage\": \"2541\", \"lpage\": \"2554\", \"name_0\": \"surname:Eissman;given-names:JM\", \"name_1\": \"surname:Dumitrescu;given-names:L\", \"name_2\": \"surname:Mahoney;given-names:ER\", \"name_3\": \"surname:Smith;given-names:AN\", \"name_4\": \"surname:Mukherjee;given-names:S\", \"name_5\": \"surname:Lee;given-names:ML\", \"name_6\": \"surname:Scollard;given-names:P\", \"name_7\": \"surname:Choi;given-names:SE\", \"name_8\": \"surname:Bush;given-names:WS\", \"name_9\": \"surname:Engelman;given-names:CD\", \"pub-id_doi\": \"10.1093/brain/awac177\", \"pub-id_pmid\": \"35552371\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"145\", \"year\": \"2022\"}, \"text\": \"Sex differences in the genetic architecture of cognitive resilience to Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 75776, \"infons\": {\"fpage\": \"e28745\", \"name_0\": \"surname:Kang;given-names:WS\", \"name_1\": \"surname:Lee;given-names:SM\", \"name_2\": \"surname:Hwang;given-names:D\", \"name_3\": \"surname:Park;given-names:HJ\", \"name_4\": \"surname:Kim;given-names:JW\", \"pub-id_doi\": \"10.1097/MD.0000000000028745\", \"pub-id_pmid\": \"35119028\", \"section_type\": \"REF\", \"source\": \"Medicine (Baltimore)\", \"type\": \"ref\", \"volume\": \"101\", \"year\": \"2022\"}, \"text\": \"Association between Unc-51-like autophagy activating kinase 2 gene polymorphisms and schizophrenia in the Korean population\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 75900, \"infons\": {\"fpage\": \"135087\", \"name_0\": \"surname:Huang;given-names:Y\", \"name_1\": \"surname:Liao;given-names:X\", \"name_2\": \"surname:Luo;given-names:J\", \"name_3\": \"surname:Liu;given-names:H\", \"name_4\": \"surname:Zhong;given-names:S\", \"name_5\": \"surname:Chen;given-names:J\", \"pub-id_doi\": \"10.1016/j.neulet.2020.135087\", \"pub-id_pmid\": \"32534097\", \"section_type\": \"REF\", \"source\": \"Neurosci Lett\", \"type\": \"ref\", \"volume\": \"735\", \"year\": \"2020\"}, \"text\": \"Expression of circular RNAs in the vascular dementia rats\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 75958, \"infons\": {\"fpage\": \"4444\", \"name_0\": \"surname:Lee;given-names:A\", \"name_1\": \"surname:Kondapalli;given-names:C\", \"name_2\": \"surname:Virga;given-names:DM\", \"name_3\": \"surname:Lewis;given-names:TL;suffix:Jr\", \"name_4\": \"surname:Koo;given-names:SY\", \"name_5\": \"surname:Ashok;given-names:A\", \"name_6\": \"surname:Mairet-Coello;given-names:G\", \"name_7\": \"surname:Herzig;given-names:S\", \"name_8\": \"surname:Foretz;given-names:M\", \"name_9\": \"surname:Viollet;given-names:B\", \"pub-id_doi\": \"10.1038/s41467-022-32130-5\", \"pub-id_pmid\": \"35915085\", \"section_type\": \"REF\", \"source\": \"Nat Commun\", \"type\": \"ref\", \"volume\": \"13\", \"year\": \"2022\"}, \"text\": \"Abeta42 oligomers trigger synaptic loss through CAMKK2-AMPK-dependent effectors coordinating mitochondrial fission and mitophagy\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 76087, \"infons\": {\"fpage\": \"436\", \"lpage\": \"441\", \"name_0\": \"surname:Naj;given-names:AC\", \"name_1\": \"surname:Jun;given-names:G\", \"name_2\": \"surname:Beecham;given-names:GW\", \"name_3\": \"surname:Wang;given-names:LS\", \"name_4\": \"surname:Vardarajan;given-names:BN\", \"name_5\": \"surname:Buros;given-names:J\", \"name_6\": \"surname:Gallins;given-names:PJ\", \"name_7\": \"surname:Buxbaum;given-names:JD\", \"name_8\": \"surname:Jarvik;given-names:GP\", \"name_9\": \"surname:Crane;given-names:PK\", \"pub-id_doi\": \"10.1038/ng.801\", \"pub-id_pmid\": \"21460841\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"43\", \"year\": \"2011\"}, \"text\": \"Common variants at MS4A4/MS4A6E, CD2AP, CD33 and EPHA1 are associated with late-onset Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 76195, \"infons\": {\"fpage\": \"233\", \"lpage\": \"236\", \"name_0\": \"surname:Shimohama;given-names:S\", \"name_1\": \"surname:Chachin;given-names:M\", \"name_2\": \"surname:Taniguchi;given-names:T\", \"name_3\": \"surname:Hidaka;given-names:H\", \"name_4\": \"surname:Kimura;given-names:J\", \"pub-id_doi\": \"10.1016/0006-8993(96)00070-4\", \"pub-id_pmid\": \"8738246\", \"section_type\": \"REF\", \"source\": \"Brain Res\", \"type\": \"ref\", \"volume\": \"716\", \"year\": \"1996\"}, \"text\": \"Changes of neurocalcin, a calcium-binding protein, in the brain of patients with Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 76298, \"infons\": {\"fpage\": \"48\", \"name_0\": \"surname:Miller;given-names:JA\", \"name_1\": \"surname:Woltjer;given-names:RL\", \"name_2\": \"surname:Goodenbour;given-names:JM\", \"name_3\": \"surname:Horvath;given-names:S\", \"name_4\": \"surname:Geschwind;given-names:DH\", \"pub-id_doi\": \"10.1186/gm452\", \"pub-id_pmid\": \"23705665\", \"section_type\": \"REF\", \"source\": \"Genome Med\", \"type\": \"ref\", \"volume\": \"5\", \"year\": \"2013\"}, \"text\": \"Genes and pathways underlying regional and cell type changes in Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 76384, \"infons\": {\"fpage\": \"1211\", \"lpage\": \"1221\", \"name_0\": \"surname:Ma;given-names:WW\", \"name_1\": \"surname:Ding;given-names:BJ\", \"name_2\": \"surname:Yuan;given-names:LH\", \"name_3\": \"surname:Zhao;given-names:L\", \"name_4\": \"surname:Yu;given-names:HL\", \"name_5\": \"surname:Xi;given-names:YD\", \"name_6\": \"surname:Xiao;given-names:R\", \"pub-id_doi\": \"10.4314/ahs.v17i4.32\", \"pub-id_pmid\": \"29937895\", \"section_type\": \"REF\", \"source\": \"Afr Health Sci\", \"type\": \"ref\", \"volume\": \"17\", \"year\": \"2017\"}, \"text\": \"Neurocalcin-delta: a potential memory-related factor in hippocampus of obese rats induced by high-fat diet\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 76491, \"infons\": {\"fpage\": \"37\", \"lpage\": \"41\", \"name_0\": \"surname:Van Damme;given-names:P\", \"name_1\": \"surname:Van Hoecke;given-names:A\", \"name_2\": \"surname:Lambrechts;given-names:D\", \"name_3\": \"surname:Vanacker;given-names:P\", \"name_4\": \"surname:Bogaert;given-names:E\", \"name_5\": \"surname:van Swieten;given-names:J\", \"name_6\": \"surname:Carmeliet;given-names:P\", \"name_7\": \"surname:Van Den Bosch;given-names:L\", \"name_8\": \"surname:Robberecht;given-names:W\", \"pub-id_doi\": \"10.1083/jcb.200712039\", \"pub-id_pmid\": \"18378771\", \"section_type\": \"REF\", \"source\": \"J Cell Biol\", \"type\": \"ref\", \"volume\": \"181\", \"year\": \"2008\"}, \"text\": \"Progranulin functions as a neurotrophic factor to regulate neurite outgrowth and enhance neuronal survival\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 76598, \"infons\": {\"fpage\": \"54\", \"issue\": \"Suppl 1\", \"lpage\": \"70\", \"name_0\": \"surname:Mackenzie;given-names:IR\", \"name_1\": \"surname:Neumann;given-names:M\", \"pub-id_doi\": \"10.1111/jnc.13588\", \"pub-id_pmid\": \"27306735\", \"section_type\": \"REF\", \"source\": \"J Neurochem\", \"type\": \"ref\", \"volume\": \"138\", \"year\": \"2016\"}, \"text\": \"Molecular neuropathology of frontotemporal dementia: insights into disease mechanisms from postmortem studies\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 76708, \"infons\": {\"fpage\": \"3631\", \"lpage\": \"3642\", \"name_0\": \"surname:Rademakers;given-names:R\", \"name_1\": \"surname:Eriksen;given-names:JL\", \"name_2\": \"surname:Baker;given-names:M\", \"name_3\": \"surname:Robinson;given-names:T\", \"name_4\": \"surname:Ahmed;given-names:Z\", \"name_5\": \"surname:Lincoln;given-names:SJ\", \"name_6\": \"surname:Finch;given-names:N\", \"name_7\": \"surname:Rutherford;given-names:NJ\", \"name_8\": \"surname:Crook;given-names:RJ\", \"name_9\": \"surname:Josephs;given-names:KA\", \"pub-id_doi\": \"10.1093/hmg/ddn257\", \"pub-id_pmid\": \"18723524\", \"section_type\": \"REF\", \"source\": \"Hum Mol Genet\", \"type\": \"ref\", \"volume\": \"17\", \"year\": \"2008\"}, \"text\": \"Common variation in the miR-659 binding-site of GRN is a major risk factor for TDP43-positive frontotemporal dementia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 76826, \"infons\": {\"fpage\": \"1871\", \"lpage\": \"1878\", \"name_0\": \"surname:Nicholson;given-names:AM\", \"name_1\": \"surname:Finch;given-names:NA\", \"name_10\": \"surname:Rademakers;given-names:R\", \"name_2\": \"surname:Thomas;given-names:CS\", \"name_3\": \"surname:Wojtas;given-names:A\", \"name_4\": \"surname:Rutherford;given-names:NJ\", \"name_5\": \"surname:Mielke;given-names:MM\", \"name_6\": \"surname:Roberts;given-names:RO\", \"name_7\": \"surname:Boeve;given-names:BF\", \"name_8\": \"surname:Knopman;given-names:DS\", \"name_9\": \"surname:Petersen;given-names:RC\", \"pub-id_doi\": \"10.1212/WNL.0000000000000445\", \"pub-id_pmid\": \"24771538\", \"section_type\": \"REF\", \"source\": \"Neurology\", \"type\": \"ref\", \"volume\": \"82\", \"year\": \"2014\"}, \"text\": \"Progranulin protein levels are differently regulated in plasma and CSF\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 76897, \"infons\": {\"fpage\": \"1187\", \"lpage\": \"1195\", \"name_0\": \"surname:Xu;given-names:HM\", \"name_1\": \"surname:Tan;given-names:L\", \"name_10\": \"surname:Yu;given-names:JT\", \"name_2\": \"surname:Wan;given-names:Y\", \"name_3\": \"surname:Tan;given-names:MS\", \"name_4\": \"surname:Zhang;given-names:W\", \"name_5\": \"surname:Zheng;given-names:ZJ\", \"name_6\": \"surname:Kong;given-names:LL\", \"name_7\": \"surname:Wang;given-names:ZX\", \"name_8\": \"surname:Jiang;given-names:T\", \"name_9\": \"surname:Tan;given-names:L\", \"pub-id_doi\": \"10.1007/s12035-016-9698-4\", \"pub-id_pmid\": \"26820675\", \"section_type\": \"REF\", \"source\": \"Mol Neurobiol\", \"type\": \"ref\", \"volume\": \"54\", \"year\": \"2017\"}, \"text\": \"PGRN is associated with late-onset Alzheimer\\u2019s disease: a case-control replication study and meta-analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 77006, \"infons\": {\"fpage\": \"2458\", \"lpage\": \"2467\", \"name_0\": \"surname:Vardarajan;given-names:BN\", \"name_1\": \"surname:Reyes-Dumeyer;given-names:D\", \"name_2\": \"surname:Piriz;given-names:AL\", \"name_3\": \"surname:Lantigua;given-names:RA\", \"name_4\": \"surname:Medrano;given-names:M\", \"name_5\": \"surname:Rivera;given-names:D\", \"name_6\": \"surname:Jimenez-Velazquez;given-names:IZ\", \"name_7\": \"surname:Martin;given-names:E\", \"name_8\": \"surname:Pericak-Vance;given-names:MA\", \"name_9\": \"surname:Bush;given-names:W\", \"pub-id_doi\": \"10.1002/alz.12567\", \"pub-id_pmid\": \"35258170\", \"section_type\": \"REF\", \"source\": \"Alzheimers Dement\", \"type\": \"ref\", \"volume\": \"18\", \"year\": \"2022\"}, \"text\": \"Progranulin mutations in clinical and neuropathological Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 77084, \"infons\": {\"fpage\": \"653\", \"lpage\": \"660\", \"name_0\": \"surname:Nag;given-names:S\", \"name_1\": \"surname:Yu;given-names:L\", \"name_2\": \"surname:Wilson;given-names:RS\", \"name_3\": \"surname:Chen;given-names:EY\", \"name_4\": \"surname:Bennett;given-names:DA\", \"name_5\": \"surname:Schneider;given-names:JA\", \"pub-id_doi\": \"10.1212/WNL.0000000000003610\", \"pub-id_pmid\": \"28087828\", \"section_type\": \"REF\", \"source\": \"Neurology\", \"type\": \"ref\", \"volume\": \"88\", \"year\": \"2017\"}, \"text\": \"TDP-43 pathology and memory impairment in elders without pathologic diagnoses of AD or FTLD\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 77176, \"infons\": {\"fpage\": \"1503\", \"lpage\": \"1527\", \"name_0\": \"surname:Nelson;given-names:PT\", \"name_1\": \"surname:Dickson;given-names:DW\", \"name_2\": \"surname:Trojanowski;given-names:JQ\", \"name_3\": \"surname:Jack;given-names:CR\", \"name_4\": \"surname:Boyle;given-names:PA\", \"name_5\": \"surname:Arfanakis;given-names:K\", \"name_6\": \"surname:Rademakers;given-names:R\", \"name_7\": \"surname:Alafuzoff;given-names:I\", \"name_8\": \"surname:Attems;given-names:J\", \"name_9\": \"surname:Brayne;given-names:C\", \"pub-id_doi\": \"10.1093/brain/awz099\", \"pub-id_pmid\": \"31039256\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"142\", \"year\": \"2019\"}, \"text\": \"Limbic-predominant age-related TDP-43 encephalopathy (LATE): consensus working group report\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 77268, \"infons\": {\"fpage\": \"641\", \"lpage\": \"652\", \"name_0\": \"surname:Rhinn;given-names:H\", \"name_1\": \"surname:Tatton;given-names:N\", \"name_2\": \"surname:McCaughey;given-names:S\", \"name_3\": \"surname:Kurnellas;given-names:M\", \"name_4\": \"surname:Rosenthal;given-names:A\", \"pub-id_doi\": \"10.1016/j.tips.2021.11.015\", \"pub-id_pmid\": \"35039149\", \"section_type\": \"REF\", \"source\": \"Trends Pharmacol Sci\", \"type\": \"ref\", \"volume\": \"43\", \"year\": \"2022\"}, \"text\": \"Progranulin as a therapeutic target in neurodegenerative diseases\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 77334, \"infons\": {\"fpage\": \"e10153\", \"name_0\": \"surname:Gomez Ravetti;given-names:M\", \"name_1\": \"surname:Rosso;given-names:OA\", \"name_2\": \"surname:Berretta;given-names:R\", \"name_3\": \"surname:Moscato;given-names:P\", \"pub-id_doi\": \"10.1371/journal.pone.0010153\", \"pub-id_pmid\": \"20405009\", \"section_type\": \"REF\", \"source\": \"PLoS ONE\", \"type\": \"ref\", \"volume\": \"5\", \"year\": \"2010\"}, \"text\": \"Uncovering molecular biomarkers that correlate cognitive decline with the changes of hippocampus\\u2019 gene expression profiles in Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 77484, \"infons\": {\"fpage\": \"1681\", \"lpage\": \"1694\", \"name_0\": \"surname:Silver;given-names:M\", \"name_1\": \"surname:Janousova;given-names:E\", \"name_2\": \"surname:Hua;given-names:X\", \"name_3\": \"surname:Thompson;given-names:PM\", \"name_4\": \"surname:Montana;given-names:G\", \"pub-id_doi\": \"10.1016/j.neuroimage.2012.08.002\", \"pub-id_pmid\": \"22982105\", \"section_type\": \"REF\", \"source\": \"Neuroimage\", \"type\": \"ref\", \"volume\": \"63\", \"year\": \"2012\"}, \"text\": \"Identification of gene pathways implicated in Alzheimer\\u2019s disease using longitudinal imaging phenotypes with sparse regression\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 77613, \"infons\": {\"fpage\": \"700\", \"lpage\": \"716\", \"name_0\": \"surname:Vounou;given-names:M\", \"name_1\": \"surname:Janousova;given-names:E\", \"name_2\": \"surname:Wolz;given-names:R\", \"name_3\": \"surname:Stein;given-names:JL\", \"name_4\": \"surname:Thompson;given-names:PM\", \"name_5\": \"surname:Rueckert;given-names:D\", \"name_6\": \"surname:Montana;given-names:G\", \"pub-id_doi\": \"10.1016/j.neuroimage.2011.12.029\", \"pub-id_pmid\": \"22209813\", \"section_type\": \"REF\", \"source\": \"Neuroimage\", \"type\": \"ref\", \"volume\": \"60\", \"year\": \"2012\"}, \"text\": \"Sparse reduced-rank regression detects genetic associations with voxel-wise longitudinal phenotypes in Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 77738, \"infons\": {\"fpage\": \"309\", \"lpage\": \"321\", \"name_0\": \"surname:Yan;given-names:Q\", \"name_1\": \"surname:Nho;given-names:K\", \"name_2\": \"surname:Del-Aguila;given-names:JL\", \"name_3\": \"surname:Wang;given-names:X\", \"name_4\": \"surname:Risacher;given-names:SL\", \"name_5\": \"surname:Fan;given-names:KH\", \"name_6\": \"surname:Snitz;given-names:BE\", \"name_7\": \"surname:Aizenstein;given-names:HJ\", \"name_8\": \"surname:Mathis;given-names:CA\", \"name_9\": \"surname:Lopez;given-names:OL\", \"pub-id_doi\": \"10.1038/s41380-018-0246-7\", \"pub-id_pmid\": \"30361487\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"26\", \"year\": \"2021\"}, \"text\": \"Genome-wide association study of brain amyloid deposition as measured by Pittsburgh Compound-B (PiB)-PET imaging\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 77851, \"infons\": {\"fpage\": \"2498\", \"lpage\": \"2513\", \"name_0\": \"surname:Kim;given-names:BM\", \"name_1\": \"surname:You;given-names:MH\", \"name_2\": \"surname:Chen;given-names:CH\", \"name_3\": \"surname:Suh;given-names:J\", \"name_4\": \"surname:Tanzi;given-names:RE\", \"name_5\": \"surname:Ho Lee;given-names:T\", \"pub-id_pmid\": \"27094130\", \"section_type\": \"REF\", \"source\": \"Hum Mol Genet\", \"type\": \"ref\", \"volume\": \"25\", \"year\": \"2016\"}, \"text\": \"Inhibition of death-associated protein kinase 1 attenuates the phosphorylation and amyloidogenic processing of amyloid precursor protein\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 77988, \"infons\": {\"fpage\": \"1328\", \"lpage\": \"1342\", \"name_0\": \"surname:Wirz;given-names:KT\", \"name_1\": \"surname:Bossers;given-names:K\", \"name_2\": \"surname:Stargardt;given-names:A\", \"name_3\": \"surname:Kamphuis;given-names:W\", \"name_4\": \"surname:Swaab;given-names:DF\", \"name_5\": \"surname:Hol;given-names:EM\", \"name_6\": \"surname:Verhaagen;given-names:J\", \"pub-id_doi\": \"10.1016/j.neurobiolaging.2012.11.008\", \"pub-id_pmid\": \"23245294\", \"section_type\": \"REF\", \"source\": \"Neurobiol Aging\", \"type\": \"ref\", \"volume\": \"34\", \"year\": \"2013\"}, \"text\": \"Cortical beta amyloid protein triggers an immune response, but no synaptic changes in the APPswe/PS1dE9 Alzheimer\\u2019s disease mouse model\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78126, \"infons\": {\"fpage\": \"678\", \"lpage\": \"692\", \"name_0\": \"surname:Linker;given-names:RA\", \"name_1\": \"surname:Lee;given-names:DH\", \"name_2\": \"surname:Ryan;given-names:S\", \"name_3\": \"surname:van Dam;given-names:AM\", \"name_4\": \"surname:Conrad;given-names:R\", \"name_5\": \"surname:Bista;given-names:P\", \"name_6\": \"surname:Zeng;given-names:W\", \"name_7\": \"surname:Hronowsky;given-names:X\", \"name_8\": \"surname:Buko;given-names:A\", \"name_9\": \"surname:Chollate;given-names:S\", \"pub-id_doi\": \"10.1093/brain/awq386\", \"pub-id_pmid\": \"21354971\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"134\", \"year\": \"2011\"}, \"text\": \"Fumaric acid esters exert neuroprotective effects in neuroinflammation via activation of the Nrf2 antioxidant pathway\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78244, \"infons\": {\"fpage\": \"11624\", \"name_0\": \"surname:Kobayashi;given-names:EH\", \"name_1\": \"surname:Suzuki;given-names:T\", \"name_10\": \"surname:Yamamoto;given-names:M\", \"name_2\": \"surname:Funayama;given-names:R\", \"name_3\": \"surname:Nagashima;given-names:T\", \"name_4\": \"surname:Hayashi;given-names:M\", \"name_5\": \"surname:Sekine;given-names:H\", \"name_6\": \"surname:Tanaka;given-names:N\", \"name_7\": \"surname:Moriguchi;given-names:T\", \"name_8\": \"surname:Motohashi;given-names:H\", \"name_9\": \"surname:Nakayama;given-names:K\", \"pub-id_doi\": \"10.1038/ncomms11624\", \"pub-id_pmid\": \"27211851\", \"section_type\": \"REF\", \"source\": \"Nat Commun\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2016\"}, \"text\": \"Nrf2 suppresses macrophage inflammatory response by blocking proinflammatory cytokine transcription\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78344, \"infons\": {\"fpage\": \"1112\", \"lpage\": \"1121\", \"name_0\": \"surname:Lee;given-names:JJ\", \"name_1\": \"surname:Wedow;given-names:R\", \"name_2\": \"surname:Okbay;given-names:A\", \"name_3\": \"surname:Kong;given-names:E\", \"name_4\": \"surname:Maghzian;given-names:O\", \"name_5\": \"surname:Zacher;given-names:M\", \"name_6\": \"surname:Nguyen-Viet;given-names:TA\", \"name_7\": \"surname:Bowers;given-names:P\", \"name_8\": \"surname:Sidorenko;given-names:J\", \"name_9\": \"surname:Karlsson Linner;given-names:R\", \"pub-id_doi\": \"10.1038/s41588-018-0147-3\", \"pub-id_pmid\": \"30038396\", \"section_type\": \"REF\", \"source\": \"Nat Genet\", \"type\": \"ref\", \"volume\": \"50\", \"year\": \"2018\"}, \"text\": \"Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 78474, \"infons\": {\"fpage\": \"169\", \"lpage\": \"181\", \"name_0\": \"surname:Hill;given-names:WD\", \"name_1\": \"surname:Marioni;given-names:RE\", \"name_2\": \"surname:Maghzian;given-names:O\", \"name_3\": \"surname:Ritchie;given-names:SJ\", \"name_4\": \"surname:Hagenaars;given-names:SP\", \"name_5\": \"surname:McIntosh;given-names:AM\", \"name_6\": \"surname:Gale;given-names:CR\", \"name_7\": \"surname:Davies;given-names:G\", \"name_8\": \"surname:Deary;given-names:IJ\", \"pub-id_doi\": \"10.1038/s41380-017-0001-5\", \"pub-id_pmid\": \"29326435\", \"section_type\": \"REF\", \"source\": \"Mol Psychiatry\", \"type\": \"ref\", \"volume\": \"24\", \"year\": \"2019\"}, \"text\": \"A combined analysis of genetically correlated traits identifies 187 loci and a role for neurogenesis and myelination in intelligence\", \"sentences\": [], \"annotations\": [], \"relations\": []}], \"annotations\": [], \"relations\": []}]}]\n",
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading weights: 100%|| 770/770 [00:00<00:00, 1530.11it/s, Materializing param=model.encoder_input_proj.2.1.weight]                                                 \n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    }
   ],
   "source": [
    "pmcid = \"PMC10286470\"\n",
    "file_name = \"papers/s13024-023-00633-4.pdf\"  # Can be a local path or a URL\n",
    "df_lst = extract_tables_lst_from_paper(pmcid, file_name)  \n",
    "for i in [2, 3]:\n",
    "    df_lst[i-1].to_csv(f\"tables/{file_name.split('/')[-1].replace('.pdf', '')}_table_{i}.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "72bce5d8",
   "metadata": {},
   "outputs": [],
   "source": [
    "# pmcid = \"PMC9945061\"\n",
    "# file_name = \"PMID36809323_table_3.pdf\"\n",
    "# df_lst = extract_tables_lst_from_paper(pmcid, file_name)  \n",
    "# for i in [3]:\n",
    "#     df_lst[i-3].to_csv(f\"tables/{file_name.split('/')[-1].replace('.pdf', '')}_table_{i}.csv\", index=False)\n",
    "# problem in PMC api"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "63c135bb",
   "metadata": {},
   "source": [
    "Try to map columns to reference columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "0755935e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>column</th>\n",
       "      <th>description</th>\n",
       "      <th>is_numeric</th>\n",
       "      <th>column_with_context</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>SNP</td>\n",
       "      <td>Variant identifier, or snp idenifier, or chr:p...</td>\n",
       "      <td>False</td>\n",
       "      <td>SNP: Variant identifier, or snp idenifier, or ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Chr</td>\n",
       "      <td>Genomic chromosome identifier. Keywords: CHR, ...</td>\n",
       "      <td>False</td>\n",
       "      <td>Chr: Genomic chromosome identifier. Keywords: ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Position</td>\n",
       "      <td>Genomic coordinate location. Keywords: BP, POS...</td>\n",
       "      <td>False</td>\n",
       "      <td>Position: Genomic coordinate location. Keyword...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Locus</td>\n",
       "      <td>Specific named genomic region or hit identity....</td>\n",
       "      <td>False</td>\n",
       "      <td>Locus: Specific named genomic region or hit id...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Reported gene</td>\n",
       "      <td>Official gene symbol or name used in gene-base...</td>\n",
       "      <td>False</td>\n",
       "      <td>Reported gene: Official gene symbol or name us...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>SNV</td>\n",
       "      <td>The count or specific list of variants in a ge...</td>\n",
       "      <td>False</td>\n",
       "      <td>SNV: The count or specific list of variants in...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Population</td>\n",
       "      <td>The ancestral group or ethnicity of the partic...</td>\n",
       "      <td>False</td>\n",
       "      <td>Population: The ancestral group or ethnicity o...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Cohort</td>\n",
       "      <td>The specific study or database name. Keywords:...</td>\n",
       "      <td>False</td>\n",
       "      <td>Cohort: The specific study or database name. K...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Sample size</td>\n",
       "      <td>Total number of subjects or observations. Keyw...</td>\n",
       "      <td>True</td>\n",
       "      <td>Sample size: Total number of subjects or obser...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Sample Information</td>\n",
       "      <td>Metadata regarding where sample data was extra...</td>\n",
       "      <td>False</td>\n",
       "      <td>Sample Information: Metadata regarding where s...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Subset Analyzed</td>\n",
       "      <td>The subgroup or demographic filter applied to ...</td>\n",
       "      <td>False</td>\n",
       "      <td>Subset Analyzed: The subgroup or demographic f...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Phenotype</td>\n",
       "      <td>The trait or biological marker being measured....</td>\n",
       "      <td>False</td>\n",
       "      <td>Phenotype: The trait or biological marker bein...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Association type</td>\n",
       "      <td>The statistical framework or study design. Key...</td>\n",
       "      <td>False</td>\n",
       "      <td>Association type: The statistical framework or...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>RA</td>\n",
       "      <td>The allele associated with the effect or risk....</td>\n",
       "      <td>False</td>\n",
       "      <td>RA: The allele associated with the effect or r...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>AF</td>\n",
       "      <td>Frequency of the reported allele in the sample...</td>\n",
       "      <td>True</td>\n",
       "      <td>AF: Frequency of the reported allele in the sa...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>P-value</td>\n",
       "      <td>The statistical significance of the associatio...</td>\n",
       "      <td>True</td>\n",
       "      <td>P-value: The statistical significance of the a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Effect Size</td>\n",
       "      <td>The magnitude and direction of the association...</td>\n",
       "      <td>True</td>\n",
       "      <td>Effect Size: The magnitude and direction of th...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Confidence Interval</td>\n",
       "      <td>The uncertainty range of the effect size. Keyw...</td>\n",
       "      <td>True</td>\n",
       "      <td>Confidence Interval: The uncertainty range of ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Imputation</td>\n",
       "      <td>The software or panel used to predict missing ...</td>\n",
       "      <td>False</td>\n",
       "      <td>Imputation: The software or panel used to pred...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Nearest gene</td>\n",
       "      <td>The gene physically closest to the lead SNP. K...</td>\n",
       "      <td>False</td>\n",
       "      <td>Nearest gene: The gene physically closest to t...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>Most severe consequence</td>\n",
       "      <td>Functional impact of the variant on the protei...</td>\n",
       "      <td>False</td>\n",
       "      <td>Most severe consequence: Functional impact of ...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                     column  \\\n",
       "0                       SNP   \n",
       "1                       Chr   \n",
       "2                  Position   \n",
       "3                     Locus   \n",
       "4             Reported gene   \n",
       "5                       SNV   \n",
       "6                Population   \n",
       "7                    Cohort   \n",
       "8               Sample size   \n",
       "9        Sample Information   \n",
       "10          Subset Analyzed   \n",
       "11                Phenotype   \n",
       "12         Association type   \n",
       "13                       RA   \n",
       "14                       AF   \n",
       "15                  P-value   \n",
       "16              Effect Size   \n",
       "17      Confidence Interval   \n",
       "18               Imputation   \n",
       "19             Nearest gene   \n",
       "20  Most severe consequence   \n",
       "\n",
       "                                          description  is_numeric  \\\n",
       "0   Variant identifier, or snp idenifier, or chr:p...       False   \n",
       "1   Genomic chromosome identifier. Keywords: CHR, ...       False   \n",
       "2   Genomic coordinate location. Keywords: BP, POS...       False   \n",
       "3   Specific named genomic region or hit identity....       False   \n",
       "4   Official gene symbol or name used in gene-base...       False   \n",
       "5   The count or specific list of variants in a ge...       False   \n",
       "6   The ancestral group or ethnicity of the partic...       False   \n",
       "7   The specific study or database name. Keywords:...       False   \n",
       "8   Total number of subjects or observations. Keyw...        True   \n",
       "9   Metadata regarding where sample data was extra...       False   \n",
       "10  The subgroup or demographic filter applied to ...       False   \n",
       "11  The trait or biological marker being measured....       False   \n",
       "12  The statistical framework or study design. Key...       False   \n",
       "13  The allele associated with the effect or risk....       False   \n",
       "14  Frequency of the reported allele in the sample...        True   \n",
       "15  The statistical significance of the associatio...        True   \n",
       "16  The magnitude and direction of the association...        True   \n",
       "17  The uncertainty range of the effect size. Keyw...        True   \n",
       "18  The software or panel used to predict missing ...       False   \n",
       "19  The gene physically closest to the lead SNP. K...       False   \n",
       "20  Functional impact of the variant on the protei...       False   \n",
       "\n",
       "                                  column_with_context  \n",
       "0   SNP: Variant identifier, or snp idenifier, or ...  \n",
       "1   Chr: Genomic chromosome identifier. Keywords: ...  \n",
       "2   Position: Genomic coordinate location. Keyword...  \n",
       "3   Locus: Specific named genomic region or hit id...  \n",
       "4   Reported gene: Official gene symbol or name us...  \n",
       "5   SNV: The count or specific list of variants in...  \n",
       "6   Population: The ancestral group or ethnicity o...  \n",
       "7   Cohort: The specific study or database name. K...  \n",
       "8   Sample size: Total number of subjects or obser...  \n",
       "9   Sample Information: Metadata regarding where s...  \n",
       "10  Subset Analyzed: The subgroup or demographic f...  \n",
       "11  Phenotype: The trait or biological marker bein...  \n",
       "12  Association type: The statistical framework or...  \n",
       "13  RA: The allele associated with the effect or r...  \n",
       "14  AF: Frequency of the reported allele in the sa...  \n",
       "15  P-value: The statistical significance of the a...  \n",
       "16  Effect Size: The magnitude and direction of th...  \n",
       "17  Confidence Interval: The uncertainty range of ...  \n",
       "18  Imputation: The software or panel used to pred...  \n",
       "19  Nearest gene: The gene physically closest to t...  \n",
       "20  Most severe consequence: Functional impact of ...  "
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Now try to map the columns with the actual col in advp\n",
    "referencing_cols = pd.read_csv(\"Rules for harmonizing ADVP papers - Main cols.csv\")\n",
    "referencing_cols[\"column_with_context\"] = referencing_cols.apply(lambda x: x[\"column\"] if pd.isna(x[\"description\"]) else x[\"column\"] + \": \" + x[\"description\"], axis = 1)\n",
    "referencing_cols"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2cf842b5",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading weights: 100%|| 199/199 [00:00<00:00, 2024.74it/s, Materializing param=pooler.dense.weight]                               \n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "BertModel(\n",
       "  (embeddings): BertEmbeddings(\n",
       "    (word_embeddings): Embedding(30522, 768, padding_idx=0)\n",
       "    (position_embeddings): Embedding(512, 768)\n",
       "    (token_type_embeddings): Embedding(2, 768)\n",
       "    (LayerNorm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
       "    (dropout): Dropout(p=0.1, inplace=False)\n",
       "  )\n",
       "  (encoder): BertEncoder(\n",
       "    (layer): ModuleList(\n",
       "      (0-11): 12 x BertLayer(\n",
       "        (attention): BertAttention(\n",
       "          (self): BertSelfAttention(\n",
       "            (query): Linear(in_features=768, out_features=768, bias=True)\n",
       "            (key): Linear(in_features=768, out_features=768, bias=True)\n",
       "            (value): Linear(in_features=768, out_features=768, bias=True)\n",
       "            (dropout): Dropout(p=0.1, inplace=False)\n",
       "          )\n",
       "          (output): BertSelfOutput(\n",
       "            (dense): Linear(in_features=768, out_features=768, bias=True)\n",
       "            (LayerNorm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
       "            (dropout): Dropout(p=0.1, inplace=False)\n",
       "          )\n",
       "        )\n",
       "        (intermediate): BertIntermediate(\n",
       "          (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
       "          (intermediate_act_fn): GELUActivation()\n",
       "        )\n",
       "        (output): BertOutput(\n",
       "          (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
       "          (LayerNorm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
       "          (dropout): Dropout(p=0.1, inplace=False)\n",
       "        )\n",
       "      )\n",
       "    )\n",
       "  )\n",
       "  (pooler): BertPooler(\n",
       "    (dense): Linear(in_features=768, out_features=768, bias=True)\n",
       "    (activation): Tanh()\n",
       "  )\n",
       ")"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# NOTE: before working in the main cell for mapping columns, write inference code here + setting up models here\n",
    "def create_embeddings_from_model(sentences, model, tokenizer):\n",
    "    input = tokenizer(sentences, padding=True, truncation=True, return_tensors='pt')\n",
    "\n",
    "    # get token embeddings\n",
    "    with torch.no_grad():\n",
    "        output = model(**input)\n",
    "    token_embeddings = output[0]\n",
    "\n",
    "    # extract mask and mean pooling for sentence embeddings\n",
    "    input_mask_expanded = input['attention_mask'].unsqueeze(-1).expand(token_embeddings.size()).float()\n",
    "    sentence_embeddings = torch.sum(token_embeddings * input_mask_expanded, 1) / torch.clamp(input_mask_expanded.sum(1), min=1e-9)\n",
    "\n",
    "    # final normalization\n",
    "    sentence_embeddings = F.normalize(sentence_embeddings, p=2, dim=1)\n",
    "\n",
    "    return sentence_embeddings\n",
    "\n",
    "# def reranking_from_model(query, candidates, candidates_scores, batch_size = 16):\n",
    "#     pairs = [[query, c] for c in candidates]\n",
    "#     all_scores = []\n",
    "\n",
    "#     with torch.no_grad():\n",
    "#         for i in range(0, len(pairs), batch_size):\n",
    "#             batch = pairs[i:i+batch_size]\n",
    "#             inputs = reranker_tokenizer(\n",
    "#                 batch,\n",
    "#                 padding=True,\n",
    "#                 truncation=True,\n",
    "#                 max_length=512,\n",
    "#                 return_tensors=\"pt\",\n",
    "#             )\n",
    "\n",
    "#             logits = reranker_model(**inputs).logits.view(-1)\n",
    "#             scores = torch.sigmoid(logits).detach().cpu().numpy()\n",
    "#             all_scores.extend(scores.tolist())\n",
    "\n",
    "#     best_idx = int(np.argmax(np.asarray(all_scores, dtype=float)))\n",
    "#     return candidates[best_idx], candidates_scores[best_idx]\n",
    "\n",
    "# def reranking_from_llm(query, query_details, candidates, candidates_details):\n",
    "#     prompt = f\"\"\"\n",
    "#     You are matching one source table column to the single best target schema field.\n",
    "\n",
    "#     Rules:\n",
    "#     - Pick exactly one candidate number that represents the same concept as the source column.\n",
    "#     - Rely on meaning (units, value types, examples), not just wording.\n",
    "#     - If no perfect match, choose the closest and mention uncertainty briefly.\n",
    "\n",
    "#     Example:\n",
    "#     Source column:\n",
    "#     - name: \"Position\"\n",
    "#     - details: Genomic coordinate location. Keywords: BP, POS, Base Pair, start, end. Examples: 45411941, 10240500:10248600 (range), build 37.\n",
    "\n",
    "#     Candidates:\n",
    "#     1. \"Variant ID | description: dbSNP identifier \n",
    "#     2. \"Chromosome position | description: genomic coordinate (bp)\n",
    "#     3. \"Effect allele | description: allele tested\n",
    "\n",
    "#     Expected output:\n",
    "#     BEST=2\n",
    "\n",
    "#     Source column:\n",
    "#     - name: {query}\n",
    "#     - details: {query_details}\n",
    "\n",
    "#     Candidates:\n",
    "#     1. {candidates[0]} | descrption: {candidates_details[0]}\n",
    "#     2. {candidates[1]} | descrption: {candidates_details[1]}\n",
    "#     3. {candidates[2]} | descrption: {candidates_details[2]}\n",
    "#     4. {candidates[3]} | descrption: {candidates_details[3]}\n",
    "#     5. {candidates[4]} | descrption: {candidates_details[4]}\n",
    "\n",
    "#     Output:\n",
    "#     BEST=\n",
    "#     \"\"\"\n",
    "\n",
    "#     inputs = llm_model_tokenizer(prompt, return_tensors=\"pt\")\n",
    "\n",
    "#     with torch.no_grad():\n",
    "#         output_ids = llm_model.generate(\n",
    "#             **inputs,\n",
    "#             max_new_tokens=64,\n",
    "#             do_sample=False,\n",
    "#             temperature=0.0,\n",
    "#         )\n",
    "\n",
    "#     resp = llm_model_tokenizer.decode(output_ids[0], skip_special_tokens=True)\n",
    "#     match = re.search(r\"BEST\\s*=\\s*(\\d+)\", resp)\n",
    "#     return candidates[int(match.group(1)) - 1] if int(match.group(1)) - 1 in range(len(candidates)) else query\n",
    "    \n",
    "\n",
    "# setting up models\n",
    "embeddings_model_name = \"NeuML/pubmedbert-base-embeddings\"\n",
    "embeddings_model_tokenizer = AutoTokenizer.from_pretrained(embeddings_model_name)\n",
    "embeddings_model = AutoModel.from_pretrained(embeddings_model_name)\n",
    "embeddings_model.eval()\n",
    "\n",
    "# setting up reranker\n",
    "# reranker_model_name = \"BAAI/bge-reranker-base\"\n",
    "# reranker_tokenizer = AutoTokenizer.from_pretrained(reranker_model_name)\n",
    "# reranker_model = AutoModelForSequenceClassification.from_pretrained(reranker_model_name)\n",
    "# reranker_model.eval()\n",
    "\n",
    "# setting up llm\n",
    "# llm_model_name = \"stanford-crfm/BioMedLM\"\n",
    "# llm_model_tokenizer = AutoTokenizer.from_pretrained(llm_model_name)\n",
    "# llm_model = AutoModelForCausalLM.from_pretrained(llm_model_name, torch_dtype=torch.float32).to(\"mps\")\n",
    "# llm_model.eval()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "38f2831b",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Try to first convert any abbreviation, then we match column with right semantic\n",
    "gwas_abbreviation_dict = {\n",
    "    \"CHR\": \"Chromosome number\",\n",
    "    \"BP\": \"Base-pair position\",\n",
    "    \"POS\": \"Position\",\n",
    "    \"SNP\": \"Single nucleotide polymorphism identifier\",\n",
    "    \"RS\": \"Reference Single nucleotide polymorphism\",\n",
    "    \"VAR\": \"Variant\",\n",
    "    \"ID\": \"identifier\",\n",
    "    \"A1\": \"Effect allele / tested allele\",\n",
    "    \"A2\": \"Other allele / non-effect allele\",\n",
    "    \"REF\": \"Reference allele (genome reference)\",\n",
    "    \"ALT\": \"Alternate allele\",\n",
    "    \"EA\": \"Effect Allele\",\n",
    "    \"NEA\": \"Non-Effect Allele\",\n",
    "    \"RA\": \"Risk Allele\",\n",
    "    \"OA\": \"Other Allele\",\n",
    "    \"AF\": \"Allele Frequency (general term)\",\n",
    "    \"RAF\": \"Risk Allele Frequency\",\n",
    "    \"EAF\": \"Effect Allele Frequency\",\n",
    "    \"MAF\": \"Minor Allele Frequency\",\n",
    "    \"BETA\": \"Effect size (regression coefficient)\",\n",
    "    \"OR\": \"Odds Ratio\",\n",
    "    \"SE\": \"Standard Error of effect estimate\",\n",
    "    \"Z\": \"Z-score statistic\",\n",
    "    \"T\": \"T-statistic\",\n",
    "    \"CI\": \"Confidence Interval\",\n",
    "    \"P\": \"P-value\",\n",
    "    \"PVAL\": \"P-value\",\n",
    "    \"LOGP\": \"Negative log10 P-value\",\n",
    "    \"Q\": \"Heterogeneity statistic (meta-analysis)\",\n",
    "    \"I2\": \"I-squared heterogeneity metric\",\n",
    "    \"HWE\": \"Hardy-Weinberg Equilibrium test statistic\",\n",
    "    \"INFO\": \"Imputation quality score\",\n",
    "    \"R2\": \"Imputation accuracy metric\",\n",
    "    \"CALLRATE\": \"Genotype call rate\",\n",
    "    \"MISSING\": \"Missing genotype rate\",\n",
    "    \"N\": \"Total sample size\",\n",
    "    \"N_CASES\": \"Number of cases (for binary traits)\",\n",
    "    \"N_CONTROLS\": \"Number of controls (for binary traits)\",\n",
    "    \"EUR\": \"European ancestry\",\n",
    "    \"AFR\": \"African ancestry\",\n",
    "    \"ASN\": \"Asian ancestry\",\n",
    "    \"AMR\": \"Admixed American ancestry\",\n",
    "    \"SAS\": \"South Asian ancestry\",\n",
    "    \"EAS\": \"East Asian ancestry\",\n",
    "    \"LD\": \"Linkage Disequilibrium\",\n",
    "    \"DPRIME\": \"LD D value\",\n",
    "    \"CADD\": \"CADD score (functional impact)\",\n",
    "    \"EQTL\": \"Expression quantitative trait locus\",\n",
    "    \"PQTL\": \"Protein QTL\",\n",
    "    \"GWGAS\": \"Gene-wide association study\",\n",
    "    \"PRS\": \"Polygenic Risk Score\",\n",
    "    \"PGS\": \"Polygenic Score\",\n",
    "    \"QC\": \"Quality Control\",\n",
    "    \"MA\": \"Meta-analysis\",\n",
    "    \"HLA\": \"Human Leukocyte Antigen region\",\n",
    "    \"HR\": \"Hazard ratio\",\n",
    "    \"HET\": \"Heterogeneity test\",\n",
    "    \"APOE4\": \"APOE 4\",\n",
    "    \"APOE*4\": \"APOE 4\",\n",
    "    \"#\": \"Number of\",\n",
    "    \"frq\": \"Frequency\",\n",
    "    '': \"Effect\",\n",
    "    \"nsnps\": \"Number of Variants\"\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "6df00f97",
   "metadata": {},
   "outputs": [],
   "source": [
    "def clean_col(col):\n",
    "    \"\"\"\n",
    "    Clean a column by replacing any possible abbreviation with their actual meaning for better semantic matching\n",
    "    \"\"\"\n",
    "    new_col = col\n",
    "    for abb in gwas_abbreviation_dict:\n",
    "        if re.search(fr\"[^a-zA-Z]{abb.lower()}[^a-zA-Z]\", new_col.lower()):\n",
    "            new_col = re.sub(fr\"([^a-zA-Z]){abb.lower()}([^a-zA-Z])\", fr\"\\1{gwas_abbreviation_dict[abb]}\\2\", new_col.lower())\n",
    "        elif re.search(fr\"^{abb.lower()}[^a-zA-Z]\", new_col.lower()):\n",
    "            new_col = re.sub(fr\"^{abb.lower()}([^a-zA-Z])\", fr\"{gwas_abbreviation_dict[abb]}\\1\", new_col.lower())\n",
    "        elif re.search(fr\"[^a-zA-Z]{abb.lower()}$\", new_col.lower()):\n",
    "            new_col = re.sub(fr\"([^a-zA-Z]){abb.lower()}$\", fr\"\\1{gwas_abbreviation_dict[abb]}\", new_col.lower())\n",
    "        elif re.search(fr\"^{abb.lower()}$\", new_col.lower()):\n",
    "            new_col = re.sub(fr\"{abb.lower()}\", gwas_abbreviation_dict[abb], new_col.lower())\n",
    "    # new_col = new_col.replace(\".\", \" \")\n",
    "    return new_col"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "b1c45269",
   "metadata": {},
   "outputs": [],
   "source": [
    "def make_col_prompt(col, example_values, num_example_values = 5):\n",
    "    \"\"\"\n",
    "    Based on the column title and some possible values of that columns, try to make a prompt\n",
    "    \"\"\"\n",
    "    col_prompt = f\"{col}: \"\n",
    "    for i in range(min(num_example_values, len(example_values))):\n",
    "        col_prompt += f\"{example_values[i]}, \"\n",
    "    return col_prompt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "a589c8c4",
   "metadata": {},
   "outputs": [],
   "source": [
    "def match_single_col_to_ref_col(col, ref_col_lst, ref_col_embeddings):\n",
    "    \"\"\"\n",
    "    match a single column to reference col given column, the list of referencing col and their embeddings\n",
    "    \"\"\"\n",
    "    # calculate column embedding\n",
    "    # col_embeddings = embeddings_model.encode(col, normalize_embeddings=True)\n",
    "    # multi_index_pattern = r\".+\\..+\"\n",
    "    # if re.search(multi_index_pattern, col):\n",
    "    #     col = col.replace(\".\", \" \")\n",
    "    #     # sub_col_lst = col.split(\".\")\n",
    "    #     # sub_col_embeddings = create_embeddings_from_model(sub_col_lst, embeddings_model, embeddings_model_tokenizer)\n",
    "    #     # col_embeddings = torch.mean(sub_col_embeddings, dim = 0)\n",
    "    #     col = col.replace(\".\", \" \")\n",
    "    #     col_embeddings = create_embeddings_from_model(col, embeddings_model, embeddings_model_tokenizer)\n",
    "    # else:\n",
    "    #     col_embeddings = create_embeddings_from_model(col, embeddings_model, embeddings_model_tokenizer)\n",
    "    col_embeddings = create_embeddings_from_model(col, embeddings_model, embeddings_model_tokenizer)\n",
    "    col_embeddings = col_embeddings.reshape(-1, 1)\n",
    "\n",
    "    # calculate similarity score\n",
    "    scores = torch.matmul(ref_col_embeddings, col_embeddings).reshape(-1)\n",
    "\n",
    "    # sort similairty score\n",
    "    top_k_indices = torch.argsort(scores, descending=True)\n",
    "\n",
    "    # verify if we even got good enough similarity\n",
    "    best_inx = top_k_indices[0].item()\n",
    "    best_score = scores[best_inx].item()\n",
    "    # second_best_inx = top_k_indices[1].item()\n",
    "    # second_best_score = scores[second_best_inx].item()\n",
    "    # need a threshold for score or else, just return col\n",
    "    # if best_score < 0.4: \n",
    "    #     return (col, 1)\n",
    "    \n",
    "    # # now do rerank\n",
    "    # candidates = [ref_col_lst[inx] for inx in top_k_indices if scores[inx] >= 0.4]\n",
    "    # candidates_scores = [scores[inx] for inx in top_k_indices if scores[inx] >= 0.4]\n",
    "    # best_ref_col, best_ref_col_scores = reranking_from_model(col, candidates, candidates_scores)\n",
    "    # return (best_ref_col, best_ref_col_scores)\n",
    "\n",
    "    if best_score >= 0.4:\n",
    "        return (ref_col_lst[best_inx], best_score)\n",
    "    return (col, 1)\n",
    "\n",
    "def match_many_col_to_ref_col(df, ref_col_df):\n",
    "    \"\"\"\n",
    "    Given a list of column and a dataframe (could be dict later on if that fits better),\n",
    "    try to match each column to the best fitted reference col, \n",
    "    return a dict of ref col : list of (col, cleaned col prompt, score)\n",
    "    \"\"\"\n",
    "    # prepare the embeddings for reference col since we reuse them\n",
    "    ref_col_lst = ref_col_df[\"column\"].to_list()\n",
    "    ref_col_context_lst = ref_col_df[\"column_with_context\"].to_list()\n",
    "    ref_col_embeddings = create_embeddings_from_model(ref_col_context_lst, embeddings_model, embeddings_model_tokenizer)\n",
    "\n",
    "    # conduct matching\n",
    "    multi_index_pattern = r\"^.+\\..+$\" # need multi index pattern for handling multi index\n",
    "    ref_col_to_col_lst = {}\n",
    "    for col in df.columns:\n",
    "\n",
    "        # extract values needed for prompt\n",
    "        cleaned_col = clean_col(col)\n",
    "        example_values = df[col].unique().tolist()\n",
    "        # prompt: {col}: example, need to delete all : first\n",
    "\n",
    "        if re.search(multi_index_pattern, cleaned_col.strip()):\n",
    "            # try to assess each part and see if which one have highest score\n",
    "            best_ref_col, best_score = None, 0\n",
    "            best_cleaned_sub_col_prompt = None\n",
    "            for sub_col in cleaned_col.split(\".\"):\n",
    "\n",
    "                # make column prompt for each sub col\n",
    "                cleaned_sub_col_prompt = make_col_prompt(sub_col, example_values)\n",
    "                \n",
    "                # matching and compare\n",
    "                ref_col, score = match_single_col_to_ref_col(cleaned_sub_col_prompt, ref_col_lst, ref_col_embeddings)\n",
    "                if score > best_score and ref_col != cleaned_sub_col_prompt:\n",
    "                    best_ref_col = ref_col\n",
    "                    best_score = score\n",
    "                    best_cleaned_sub_col_prompt = cleaned_sub_col_prompt\n",
    "            \n",
    "            # if we have a best one vs not\n",
    "            if best_ref_col is not None:\n",
    "                if best_ref_col not in ref_col_to_col_lst:\n",
    "                    ref_col_to_col_lst[best_ref_col] = []\n",
    "                ref_col_to_col_lst[best_ref_col].append((col, best_cleaned_sub_col_prompt, score))\n",
    "            else:\n",
    "                if col not in ref_col_to_col_lst:\n",
    "                    ref_col_to_col_lst[col] = []\n",
    "                ref_col_to_col_lst[col].append((col, best_cleaned_sub_col_prompt, 1))\n",
    "        else:\n",
    "            # match a single col with best fit referencing col and return a dict\n",
    "            # make the col prompt\n",
    "            # extra steps to remove .\n",
    "            cleaned_col = cleaned_col.replace(\".\", \"\")\n",
    "            cleaned_col_prompt = make_col_prompt(cleaned_col, example_values)\n",
    "\n",
    "            # matching for best col\n",
    "            ref_col, score = match_single_col_to_ref_col(cleaned_col_prompt, ref_col_lst, ref_col_embeddings)\n",
    "            # if we still get same col\n",
    "            if ref_col == cleaned_col_prompt: \n",
    "                ref_col = col\n",
    "            if ref_col not in ref_col_to_col_lst:\n",
    "                ref_col_to_col_lst[ref_col] = []\n",
    "            ref_col_to_col_lst[ref_col].append((col, cleaned_col_prompt, score))\n",
    "        # ref_col, score = match_single_col_to_ref_col(cleaned_col_prompt, ref_col_lst, ref_col_embeddings)\n",
    "        # if ref_col == cleaned_col_prompt: \n",
    "        #     ref_col = col\n",
    "        # if ref_col not in ref_col_to_col_lst:\n",
    "        #     ref_col_to_col_lst[ref_col] = []\n",
    "        # ref_col_to_col_lst[ref_col].append((col, cleaned_col_prompt, score))\n",
    "\n",
    "    return ref_col_to_col_lst"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "251adf23",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "s42003-019-0537-9_table_1.csv\n",
      "Lobe.: [('Lobe.', 'Lobe: Parietal, Temporal, Occipital, ', 1)]\n",
      "Nearest gene: [('Annotation (nearby).', 'Annotation (nearby): UTR3, Intronic, Intergenic, Upstream, ', 0.6490411758422852)]\n",
      "Position: [('Gene.', 'Gene: ZIC4, MSRB3, DAAM1 (dist = 24302), FGFRL1, CENPW (dist = 175626), ', 0.4209498167037964), ('Position.', 'Position: 147106319, 65793942, 59631075, 1013382, 126845380, ', 0.5123594403266907)]\n",
      "SNP: [('Rs-ID.', 'reference single nucleotide polymorphism-identifier: rs2279829, rs146354218, rs147148763, rs74921869, rs1337736, ', 0.6220928430557251)]\n",
      "Chr: [('Chr.', 'Chromosome number: 3, 12, 14, 4, 6, ', 0.6646981239318848)]\n",
      "RA: [('A1.', 'Effect allele / tested allele: T, A, G, ', 0.6893497109413147), ('A2.', 'Other allele / non-effect allele: C, G, GTTGT, ', 0.5696174502372742)]\n",
      "AF: [('European ancestry meta-analysis.Frq', 'european ancestry meta-analysis: 0.21, 0.37, 0.13, 0.2, 0.23, ', 0.4252067804336548), ('European ancestry meta-analysis.Effect', 'European ancestry meta-analysis: - 0.92, 0.69, - 0.83, - 0.84, - 0.64, ', 0.4634610116481781)]\n",
      "Confidence Interval: [('European ancestry meta-analysis.SE', 'Standard Error of effect estimate: 0.16, 0.11, 0.14, 0.12, 0.1, ', 0.6344603300094604), ('Multi-ethnic meta-analysis.Effect', 'Effect: - 0.95, 0.69, - 0.85, - 0.82, - 0.64, ', 0.4803103804588318), ('Multi-ethnic meta-analysis.SE', 'Standard Error of effect estimate: 0.15, 0.11, 0.14, 0.12, 0.09, ', 0.6245501637458801), ('Replication sample ( n = 8,789).SE', 'standard error of effect estimate: 0.18, 0.08 1.6, ', 0.647702693939209)]\n",
      "P-value: [('European ancestry meta-analysis.p', 'P-value: 5.3  10 - 9, 1.2  10 - 9, 6.6  10 - 9, 5.9  10 - 9, 8.1  10 - 8, ', 0.5993689894676208), ('Multi-ethnic meta-analysis.p', 'P-value: 4.4  10 - 10, 6.4  10 - 10, 2.9  10 - 9, 6.2  10 - 9, 4.0  10 - 8, ', 0.6031227111816406), ('Replication sample ( n = 8,789).p', 'p-value: 6.0  10 - 4, 6.4  10 - 5, 3.1  10 - 11, 3.0  10 - 2, 8.0  10 - 7, ', 0.6468125581741333)]\n",
      "Population: [('Multi-ethnic meta-analysis.Frq', 'multi-ethnic meta-analysis: 0.21, 0.37, 0.13, 0.2, 0.23, ', 0.4252067804336548)]\n",
      "Sample size: [('Replication sample ( n = 8,789).N', 'replication sample ( Total sample size = 8,789): 16,015, 15,789, 15,220, 12,424, 16,016, ', 0.4269945025444031), ('Replication sample ( n = 8,789).R 2 (%)', 'replication sample ( Total sample size = 8,789): 0.24, 0.23 0.12, 0.27, 0.19, 0.17, ', 1), ('Replication sample ( n = 8,789).Frq', 'replication sample ( total sample size = 8,789): 0.22, - 0.37 0.54, - 0.80, 0.19 - 0.22, 0.23 - 0.46, ', 0.4252721071243286), ('Replication sample ( n = 8,789).Effect', 'replication sample ( Total sample size = 8,789): 0.63, 0.14, 0.12, 0.1, 0.09, ', 0.5168788433074951)]\n",
      "\n",
      "nihms-1797266_table_8.csv\n",
      "Subset Analyzed: [('APOE4 stratification.', 'APOE 4 stratification: NO, APOE4 carriers, non- APOE4 carriers, ', 0.6221021413803101)]\n",
      "SNP: [('SNP (rs).', 'single nucleotide polymorphism identifier (Reference Single nucleotide polymorphism): rs146729640, ', 0.4884442090988159)]\n",
      "Studies combined.: [('Studies combined.', 'Studies combined: Amish and NIA- LOAD, ', 1)]\n",
      "Sample size: [('Number of observations.', 'Number of observations: 4,872, 2,293, 2,579, ', 0.6042938828468323)]\n",
      "Effect Size: [('Meta-Analysis results.Model', 'Meta-Analysis results: Fixed effect, Random effects, ', 0.41063129901885986), ('Meta-Analysis results.Hazard Ratio', 'Hazard Ratio: 2.07, 3.0, 1.5, 1.76, 3.56, ', 0.5157300233840942)]\n",
      "Confidence Interval: [('Meta-Analysis results.Hazard Ratio 95%CI', 'hazard ratio 95%Confidence Interval: 1.54; 2.78, 0.72; 12.50, 1.072; 2.09, 0.86; 3.60, 2.19; 5.78, ', 0.5328888297080994)]\n",
      "P-value: [('Meta-Analysis results.P-value', 'P-value-value: <0.0001, 0.1308, 0.0178, 0.1219, 0.0831, ', 0.6912790536880493)]\n",
      "\n",
      "s13024-023-00633-4_table_2.csv\n",
      "Chr: [('Chr.', 'Chromosome number: 1, 2, 7, 8, 11, ', 0.6763196587562561)]\n",
      "Position: [('Position.', 'Position: 207573951, 127135234, 91069758, 30963282, 60173126, ', 0.48896729946136475)]\n",
      "SNP: [('Variant.', 'Variant: rs1752684, rs6733839, rs705353, rs117523305, rs7232, ', 0.6578899621963501)]\n",
      "RA: [('A1/A2 a (MAF).', 'effect allele / tested allele/other allele / non-effect allele a (Minor Allele Frequency): A/G (0.198), T/C (0.403), G/A (0.252), C/G (0.016), A/T (0.361), ', 0.7460706830024719)]\n",
      "Nearest gene: [('Nearest Gene.', 'Nearest Gene: CR1, BIN1, CYP27C1, CDK14, PURG TEX15, ', 0.5540646314620972)]\n",
      "Cognitive Domain.: [('Cognitive Domain.', 'Cognitive Domain: Executive Function, Language, Memory, ', 1)]\n",
      "Cohorts b.: [('Cohorts b.', 'Cohorts b: All, Clinic, Community, Community All, ', 1)]\n",
      "Effect Size: [('Genetic effects. G (SE)', 'Effect g (standard error of effect estimate): -0.028 (0.007), -0.042 (0.012), -0.020 (0.009), -0.026 (0.007), -0.054 (0.012), ', 0.5935451984405518), ('Genetic effects. G  Age (SE)', 'Effect g  age (standard error of effect estimate): -0.002 (0.001), -0.001 (0.001), -0.004 (0.001), 0.000 (0.001), -0.005 (0.001), ', 0.4552510380744934)]\n",
      "P-value: [('Genetic effects.P G', 'P-value g: 1.53  10 -4, 5.91  10 -4, 2.98  10 -2, 3.90  10 -4, 1.26  10 -5, ', 0.6706618070602417), ('Genetic effects.P G  Age', 'P-value g  age: 6.05  10 -3, 7.52  10 -2, 4.90  10 -2, 6.84  10 -2, 5.07  10 -2, ', 0.6131654977798462), ('Genetic effects.P Joint', 'P-value joint: 5.05  10 -5, 6.88  10 -4, 2.40  10 -2, 6.25  10 -4, 1.63  10 -5, ', 0.6872744560241699)]\n",
      "\n",
      "s13024-023-00633-4_table_3.csv\n",
      "Chr: [('Chr.', 'Chromosome number: 2, 4, 8, 9, 16, ', 0.6527141332626343)]\n",
      "Position: [('Position.', 'Position: 127135234, 179150192, 102060792, 10155013, 83947805, ', 0.4991665482521057)]\n",
      "SNP: [('Variant.', 'Variant: rs6733839, rs73005629, rs56162098, rs145989094, rs12447050, ', 0.6541426181793213), ('Individual locus. G  Age (SE)', 'individual locus: -0.002 (0.001) -0.002 (0.001), -0.007 (0.002) -0.006 (0.002), 0.001 (0.001) -0.001 (0.001), -0.006 (0.002) -0.008 (0.003), -0.004 (0.003) -0.008 (0.003), ', 0.42733120918273926)]\n",
      "RA: [('A1/A2 a (MAF).', 'effect allele / tested allele/other allele / non-effect allele a (Minor Allele Frequency): T/C (0.403), C/T (0.022), C/T (0.138), A/T (0.013), C/T (0.400), ', 0.7249321341514587)]\n",
      "Nearest gene: [('Nearest Gene.', 'Nearest Gene: BIN1 CYP27C1, LOC107984373, NCALD, PTPRD, OSGIN1 MLYCD, ', 0.5613346099853516)]\n",
      "Cognitive Domain.: [('Cognitive Domain.', 'Cognitive Domain: Language Memory, Exec Function Memory, Language, Memory, ', 1)]\n",
      "Effect Size: [('Individual locus. G (SE)', 'Effect g (standard error of effect estimate): -0.031 (0.006) -0.039 (0.006), -0.048 (0.019) -0.062 (0.021), 0.019 (0.008) -0.013 (0.009), -0.056 (0.026) -0.058 (0.028), -0.093 (0.025) -0.058 (0.028), ', 0.5734248161315918)]\n",
      "P-value: [('Individual locus.P G', 'P-value g: 6.90  10 -8 2.20  10 -9, 1.21  10 -2 3.97  10 -3, 1.64  10 -2 1.68  10 -1, 3.22  10 -2 4.56  10 -2, 3.40  10 -4 4.56  10 -2, ', 0.6728843450546265), ('Individual locus.P G  Age', 'P-value g  age: 3.62  10 -3 1.62  10 -3, 2.06  10 -4 8.12  10 -3, 1.84  10 -1 1.71  10 -1, 1.26  10 -2 3.14  10 -3, 9.29  10 -2 3.14  10 -3, ', 0.5825444459915161), ('Individual locus.P Joint', 'P-value joint: 2.70  10 -8 2.37  10 -9, 1.19  10 -4 2.11  10 -3, 3.08  10 -2 2.17  10 -1, 1.04  10 -2 5.92  10 -3, 8.84  10 -4 5.92  10 -3, ', 0.6739028692245483), ('Pleiotropy.P Placo,G', 'P-value placo,g: 2.72  10 -10, 3.69  10 -3, 5.85  10 -5, 2.70  10 -2, 1.22  10 -2, ', 0.5555859804153442), ('Pleiotropy.P Placo,G  Age', 'P-value placo,g  age: 9.11  10 -4, 9.32  10 -4, 5.74  10 -3, 2.79  10 -3, 3.47  10 -2, ', 0.5390950441360474), ('Pleiotropy.P Placo, Joint', 'P-value placo, joint: 9.01  10 -12, 6.16  10 -5, 6.86  10 -6, 9.93  10 -4, 4.65  10 -3, ', 0.5947532653808594)]\n",
      "\n",
      "ACEL-22-e13938_table_4.csv\n",
      ": [('', ': 2-order interactions, 3- and 4-order interactions, B: APOE *4 + males, 3- order interaction, C: APOE *4 - males, ', 1)]\n",
      "SNP: [('Gene-gene combination', 'Gene-gene combination: rs3757949/rs7092522, rs62491484/rs35435718, rs62491484/rs28672744/ rs35435718, rs2686186/rs7092522/ rs12316064/ rs35511346, rs4674302/rs718630, ', 0.5972591638565063)]\n",
      "Reported gene: [('Genes included in the combination', 'Genes included in the combination: GATA4 / IDE, NEIL2 / PGPEP1L, NEIL2 / TXNRD1 / PGPEP1L, FDFT1 / IDE / IGF1 / TXNRD1, AOX1 / PTPN1, ', 0.4906756281852722)]\n",
      "Pathways involved: [('Pathways involved', 'Pathways involved: Stress response/ Metabolism, DNA repair/Stress response, Metabolism, DNA repair, DNA repair/ Metabolism, ', 1)]\n",
      "Effect Size: [('OR(95% CI.)', 'odds ratio(95% Confidence Interval: 1.42 (1.25, 1.62), 1.36 (1.19, 1.54), 1.56 (1.38, 1.76), 1.96 (1.72, 2.22), 1.46 (1.27, 1.69), ', 0.41210195422172546)]\n",
      "P-value: [('p -value (10,000 permutations).APOE *4 + females', 'P-value -value (10,000 permutations): 0.0001, 0.0002, 0.8127, 0.5017, 0.9976, ', 0.46324244141578674), ('p -value (10,000 permutations).APOE *4 + males', 'P-value -value (10,000 permutations): 0.1945, 0.6143, 0.8034, 0.9389, 0.0001, ', 0.479831337928772), ('p -value (10,000 permutations).APOE *4 - females', 'P-value -value (10,000 permutations): 0.7051, 0.8744, 0.9658, 0.6944, 0.9709, ', 0.4692883789539337), ('p -value (10,000 permutations).APOE *4 - males', 'P-value -value (10,000 permutations): 0.6106, 0.0294, 0.4729, 0.824, 0.9489, ', 0.48043057322502136)]\n",
      "\n",
      "41591_2023_Article_2268_table_2.csv\n",
      "SNP: [('GWASmeta-analysis.SNP', 'Single nucleotide polymorphism identifier: rs6011998, rs13079464, rs2425884, rs10817108, rs2425881, ', 0.6490005850791931)]\n",
      "Chr: [('GWASmeta-analysis.chr:position', 'Chromosome number:position: 20:45269867, 3:13822439, 20:45258292, 9:113658671, 20:45255618, ', 0.672398030757904)]\n",
      "RA: [('GWASmeta-analysis.EA/OA', 'effect allele/Other Allele: C/T, C/G, A/G, G/A, T/A, ', 0.5965904593467712)]\n",
      "Reported gene: [('GWASmeta-analysis.Nearest gene(s)', 'Nearest gene(s): SLC13A3, WNT7A, LPAR1, ITGB5 , UMPS, CENPF , KCNK2, ', 0.5279557108879089)]\n",
      "Effect Size: [('i-Share(dichotomous).OR(95% CI)', 'odds ratio(95% Confidence Interval): 1.26 (0.83-1.92), 1.12 (0.91-1.40), 1.18 (0.95-1.45), 0.90 (0.69-1.17), 1.47 (1.03-2.01), ', 0.5026836395263672), ('i-Share (continuous).(SE)', 'Effect(standard error of effect estimate): 0.164 (0.04), 0.014 (0.02), 0.077 (0.02), 0.058 (0.03), 0.063 (0.03), ', 0.6067800521850586), ('Nagahama(dichotomous).OR(95% CI)', 'odds ratio(95% Confidence Interval): 1.69 (1.33-2.13), 1.16 (0.97-1.40), 1.29 (1.09-1.52), 1.18 (0.98-1.43), 1.18 (1.01-1.37), ', 0.48577362298965454), ('Nagahama (continuous).(SE)', 'Effect(standard error of effect estimate): 0.037 (0.008), 0.015 (0.006), 0.026 (0.005), 0.017 (0.006), 0.014 (0.005), ', 0.58628910779953)]\n",
      "P-value: [('i-Share(dichotomous).P value', 'P-value value: 0.28, 0.29, 0.13, 0.44, 1.4010 -02, ', 0.7504112720489502), ('i-Share (continuous).P value', 'P-value value: 4.2010 -05, 0.50, 2.9810 -04, 2.2310 -02, 2.6210 -02, ', 0.7592149972915649), ('Nagahama(dichotomous).P value', 'P-value value: 1.2210 -05, 0.11, 3.4810 -03, 0.07, 3.6610 -02, ', 0.7511454820632935), ('Nagahama (continuous).P value', 'P-value value: 6.2110 -07, 1.5010 -02, 1.7710 -06, 4.1010 -03, 4.6810 -03, ', 0.7479268312454224)]\n",
      "\n",
      "nihms-1797266_table_5.csv\n",
      "Chr: [('Chr', 'Chromosome number: 17, 9, 6, ', 0.6750222444534302)]\n",
      "SNV: [('Variant', 'Variant: 17:11183894_G_A, 9:128627091_C_G, 6:149756107_C_G, ', 0.4719577133655548)]\n",
      "SNP: [('Variant (rs)', 'variant (Reference Single nucleotide polymorphism): rs146729640, rs534551495, rs145348074, ', 0.5827505588531494)]\n",
      "Cohort: [('Data set', 'Data set: Amish, NIA-LOAD, ', 0.49322229623794556)]\n",
      "Sample size: [('# of subjects', 'Number of of subjects: 1520, 3350, ', 0.662908136844635)]\n",
      "AF: [('MAF', 'Minor Allele Frequency: 0.012, 0.038, 0.006, 0.024, 0.013, ', 0.6308693885803223)]\n",
      "Effect Size: [('Hazard ratio', 'Hazard ratio: 6.38, 1.49, 2.82, 0.98, 3.35, ', 0.5193595886230469)]\n",
      "P-value: [('P-value', 'P-value-value: 5.59e-09, 0.0217, 4.83e-08, 0.9356, 5.26e-08, ', 0.7559226751327515)]\n",
      "\n",
      "nihms-1797266_table_7.csv\n",
      "Subset Analyzed: [('APOE4 stratification NIA-LOAD', 'APOE 4 stratification nia-load: No, APOE4 carriers, non- APOE4 carriers, ', 0.5843316912651062)]\n",
      "SNP: [('SNP (chr:pos)', 'Single nucleotide polymorphism identifier (chromosome number:position): 17:11183894_G_A, ', 0.5955944061279297), ('SNP (rs)', 'single nucleotide polymorphism identifier (Reference Single nucleotide polymorphism): rs146729640, ', 0.4884442090988159)]\n",
      "Sample size: [('N (AD)', 'Total sample size (ad): 1,785, 1,326, 459, ', 0.6699733734130859), ('N (controls)', 'Total sample size (controls): 1,565, 572, 993, ', 0.755468487739563)]\n",
      "Effect Size: [('Hazard Ratio', 'Hazard Ratio: 1.49, 1.35, 1.82, ', 0.5352332592010498)]\n",
      "Confidence Interval: [('Hazard Ratio 95% CI', 'hazard ratio 95% Confidence Interval: 1.06 ; 2.08, 0.95 ; 1.93, 0.96 ; 3.43, ', 0.5930284261703491)]\n",
      "P-value: [('P-value', 'P-value-value: 0.0217, 0.095, 0.0658, ', 0.746171772480011)]\n",
      "Gene: [('Gene', 'Gene: SHISA6, ', 1)]\n",
      "\n",
      "41591_2023_Article_2268_table_1.csv\n",
      "Locus: [('Region', 'Region: 20q13.12, 3p25.1, 9q31.3, 3q21.2, 1q41, ', 0.6903148889541626)]\n",
      "SNP: [('SNPALL', 'SNPALL: rs6011998, rs13079464, rs2425884, rs10817108, rs2425881, ', 0.6580637693405151)]\n",
      "Chr: [('chr:position', 'Chromosome number:position: 20:45269867, 3:13822439, 20:45258292, 9:113658671, 20:45255618, ', 0.672398030757904)]\n",
      "RA: [('EA/OA', 'effect allele/Other Allele: C/T, C/G, A/G, G/A, T/A, ', 0.5965904593467712)]\n",
      "AF: [('EAF', 'Effect Allele Frequency: 0.95, 0.46, 0.57, 0.21, 0.83, ', 0.6164335608482361)]\n",
      "Most severe consequence: [('Function', 'Function: intronic, intergenic, UTR3, exonic, ', 0.5250861048698425)]\n",
      "Reported gene: [('Nearest gene(s)', 'Nearest gene(s): SLC13A3, WNT7A, LPAR1, ITGB5 , UMPS, CENPF , KCNK2, ', 0.5279557108879089)]\n",
      "Effect Size: [('Effect ()', 'effect (Effect): 0.087, 0.026, 0.029, 0.033, 0.032, ', 0.5356194972991943)]\n",
      "Confidence Interval: [('SE', 'Standard Error of effect estimate: 0.009, 0.004, 0.005, 0.012, 0.01, ', 0.5811103582382202)]\n",
      "Z - score: [('Z - score', 'Z-score statistic - score: 10.65, 8.7, 8.63, 8.2, 7.68, ', 1)]\n",
      "Dir: [('Dir', 'Dir: ++++, +-+-, +++?, +-+?, +???, ', 1)]\n",
      "Sample size: [('N ext-PVS/ N total', 'n ext-pvs/ Total sample size total: 9,502/39,128, 9,614/39,822, 9,550/39,516, 9,496/39,087, 8,426/34,530, ', 0.5367323756217957)]\n",
      "P-value: [('P valueEUR', 'P-value valueeur: 1.9010 -24, 8.6410 -17, 2.6010 -18, 1.0710 -15, 2.0210 -15, ', 0.7449556589126587), ('P valueAll', 'P-value valueall: 1.8010 -26, 3.4110 -18, 6.0210 -18, 2.4610 -16, 1.5910 -14, ', 0.7230997085571289), ('Het P value', 'Heterogeneity test p-value value: 0.11, 0.59, 0.14, 0.75, 0.06, ', 0.5893433094024658)]\n",
      "\n",
      "nihms-1797266_table_6.csv\n",
      "Subset Analyzed: [('APOE4 stratification Amish', 'APOE 4 stratification amish: No, APOE4 carriers, non- APOE4 carriers, ', 0.6200578212738037)]\n",
      "SNP: [('SNP (chr:pos)', 'Single nucleotide polymorphism identifier (chromosome number:position): 17:11183894_G_A, ', 0.5955944061279297), ('SNP (rs)', 'single nucleotide polymorphism identifier (Reference Single nucleotide polymorphism): rs146729640, ', 0.4884442090988159)]\n",
      "Confidence Interval: [('N (CI)', 'Total sample size (confidence interval): 362, 105, 257, ', 0.691734254360199), ('Hazard Ratio 95% CI', 'hazard ratio 95% Confidence Interval: 3.42 ; 11.90, 1.16 ; 7.40, 4.25 ; 18.91, ', 0.5840392112731934)]\n",
      "Sample size: [('N (CU)', 'Total sample size (cu): 1,160, 290, 870, ', 0.6655024290084839)]\n",
      "Effect Size: [('Hazard Ratio', 'Hazard Ratio: 6.38, 2.93, 8.96, ', 0.5116651058197021)]\n",
      "P-value: [('P-value', 'P-value-value: 5.59e-09, 0.0232, 8.44e-09, ', 0.7551623582839966)]\n",
      "Gene: [('Gene', 'Gene: SHISA6, ', 1)]\n",
      "\n",
      "ACEL-22-e13938_table_3.csv\n",
      "SNP: [('Variant', 'Variant: rs13183641, rs4561, rs2348974, rs6556881, rs34886287, ', 0.6433566212654114)]\n",
      "Chr: [('Chr.', 'Chromosome number: 5, 8, 9, 10, 19, ', 0.6747028827667236)]\n",
      "Position: [('Position', 'Position: 95,147,287, 95,152,313, 95,143,394, 95,134,419, 80,077,309, ', 1)]\n",
      "Reported gene: [('Gene SYMBOL', 'Gene SYMBOL: RHOBTB3, MSH3, FDFT1, RP11-\\xad 297N6.4, PAPPA, ', 0.5255099534988403)]\n",
      "RA: [('EA', 'Effect Allele: T, G, C, A, ', 0.6581827402114868)]\n",
      "Effect Size: [('OR', 'Odds Ratio: 0.873, 0.886, 1.119, 0.908, 1.11, ', 0.4957042336463928), ('OR_1', 'Odds Ratio_1: 0.909, 0.91, 1.093, 0.925, 1.105, ', 0.4880014955997467)]\n",
      "P-value: [('p', 'P-value: 0.003, 0.008, 0.015, 0.038, 0.029, ', 0.6428716778755188), ('p_1', 'P-value_1: 0.01, 0.011, 0.014, 0.034, 0.007, ', 0.7438563108444214)]\n",
      "\n",
      "nihms-1797266_table_3.csv\n",
      "Position: [('Location and base pair change (build hg19)', 'Location and base pair change (build hg19): 6:149756107_C_G, 9:128627091_C_G, 17:11183894_G_A, ', 0.6547622084617615)]\n",
      "SNP: [('SNP (rs)', 'single nucleotide polymorphism identifier (Reference Single nucleotide polymorphism): rs145348074, rs534551495, rs146729640, ', 0.5564624071121216)]\n",
      "RA: [('MAF', 'Minor Allele Frequency: 0.0237382, 0.0379333, 0.0119911, ', 0.5966141223907471)]\n",
      "Most severe consequence: [('Variant type', 'Variant type: Intergenic, Intron, ', 0.5914268493652344)]\n",
      "Effect Size: [('Hazard Ratio', 'Hazard Ratio: 3.35, 2.82, 6.38, ', 0.5270391702651978)]\n",
      "P-value: [('P-value', 'P-value-value: 5.26e-08, 4.83e-08, 5.59e-09, ', 0.722084641456604)]\n",
      "Nearest gene: [('Overlapping Genes', 'Overlapping Genes: Intergenic variant. Closest gene: ZC3H12D, PBX3, SHISA6, ', 0.6498894095420837), ('Genes within +/- 500Kb', 'Genes within +/- 500Kb: UST, UST-AS1, LOC105378047, TAB2, TAB2-AS1, SUMO4, ZC3H12D, PPIL4, GINM1, KATNA1, LATS1, LOC645967, NUP43, PCMT1, LRP11, RAET1E-AS1, RAET1E, RAET1G, LOC105378052, GAPVD1, MAPKAP1, LOC51145, PBX3, LOC10192911, MVB12B, TMEM220-AS1, DNAH9, TMEM238L, PIRT, SHISA6,, ', 0.5745738744735718)]\n",
      "\n",
      "Recent paper on AD GWAS (1)_table_1.csv\n",
      "Locus: [('Locus', 'Locus: VWA5B2, TRANK1, ', 0.5276135802268982)]\n",
      "SNP: [('SNP', 'Single nucleotide polymorphism identifier: rs9837978, rs9867455, ', 0.6051782369613647)]\n",
      "Chr: [('Chromosome', 'Chromosome: 3, ', 0.7065351009368896)]\n",
      "Position: [('Position', 'Position: 183936947, 36953424, ', 0.5131635665893555)]\n",
      "RA: [('Effect allele', 'Effect allele: G, A, ', 0.7219094038009644), ('Reference allele', 'Reference allele: A, T, ', 0.5889570713043213), ('Mean effect allele frequency', 'Mean effect allele frequency: 0.287, 0.405, ', 0.6255718469619751)]\n",
      "P-value: [('P, MR-MEGA', 'P-value, mr-mega: 2.32E - 07, 7.33E - 08, ', 0.6305118799209595), ('P, ancestry heterogeneity', 'P-value, ancestry heterogeneity: 5.96E - 01, 1.18E - 01, ', 0.6734113693237305), ('P, random effects', 'P-value, random effects: 3.75E - 08, 3.49E - 08, ', 0.6263864040374756)]\n",
      "Effect Size: [('beta, random effects', 'Effect size (regression coefficient), random effects: - 0.053, - 0.042, ', 0.5624829530715942), ('I2', 'I-squared heterogeneity metric: 0.0, ', 0.46090173721313477)]\n",
      "Confidence Interval: [('SE, random effects', 'Standard Error of effect estimate, random effects: 0.009, 0.008, ', 0.5634124875068665)]\n",
      "AF: [('Minimum effect allele frequency', 'Minimum effect allele frequency: 0.171, 0.326, ', 0.6242308616638184), ('Maximum effect allele frequency', 'Maximum effect allele frequency: 0.45, 0.503, ', 0.6445791721343994)]\n",
      "\n",
      "Recent paper on AD GWAS (1)_table_2.csv\n",
      "Locus: [('Locus', 'Locus: BIN1, INPP5D, RHOH, CTSB, ECHDC3, ', 0.5405474901199341)]\n",
      "SNP: [('SNP', 'Single nucleotide polymorphism identifier: rs6733839, rs10933431, rs2245466, rs1065712, rs7912495, ', 0.6305140256881714)]\n",
      "Chr: [('Chromosome', 'Chromosome: 2, 4, 8, 10, 14, ', 0.684895932674408)]\n",
      "Position: [('Position', 'Position: 127892810, 233981912, 40198846, 11702122, 11718713, ', 0.5232166051864624)]\n",
      "RA: [('Effect allele', 'Effect allele: T, C, A, ', 0.6809673309326172), ('Reference allele', 'Reference allele: C, G, ', 0.578101396560669)]\n",
      "Novel /uniFB01 ne- mapping: [('Novel /uniFB01 ne- mapping', 'Novel /uniFB01 ne- mapping: No, Yes, ', 1)]\n",
      "SNV: [('Nsnps_in_credible_set', 'Number of Variants_in_credible_set: 1, 32, 2, ', 0.6518357992172241)]\n",
      "P-value: [('Posterior probability', 'Posterior probability: 1.0, 0.91, 0.965, ', 0.43237727880477905), ('P, MR-MEGA', 'P-value, mr-mega: 5.74E - 92, 1.63E - 16, 3.82E - 09, 6.42E - 09, 1.35E - 13, ', 0.6117170453071594), ('P, ancestry heterogeneity', 'P-value, ancestry heterogeneity: 3.13E - 01, 7.90E - 02, 9.31E - 01, 7.60E - 01, 6.02E - 01, ', 0.6520999073982239)]\n",
      "Population: [('chisq, ancestry (% total)', 'chisq, ancestry (% total): 14.27, 41.675, 0.128, 4.551, 48.894, ', 0.44168075919151306)]\n",
      "Effect Size: [('I2', 'I-squared heterogeneity metric: 58.01, 59.47, 49.16, 0.0, 57.45, ', 0.4712890684604645)]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for file in os.listdir(\"./tables\"):\n",
    "    if \"table\" in file and \".csv\" in file and \"harmonized\" not in file:\n",
    "        print(file)\n",
    "        df = pd.read_csv(f\"./tables/{file}\")\n",
    "        df.columns = ['' if 'Unnamed:' in col else col for col in df.columns]\n",
    "        col_to_ref_col = match_many_col_to_ref_col(df, referencing_cols)\n",
    "        for ref_col in col_to_ref_col:\n",
    "            print(f\"{ref_col}: {col_to_ref_col[ref_col]}\")\n",
    "        print()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8012a639",
   "metadata": {},
   "source": [
    "Final formatting of the tables"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "43a8ed1a",
   "metadata": {},
   "outputs": [],
   "source": [
    "# We now can try to use these dictionary to make a final dataset\n",
    "# need a list of columns that we sure that there might be multiple copies of it that we can melt them into many rows\n",
    "possible_ref_col_to_melt = [\"P-value\", \"Effect Size\", \"Confidence Interval\", \"AF\"]\n",
    "def format_original_table(df, remove_unique_col = False):\n",
    "    # map the columns\n",
    "    new_col_to_old_col_lst = match_many_col_to_ref_col(df, referencing_cols)\n",
    "    df_with_ref_col = None\n",
    "    new_col_to_not_melt = [] # list of columns that are stable and not need to be melt\n",
    "    new_col_to_old_col_lst_to_melt = {}\n",
    "    for new_col in new_col_to_old_col_lst:\n",
    "        if len(new_col_to_old_col_lst[new_col]) == 1:\n",
    "            if (not remove_unique_col) or (remove_unique_col and new_col_to_old_col_lst[new_col][0][2] != 1):\n",
    "                if df_with_ref_col is None:\n",
    "                    df_with_ref_col = df[[new_col_to_old_col_lst[new_col][0][0]]]\n",
    "                    df_with_ref_col = df_with_ref_col.rename({new_col_to_old_col_lst[new_col][0][0]: new_col}, axis = 1)\n",
    "                else:\n",
    "                    df_with_ref_col[new_col] = df[new_col_to_old_col_lst[new_col][0][0]]\n",
    "                # add these single col to the list of not melt\n",
    "                new_col_to_not_melt.append(new_col)\n",
    "        else:\n",
    "            if new_col in possible_ref_col_to_melt:\n",
    "                old_col_lst = []\n",
    "                for col, _, _ in new_col_to_old_col_lst[new_col]:\n",
    "                    old_col_lst.append(col)\n",
    "                    if df_with_ref_col is None:\n",
    "                        df_with_ref_col = df[[col]]\n",
    "                    else:\n",
    "                        df_with_ref_col[col] = df[col]\n",
    "                new_col_to_old_col_lst_to_melt[new_col] = old_col_lst.copy()\n",
    "            else:\n",
    "                # make multiple copies with notes\n",
    "                for inx, (col, _, _) in enumerate(new_col_to_old_col_lst[new_col]):\n",
    "                    if df_with_ref_col is None:\n",
    "                        df_with_ref_col = df[[col]]\n",
    "                        df_with_ref_col = df_with_ref_col.rename({col: f\"{new_col}_{inx + 1}\"}, axis = 1)\n",
    "                    else:\n",
    "                        df_with_ref_col[f\"{new_col}_{inx + 1}\"] = df[col]\n",
    "                    df_with_ref_col[f\"{new_col}_{inx + 1} notes\"] = col\n",
    "                    # add these cols in group but not need to melt\n",
    "                    new_col_to_not_melt.append(f\"{new_col}_{inx + 1}\")\n",
    "                    new_col_to_not_melt.append(f\"{new_col}_{inx + 1} notes\")\n",
    "    # Melting stage\n",
    "    if len(new_col_to_old_col_lst_to_melt) > 0:\n",
    "        # now melting column in same groups\n",
    "        # Instead of keep melting, for each group, we make a new dataset of \n",
    "        # [stable col] + [to be melt col] => melt them as a new df\n",
    "        # do this for each gorup and then join together based on stable col\n",
    "        df_with_melt_col = None \n",
    "        for new_col in new_col_to_old_col_lst_to_melt:\n",
    "            temp_df = deepcopy(df_with_ref_col[new_col_to_not_melt + new_col_to_old_col_lst_to_melt[new_col]])\n",
    "            # create a temp row id for stable join\n",
    "            temp_df[\"_row_id\"] = np.arange(temp_df.shape[0])\n",
    "            temp_df = temp_df.melt(\n",
    "                id_vars = new_col_to_not_melt + [\"_row_id\"],    \n",
    "                value_vars = new_col_to_old_col_lst_to_melt[new_col], \n",
    "                var_name = f\"{new_col} notes\",      \n",
    "                value_name = f\"{new_col}\"\n",
    "            )\n",
    "            if df_with_melt_col is None:\n",
    "                df_with_melt_col = deepcopy(temp_df)\n",
    "            else:\n",
    "                df_with_melt_col = df_with_melt_col.merge(temp_df, how = \"inner\", on = [\"_row_id\"] + new_col_to_not_melt)\n",
    "        df_with_melt_col = df_with_melt_col.drop(\"_row_id\", axis = 1)\n",
    "        return df_with_melt_col\n",
    "    else:\n",
    "        return df_with_ref_col"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "0e26f716",
   "metadata": {},
   "outputs": [],
   "source": [
    "modified_df_all = None\n",
    "for file in os.listdir(\"./tables\"):\n",
    "    if \".csv\" in file and \"table\" in file and \"harmonized\" not in file:\n",
    "        df = pd.read_csv(f\"./tables/{file}\")\n",
    "        df.columns = ['' if 'Unnamed:' in col else col for col in df.columns]\n",
    "        modified_df = format_original_table(df, remove_unique_col = False)\n",
    "        modified_df[\"file_name\"] = file\n",
    "        if modified_df_all is None:\n",
    "            modified_df_all = modified_df.copy()\n",
    "        else:\n",
    "            modified_df_all = pd.concat([modified_df_all, modified_df], ignore_index = True)\n",
    "        modified_df.to_csv(f\"./harmonized_tables/{file.replace('.csv', '')}_harmonized.csv\", index = False)\n",
    "modified_df_all.to_csv(\"./harmonized_tables/harmonized_table.csv\", index = False)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.19"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
