{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "ad359987",
   "metadata": {},
   "source": [
    "### Test on ADVP 1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "33002256",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/justpqa/advpai/venv/lib/python3.10/site-packages/pypdf/_crypt_providers/_cryptography.py:32: CryptographyDeprecationWarning: ARC4 has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.ARC4 and will be removed from cryptography.hazmat.primitives.ciphers.algorithms in 48.0.0.\n",
      "  from cryptography.hazmat.primitives.ciphers.algorithms import AES, ARC4\n",
      "/Users/justpqa/advpai/venv/lib/python3.10/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "from gwas_table_extraction import *"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "95dff043",
   "metadata": {},
   "source": [
    "Import reference table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "124c09ef",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#dbSNP_hg38_chr</th>\n",
       "      <th>dbSNP_hg38_position</th>\n",
       "      <th>Top SNP</th>\n",
       "      <th>P-value</th>\n",
       "      <th>LocusName</th>\n",
       "      <th>RA 1(Reported Allele 1)</th>\n",
       "      <th>nonref_allele</th>\n",
       "      <th>nonref_effect</th>\n",
       "      <th>OR_nonref</th>\n",
       "      <th>nearest_gene_symb</th>\n",
       "      <th>Study type</th>\n",
       "      <th>Study Design</th>\n",
       "      <th>Pubmed PMID</th>\n",
       "      <th>Population_map</th>\n",
       "      <th>Cohort_simple3</th>\n",
       "      <th>Sample size</th>\n",
       "      <th>Analysis group</th>\n",
       "      <th>Phenotype</th>\n",
       "      <th>Phenotype-derived</th>\n",
       "      <th>most_severe_consequence</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr1</td>\n",
       "      <td>6434683</td>\n",
       "      <td>rs12074379</td>\n",
       "      <td>0.00726</td>\n",
       "      <td>ESPN</td>\n",
       "      <td>T</td>\n",
       "      <td>T</td>\n",
       "      <td>NR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ESPN</td>\n",
       "      <td>SNP-based</td>\n",
       "      <td>Disease risk</td>\n",
       "      <td>30636644</td>\n",
       "      <td>Caucasian</td>\n",
       "      <td>ADGC, CHS, CHARGE, HRS</td>\n",
       "      <td>10191</td>\n",
       "      <td>Plan 3 (only females)</td>\n",
       "      <td>AD</td>\n",
       "      <td>AD</td>\n",
       "      <td>intron_variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr1</td>\n",
       "      <td>6434683</td>\n",
       "      <td>rs12074379</td>\n",
       "      <td>8.51E-40</td>\n",
       "      <td>NR</td>\n",
       "      <td>T</td>\n",
       "      <td>T</td>\n",
       "      <td>NR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ESPN</td>\n",
       "      <td>SNP-based</td>\n",
       "      <td>eQTL</td>\n",
       "      <td>30636644</td>\n",
       "      <td>Caucasian</td>\n",
       "      <td>ADGC, CHS, CHARGE, HRS</td>\n",
       "      <td>10191</td>\n",
       "      <td>Plan 3 (only females)</td>\n",
       "      <td>ESPN (ILMN_1806710) expression</td>\n",
       "      <td>Expression</td>\n",
       "      <td>intron_variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr1</td>\n",
       "      <td>8708071</td>\n",
       "      <td>rs112053331</td>\n",
       "      <td>0.0009</td>\n",
       "      <td>RERE</td>\n",
       "      <td>NR</td>\n",
       "      <td>NR</td>\n",
       "      <td>NR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>RERE</td>\n",
       "      <td>SNP-based</td>\n",
       "      <td>Cross phenotype</td>\n",
       "      <td>30010129</td>\n",
       "      <td>Caucasian</td>\n",
       "      <td>IGAP</td>\n",
       "      <td>54162</td>\n",
       "      <td>All</td>\n",
       "      <td>AD</td>\n",
       "      <td>AD</td>\n",
       "      <td>intron_variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr1</td>\n",
       "      <td>8708071</td>\n",
       "      <td>rs112053331</td>\n",
       "      <td>0.08392</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NR</td>\n",
       "      <td>NR</td>\n",
       "      <td>NR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>RERE</td>\n",
       "      <td>Gene-based</td>\n",
       "      <td>Cross phenotype</td>\n",
       "      <td>30010129</td>\n",
       "      <td>Caucasian</td>\n",
       "      <td>IGAP</td>\n",
       "      <td>54162</td>\n",
       "      <td>All</td>\n",
       "      <td>AD</td>\n",
       "      <td>AD</td>\n",
       "      <td>intron_variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr1</td>\n",
       "      <td>11487007</td>\n",
       "      <td>rs2379135</td>\n",
       "      <td>0.0156</td>\n",
       "      <td>PTCHD2</td>\n",
       "      <td>NR</td>\n",
       "      <td>NR</td>\n",
       "      <td>NR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DISP3</td>\n",
       "      <td>SNP-based</td>\n",
       "      <td>Endophenotype</td>\n",
       "      <td>22245343</td>\n",
       "      <td>Caucasian</td>\n",
       "      <td>ADNI</td>\n",
       "      <td>757</td>\n",
       "      <td>All</td>\n",
       "      <td>MRI</td>\n",
       "      <td>Imaging</td>\n",
       "      <td>intron_variant</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  #dbSNP_hg38_chr  dbSNP_hg38_position      Top SNP   P-value LocusName  \\\n",
       "0            chr1              6434683   rs12074379   0.00726      ESPN   \n",
       "1            chr1              6434683   rs12074379  8.51E-40        NR   \n",
       "2            chr1              8708071  rs112053331    0.0009      RERE   \n",
       "3            chr1              8708071  rs112053331   0.08392       NaN   \n",
       "4            chr1             11487007    rs2379135    0.0156    PTCHD2   \n",
       "\n",
       "  RA 1(Reported Allele 1) nonref_allele nonref_effect  OR_nonref  \\\n",
       "0                       T             T            NR        NaN   \n",
       "1                       T             T            NR        NaN   \n",
       "2                      NR            NR            NR        NaN   \n",
       "3                      NR            NR            NR        NaN   \n",
       "4                      NR            NR            NR        NaN   \n",
       "\n",
       "  nearest_gene_symb  Study type      Study Design  Pubmed PMID Population_map  \\\n",
       "0              ESPN   SNP-based      Disease risk     30636644      Caucasian   \n",
       "1              ESPN   SNP-based              eQTL     30636644      Caucasian   \n",
       "2              RERE   SNP-based  Cross phenotype      30010129      Caucasian   \n",
       "3              RERE  Gene-based  Cross phenotype      30010129      Caucasian   \n",
       "4             DISP3   SNP-based     Endophenotype     22245343      Caucasian   \n",
       "\n",
       "           Cohort_simple3 Sample size         Analysis group  \\\n",
       "0  ADGC, CHS, CHARGE, HRS       10191  Plan 3 (only females)   \n",
       "1  ADGC, CHS, CHARGE, HRS       10191  Plan 3 (only females)   \n",
       "2                    IGAP       54162                    All   \n",
       "3                    IGAP       54162                    All   \n",
       "4                    ADNI         757                    All   \n",
       "\n",
       "                        Phenotype Phenotype-derived most_severe_consequence  \n",
       "0                              AD                AD          intron_variant  \n",
       "1  ESPN (ILMN_1806710) expression        Expression          intron_variant  \n",
       "2                              AD                AD          intron_variant  \n",
       "3                              AD                AD          intron_variant  \n",
       "4                             MRI           Imaging          intron_variant  "
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "advp = pd.read_csv(\"advp.variant.records.hg38.tsv\", sep='\\t')\n",
    "advp.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "25882935",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "RangeIndex: 6346 entries, 0 to 6345\n",
      "Data columns (total 20 columns):\n",
      " #   Column                   Non-Null Count  Dtype  \n",
      "---  ------                   --------------  -----  \n",
      " 0   #dbSNP_hg38_chr          6346 non-null   object \n",
      " 1   dbSNP_hg38_position      6346 non-null   int64  \n",
      " 2   Top SNP                  6346 non-null   object \n",
      " 3   P-value                  6338 non-null   object \n",
      " 4   LocusName                6244 non-null   object \n",
      " 5   RA 1(Reported Allele 1)  6222 non-null   object \n",
      " 6   nonref_allele            6222 non-null   object \n",
      " 7   nonref_effect            6224 non-null   object \n",
      " 8   OR_nonref                3390 non-null   float64\n",
      " 9   nearest_gene_symb        6336 non-null   object \n",
      " 10  Study type               6346 non-null   object \n",
      " 11  Study Design             6346 non-null   object \n",
      " 12  Pubmed PMID              6346 non-null   int64  \n",
      " 13  Population_map           6346 non-null   object \n",
      " 14  Cohort_simple3           6346 non-null   object \n",
      " 15  Sample size              6346 non-null   object \n",
      " 16  Analysis group           6322 non-null   object \n",
      " 17  Phenotype                6346 non-null   object \n",
      " 18  Phenotype-derived        6346 non-null   object \n",
      " 19  most_severe_consequence  6332 non-null   object \n",
      "dtypes: float64(1), int64(2), object(17)\n",
      "memory usage: 991.7+ KB\n"
     ]
    }
   ],
   "source": [
    "advp.info()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "883238a2",
   "metadata": {},
   "source": [
    "Test papers along with their PMC and PMID\n",
    "- 1-s2.0-S0197458018303816-main.pdf - 30448613 - PMC6331247\n",
    "- 1-s2.0-S0197458019300727-main.pdf - 30979435 - PMC6783343\n",
    "- s11357-019-00071-5.pdf - 31055733 - PMC6544706\n",
    "- s41588-018-0311-9.pdf - 30617256 - PMC6836675\n",
    "- s41588-019-0358-2.pdf - 30820047 - PMC6463297"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "4b5070a9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Not accurate extraction for test_papers/1-s2.0-S0197458018303816-main.pdf\n",
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Error while extracting tables from test_papers/s11357-019-00071-5.pdf\n",
      "Not accurate extraction for test_papers/s11357-019-00071-5.pdf\n",
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Not accurate extraction for test_papers/s41588-019-0358-2.pdf\n"
     ]
    }
   ],
   "source": [
    "# paper file - PMID - PMCID - table id\n",
    "test_papers_info = [\n",
    "    (\"test_papers/1-s2.0-S0197458018303816-main.pdf\", 30448613, \"PMC6331247\", [1, 2, 3]),\n",
    "    (\"test_papers/1-s2.0-S0197458019300727-main.pdf\", 30979435, \"PMC6783343\", [1]),\n",
    "    (\"test_papers/s11357-019-00071-5.pdf\", 31055733, \"PMC6544706\", [1, 3, 4]), # got error in PMC\n",
    "    (\"test_papers/s41588-018-0311-9.pdf\", 30617256, \"PMC6836675\", [1]),\n",
    "    (\"test_papers/s41588-019-0358-2.pdf\", 30820047, \"PMC6463297\", [1, 2])\n",
    "]\n",
    "\n",
    "for file_name, pmid, pmcid, inx_lst in test_papers_info:\n",
    "    try:\n",
    "        df_lst = extract_tables_lst_from_paper(pmcid, file_name)\n",
    "    except:\n",
    "        print(f\"Error while extracting tables from {file_name}\")\n",
    "    try:\n",
    "        for i in inx_lst:\n",
    "            df_lst[i-1].to_csv(f\"test_tables/{file_name.split('/')[-1].replace('.pdf', '')}_table_{i}.csv\", index=False)\n",
    "    except:\n",
    "        print(f\"Not accurate extraction for {file_name}\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.19"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
