{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "ad359987",
   "metadata": {},
   "source": [
    "### Test on ADVP 1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "33002256",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/justpqa/advpai/venv/lib/python3.10/site-packages/pypdf/_crypt_providers/_cryptography.py:32: CryptographyDeprecationWarning: ARC4 has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.ARC4 and will be removed from cryptography.hazmat.primitives.ciphers.algorithms in 48.0.0.\n",
      "  from cryptography.hazmat.primitives.ciphers.algorithms import AES, ARC4\n",
      "/Users/justpqa/advpai/venv/lib/python3.10/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "from gwas_table_extraction import *"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "95dff043",
   "metadata": {},
   "source": [
    "Import reference table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "124c09ef",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#dbSNP_hg38_chr</th>\n",
       "      <th>dbSNP_hg38_position</th>\n",
       "      <th>Top SNP</th>\n",
       "      <th>P-value</th>\n",
       "      <th>LocusName</th>\n",
       "      <th>RA 1(Reported Allele 1)</th>\n",
       "      <th>nonref_allele</th>\n",
       "      <th>nonref_effect</th>\n",
       "      <th>OR_nonref</th>\n",
       "      <th>nearest_gene_symb</th>\n",
       "      <th>Study type</th>\n",
       "      <th>Study Design</th>\n",
       "      <th>Pubmed PMID</th>\n",
       "      <th>Population_map</th>\n",
       "      <th>Cohort_simple3</th>\n",
       "      <th>Sample size</th>\n",
       "      <th>Analysis group</th>\n",
       "      <th>Phenotype</th>\n",
       "      <th>Phenotype-derived</th>\n",
       "      <th>most_severe_consequence</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr1</td>\n",
       "      <td>6434683</td>\n",
       "      <td>rs12074379</td>\n",
       "      <td>0.00726</td>\n",
       "      <td>ESPN</td>\n",
       "      <td>T</td>\n",
       "      <td>T</td>\n",
       "      <td>NR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ESPN</td>\n",
       "      <td>SNP-based</td>\n",
       "      <td>Disease risk</td>\n",
       "      <td>30636644</td>\n",
       "      <td>Caucasian</td>\n",
       "      <td>ADGC, CHS, CHARGE, HRS</td>\n",
       "      <td>10191</td>\n",
       "      <td>Plan 3 (only females)</td>\n",
       "      <td>AD</td>\n",
       "      <td>AD</td>\n",
       "      <td>intron_variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr1</td>\n",
       "      <td>6434683</td>\n",
       "      <td>rs12074379</td>\n",
       "      <td>8.51E-40</td>\n",
       "      <td>NR</td>\n",
       "      <td>T</td>\n",
       "      <td>T</td>\n",
       "      <td>NR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ESPN</td>\n",
       "      <td>SNP-based</td>\n",
       "      <td>eQTL</td>\n",
       "      <td>30636644</td>\n",
       "      <td>Caucasian</td>\n",
       "      <td>ADGC, CHS, CHARGE, HRS</td>\n",
       "      <td>10191</td>\n",
       "      <td>Plan 3 (only females)</td>\n",
       "      <td>ESPN (ILMN_1806710) expression</td>\n",
       "      <td>Expression</td>\n",
       "      <td>intron_variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr1</td>\n",
       "      <td>8708071</td>\n",
       "      <td>rs112053331</td>\n",
       "      <td>0.0009</td>\n",
       "      <td>RERE</td>\n",
       "      <td>NR</td>\n",
       "      <td>NR</td>\n",
       "      <td>NR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>RERE</td>\n",
       "      <td>SNP-based</td>\n",
       "      <td>Cross phenotype</td>\n",
       "      <td>30010129</td>\n",
       "      <td>Caucasian</td>\n",
       "      <td>IGAP</td>\n",
       "      <td>54162</td>\n",
       "      <td>All</td>\n",
       "      <td>AD</td>\n",
       "      <td>AD</td>\n",
       "      <td>intron_variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr1</td>\n",
       "      <td>8708071</td>\n",
       "      <td>rs112053331</td>\n",
       "      <td>0.08392</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NR</td>\n",
       "      <td>NR</td>\n",
       "      <td>NR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>RERE</td>\n",
       "      <td>Gene-based</td>\n",
       "      <td>Cross phenotype</td>\n",
       "      <td>30010129</td>\n",
       "      <td>Caucasian</td>\n",
       "      <td>IGAP</td>\n",
       "      <td>54162</td>\n",
       "      <td>All</td>\n",
       "      <td>AD</td>\n",
       "      <td>AD</td>\n",
       "      <td>intron_variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr1</td>\n",
       "      <td>11487007</td>\n",
       "      <td>rs2379135</td>\n",
       "      <td>0.0156</td>\n",
       "      <td>PTCHD2</td>\n",
       "      <td>NR</td>\n",
       "      <td>NR</td>\n",
       "      <td>NR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DISP3</td>\n",
       "      <td>SNP-based</td>\n",
       "      <td>Endophenotype</td>\n",
       "      <td>22245343</td>\n",
       "      <td>Caucasian</td>\n",
       "      <td>ADNI</td>\n",
       "      <td>757</td>\n",
       "      <td>All</td>\n",
       "      <td>MRI</td>\n",
       "      <td>Imaging</td>\n",
       "      <td>intron_variant</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  #dbSNP_hg38_chr  dbSNP_hg38_position      Top SNP   P-value LocusName  \\\n",
       "0            chr1              6434683   rs12074379   0.00726      ESPN   \n",
       "1            chr1              6434683   rs12074379  8.51E-40        NR   \n",
       "2            chr1              8708071  rs112053331    0.0009      RERE   \n",
       "3            chr1              8708071  rs112053331   0.08392       NaN   \n",
       "4            chr1             11487007    rs2379135    0.0156    PTCHD2   \n",
       "\n",
       "  RA 1(Reported Allele 1) nonref_allele nonref_effect  OR_nonref  \\\n",
       "0                       T             T            NR        NaN   \n",
       "1                       T             T            NR        NaN   \n",
       "2                      NR            NR            NR        NaN   \n",
       "3                      NR            NR            NR        NaN   \n",
       "4                      NR            NR            NR        NaN   \n",
       "\n",
       "  nearest_gene_symb  Study type      Study Design  Pubmed PMID Population_map  \\\n",
       "0              ESPN   SNP-based      Disease risk     30636644      Caucasian   \n",
       "1              ESPN   SNP-based              eQTL     30636644      Caucasian   \n",
       "2              RERE   SNP-based  Cross phenotype      30010129      Caucasian   \n",
       "3              RERE  Gene-based  Cross phenotype      30010129      Caucasian   \n",
       "4             DISP3   SNP-based     Endophenotype     22245343      Caucasian   \n",
       "\n",
       "           Cohort_simple3 Sample size         Analysis group  \\\n",
       "0  ADGC, CHS, CHARGE, HRS       10191  Plan 3 (only females)   \n",
       "1  ADGC, CHS, CHARGE, HRS       10191  Plan 3 (only females)   \n",
       "2                    IGAP       54162                    All   \n",
       "3                    IGAP       54162                    All   \n",
       "4                    ADNI         757                    All   \n",
       "\n",
       "                        Phenotype Phenotype-derived most_severe_consequence  \n",
       "0                              AD                AD          intron_variant  \n",
       "1  ESPN (ILMN_1806710) expression        Expression          intron_variant  \n",
       "2                              AD                AD          intron_variant  \n",
       "3                              AD                AD          intron_variant  \n",
       "4                             MRI           Imaging          intron_variant  "
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "advp = pd.read_csv(\"advp.variant.records.hg38.tsv\", sep='\\t')\n",
    "advp.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "25882935",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "RangeIndex: 6346 entries, 0 to 6345\n",
      "Data columns (total 20 columns):\n",
      " #   Column                   Non-Null Count  Dtype  \n",
      "---  ------                   --------------  -----  \n",
      " 0   #dbSNP_hg38_chr          6346 non-null   object \n",
      " 1   dbSNP_hg38_position      6346 non-null   int64  \n",
      " 2   Top SNP                  6346 non-null   object \n",
      " 3   P-value                  6338 non-null   object \n",
      " 4   LocusName                6244 non-null   object \n",
      " 5   RA 1(Reported Allele 1)  6222 non-null   object \n",
      " 6   nonref_allele            6222 non-null   object \n",
      " 7   nonref_effect            6224 non-null   object \n",
      " 8   OR_nonref                3390 non-null   float64\n",
      " 9   nearest_gene_symb        6336 non-null   object \n",
      " 10  Study type               6346 non-null   object \n",
      " 11  Study Design             6346 non-null   object \n",
      " 12  Pubmed PMID              6346 non-null   int64  \n",
      " 13  Population_map           6346 non-null   object \n",
      " 14  Cohort_simple3           6346 non-null   object \n",
      " 15  Sample size              6346 non-null   object \n",
      " 16  Analysis group           6322 non-null   object \n",
      " 17  Phenotype                6346 non-null   object \n",
      " 18  Phenotype-derived        6346 non-null   object \n",
      " 19  most_severe_consequence  6332 non-null   object \n",
      "dtypes: float64(1), int64(2), object(17)\n",
      "memory usage: 991.7+ KB\n"
     ]
    }
   ],
   "source": [
    "advp.info()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "883238a2",
   "metadata": {},
   "source": [
    "Test papers along with their PMC and PMID\n",
    "- 1-s2.0-S0197458018303816-main.pdf - 30448613 - PMC6331247\n",
    "- 1-s2.0-S0197458019300727-main.pdf - 30979435 - PMC6783343\n",
    "- s11357-019-00071-5.pdf - 31055733 - PMC6544706\n",
    "- s41588-018-0311-9.pdf - 30617256 - PMC6836675\n",
    "- s41588-019-0358-2.pdf - 30820047 - PMC6463297"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "4b5070a9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "objc[2241]: Class AVFFrameReceiver is implemented in both /Users/justpqa/advpai/venv/lib/python3.10/site-packages/av/.dylibs/libavdevice.62.1.100.dylib (0x136d043a8) and /Users/justpqa/advpai/venv/lib/python3.10/site-packages/cv2/.dylibs/libavdevice.61.3.100.dylib (0x13ebcc3a8). This may cause spurious casting failures and mysterious crashes. One of the duplicates must be removed or renamed.\n",
      "objc[2241]: Class AVFAudioReceiver is implemented in both /Users/justpqa/advpai/venv/lib/python3.10/site-packages/av/.dylibs/libavdevice.62.1.100.dylib (0x136d043f8) and /Users/justpqa/advpai/venv/lib/python3.10/site-packages/cv2/.dylibs/libavdevice.61.3.100.dylib (0x13ebcc3f8). This may cause spurious casting failures and mysterious crashes. One of the duplicates must be removed or renamed.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Not accurate extraction for test_papers/1-s2.0-S0197458018303816-main.pdf\n",
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Error while extracting tables from test_papers/s11357-019-00071-5.pdf\n",
      "Not accurate extraction for test_papers/s11357-019-00071-5.pdf\n",
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Successfully retrieve number of columns\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n",
      "Usage of TableItem.export_to_dataframe() without `doc` argument is deprecated.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Not accurate extraction for test_papers/s41588-019-0358-2.pdf\n"
     ]
    }
   ],
   "source": [
    "# paper file - PMID - PMCID - table id\n",
    "test_papers_info = [\n",
    "    (\"test_papers/1-s2.0-S0197458018303816-main.pdf\", 30448613, \"PMC6331247\", [1, 2, 3]),\n",
    "    (\"test_papers/1-s2.0-S0197458019300727-main.pdf\", 30979435, \"PMC6783343\", [1]),\n",
    "    (\"test_papers/s11357-019-00071-5.pdf\", 31055733, \"PMC6544706\", [1, 3, 4]), # got error in PMC\n",
    "    (\"test_papers/s41588-018-0311-9.pdf\", 30617256, \"PMC6836675\", [1]),\n",
    "    (\"test_papers/s41588-019-0358-2.pdf\", 30820047, \"PMC6463297\", [1, 2])\n",
    "]\n",
    "\n",
    "for file_name, pmid, pmcid, inx_lst in test_papers_info:\n",
    "    try:\n",
    "        df_lst = extract_tables_lst_from_paper(pmcid, file_name)\n",
    "    except:\n",
    "        print(f\"Error while extracting tables from {file_name}\")\n",
    "    try:\n",
    "        for i in inx_lst:\n",
    "            df_lst[i-1].to_csv(f\"test_tables/{file_name.split('/')[-1].replace('.pdf', '')}_table_{i}.csv\", index=False)\n",
    "    except:\n",
    "        print(f\"Not accurate extraction for {file_name}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "dacbe3b7",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Asking to truncate to max_length but no maximum length is provided and the model has no predefined maximum length. Default to no truncation.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "s11357-019-00071-5_table_1.csv\n",
      "Locus: [('Gene', 'Gene: CR1, BIN1, OR2S2, MS4A6A, MS4A4E, ', 0.5280643701553345)]\n",
      "Chr: [('Chr', 'Chromosome number: 1, 2, 9, 11, 14, ', 0.6727339625358582)]\n",
      "P-value: [('p value.', 'P-value value: 4.70 /C2 10 /C0 6, 3.67 3 10 L 8, 8.45 3 10 L 7, 8.68 /C2 10 /C0 6, 2.10 /C2 10 /C0 6, ', 0.4075296223163605), ('Age 60 e 79 y (younger).p value', 'P-value value: 5.41 /C2 10 /C0 9, 6.06 3 10 L 14, 5.43 3 10 L 6, 1.08 /C2 10 /C0 5, 1.45 /C2 10 /C0 6, ', 0.4264649748802185), ('Age /C21 80 y (older).p value', 'P-value value: 0.027, 0.340, 0.363, 3.38 /C2 10 /C0 4, 5.53 /C2 10 /C0 4, ', 0.5158348083496094), ('Heterogeneity between the younger & older age groups.p value', 'P-value value: 0.159, 6.86 3 10 L 4, 0.0695, 0.587, 0.863, ', 0.5418775677680969)]\n",
      "SNP: [('Top SNP in gene.', 'top Single nucleotide polymorphism identifier in gene: rs3818361, rs6431219, rs12378268, rs1834550, rs184909761, ', 0.6773247122764587)]\n",
      "RA: [('A1/A2.', 'effect allele / tested allele/Other allele / non-effect allele: A/G, T/C, C/T, G/A, T/A, ', 0.6531670689582825)]\n",
      "AF: [('Age 60 e 79 y (younger).Frq', 'Frequency: 0.204, 0.419, 0.32, 0.389, 0.353, ', 0.4497758448123932), ('Age /C21 80 y (older).Frq', 'Frequency: 0.195, 0.408, 0.32, 0.402, 0.36, ', 0.4339182376861572)]\n",
      "Sample size: [('Age 60 e 79 y (younger).N', 'Total sample size: 14,867, 14,225, 14,185, 14,677, 12,169, ', 0.6539657115936279), ('Age /C21 80 y (older).N', 'Total sample size: 6554, 6341, 6339, 6473, 5340, ', 0.654046356678009)]\n",
      "Effect Size: [('Age 60 e 79 y (younger).OR (95% CI)', 'odds ratio (95% Confidence Interval): 1.21 (1.14, 1.29), 1.23 (1.17, 1.30), 1.14 (1.08, 1.21), 0.89 (0.84, 0.94), 0.86 (0.81, 0.92), ', 0.48151111602783203), ('Age /C21 80 y (older).OR (95% CI)', 'odds ratio (95% Confidence Interval): 1.12 (1.01, 1.23), 1.04 (0.96, 1.13), 0.86 (0.80, 0.94), 0.85 (0.78, 0.93), 0.98 (0.90, 1.06), ', 0.48938655853271484)]\n",
      "\n",
      "1-s2.0-S0197458019300727-main_table_1.csv\n",
      "Locus: [('Gene', 'Gene: CR1, BIN1, OR2S2, MS4A6A, MS4A4E, ', 0.5280643701553345)]\n",
      "Chr: [('Chr', 'Chromosome number: 1, 2, 9, 11, 14, ', 0.6727339625358582)]\n",
      "P-value: [('p value.', 'P-value value: 4.70 /C2 10 /C0 6, 3.67 3 10 L 8, 8.45 3 10 L 7, 8.68 /C2 10 /C0 6, 2.10 /C2 10 /C0 6, ', 0.4075296223163605), ('Age 60 e 79 y (younger).p value', 'P-value value: 5.41 /C2 10 /C0 9, 6.06 3 10 L 14, 5.43 3 10 L 6, 1.08 /C2 10 /C0 5, 1.45 /C2 10 /C0 6, ', 0.4264649748802185), ('Age /C21 80 y (older).p value', 'P-value value: 0.027, 0.340, 0.363, 3.38 /C2 10 /C0 4, 5.53 /C2 10 /C0 4, ', 0.5158348083496094), ('Heterogeneity between the younger & older age groups.p value', 'P-value value: 0.159, 6.86 3 10 L 4, 0.0695, 0.587, 0.863, ', 0.5418775677680969)]\n",
      "SNP: [('Top SNP in gene.', 'top Single nucleotide polymorphism identifier in gene: rs3818361, rs6431219, rs12378268, rs1834550, rs184909761, ', 0.6773247122764587)]\n",
      "RA: [('A1/A2.', 'effect allele / tested allele/Other allele / non-effect allele: A/G, T/C, C/T, G/A, T/A, ', 0.6531670689582825)]\n",
      "AF: [('Age 60 e 79 y (younger).Frq', 'Frequency: 0.204, 0.419, 0.32, 0.389, 0.353, ', 0.4497758448123932), ('Age /C21 80 y (older).Frq', 'Frequency: 0.195, 0.408, 0.32, 0.402, 0.36, ', 0.4339182376861572)]\n",
      "Sample size: [('Age 60 e 79 y (younger).N', 'Total sample size: 14,867, 14,225, 14,185, 14,677, 12,169, ', 0.6539657115936279), ('Age /C21 80 y (older).N', 'Total sample size: 6554, 6341, 6339, 6473, 5340, ', 0.654046356678009)]\n",
      "Effect Size: [('Age 60 e 79 y (younger).OR (95% CI)', 'odds ratio (95% Confidence Interval): 1.21 (1.14, 1.29), 1.23 (1.17, 1.30), 1.14 (1.08, 1.21), 0.89 (0.84, 0.94), 0.86 (0.81, 0.92), ', 0.48151111602783203), ('Age /C21 80 y (older).OR (95% CI)', 'odds ratio (95% Confidence Interval): 1.12 (1.01, 1.23), 1.04 (0.96, 1.13), 0.86 (0.80, 0.94), 0.85 (0.78, 0.93), 0.98 (0.90, 1.06), ', 0.48938655853271484)]\n",
      "\n",
      "s41588-019-0358-2_table_1.csv\n",
      "SNP: [('', ': rs4844610, rs6733839, rs10933431, rs9271058, rs75932628, ', 0.6114726066589355)]\n",
      "Chr: [('Chr..Chr.', 'chr: 1, 2, 6, 7, 8, ', 1)]\n",
      "Position: [('Position b.Position', 'Position: 207802552, 127892810, 233981912, 32575406, 41129252, ', 0.5059198141098022)]\n",
      "Locus: [('Closest gene c.Closest gene', 'Closest gene c: CR1, BIN1, INPP5D, HLA-DRB1, TREM2, ', 0.5145701169967651)]\n",
      "RA: [('Major/ minor alleles.Major/ minor alleles', 'Major/ minor alleles: C/A, C/T, C/G, T/A, A/C, ', 0.572628915309906)]\n",
      "AF: [('MAF d.MAF', 'maf: 0.187, 0.407, 0.223, 0.27, 0.008, ', 0.6273393630981445)]\n",
      "Sample size: [('stage 1 discovery ( n /thinspace = /thinspace63,926).OR', 'stage 1 discovery ( Total sample size /thinspace = /thinspace63,926): 1.16, 1.18, 0.9, 1.1, 2.01, ', 0.5167749524116516), ('stage 2 ( n /thinspace = /thinspace18,845).OR', 'stage 2 ( Total sample size /thinspace = /thinspace18,845): 1.20, 1.23, 0.92, 1.11, 2.50, ', 0.5263794660568237)]\n",
      "Confidence Interval: [('stage 1 discovery ( n /thinspace = /thinspace63,926).95%Ci', '95%confidence interval: 1.12-1.20, 1.15-1.22, 0.87-0.94, 1.06-1.14, 1.65-2.44, ', 0.7427425384521484), ('stage 2 ( n /thinspace = /thinspace18,845).95%Ci', '95%confidence interval: 1.13-1.27, 1.18-1.29, 0.87-0.97, 1.06-1.17, 1.56-4.00, ', 0.7503045797348022), ('Overall stage 1/thinspace + /thinspacestage 2 ( n /thinspace = /thinspace82,771).95%Ci', '95%confidence interval: 1.13-1.21, 1.17-1.23, 0.88-0.94, 1.07-1.13, 1.73-2.49, ', 0.733658492565155)]\n",
      "stage 1 discovery ( n /thinspace = /thinspace63,926).P: [('stage 1 discovery ( n /thinspace = /thinspace63,926).P', None, 1)]\n",
      "P-value: [('stage 2 ( n /thinspace = /thinspace18,845).P', 'p-value: 3.8 × /thinspace10 - 10, 2.0 × /thinspace10 - 18, 3.2 × /thinspace10 - 3, 5.7 × /thinspace10 - 5, 1.5/thinspace × /thinspace10 - 4, ', 0.40690258145332336), ('Overall stage 1/thinspace + /thinspacestage 2 ( n /thinspace = /thinspace82,771).Meta P', 'meta p-value: 3.6 × /thinspace10 - 24, 2.1 × /thinspace10 - 44, 3.4 × /thinspace10 - 9, 1.4 × /thinspace10 - 11, 2.7 × /thinspace10 - 15, ', 0.5404720902442932)]\n",
      "Effect Size: [('Overall stage 1/thinspace + /thinspacestage 2 ( n /thinspace = /thinspace82,771).OR', 'odds ratio: 1.17, 1.20, 0.91, 1.10, 2.08, ', 0.5248802900314331)]\n",
      "Overall stage 1/thinspace + /thinspacestage 2 ( n /thinspace = /thinspace82,771).I 2 (%), P: [('Overall stage 1/thinspace + /thinspacestage 2 ( n /thinspace = /thinspace82,771).I 2 (%), P', None, 1)]\n",
      "\n",
      "s41588-018-0311-9_table_1.csv\n",
      "Locus: [('Region.Locus', 'Locus: 1, 2, 3, 4, 5, ', 0.6077675819396973), ('Gene', 'Gene: ADAMTS4, CR1, BIN1, INPPD5, HESX1, ', 0.4769435226917267)]\n",
      "Chr: [('Chr', 'Chromosome number: 1, 2, 3, 4, 6, ', 0.6408736705780029)]\n",
      "SNP: [('Case-control status (phase 1).SNP', 'Single nucleotide polymorphism identifier: rs4575098, rs6656401, rs4663105, rs10933431, nan, ', 0.6476064920425415), ('AD-by-proxy (phase 2).SNP', 'Single nucleotide polymorphism identifier: rs4575098, rs679515, rs4663105, rs10933431, rs184384746, ', 0.6603267788887024), ('Overall (phase 3).SNP', 'Single nucleotide polymorphism identifier: rs4575098, rs2093760, rs4663105, rs10933431, rs184384746, ', 0.660340428352356)]\n",
      "Case-control status (phase 1).P: [('Case-control status (phase 1).P', None, 1)]\n",
      "AD-by-proxy (phase 2).P: [('AD-by-proxy (phase 2).P', None, 1)]\n",
      "Position: [('Overall (phase 3).bp', 'Base-pair position: 161155392, 207786828, 127891427, 233981912, 57226150, ', 0.6496379375457764)]\n",
      "RA: [('A1', 'Effect allele / tested allele: A, C, G, T, ', 0.6727954149246216), ('A2', 'Other allele / non-effect allele: G, A, C, T, ', 0.6641621589660645)]\n",
      "AF: [('MAF', 'Minor Allele Frequency: 0.240, 0.205, 0.415, 0.235, 0.002, ', 0.6490474939346313)]\n",
      "Z: [('Z', 'Z-score statistic: 6.36, 8.82, 13.94, - 6.13, 5.69, ', 1)]\n",
      "P: [('P', 'P-value: 2.05 /thinspace × 10 - 10, 1.10 /thinspace × 10 - 18, 3.38 /thinspace × 10 - 44, 8.92 /thinspace × 10 - 10, 1.24 /thinspace × 10 - 8, ', 1)]\n",
      "Direction: [('Direction', 'Direction: ? +++, ++++, ?---, ??? +, ? + - +, ', 1)]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "referencing_col_df = pd.read_csv(\"Rules for harmonizing ADVP papers - Main cols.csv\")\n",
    "referencing_col_df[\"column_with_context\"] = referencing_col_df.apply(lambda x: x[\"column\"] if pd.isna(x[\"description\"]) else x[\"column\"] + \": \" + x[\"description\"], axis = 1)\n",
    "\n",
    "gwas_column_matching_engine = GWASColumnMatchingEngine(referencing_col_df)\n",
    "\n",
    "for file in os.listdir(\"./test_tables\"):\n",
    "    if \"table\" in file and \".csv\" in file and \"harmonized\" not in file:\n",
    "        print(file)\n",
    "        df = pd.read_csv(f\"./test_tables/{file}\")\n",
    "        df.columns = ['' if 'Unnamed:' in col else col for col in df.columns]\n",
    "        col_to_ref_col = gwas_column_matching_engine.match_many_col_to_ref_col(df)\n",
    "        for ref_col in col_to_ref_col:\n",
    "            print(f\"{ref_col}: {col_to_ref_col[ref_col]}\")\n",
    "        print()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3e09de28",
   "metadata": {},
   "outputs": [],
   "source": [
    "modified_df_all = None\n",
    "for file in os.listdir(\"./test_tables\"):\n",
    "    if \".csv\" in file and \"table\" in file and \"harmonized\" not in file:\n",
    "        df = pd.read_csv(f\"./test_tables/{file}\")\n",
    "        df.columns = ['' if 'Unnamed:' in col else col for col in df.columns]\n",
    "        modified_df = format_original_table(df, gwas_column_matching_engine, remove_unique_col = False)\n",
    "        modified_df[\"file_name\"] = file\n",
    "        if modified_df_all is None:\n",
    "            modified_df_all = modified_df.copy()\n",
    "        else:\n",
    "            modified_df_all = pd.concat([modified_df_all, modified_df], ignore_index = True)\n",
    "        modified_df.to_csv(f\"./harmonized_test_tables/{file.replace('.csv', '')}_harmonized.csv\", index = False)\n",
    "modified_df_all.to_csv(\"./harmonized_test_tables/harmonized_table.csv\", index = False)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.19"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
